# Introduction {#chapter01}

## Denitrification and the nitrogen cycle

Many chemical elements of biological importance are cycled through the biosphere by a combination of physical and biological processes, in what are termed biogeochemical cycles.
One of these cycles is the nitrogen cycle shown schematically in Figure 1.1.

```{r fig1-1, fig.cap="The nitrogen cycle. The oxidation state of nitrogen in each intermediate is given in brackets. Denitrification is shown by black arrows, nitrogen fixation by a dark grey arrow and all other reactions by light grey arrows. Note that nitrification involves two reactions, oxidation of ammonia to nitrite and of nitrite to nitrate, but these have never been found together in the same organism. ANAMOX (anaerobic ammonia oxidation) is a poorly-characterised process in which nitrite and ammonia combine to give water and dinitrogen [@society_for_general_microbiology_nitrogen_1992].", fig.align="center", echo=FALSE, out.width="50%"}
include_graphics(path = "figure/fig1.1.jpg")
```

The cycle involves the interconversion between various oxidation states of nitrogen and is predominantly carried out by micro-organisms. A simplified view of the nitrogen cycle is one of a balance between two opposing processes. In nitrogen fixation atmospheric dinitrogen gas is reduced to ammonia, allowing the assimilation of nitrogen into biological molecules. Nitrogen is released back into the environment as ammonium during decay, oxidised to nitrate and converted to dinitrogen gas by the process of denitrification, the left hand branch of Figure 1.1. This involves the sequential reduction of nitrate to nitrite, nitric oxide, nitrous oxide and finally nitrogen, in a series of enzyme-catalysed reactions. The individual reactions of denitrification can be referred to as respiratory N-oxide reduction, because the process is used to generate ATP via a proton electrochemical gradient across the cytoplasmic membrane. The term dissimilatory N-oxide reduction is also used to indicate that the products are lost to the environment; this distinguishes the reactions from the assimilatory reductions used by plants and bacteria to incorporate nitrogen into biomass.

### The importance of denitrification

Denitrification was first reported last century [@gayon_recherches_1886] and it has since been the focus of much research, spanning a variety of disciplines. Factors that affect the levels of N-oxides in the environment are of great ecological interest. For example, nitrous oxide, the penultimate compound in the pathway, has been implicated as a contributor to global warming and as an agent capable of destroying stratospheric ozone [@rasmussen_atmospheric_1986]. Denitrification is a route by which soil nitrogen may be lost, with implications for agricultural productivity [@hauck_notitle_1981]. The removal of nitrogenous compounds in waste water treatment by bioremediation is an important field, particularly in small countries such as the Netherlands [@hutchins_biodegradation_1991], and the contribution of polluting nitrates in groundwater to eutrophication is another concern [@olson_fertilizers_1977]. Nitrate and nitrite levels in the environment may also have health consequences, through conditions such as methaemoglobinaemia [@levallois_contamination_1994] and certain stomach cancers (caused by conversion of nitrites to N-nitroso compounds in the stomach) [@calmels_characterization_1996]. Information about microbial N-oxide metabolism may therefore be relevant to many researchers.

In recent years denitrifying bacteria have proven to be extremely interesting systems for the study of fundamental prokaryotic biochemistry, physiology and molecular genetics, and this aspect of denitrification is the main concern of this thesis. As a bioenergetic process, the enzymes involved may be used to investigate electron transfer and proton translocation, both of which are poorly understood at present. Biogenesis of the enzymes, in terms of targeting and compartmentalisation, folding and assembly of their various metal centres is another fundamental process. The catalytic mechanisms of each enzyme also attract research, particularly as several of the proteins involved are soluble, easy to purify in large amounts and, in some cases, have had their crystal structures determined. Finally there is always the possibility that study of prokaryotic systems may provide structural models for analogous eukaryotic processes. As an example, the bacterial cytochrome _cd_$_1$ nitrite reductase (Section 1.5) generates bound nitric oxide which must be released from a haem group. In eukaryotes, nitric oxide participates in signal transduction by binding to a haem group on the enzyme guanyl cyclase, from which it must also be released [@bredt_nitric_1994].

## The denitrifying micro-organisms

Denitrification research has traditionally focused on the Gram-negative bacteria (those containing a compartment between the cytoplasmic and outer membranes termed the periplasm) and until quite recently, respiratory N-oxide reduction was considered to be an exclusively prokaryotic attribute. This situation has altered somewhat with the discovery that at least two species of fungus, _Fusarium oxysporum_ and _Cylindrocarpon tonkinense_ can reduce nitrite to nitrous oxide [@shoun_denitrification_1991; @usuda_denitrification_1995]. In _F. oxysporum_, N-oxide reduction is localised to the mitochondrion [@kobayashi_denitrification_1996] and is comprised of distinct enzymes for the dissimilatory reduction of nitrate [@usuda_denitrification_1995], nitrite [@kobayashi_copper-containing_1995] and nitric oxide [@nakahara_cytochrome_1993]. The system is significantly different to that found in bacteria, particularly the nitric oxide reductase which is a unique cytochrome P450-type enzyme.

The bacteria studied to date tend to fall into two groups; those in which at least one enzyme of the denitrification pathway has attracted interest, although the complete pathway has not been investigated (or may not be present), and those in which the complete denitrification pathway from nitrate through to dinitrogen has been characterised. _Escherichia coli_ falls into the former category; although not a denitrifier, it possesses a membrane-bound nitrate reductase [@enoch_role_1974] and a putative periplasmic nitrate reductase [@berks_napedabc_1995], encoded by the _nar_ and _nap_ gene clusters, respectively, which are essentially identical to those found in the denitrifying bacteria. The genetic amenability of this organism and its wider importance in biochemical and genetic research has encouraged investigation of these enzymes; for example extensive sequencing of the _E. coli_ genome allowed the identification of the _E. coli_ nap gene cluster during a DNA database search using the _Thiosphaera pantotropha nap_ genes [@richterich_u00008_1993].

Also in the first category are denitrifiers where one enzyme in the pathway has been singled out as being of specific interest. Examples include _Achromobacter cycloclastes_ and _Alcaligenes faecalis_ S-6; these organisms contain a copper nitrite reductase (Section 1.4.3), the X-ray crystal structures of which have been determined to high resolution [@godden_23_1991; @kukimoto_x-ray_1994]. This work has provided a great deal of information on copper centres and possible mechanisms of nitrite reduction.

However, the core of denitrification research concerns bacteria from the latter category; those in which the four enzymes required for complete denitrification have been identified and at least partially characterised. Many such species have been described [@zumft_denitrifying_1992] but the best studied are _Pseudomonas aeruginosa_, _Pseudomonas stutzeri_ and _Paracoccus denitrificans_ (and its close relatives, particularly _T. pantotropha_). _P. denitrificans_ has long been of interest in the field of bioenergetics, due to the close similarity between its aerobic respiratory chain and that of the eukaryotic mitochondrion [@john_paracoccus_1975]. _P. denitrificans_ has also been the subject of a great deal of metabolic research, due to its nutritional versatility. As well as growing anaerobically with nitrate as terminal electron acceptor, _P. denitrificans_ can grow chemoautotrophically on inorganic substrates such as hydrogen or thiosulphate, chemoheterotrophically on organic acids such as acetate or succinate and methylotrophically, on methanol or methylamine. As with other bacteria denitrifying growth in _P. denitrificans_ is an anaerobic process used as an alternative to aerobic respiration when oxygen, the preferred terminal electron acceptor, is in short supply.

## Organisation of the denitrifying electron transport chain

Although denitrification is relatively widespread amongst the prokaryotes, detailed biochemical information is available for only a small number of organisms. However, the current model of the organisation of the denitrifying respiratory chain is shown in Figure 1.2.

```{r fig1-2, fig.cap="The denitrifying electron transport chain of _Paracoccus denitrificans_. The enzymes shown are: NarGHI, nitrate reductase, FbcFBC, cytochrome _bc_$_1$ complex, NorBC, nitric oxide reductase, NirS, cyochrome _cd_$_1$ nitrite reductase, NosZ, nitrous oxide reductase. 'Q' represents the ubiquinol/ubiquinone pool. Chemical reactions are indicated by solid arrows, proton translocation by dashed arrows and electron transfer by dotted arrows, with the stoichiometry shown in each case. Proton translocation at the cytochrome _bc_$_1$ complex is described by the proton-motive Q-cycle (see text for details). Not shown is the periplasmic nitrate reductase, which is not present during anaerobic denitrifying growth conditions.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

This diagram is based on the organism _Paracoccus denitrificans_, but is probably similar in other bacteria, particularly _Ps. aeruginosa_ and _Ps. stutzeri_. During anaerobic denitrifying growth, a standard tricarboxylic acid cycle is employed, generating the reductants NADH and succinate. The cytoplasmic membrane thus contains prokaryotic homologues of the classical respiratory complexes: complex I (NADH-ubiquinone oxidoreductase), complex II (succinate-ubiquinone oxidoreductase) and complex III (ubiquinol-cytochrome _c_ oxidoreductase), in addition to the integral membrane proteins nitrate and nitric oxide reductase. Nitrate reductase is believed to accept electrons from the ubiquinol pool. In contrast, the other three N-oxide reductases are believed to receive electrons from the cytochrome _bc_$_1$ complex via small periplasmic electron transport proteins. Net proton translocation is achieved by the operation of the proton motive Q-cycle in the cytochrome _bc_$_1$ complex [@trumpower_cytochrome_1990] and the activity of Complex I. The reactions of N-oxide reduction are therefore periplasmic, with the exception of nitrate reduction which occurs in the cytoplasm [@ballard_respiratory_1988]; this implies a transporter for entry of nitrate and export of nitrite to the periplasm (Section 1.4.1).

Denitrification is an energy conserving process. Consideration of the reduction potentials associated with each step (Table 1.1) suggests that each reaction is relatively exergonic, yielding sufficient free energy to drive the synthesis of ATP.

```{r table11, echo=FALSE, results='asis'}
table1_1 <- data.frame(Reaction = c("$\\mathrm{2NO_3^- + 4H^+ + 4e^-}$ $\\rightarrow$ $\\mathrm{2NO_2^- + 2H_2O}$", "$\\mathrm{2NO_2^- + 2H^+ + 2e^-}$ $\\rightarrow$ $\\mathrm{2NO + 2H_2O}$", "$\\mathrm{2NO + 2H^+ + 2e^-}$ $\\rightarrow$ $\\mathrm{N_2O + H_2O}$", "$\\mathrm{N_2O + 2H^+ + 2e^-}$ $\\rightarrow$ $\\mathrm{N_2 + H_2O}$", "**Overall**", "$\\mathrm{2NO_3^- + 2H^+ + 5H_2}$ $\\rightarrow$ $\\mathrm{N_2 + 6H_2O}$", "$\\mathrm{O_2 + 4H^+ + 4e^-}$ $\\rightarrow$ $\\mathrm{2H_2O}$", "$\\mathrm{NAD^+ + 2H^+ + 2e^-}$ $\\rightarrow$ $\\mathrm{NADH + H^+}$"),
                       Couple = c("NO$_3^-$/NO$_2^-$", "NO$_2^-$/NO", "NO/N$_2$O", "N$_2$O/N$_2$",
                                  "", "", "O$_2$/H$_2$O", "NAD$^+$/NADH"),
                       `$\\Delta$_E_$^{o'}$ (V)` = c("+0.420", "+0.375", "+1.175", "+1.355",
                                                     "", "", "+0.800", "-0.320"),
                       `$\\Delta$_G_$^{o'}$ (kJ mol$^{-1}$) per reaction` = c("-285.6", "-134.1", 
                                                                              "-288.5", "-323.3", 
                                                                              "", "-1031.5", 
                                                                              "-438.5", ""),
                       `$\\Delta$_G_$^o$ (kJ mol$^{-1}$) per NADH` = c("-142.8", "-134.1", 
                                                                       "-288.5", "-323.3", 
                                                                       "", "-206.3", 
                                                                       "-219.2", ""),
                        check.names = FALSE)

kable(table1_1,
      caption = "Reduction potentials and thermodynamics of the reactions of denitrification compared with those of the oxygen/water couple, assuming NADH as the reductant.
The four reactions of denitrification are shown for the reduction of two molecules of nitrate; this is to balance the overall equation as two nitric oxide molecules are required to dimerise, forming one molecule of nitrous oxide. The reduction potentials of each couple under biological standard state conditions (1 atm pressure for gases, solutions at 1 M, pH 7.0, at 298 K) are shown. Assuming NADH to be the reductant for each reaction allows the free energy change to be calculated as shown in column four, using the equation $\\Delta$_G_°' = -nF$\\Delta$_E_°', where $\\Delta$_G_°' = Gibbs free energy change, n = number of electrons transferred, F = 96 500 C mol$^-1$ and $\\Delta$_E_°' = redox span between reductant and oxidant. This shows that the reduction of nitric and nitrous oxides is associated with a greater free energy change per reducing equivalent (two electrons) than that of oxygen to water. The overall denitrification reaction also has a comparable free energy change to that of oxygen reduction. However, as explained in the main text, standard state conditions are unlikely to occur in the cell and, additionally, the topology of the denitrifying electron transport chain results in a lower ATP:2e$^-$ stoichiometry than that of the aerobic respiratory chain.")
```

With NADH as electron donor (<i>E</i>$^o$' for the NADH/NAD$^+$ couple is -320 mV), the redox span to nitric and nitrous oxide is actually greater than that for the oxygen/water couple. However, it must be born in mind that these values refer to standard state conditions and will be different in the cell; for instance the actual concentration of N$_2$O will be rather lower than one atmosphere. Furthermore, the spatial arrangement of the components involved in the electron transport sequence influences the stoichiometry of net proton translocation per pair of electrons passing through the electron transport chain. Termination of the denitrifying electron transport chain with nitrous oxide reductase results in no net proton translocation across the redox span from cytochrome _c_ to nitrous oxide, whereas oxygen reduction by the cytochrome _aa_$_3$ oxidase results in the equivalent of four protons translocated per pair of electrons. Thus, assuming a constant H$^+$:ATP ratio for the ATP synthase, nitrous oxide respiration gives a lower ATP:2e$^-$ ratio than does oxygen respiration. Nevertheless, denitrification represents an important alternative mode of respiration for those bacteria which have evolved the necessary enzymatic machinery, allowing them to exploit a wider range of environmental niches than obligate aerobes.

In the following sections, the enzymes of denitrification are discussed in more detail. The order of discussion is the same as that of each reaction in the pathway, with the exception of the cytochrome _cd_$_1$ nitrite reductase; as this enzyme is the main focus of the project it is treated in greater detail after the others.

## Enzymes of the denitrifying electron transport chain

### The membrane-bound nitrate reductase

Membrane-bound nitrate reductase (Nar) is widely distributed, being found in many enteric bacteria as well as the denitrifying species. The most extensively characterised examples are from _E. coli_ [@clegg_purification_1976] and _P. denitrificans_ [@ballard_molecular_1987]. Nitrate reductase from these sources is composed of three subunits with molecular masses of around 140 kDa ($\alpha$), 60 kDa ($\beta$) and 25 kDa ($\gamma$), as judged by SDS-PAGE. The $\gamma$ subunit anchors the other two subunits to the cytoplasmic side of the cytoplasmic membrane and an $\alpha\beta$ complex can be released from inside-out membrane vesicles by washing with EDTA or magnesium-free buffer [@ballard_respiratory_1988]. Loss of the $\gamma$ subunit during purification is accompanied by an apparent proteolytic cleavage at the C-terminus of the $\beta$ subunit [@demoss_characterization_1981]. The inference from these studies is that the a$\beta$ complex contacts the $\gamma$ subunit through a region near the carboxy end of the $\beta$ subunit polypeptide chain. Stoichiometry of the subunits is uncertain, but is thought most likely to be $\alpha1\beta1\gamma1$; in _E. coli_ the genes appear to have overlapping start and stop codons, which would result in translational coupling [@oppenheim_translational_1980] and a stoichiometric synthesis of the subunits.

Membrane-bound nitrate reductase is a quinol oxidase, receiving from the quinol pool the two electrons required to reduce a molecule of nitrate. The electrons are transported through the complex by a variety of cofactors. The $\gamma$ subunit contains two _b_-type haems [@ballard_respiratory_1988]; modelling studies suggest that these conduct electrons from ubiquinol across the cytoplasmic membrane, leaving two protons in the periplasm in a classical “redox loop” mechanism [@berks_sequence_1995]. The $\alpha\beta$ complex contains several iron-sulphur clusters [@johnson_iron-sulfur_1985] and the $\alpha$ subunit contains in addition a complex cofactor termed molybdopterin guanine dinucleotide (MGD) [@wootton_enzymes_1991]. All bacterial nitrate reductases contain this cofactor, a variant of molybdopterin which in this case is conjugated to guanosine monophosphate. MGD-dependent enzymes form a large family, whose members include soluble and membrane-bound nitrate reductases, formate dehydrogenases and S-oxide reductases. Amino acid similarities between these sequences have been used to establish a number of sub-families and to identify conserved regions involved in the binding of metal cofactors [@berks_enzymes_1995].

A great deal of spectroscopic data has been amassed for the MGD cofactor, which receives electrons from the neighbouring $\beta$ subunit, binds nitrate and catalyses an oxo-transfer reaction between nitrate and water [@craig_reduction_1989]. EPR spectroscopy and mechanistic studies indicate that the molybdenum atom cycles between Mo(IV) and Mo(VI) states and is the site of the oxo-transfer reaction [@cramer_molybdenum_1983]. The complexity of the cofactor, in terms of the organic dithiolene moiety, is argued to provide a specific binding site for the Mo atom [@bellion_biological_1992]. This has been confirmed by two recent crystal structures of the enzymes dimethyl sulphoxide reductase [@schindelin_crystal_1996] and formate dehydrogenase H [@boyington_crystal_1997]. Both of these structures show that the molybdenum ion is co-ordinated by two MGD molecules through the dithiolene groups.

The structural genes for the three subunits of the complex, _narGHI_, have been cloned and sequenced from _E. coli_. A fourth gene, _narJ_, is found in this operon which encodes a cytoplasmic protein thought to be involved in biosynthesis of the complex [sodergren_nari_1988]. This has been referred to as the $\delta$ subunit but does not copurify with the enzyme. The T. pantotropha _narJ_ and _narHI_ genes have also been sequenced [@berks_sequence_1995]. In _E. coli_, a second nitrate reductase operon, _narZYWV_, arising from gene duplication, is also found [@blasco_nitrate_1990]. The _E. coli narGHIJ_ operon is also linked to three other genes nar genes, _narXL_, a two-component nitrate/nitrite sensing regulator [@nohno_narx_1989], Section 1.7.3 and _narK_, a putative nitrite exporter [@rowe_nark_1994], or nitrate/nitrite antiporter [@demoss_nark_1991]. Nitrate reduction is cytoplasmic; in _E. coli_ the product nitrite may be reduced to ammonia by a cytoplasmic sirohaem-containing enzyme [@peakman_nucleotide_1990] or under some circumstances, exported to the periplasm by _narK_ and reduced there by a _c_ haem-containing nitrite reductase [@kajie_purification_1986]. A similar system for nitrite export must operate in the denitrifying bacteria and has often assumed to be a nitrate/nitrite antiporter. No such protein has yet been identified in denitrifying bacteria, though other transport systems have been identified in prokaryotes In the cyanobacteria these include an active transporter of the ABC type [@omata_structure_1995] and a sodium-linked symporter [@rodriguez_nitrate_1992], both of which may transport nitrate and nitrite, although nitrate reduction in these organisms is assimilatory rather than respiratory.

### The periplasmic nitrate reductase

Periplasmic nitrate reductases have been purified from several bacteria, including _R. capsulatus_ [@mcewan_periplasmic_1987], _T. pantotropha_ [@berks_purification_1994] and _A. eutrophus_ [@siddiqui_structure_1993]. In addition, the complete gene locus _nap_ has been mapped and sequenced from _T. pantotropha_ [@berks_napedabc_1995], _R. sphaeroides_ [@reyes_isolation_1996] and _E. coli_ [@richterich_u00008_1993]. Although purified from a relatively small number of species the enzyme is probably widespread amongst the _Proteobacteria_; probes for the gene have been used to isolate up to forty species of bacteria from soil and water environments in one locale [@carter_soil_1995].

The enzyme is a heterodimer of two subunits, NapA (90 kDa) and NapB (16 kDa) [@berks_purification_1994]. NapB is a dihaem cytochrome _c_$_{552}$, which is believed to accept electrons from ubiquinol, via a quinol oxidase complex (possibly the _napE_ gene product) and a tetrahaem cytochrome _c_ (encoded by _napC_, [@berks_napedabc_1995]). NapA is the catalytic site, containing the MGD cofactor and, at least in _T. pantotropha_, an iron-sulphur cluster [@breton_characterization_1994]. The enzyme is catalytically distinct from the membrane-bound nitrate reductase, in that it is not competitively inhibited by azide, nor can it reduce chlorate or bromate [@berks_purification_1994].

Extensive sequencing of the _nap_ loci in _T. pantotropha_, _R. sphaeroides_ and _E. coli_ (the _aeg-46.5_ locus) has revealed a number of open reading frames besides those of the structural genes. _T. pantotropha_ contains the aforementioned _napE_ and _napC_, as well as _napD_, coding for a cytoplasmic protein of unknown function. _E. coli_ contains _napF_, _napG_ and _napH_; these proteins have been suggested to bind iron-sulphur clusters and to be involved in electron transport to NapAB [@berks_enzymes_1995]. In addition, there are a number of genes downstream of the _nap_ cluster involved in _c_-type cytochrome biogenesis [@thony-meyer_bacterial_1994] and _narP_, a nitrate/nitrite response regulator (Section 1.7.3).

The physiological role of the periplasmic nitrate reductase is not especially clear and may even vary in different organisms. In _T. pantotropha_, the enzyme was initially assigned a role in aerobic denitrification, a feature that originally distinguished this organism from the very closely related _P. denitrificans_ [@stouthamer_metabolic_1992]. One rationale for this is that oxygen is known to inhibit nitrate transport to the cytoplasm [@noji_molecular_1987], the site of reduction by the membrane-bound reductase, whereas a periplasmic location would effectively bypass this route of inhibition. However, more recent research seems to suggest that _T. pantotropha_ does not denitrify aerobically to the extent previously reported [@robertson_confirmation_1995], perhaps as a result of repeated sub-culturing in the laboratory. Furthermore, _P. denitrificans_ also possesses a periplasmic nitrate reductase and synthesis of the other enzymes of denitrification appears to be quite strictly controlled by anaerobiosis. However, the periplasmic nitrate reductase may be expressed under conditions of stress (such as entering stationary phase, [@siddiqui_structure_1993]) and as at least pre-synthesised cytochrome _cd_$_1$ nitrite reductase appears to persist on switching from anaerobic to aerobic growth [@baumann_dynamics_1996], the enzyme may play a role in aerobic denitrification as an adaptation to cope with rapidly changing levels of aeration in the environment.

Other roles have been suggested for the enzyme, involving the concepts of redox balance and redox poise. In photoheterotrophs and chemoheterotrophs, an optimum ratio of ubiquinol:ubiquinone is required to maintain turnover of enzyme complexes such as NADH dehydrogenase, the cytochrome _bc_$_1$ complex and the photosynthetic reaction centre [@berks_enzymes_1995]. Certain growth conditions such as low light intensity or the provision of highly reduced carbon substrate can lead to an increase in the UQH$_2$:UQ ratio and a fall in growth rate. The quinol oxidase activity of periplasmic nitrate reductase would alleviate this by consuming ubiquinol. There is good evidence for this model; in _T. pantotropha_, aerobic growth on the highly-reduced substrate butyrate results in a ten-fold increase in Nap expression compared to succinate [@richardson_influence_1992], whilst nitrate reduction in photosynthetic _R. capsulatus_ is markedly increased under low light intensity [@richardson_role_1988].


### The copper nitrite reductase

Copper nitrite reductase is one of two types of eubacterial dissimilatory nitrite reductase, the other being the cytochrome _cd_$_1$ enzyme discussed in Section 1.5. The two enzymes are distributed fairly evenly amongst bacteria with a slight preponderance of the cytochrome _cd_$_1$ enzyme [@averill_dissimilatory_1996]. There is no correlation between genus (or even species) and the type of nitrite reductase present; for example _Alcaligenes eutrophus_ H16 contains the _cd_$_1$ type [@sann_cytochrome_1994] as does _A. faecalis_ strain IAM 1015 [@iwasaki_cytochrome_1971], whereas _A. faecalis_ S-6 contains the copper enzyme [@kakutani_purification_1981]. The two enzymes are never found in the same organism but are functionally equivalent; expression of the copper nitrite reductase gene _nirK_ in a _Ps. stutzeri_ mutant lacking cytochrome _cd_$_1$ restores nitrite reductase activity [@glockner_copper-containing_1993].

Copper nitrite reductases have been purified from several Gram-negative bacteria, including _Achromobacter cycloclastes_ [@iwasaki_nitrite_1972], _Alcaligenes faecalis_ S-6 [@kakutani_purification_1981] and _Pseudomonas aureofaciens_ [@zumft_type_1987] and one Gram-positive bacterium, _Bacillus halodenitrificans_ [@denariaz_denitrifying_1991]. It now seems clear that all the enzymes are homotrimeric, with each 37 kDa subunit containing a variant of a type I centre and one type II copper centre. Type I centres are mononuclear, cysteine-ligated, have high redox potentials and are an intense blue colour (but see below). They are always involved in electron transfer, rather than catalysis. Type II copper centres contain a single copper atom, with spectroscopic properties similar to those of inorganic copper complexes. They are ligated predominantly by histidine residues and are involved with catalysis at the active site, where substrate or exogenous ligands can displace a water-derived ligand. The type I centre in the copper nitrite reductase from _A. cycloclastes_ is unusual in that it produces an intense green absorption at 458 nm; this is thought to be due to a short Cu-S-Met distance and substantial charge transfer through this bond [@han_resonance_1993].

Copper nitrite reductases are relatively well understood due to the elucidation of two X-ray crystal structures from the organisms _A. cycloclastes_ [@godden_23_1991] and _A. faecalis_ S-6 [@kukimoto_x-ray_1994]. These have revealed that each monomer forms two $\beta$ barrel domains stabilised by a helical region. The histidine, cysteine and methionine ligands to the copper centres have also been identified; the type II copper centres are positioned at the subunit interfaces and share ligands from adjacent subunits. These centres can be removed and reconstituted, showing them to be the site of nitrite reduction. Some site directed mutagenesis of copper nitrite reductase has been carried out, which has identified the important role of the copper ligands [@kukimoto_x-ray_1994], and the negatively-charged surface residues thought to interact with the electron donor pseudoazurin (Section 1.4.6 [@kukimoto_studies_1996]). A mechanism of nitrite reduction has been proposed which is quite similar to the cytochrome _cd_$_1$ enzyme (Section 1.5), involving protonation of the N-bound nitrite molecule by a neighbouring histidine residue, followed by dehydration to give a cuprous-NO+ species and followed by release of nitric oxide. The active site of copper nitrite reductase is comparable to that of carbonic anhydrase in that a number of similarly placed histidine residues act as proton donors [@strange_substrate-binding_1995]. However, there may be some mechanistic differences between the copper nitrite reductase and cytochrome _cd_$_1$ -type nitrite reductase. No evidence for a Cu(I)-NO+ species has been obtained by H${_2}^{18}$O exchange experiments using the copper nitrite reductase, [@hulse_evidence_1989] and the enzyme also seems to produce significant amounts of N$_2$O as judged by isotope exchange experiments [@jackson_evidence_1991]. This occurs when nitric oxide levels are quite high, implying reaction between a labile Cu-NO species and NO or NO$_2^-$. However, as with cytochrome _cd_$_1$, this may not occur physiologically, as NO is normally rapidly removed in the cell by the activity of nitric oxide reductase (see below).

### The nitric oxide reductase
For many years, it was unclear whether the product of nitrite reduction was nitric oxide or nitrous oxide and, if the former, whether nitric oxide was a free intermediate in denitrification. This situation has now been resolved by a number of methods: (1) trapping experiments using haemoglobin have shown nitric oxide to be freely diffusible [@goretski_trapping_1988], (2) membranes of mutants lacking nitrite reductase still possess a nitric oxide reductase activity [@zumft_defects_1988] and most conclusively, (3) a distinct nitric oxide reductase enzyme has been identified, purified and shown to be essential for denitrification by gene disruption. Nitric oxide reductase has now been purified from several sources, including _P. denitrificans_ [@carr_nitric_1990], _A. cycloclastes_ [@jones_nitric_1993] and _Ps. stutzeri_ [@heiss_formation_1989]. On SDS-PAGE gels two subunits are seen; a cytochrome _c_ of 17 kDa containing a single haem and a larger subunit (of apparent molecular mass 37 kDa, though actually 53 kDa from the gene sequence), binding a _b_-type haem. The enzyme also contains non-haem iron [@heiss_formation_1989]. Biochemical studies of the protein solubilised in detergent vesicles have been carried out; cytochrome _c_$_{550}$ can donate electrons to the enzyme and the K$_m$ for nitric oxide under these conditions is sub-micromolar [@carr_nitric_1990]. This measurement seems reasonable as NO concentrations in whole cells are generally 10$^{-7}$ M or less [@goretski_steady-state_1990].

Redox titrations have revealed only two redox components in the enzyme; the _c_ haem component (+225 mV in _P. denitrificans_, +280 mV in _Ps. stutzeri_) and the _b_ haem component (+305 mV or +322 mV in the same organisms respectively, Chan and Ferguson, unpublished work, [@kastrau_nitric_1994]). Early studies reported a stoichiometry of two haems per NorBC complex [@heiss_formation_1989], but more recent work indicates that there are two _b_-type haems (one high-spin that can bind carbon monoxide, the other low-spin), one _c_ haem and one non-haem iron per enzyme complex ([@fujiwara_cytochrome_1996; @girsch_purification_1997]). The membrane-bound nature of the enzyme makes purification quite difficult and it now seems that the earlier estimates of haem stoichiometry were erroneous.

The problems outlined above have made elucidation of a reaction mechanism for nitric oxide reductase quite problematic; in particular it was difficult to envisage how dimerisation of two NO molecules could be achieved at a single haem binding site. However, the cloning and sequencing of the _norBC_ genes has led to several suggestions for a catalytic mechanism. The NorB subunit has structural similarity to subunit I of cytochrome _c_ oxidase [@van_der_oost_heme-copper_1994], in that both are predicted to have 12 transmembrane helices, with 6 conserved histidines that act as ligands (in the oxidase, to haems _a_ and _a_$_3$ and to the Cu$_{A3}$ site, [@iwata_structure_1995]. Furthermore, like nitric oxide reductase the more primitive cytochrome oxidase, cytochrome _cbb_$_3$, has a subunit II (FixN) possessing a cytochrome _c_, rather than a binuclear Cu$_A$ centre [@saraste_cytochrome_1994]. These analogies have strengthened the case for two _b_-type haems and the conserved histidine residues strongly suggest liganding to the non-haem iron identified in later preparations of nitric oxide reductase. This raises the intriguing possibility of NO reduction at a binuclear haem-non-haem iron centre. Modelling of this enzyme family has also been used to suggest an evolutionary progression from nitric oxide reductase, through cytochrome _cbb_$_3$ to the cytochrome _c_ oxidases, and to imply that the cytochromes _cbb_$_3$ may have a different mechanism of proton pumping to the haem-copper enzymes, whilst nitric oxide reductase is probably not a proton pump [@van_der_oost_heme-copper_1994]. It is notable that cytochrome _c_ oxidase is capable of nitric oxide reduction [@zhao_cytochrome_1995] and a recent paper describing nitric oxide reductase from _T. pantotropha_ indicates that it has a cytochrome oxidase activity [@fujiwara_cytochrome_1996].

### The nitrous oxide reductase

Nitrous oxide reductase, the final enzyme of the denitrification pathway is one of the most comprehensively studied of the denitrifying enzymes. It has been purified from numerous sources, including _Ps. stutzeri_ [@coyle_nitrous_1985], _P. denitrificans_ [@snyder_purification_1987], _Ps. aeruginosa_ [@soohoo_purification_1991] and _A. faecalis_ [@matsubara_new-type_1972]. The enzyme appears essentially identical from all sources, being a homodimer of subunits each with a molecular mass of around 70 kDa. Nitrous oxide reductase is a soluble, periplasmic enzyme and contains four copper atoms per polypeptide [@coyle_nitrous_1985]. Spectroscopic models indicate that two types of site are present; a Cu$_A$ site (sometimes termed the “purple copper centre” due to its distinctive colour), related to that found in cytochrome _c_ oxidase and which functions in electron transfer, and a catalytic site, variously termed Cu$_B$, Cu$_C$ or Cu$_Z$, the latter term being preferred. Of these the Cu$_A$ site is the better understood and has been extensively modelled on the basis of EPR and MCD data. It is similar to the site found in cytochrome _c_ oxidase [@kelly_two_1993]; sequence similarity between the C-terminal of nitrous oxide reductase and cytochrome _c_ oxidase subunit II (which binds Cu$_A$) has been used to predict the ligands to the nitrous oxide reductase copper centre [@zumft_derived_1992]. In general, these predictions are supported by spectroscopic studies. The EPR spectrum of this centre is quite unusual and suggests that one electron is delocalised over both copper atoms, making the centre a one-electron acceptor [@farrar_model_1991]. The catalytic Cu$_Z$ site is rather less well understood, largely because it has fewer distinct spectroscopic properties. It has also been modelled by [@farrar_model_1991] and it is also thought to be dinuclear, containing 2 Cu(II) atoms. Reduction using dithionite is thought to reduce only one of these atoms to Cu(I), giving a dead-end product which is enzymically inactive. Ligands to the Cu$_Z$ site have been suggested by examining conserved residues known not to be involved with the Cu$_A$ site; 8 histidines and a single cysteine are possible candidates.

The enzymology of nitrous oxide reductase is rather complex. Two forms of the protein can be purified; the purple form (isolated anaerobically) and the pink form (isolated aerobically), with distinct spectroscopic features [@riester_nitrous_1989], the anaerobic form having a higher specific activity. Reduction of the enzyme with dithionite followed by oxidation by molecular oxygen converts the purple form to the pink form. Dithionite reduction alone produces a third form of the enzyme, the “blue” form which still contains oxidised copper; this is associated with the Cu$_Z$ centre. The enzyme is inhibited by acetylene, azide, carbon monoxide and cyanide [@kristjansson_first_1980], which presumably can act as copper ligands at one or both of the copper centres.

The purified enzyme shows the highest activities when assayed with reduced viologens as electron donors, activities using the presumed physiological donors are lower. However, turnover-dependent inactivation has been noted for some of these donors _e.g._ [@berks_purification_1993]. The activity of the enzyme from some species has been increased by incubation with reduced viologens [@snyder_purification_1987] and also by exposure to high pH buffer [@coyle_nitrous_1985]; the latter induces a permanent chemical change and it has been suggested that this reaction converts inactive forms of the Cu$_Z$ site to active species, perhaps by regenerating a thiolate ion [@berks_enzymes_1995].

The complexity of the copper sites in nitrous oxide reductase has slowed progress in the understanding of its mechanism. However, the dinuclear nature of the Cu$_Z$ site suggests that nitrous oxide is bound as a bridging ligand between the two copper atoms; following electron transfer from the Cu$_A$ site, the N-O bond could be activated for cleavage, releasing dinitrogen. The intermolecular electron transfer is another intriguing aspect of this enzyme, in particular why the unusual purple copper centre is used in preference to the more normal type I blue copper centre. The electron pathway through the enzyme (whether through hydrogen bonds/through space or through covalent bonds) is another problem that remains to be elucidated.

The structural gene for nitrous oxide reductase, _nosZ_, has been cloned and sequenced from a number of sources, allowing the sequence analyses described previously. In _Ps. stutzeri_ [@braun_structural_1992], it is part of a large denitrification cluster containing a number of genes believed to be involved in biosynthesis of the enzyme. Similar clusters are thought to exist in _Ps. aeruginosa_ [@zumft_derived_1992] and _P. denitrificans_ [@hoeren_sequence_1993], though there is no evidence of the accessory genes in _A. eutrophus_ [@zumft_derived_1992]. The _nos_ gene cluster is discussed more fully in Section 1.6, in relation to the other genes of denitrification. An interesting feature of the _nosZ_ genes is that the encoded NosZ protein appears to have an unusually long periplasmic targeting sequence (58-60 amino acids), compared to the more usual 26 or less [@von_heijne_signal_1985]. Another unusual feature is that the N-termini of the targeting sequences contain a quite highly conserved motif; these features have been suggested to point to a common export pathway for periplasmic proteins with complex metal cofactors [@berks_common_1996].

### Small electron transfer proteins

The periplasms of Gram-negative bacteria contain a variety of small, soluble redox proteins. These are assumed to mediate electron transfer between membrane-bound redox complexes, such as the cytochrome _bc_$_1$ complex and the periplasmic enzymes, including those of denitrification, although in few cases has an _in vivo_ reaction been directly demonstrated. The small electron transfer proteins group into two types: the small cytochromes _c_ and the cupredoxins.

The cytochromes _c_ are a large family of proteins, that have been grouped into various families by consideration of size, sequence similarity and physicochemical properties [@ambler_structure_1980]. They have been popular subjects for the study of protein structure and function for many years and a vast literature exists on the subject, which can only be briefly summarised here. The periplasmic cytochromes _c_ involved in electron transfer include cytochrome _c_$_{550}$ from _P. denitrificans_, cytochrome _c_$_{551}$ from _Ps. aeruginosa_ and cytochrome _c_$_2$ from Rhodobacter capsulatus. These proteins are characterised as Class I cytochromes _c_: they are generally small (80-120 amino acids), and contain one covalently bound _c_ haem. The haem group is low-spin, co-ordinated by histidine and methionine and found near the N terminus of the protein. Many crystal structures have been determined, revealing a mostly a helical structure, a crevice in which the haem group is buried and a positively-charged face around the haem edge, through which redox partners may interact by electrostatic interactions [@hall_reaction_1987].

The cupredoxins are a family of small (around 13 kDa) copper-containing proteins containing what are known as type I or “blue copper” centres. They contain a single copper atom which is involved in electron transfer with donor and acceptor proteins. Cupredoxins absorb intensely near 600 nm giving them an intense blue colour. The copper atom is
liganded by two histidine residues, a cysteine and a weak methionine ligand; subtle variations in the ligand geometry give rise to four spectroscopic sub-classes of cupredoxin: (i) azurins, (ii) pseudoazurins, (iii) plastocyanin/amicyanin, (iv) stellacyanin and associated proteins. Examples of cupredoxins include pseudoazurin from _T. pantotropha_ [@moir_purification_1993], azurin from _Ps. aeruginosa_ [@parr_purification_1976] and amicyanin from _P. denitrificans_ [@husain_properties_1986]. A number of X-ray crystal structures are known which show a high degree of similarity; cupredoxins contain a Greek key $\beta$-barrel fold, consisting of 8 $\beta$ sheets. As with the cytochromes _c_, surface regions that govern the interactions between the cupredoxins and their redox partners have been defined; in azurin, His-117 is thought to transfer electrons to partners bound by a patch of hydrophobic residues around this residue [@van_de_kamp_involvement_1990]. However, recent analysis of several redox partners has introduced the concept of pseudospecific interactions, in which both hydrophobic patches and oppositely charged surface regions act to bring proteins together and promote electron transfer [@williams_pseudospecific_1995].

Proteins are generally assumed to be physiological redox partners on the basis of their reactivity in _in vitro_ assays, as it is difficult to prove an _in vivo_ relationship. Where this has been attempted, mutant strains lacking the small redox protein under study have been used. A mutant of _P. denitrificans_ lacking cytochrome _c_$_{550}$ has been shown to denitrify at a similar rate to the wild-type organism but anaerobic growth becomes sensitive to diethyldithiocarbamate (a copper chelator), suggesting a role for a copper protein (probably pseudoazurin) in electron transport [@moir_properties_1994]. It is often the case that other redox proteins can substitute for an absent protein in redox carrier mutants; for example a mutant of _R. capsulatus_ lacking cytochrome _c_$_2$ can still grow photosynthetically, because a membrane-bound cytochrome _c_ takes over its role [@jenney_novel_1993].

Redox partners may also be inferred from co-expression. In _P. denitrificans_ grown methylotrophically, two additional cytochromes (_c_$_{551i}$ and _c_$_{553i}$, [@husain_characterization_1986]) and the copper protein amicyanin [@husain_properties_1986] appear, suggesting that they are involved in electron transfer to or from the methanol dehydrogenase.

## The cytochrome _cd_$_1$ nitrite reductase

Cytochrome _cd_$_1$ nitrite reductase is the alternative dissimilatory nitrite reductase to the copper enzyme (Section 1.4.3). Cytochrome _cd_$_1$ was first isolated from _Ps. aeruginosa_ [@horio_terminal_1958]. It reduces oxygen and has been referred to as soluble cytochrome oxidase, but it was later shown that the physiological role of the enzyme is nitrite reduction. This was based on the findings that the K$_m$ for nitrite is much lower than that for oxygen and that the enzyme is induced during anaerobic growth on nitrate [@yamanaka_biological_1963]. However oxygen reduction still attracts interest; it is known to be a four-electron reduction to water [@timkovich_evidence_1979] and may still prove relevant in the study of membrane bound cytochrome oxidases.

Cytochrome _cd_$_1$ has since been identified, isolated and studied in detail from many sources; some of these include _P. denitrificans_ [@lam_nitrite_1969], _T. pantotropha_ [@moir_purification_1993], _Ps. stutzeri_ [@zumft_defects_1988], _A. faecalis_ [@iwasaki_cytochrome_1971], _A. eutrophus_ [@sann_cytochrome_1994], _Thiobacillus denitrificans_ [@sawhney_sulphide-linked_1978], _Magnetospirillum magnetotacticum_ [yamazaki_nitrite_1995] and _Roseobacter denitrificans_ (formerly _Erythrobacter sp._ Och114 [@doi_two_1989]). Their gross properties are all similar, though as will be discussed later, there are important differences between them. The enzyme from _Ps. aeruginosa_ is the most intensively studied of the family in terms of biochemical mechanistic data and most of the discussion below pertains to data from this enzyme.

### Subunit structure

Cytochrome _cd_$_1$ from all sources is a dimer of two identical subunits of molecular masses around 55-65 kDa, as judged by SDS-PAGE, size exclusion chromatography and sedimentation analysis [@kuronen_new_1972]. Each subunit contains two haem groups; a covalently-bound _c_ haem identical to that found in mitochondrial cytochrome _c_ and a non-covalently bound _d_$_1$ haem, which is unique to this enzyme (Figure 1.3). 

```{r fig1-3, fig.cap="(a) Structure of _d_$_1$ haem and (b) of protoporphyrin IX. The _d_$_1$ haem is unique to cytochrome _cd_$_1$. Protoporphyrin IX can occur as a non-covalently bound prosthetic group, where it is referred to as _b_-haem. In _c_-type cytochromes, protoporphyrin IX binds covalently via the vinyl side chains to two cysteine residues in the polypeptide at the consensus site CXXCH (X = any amino acid).", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

The latter haem is distinguished by the presence of oxo and carboxylate moieties and partial saturation on two adjacent pyrroles (I and II) of the porphyrin ring. Its structure has been confirmed by NMR spectroscopy and total chemical synthesis [@chang_evidence_1986]. Haem _d_$_1$ can be removed from the protein to give an inactive semi-apo protein; activity can be restored by reconstitution with the haem group, or to a much lesser extent, _a_-type haem [@hill_reconstitution_1978].

### Properties of the haem groups in cytochrome _cd_$_1$

Haem groups and their ligands are fundamental determinants of the catalytic properties of enzymes like cytochrome _cd_$_1$. The ligands to the haems are amino acid residues from the polypeptide chain; they control access of the substrate and the electrochemistry of the reaction by modulating the reduction potential of the haem. Cytochrome _cd_$_1$ has been investigated by numerous spectroscopic methods, including UV-visible spectroscopy, nuclear magnetic resonance (NMR), electron paramagnetic resonance (EPR) and magnetic circular dichroism (MCD), from which models have been developed to explain the observed spin states and absorption features of the haem groups. In addition, a number of haem redox potentials have been measured for the enzyme.

In fully oxidised cytochrome _cd_$_1$ both haems are in a low spin state, indicating that they are hexacoordinate. Reduction causes the _d_$_1$ haem to become high spin indicating that it has lost an axial ligand [@walsh_magnetic_1979]. This implies that the _d_$_1$ haem is the site of nitrite binding and a number of ligands have been used to probe this binding-site, including azide, imidazole [@walsh_characterization_1981], cyanide [@barber_reactions_1978] and carbon monoxide [@parr_reaction_1975]. Similar behaviour has been noted for the periplasmic cytochrome _c_ peroxidase enzyme of _P. denitrificans_, in which peroxide access at the catalytic site is thought to be blocked by a sixth axial ligand until the electron receiving haem group is fully reduced; this would prevent partial reduction of hydrogen peroxide to more reactive oxygen species [@gilmour_kinetics_1994].

The identity of the ligands to the haems has taken a long time to resolve and apparently conflicting results have been reported for different cytochromes _cd_$_1$. Low temperature MCD and EPR spectra of the _Ps. aeruginosa_ enzyme in the oxidised state suggest histidine-methionine ligation at the _c_-haem and possibly bis-histidine ligation at the _d_$_1$ haem [@sutherland_investigation_1986]. This is also the case for _Ps. stutzeri_ [@cheesman_two_1997]. In contrast, EPR and Mössbauer spectroscopy of the enzyme from _Th. denitrificans_ suggested bis-histidine ligation at both haems [@huynh_mossbauer_1982], whilst the _T. pantotropha_ crystal structure (Section 1.5.3.1) has bis-histidine at the _c_ haem and an unusual histidine-tyrosine at the _d_$_1$ haem [@fulop_anatomy_1995]. Midpoint redox potentials of the haems have been reported for _Ps. aeruginosa_ by several groups, with some disagreement. The values reported to date are summarised in Table 1.2.

```{r table12, echo=FALSE, results='asis'}
table1_2 <- data.frame(Organism = c("_Pseudomonas aeruginosa_", "", "", "", "", "", "", "", 
                                    "_Pseudomonas nautica_ 617", "_Magnetospirillum magnetotacticum_"),
                       `_E_$^{o'}$ of _c_ haem (mV)` = c("", "288", "290", "", "", "294", "288", "", "234 (pH 7.6)", "191"),
                       `_E_$^{o'}$ of _d_$_1$ haem (mV)` = c("", "", "", "", "", "287", "", "", "199 (pH 7.6)", "180"),
                       `_E_$^{o'}$ (_c_ - _d_$_1$) (mV)` = c("73", "", "40-100", "24", "60", "", "", "25 (pH 8)", "", ""),
                       Reference = c("Horio et al. (1961) [@horio_relative_1961]", "Kamen and Horio (1970) [@kamen_bacterial_1970]", "Wharton et al. (1973) [@wharton_cytochrome_1973]", "Shimada and Orii (1976) [@shimada_oxidation-reduction_1976]", "Blatt and Pecht (1979) [@blatt_allosteric_1979]", "Schichman and Gray (1981) [@schichman_kinetics_1981]", "Silvestrini et al. (1982) [@silvestrini_kinetics_1982]", "Carson et al. (1986) [@carson_variations_1986]", "Besson et al. (1995) [@besson_cytochrome_1995]", "Yamazaki et al. (1995) [@yamazaki_nitrite_1995]"),
                       check.names = FALSE)

kable(table1_2,
      caption = "Standard reduction potentials reported by other workers of the haem groups in cytochrome _cd_$_1$. The figures for the difference between the midpoint redox potentials of the _c_ and _d_$_1$ haems are given where this measurement was the intention of the experiment, rather than a direct
measurement of the two values. The pH at which the reduction potential was measured is reported where it differs from pH 7.0.")
```

Measurement seems to be complicated by a strong cooperative redox reaction between the haems [@blatt_allosteric_1979] and by the possibility of _d_$_1$ haem loss and protein denaturation during the experiment (Alrik Koppenhöfer, personal communication). The _c_ haem value seems the more consistent of the two; for _Ps. aeruginosa_ this is estimated at around +288 mV (pH 7.0). The _d_$_1$ value has been extrapolated from data used to measure the difference between the two haems and has been variously estimated as being between 25 and 70 mV lower. For the enzyme from _M. magnetotacticum_ (which may be an atypical member of the family), the _c_ and _d_$_1$ values are estimated as +191 and +180 mV respectively [@yamazaki_nitrite_1995]. The corresponding values reported for _Pseudomonas nautica_ strain 617 cytochrome _cd_$_1$ are +234 mV and +199 mV (pH 7.6) [@besson_cytochrome_1995]. The _c_ haem in the semi-apo form of _T. pantotropha_ cytochrome _cd_$_1$ has been estimated as +243 mV (Koppenhöfer and Ferguson, unpublished work). The values are in the expected range given that the _c_ haem is expected to receive electrons from donors of potentials around +250 mV, though the _d_$_1$ values are somewhat lower than expected given that electrons are thought to flow from the _c_ to the _d_$_1$ haem.

### Folding and conformation of cytochrome _cd_$_1$

At present seven primary sequences of cytochromes _cd_$_1$ are known, from the organisms _Pseudomonas aeruginosa_ [@silvestrini_nitrite_1989], _Pseudomonas stutzeri_ Zobell [@jungst_nirstbm_1991], _Pseudomonas stutzeri_ JM300 [@smith_isolation_1992], _Paracoccus denitrificans_ IFO 12442 [@ohshima_cloning_1993], _Paracoccus denitrificans_ PD1222 [@de_boer_isolation_1994], _Alcaligenes eutrophus_ H16 [@rees_structural_1997] and _Thiosphaera pantotropha_ (this work, Chapter 3). The first to be obtained was from _Ps. aeruginosa_ and secondary structure predictions for this protein indicated that it contained two domains; a predominantly $\alpha$ helical N-terminal portion (including the _c_-haem region) and folds of largely $\beta$-sheet towards the carboxy terminus [@silvestrini_nitrite_1989]. Further evidence of a two domain structure was obtained from partial digests with the alkaline serine protease subtilisin, which indicated that a dimer containing only _d_$_1$ haem could be obtained [@horowitz_controlled_1982]. Circular dichroism spectroscopy using the _Ps. aeruginosa_ enzyme indicated that the proportions of $\alpha$ helix and $\beta$-sheet in the oxidised enzyme were 16% and 48% respectively, with a change on reduction to 35% $\beta$-sheet [@tordi_circular-dichroic_1984].

Conformational changes have been observed when the enzyme is reduced, although the precise character of these changes is unknown. In addition to a decrease in $\beta$-sheet content the radius of gyration decreases as measured by small angle X-ray scattering [@berger_small-angle_1980], the thermal stability of the protein decreases [@mitra_heat_1981] and crystals of cytochrome _cd_$_1$ crack on reduction [@akey_characterization_1980]. These observations point to some kind of rearrangement in quaternary structure with a change in redox state.

The orientation and relative position of the haem groups has long been a contentious issue. A slow electron transfer rate between the haems as observed by some groups suggested that they may be quite far apart. These data are contradicted by fluorescence quenching studies which indicated that the haems were grouped at one end of a subunit
[@mitra_location_1980] and by the recently obtained structure of the _T. pantotropha_ enzyme (Section 1.5.3.1). In the latter case, the haems are at a relative angle of about 60° and the iron-iron distance between haems within a subunit is 20.6 Å [@fulop_anatomy_1995].

#### Crystal structure of the T. pantotropha enzyme

Cytochrome _cd_$_1$ from _Ps. aeruginosa_ was first isolated over thirty years ago. Crystallisation of the enzyme was reported almost immediately [@yamanaka_crystalline_1963] but although other groups have also obtained crystals ([@akey_characterization_1980], tegoni_crystallization_1994), none of this work has led to a published X-ray structure. However, the crystal structure of cytochrome _cd_$_1$ in the oxidised state from _T. pantotropha_ has recently been determined (Figure 1.4). 

```{r fig1-4, fig.cap="Crystal structure of cytochrome _cd_$_1$ in the oxidised form from _T. pantotropha_. Taken from Fülöp et al. (1995) [@fulop_anatomy_1995]. (a) shows the dimeric molecule with the $\\alpha$-helical _c_-haem domain at the top and the $\\beta$-propeller _d_$_1$ haem domain below. (b) shows the _d_$_1$ haem domains of each subunit from below, clearly illustrating how the eight-fold $\\beta$-propeller structure forms a channel to bind the _d_$_1$ haem. The chain of each subunit is coloured from blue (N-terminus) to red (C-terminus).", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

The _T. pantotropha_ structure was initially determined at a resolution of 1.55 $\unicode{x212b}$ from diffraction data and using the primary sequence of the _P. denitrificans_ PD1222 protein as a guide [@fulop_anatomy_1995]. The latter sequence was assumed to be highly homologous to that of the protein from _T. pantotropha_ and this has been shown subsequently to be the case (Chapter 3).

The crystal space group contains a dimer with each monomer organised in two domains, as predicted by earlier studies. The smaller domain is predominantly $\alpha$-helical, binds the _c_ haem and resembles a Class I fold to some degree, though with some significant differences. The ligands to this haem are His-17 and His-69; this is rather unexpected as bis-histidine ligation is normally associated with low redox potential [@moore_cytochromes_1990], whereas the _c_ haem is thought to have a redox potential in the +250 mV range. The
larger domain is an eight-bladed $\beta$-propeller structure, a fold previously identified in methanol dehydrogenase [@xia_three-dimensional_1992]. Propeller structures with fewer blades are also seen in methylamine dehydrogenase (seven blades [@vellieux_structure_1989]) and neuraminidase (six blades [@varghese_structure_1983]). The central channel of this structure contains the _d_$_1$ haem, which is ligated by His-200 and interestingly, by Tyr-25. Not only is this an unusual haem ligand, but it originates in the _c_ haem domain; hence although separate, there is a distinct structural link between the domains. There are also a large number of important hydrogen bonds at the domain interface, and between subunits.

The structure answers some of the earlier questions about haem position and environment. The two haem types are at an angle of about 60° to each other within a monomer and separated by an iron-iron distance of 20.6 $\unicode{x212b}$. The haem edge-edge distance in a monomer is 11.0 $\unicode{x212b}$. Haem distances between subunits are even larger; _c_ haem iron-iron is 42.4 $\unicode{x212b}$. _c_ haem-_d_$_1$ haem is 47.0 $\unicode{x212b}$ and _d_$_1$ haem iron-iron is 50.5 $\unicode{x212b}$, with no edges closer than 38.5 $\unicode{x212b}$. This would seem to rule out a simple iron-iron electron transfer mechanism and also discounts a binuclear haem binding site for oxygen reduction. The reported crystal structure, in combination with more recent work on reduced crystals of the enzyme and previous mechanistic studies, has allowed the development of a model for nitrite reduction which is discussed in the next section.

### Kinetic and mechanistic studies of the enzyme
The mechanism of nitrite reduction and electron transfer in cytochrome _cd_$_1$ has been investigated by a number of methods. Classical enzymological studies have been limited by the fact that the product of nitrite reduction, nitric oxide, is a potent inhibitor of the enzyme and may also bind to the haem groups of donor proteins used in the assay system; this leads to rather complex patterns of product inhibition and rate equations ([@dhesi_patterns_1984], [@robinson_implications_1979]). However, a considerable amount of information has been gained from isotopic labelling techniques, spectroscopy and fast kinetic methods and data so obtained, combined with structural information, have led to a coherent model for nitrite reduction (Figure 1.5).

```{r fig1-5, fig.cap="Proposed mechanism for nitrite reduction by cytochrome _cd_$_1$ from _Thiosphaera pantotropha_ (Fülöp et al., 1995 [@fulop_anatomy_1995]). (a) Nitrite binds at the reduced _d_$_1$ haem which is pentacoordinate, displacing a water molecule. (b) His-345 and His-388 act as proton donors, allowing the abstraction of an oxygen atom from bound nitrite. (c) This leaves nitric oxide bound at the oxidised _d_$_1$ haem. (d) Bound NO is then displaced by replacement with Tyr-25. The _d_$_1$ haem is re-reduced by an electron from the _c_-haem, displacing Tyr-25 and allowing the cycle to begin again.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Electrons from donor proteins are believed to enter the enzyme at the _c_ haem [@parr_electron-transfer_1977]. This is followed by an intramolecular electron transfer to the _d_$_1$ haem. Reduction of this haem causes the loss of one of its axial ligands, allowing productive binding of nitrite. Isotope exchange studies have shown that the nitrite ion is protonated, then dehydrated to give an enzyme-bound nitrosyl intermediate [@kim_catalysis_1984]. Some earlier exchange studies also indicated that a substantial amount of reaction product was nitrous oxide [@kim_15n_1983]; however this has now been discounted as an artefact of the assay system, as _P. denitrificans_ cytochrome _cd_$_1$ assayed using isoascorbate as electron donor generates stoichiometric amounts of nitric oxide from nitrite [@carr_energy-conserving_1989]. A bound Fe(III)-NO+ intermediate would not be detectable by EPR, but has recently been observed by FTIR spectroscopy [@wang_direct_1996]. Furthermore, stopped-flow kinetic studies show that the first observable species is enzyme-bound nitric oxide at the oxidised _d_$_1$ haem, with the _c_ haem reduced [@silvestrini_reaction_1990]. It is notable that in this assay system, a slow intramolecular electron transfer from the _c_ haem to the _d_$_1$ haem occurs, giving fully-reduced NO-bound enzyme as the final product. Bound NO has also been observed by EPR [@legall_purification_1979]. This implies that the protonation/dehydration step is very fast and is followed by rapid oxidation of the _d_$_1$ haem. The _c_ haem is also re-reduced rapidly by a second electron; this rate is increased by the presence of NO at the _d_$_1$ haem [@silvestrini_reaction_1990].

In _T. pantotropha_ cytochrome _cd_$_1$, Tyr-25 is proposed to be the axial ligand which leaves the _d_$_1$ haem on reduction. This ligand is then proposed to “swing back”, displacing the nitric oxide molecule from the oxidised _d_$_1$ haem. Such a scheme would mean that NO had to be displaced before the _d_$_1$ haem could be reduced by the second electron from the _c_ haem (in contrast to [@silvestrini_reaction_1990]); this makes mechanistic sense as normally, Fe(II)-NO complexes are very stable, with a half-life for dissociation of the NO in the region of days [@traylor_why_1992]. This could still allow for the slow intramolecular electron transfer rate observed in some stop-flow experiments which has been assigned as the rate-limiting step [@silvestrini_reaction_1990]; the electron would have to "wait” at the _c_ haem for ligand reorganisation around the haem _d_$_1$ co-ordination sphere. However, the observed intramolecular transfer rate may vary according to the precise experimental method; more recent work using the technique of pulse radiolysis suggests that it may not be as slow as was previously thought [@kobayashi_pulse_1997].

Although the scheme outlined above is very plausible and well-supported by evidence, there are still several issues to be resolved. One of these is the precise nature of the enzyme cycle; recently it has been shown that on reduction, the _c_ haem of cytochrome _cd_$_1$ from _T. pantotropha_ changes from bis-histidine to histidine-methionine ligation [@williams_time-resolved_1996]. Thus it may be that the fully-oxidised form of the enzyme is an artefact of purification and that during the catalytic cycle _in vivo_, the completely oxidised state is not regenerated. Another problem is relating mechanistic data concerning the enzyme from different sources to the crystal structure of the enzyme from _T. pantotropha_, particularly as mounting spectroscopic evidence indicates that axial haem ligation does seem genuinely to differ between cytochrome _cd_$_1$ enzymes from different species; this point is discussed more fully in Chapter 3.

## Genetic analyses of denitrification

As outlined in the previous sections, a large number of genes involved in denitrification have been mapped and sequenced from several bacterial species. The first of these was the structural gene for nitrous oxide reductase, _nosZ_, from _Ps. stutzeri_ Zobell; the gene was isolated using Tn5 transposon mutagenesis and analysis of mutants deficient in denitrifying growth [@zumft_isolation_1985]. Subsequent cloning and sequencing of the flanking DNA led to the discovery of a large cluster of over 30 denitrification genes in this organism, spanning over 30 kb [@braun_structural_1992]. Other workers have built on this approach by screening digested genomic DNA in different organisms using oligonucleotide probes for specific denitrification genes, isolating large hybridising fragments and sequencing. In this way, similar denitrification gene clusters have been identified in _Ps. aeruginosa_ and _P. denitrificans_. Maps of the largest contiguous clusters as presently known are shown in Figure 1.6.

```{r fig1-6, fig.cap="Denitrification gene clusters in _Ps. stutzeri_ Zobell, _Ps. aeruginosa_ and _P. denitrificans_ PD1222. Loci within the clusters that contain genes for nitrite- (_nir_), nitric oxide- (_nor_) and nitrous oxide- (_nos_) reduction are indicated. Genes in these regions are labelled with single letters. Numbered genes are open reading frames that have not been designated a gene name. Homologous genes that are found in all three clusters are indicated by identical shading; the structural genes for nitrite reductase (_nirS_) and nitric oxide reductase (_norCB_) are shaded black. Note that a gene downstream of _norD_ designated _fnrD_ in _Ps. stutzeri_ Zobell is thought to be a homologue of _dnr_ (_Ps. aeruginosa_) and _nnr_ (_P. denitrificans_), Zumft and Körner (1997) [@zumft_enzyme_1997] but its sequence has not yet been reported. The figure was compiled from sequence data contained in the following GenBank accession numbers: _Ps. stutzeri_ Zobell Z69589, 250198, X56813, X53676, Z13988, S41769, X65277, Z17423, S51580, Z73914, Z28384; _Ps. aeruginosa_ D50473, D38133, D37883, X16452, D50019, D84475; _P. denitrificans_ U28078, U05002, U17435. Additional data for P. denitrificans was derived from de Boer (1996) [@de_boer_regulation_1996].", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Knowledge of these gene sequences has allowed analysis of the putative gene products using a range of methods. Some of these are theoretical; for instance the sequence can be analysed for known motifs (such as the CXXCH _c_ haem binding motif), or subjected to computer-based methods which may predict properties such as sub-cellular location, the presence of a periplasmic targeting sequence or the number of transmembrane helices in a membrane protein. Database searching may also reveal possible homologues whose function is already known. All of these methods have been applied to the genes of denitrification.

In general, experimental verification of gene function lags behind the sequencing effort. However, some progress has been made in this area. The commonest approach is to inactivate the gene of interest, usually by inserting an antibiotic resistance cassette into the gene by homologous recombination, and then to look for phenotypic effects under denitrifying growth conditions. This approach has the disadvantage that it may have polar effects on genes downstream of the target gene, but such effects can normally be recognised by complementation experiments. Using the methods outlined above, a number of similarities in the denitrification gene clusters of _P. denitrificans_, _Ps. stutzeri_ and _Ps. aeruginosa_ are apparent, that can be summarised as follows.

(i) All three clusters contain the structural genes for nitric oxide reductase, _norCB_, and nitrite reductase, _nirS_, quite close to each other. In addition, the larger _Ps. stutzeri_ cluster contains the nitrous oxide reductase gene, _nosZ_, and associated genes _nosDFY_, which are thought to be involved in biosynthesis of the enzyme. The _nosZ_ gene has also been sequenced in _P. denitrificans_ and appears to be associated with other _nos_ genes as in _Ps. stutzeri_.

(ii) Each organism contains a set of homologous genes that tend to be associated with the structural gene for a denitrification enzyme and appear to be involved with its biosynthesis. In _P. denitrificans_, the genes downstream of _nirS_, _nirECFD_, have all been inactivated by insertional mutagenesis and this results in a cytochrome _cd_$_1$ that lacks _d_$_1$ haem; hence these genes are assigned a role in haem _d_$_1$ biosynthesis [@de_boer_isolation_1994]. Genes of this type that are conserved between the three clusters include _nirF_ (a putative “scaffold” protein for haem _d_$_1$ assembly), _nirC_ (a small c-type cytochrome) and _nirE_ (a S-adenosylmethionine uroporphyrinogen (III) C-methyl transferase).

(iii) A third class of genes encodes putative electron transport proteins and these tend to be less highly conserved between species. Examples include _nirM_, which encodes the Class I cytochrome _c_$_{551}$ in _Ps. aeruginosa_ [@arai_cloning_1990] and _nirT_, encoding a membrane-bound tetrahaem cytochrome _c_ in _Ps. stutzeri_ [@jungst_nirstbm_1991]. The latter protein is not found in the other denitrification clusters, but has a homologue, _napC_, in the periplasmic nitrate reductase operon of _T. pantotropha_.

(iv) Each cluster contains several genes that appear to be involved in the regulation of denitrification. Predominant amongst these are members of the FNR family of proteins, discussed more fully in Section 1.7.1. Binding sites for these proteins are also detectable upstream of several denitrification operons. In addition, two other regulatory proteins are of note. One of these is _nirQ_ [@jungst_interdependence_1992] (designated _norQ_ in P. denitrificans [@de_boer_mutational_1996]). Inactivation of _nirQ_ abolishes nitrite and nitric oxide reduction _in vivo_ though the effects are complex; in _Ps. stutzeri_ nitric oxide reductase becomes expressed to a two-fold higher degree and nitrite reductase retains _in vitro_ activity. NirQ has some sequence similarity to NtrC, part of a two-component regulatory system which is linked to RNA polymerase containing the alternative sigma factor $\sigma^{54}$. The other protein is _nosR_ [@cuypers_nosr_1992] (with a second homologue, _nirI_ in _P. denitrificans_, de Boer et al. 1994). This is predicted to be a membrane-bound protein and may possess an iron-sulphur cluster. Inactivation of _nosR_ in _Ps. stutzeri_ results in the loss of nitrous oxide reductase and in a _P. denitrificans_ _nirI_ mutant, the _nirS_ promoter is inactivated and cytochrome _cd_$_1$ is absent (de Boer, 1996).

(iv) Finally, there are a number of genes to which no function can be assigned, or that do not appear to be involved in denitrification. One such example is _nirJ_ from _P. denitrificans_, insertional mutagenesis of which has no effect on anaerobic growth [@de_boer_isolation_1994]. Other examples include some recently sequenced open reading frames in _Ps. stutzeri_, such as _orf247_ (a short-chain alcohol dehydrogenase) and _orf57_, which has homology to a hypothetical _E. coli_ protein, YEBC [@glockner_sequence_1996].

The main feature of the denitrification gene clusters identified to date is that they contain homologous genes, but the gene order and content of the clusters are not conserved. For instance the _nor_ genes are upstream of the _nir_ operon in _Ps. aeruginosa_ and _P. denitrificans_, but are downstream in _Ps. stutzeri_. Similar genes are also often transcribed in different directions amongst the three clusters. This has led to the description of a “mosaic" gene organisation in these clusters [@zumft_enzyme_1997] and supports a recent proposal that gene order is not conserved in evolution [@mushegian_gene_1996]. A final feature of note is that genes for the membrane-bound and periplasmic nitrate reductases, _nar_ and _nap_ respectively, are not found within the large denitrification gene clusters described above.


## Regulation of denitrification

Respiratory N-oxide reduction is an anaerobic process and its regulation might therefore be expected to be under the control of both oxygen tension and ambient N-oxide concentration. Both of these aspects have been investigated, particularly in _E. coli_ due to its genetic amenability, though similar systems are becoming apparent in the denitrifying bacteria. It is now clear that the control systems involved act largely at the level of transcription. Several systems comprising DNA-binding transcriptional activators (or repressors), and the sequence motifs to which they bind, are now understood in some detail and are described in the following sections.

### Anaerobic response regulators: the FNR family

The anaerobic regulator FNR was first identified in mutants of _E. coli_, which were deficient in anaerobic growth on fumarate and nitrite [@lambden_mutants_1976]. Complementation analysis of these mutants revealed an open reading frame encoding the FNR protein. The protein has a subunit mass of around 28 kDa and shows high homology to the cyclic AMP receptor protein [@shaw_homology_1983]. The most highly conserved regions are a DNA-binding helix-turn-helix motif, an N-terminal $\beta$-roll and an RNA polymerase contact site. However, the FNR protein contains an N-terminal extension with four cysteine residues, and a further internal cysteine at position 122. The function of the cysteines is to bind iron; the activity of FNR can be correlated to its iron content, which varies with the method of purification, and can be reconstituted to some degree [@green_reconstitution_1996]. More recently, it has been suggested that the N-terminal region forms an iron-sulphur cluster, which is labile in the presence of oxygen [@khoroshilova_association_1995].

FNR is a transcriptional regulator which can act under anaerobic conditions in either a positive or a negative fashion, depending on the target gene. It has been suggested that FNR senses oxygen directly, possibly through a different binding affinity for Fe(II) and Fe(III) becker_o2_1996, but FNR-dependent promoter activity can also be stimulated by varying the redox potential in the cell [@unden_effect_1990], suggesting it may react more indirectly to the redox state. However, currently the consensus of opinion supports the idea that the inactive form of FNR is generated as a result of the destruction of its iron-sulphur cluster in the presence of oxygen [@khoroshilova_association_1995]. Under anaerobic conditions, FNR is believed to dimerise and bind to a target sequence upstream of the FNR-dependent gene. Many such gene clusters have been identified in _E. coli_ and other bacteria; the consensus target sequence (or “FNR box”) is an inverted repeat, TTGATN$_4$ATCAA and in _E. coli_, this is generally centred at about -41.5 nucleotides relative to the transcription start point [@walker_promoter_1991] in genes that are activated by FNR. DNA-binding by FNR can then enhance transcription from the downstream promoter by contacting RNA polymerase. FNR can also negatively regulate transcription by binding at a different position and blocking RNA polymerase. The _fnr_ gene is itself negatively auto-regulated in this way [@spiro_regulation_1987]. Binding to the FNR box has been demonstrated by a number of methods, including DNase I footprinting [@sharrocks_fnr_1991] and electrophoretic mobility shift assays [@ziegelhoffer_vitro_1995]. Having identified FNR-dependent promoters, reporter gene fusions have been employed to measure induction or repression at the promoter under different growth conditions [@park_regulation_1995].

Site-directed mutagenesis has been used extensively to determine both the key amino acids in the FNR protein and the nucleotides at its binding site. This work has identified the conserved regions in the N-terminal cysteines of FNR, the RNA polymerase contact site and the DNA-binding domain. Mutations in residues 81-87 of the _E. coli_ FNR protein can give a protein which is competent for DNA binding but not transcriptional activation, establishing the concept of a positive control “patch” which interacts with RNA polymerase [@williams_role_1991]. In this region, Gly-85 and Gln-88 are invariant but residues 86 and 87 vary in other members of the FNR family (see below). Site-directed mutagenesis has also established the residues thought to be critical in the DNA-binding domain; this is the most highly conserved region of the FNR family [@spiro_fnr_1994]. A number of FNR homologues have now been identified that fall into several distinct groups (see below) but intriguingly, the consensus FNR box appears to be very similar regardless of which FNR homologue it binds. There may be some promiscuous binding between FNR proteins and their target sequences, but on the whole the homologues do not appear to substitute for one another _in vivo_, suggesting rather subtle differences in the specificity of the different FNR-like proteins. However, only two amino acid substitutions in FNR allows binding to a CRP consensus site (and _vice-versa_, [@spiro_interconversion_1990]), and only three base changes in the FNR binding site allow binding by the CRP protein [@sawers_transcriptional_1997].

Much of the detailed biochemical and genetic characterisation of FNR has been carried out in _E. coli_, the organism in which it was first identified. More recently, a number of FNR homologues have been identified in several other species of bacteria and the FNR binding consensus has been found upstream of a large number of open reading frames. Many of these FNR homologues have been cloned and sequenced. Phylogenetic analysis of the sequences has revealed a superfamily of proteins (Figure 1.7). 

```{r fig1-7, fig.cap="Unrooted phylogenetic tree of the FNR family of transcriptional activator proteins. Protein sequences that showed a high similarity to the FNR protein from _E. coli_ were retrieved from the GenBank database and aligned using the program Clustal W (Thompson et al. 1994 [@thompson_clustal_1994]). The phylogenetic tree was constructed by the method of neighbour-joining (Saitou and Nei, 1987 [@saitou_neighbor-joining_1987]) and analysed by bootstrapping (Felsenstein, 1985 [@felsenstein_confidence_1985]) using 1000 trials. The final tree was displayed using the program Treeview (Page, 1996 [@page_treeview_1996]). More details of these methods are given in Chapter 3. The tree shows that the FNR family falls into three main groups (van Spanning et al., 1997 [@van_spanning_fnrp_1997]): (A) the FNR subgroup, (B) the FixK subgroup and (C) the NNR subgroup. The FixK subgroup can be subdivided further into two groups, depending on the presence of an N-terminal cysteine cluster (van Spanning et al., 1997 [@van_spanning_fnrp_1997]). The NtcA proteins (Frías et al. 1993 [@frias_general_1993]) also cluster together within the NNR subgroup. Abbreviations: Syn1, _Synechococcus_ sp. WH7803, Psa, _Pseudomonas aeruginosa_, Ec, _Escherichia coli_, Sp, _Shewanella putrefaciens_, Bp, _Bordetella pertussis_, Av, _Azotobacter vinelandii_, Rh, _Rhizobium_ sp. IC3342, Ap, _Actinobacillus pleuropneumoniae_, Bj, _Bradyrhizobium japonicum_, Rp, _Rhodopseudomonas palustris_, Pd, _Paracoccus denitrificans_, St, _Salmonella typhimurium_, Rl, _Rhizobium leguminosarum_, Hi, _Haemophilus influenzae_, Rs, _Rhodobacter sphaeroides_, Tf, _Thiobacillus ferrooxidans_, Re, _Rhizobium etli_, Cy, _Cyanothece_ sp. ATCC 51142, Ac, _Azorhizobium caulinodans_, Syn2, _Synechococcus_ sp. PCC 7942, Syn3, _Synechocystis_ sp. PCC 6803, An, _Anabaena_ sp. PCC 7120, Rm, _Rhizobium meliloti_, Pss, _Pseudomonas stutzeri_ Zobell, Bs, _Bacillus subtilis_.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Examples include NNR from _P. denitrificans_ [@van_spanning_nitrite_1995], ANR and DNR from _Ps. aeruginosa_ [@ye_anaerobic_1995; @arai_expression_1995] and FixK from _Rhizobium meliloti_ [@batut_fixk_1989]. FNR homologues have also been identified in Gram-positive bacteria [@irvine_lactobacillus_1993] and the FNR box has been identified upstream of the CYP 55 gene in the fungus _F. oxysporum_ [@tomura_nitric_1994].

Figure 1.7 shows that the FNR family falls into three main groups, which van Spanning et al. [@van_spanning_fnrp_1997] have labelled types A, B and C, with a further subdivision of group B into B1 and B2. This grouping reflects the different functional domains within the proteins. Group A are the prototype FNR proteins containing the N-terminal cysteine cluster, RNA polymerase contact site, dimerisation domain and DNA-binding helix-turn-helix motif. Group B1 also contains all of these domains, but in groups B2 and C (the FixK and NNR subfamilies), the cysteine cluster is not present and there are differences in the putative RNA polymerase contact site. The lack of an iron-binding cysteine cluster is intriguing, as it is this that is proposed to respond to anaerobiosis in the FNR proteins. FixK and NNR clearly do respond to anaerobiosis in some way, as shown by strains carrying mutations in these genes [@anthamatten_characterization_1992; @tosques_cloning_1996], but the mechanism of the response is at present not clear. The differences in the RNA polymerase contact site have led some groups to suggest that the promoter regions of genes controlled in this manner bind an alternative sigma factor on RNA polymerase [@winteler_homologous_1996]. This is discussed further in Chapter 7. In many cases different FNR homologues have been found in the same organism; for instance in _P. denitrificans_, FnrP is thought to regulate transcription of membrane-bound nitrate reductase, cytochrome _c_ peroxidase and _cbb_$_3$ cytochrome oxidase, whilst NNR controls the nitrite and nitric oxide reductases [@van_spanning_fnrp_1997]. Similarly in _Ps. aeruginosa_, the homologues ANR and DNR both seem to control denitrification genes [@arai_expression_1995], whilst in _Ps. stutzeri_, mutation of FnrA (an ANR homologue) has no effect on denitrifying growth, suggesting the presence of a second FNR-like protein [@cuypers_anaerobic_1993].


### The ArcA/B system
In _E. coli_ a second system for oxygen-regulated transcriptional control is known, the ArcA/B system. This is a two-component sensor/regulator signal transduction system, of a type commonly found where response to an environmental signal is required [@stock_protein_1989]. ArcB is a transmembrane protein, with a periplasmic N-terminal sensor domain and a cytoplasmic C-terminal transmitter domain [@iuchi_arcb_1990]. ArcA is a cytoplasmic protein, with an N-terminal receiver domain and a C-terminal DNA-binding domain [@iuchi_purification_1992].

The action of the ArcA/B system has been recently reviewed in Unden et al. [@unden_oxygen_1994]. Under low oxygen tension, the transmembrane protein ArcB responds indirectly to a signal at the cytoplasmic membrane (possibly the UQH$_2$/UQ ratio) and is autophosphorylated at His-292 by ATP. The phosphate is transferred intramolecularly to Asp-576, followed by a second phosphorylation at His-292. This double-phosphorylated species is then competent to phosphorylate the cytoplasmic ArcA protein at an N-terminal Asp residue. Once phosphorylated, ArcA becomes active and can bind DNA upstream of the genes to be regulated. ArcA-P acts for most genes as a repressor, blocking transcription under anaerobic conditions. Genes affected in this way include those encoding the citric acid cycle enzymes. ArcA-P can also act as an activator; the _E. coli_ _cydAB_ operon encoding cytochrome _d_
oxidase possesses two ArcA-P binding sites upstream of its promoter, which have been identified by DNase I footprinting [@lynch_transcriptional_1996]. This work has lead to the identification of a consensus ArcA-P binding site. LacZ reporter gene fusions have been used to show that the Arc A/B system exerts its effects in the range 10-20% air saturation, whereas Fnr functions in the 0-10% range [@tseng_effect_1996]. Hence in _E. coli_ the two systems co-ordinate the control of respiratory gene expression to ensure optimal use of oxygen as environmental conditions change.

Unlike FNR the ArcA/B system is only well characterised in _E. coli_ and to date, none of the denitrification genes have been shown to be under ArcA/B control. However, the system is important as a complementary response to the FNR system and it is possible that something similar will be found in denitrifying bacteria.

### Sensors of nitrate and nitrite
In _E. coli_ nitrate and nitrite are sensed by two, interacting, two-component regulatory systems known as NarX/L and NarQ/P [@stewart_nitrate_1993; @rabin_dual_1993]. These function in a similar fashion to the ArcA/B system described earlier. NarX and NarQ are transmembrane proteins, possessing a periplasmic domain which can sense nitrate or nitrite. A region within this domain termed the “P-box” has been shown by site-directed mutagenesis to be involved in nitrate/nitrite sensing [@cavicchioli_role_1996]. This results in an autophosphorylation at the C-terminal cytoplasmic domain, which in turn goes on to phosphorylate the cytoplasmic component, NarL or NarP. Activated NarL/P can then go on to bind to a target DNA sequence and enhance or repress transcription of nitrate/nitrite dependent genes. A consensus site, the “NarL heptamer”, has been proposed with the recognition sequence TACYNKT (where Y=C or T, N=any nucleotide and K=A or C) [@tyson_nitrite_1994]. The NarL heptamer appears as a single heptamer, a direct or inverted repeat, in either orientation and apparently between +20 and -220 nucleotides from the transcription start site. The interaction between the homologous NarX/L and NarQ/P
systems is quite complex; essentially it seems that nitrate stimulates NarX/Q to phosphorylate NarL/P, nitrite stimulates NarQ to phosphorylate both NarL and NarP and finally that NarX can act to dephosphorylate NarL-phosphate in the absence of nitrate [@rabin_dual_1993].

No analogous system has yet been identified in the denitrifying bacteria, but all four N oxides have been reported to increase the expression of at least one reductase, particularly in _Ps. stutzeri_ and _Ps. aeruginosa_. In the latter organism, expression of the cytochrome _cd_$_1$ nitrite reductase appears to rely more on the presence of nitrate or nitrite than on anaerobiosis [@zennaro_nitrite_1993]. Levels of nitrous oxide reductase in _Ps. stutzeri_ are also regulated by nitrate, nitrous oxide and anaerobiosis [@korner_expression_1989]. There also appears to be a regulatory link between the nitrite and nitric oxide reductases. In _Ps. stutzeri_, mutation in nitric oxide reductase reduces nitrite reductase activity, though its expression level is similar. The converse is true for mutants in nitrite reductase [@heiss_formation_1989]. This implies that nitric oxide may induce its reductase and that there is a functional coupling between these two enzymes.

It seems likely that control of denitrification will prove to be a complex process, with several hierarchical levels that act to “fine-tune” the response to changing environmental conditions. This is already becoming apparent in the systems described previously for _E. coli_. One example of a complex promoter system is the _focApfl_ operon, encoding pyruvate formate lyase and the _focA_ gene product. This system is regulated by both ArcA and FNR, which can act at several different promoters in the operon. One promoter, P6A, overlaps an FNR-dependent promoter, modulating its activity by competition for FNR- and RNA polymerase-binding [@kaiser_overlapping_1997]. There are also several examples of promoters that are regulated by the FNR, Arc A/B and NarX/L components [@bongaerts_transcriptional_1995].

Although the regulation of denitrification is known to occur primarily at the level of transcription, transcriptional analysis of denitrification genes has been reported in relatively few cases. Northern blot analysis has been reported for the _norBC_ genes of _Ps. stutzeri_ which are co-transcribed [@heiss_formation_1989], and for the _nosZ_ gene which is apparently under the control of 6 alternate promoters [@cuypers_multiple_1995]. Also in _Ps. stutzeri_, transcription of the _nirS_ gene for cytochrome _cd_$_1$ nitrite reductase occurs as two transcripts, _nirS_ and _nirST_, and is controlled by anaerobiosis and nitrate [@zumft_notitle_1995]. The _fnrA_ gene from this organism is also known to be transcribed as a single 0.85 kb species [@heiss_formation_1989]. Induction of the mRNAs for nitrate, nitrite and nitrous oxide reductases in _P. denitrificans_ during the switch to anaerobiosis has also been reported [@baumann_dynamics_1996]. More commonly, transcriptional and translational features are inferred from inspection of the gene sequence; hence in _P. denitrificans_, the _nirE_ and _nirC_ genes have overlapping stop and start codons [@de_boer_isolation_1994], implying that they are co-translated, whilst stem-loop analysis of the _nir_ operon in _Ps. aeruginosa_ has been used to suggest that the 11 genes _nirSMCFDLGHJEN_ are transcribed on a polycistronic mRNA [@kawasaki_gene_1997].

## Aims of the present work

At the outset of this project the X-ray crystal structure of cytochrome _cd_$_1$ nitrite reductase from _T. pantotropha_ had been determined from diffraction data and the _P. denitrificans_ PD1222 sequence, with no complete protein sequence from the _T. pantotropha_ protein. The initial aim of the project was to clone the structural gene, _nirS_, from _T. pantotropha_, so as to obtain the polypeptide sequence and refine this crystal structure. Mapping and sequencing of the _nirS_ gene and surrounding regions was also essential for subsequent DNA manipulations. It was also of interest to determine whether the organisation of the _nir_ cluster is similar to that of the closely-related _P. denitrificans_.

Having cloned the _nirS_ gene, the next aim would be to investigate several expression systems for the enzyme. This work would involve cloning the gene into a suitableexpression vector and trying to transform a number of heterologous hosts, to determine which ones expressed the protein most efficiently. As the _d_$_1$ haem group is unique to cytochrome _cd_$_1$, it would also be advantageous to use heterologous hosts which are able to synthesise this group, so as to express the active holoprotein. Development of a suitable expression system is an essential prerequisite for future site-directed mutagenesis studies.

Finally, transcriptional regulation of the _nirS_ and associated genes is of interest. Knowledge of this process may help to understand the expression of the protein, so a third aim of the project was to investigate the mRNA transcribed from the _nirS_ gene, using methods such as Northern blotting and determination of the transcription start site. This work was expected to help to define novel promoter regions in the denitrification genes of _P. denitrificans_ and _T. pantotropha_, and to provide insight into the transcriptional regulation of denitrification.
