# Sequencing and analysis of the _nirS_ gene from _Thiosphaera pantotropha_ {#chapter03}

## Introduction

To date the primary sequences of six cytochrome _cd_$_1$ proteins, obtained by DNA sequencing of the _nirS_ gene, have been published. These are from the organisms _Ps. aeruginosa_ NTCC 6750 [@silvestrini_kinetics_1982], _Ps. stutzeri_ JM300 [@smith_isolation_1992], _Ps. stutzeri_ Zobell [@jungst_nirstbm_1991], _P. denitrificans_ PD1222 [@de_boer_isolation_1994], _P. denitrificans_ IFO 12442 [@ohshima_cloning_1993] and _A. eutrophus_ H16 [@rees_structural_1997]. Additionally, small lengths of peptide sequence have been determined from several cytochromes _cd_$_1$ as a result of either N-terminal or internal protein sequencing, _e.g._ from _Pseudomonas stutzeri_ JM300 [@weeg-aerssens_purification_1991] and _Thiobacillus denitrificans_ [@hole_characterization_1996]. The initial aim of the present work, the sequencing of the structural gene for cytochrome _cd_$_1$ (nirS) from _T. pantotropha_, was important for several reasons. First, the published crystal structure of the enzyme was determined from the electron density map using comparison to the sequence of the protein from _P. denitrificans_ PD1222 [@fulop_anatomy_1995]. Although this procedure fitted the diffraction data well, the authentic protein sequence was required to resolve any ambiguities, for instance in the N-terminus of the protein which was disordered in the crystal structure, and to refine the structure. Second, knowledge of the _nirS_ nucleotide sequence was essential for future work involving manipulation of the gene; for example restriction mapping, cloning procedures and mutagenesis. Mapping of the regions either side of the _nirS_ gene would also facilitate cloning and allow comparison of the nir region in _T. pantotropha_ to the closely-related _P. denitrificans_ PD1222. Third, knowledge of the cytochrome _cd_$_1$ peptide sequence allows comparison with other cytochromes _cd_$_1$ and analysis of the similarities and differences within this group of enzymes.

This chapter describes the strategy used to sequence the _nirS_ gene from _T. pantotropha_. The DNA sequence is reported and analysed, including comparison with the sequences from two strains of _P. denitrificans_. The translated cytochrome _cd_$_1$ sequence is reported and compared with other known cytochrome _cd_$_1$ sequences, using several computer-based methods. This work reveals that there are important differences between the proteins and the comparison with cytochrome _cd_$_1$ from _Ps. stutzeri_ Zobell is explored in more detail.

Finally the relationship between the proposed haem ligands and the mechanism of cytochrome _cd_$_1$ is discussed, in the light of recent spectroscopic and crystallographic data.

## Sequencing strategy

The _nirS_ gene from _T. pantotropha_ was sequenced in collaboration with Dr. Simon Baker, from a series of overlapping PCR products. Initial studies begun before the present project by Dr. Simon Baker had generated a PCR product using degenerate primers that were designed using short lengths of peptide sequence obtained from _T. pantotropha_ cytochrome _cd_$_1$  [@moir_aspects_1993]. The sequence of the PCR product indicated that the _nirS_ DNA sequence of _T. pantotropha_ was very similar to that from _P. denitrificans_ PD1222, and so further primers for PCR and sequencing were designed using the sequence of the _nirS_ gene from _P. denitrificans_ PD1222 [@de_boer_isolation_1994]. Table 3.1 summarises the primers used.

```{r table31, echo=FALSE, results='asis'}
table3_1 <- data.frame(`Primer direction` = c("Forward", "", "", "", "", "", "", "", "", "",
                                              "Reverse", "", "", "", "", "", "", "", "", "", 
                                              "", "", "", "", ""),
                       `Primer name` = c("460F", "610F", "MIDF", "1053F", "1000F", "CT2F", "CTIF", "1572F", "1631F", "1918F", 
                                         "228R", "242R", "284R", "811R", "460R", "1073R", "300R", "MIDR", "1572R", "1661R", 
                                         "1982R", "CT2R", "CTIR", "nirS2R", "nirSR"),
                       `Equivalent bases in _P. denitrificans_ _nirS_ sequence(GenBank U05002)` = c("473-492", "610-629", "913-932", "1053-1072", "1192-1211", "1346-1364", "1352-1369", "1572-1590", "1630-1650", "1918-1940", "756-737", "714-697", "700-682", "810-790", "932-913", "1072-1051", "1211-1192", "1461-1442", "1592-1574", "1660-1642", "1982-1964", "2266-2248", "2280-2261", "2444-2427", "2447-2430"),
                       `Sequence (5'-3')` = c("ACCTCGGCCTTAACAAAGGT", "CTGGCCCTTGTCCTTGGGCC", "GGCTTCGACTACCTGCAAAG", "ATTCGGCATGAAGGAGATGCGC", "GACGGCACCACCTATGAGAT", "CCGAGATCAAGATCGGCTC", "TCAAGATCGGCTCGGAAG", "GCCCGAGTTCATCGTGAAC", "GACCTCAAGAACCTCAAGACC", "CCCGACAATGCCTGGAAGATCCT", 
                                              "TACGTCCTGCTGTGCAAGGT", "GTTGTCCGTCTTGGTCTT", "TCTTGTGATCCTCGAGTG", "GTTGTATTGGGCGTCGGACAG", "CTTTGCAGGTAGTCGAAGCC", "GCATCTCCTTCATGCCGAATTC", "ATCTCATAGGTGGTGCCGTC", "GTCCATGATGACGTATTGCG", "ACGTTCACGATGAACTCGG", "CGATCTCGGTGGTCTTGAG", 
                                              "TTGATGAAGAGCGAGCCACC", "GTTCGAGGGTCTTGTCGTCC", "GATGACGTGCTTCAGTTCGA", "GTTCGTCACCGTCTTGCC", "CCCGTTCGTCACCGTCTT"),
  check.names = FALSE)

kable(table3_1,
      caption = "Primers used to sequence the _nirS_ gene from _T. pantotropha_. The equivalent positions in the GenBank sequence accession number U05002 (the sequence of the genes _nirI_, _nirS_, _nirE_, _nirC_ and _nirF_ from _P. denitrificans_ PD1222) are shown. Primer sequences are in the 5' to 3' direction for both forward and reverse primers.")
```

The _nirS_ gene was cloned as four overlapping PCR products, generating the plasmids pAP4. pMID1, pCU2p1 and pNDp1. Figure 3.1 shows the sizes of these four PCR products, the primers used to generate them and their relation to the _nirS_ gene from _P. denitrificans_ PD1222.

```{r fig3-1, fig.cap="PCR products used to sequence the _nirS_ gene of _T. pantotropha_. The _nir_ region of _P. denitrificans_ PD1222 is shown, numbered according to the GenBank sequence U05002. Below is a magnified view of the _P. denitrificans_ PD1222 _nirS_ gene, showing its relationship to the four overlapping PCR products AP4, MID1, CU2p2 and NDp1. The lengths of the PCR products and the primer pairs (Table 3.1) used to generate them are shown.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-1.png")
```

Three different clones of each product were used for sequencing, and each clone was sequenced at least twice in both directions. This procedure resolved any ambiguities at each base. Additional sequence information from a genomic clone (plasmid pTNIR3, described in Chapter 4) is completely consistent with the sequence derived from these PCR
products.

## Results
Altogether, 2008 bp of DNA sequence was obtained, including that from the plasmid pTNIR3 (which contributed 139 bp of upstream sequence, Chapters 4 and 7). This covers the _nirS_ open reading frame (bases 139-1926) and short flanking regions at the 5' and 3' ends. The complete DNA sequence is presented in Figure 3.2, with the translated cytochrome _cd_$_1$ polypeptide sequence aligned below. Also indicated in this figure are features of interest within the non-coding regions of the DNA.

```{r fig3-2, fig.cap="Sequence of the _nirS_ gene and its product, cytochrome _cd_$_1$, from _T. pantotropha_. The ATG start codons of open reading frames are labelled with the gene name and an arrow indicating the direction of transcription. Inverted repeats in the DNA sequence are shown by inward facing pairs of arrows. Putative Shine-Delgarno ribosome binding sites are dash-underlined. The _c_-haem binding site in cytochrome _cd_$_1$ is single-underlined. The putative NNR binding site upstream of _nirS_ is boxed. The vertical downward-facing arrow indicates the predicted cleavage site of the periplasmic targeting sequence.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-2.png")
```

### DNA sequence features
Three imperfect inverted repeats are prominent in the DNA sequence, at positions 58-79, 1937-1946 and 1951-1968, and one perfect inverted repeat is found at positions 80-89. Similar repeats are found in the sequences from _P. denitrificans_ PD1222 [@de_boer_isolation_1994] and _P. denitrificans_ IFO 12442 [@ohshima_cloning_1993]. The positions, sequences and previously assigned functions of these repeats are summarised in Table 3.2.

```{r table32, echo=FALSE, results='asis'}
table3_2 <- data.frame(`DNA sequence of repeat in _T. pantotropha_` = c("Repeat 1: 58-79<br />5'-GGCCTTAACAATGGTCAAAGCC-3'", "Repeat 2: 80-89<br />5'-TTGCGCGCAA-3'", "Repeat 3: 1937-1946<br />5'-GGTTACGGACC-3'", "Repeat 4: 1952-1969<br />5'-GGGGGCGTTCGCGCCCCC-3'"),
                       `DNA sequence of repeat in _P. denitrificans_ PD1222` = c("478-499<br />5'-GGCCTTAACAAAGTCAAAGCC-3'", "500-509<br />5'-CCGCGCGCGG-3'", "2357-2366<br />5'-GGTTGCAACC-3'", "2372-2389<br />5'-GGGGGCGTTCGCGCCCCC-3'"),
                       `Previously assigned function of repeat` = c("Binding site for NNR, an anaerobic transcriptional activator protein of the FNR family", "No function assigned", "No function assigned", "Repeat structure assigned as binding site for an oxygen responsive element (ORE). 5' GGGGGC assigned as a -10 RNA polymerase binding site"),
                       check.names = FALSE)

kable(table3_2,
      caption = "Comparison of inverted repeats in the DNA sequence of the nir region from _T. pantotropha_ with those in the same region of _P. denitrificans_ PD1222. The sequences are numbered according to the GenBank accession numbers U75413 (_T. pantotropha_) and U05002 (_P. denitrificans_ PD1222). Previously assigned functions of the repeats in the _P. denitrificans_ PD1222 sequence are taken from the GenBank sequence annotation and from de Boer et al. (1994) [@de_boer_isolation_1994].",
      escape = FALSE) %>% 
  column_spec(1, monospace = TRUE) %>% 
  column_spec(2, monospace = TRUE) %>% 
  column_spec(3, width = "25%")
```

Two partial open reading frames can be identified in the _T. pantotropha_ DNA sequence (Figure 3.2). The first of these begins at position 17 and reads in the opposite orientation to the _nirS_ gene. The first four amino acids are Met-Ala-Met-Arg. This appears to correspond to the start of the _nirI_ gene (Section 3.4.1), which in _P. denitrificans_ PD1222 begins Met-Ala-Met-Gly [@de_boer_isolation_1994]. The second ORF at position 2001 begins Met-Ala-Gly; this seems to be the start of the _nirE_ gene (Section 3.4.1), which begins with an identical sequence in P.denitrificans PD1222 [@de_boer_isolation_1994]. The _nirS_ genes of _T. pantotropha_ and _P. denitrificans_ PD1222 are 94.1% identical and the GC content of the _nirS_ gene is 64%.

### Primary sequence of cytochrome _cd_$_1$ from _T. pantotropha_
The _nirS_ gene codes for a protein of 596 amino acids, the first 29 of which are predicted to be a cleavable periplasmic targeting sequence (see Discussion, Section 3.4.3 and Figure 3.3). The six previously published cytochrome _cd_$_1$ sequences were downloaded from the GenBank and Swissprot databases and processed using the program SignalP, which predicts targeting sequences according to the rules developed by von Heijne [@von_heijne_new_1986] and uses algorithms specifically tested on protein sequences from Gram-negative bacteria  [@nielsen_identification_1997]. The sequences were edited to remove the predicted targeting sequence and aligned using the ClustalW program [@thompson_clustal_1994]. The multiple sequence alignment was then formatted using the program ALSCRIPT [@barton_alscript_1993] to generate Figure 3.4. In this figure, identity between four or more of the sequences is indicated by black shading and similarity is indicated by various grades of grey shading, using amino acid similarity groups based on physicochemical properties. Other shadings are described in the figure legend.

```{r fig3-3, fig.cap="Predicted cleavage site of the periplasmic targeting sequence in cytochrome _cd_$_1$ from _T. pantotropha_. The SignalP program (Nielsen et al., 1997 [@nielsen_identification_1997]) predicts the cleavage site on the basis of three scores, termed C, S and Y. The residue lying immediately after the cleavage site has high C and Y scores and a sharp gradient in the S score. In the above figure it can be seen that the residue Q30 is predicted to be this residue. As described in the text, an N-terminal glutamine as the first residue of the mature protein is fully consistent with experimental data.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-3.png")
```

```{r fig3-4, fig.cap="Multiple alignment of cytochrome _cd_$_1$ sequences. Six previously published cytochrome _cd_$_1$ sequences (see main text for details) were downloaded from the GenBank and SwissProt databases. The sequence of cytochrome _cd_$_1$ from _T. pantotropha_ was added to the group. All sequences were edited to remove the predicted periplasmic targeting sequence and aligned using the Program ClustalW [@thompson_clustal_1994]. The multiple alignment was then processed using the program ALSCRIPT [@barton_alscript_1993], to shade homologous regions and other features of interest. Identity within four or more sequences is indicated by black shading; similarity is indicated by levels of grey shading based on the physicochemical properties of the amino acids. Conserved methionine and histidine residues which could be potential haem ligands are indicated in white on dark-grey and conserved aspartate residues at the end of the third strand in each $\\beta$ propeller (see Appendix A) are shown in white on light-grey. The haem ligands His-17, His-69 (_c_-haem), His-200 and Tyr-25 (_d_$_1$ haem) in the oxidised _T. pantotropha_ enzyme are shown by open boxes. The secondary structure elements of the _T. pantotropha_ enzyme  [@fulop_anatomy_1995] are shown above the alignment. Abbreviations: tp9263, _T. pantotropha_ LMD 92.63; pd1222, _P. denitrificans_ PD1222; pd12442, _P. denitrificans_ IFO 12442; pss14405, _Ps. stutzeri_ Zobell; pss300, _Ps. stutzeri_ JM300; psa6750, _Ps. aeruginosa_ NTCC 6750; aeh16, _A. eutrophus_ H16.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-4.png")
```

The multiple sequence file was processed using three other computational methods in order to compare the seven sequences. Each cytochrome _cd_$_1$ sequence was compared to the other six using the GCG program BESTFIT [@devereux_comprehensive_1984]. This program generates a percentage identity and similarity for each pair, the results of which are summarised in Table 3.3.

```{r table33, echo=FALSE, results='asis'}
table3_3 <- data.frame(col1 = c("_Pd1222_", "_Pd12442_", "_Pss14405_", "_Pss300_", "_Psa6750_", "_AeH16_", ""),
                       col2 = c("**97.8**<br />97.1", "**95.0**<br />91.6", "**73.5**<br />55.0", "**71.8**<br />53.8", "**78.5**<br />65.8", "**78.6**<br />61.8", "_Tp9263_"), 
                       col3 = c("", "**96.7**<br />94.0", "**78.5**<br />66.1", "**72.4**<br />54.4", "**78.5**<br />66.1", "**78.8**<br />62.4", "_Pd1222_"),
                       col4 = c("", "", "**73.7**<br />55.4", "**71.6**<br />52.7", "**77.5**<br />64.1", "**77.6**<br />60.8", "_Pd12442_"),
                       col5 = c("", "", "", "**93.7**<br />89.2", "**75.2**<br />58.9", "**74.1**<br />59.2", "_Pss14405_"),
                       col6 = c("", "", "", "", "**73.7**<br />57.2", "**73.2**<br />59.2", "_Pss300_"),
                       col7 = c("", "", "", "", "", "**80.3**<br />65.5", "_Psa6750_"))

kable(table3_3,
      caption = "Matrix of similarities and identities between the cytochromes _cd_$_1$.Pairwise comparisons were determined using the program BESTFIT, part of the GCG 8.0 package. In each box the upper bold figure is the percentage similarity for the pair, the lower is percentage identity. Sequence abbreviations are: _Tp9263_, _T. pantotropha_ LMD 92.63, _Pd1222_, _P. denitrificans_ PD1222, _Pd12442_, _P. denitrificans_ IFO 12442, _Pss14405_, _Ps. stutzeri_ ATCC 14405, _Pss300_, _Ps. stutzeri_ JM300, _Psa6750_, _Ps. aeruginosa_ NTCC 6750, _AeH16_, _A. eutrophus_ H16.",
      escape = FALSE,
      col.names = NULL)
```

Phylogenetic relationships between the aligned sequences were inferred using ClustalW. To generate the unrooted phylogram shown in Figure 3.5a, the multiple alignment was processed using a neighbour-joining method [@saitou_neighbor-joining_1987] to calculate divergence distance between the sequences. The statistical significance of the tree was analysed by the method of bootstrapping [@felsenstein_confidence_1985] with 1000 trials and the output was visualised with the program TreeView [@page_treeview_1996]. Figure 3.5b is a phylogenetic tree inferred from 16S ribosomal RNA sequences and used for comparison with the protein sequence tree.

```{r fig3-5, fig.cap="Phylogenetic tree of cytochrome _cd_$_1$ sequences compared with a phylogenetic tree of 16S rRNA sequences from the same organisms. (a) The sequences were aligned using the ClustalW program and the tree was calculated using the neighbour-joining method of Saitou and Nei [@saitou_neighbor-joining_1987]. The tree was then analysed using the method of bootstrapping, with 1000 trials and visualised with the program TreeView. Sequence abbreviations are as in Table 3.3. (b) The 16S rRNA tree was calculated in the same way as for the cytochromes _cd_$_1$, except that gaps were ignored due to the partial nature of the _T. pantotropha_ LMD 92.63 and _P. denitrificans_ PD1222 sequences (obtained from Dr. D.J. Richardson, University of East Anglia). Complete 16S rRNA gene sequences for _Ps. stutzeri_ ATCC 14405, _Ps. stutzeri_ JM300, _Ps. aeruginosa_ DSM 50071 (Psa50071) and _P. denitrificans_ LMG 4218 (Pd4218) were taken from the GenBank database.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-5.png")
```

Complete or partial rRNA sequence is known for four of the organisms under study as indicated in the figure legend; a different strain of _Ps. aeruginosa_ and a third _P. denitrificans_ strain were added to this group for comparison. The incomplete 16S rRNA sequences of _P. denitrificans_ PD1222 and _T. pantotropha_ were obtained from Dr. D.J. Richardson, University of East Anglia (J.P. Carter, S. Spiro and D.J. Richardson, unpublished results). The tree was produced in a similar manner to that for the cytochrome _cd_$_1$ sequences, except that gaps in the alignment were ignored owing to the partial nature of some of the 16S rRNA sequences. Finally the amino acid composition, molecular weight and predicted isoelectric point of each sequence was analysed using the program ProtParam [@appel_new_1994]. These data are presented in Table 3.4.

```{r table34, echo=FALSE, results='asis'}
table3_4 <- data.frame(`Sequence ID` = c("**Length**", "**Molecular weight (Da)**", "**Theoretical pI**", "**Amino acid (%)**", "**Ala**", "**Arg**", "**Asn**", "**Asp**", "**Cys**", "**Gln**", "**Glu**", "**Gly**", "**His**", "**Ile**", "**Leu**", "**Lys**", "**Met**", "**Phe**", "**Pro**", "**Ser**", "**Thr**", "**Trp**", "**Tyr**", "**Val**"),
                       Tp9263 = c("567", "62 538.9", "4.75", "", "8.8", "3.7", "3.7", "7.8", "0.4", "3.2", "7.6", "8.3", "2.5", "5.1", "7.6", "5.5", "2.1", "3.2", "6.0", "5.1", "7.4", "1.9", "3.7", "6.5"),
                       Pd1222 = c("568", "62 596.1", "4.73", "", "9.3", "3.5", "3.7", "7.6", "0.4", "3.3", "7.7", "8.5", "2.5", "5.3", "7.6", "5.5", "2.5", "3.2", "6.0", "4.8", "7.0", "1.9", "3.7", "6.2"),
                       Pd12442 = c("568", "62 687.5", "5.02", "", "9.3", "4.0", "4.0", "6.9", "0.5", "3.2", "7.2", "8.5", "2.5", "5.5", "7.4", "5.6", "2.3", "3.2", "6.3", "4.6", "7.0", "1.9", "3.9", "6.2"),
                       Ps14405 = c("534", "59 532.7", "6.44", "", "7.3", "3.6", "4.3", "6.4", "0.4", "2.4", "6.6", "7.5", "2.6", "5.8", "6.4", "8.6", "2.6", "2.6", "5.2", "5.6", "7.3", "2.2", "3.9", "8.6"),
                       Ps300 = c("527", "58 862.8", "6.59", "", "8.0", "4.4", "4.2", "6.8", "0.4", "2.3", "6.3", "7.0", "2.7", "5.9", "6.1", "8.2", "2.8", "2.7", "5.5", "5.5", "7.8", "2.5", "3.4", "7.8"),
                       Pa6750 = c("543", "60 178.2", "7.39", "", "7.9", "4.2", "3.7", "7.0", "0.4", "4.2", "5.0", "7.7", "2.9", "5.5", "6.8", "7.7", "2.0", "2.6", "6.3", "6.4", "5.7", "2.0", "3.7", "8.1"),
                       Aeh16 = c("529", "58 000.0", "8.44", "", "9.8", "4.3", "3.4", "7.4", "0.4", "2.3", "4.5", "7.9", "2.5", "5.5", "7.6", "8.1", "1.7", "2.6", "6.0", "5.1", "7.4", "1.7", "4.0", "8.0"),
                       check.names = FALSE)

kable(table3_4,
      escape = FALSE,
      caption = "Table 3.4 Comparison of the physicochemical properties of cytochrome _cd_$_1$ from different sources. Sequence abbreviations are as in Table 3.3 and Figure 3.4. Cytochrome _cd_$_1$ sequences were retrieved from the GenBank and SwissProt databases. The data presented in the table was obtained using the program ProtParam [@appel_new_1994].")
```

## Discussion
### Features of the DNA sequence
The _nirS_ gene and its immediate flanking regions are very similar in both _T. pantotropha_ and the two previously published _P. denitrificans_ sequences; in particular the _nirS_ open reading frame from _T. pantotropha_ is 94.1% identical to that of _P. denitrificans_ PD1222. The GC content of the two genes is 64%, a typical figure for genes from _Paracoccus_ species, reflecting the high GC content of the genome (68% [@steinrucke_genetics_1993]).

In _P. denitrificans_ PD1222, an open reading frame was identified both upstream and downstream of the _nirS_ gene. The upstream ORF was designated _nirI_ and has some sequence homology to the regulatory protein _nosR_ [@de_boer_isolation_1994; @hoeren_sequence_1993], in that it is predicted to be an integral membrane protein with a putative iron-sulphur cluster. It is transcribed divergently from the _nirS_ gene. Insertional mutation in _nirI_ results in the loss of cytochrome _cd_$_1$ [@de_boer_isolation_1994], Chapter 7. The _nirI_ ORF is also present in the sequence from _P. denitrificans_ IFO 12442, although Ohshima et al. (1993) [@ohshima_cloning_1993] failed to identify it as such. Translation of the first 37 codons from the latter sequence shows 97% identity to the _P. denitrificans_ PD1222 NirI protein, with a substitution of Phe for Leu at position 33. In the _T. pantotropha_ sequence, the start of the _nirI_ gene can also be discerned, with a substitution of Arg for Gly at position 4.

Downstream of _nirs_, de Boer et al. (1994) [@de_boer_isolation_1994] identified the gene _nirE_. The product of this gene shows similarity to a number of S-adenosyl-L-methionine uroporphyrinogen (III) methyltransferases such as the sirohaem biosynthetic gene _cysG_ of _E. coli_ [@peakman_nucleotide_1990], and _cobA_, a cobalamin biosynthesis gene of _S. typhimurium_ [@roth_characterization_1993], and is thought to be involved in methyl transfer to the isobacteriochlorin ring during the biosynthesis of haem _d_$_1$. Insertional inactivation of _nirE_ results in the formation of inactive cytochrome _cd_$_1$, lacking the _d_$_1$ haem [@de_boer_isolation_1994]. This open reading frame was identified (but not characterised) in _P. denitrificans_ IFO 12442 [@ohshima_cloning_1993].

Translation of the first 27 codons from the _P. denitrificans_ IFO 12242 gene shows 81% identity to the _P. denitrificans_ PD1222 NirE product. Once again, the start of the _nirE_ gene can be identified in the _T. pantotropha_ sequence. These sequence data indicate that the structure of the _nir_ gene cluster in the immediate vicinity of _nirS_ is highly conserved in _T. pantotropha_ and the two strains of _Paracoccus denitrificans_ characterised to date. Further mapping of this region, described in Chapter 4, confirms this observation.

The non-coding regions flanking the _nirS_ gene contain a number of inverted repeats which are also quite highly conserved. In the _T. pantotropha_ sequence, the first of these (repeat 1, Table 3.1) begins 81 bp upstream of the _nirS_ initiation codon. In _P. denitrificans_ PD1222 this sequence was identified as containing a putative binding site for NNR, a transcriptional activator protein belonging to the FNR family [@spiro_fnr_1994], Chapter 7. The NNR gene has been cloned and sequenced in this organism [@van_spanning_nitrite_1995] and insertional mutagenesis results in the loss of nitrite and nitric oxide reductases. The putative NNR binding sequences are very similar, with a single substitution of T for A at position 12 in the _T. pantotropha_ sequence. This lies in the non-conserved region of the NNR box consensus and is unlikely to be important. An identical sequence to the _P. denitrificans_ PD1222 NNR box is found in the _P. denitrificans_ IFO 12442 sequence, although Ohshima et al. (1993) [@ohshima_cloning_1993] did not identify it as such. Instead, they assigned the TAA trinucleotide as a stop sequence for a putative upstream open reading frame and included part of the NNR box sequence in a larger, downstream repeat. Both of these assignments appear to be erroneous in the light of other data. Expression of cytochrome _cd_$_1$ in all three organisms is, therefore, concluded to be transcriptionally regulated by the NNR protein. An alignment of these three NNR boxes with the FNR consensus sequence is shown in Figure 3.6.

```{r fig3-6, fig.cap="Putative NNR binding sites in the upstream regions of the _nirS_ gene aligned with the FNR consensus binding site. The NNR boxes upstream of _nirS_ in _T. pantotropha_ LMD 92.63 (Tp9263), _P. denitrificans_ PD1222 (Pd1222) and _P. denitrificans_ IFO 12442 (Pd12442) are shown as part of a larger inverted repeat, aligned to the consensus FNR binding site derived from anaerobically-regulated _E. coli_ genes [@spiro_fnr_1994].", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-6.png")
```

Immediately downstream of the NNR box is a small, perfect inverted repeat (repeat 2, Table 3.2). This structure is also seen in the two _P. denitrificans_ sequences but interestingly, in both cases, the first and last dinucleotides are substituted by CC and GG respectively. Hence the repeat structure is preserved, although the sequences differ. No putative function has been assigned to this repeat, but its structural conservation implies that it has some importance. A possible function is the binding of a transcriptional repressor protein; this possibility is explored more fully in Chapter 7.

Two more repeat sequences are found between the _nirS_ and nirE genes in _T. pantotropha_. The first of these (repeat 3, Table 3.2) was identified in the _P. denitrificans_ PD1222 sequence as a perfect 10 bp inverted repeat and is also found in the _P. denitrificans_ IFO 12442 sequence. However, in _T. pantotropha_ the sequence differs at positions five and seven and so is an imperfect repeat. The significance of this repeat is unclear.

Finally, a large imperfect repeat (repeat 4, Table 3.2) is found 6 bp downstream of repeat 3 in _T. pantotropha_. The sequence of this repeat is perfectly conserved in the _P. denitrificans_ PD1222 sequence, but less so in _P. denitrificans_ IFO 12442 where the corresponding sequence differs at positions one and five. In _P. denitrificans_ PD1222 two assignments were made to this sequence; the first GGGGGC hexanucleotide was suggested to be a putative -10 promoter sequence for the _nirE_ gene [@de_boer_isolation_1994], based on the promoter consensus for purple non-sulphur bacteria of Steinrücke and Ludwig (1993) [@steinrucke_genetics_1993], whilst the structure as a whole was suggested by the same authors to be a putative oxygen responsive element (ORE). An ORE is a region of dyad symmetry found upstream of genes that are regulated by oxygen; it was first identified in photosynthetic genes from the genus _Rhodobacter_ [@klug_dna_1991] and is thought to bind a protein that represses transcription under aerobic conditions. Steinrücke and Ludwig (1993) [@steinrucke_genetics_1993] have proposed that some _Paracoccus_ are also ORE-regulated. Both of the assignments proposed by de Boer et al. (1994) [@de_boer_isolation_1994] were tentative and both now appear unlikely. Based on the finding that _nirS_ is transcribed as a single mRNA species (see Chapter 7 for data and discussion), this feature could be involved in transcriptional termination.

### Analysis of the seven primary sequences of cytochromes _cd_$_1$

A ClustalW multiple alignment of the six previously published cytochrome _cd_$_1$ sequences plus the _T. pantotropha_ sequence is shown in Figure 3.4. It is clear that as a group, the cytochromes _cd_$_1$ are extremely highly conserved. In this alignment identity between four or more sequences is indicated by black shading. A large number of identical regions are apparent, particularly around the _c_-haem binding region and in the haem _d_$_1$ binding domain (which corresponds to residues 135-567 in the mature _T. pantotropha_ sequence). The _T. pantotropha_ enzyme shows most homology to the two _P. denitrificans_ sequences (97.1% identity to _P. denitrificans_ PD1222); of all the pairwise comparisons, the lowest identity is still considerable (52.7% between _Ps. stutzeri_ JM300 and _P. denitrificans_ IFO 12442). However, variation is apparent within the N-terminal region, a point which is returned to presently.

The ClustalW alignment was used as the basis for construction of a phylogenetic tree. To generate this tree, the ClustalW program uses the neighbour-joining method of Saitou and Nei  [@saitou_neighbor-joining_1987]. Briefly, this algorithm compares all pairs of sequences in an alignment and computes a percentage divergence between them. A correction is included to take account of the possibility that multiple mutations can occur at one site. Pairs of sequences are then joined and connected to the other sequences by an internal branch, such that the total branch length is minimised. The joined sequences are merged and the process is repeated, so an alignment of N sequences allows an unrooted tree with N-3 internal branches. To place confidence limits on the tree, the method of bootstrapping [@felsenstein_confidence_1985] was employed. This process is quite complex, but essentially for an alignment of N residues in length, a random sample of N sites is taken, from which a distance matrix and tree can be calculated. This process is repeated a large number of times (1000 is a typical figure) and a site may be selected several times, or not at all, in any given sample. If the sample trees are similar to one another and the tree from the complete data set, the tree is strongly supported. A particular grouping of sequences is considered significant at the 95% confidence level if it occurs in 95% or more of the bootstrap samples. The biological significance of the tree, in terms of phylogenetic relationships will always be a rather subjective judgement, based on factors such as agreement between trees from different data sources (e.g. protein and rRNA sequences) and other taxonomic schemes for grouping organisms.

Figure 3.5a shows that the _T. pantotropha_ sequence lies closest to that from _P. denitrificans_ PD1222 and that these two form a group with _P. denitrificans_ IFO 12442. This group is most distant from the two strains of _Ps. stutzeri_ which group together. The proteins from _Ps. aeruginosa_ and _A. eutrophus_ are approximately equidistant from these two groups and closer to each other than the other enzymes. This grouping is what would be expected from the current consensus opinion of the phylogeny of the _Proteobacteria_ [@olsen_winds_1994]. For comparison, a phylogenetic tree constructed from 16S ribosomal RNA sequences is shown in Figure 3.5b. 16S rRNA sequences are widely used to infer phylogeny (the advantages and limitations of the method are discussed in Fox et al. (1992) [@fox_how_1992], and the results agree closely with that obtained from the cytochrome _cd_$_1$ sequences. The main results of this analysis can be summarised as follows: (1) comparison of cytochrome _cd_$_1$ and 16S rRNA sequences between the organisms under consideration gives estimates of relatedness which agree well with one another and with current ideas about bacterial taxonomy and (2) the high similarity of the cytochrome _cd_$_1$ sequences indicates an interspecies transfer of the _nirS_ gene in the evolutionary past and/or a strong relationship between structure and function in the protein.

### Relationship of the primary sequence to the structure of cytochrome _cd_$_1$
Sequencing of the _nirS_ gene from _T. pantotropha_ has allowed the crystal structure of the enzyme, originally deduced from electron density by comparison with the _P. denitrificans_ PD1222 sequence [@fulop_anatomy_1995], to be checked for errors and refined [@baker_cytochrome_1997]. There were 14 discrepancies between the density map and the _P. denitrificans_ PD1222 sequence in the original publication of the oxidised crystal structure; the translated protein sequence has confirmed 11 of these (three were shown to be the same as the _P. denitrificans_ sequence), as well as revealing four others that were not apparent earlier (counting from the N-terminus of the mature protein these are K2OR, G101A, E339S and A499T, the corrected amino acid second). The oxidised crystal structure has now been determined at the higher resolution of 1.28 $\unicode{x212b}$ and is fully consistent with the protein sequence presented in this chapter.

The crystal structure of cytochrome _cd_$_1$ contains 559 residues, giving a calculated molecular weight of 62 249 Da per monomer including the covalently-bound _c_ haem moiety. The monomeric molecular weight as measured by electrospray mass spectrometry was 63 091 +/- 8 Da [@moir_purification_1993], a discrepancy of 842 Da (when using the _P. denitrificans_ sequence for comparison). From this, it was argued that eight N-terminal residues of the mature protein in the crystal structure were not visible, due to disorder in the crystal packing [@fulop_anatomy_1995]. This has been confirmed by sequencing of the _nirS_ gene. The _T. pantotropha_ _nirS_ gene encodes a protein of 596 amino acids, with a calculated molecular weight of 66 045 Da, including the _c_ haem. The first 29 residues are predicted to be a cleavable periplasmic targeting sequence (Figure 3.3), giving a molecular weight for the mature protein of 63 141 Da. The discrepancy between the latter figure and the molecular weight calculated from the crystal structure is 892 Da, which is accounted for by the first eight residues in the mature _T. pantotropha_ sequence. These residues are QEQVAPPK, an additional molecular weight of 896 Da. The N-terminus of cytochrome _cd_$_1$ from _T. pantotropha_ is known to be blocked, as it is resistant to Edman degradation [@moir_aspects_1993]. An N-terminal glutamine, as postulated above, can account for this by cyclisation to pyroglutamate. The N-terminus of cytochrome _cd_$_1$ from _P. denitrificans_ IFO 12442 has also been reported to be blocked [@ohshima_cloning_1993]. However, the equivalent eight residue sequence in the two Paracoccus sequences is QEQAAPPK and a further difference (counting from the N-terminus of the uncleaved protein, residue 27 is alanine in the _Paracoccus_ sequences, valine in _T. pantotropha_) makes the predicted signal cleavage site one residue earlier in both _Paracoccus_ sequences, giving AQEQAAPPK as the mature N-terminus. This would not be expected to show blockage, but in the absence of further experimental data, the predicted mature sequences have been used in this chapter and the effect on the alignment is insignificant.

The crystal structure of cytochrome _cd_$_1$ from _T. pantotropha_ showed that the _c_ haem was ligated by His-17 and His-69 (numbering is with the targeting sequence removed) and the _d_$_1$ haem by Tyr-25 and His-200 [@fulop_anatomy_1995]. Both of these arrangements were somewhat unexpected; first, bis-histidyl ligation is normally associated with low-potential haems [@moore_cytochromes_1990; @wallace_functional_1992], whereas the _c_-haem in cytochrome _cd_$_1$ has a midpoint redox potential of around 243 mV (A. Koppenhöfer, unpublished data). Second, tyrosyl ligation to haem groups, which is most common in bacterial catalases [@dawson_active_1991] also results in a low redox potential, as illustrated by mutant forms of haemoglobin [@nagai_reduction_1983] and myoglobin [@hildebrand_proximal_1995]. The midpoint redox potential of the _d_$_1$ haem in cytochrome _cd_$_1$ from _T. pantotropha_ is not known but is likely to be a few tens of millivolts lower than that of the _c_ haem (Chapter 1, Table 1.2). However, the protein sequence confirms the presence of all of these residues.

His-17 and Tyr-25 are located towards the N-terminus of the protein, which Figure 3.4 shows to be the least conserved region of the cytochromes _cd_$_1$. Intriguingly, inspection of Figure 3.4 shows that these residues are conserved only in _T. pantotropha_ and the two strains of _Paracoccus_. The other proteins lack the N-terminal extension seen in these sequences which contains the two haem ligands. Of the other proteins, only that from _Ps. aeruginosa_ contains a histidine or tyrosine residue at even roughly similar positions in the N-terminus before the _c_ haem binding site. This finding was extremely unexpected. Ligands to the haem groups control several important factors such as the redox potential, substrate accessibility and the reaction mechanism; in _T. pantotropha_, Tyr-25 has been postulated to play a crucial role in the release of nitric oxide from the haem group [@fulop_anatomy_1995]. The importance of these residues and the overall high similarity of the proteins suggests that functional amino acids should be highly conserved. For this reason, it was decided to re-sequence the relevant segment of the _nirS_ sequence from _Ps. stutzeri_ Zobell so as to examine the possibility that the sequence had been mis-reported, or that errors in the sequence (such as base changes or frame shifts) could have resulted in the apparent absence of at least Tyr 25. Two sets of forward and reverse primers, flanking the N-terminal region of the _nirS_ protein, were designed and used to amplify genomic DNA from _Ps. stutzeri_ Zobell, as shown in Figure 3.7. 

```{r fig3-7, fig.cap="Early region of the _nirS_ gene from _Ps. stutzeri_ ATCC 14405, showing primers used for re-sequencing and the sequence obtained. The downward arrow indicates a deletion of about 40 amino acids seen when the cytochrome _cd_$_1$ sequence is aligned to that of _T. pantotropha_. Primers designed to flank this region (FOR1, FOR2, REV1 and REV2) are indicated by horizontal arrows. The sequence obtained is annotated as follows; dotted underline = sequenced once in one direction, single underline = sequenced twice in forward direction, double underline = sequenced in both directions.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-7.png")
```

The four resulting PCR products were directly sequenced from low-melting point agarose using these primers. Figure 3.7 indicates the region, direction and strand coverage (forward, reverse or both) of the sequence obtained. This sequence was 100% identical to that deposited in the GenBank database. It is thus clear that the gene sequence is correct and that the tyrosyl and histidyl ligands in the protein from _T. pantotropha_ really are absent from at least the _Ps. stutzeri_ Zobell enzyme.

This result raises a number of possibilities. First, the _c_- and _d_$_1$ haems in cytochrome _cd_$_1$ in the oxidised state from _Ps. stutzeri_ (and from other species) may be liganded by tyrosyl and histidyl residues lying elsewhere in the polypeptide chain. This would result in a substantially different tertiary structure in these enzymes and given the overall high sequence homology, does not seem to be a credible proposal. Second, the other cytochromes _cd_$_1$ may have different haem ligands (such as the conserved methionines and histidines seen in Figure 3.4), which could preserve a similar structure to the _T. pantotropha_ protein, but might be expected to result in different spectroscopic and biochemical properties. Third, the oxidised _T. pantotropha_ enzyme structure may not be representative of the physiologically-active structure found _in vivo_; the latter, hypothetical, structure may have the haem ligands that are conserved between all cytochrome _cd_$_1$ sequences.

In order to examine the optical spectrum of cytochrome _cd_$_1$ from _Ps. stutzeri_ Zobell more closely, it was purified using the same procedure as that for the _T. pantotropha_ protein. This protocol gave a product that was approximately 90-95% pure as judged by SDS-PAGE. The spectra of the oxidised and reduced forms of the enzyme are shown in Figure 3.8.

```{r fig3-8, fig.cap="Visible spectra of cytochrome _cd_$_1$ (a) in the oxidised and (b) in the reduced forms from _Ps. stutzeri_ Zobell. As isolated, cytochrome _cd_$_1$ from _Ps. stutzeri_ Zobell exhibited absorbance maxima at 416, 526, 554 and 645 nm. Following reduction with dithionite maxima were observed at 421 (not shown), 464, 526, 553, 557 and 660 nm. The reversal in the relative amplitudes of the peaks at 548 and 554 nm referred to in the text is clearly visible in (b).", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-8.png")
```

Overall, the spectra of the seven cytochromes _cd_$_1$ that have been sequenced are very similar. In the oxidised form there are peaks at or near 411 and 640 nm, with shoulders around 561, 325 and 360 nm. In the reduced form, peaks at 418, 460, 521, 548, 554 and 625 nm (dithionite-reduced) or 655 nm (ascorbate-reduced) are seen. The absorbances at 655/625 and 460 nm are assigned to the _d_$_1$ haem, whereas the 548-554, 521/525 and 418/411 nm peaks are the $\alpha$, $\beta$ and $\gamma$ (Soret) peaks of the _c_-type haem. However, the enzyme from _Ps. stutzeri_ Zobell shows a small, but interesting difference. In the split $\alpha$-peak of the reduced _c_ haem at 548-554 nm, the first peak of the pair is slightly higher for the enzymes from _T. pantotropha_, _P. denitrificans_ and _Ps. aeruginosa_. In the purified enzyme from _Ps. stutzeri_ Zobell, this situation is reversed. This feature can be seen in published spectra by other workers [@zumft_defects_1988] and was also specifically noted for the enzyme from _Ps. stutzeri_ JM300 [@weeg-aerssens_purification_1991]. Interestingly, if the _d_$_1$ haem is removed from the _T. pantotropha_ protein to give the semi-apo enzyme, the relative heights of the split $\alpha$-peaks in the reduced state become reversed, so as to resemble the _Ps. stutzeri_ enzymes (Figure 3.9).

```{r fig3-9, fig.cap="Visible spectra of (a) holocytochrome _cd_$_1$ and (b) semi-apo cytochrome _cd_$_1$ in the reduced forms from _T. pantotropha_. The reduced holoprotein exhibited absorbance maxima at 418, 460 (not shown), 525, 548, 554 and 650 nm (a). Following removal of the _d_$_1$ haem the maxima at 460 and 650 nm (not shown) are lost and the amplitude of the peaks at 548 and 554 nm is reversed as compared to the holoprotein, as shown in (b).", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-9.png")
```

From the _T. pantotropha_ crystal structure, we know that the Tyr-25 ligand originates in the _c_ haem domain. Hence this result could be interpreted as showing that the _d_$_1$ haem domain has a subtle effect on the absorption spectrum of the _c_ haem, which is altered when the _d_$_1$ haem is removed. The apparent lack of this effect in the _Ps. stutzeri_ enzymes might be taken as evidence that a similar link between the _c_ and _d_$_1$ haem domains does not exist in these proteins. A second interpretation is that _d_$_1$ haem loss is more extensive during the purification of the _Ps. stutzeri_ enzymes, yielding a species more like the semi-apo enzyme. However, the observation of this spectrum by other groups and previously reported activity measurements [@weeg-aerssens_purification_1991] argue against this idea.

In fact, mounting spectroscopic evidence indicates that haem ligation is genuinely not identical in different cytochromes _cd_$_1$. Magnetic circular dichroism (MCD) and electron paramagnetic resonance (EPR) spectroscopy of the cytochrome _cd_$_1$ from _Ps. aeruginosa_ have indicated Met-His ligation at the _c_ haem and His-His ligation at the _d_$_1$ haem  [@sutherland_investigation_1986]. EPR and Mössbauer spectroscopy of the enzyme from _Thiobacillus denitrificans_ suggested His-His ligation at both haems [@huynh_mossbauer_1982]. More recently, the _T. pantotropha_ and _Ps. stutzeri_ proteins in the oxidised form have been compared using EPR and MCD spectroscopy at the University of East Anglia [@cheesman_two_1997]. The latter method in particular is regarded as the most accurate diagnostic of haem ligation [@andersson_comparative_1995]. The EPR and MCD data are interpreted as being consistent with His-His ligation at the _c_ haem in the _T. pantotropha_ enzyme although, interestingly, a very small proportion of the sample appears to have Met-His ligation. The _d_$_1$ haem ligands are more difficult to assign in the absence of data from model compounds, but are thought to be consistent with His-Tyr or possibly His-OH ligation in both cases. However, the _c_ haem in the oxidised _Ps. stutzeri_ enzyme is clearly Met-His ligated, as was previously reported for cytochrome _cd_$_1$ from _Ps. aeruginosa_ [@sutherland_investigation_1986].

In summary, it is apparent that there are significant differences in haem ligation amongst the cytochromes _cd_$_1$, though bis-histidyl ligation at the _d_$_1$ haem and methionyl histidyl ligation at the _c_ haem are commonly assigned to several members of the group. This brings us to the question of whether the _T. pantotropha_ cytochrome _cd_$_1$ crystal structure is anomalous or in some way misleading. There can be no doubt that the structure is “correct” in terms of the integrity of the diffraction data, its interpretation, the chain tracing methods used and correlation with the protein sequence; the question is whether the structure is physiologically relevant.

As described above, MCD spectra of the _T. pantotropha_ enzyme confirm the bis histidyl ligation seen in the oxidised crystal structure, whereas the enzymes from _Ps. stutzeri_ and _Ps. aeruginosa_ are thought to have Met-His ligation to the _c_ haem in the oxidised state. Recent work [@williams_time-resolved_1996] has shown that in reduced crystals of cytochrome _cd_$_1$ from _T. pantotropha_ (which do not crack when reduced by sodium dithionite, unlike crystals of the _Ps. aeruginosa_ enzyme [@akey_characterization_1980], dramatic changes in haem ligation are observed. Tyr-25 is seen to move away from the _d_$_1$ haem, as predicted by spectroscopic and mechanistic studies (Chapter 1, Section 1.5.4), but at the _c_ haem His-17 is no longer a ligand. Instead, a loop of residues (100-114) moves upwards in the structure and Met-106 becomes a new _c_ haem ligand. This arrangement means that the _T. pantotropha_ enzyme in the reduced state more closely resembles the predicted ligation of the enzymes from _Ps. aeruginosa_ and _Ps. stutzeri_; additionally Met-106 is a conserved residue in all cytochrome _cd_$_1$ sequences.

These data raise the possibility that the physiologically active form of _T. pantotropha_ cytochrome _cd_$_1$ is Met-His ligated at the _c_ haem at all times, and that the fully-oxidised His His ligated species is a relaxed or resting artefact which is not returned to in the catalytic cycle. Three alternative reaction schemes are outlined in Figure 3.10. 

```{r fig3-10, fig.cap="Alternative reaction schemes for the reduction of nitrite by cytochrome _cd_$_1$ starting from the fully oxidised enzyme. (A) The _c_-haem switches to Met-His ligation on reduction. After intramolecular electron transfer to the _d_$_1$ haem, the re-oxidised _c_-haem switches back to His-His ligation. The fully oxidised enzyme is regenerated. (B) The _c_-haem becomes Met-His ligated on reduction and remains in this state even after reoxidisation. The fully oxidised state is regenerated, but His-His ligation is not. (C) The _c_-haem becomes His-Met ligated on reduction. After internal electron transfer to the _d_$_1$ haem, the _c_-haem is instantly re-reduced by an external electron donor protein and stays in the Met-His ligation state. Neither His-His ligation nor the fully oxidised enzyme are regenerated. Note that all schemes assume Tyr-25 returns to the _d_$_1$ haem after NO release. Dashed arrows indicate intramolecular electron transfer. M = methionine, H = histidine, Y = tyrosine.", fig.align="center", echo=FALSE, out.width="75%"}
include_graphics(path = "figure/fig3-10.png")
```

In the first, the _c_ haem returns to His-His ligation when it becomes reoxidised following intramolecular electron transfer to the _d_$_1$ haem. In the second, the _c_ haem remains Met-His ligated after reoxidation and a fully-oxidised His-His species is not regenerated. In the third, a fully oxidised enzyme is not regenerated because re-reduction of the _c_ haem following the first intramolecular electron transfer to the _d_$_1$ haem is very rapid, preventing the formation of the oxidised His-His ligation state. Further data using reduced crystals of the _T. pantotropha_ enzyme [@williams_time-resolved_1996] has provided some insight into which of these schemes may be correct. It has proved possible to soak the reduced crystals in potassium nitrite and then rapidly freeze them, capturing what are proposed to be structural intermediates in the catalytic cycle. The fully-reduced enzyme molecules in the crystal carry two electrons, which allows two turnovers of the enzyme. The freeze-quenched structures contain molecules captured after the binding of the second nitrite molecule and different structures show either nitrite or nitric oxide bound at the _d_$_1$ haem. It is clear from the diffraction data that following the reduction of nitrite to nitric oxide at the _d_$_1$ haem, the _c_ haem has returned to His-His ligation, as in Figure 3.10a. Such a rearrangement of the haem ligands during catalysis was unexpected and it has been suggested that this is the rate-limiting step of the reaction [@williams_time-resolved_1996]. It is also suggested that the mechanism prevents reduction of the _d_$_1$ haem whilst NO is still bound; as His-17 and Tyr 25 are on the same stretch of the polypeptide chain, return of His-17 to the _c_ haem would facilitate the re-ligation of the _d_$_1$ haem by Tyr-25, displacing the bound NO. The _c_ haem would then be re-reduced, change to Met-His ligation and transfer its electron to the _d_$_1$ haem where Tyr-25 would be displaced to begin a new catalytic cycle.

However, the data from the experiments of Williams [@williams_time-resolved_1996] are not complete. Crystalline _cd_$_1$ molecules can undergo only two turnovers whereas in the periplasm it is possible that a constant supply of electrons from donor proteins would act to keep the _c_ haem reduced and in the Met-His state. Notably, stopped flow experiments using the enzyme from _Ps. aeruginosa_ showed that the _c_ haem was more than 90% reduced at all times [@silvestrini_reaction_1990]. The conservation of Met-106 in all cytochrome _cd_$_1$ sequences is also a strong argument for this residue to be more functionally important than His-17 in _T. pantotropha_ cytochrome _cd_$_1$. The issue will ultimately be resolved by further kinetic studies of cytochrome _cd_$_1$, in conjunction with site-directed mutagenesis of His-17 and Met-106.

The role of the non-conserved Tyr-25 _d_$_1$ haem ligand remains more problematic. It is clearly a ligand to the _c_ haem in the oxidised crystal structure and can also be seen near the haem, having moved away, in the reduced state. Notably however, it has not yet been observed to return to the _d_$_1$ haem in the crystal structure after NO release; this step remains hypothetical. Therefore at present it is not possible to say with certainty whether Tyr-25 plays a crucial role in the mechanism of _T. pantotropha_ cytochrome _cd_$_1$. The lack of sequence conservation in an otherwise highly homologous group of proteins does suggest that Tyr-25 may not be an obligatory residue. Alternatively we have to accept that for some reason, within a set of enzymes that are so strongly conserved as to suggest high structural homology, different functional residues are used for catalysis.

The clearest way to resolve this issue, as with the problem of haem ligation in the _c_ domain, will be using site-directed mutagenesis to generate changes at Tyr-25 and analysis of the resultant proteins. As with all site-directed mutagenesis experiments, care will have to be taken to distinguish effects arising due to gross perturbation of the protein tertiary structure from those that involve only alteration around the haem ligation site. Bearing this in mind, obvious candidates for substitution at Tyr-25 are (1) phenylalanine, the same basic structure but lacking the liganding -OH moiety, (2) serine, of small size with an -OH group and (3) histidine, of similar size with the possibility of acting as a ligand, having been tentatively identified in other cytochromes _cd_$_1$. This work is currently underway in our laboratory and some early data are presented in Chapter 6. Further crystal structures of other cytochromes _cd_$_1$ will also be of great interest. The purified cytochrome _cd_$_1$ from _Ps. stutzeri_ Zobell, described in Section 3.4.3, was used for crystallisation trials by Dr. Vilmos Fülöp in the Laboratory of Molecular Biophysics and gave small crystals, though not of diffraction quality. However, we have since learned that the group of Dr. E.N. Baker (Massey University, New Zealand) is close to solving this structure, which should be available in the near future. Very recently the structure of cytochrome _cd_$_1$ from _Ps. aeruginosa_ has been determined (V. Fülöp, personal communication), although the details have not yet been published. Some aspects of this structure are discussed in Chapter 8.
