# Cloning and expression of the *nirS* gene from *T. pantotropha* in *Escherichia coli* {#chapter04}

## Introduction

The aim of the work described in this chapter was to investigate the expression of the cloned *nirS* gene from *T. pantotropha* in *Escherichia coli*. Although *E. coli* does not synthesise the *d*$_1$ haem group, and so cannot express holocytochrome *cd*$_1$, expression studies in this organism are still valuable for two reasons. First, *E. coli* provides a convenient experimental system to test gene expression, due to its widespread use in molecular biology, the consequent large technical literature, number of methods and range of plasmids available. This means that experiments using *E. coli* can be carried out with relative ease and efficiency. Second it is probable that in expression systems that can synthesise the holocytochrome (see Chapters 5 and 6), certain site-directed mutants of the nitrite reductase, the ultimate aim of this work, will be inactive and the host cells will be unable to grow under denitrifying conditions because of the accumulation of toxic nitrite. However, using *E. coli* (Glockner et al., 1993 [@glockner_copper-containing_1993]) and *Ps. putida* (Silvestrini et al., 1992 [@silvestrini_expression_1992]) it has already proved possible to express the semi-apo enzyme containing only *c* haem and then to reconstitute the enzyme with purified *d*$_1$ haem. The latter approach may be a way of producing sufficient quantities of mutant enzyme for kinetic studies. Previously published data concerning the expression of cytochrome *cd*$_1$ in *E. coli* and the expression of other *c* type cytochromes, both exogenous and endogenous (Iobbi-Nivol et al., 1994 [@iobbi-nivol_reassessment_1994]) indicated that *c* haem attachment to the *T.pantotropha* enzyme should not pose a problem. However, a potential problem was that *E. coli* is quite distantly related to *T. pantotropha* and so aspects of its genetics such as promoter sequences and codon usage may be quite different.

Although seven *nirS* genes have now been sequenced (including that from *T. pantotropha*, Chapter 3), expression of cytochrome *cd*$_1$ from the cloned gene has only been studied in any detail for *Ps. aeruginosa*, *Ps. stutzeri* Zobell and *P. denitrificans*. Three organisms have been used as heterologous hosts for expression. Silvestrini et al. (1992) [@silvestrini_expression_1992] have expressed cytochrome *cd*$_1$ from *Ps. aeruginosa* in *Ps. putida*. In common with *E. coli* this organism does not make *d*$_1$ haem and thus the semi-apoprotein, containing *c*-haem was produced. Glockner et al. (1993) [@glockner_copper-containing_1993] have expressed cytochrome *cd*$_1$ from *Ps. stutzeri* Zobell in *Ps. aureofaciens* and *E. coli* with similar results. Ohshima et al. (1993) [@ohshima_cloning_1993] have expressed the *P. denitrificans* IFO 12442 *nirS* gene in *E. coli*, but in this case it was claimed that active holoprotein was synthesised. Nitric oxide reductase activity was also apparently detectable in this system. An alternative approach to expression of the *nirS* gene has been described by Zennaro et al. (1993) [@zennaro_nitrite_1993], who inactivated the *nirS* gene in *Ps. aeruginosa* using a tetracycline cassette and then complemented the mutants with the wild-type gene on a plasmid. This met with some success and is discussed further in Chapter 6.

In all cases where semi-apo cytochrome *cd*$_1$ is produced, it has proved possible to obtain holoprotein by reconstitution with purified haem *d*$_1$. This has been performed for *Ps. aeruginosa* (Silvestrini et al., 1992 [@silvestrini_expression_1992]) and *Ps. stutzeri* Zobell (Glockner et al., 1993 [@glockner_copper-containing_1993]). It is also possible to remove haem *d*$_1$ chemically from the purified native holoprotein and reconstitute the semi-apo enzyme prepared in this manner to recover most of the activity; this has been described for cytochrome *cd*$_1$ from *Ps. aeruginosa* (Hill and Wharton, 1978 [@hill_reconstitution_1978]) and *Ps. stutzeri* M300 (Weeg-Aerssens et al., 1991 [@weeg-aerssens_purification_1991]).

This chapter describes a series of experiments carried out to express cytochrome *cd*$_1$ from *T. pantotropha* in *E. coli*. First, mapping of the *nir* gene locus, which was performed to identify a suitable fragment that contained the full-length *nirS* gene for sub-cloning, is discussed. Cloning of the *nirS* gene into a broad-host range vector is then described. Expression of the gene in *E. coli* under aerobic and anaerobic conditions is analysed. The final section describes the partial purification of the expressed semi-apo cytochrome *cd*$_1$ and a first attempt to reconstitute this material with purified haem *d*$_1$. Throughout the chapter comparisons with the results of similar experiments from other laboratories are made.

## Results

### Sub-cloning of the full-length *nirS* gene from *T. pantotropha*

The *T. pantotropha* *nirS* gene was cloned initially as a set of four overlapping PCR products for sequencing (Chapter 3). For further work, a full length copy of the gene was required. In theory this could be produced by PCR using primers flanking the 5'- and 3' ends of the gene. PCR reactions using such primers (460F, nirSF, CT3R and nirS2R, Table 3.1, Chapter 3) were performed and appeared to produce fragments of the expected lengths (results not shown). However, extreme difficulty was experienced when attempting to clone the full length products. A large number of remedies were tried, including proteinase K treatment of the PCR products (Hitti and Bertino, 1994 [@hitti_proteinase_1994]), the use of TA cloning vectors and different methods of PCR product purification, but none of the putative clones obtained after blue-white selection contained the desired insert when analysed by restriction mapping. In all cases, it appeared that rearrangement of the DNA had occurred. At present, the best explanation for this is that a recurrent error during synthesis of the longer PCR products prevented successful cloning, perhaps because of the expression of a toxic product from the DNA.

It therefore became necessary to obtain a genomic clone of the *nirS* gene. Probing a cosmid library prepared from *T. pantotropha* genomic DNA by Southern blotting with a *nirS* gene probe (the MID1 PCR product, Figure 3.1, Chapter 3) failed to identify a positive clone. However, blots of total genomic DNA, digested with *PstI*, *NotI* and *DdeI*, gave single bands in the size range 4.5-6 kbp (results not shown). Attempts to clone these fragments were in progress, when a plasmid containing the *T. pantotropha* *nirS* gene was obtained from Thon de Boer, Vrije Universiteit, Amsterdam. The plasmid, pTNIR3, contains the *nirS* gene on a 7.5 kbp *SphI* fragment cloned in the vector pGEM7zf(+).

Plasmid pTNIR3 was analysed by restriction mapping so as to determine the best way of generating a fragment containing the *nirS* gene for further work. Figure 4.1 shows a restriction map of pTNIR3 obtained using six enzymes.

```{r fig4-1, fig.cap="Physical map of the plasmid pTNIR3 produced using six restriction enzymes. Plasmid pTNIR3, containing the _nirS_ gene from _T. pantotropha_, was obtained from Thon de Boer, Vrije Universiteit, Amsterdam. It consists of a 7.5 kbp _SphI_ fragment of genomic DNA cloned in pGEM7zf (+). The plasmid was digested with 6 enzymes, both singly and in combination, and the fragments sized by agarose gel electrophoresis to generate the map. Enzyme abbreviations: Sp, _SphI_; B, _BamHI_; E, _EcoRI_; H, _HindIII_; P, _PstI_; S, _SalI_", fig.align="center", echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

This indicated that the 7.5 kbp insert could be conveniently cut into two fragments of 5.1 kbp and 2.4 kbp using *PstI* and *SphI*. These fragments were cloned into the pBS(-) plasmid and the ends of the inserts were sequenced using universal M13 primers. The restriction mapping data in conjunction with the partial DNA sequences were used to align the plasmid pTNIR3 and its derivatives with the sequence of the *nir* region from *P. denitrificans* PD1222 (de Boer et al., 1994 [@de_boer_isolation_1994]), as shown in Figure 4.2.

```{r fig4-2, fig.cap="Location of denitrification genes of _T. pantotropha_ on the plasmid pTNIR3 as determined by physical mapping and limited sequencing. Plasmid pTNIR3 was digested with _PstI_ and _SphI_ and the fragments were sub-cloned into pBS(-). The ends of the fragments were then sequenced using M13 universal primers. The plasmids pTNIR3, pB5K and pBNIR are aligned to previously published sequence from _P. denitrificans_ PD1222, covering the _nir_ and _nor_ loci (U05002 and U28078 respectively). No direct evidence was obtained for the presence of the _nirJ_ gene in _pTNIR3_, although it is assumed to be present by analogy to _P. denitrificans_ PD1222 (de Boer 1996) [@de_boer_regulation_1996].", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

The 5.1 kbp fragment lies upstream of the *nirS* gene and by analogy to *P. denitrificans* PD1222 would contain the genes *nirI*, *nirJ*, *norС* and part of the *norB* gene, in which lies the *SphI* site. The plasmid containing this fragment is designated pB5K. The 2.4 kbp fragment contains the *nirS* gene, with the *PstI* site at the 5'-end of the gene, just upstream of the NNR box, and the *SphI* site lying in the early region of the downstream *nirE* gene. This plasmid was designated pBNIR and was selected for further sub-cloning and expression experiments.

### Cloning the *nirS* gene into the broad host range expression vector pMMB67EH

The broad host range expression plasmid pMMB67EH (Fürste et al., 1986 [@furste_molecular_1986]) was chosen as a suitable vector for expression experiments. A diagram of the vector is shown in Figure 4.3.

```{r fig4-3, fig.cap="Diagram of the broad host range expression vector pMMB67EH, showing relevant features. pMMB67EH (Fürste et al., 1986 [@furste_molecular_1986]) was used for cloning and expression of the _nirS_ gene from _T. pantotropha_. Features are described fully in the main text and are as Tollows: _bla_, $\beta$-lactamase gene conferring ampicillin resistance, P$_{tac}$, _tac_ promoter, _rrnB_, terminator derived from the _E. coli_ 5S rRNA gene, _oriV_, vegetative origin of replication, _oriT_, origin of transfer, _lacI_$^q$, over-produced form of the _lacI_ repressor protein. The vector contains an mp18 multiple cloning site and 6 of the restriction enzyme sites that cut in this region are indicated.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

It contains a number of useful features, which are as follows: (1) A *tac* promoter, which is a strong hybrid promoter constructed from the *trp* and *lac* promoters (de Boer et al., 1983 [@de_boer_tac_1983]) for driving transcription, (2) the mp18 multiple cloning site which is compatible with the pBS(-) plasmid, in two possible orientations (these are designated pMMB67EH and HE), (3) a transcriptional terminator, derived from the *rrnB* gene, (4) the $\beta$-lactamase gene conferring ampicillin resistance, (5) the *lacI*$^q$ gene, encoding a mutant form of the *lacI* repressor (Gronenborn, 1976 [@gronenborn_overproduction_1976]) which is overproduced to prevent transcription in the absence of IPTG and (6) an origin of transfer, *oriT*, allowing transfer of the plasmid to other Gram-negative species by conjugation (Derbyshire and Willetts, 1987 [@derbyshire_mobilization_1987]). This last feature allows the use of the same construct to investigate expression in other bacteria, which is described in Chapters 5 and 6.

In theory the 2.4 kbp *PstI-SphI* fragment derived from the plasmid pBNIR should be easily cloned into the mp18 cloning site of pMMB67EH. However, difficulty was experienced due to the problem of vector carryover as shown in Figure 4.4.

```{r fig4-4, fig.cap="Restriction digests illustrating the problem of vector carryover during cloning of the _T. pantotropha_ _nirS_ gene into pMMB67EH. A 2.4 kbp _PstI-SphI_ fragment of pBNIR was isolated and used in a ligation reaction with pMMB67EH, also digested with _PstI-SphI_. _E. coli_ JM83 was then transformed with the ligation mix and ampicillin resistant clones screened for the presence of the desired plasmid by restriction digests using _PstI_ plus _SphI_ and electrophoresis on a 0.8% agarose gel. However, in this case, all the plasmids recovered are either the original pBNIR plasmid (lanes 3, 6 and 7) or the pMMB67EH plasmid (lanes 1, 2, 4, 5 and 8) that has re-ligated due to incomplete digestion. Solutions to the problem are described in the text. The gel is displayed as a negative image to show faint bands more clearly. Lanes labelled III and II contain DNA size standards, the sizes of some of which are indicated at the left of the gel.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

This problem occurs when a fragment of similar size to the vector fragment is isolated from an agarose gel. A small amount of vector can be carried over with the desired fragment and, as pBS(-) is smaller than pMMB67EH, religation of the original plasmid occurs with higher efficiency producing clones containing the original vector. This is a particular problem when the two vectors contain the same antibiotic resistance determinant (in this case ampicillin). To circumvent this problem, a kanamycin resistance cassette was inserted in the *PstI* site of pBNIR to give the plasmid pBNIRK. The cassette and *nirS* gene were then isolated together as an *XbaI-SphI* fragment and cloned into pMMB67EH. This step introduces a new antibiotic resistance determinant and also increases the size of the insert by 1.2 kbp, thus improving its separation from the vector fragment. Several positive clones were selected by kanamycin resistance. The isolated plasmid, pMMBSEK, was then cut with *PstI* and subjected to partial religation to remove the kanamycin cassette. Cells of *E. coli* JM83 were re-transformed and ampicillin resistant clones were streaked onto plates containing either ampicillin or ampicillin plus kanamycin. Kanamycin sensitive clones, which had lost the cassette, were re-isolated and the construct was confirmed by restriction digests (Figure 4.5).

```{r fig4-5, fig.cap="Restriction digest analysis of the plasmids pMMBSE and pMMBSEK. Digests of pMMBSE were performed using _SalI_ (S), _BamHI_ (B), _HindIII_ (H), _BglII_ (Bg), _EcoRI_ (E) and _PstI_ (P) and the fragments were separated by electrophoresis on a 0.8% agarose gel. The fragments shown above are of the expected sizes from the 2.4 kbp fragment of pBNIR, containing the _nirS_ gene, in pMMB67EH. Lane 7 contains a _HindIII_ digest of the plasmid pMMBSEK. This clearly shows an extra fragment of 1.38 kbp from pMMBSEK, which arises due to an extra _HindIII_ site in the centre of the kanamycin cassette. The lane labelled III contains DNA size markers, the sizes of some of which are indicated at the left of the gel.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

This procedure yielded two plasmids for further work: (1) pMMBSE, containing the *nirS* gene together with its upstream region (the NNR box and promoter region) and part of the downstream *nirE* gene, and (2) pMMBSEK, in which the *tac* promoter and the NNR box are separated by a kanamycin resistance cassette, transcribed in the same direction as the *nirS* gene. A list of the plasmids referred to in this chapter is given in Table 4.1.

```{r table41, echo=FALSE, results='asis'}
table4_1 <- data.frame(`Plasmid name` = c("pBS(-)", "pMMB67EH", "pTNIR3", "pBNIR", "pB5K", "pMMBSEK", "pMMBSE"),
                       Description = c("General purpose cloning vector", "Broad host range expression vector", "7.5 kbp _SphI_ fragment of genomic DNA from _T. pantotropha_ containing _nirS_ and flanking DNA", "2.4 kbp Pstl-Sphl fragment of pTNIR3, containing the _nirS_ gene, cloned in pBS(-)", "5.1 kbp _PstI-SphI_ fragment of pTNIR3, containing the genes _nirI-norB_, cloned in pBS(-)", "2.4 kbp _Xba-SphI_ fragment of pBNIR, containing the _kan_ gene (kanamycin resistance) in the _PstI_ site, and the _nirS_ gene, cloned in pMMB67EH", "pMMBSEK cut with _PstI_ to remove kan gene and religated
"),
                       `Source or reference` = c("Stratagene", "Fürste et al. (1986) [@furste_molecular_1986]", "Obtained from Thon de Boer, Vrije Universiteit, Amsterdam", "This work", "This work", "This work", "This work"),
                       check.names = FALSE)

kable(table4_1,
      escape = FALSE,
      caption = "Plasmids referred to in this chapter")
```

### Expression from pMMBSE in *E. coli* JM83 under different growth conditions

In order to obtain a complete analysis of expression from the *nirS* gene in *E. coli* JM83, cells of wild-type *E. coli* JM83, *E. coli* JM83 [pMMB67EH], *E. coli* JM83 [pMMBSEK] and *E. coli* JM83 [pMMBSE] were grown under both aerobic (in LB medium) anaerobic (minimal medium containing fumarate plus nitrite) conditions. Additionally, cultures of *E. coli* JM83 [pMMBSEK] and *E. coli* [pMMBSE] were induced with 1 mM IPTG during the early exponential phase of growth to activate transcription from the *tac* promoter. Twelve different cultures were grown as outlined in Table 4.2.

```{r table42, echo=FALSE, results='asis'}
table4_2 <- data.frame(`Culture number` = 1:12,
                       `Growth conditions` = c("aerobic", "anaerobic", "aerobic", "anaerobic", "aerobic", "aerobic", "anaerobic", "anaerobic", "aerobic", "aerobic", "anaerobic", "anaerobic"),
                       `Plasmid present` = c("", "", "pMMB67EH", "pMMB67EH", "pMMBSEK", "pMMBSEK", "pMMBSEK", "pMMBSEK", "pMMBSE", "pMMBSE", "pMMBSE", "pMMBSE"),
                       `IPTG added` = c("no", "no", "no", "no", "no", "yes", "no", "yes", "no", "yes", "no", "yes"),
                       check.names = FALSE)

kable(table4_2,
      escape = FALSE,
      caption = "Different growth conditions used to compare the growth of _E. coli_ JM83 with that of _E. coli_ containing either pMMB67EH, pMMBSEK or pMMBSE. Wild-type _E. coli_ JM83 and _E. coli_ JM83 [pMMB67EH] were grown either aerobically or anaerobically. _E. coli_ JM83 containing either pMMBSEK or pMMBSE were also grown under these conditions; additionally, these strains could be induced with IPTG during growth under either condition. This gave rise to the 12 different cultures indicated in the table.")
```

50 ml of each medium was inoculated with a 1:100 dilution of overnight culture that had been grown in 1.5 ml LB medium plus appropriate antibiotics. Aerobic cultures were sampled every 30 minutes and the absorbance at 650 nm was recorded to give the growth curves shown in Figure 4.6.

```{r fig4-6, fig.cap="Comparison of the aerobic growth of _E. coli_, E. coli [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]. Overnight cultures of wild-type cells and those carrying plasmid were inoculated into LB medium at a dilution of 1/100 and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). Symbols: (&#9670;) _E. coli_ JM83, (&#9632;) _E. coli_ [pMMB67EH], (&#9650;) _E. coli_ [pMMBSEK], (x) _E. coli_ [pMMBSEK] induced with 1 mM IPTG, (&#9633;) _E. coli_ [pMMBSE], (&#9679;) _E. coli_ [pMMBSE] induced with 1 mM IPTG.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Reliable growth curves were not obtained for the anaerobic cultures, because in the cultures containing the *nirS* gene the cells tended to sediment and adhere quite strongly to the lower wall of the culture tube. The cells could be resuspended by vigorous vortexing, but this was avoided to maintain anaerobic conditions in the tubes. Instead, anaerobic cultures were vortexed once after 12 hours to ascertain that they had entered early exponential phase. At this point, cultures containing *E. coli* [pMMBSE] and E. coli [pMMBSEK] were induced with 1 mM IPTG and the cultures were left overnight for a total of 24 hours. Comparison with the early growth curve of the anaerobic culture of *E. coli* [pMMB67EH] indicated that this growth period was sufficient to reach stationary phase without over-growth (Figure 4.7).

```{r fig4-7, fig.cap="Comparison of the anaerobic growth of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH]. Overnight cultures of wild-type cells and those carrying pMMB67EH were inoculated into _E. coli_ anaerobic medium (Materials and Methods) and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). As explained in the main text, growth curves for cells containing the other plasmids, pMMBSE and pMMBSEK, could not be obtained. Symbols: (&#9670;) _E. coli_ JM83, (&#9632;) _E. coli_ [pMMB67EH].", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

After 24 hours, the cultures were vortexed to record the final optical density and then harvested. Aerobic cultures were harvested when judged to have just entered stationary phase. Total soluble extracts were prepared from each culture by sonication and ultracentrifugation of the lysate.

The protein profile in the extracts from each of the 12 cultures was first investigated using SDS-PAGE and staining for protein using Coomassie Blue (Figure 4.8).

```{r fig4-8, fig.cap="Comparison of the protein content of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]. 40 $\\mu$g of total protein per lane was separated by SDS-PAGE on an 8% acrylamide gel and the gel was stained for protein using Coomassie Brilliant Blue R-250. Lane M contains molecular weight markers, the sizes of some of which are indicated at the left of the gel. The lanes are labelled to indicate the growth conditions, plasmid present and presence or absence of IPTG. A 63 kDa band, corresponding to expressed _T. pantotropha_ cytochrome _cd_$_1$, is visible in lanes 4 and 8.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Equal amounts of protein from each sample were loaded in each lane and separated on 8% acrylamide gels. A band corresponding to the size of mature cytochrome *cd*$_1$ (63 kDa) was apparent only in the samples induced using IPTG from *E. coli* [pMMBSE], grown either aerobically or anaerobically, and is more prominent in the latter sample.

Next, an identical set of samples was separated by SDS-PAGE, transferred to nitrocellulose by semi-dry electroblotting and analysed by Western blotting using a mouse antibody to *P. denitrificans* PD1222 cytochrome *cd*$_1$ (obtained from Thon de Boer, Vrije Universiteit, Amsterdam). For comparison, a soluble extract from *T. pantotropha* grown anaerobically with nitrate was run on the same gel. Only the aerobic and anaerobic samples from *E. coli* [pMMBSE] that had been induced using IPTG gave a positive reaction for the presence of cytochrome *cd*$_1$ (Figure 4.9).

```{r fig4-9, fig.cap="Western blot of protein samples from _E. coli_ JM83, _E. coli_ [pMMB67EH], E. coli [pMMBSEK] and _E. coli_ [pMMBSE], probed using antibody to cytochrome _cd_$_1$. 40 $\\mu$g of protein from the cultures indicated above the lanes was separated by electrophoresis on an 8% SDS-PAGE gel and transferred to nitrocellulose by semi-dry electroblotting. The membrane was then probed a cytochrome _cd_$_1$ antibody, detected colorimetrically with a secondary antibody coupled to alkaline phosphatase, as detailed in Materials and Methods. The lane labelled Tp contained 20 $\\mu$g of protein from a soluble extract of wild-type _T. pantotropha_ grown anaerobically with nitrate.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

The band in the anaerobic sample was somewhat stronger than that in the aerobic lane, and both bands ran at the same position as the wild type protein.

Finally, the samples were analysed for the presence of *c*-type cytochromes by SDS PAGE followed by haem staining. A sample of purified cytochrome *cd*$_1$ from *T. pantotropha* was run on this gel for comparison (Figure 4.10).

```{r fig4-10, fig.cap="Comparison of the _c_-type cytochrome content of _E. coli_ [pMMBSE] grown under aerobic or anaerobic conditions. 100 $\\mu$g of protein from the soluble extracts indicated was separated by electrophoresis on an 8% SDS-PAGE gel and stained for the presence of covalently-attached haem using tetramethylbenzidine and H$_2$O$_2$, as described in Materials and Methods. The lane labelled Tp contained 5 $\\mu$g of purified cytochrome _cd_$_1$ for comparison.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Once again, the only positive lanes were those containing protein samples from *E. coli* [pMMBSE] grown either aerobically or anaerobically. The anaerobic sample stained appreciably more strongly than the aerobic sample, and the mobilities of the haem stained bands were identical to that of the wild-type cytochrome *cd*$_1$.

The soluble extract from anaerobically-grown *E. coli* [pMMBSE] was pink. A sample of the extract was analysed using UV-visible spectroscopy (Figure 4.11).

```{r fig4-11, fig.cap="Visible spectrum of soluble extract from _E. coli_ [pMMBSE] grown anaerobically and induced with 1 mM IPTG. 1 ml of soluble extract was prepared from 50 ml of cells and used directly for spectroscopy in 50mM Tris-HCl, pH 8.0. Absorbance maxima are at 416 (not shown), 522, 548 and 554 nm.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

The spectrum obtained was extremely similar to that of chemically-prepared semi-apo cytochrome *cd*$_1$ in the reduced form and showed absorbance maxima at 416, 521, 548 and The peaks at 548 and 554 nm were reversed in magnitude when compared to the spectrum of the holoprotein, as was observed with the chemically-prepared semi-apoprotein (Chapter 3, Figure 3.9).

To partially purify the expressed semi-apo cytochrome *cd*$_1$, a 1 l culture of *E. coli* [pMMBSE] was grown anaerobically, induced with 1 mM IPTG in early exponential phase and harvested at stationary phase, the final A$_{650}$ being around 0.7. A soluble extract was prepared and subjected to ion exchange chromatography on a small DEAE-Sepharose column. The elution profile is shown in Figure 4.12.

```{r fig4-12, fig.cap="Elution profile during ion exchange chromatography of soluble extract from anaerobically-grown _E. coli_ [pMMBSE] induced using IPTG. 1 l of cells was grown anaerobically to stationary phase and a soluble extract was prepared by sonication in 50 mM Tris-HCl pH 8.0 followed by ultracentrifugation. The extract was bound to a DEAE-Sepharose CL6B column (1 x 10 cm) equilibrated in the same buffer and eluted with a 200 ml gradient (0-400 mM), also in 50 mM Tris-HCl pH 8.0. Symbols: Elution of haem proteins was followed by absorption at 410 nm (&#9632;), NaCl gradient (&#9670;).", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Three major peaks absorbing at 410 nm were apparent, the last which eluted at around 250 mM NaCl which is the expected position for cytochrome *cd*$_1$. Fractions from this peak were pooled and concentrated, and the spectrum of the pooled fractions was recorded (Figure 4.13).

```{r fig4-13, fig.cap="UV-visible spectrum of recombinant semi-apo cytochrome _cd_$_1$ following ion-exchange chromatography. The peak fractions (28-31) in Figure 4.12 were pooled and concentrated to approximately 200 $\\mu$l. 40 $\\mu$l of the concentrated material was then diluted 5-fold in 50 mM Tris-HCl pH 8.0 and the spectrum was recorded. Absorbance maxima are at 255, 294, 317, 416, 522,548 and 554 nm.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

Reconstitution of the semi-apoprotein was attempted using haem *d*$_1$ purified from a stock of native cytochrome *cd*$_1$. In the reconstitution experiment 130 $\mu$l of concentrated material from the ion exchange column, containing approximately 18.8 $\mu$M semi-apo cytochrome *cd*$_1$ was mixed with 150 $\mu$l of haem *d*$_1$, prepared from approximately 40 mg of partially purified cytochrome *cd*$_1$, in a total volume of 1 ml containing 50 mM Tris-HCl, pH 8.0. The solution was incubated at 4 °C for 30 minutes, with gentle mixing by inversion. On mixing the colour of the solution changed from pink to green. The sample was then diluted to 15 ml with 50 mM Tris-HCl pH 8.0, concentrated in a 50 kDa cut-off centrifugal filtration unit, diluted once more to 15 ml and reconcentrated to approximately 300 $\mu$l, with the aim of retaining cytochrome *cd*$_1$ plus bound haem and removing unbound haem *d*$_1$. After concentration, the spectrum of the sample was recorded as isolated and after the addition of sodium dithionite. These spectra are shown in Figure 4.14.

```{r fig4-14, fig.cap="Visible spectra of recombinant cytochrome _cd_$_1$ following reconstitution with purified haem _d_$_1$ (a) as prepared, (b) following reduction using sodium dithionite, compared with (c) native cytochrome _cd_$_1$ from _T. pantotropha_. Approximately 2.4 nmol of recombinant semi-apo cytochrome _cd_$_1$ was reconstituted with haem _d_$_1$ as described in the main text and concentrated to approximately 300 $\\mu$l. A 200 $\\mu$l sample was used to record spectrum (a) and was then reduced using sodium dithionite to record spectrum (b). Spectrum (c) was recorded from 1.5 $\\mu$M of purified native cytochrome _cd_$_1$ in 50 mM Tris-HCl pH 8.0, also reduced using sodium dithionite.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

In the reduced form, the reconstituted protein exhibited a prominent shoulder at 460 nm and a broad peak at around 625 nm, both of which are diagnostic for the presence of bound haem *d*$_1$. The activity of the expressed semi-apo cytochrome *cd*$_1$, reconstituted cytochrome *cd*$_1$ and native cytochrome *cd*$_1$ (purified from *T. pantotropha*) was assayed using reduced methyl viologen as an electron donor. Assay traces for the three samples are shown in Figure 4.15 and estimates of turnover number are summarised below the traces.

```{r fig4-15, fig.cap="Assays for nitrite reductase activity using reduced methyl viologen as an electron donor: (a) recombinant semi-apo cytochrome _cd_$_1$, (b) recombinant cytochrome _cd_$_1$ following reconstitution with _d_$_1$ haem, (c) native cytochrome _cd_$_1$ purified from _T. pantotropha_. All assays were performed anaerobically in a volume of 1 ml as described in Materials and Methods. The traces show the decrease in absorbance at 600 nm as the methyl viologen becomes oxidised. The reaction rates used for the calculations described in the main text were obtained by subtraction of the drift rate (before nitrite addition) from the initial rate (after nitrite addition)", fig.align='center', echo=FALSE}
include_graphics(path = "figure/placeholder.png")
```

## Discussion

When grown aerobically all of the *E. coli* strains exhibited similar growth rates (Figure 4.6). The final optical density reached was similar for all strains carrying a plasmid selected by ampicillin resistance (pMMB67EH or pMMBSE), but slightly lower for plasmids selected by kanamycin resistance (pMMBSEK). As no cytochrome *cd*$_1$ was expressed from plasmid pMMBSEK (Figure 4.9), this effect seems likely to result from the stress of countering the antibiotic. Thus it is concluded that, aerobically, the *nirS* gene exerts no toxic effects on *E. coli* JM83.

Under anaerobic conditions, a greater disparity was observed between cultures of *E. coli* when compared with cultures carrying the different plasmids. Cells carrying the parent plasmid pMMB67EH, selected by ampicillin resistance, showed a pronounced lag when compared to the wild-type strain, taking around 24 hours to reach stationary phase as compared to around 16 hours for the wild-type (Figure 4.7) and the final optical density reached was slightly lower. This again suggests that the combined stresses of growing anaerobically in the minimal medium and countering the effects of the antibiotic acted together to slow the growth of the cells. Optical densities of cultures containing anaerobically grown *E. coli* [pMMBSE] and *E. coli* [pMMBSEK] could not be accurately determined because the cells in these cultures tended to sediment and adhere strongly to the walls of the tube. However, the final density of these cultures did appear to be somewhat lower than that of *E. coli* [pMMB67EH] grown anaerobically. The reduced growth rate and adherence to the walls of the tube occurred in *E. coli* [pMMBSE] and *E. coli* [pMMBSEK] irrespective of the presence of IPTG and so do not appear to be a consequence of expression of the *nirS* gene. It seems that cells in the latter two cultures were experiencing stress arising from an as-yet unidentified factor. One possible cause might be expression of the truncated *nirE* gene during anaerobic growth. The first 648 bp (encoding 216 amino acids) of the *nirE* gene are present on the plasmids pMMBSE and pMMBSEK and it is possible that the truncated gene product is toxic to the cells. Plasmids containing the upstream region of the *nirE* gene have also been observed to reduce yields of cells and DNA during routine cloning procedures in *E. coli* (Dr. Simon Baker, personal communication).

A Western blot of extracts from cells grown under various conditions (Figure 4.9) showed that cytochrome *cd*$_1$ was expressed only from pMMBSE when induced using IPTG. No cytochrome *cd*$_1$ was detectable without induction, indicating that the endogenous *nirS* promoter is not functional in *E. coli* under either aerobic or anaerobic conditions. Further evidence for this assertion is supplied by strains carrying the plasmid pMMBSEK, a construct where a kanamycin cassette lies between the *tac* promoter and the NNR box. In this construct, transcription from the kanamycin gene promoter would be expected to interfere with transcription from the upstream *tac* promoter, but not with the downstream endogenous promoter region, were it to be active. This assumes that the NNR box and downstream regions contain all the necessary information for transcriptional activation of the *nirS* gene; Chapter 7 discusses the region in more detail, but by analogy to FNR-activated genes it is believed that the above assumption is true. In cultures of *E. coli* [pMMBSEK] no expression was observed under any growth condition, even when induced with IPTG.

Under anaerobic growth conditions expression of cytochrome *cd*$_1$ in *E. coli* was enhanced, as judged by Western blotting. *E. coli* has only been shown quite recently to synthesise a range of previously uncharacterised *c*-type cytochromes and it has been demonstrated that these are most abundant during anaerobic growth (Iobbi-Nivol et al., 1994 [@iobbi-nivol_reassessment_1994]). In the case of the endogenous *c*-type cytochromes the increase under anaerobic growth conditions is the result of several factors, including activation by FNR and other transcriptional activators, such as the NarX/L family (Darwin and Stewart, 1995 [@darwin_nitrate_1995]). However, for cytochrome *cd*$_1$ these factors are unlikely to be operative as the cloned fragment contains only the NNR box, which is probably not recognised by the *E. coli* FNR protein (Spiro, 1994 [@spiro_fnr_1994]). The increased amounts of cytochrome *cd*$_1$ under anaerobic growth conditions are due in large part to the increased expression levels of the cytochrome *c* biogenesis genes under these conditions (Thony-Meyer et al., 1995 [@thony-meyer_escherichia_1995]). The amount of covalently-attached *c* haem increased in parallel with the increased levels of cytochrome *cd*$_1$ polypeptide under anaerobic conditions (Figure 4.10), indicating that haem supply is increased anaerobically in *E. coli*. Previous studies have shown that expression of the *hemA* gene in *E. coli*, which encodes glutamyl-tRNA dehydrogenase, a haem precursor biosynthetic enzyme, is enhanced 2.5-fold under anaerobic as compared to aerobic growth conditions (Darie and Gunsalus, 1994 [@darie_effect_1994]). It has also been demonstrated that expression of cytochrome b$_5$ in *E. coli* provides a "haem sink", which increases the cellular haem content (Woodard and Dailey, 1995 [@woodard_regulation_1995]), and a similar effect may result from cytochrome *cd*$_1$ expression.

Western blotting and protein staining of the protein extracts showed that the expressed cytochrome *cd*$_1$ was the same size as native cytochrome *cd*$_1$ from *T. pantotropha* (Figures 4.8, 4.9). This indicates that the targeting sequence was recognised by the periplasmic export complex of *E. coli* and efficiently cleaved. It also suggests that the protein is exported to the periplasm. The fact that *c*-type haem is covalently attached also suggests correct targeting, as attachment of *c*-type haem is believed to take place in the periplasm (Page and Ferguson, 1990 [@page_apo_1990]). Thus it appears that recognition of the foreign *c*-type haem attachment site, processing and export to the periplasm can all occur in *E. coli*, implying that the steps are fundamentally similar to those in *P. denitrificans* and other Gram-negative bacteria. Similar observations have been made for other *c*-type cytochromes although the presence or absence of the *c*-type haem seems to depend very much on the particular protein. Examples of *c*-type cytochromes that have been successfully expressed as the holoprotein in *E. coli* include cytochrome *c*$_2$ from *Rhodobacter sphaeroides* (Grisshammer et al., 1991 [@grisshammer_expression_1991], though not the same protein from *R. viridis*), cytochrome *c*$_{550}$ from *P. denitrificans* (Sambongi et al., 1996 [@sambongi_alteration_1996]), cytochrome *c*$_{553}$ (Pollock and Voordouw, 1994 [@pollock_aerobic_1994]) and DcrA (Fu et al., 1994 [@fu_dcra_1994]) from *Desulfovibrio vulgaris*, mitochondrial cytochrome *c* from *Saccharomyces cerevisiae* (Sambongi et al., 1996 [@sambongi_alteration_1996]) and cytochrome *c*$_{550}$ from *Thiobacillus versutus* (Ubbink et al., 1992 [@ubbink_cytochrome_1992]). The optimal growth conditions also seem to be specific to each case; cytochrome *c*$_2$ required anaerobic growth, whilst *T. versutus* cytochrome *c*$_550$ was optimally expressed semi-anaerobically and the *D. vulgaris* cytochrome was optimally expressed under aerobic growth conditions. These observations have led to the suggestion that successful covalent haem attachment in *E. coli* may depend on the physical properties of the apoprotein. Anomalously, cytochrome *c*$_{552}$ from the thermophilic bacterium *Hydrogenobacter thermophilus* can apparently attach *c*-type haem in the cytoplasm of *E. coli*, as demonstrated using expression from a gene in which the periplasmic largeting sequence has been deleted (Sambongi and Ferguson, 1994 [@sambongi_synthesis_1994]). This is believed to result from spontaneous covalent attachment to the apoprotein, which possesses a highly ordered structure due to its thermostability.

Heterologous expression of *c*-type cytochromes is thus similar to that of other metalloproteins which have been expressed in *E. coli*, in that the complexity of the both the protein and its endogenous biosynthetic pathway both come to bear on the ease of expression. To give some examples, a number of proteins possessing the molybdopterin guanine dinucleotide cofactor (found in the dissimilatory nitrate reductases) have been successfully expressed (Garrett and Rajagapolan, 1994 [@garrett_molecular_1994]), as *E. coli* possesses several enzymes that contain the cofactor (and therefore the biosynthetic machinery to process such proteins) Iron-sulphur proteins have also been expressed relatively easily (Vidakovic et al., 1996 [@vidakovic_expression_1996]), as have simple copper proteins such as pseudoazurin (Leung et al., 1997 [@leung_pseudoazurin_1997]), but more complex copper proteins such as nitrous oxide reductase are expressed as the apoprotein (Viebrock and Zumft, 1988 [@viebrock_molecular_1988]) because of the absence of endogenous accessory gene products.

A spectrum of the soluble extract from anaerobically grown *E. coli* [pMMBSE] that had been induced using IPTG is shown in Figure 4.11. The quality of the spectrum was quite surprising, as it was very similar to that of pure semi-apo cytochrome *cd*$_1$, suggesting that there was very little contamination by endogenous *c*-type cytochromes and that the expressed cytochrome *cd*$_1$ was the major cytochrome *c* in the cells. In *E. coli* 712, a strain of *E. coli* that over-produces cytochromes *c*, five endogenous *c*-type cytochromes can be detected by haem staining after SDS-PAGE (Iobbi-Nivol et al., 1994 [@iobbi-nivol_reassessment_1994]). It appears that these cytochromes are less abundant in *E. coli* JM83, though a faint band corresponding to the *nrfA* gene product (a tetra- or possibly pentahaem nitrite reductase, Darwin et al., 1993 [@darwin_regulation_1993]) could be seen in some haem-stained extracts from *E. coli* [pMMBSE]. The expressed cytochrome *cd*$_1$ shows the same characteristic absorption spectrum as the reduced chemically prepared form, with peaks at 416, 521, 548 and 554 nm, all originating from the *c*-haem, and the reversal relative to the holoprotein in the amplitude of the split peak at 548-554 nm noted in Chapter 3. Additionally, a peak can be seen at 317 nm. This peak is ascribed to low-spin reduced *c*-haem (Barber et al., 1976 [@barber_spectral_1976]) and cannot normally be seen in the chemically-prepared semi-apo protein, as the addition of reducing agents such as sodium dithionite masks absorbance in this region. The expressed cytochrome *cd*$_1$ protein is reduced as isolated and remains stable in this form for several days at -20 °C. The same property and the 317 nm absorbance band have also been noted for the recombinant enzyme from *Ps. aeruginosa* when expressed in *Ps. putida* (Silvestrini et al., 1992 [@silvestrini_expression_1992]).

When assayed with reduced methyl viologen, expressed semi-apo cytochrome *cd*$_1$ is devoid of detectable nitrite reductase activity (Figure 4.15). This is unsurprising, as the *d*$_1$ haem has previously been shown to be the site of this reaction (Silvestrini et al., 1990 [@silvestrini_reaction_1990]), although it has been reported that about 5% of the cytochrome *c* oxidase activity of native cytochrome *cd*$_1$ can be restored to the chemically prepared semi-apo protein by reconstitution using *a*-type haem (Hill and Wharton, 1978 [@hill_reconstitution_1978]). However, in the case of the expressed protein in *E. coli* it seems that alternative haems are not inserted *in vivo*, as no activity was present and the only spectroscopic features were those of *c*-haem. In contrast to this result, Ohshima et al. (1993) [@ohshima_cloning_1993] reported that expression of the *nirS* gene from *P. denitrificans* IFO 12442 in *E. coli* gave measurable nitrite reductase and nitric oxide activities. However, the construct used in these experiments contained 10 kbp of downstream DNA in addition to the *nirS* gene and as the same downstream region in *P. denitrificans* PD1222 contains the genes for haem *d*$_1$ biosynthesis (de Boer et al., 1994 [@de_boer_isolation_1994]), it may be that the same genes are present in *P. denitrificans* IFO 12242, thus allowing the formation of some active cytochrome *cd*$_1$ in *E. coli*. The nitric oxide reductase activity is harder to explain, as the genes for this enzyme are upstream of the *nirS* gene in *P. denitrificans* PD1222 (de Boer et al., 1996 [@de_boer_mutational_1996]) (and in *T. pantotropha*, as shown by mapping the plasmid pTNIR3, Figure 4.2). It should be noted that the activities reported by Ohshima et al. (1993) [@ohshima_cloning_1993] are extremely low (in the order of units per gram of protein) and are barely above background level. Furthermore, the expressed protein was detected only by Western blotting and all work was performed with crude extracts, with no direct demonstration that holoprotein was formed (*e.g.* by partial purification and spectroscopy). The results of Ohshima et al. (1993) [@ohshima_cloning_1993] should therefore be viewed with some scepticism.

To reconstitute expressed cytochrome *cd*$_1$ using haem *d*$_1$ a soluble extract was prepared from 1 l of cells and partially purified on a small ion exchange column, using the same elution conditions as those employed during the purification of native cytochrome *cd*$_1$. Figure 4.12 shows the elution profile from this column, which confirms that semi-apo cytochrome *cd*$_1$, eluting at around 250 mM NaCl, was the major cytochrome *c* in the extract. SDS-PAGE of the peak fractions followed by staining for protein and haem showed the presence of a number of other proteins in the sample, although the expressed cytochrome *cd*$_1$ was the only haem protein in the peak (data not shown). Further purification of the sample was not considered to be a sensible option because of the relatively low amounts of expressed protein obtained (about 1 mg from 1 l). Other groups have reported reconstitution of whole e cell extracts (Glockner et al., 1993 [@glockner_copper-containing_1993]), so purity of the sample was not considered to be of paramount importance.

On mixing purified haem *d*$_1$ and expressed cytochrome *cd*$_1$ the pink colour was lost from the semi-apoprotein as the reduced *c*-haem was oxidised by the added haem *d*$_1$. After removal of excess haem *d*$_1$, the spectrum of the now oxidised protein was recorded (Figure 4.14a). The absorbance maxima were at similar positions to those in the native oxidised protein (407, 525 and 640 nm), although the peak at 407 nm was somewhat broader. The rise at around 640 nm due to the *d*$_1$ haem was not as pronounced as that seen in the native enzyme, although there was clearly some absorption in this region. However, there was a clear shoulder at around 440 nm, indicative of absorption by the *d*$_1$ haem.

The spectroscopic features became much clearer when the sample was reduced with sodium dithionite (Figure 4.14b). The double peak of the *c*-type haem was apparent, and the split peaks were of similar magnitude, which is indicative of holo cytochrome *cd*$_1$. There was also a peak at around 625 nm and the shoulder at 440 nm moved to 460 nm and increased in size. All of these features indicate that at least some of the protein has bound the *d*$_1$ haem to yield holoenzyme, with the 625 nm peak being shifted due to the binding of breakdown products from the sodium dithionite (Parr et al., 1974 [@parr_reaction_1974]). However, the attenuated size of the 460 nm shoulder and 625 nm peak as compared to the native reduced protein (Figure 4.14c) suggests that not all of the enzyme has been reconstituted, and the spectrum is therefore a composite of the holo- and semi-apo forms.

The holo- and semi-apo forms of cytochrome *cd*$_1$ can be further separated using ion exchange chromatography, as the holo- form is more negatively charged and so binds to a DEAE-Sepharose column more tightly. However, this step was not performed, again due to the low amounts of material obtained. The sample was assayed for nitrite reductase activity, using reduced methyl viologen as electron donor and a trace from this assay is shown in Figure 4.15. The reconstituted sample is clearly active as a nitrite reductase. Estimates of turnover number are quite difficult to obtain from this experiment, because the mixture of reconstituted holoprotein and remaining semi-apoprotein means that the amount of reconstituted material cannot be estimated spectrophotometrically, using previously determined extinction coefficients for cytochrome *cd*$_1$. However, a rough calculation was made as follows. 50 $\mu$l of reconstituted material was assayed in a volume of 1 ml, and a methyl viologen oxidation rate of 0.109 $\mu$mol min$^{-1}$ was recorded. Assuming that the absorbance of the reduced semi-apoprotein is not greatly different at 548 nm to that of the holoprotein, the extinction coefficient at 548 nm is 48.95 mM$^{-1}$ cm$^{-1}$, indicating that the material used in the 1 ml reconstitution reaction contained approximately 2.4 nmol of cytochrome *cd*$_1$. As 50 $\mu$l of this (= 0.4 nmol) was used in the assay, the turnover number is 109/0.4 giving approximately 273 $\mu$mol of methyl viologen oxidised per minute per $\mu$mol of total cytochrome *cd*$_1$ (*i.e.* reconstituted plus semi-apo protein). This is a similar figure to a typical assay of native cytochrome *cd*$_1$ from *T. pantotropha*. However, as already noted the spectrum of the reconstituted material (Figure 4.14b) differs to that of the native protein, indicating that not all of the protein has been reconstituted. Table 4.3 shows the ratios of some of the peak absorbances of the semi-apo, reconstituted and native proteins in the reduced and oxidised states.

```{r table43, echo=FALSE, results='asis'}
table4_3a <- data.frame(`Absorbance ratio` = c("418:522", "418:548", "418:554", "418:650", "548:650", "554:650"),
                       `native cytochrome _cd_$_1$ reduced` = c("7.48", "6.60", "6.68", "8.06", "1.22", "1.21"),
                       `expressed semi-apo cytochrome _cd_$_1$ reduced` = c("5.73", "5.72", "5.62", "", "", ""),
                       `reconstituted expressed cytochrome _cd_$_1$ reduced` = c("3.36", "3.32", "3.32", "4.25", "1.28", "1.28"),
                       check.names = FALSE)

table4_3b <- data.frame(`Absorbance ratio` =  c("407:525", "406:645", "526:645"),
                        `native cytochrome _cd_$_1$ oxidised` = c("6.84", "14.44", "15.31"),
                        `reconstituted expressed cytochrome _cd_$_1$ reduced` = c("2.59", "", ""),
                  check.names = FALSE)

kable(list(table4_3a, table4_3b),
      escape = FALSE,
      caption = "Comparison of the ratios of the absorbance maxima in (left) the reduced and (right) the oxidised states of native holo- and chemically prepared semi-apo cytochrome _cd_$_1$ from _T. pantotropha_ with those of the recombinant protein from _E. coli_.  Data for native cytochrome _cd_$_1$ was calculated using extinction coefficients obtained from Alrik Koppenhöfer; the absorbances for the recombinant protein were obtained from the spectra presented in this chapter.")
```

The figures indicate that the ratio of the Soret peak (the absorbance of which is contributed to by both *c* and *d*$_1$ haems) to peaks composed primarily of absorbance from the *c* haem is lower by a factor of about two in the reconstituted protein compared to native cytochrome *cd*$_1$. The spectroscopic features of the reduced reconstituted enzyme thus allow the crude estimate that the reconstitution was about 50% efficient.

A substantial amount of the maximum activity was, therefore, recovered following reconstitution of the expressed semi-apo enzyme, although it is considerably lower than that reported for reconstitutions of the chemically-prepared semi-apo enzyme from *Ps. aeruginosa* (Hill and Wharton, 1978 [@hill_reconstitution_1978]) and *Ps. stutzeri* JM300 (Weeg-Aerssens et al., 1991 [@weeg-aerssens_purification_1991]), in which almost 100% recovery was reported. There are a number of reasons why this might be the case. First, no attempt was made to optimise the procedure. Partial reconstitution of the recombinant semi-apoprotein was achieved quite easily in the present work, but attempts to reconstitute the chemically-prepared semi-apo cytochrome *cd*$_1$ from *T. pantotropha* in this laboratory have proved to be technically challenging and success requires quite specific salt concentrations, buffer, haem *d*$_1$:protein ratios and pH (A. Koppenhöfer, personal communication). It is probable that the procedure for reconstituting the recombinant semi-apo enzyme could be improved considerably by varying the conditions of the experiment. Secondly, the partial purity of the enzyme sample from *E. coli* may mean that some of the haem *d*$_1$ bound non-specifically to other proteins and so was not available for reconstitution.

Bearing the above points in mind, it is concluded that expression in *E. coli* is a viable system for producing recombinant holo-cytochrome *cd*$_1$ in the absence of an alternative system which can express the holoenzyme. The value of using *E. coli* is in the ability to express enzymically-inactive mutants of cytochrome *cd*$_1$ which could then be reconstituted, as outlined in the introduction to this chapter. There are a few disadvantages to the approach, one of which is that large quantities of native cytochrome *cd*$_1$ must be purified in order to extract haem *d*$_1$ for reconstitution (though in the above experiments, relatively small quantities were used due to the scale of the experiment). A second problem is the possibility of heterogeneity between reconstituted samples which may make it difficult to interpret the results of some experiments, such as enzyme kinetics. This could probably be overcome to some extent by optimisation and standardisation of the procedure, though it is unlikely that reconstituted samples will be suitable for crystallisation unless the reconstitution is very uniform and efficient. The yield of expressed protein in the present work was quite low (around 1 mg l$^{-1}$) and could probably be improved given time to optimise the conditions. Increased amounts of expressed protein would allow further purification of the sample, to reduce the possibility of non-specific binding of haem *d*$_1$ outlined above. In the only other quantitative study of heterologous *nirS* expression, around 3 mg l$^{-1}$ of recombinant protein was obtained from cells of *Ps. putida* (Silvestrini et al., 1992 [@silvestrini_expression_1992]). Improvements in yield using the present construct may be achieved by varying the time of induction with IPTG or possibly varying the IPTG concentration. Investigation of expression levels in other strains of *E. coli* which overproduce *c*-type cytochromes, such as *E. coli* 7120 and its derivatives, Darwin et al., 1993 [@darwin_regulation_1993]) might also be productive. The present construct, pMMBSE, is somewhat inconvenient; it would be better to try and clone only the *nirS* structural gene under the *tac* promoter, without the upstream region. This region contains an inverted repeat adjacent to the NNR box and removal of such structures has been demonstrated to improve in other cases (Yamamoto et al., 1987 [@yamamoto_blue_1987]). Removal of the truncated *nirE* gene may also be advantageous in view of the possible toxicity effects discussed previously. In addition to these experiments, a different vector system might be used. The pMMB67EH vector is quite large and cumbersome and several newer alternatives are now available, such as the pET vector systems (Studier et al., 1990 [@studier_use_1990]), in which transcription is driven from a T7 RNA polymerase promoter, using modified *E. coli* strains that have the gene for this polymerase integrated into their genomic DNA. Figures for protein expression in the range of tens of milligrams per litre are frequently quoted for the latter system (though success always depends on the specific nature of the protein under investigation) and it would be of interest to investigate the expression of cytochrome *cd*$_1$ in such a system.
