`r if(knitr:::is_latex_output()) '\\appendix'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 


# Comparison of the $\beta$-propeller structure in cytochrome _cd_$_1$ with other $\beta$-propeller structures

Residues 135-567 of _T. pantotropha_ cytochrome _cd_$_1$ form a rigid $\beta$-propeller domain which provides a binding pocket for the _d_$_1$ haem. The propeller structure consists of an eightfold repeat of a four-stranded antiparallel $\beta$-sheet motif and is so named because each set of four strands is arranged in a circular domain, resembling a propeller with eight blades. A schematic diagram of this domain is shown in Figure A1.

```{r figA-1, fig.cap="Schematic diagram of the $\\beta$-propeller _d_$_1$-haem binding domain of _T. pantotropha_ cytochrome _cd_$_1$. The symbol H represents histidine residues; on blade 1, His-200, blade 4, His-345 and blade 5, His-388. The “Velcro” closure of the ring is indicated by six parallel lines between the 2 $\\beta$-strands of blade 8, each line representing a hydrogen bond. The lollipop symbol represents conserved aspartate residues at the end of strand 3 within each blade.", fig.align='center', echo=FALSE}
include_graphics(path = "figure/figa-1.png")
```

The sheets are packed face to face, providing structural stability via mainly hydrophobic interaction between them. The rigidity of the propeller structure is also maintained by interactions that act to close (or “Velcro”) the propeller ring. In cytochrome _cd_$_1$ this is achieved by forming the last blade (conventionally assigned as the blade where closure occurs) from both termini of the propeller domain; the C-terminal innermost strand of blade 8 bonds via six main chain hydrogen bonds to one of the other three antiparallel sheets which are provided by the N-terminus.

Propeller structures are being found in an increasing number of proteins, from both prokaryotes and eukaryotes. Cytochrome _cd_$_1$ provides only the second example of an eight bladed propeller, the other being that of the heavy subunit of methanol dehydrogenase, the structure of which has been determined from _Methylobacterium extorquens_ (Ghosh et al., 1995 [@ghosh_refined_1995]) and _Methylophilus methylotrophus_ W3A1 (Xia et al., 1996 [@xia_determination_1996]). Other propeller structures are known with seven blades (fungal galactose oxidase from _Dactylium dendroides_ (Ito et al., 1991 [@ito_novel_1991]), methylamine dehydrogenase from _P. denitrificans_ (Vellieux et 1989 [@vellieux_structure_1989], and eukaryotic G-protein $\beta$-subunit, Sondek et al., 1996 [@sondek_crystal_1996]), six blades (influenza neuraminidase, Varghese et al., 1983 [@varghese_structure_1983], and bacterial sialidase, Gaskell et al., 1995 [@gaskell_three_1995]) and four blades (hemopexin, Faber et al., 1995 [@faber_18_1995], and collagenase, Li et al., 1995 [@li_structure_1995]). All propeller domains contain a central tunnel, preserved by the rigidity of the structure, which is used toco-ordinate a ligand or prosthetic group involved in catalysis or some other protein function. In the case of cytochrome _cd_$_1$, the channel is filled by the _d_$_1$ haem, which makes a large number of contacts with protein side chains and solvating water molecules lining the channel.

The emergence of several $\beta$-propeller structures prompted us to ask whether the cytochrome _cd_$_1$ crystal structure and sequence indicates any determinants that could account for such structures. This work was mostly carried out by Drs. Vilmos Fülöp and Simon Baker, but it results from analysis of the cytochrome _cd_$_1$ sequence reported in Chapter 3 and is published in a joint paper (Baker et al., 1997b [@baker_cytochrome_1997]), and so it is described here in some detail.

Direct sequence comparison of cytochrome _cd_$_1$ to methanol dehydrogenase, the other example of an eight-bladed propeller protein, reveals no significant homology. Despite this, the _d_$_1$ haem binding domain can be very closely superimposed onto the eight-bladed propeller of methanol dehydrogenase, using the program STAMP (Russell and Barton,
1992 [@russell_multiple_1992]), an alignment program for 3-D structures, rather than primary sequences. Essentially the same propeller can therefore be achieved with two different sequences, though in MDH ring closure is achieved differently; the last blade is formed from three C-terminal antiparallel sheets and an outermost sheet from the N-terminus. This feature is also seen in the other seven and six-bladed propeller structures. Visual inspection of the cytochrome _cd_$_1$ protein sequence reveals that the third sheet of each blade almost always ends with an aspartate residue (residues 183, 224, 272, 326, 370, 474 and 535, with Asp 420 displaced two residues from the C-terminal end of blade 5, sheet 3), as shown in Figure 3.13. These residues are predominantly exposed to the solvent and seven of the eight are superimposable on each other when the third sheets from each blade are aligned. Methanol dehydrogenase (MDH) is one of a family of bacterial dehydrogenases termed the PQQ-binding proteins (after the pyrolloquinoline quinone cofactor that they possess), which includes low specificity alcohol dehydrogenases (ADH) and periplasmic glucose dehydrogenases (GDH). Alignment of 15 sequences from this family to _M. extorquens_ MDH gives 23-98% identity, suggesting a common fold in the PQQ-binding domain; it has been suggested previously (Cozier et al., 1995 [@cozier_structure_1995]) that the $\alpha$-subunit of ADH from _Acetobacter aceti_ will possess an MDH-like eight-bladed $\beta$-propeller domain.

The STAMP program was used to overlay the structural primary sequence alignment of the MDH and cytochrome _cd_$_1$ $\beta$-propeller domains onto the 15 PQQ-binding protein sequences. This allowed the prediction of the positions of seven of the eight blades in these sequences (the eighth is harder to predict due to the differing method of ring closure described previously). The alignment revealed terminal $\beta$-sheet aspartate residues in four out of eight blades in the MDH sequences, at identical positions to those in cytochrome _cd_$_1$. Of the 105 predicted propeller blades (seven blades in the 15 sequences), 60 contained a terminal $\beta$-sheet aspartate residue, preceded by hydrophobic amino acids. In contrast, 37 of the predicted blades contained a previously identified feature named the "tryptophan docking motif" (Ghosh et al., 1995 [@ghosh_refined_1995]). This feature was first noted in methanol dehydrogenase and consists of a stabilising stacking interaction between the indole ring of a tryptophan residue on one propeller blade and a glycine residue on an adjacent blade. The feature is entirely absent from cytochrome _cd_$_1$ and cannot therefore be essential for propeller stability. The weak consensus derived from common residues in all the blades of the _d_$_1$ haem domain was also used to search for possible propeller structures in the protein sequence databases with less success. It is concluded that the terminal aspartate motif plays a common role in cytochrome _cd_$_1$ and the PQQ-binding proteins, but at present this role is unclear. The aspartates are exposed to the solvent and a role in structural integrity seems unlikely; they may be involved in the pathway of folding in the periplasm.

Although there seem to be relationships between the PQQ-binding proteins and the 8 bladed propeller of cytochrome _cd_$_1$, these cannot be related to a number of other features in known propeller structures. In addition to the aforementioned tryptophan docking motif, tryptophan residues are also involved in a large family of proteins called the “WD repeat” family, which includes the $\beta$-subunit of the eukaryotic G-proteins (Garcia-Higuera et al., 1996 [@garcia-higuera_folding_1996]). In these propeller structures, long repeats are found that terminate with the residues Trp-Asp, at the end of the third strand in each blade. The Trp residue is buried between sheets and interacts with an Asp-His-Ser/Thr triad, whilst the Asp residue stabilises the loop between the third and fourth strands of a blade by a number of hydrogen bonds. Again, this is very different to any structural motif found in cytochrome _cd_$_1$ or other propeller proteins, although Sondek et al. (1996) [@sondek_crystal_1996] have noted a weak sequence similarity between the WD-repeat proteins and some of the more strongly conserved repeats in fungal galactose oxidase. However, the third strand of each blade does terminate with Asp in the WD repeat proteins.

The six-bladed propeller protein bacterial sialidase contains a consensus repeat sequence Ser/Thr-Xaa-Asp-[Xaa]$_n$-Gly-Xaa-Thr-Trp/Phe, termed the “Asp box” (Gaskell et al., 1995 [@gaskell_three_1995]). This appears to dictate the fold and integrity of the propeller and again, the Asp residue falls near the end of strand 3 in each $\beta$-sheet. However, the Asp box is not present in cytochrome _cd_$_1$. A final type of motif, known as _Drosophila kelch_, has been identified in mouse MIPP protein, poxvirus and fungal galactose oxidase (Bork and Doolittle, 1994 [@bork_drosophila_1994]), where one motif is found in each of the seven blades. This does not contain the Asp residue prevalent in the other motifs and cannot be used to describe cytochrome _cd_$_1$. Finally, the four-bladed propeller structures (hemopexin and collagenase) are quite differently arranged to the other examples, in that adjacent strands in a sheet are perpendicular and not stacked on one another. Stabilisation in these structures is brought about by a calcium ion and a disulphide bond between the first and last blades. These features are not relevant to cytochrome _cd_$_1$.

In summary, although the propeller domain is a common feature to several proteins, the fold seems to be achieved in a number of different ways which are not dictated by sequence features common to all the proteins. Aspartate residues in cytochrome _cd_$_1$ and a number of other proteins, particularly the PQQ-binding proteins, seem to be important in
some as yet unknown way, but are not necessarily involved with the integrity of folding and do not form a clear basis for prediction of the fold from protein sequence alone. Each group of propeller proteins (cytochrome _cd_$_1$ type, PQQ-type, WD repeat-type, Asp-box-type and _kelch_-type) possesses some conserved residues. However, given the wide distribution of propeller proteins, with no fundamental common sequence features even at a low level, the model of propeller formation by divergent evolution proposed by Bork and Doolittle (1994) [@bork_drosophila_1994] seems less likely. Murzin (1992) [@murzin_structural_1992] suggests that propeller formation is merely a variation on the packing of $\beta$-sheets which can be extended from a consideration of the geometry that defines an ordinary $\beta$-sandwich protein and does not require any special sequence constraints. The analysis presented here seems to support this idea.


# Classification of _T. pantotropha_

Throughout the present work the principal organism under study has been referred to by the name _Thiosphaera pantotropha_. However, during the course of the project the taxonomic position of this organism has come under scrutiny by a number of workers. In particular, it has been suggested that the organism should be re-classified as _Paracoccus denitrificans_ (Ludwig et al., 1993 [@ludwig_transfer_1993]). This argument was based on two observations: first, the 16S rRNA sequence of _T. pantotropha_ is identical to that of _P. denitrificans_ strain LMG 4218 and second, 100% DNA homology is observed between these two organisms in hybridisation experiments. Additionally, some of the metabolic differences that originally distinguished _T. pantotropha_ from _P. denitrificans_ (Robertson and Kuenen, 1983 [@robertson_thiosphaera_1983]), particularly aerobic denitrification, appear to have become attenuated since the original isolation of _T. pantotropha_.

However, recent work has shown that there is some confusion as to the history of various strains of _P. denitrificans_ which are deposited in several culture collections around the world. The relationship between strains of _P. denitrificans_ has been re-investigated by examining the _c_-type cytochrome profiles using SDS-PAGE and by analysis of the protein sequences of cytochrome _c_$_{550}$ from each strain (Goodhew et al., 1996 [@goodhew_cytochromes_1996]). The analysis indicates that _P. denitrificans_ LMG 4218 is closely related to _T. pantotropha_, but also that _P. denitrificans_ LMG 4218 is not derived from the original type strain of _P. denitrificans_ (Beijerinck and Minkman, 1910 [@beijerinck_bildung_1910]), as was thought previously. Instead, _P. denitrificans_ LMG 4218 is placed in a group that includes a later strain of _P. denitrificans_, _P. denitrificans_ LMD 52.44. The revised grouping of the strains would affect the way in which they were sub-divided, were consideration to be given to the creation of separate species within the genus _Paracoccus_.

To address the latter question, fatty acid methyl ester analysis (FAMES) has been used (Thompson et al., 1997 [@thompson_variation_1997]). This technique uses gas chromatography to analyse the relative amounts of different fatty acids in the cell membranes of bacteria. Taxonomic classification based on these measurements has been used successfully (Rainey et al., 1994 [@rainey_genome_1994]) and the method is particularly well-suited to the grouping of strains at the sub-species level (Thompson et al., 1993 [@thompson_subgrouping_1993]). The results are so far unpublished, but they broadly support the assertion that _P. denitrificans_ LMD 52.44, _P. denitrificans_ LMG 4218 and _T. pantotropha_ form a separate group within the genus _Paracoccus_, and also correlate well with a recent analysis of 16S rRNA sequences which suggested that the organism _Thiobacillus versutus_ should be transferred to _Paracoccus versutus_ (Katayama et al., 1995 [@katayama_paracoccus_1995]). Although all of the _P. denitrificans_ strains studied were very similar in many respects, Thompson et al. (1997) [@thompson_variation_1997] assert that there is sufficient variation within the genus to propose the creation of separate species names, and it is suggested that _T. pantotropha_ be renamed _P. pantotrophus_, to reflect the findings and to promote continuity of the _pantotrophus_ species name.

The taxonomic status of _T. pantotropha_ therefore remains undecided, although some workers have recently adopted its transfer to _P. denitrificans_, based on the work of Ludwig et al. (1993) [@ludwig_transfer_1993]. However, in the present work the name _T. pantotropha_ has been retained, to avoid confusion between previous work in which this name was used and the much larger body of literature that refers specifically to _P. denitrificans_.
