<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A Comparison of the \(\beta\)-propeller structure in cytochrome cd\(_1\) with other \(\beta\)-propeller structures | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="A Comparison of the \(\beta\)-propeller structure in cytochrome cd\(_1\) with other \(\beta\)-propeller structures | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A Comparison of the \(\beta\)-propeller structure in cytochrome cd\(_1\) with other \(\beta\)-propeller structures | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="8-chapter08.html"/>
<link rel="next" href="B-classification-of-t-pantotropha.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures" class="section level1" number="9">
<h1><span class="header-section-number">A</span> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</h1>
<p>Residues 135-567 of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> form a rigid <span class="math inline">\(\beta\)</span>-propeller domain which provides a binding pocket for the <em>d</em><span class="math inline">\(_1\)</span> haem. The propeller structure consists of an eightfold repeat of a four-stranded antiparallel <span class="math inline">\(\beta\)</span>-sheet motif and is so named because each set of four strands is arranged in a circular domain, resembling a propeller with eight blades. A schematic diagram of this domain is shown in Figure A1.</p>
<div class="figure" style="text-align: center"><span id="fig:figA-1"></span>
<img src="figure/figa-1.png" alt="Schematic diagram of the $\beta$-propeller _d_$_1$-haem binding domain of _T. pantotropha_ cytochrome _cd_$_1$. The symbol H represents histidine residues; on blade 1, His-200, blade 4, His-345 and blade 5, His-388. The “Velcro” closure of the ring is indicated by six parallel lines between the 2 $\beta$-strands of blade 8, each line representing a hydrogen bond. The lollipop symbol represents conserved aspartate residues at the end of strand 3 within each blade." width="816" />
<p class="caption">
Figure A.1: Schematic diagram of the <span class="math inline">\(\beta\)</span>-propeller <em>d</em><span class="math inline">\(_1\)</span>-haem binding domain of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The symbol H represents histidine residues; on blade 1, His-200, blade 4, His-345 and blade 5, His-388. The “Velcro” closure of the ring is indicated by six parallel lines between the 2 <span class="math inline">\(\beta\)</span>-strands of blade 8, each line representing a hydrogen bond. The lollipop symbol represents conserved aspartate residues at the end of strand 3 within each blade.
</p>
</div>
<p>The sheets are packed face to face, providing structural stability via mainly hydrophobic interaction between them. The rigidity of the propeller structure is also maintained by interactions that act to close (or “Velcro”) the propeller ring. In cytochrome <em>cd</em><span class="math inline">\(_1\)</span> this is achieved by forming the last blade (conventionally assigned as the blade where closure occurs) from both termini of the propeller domain; the C-terminal innermost strand of blade 8 bonds via six main chain hydrogen bonds to one of the other three antiparallel sheets which are provided by the N-terminus.</p>
<p>Propeller structures are being found in an increasing number of proteins, from both prokaryotes and eukaryotes. Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> provides only the second example of an eight bladed propeller, the other being that of the heavy subunit of methanol dehydrogenase, the structure of which has been determined from <em>Methylobacterium extorquens</em><span class="citation"><sup><a href="#ref-ghosh_refined_1995" role="doc-biblioref">353</a></sup></span> and <em>Methylophilus methylotrophus</em> W3A1.<span class="citation"><sup><a href="#ref-xia_determination_1996" role="doc-biblioref">354</a></sup></span> Other propeller structures are known with seven blades (fungal galactose oxidase from <em>Dactylium dendroides</em>,<span class="citation"><sup><a href="#ref-ito_novel_1991" role="doc-biblioref">355</a></sup></span> methylamine dehydrogenase from <em>P. denitrificans</em>,<span class="citation"><sup><a href="#ref-vellieux_structure_1989" role="doc-biblioref">152</a></sup></span> and eukaryotic G-protein <span class="math inline">\(\beta\)</span>-subunit,),<span class="citation"><sup><a href="#ref-sondek_crystal_1996" role="doc-biblioref">356</a></sup></span> six blades (influenza neuraminidase<span class="citation"><sup><a href="#ref-varghese_structure_1983" role="doc-biblioref">153</a></sup></span> and bacterial sialidase)<span class="citation"><sup><a href="#ref-gaskell_three_1995" role="doc-biblioref">357</a></sup></span> and four blades (hemopexin<span class="citation"><sup><a href="#ref-faber_18_1995" role="doc-biblioref">358</a></sup></span> and collagenase).<span class="citation"><sup><a href="#ref-li_structure_1995" role="doc-biblioref">359</a></sup></span> All propeller domains contain a central tunnel, preserved by the rigidity of the structure, which is used toco-ordinate a ligand or prosthetic group involved in catalysis or some other protein function. In the case of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, the channel is filled by the <em>d</em><span class="math inline">\(_1\)</span> haem, which makes a large number of contacts with protein side chains and solvating water molecules lining the channel.</p>
<p>The emergence of several <span class="math inline">\(\beta\)</span>-propeller structures prompted us to ask whether the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> crystal structure and sequence indicates any determinants that could account for such structures. This work was mostly carried out by Drs. Vilmos Fülöp and Simon Baker, but it results from analysis of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequence reported in Chapter 3 and is published in a joint paper,<span class="citation"><sup><a href="#ref-baker_cytochrome_1997" role="doc-biblioref">262</a></sup></span> and so it is described here in some detail.</p>
<p>Direct sequence comparison of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> to methanol dehydrogenase, the other example of an eight-bladed propeller protein, reveals no significant homology. Despite this, the <em>d</em><span class="math inline">\(_1\)</span> haem binding domain can be very closely superimposed onto the eight-bladed propeller of methanol dehydrogenase, using the program STAMP,<span class="citation"><sup><a href="#ref-russell_multiple_1992" role="doc-biblioref">360</a></sup></span> an alignment program for 3-D structures, rather than primary sequences. Essentially the same propeller can therefore be achieved with two different sequences, though in MDH ring closure is achieved differently; the last blade is formed from three C-terminal antiparallel sheets and an outermost sheet from the N-terminus. This feature is also seen in the other seven and six-bladed propeller structures. Visual inspection of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> protein sequence reveals that the third sheet of each blade almost always ends with an aspartate residue (residues 183, 224, 272, 326, 370, 474 and 535, with Asp 420 displaced two residues from the C-terminal end of blade 5, sheet 3), as shown in Figure 3.13. These residues are predominantly exposed to the solvent and seven of the eight are superimposable on each other when the third sheets from each blade are aligned. Methanol dehydrogenase (MDH) is one of a family of bacterial dehydrogenases termed the PQQ-binding proteins (after the pyrolloquinoline quinone cofactor that they possess), which includes low specificity alcohol dehydrogenases (ADH) and periplasmic glucose dehydrogenases (GDH). Alignment of 15 sequences from this family to <em>M. extorquens</em> MDH gives 23-98% identity, suggesting a common fold in the PQQ-binding domain; it has been suggested previously<span class="citation"><sup><a href="#ref-cozier_structure_1995" role="doc-biblioref">361</a></sup></span> that the <span class="math inline">\(\alpha\)</span>-subunit of ADH from <em>Acetobacter aceti</em> will possess an MDH-like eight-bladed <span class="math inline">\(\beta\)</span>-propeller domain.</p>
<p>The STAMP program was used to overlay the structural primary sequence alignment of the MDH and cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <span class="math inline">\(\beta\)</span>-propeller domains onto the 15 PQQ-binding protein sequences. This allowed the prediction of the positions of seven of the eight blades in these sequences (the eighth is harder to predict due to the differing method of ring closure described previously). The alignment revealed terminal <span class="math inline">\(\beta\)</span>-sheet aspartate residues in four out of eight blades in the MDH sequences, at identical positions to those in cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Of the 105 predicted propeller blades (seven blades in the 15 sequences), 60 contained a terminal <span class="math inline">\(\beta\)</span>-sheet aspartate residue, preceded by hydrophobic amino acids. In contrast, 37 of the predicted blades contained a previously identified feature named the “tryptophan docking motif.”<span class="citation"><sup><a href="#ref-ghosh_refined_1995" role="doc-biblioref">353</a></sup></span> This feature was first noted in methanol dehydrogenase and consists of a stabilising stacking interaction between the indole ring of a tryptophan residue on one propeller blade and a glycine residue on an adjacent blade. The feature is entirely absent from cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and cannot therefore be essential for propeller stability. The weak consensus derived from common residues in all the blades of the <em>d</em><span class="math inline">\(_1\)</span> haem domain was also used to search for possible propeller structures in the protein sequence databases with less success. It is concluded that the terminal aspartate motif plays a common role in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and the PQQ-binding proteins, but at present this role is unclear. The aspartates are exposed to the solvent and a role in structural integrity seems unlikely; they may be involved in the pathway of folding in the periplasm.</p>
<p>Although there seem to be relationships between the PQQ-binding proteins and the 8 bladed propeller of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, these cannot be related to a number of other features in known propeller structures. In addition to the aforementioned tryptophan docking motif, tryptophan residues are also involved in a large family of proteins called the “WD repeat” family, which includes the <span class="math inline">\(\beta\)</span>-subunit of the eukaryotic G-proteins.<span class="citation"><sup><a href="#ref-garcia-higuera_folding_1996" role="doc-biblioref">362</a></sup></span> In these propeller structures, long repeats are found that terminate with the residues Trp-Asp, at the end of the third strand in each blade. The Trp residue is buried between sheets and interacts with an Asp-His-Ser/Thr triad, whilst the Asp residue stabilises the loop between the third and fourth strands of a blade by a number of hydrogen bonds. Again, this is very different to any structural motif found in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> or other propeller proteins, although Sondek et al. (1996)<span class="citation"><sup><a href="#ref-sondek_crystal_1996" role="doc-biblioref">356</a></sup></span> have noted a weak sequence similarity between the WD-repeat proteins and some of the more strongly conserved repeats in fungal galactose oxidase. However, the third strand of each blade does terminate with Asp in the WD repeat proteins.</p>
<p>The six-bladed propeller protein bacterial sialidase contains a consensus repeat sequence Ser/Thr-Xaa-Asp-[Xaa]<span class="math inline">\(_n\)</span>-Gly-Xaa-Thr-Trp/Phe, termed the “Asp box.”<span class="citation"><sup><a href="#ref-gaskell_three_1995" role="doc-biblioref">357</a></sup></span> This appears to dictate the fold and integrity of the propeller and again, the Asp residue falls near the end of strand 3 in each <span class="math inline">\(\beta\)</span>-sheet. However, the Asp box is not present in cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. A final type of motif, known as <em>Drosophila kelch</em>, has been identified in mouse MIPP protein, poxvirus and fungal galactose oxidase,<span class="citation"><sup><a href="#ref-bork_drosophila_1994" role="doc-biblioref">363</a></sup></span> where one motif is found in each of the seven blades. This does not contain the Asp residue prevalent in the other motifs and cannot be used to describe cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Finally, the four-bladed propeller structures (hemopexin and collagenase) are quite differently arranged to the other examples, in that adjacent strands in a sheet are perpendicular and not stacked on one another. Stabilisation in these structures is brought about by a calcium ion and a disulphide bond between the first and last blades. These features are not relevant to cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.</p>
<p>In summary, although the propeller domain is a common feature to several proteins, the fold seems to be achieved in a number of different ways which are not dictated by sequence features common to all the proteins. Aspartate residues in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and a number of other proteins, particularly the PQQ-binding proteins, seem to be important in
some as yet unknown way, but are not necessarily involved with the integrity of folding and do not form a clear basis for prediction of the fold from protein sequence alone. Each group of propeller proteins (cytochrome <em>cd</em><span class="math inline">\(_1\)</span> type, PQQ-type, WD repeat-type, Asp-box-type and <em>kelch</em>-type) possesses some conserved residues. However, given the wide distribution of propeller proteins, with no fundamental common sequence features even at a low level, the model of propeller formation by divergent evolution proposed by Bork and Doolittle (1994)<span class="citation"><sup><a href="#ref-bork_drosophila_1994" role="doc-biblioref">363</a></sup></span> seems less likely. Murzin (1992)<span class="citation"><sup><a href="#ref-murzin_structural_1992" role="doc-biblioref">364</a></sup></span> suggests that propeller formation is merely a variation on the packing of <span class="math inline">\(\beta\)</span>-sheets which can be extended from a consideration of the geometry that defines an ordinary <span class="math inline">\(\beta\)</span>-sandwich protein and does not require any special sequence constraints. The analysis presented here seems to support this idea.</p>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-vellieux_structure_1989" class="csl-entry">
(152) Vellieux, F. M., Huitema, F., Groendijk, H., Kalk, K. H., Jzn, J. F., Jongejan, J. A., Duine, J. A., Petratos, K., Drenth, J., and Hol, W. G. (1989) Structure of quinoprotein methylamine dehydrogenase at 2.25 <span>A</span> resolution. <em>The EMBO journal</em> <em>8</em>, 2171–2178.
</div>
<div id="ref-varghese_structure_1983" class="csl-entry">
(153) Varghese, J. N., Laver, W. G., and Colman, P. M. (1983) Structure of the influenza virus glycoprotein antigen neuraminidase at 2.9 <span>A</span> resolution. <em>Nature</em> <em>303</em>, 35–40.
</div>
<div id="ref-baker_cytochrome_1997" class="csl-entry">
(262) Baker, S. C., Saunders, N. F., Willis, A. C., Ferguson, S. J., Hajdu, J., and Fülöp, V. (1997) Cytochrome cd1 structure: Unusual haem environments in a nitrite reductase and analysis of factors contributing to beta-propeller folds. <em>Journal of Molecular Biology</em> <em>269</em>, 440–455.
</div>
<div id="ref-ghosh_refined_1995" class="csl-entry">
(353) Ghosh, M., Anthony, C., Harlos, K., Goodwin, M. G., and Blake, C. (1995) The refined structure of the quinoprotein methanol dehydrogenase from <span>Methylobacterium</span> extorquens at 1.94 <span>A</span>. <em>Structure (London, England: 1993)</em> <em>3</em>, 177–187.
</div>
<div id="ref-xia_determination_1996" class="csl-entry">
(354) Xia, Z., Dai, W., Zhang, Y., White, S. A., Boyd, G. D., and Mathews, F. S. (1996) Determination of the gene sequence and the three-dimensional structure at 2.4 angstroms resolution of methanol dehydrogenase from <span>Methylophilus</span> <span>W3A1</span>. <em>Journal of Molecular Biology</em> <em>259</em>, 480–501.
</div>
<div id="ref-ito_novel_1991" class="csl-entry">
(355) Ito, N., Phillips, S. E., Stevens, C., Ogel, Z. B., McPherson, M. J., Keen, J. N., Yadav, K. D., and Knowles, P. F. (1991) Novel thioether bond revealed by a 1.7 <span>A</span> crystal structure of galactose oxidase. <em>Nature</em> <em>350</em>, 87–90.
</div>
<div id="ref-sondek_crystal_1996" class="csl-entry">
(356) Sondek, J., Bohm, A., Lambright, D. G., Hamm, H. E., and Sigler, P. B. (1996) Crystal structure of a <span>G</span>-protein beta gamma dimer at 2.<span>1A</span> resolution. <em>Nature</em> <em>379</em>, 369–374.
</div>
<div id="ref-gaskell_three_1995" class="csl-entry">
(357) Gaskell, A., Crennell, S., and Taylor, G. (1995) The three domains of a bacterial sialidase: A beta-propeller, an immunoglobulin module and a galactose-binding jelly-roll. <em>Structure (London, England: 1993)</em> <em>3</em>, 1197–1205.
</div>
<div id="ref-faber_18_1995" class="csl-entry">
(358) Faber, H. R., Groom, C. R., Baker, H. M., Morgan, W. T., Smith, A., and Baker, E. N. (1995) 1.8 <span>A</span> crystal structure of the <span>C</span>-terminal domain of rabbit serum haemopexin. <em>Structure (London, England: 1993)</em> <em>3</em>, 551–559.
</div>
<div id="ref-li_structure_1995" class="csl-entry">
(359) Li, J., Brick, P., O’Hare, M. C., Skarzynski, T., Lloyd, L. F., Curry, V. A., Clark, I. M., Bigg, H. F., Hazleman, B. L., and Cawston, T. E. (1995) Structure of full-length porcine synovial collagenase reveals a <span>C</span>-terminal domain containing a calcium-linked, four-bladed beta-propeller. <em>Structure (London, England: 1993)</em> <em>3</em>, 541–549.
</div>
<div id="ref-russell_multiple_1992" class="csl-entry">
(360) Russell, R. B., and Barton, G. J. (1992) Multiple protein sequence alignment from tertiary structure comparison: Assignment of global and residue confidence levels. <em>Proteins</em> <em>14</em>, 309–323.
</div>
<div id="ref-cozier_structure_1995" class="csl-entry">
(361) Cozier, G. E., Giles, I. G., and Anthony, C. (1995) The structure of the quinoprotein alcohol dehydrogenase of <span>Acetobacter</span> aceti modelled on that of methanol dehydrogenase from <span>Methylobacterium</span> extorquens. <em>The Biochemical Journal</em> <em>308 ( Pt 2)</em>, 375–379.
</div>
<div id="ref-garcia-higuera_folding_1996" class="csl-entry">
(362) Garcia-Higuera, I., Fenoglio, J., Li, Y., Lewis, C., Panchenko, M. P., Reiner, O., Smith, T. F., and Neer, E. J. (1996) Folding of proteins with <span>WD</span>-repeats: Comparison of six members of the <span>WD</span>-repeat superfamily to the <span>G</span> protein beta subunit. <em>Biochemistry</em> <em>35</em>, 13985–13994.
</div>
<div id="ref-bork_drosophila_1994" class="csl-entry">
(363) Bork, P., and Doolittle, R. F. (1994) Drosophila kelch motif is derived from a common enzyme fold. <em>Journal of Molecular Biology</em> <em>236</em>, 1277–1282.
</div>
<div id="ref-murzin_structural_1992" class="csl-entry">
(364) Murzin, A. G. (1992) Structural principles for the propeller assembly of beta-sheets: The preference for seven-fold symmetry. <em>Proteins</em> <em>14</em>, 191–201.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="8-chapter08.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="B-classification-of-t-pantotropha.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
