# Preliminary Content {-}

A thesis submitted to the board of the Faculty of Biological Sciences at the
University of Oxford in partial fulfillment of the requirements for the degree of
Doctor of Philosophy.

## Abstract {-}

The _nirS_ gene, which encodes the enzyme cytochrome _cd_<sub>1</sub> nitrite reductase in the denitrifying
bacterium _Thiosphaera pantotropha_, was cloned and sequenced. The sequence of this
cytochrome _cd_<sub>1</sub> was found to be very similar to those of the proteins not only from the
closely-related organism _P. denitrificans_ but also several other species of bacteria. However,
the N-terminal region of these proteins shows less similarity than the overall protein
sequence. Furthermore, sequence analysis in combination with spectroscopic and structural
data indicates that the ligands to the _c_- and _d_<sub>1</sub>-haem groups are not identical in cytochromes
_cd_<sub>1</sub> from different sources. Implications for the mechanism of the enzyme are discussed.

Three different approaches were taken to the problem of expressing recombinant
cytochrome _cd_<sub>1</sub> from the cloned _nirS_ gene. First, the gene was expressed in _Escherichia coli_,
yielding a semi-apo enzyme that contained only the _c_-type haem. The protein could be
reconstituted to some degree using purified haem _d_<sub>1</sub>, producing an enzyme with some of the
spectroscopic features of the native holoenzyme and around 50% of the native nitrite
reductase activity. This approach should be of use in the future expression of site-directed
mutants of cytochrome _cd_<sub>1</sub>, which would be structurally unstable or enzymically inactive if
expressed as the holoenzyme in a heterologous denitrifying host.

Second, the _T. pantotropha_ _nirS_ gene was expressed in the denitrifying bacterium _Ps. aeruginosa_ which synthesises an endogenous cytochrome _cd_<sub>1</sub>. Under the growth conditions employed, a mixture of holo- and semi-apo recombinant cytochrome _cd_<sub>1</sub> was produced. Differences in the visible spectrum of the holoenzyme and attenuated nitrite reductase and cytochrome _c_ oxidase activities suggested that it was not fully assembled in this system. However, the system may be of use in distinguishing site-directed mutants of the enzyme that are enzymically inactive from those that retain some activity.

Third, an unmarked deletion was constructed in the genomic copy of the _T. pantotropha_ _nirS_ gene which abolished the synthesis of cytochrome _cd_<sub>1</sub>. It then proved possible to restore denitrifying growth to the mutant by complementation with the cloned wild-type _nirS_ gene. However, complementation using a site-directed mutant of cytochrome _cd_<sub>1</sub>, in which the _d_<sub>1</sub> haem ligand Tyr-25 was replaced with phenylalanine failed to restore growth and the cells contained greatly attenuated levels of the mutant cytochrome _cd_<sub>1</sub>. Possible reasons for this result are discussed. These data illustrate the problem of finding an expression system that can synthesise the holoenzyme but does not rely upon its activity to maintain the growth of the host organism.

The transcription start site of the _nirS_ gene was determined using 5'-RACE and primer
extension. The _nirS_ gene was found to be transcribed as a monocistronic mRNA.
Comparison of the upstream region of the _nirS_ gene with other denitrification genes indicates
a common mechanism of transcriptional regulation and suggests that such genes may be
transcribed using a specific sigma factor.

## Acknowledgements {-}

My thanks first to Stuart, for taking me on in the first place, organising the funding and for his advice and encouragement during the project, particularly in the writing of this thesis. Thanks are also due to the Wellcome Trust, for their absurdly generous studentship, to Graham Pettigrew, who got me into bacterial _c_-type cytochromes in the first place, and to Thon de Boer, for his cytochrome _cd_<sub>1</sub> antibody and for the gift of pTNIR3, the plasmid that saved the project! Thanks too, to Val for a huge number of perfect primers and to Colin for excellent photography.

Lab 5-9 has been graced by a number of wonderful postdocs; my thanks especially to Simon, for his support throughout the project and for not laughing (too much) during my transformation (ho ho) to a molecular biologist. And a big thank-you to Dudley, for advice on almost everything and conversation during the late nights, long weekends and even the more normal times of day. Variety has certainly been the spice of life in 5-9 thanks to Dave, Ben, Yoshi, Fiona, Alrik, Neeti, Isobel, our latest arrivals and all our visiting students. Outside of 5-9, thanks especially to Pam, Vilmos and Janos, Andy, the 6th floor crowd and of course Gerry and Rachel for much-needed caffeine and sugar, with a smile.

On the brief occasions that I escaped to the world outside, the Oxford “1st wave”, John, Hannah, Jacqui, Caroline and Sara had a lot to answer for. Thanks too to the Oxford “2nd wave”, especially Mat and Lucy, Jem and Lisa, Ralph, Charlie, Craig, Mark and Amanda and of course Marian, for rescuing me and making me realise it wasn't all bad! Most of all, thanks to the folks in Edinburgh, for great holidays, time spent amongst mountains, accommodation, beers, entertainment and empathy: Mark, Leo, Jo, Anita and Lorna, and to my fellow "refugees in England” or further abroad; Mary Jane, James and Nim, Marc and Karen, and Colin and Michelle. I'd never have made it without all of you. I must also thank Mark E. Smith and The Fall, for twenty years of brilliant records.

Last, but definitely not least, a big thank-you to my parents, for all their support over the last eight years. I guess this is for you.

## Publications {-}

Some of the work in this thesis has been published.

1. Baker, S.C., Saunders, N.F.W., Willis, A.C., Ferguson, S.J., Hajdu, J. and Fülöp, V. (1997). Cytochrome _cd_<sub>1</sub> structure: unusual haem environments in a nitrite reductase and analysis of factors contributing to β-propeller folds. J. Mol. Biol. 269: 440-455.

2. Williams, P.A., Fülöp, V., Garman, E.F., Saunders, N.F.W., Ferguson, S.J. and Hajdu, J. (1997). Haem-ligand switching during catalysis in crystals of a nitrogen cycle enzyme. Nature 389: 406-412.

**Conference abstract**

3. Baker, S.C., Saunders, N., Fülöp, V., Hajdu, J. and Ferguson, S.J. (1995). The structure of _cd_<sub>1</sub> nitrite reductase and analysis of the nirS gene of the denitrifying bacterium _Thiosphaera pantotropha_. Poster presented at the Beijerinck Centennial Conference on Microbial Physiology and Gene Regulation: Emerging Principles and Applications. 10-14 December 1995, The Hague, The Netherlands.

**Publications not directly related to the present work**

4. Goodhew, C.F., Pettigrew, G.W., Devreese, B., Beeumen, J.V., van Spanning, R.J.M., Baker, S.C., Saunders, N.F.W., Ferguson, S.J. and Thompson, I.P. (1996). The cytochromes _c_-550 of _Paracoccus denitrificans_ and _Thiosphaera pantotropha_: a need for re-evaluation of the history of _Paracoccus cultures_. FEMS Microbiol. Letts. 137: 95-101.

5. Page, M.D., Saunders, N.F.W. and Ferguson, S.J. (1997). Disruption of the _Pseudomonas aeruginosa dipZ_ gene, encoding a putative protein-disulfide reductase, leads to partial pleiotropic deficiency in _c_-type cytochrome biogenesis. Microbiol. 143: 3111-3122.

6. Hu, W., Van Driessche, G., Devreese, B., Goodhew, C.F., McGinnity, D.F., Saunders, N., Fülöp, V., Pettigrew, G.W. and Van Beeumen, J.J. (1997). Structural characterization of _Paracoccus denitrificans_ cytochrome _c_ peroxidase and assignment of the low and high potential heme sites. Biochemistry 36: 7958-7966.

## List of Figures {-}

|Chapter|Figure|Title|
|:-|:-|-:|
|Chapter 1|\@ref(fig:fig1-1)|The Nitrogen Cycle|
||\@ref(fig:fig1-2)|<nobr>The denitrifying electron transport chain of _Paracoccus denitrificans_</nobr>|
||\@ref(fig:fig1-3)|Structure of (a) _d_$_1$ haem and (b) of protoporphyrin IX|
||\@ref(fig:fig1-4)|Crystal structure of cytochrome _cd_$_1$ in the oxidised form from _T. pantotropha_|
||\@ref(fig:fig1-5)|Proposed mechanism for nitrite reduction by cytochrome _cd_$_1$ from _Thiosphaera pantotropha_|
||\@ref(fig:fig1-6)|Denitrification gene clusters in _Ps. stutzeri_ Zobell, _Ps. aeruginosa_ and _P. denitrificans_ PD1222|
||\@ref(fig:fig1-7)|Unrooted phylogenetic tree of the FNR family of transcriptional activator proteins|
|Chapter 3|\@ref(fig:fig3-1)|PCR products used to sequence the _nirS_ gene of _T. pantotropha_|
||\@ref(fig:fig3-2)|Sequence of the _nirS_ gene and its product, cytochrome _cd_$_1$, from _T. pantotropha_|
||\@ref(fig:fig3-3)|Predicted cleavage site of the periplasmic targeting sequence in cytochrome _cd_$_1$ from _T. pantotropha_|
||\@ref(fig:fig3-4)|Multiple alignment of cytochrome _cd_$_1$ sequences|
||\@ref(fig:fig3-5)|Phylogenetic tree of cytochrome _cd_$_1$ sequences compared with a phylogenetic tree of 16S rRNA sequences from the same organisms|
||\@ref(fig:fig3-6)|Putative NNR binding sites in the upstream regions of the _nirS_ gene aligned with the FNR consensus binding site|
||\@ref(fig:fig3-7)|Early region of the _nirS_ gene from _Ps. stutzeri_ ATCC 14405, showing primers used for re-sequencing and the sequence obtained|
||\@ref(fig:fig3-8)|Visible spectra of cytochrome _cd_$_1$ (a) in the oxidised and (b) in the reduced forms from _Ps. stutzeri_ Zobell|
||\@ref(fig:fig3-9)|Visible spectra of (a) holocytochrome _cd_$_1$ and (b) semi-apo cytochrome _cd_$_1$ in the reduced forms from _T. pantotropha_|
||\@ref(fig:fig3-10)|Alternative reaction schemes for the reduction of nitrite by cytochrome _cd_$_1$ starting from the fully oxidised enzyme|
|Chapter 4|\@ref(fig:fig4-1)|Physical map of the plasmid pTNIR3 produced using six restriction enzymes|
||\@ref(fig:fig4-2)|Location of denitrification genes of _T. pantotropha_ on the plasmid pTNIR3 as determined by physical mapping and limited sequencing.|
||\@ref(fig:fig4-3)|Diagram of the broad host range expression vector pMMB67EH, showing relevant features|
||\@ref(fig:fig4-4)|Restriction digests illustrating the problem of vector carryover during cloning of the _T. pantotropha_ _nirS_ gene into pMMB67EH|
||\@ref(fig:fig4-5)|Restriction digest analysis of the plasmids pMMBSE and pMMBSEK|
||\@ref(fig:fig4-6)|Comparison of the aerobic growth of _E. coli_, _E. coli_ [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]|
||\@ref(fig:fig4-7)|Comparison of the anaerobic growth of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH]|
||\@ref(fig:fig4-8)|Comparison of the protein content of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]. |
||\@ref(fig:fig4-9)|Western blot of protein samples from _E. coli_ JM83, _E. coli_ [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE], probed using antibody to cytochrome _cd_$_1$|
||\@ref(fig:fig4-10)|Comparison of the _c_-type cytochrome content of _E. coli_ [pMMBSE] grown under aerobic or anaerobic conditions|
||\@ref(fig:fig4-11)|Visible spectrum of soluble extract from _E. coli_ [pMMBSE] grown anaerobically and induced with 1 mM IPTG|
||\@ref(fig:fig4-12)|Elution profile during ion exchange chromatography of soluble extract from anaerobically-grown _E. coli_ [pMMBSE] induced using IPTG|
||\@ref(fig:fig4-13)|UV-visible spectrum of recombinant semi-apo cytochrome _cd_$_1$ following ion-exchange chromatography|
||\@ref(fig:fig4-14)|Visible spectra of recombinant cytochrome _cd_$_1$ following reconstitution with purified haem _d_$_1$ (a) as prepared, (b) following reduction using sodium dithionite, compared with (c) native cytochrome _cd_$_1$ from _T. pantotropha_|
||\@ref(fig:fig4-15)|Assays for nitrite reductase activity using reduced methyl viologen as an electron donor: (a) recombinant semi-apo cytochrome _cd_$_1$, (b) recombinant cytochrome _cd_$_1$ following reconstitution with _d_$_1$ haem, (c) native cytochrome _cd_$_1$ purified from _T. pantotropha_|
|Chapter 5|\@ref(fig:fig5-1)|Re-isolation of intact plasmid pMMBSE from _Ps. aeruginosa_ PAO1|
||\@ref(fig:fig5-2)|Comparison of the growth curves of _Ps. aeruginosa_ and _Ps. aeruginosa_ [pMMBSE] under different growth conditions|
||\@ref(fig:fig5-3)|Comparison of (a) the growth curve and (b) nitrite accumulation of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE]|
||\@ref(fig:fig5-4)|Comparison of the protein content of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE] under different growth conditions|
||\@ref(fig:fig5-5)|Western blot of protein samples from _Ps. aeruginosa_ or _Ps. aeruginosa_ [pMMBSE] grown under different growth conditions, probed using antibody to cytochrome _cd_$_1$|
||\@ref(fig:fig5-6)|Comparison of the _c_-type cytochrome content of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE] under different growth conditions|
||\@ref(fig:fig5-7)|Elution profile during ion-exchange chromatography of soluble extract from _Ps. aeruginosa_ [pMMBSE] grown semi-aerobically with nitrate and induced using IPTG|
||\@ref(fig:fig5-8)|Analysis of the _c_-type cytochrome content in the elution profile of soluble extract from _Ps. aeruginosa_ [pMMBSE] following ion exchange chromatography|
||\@ref(fig:fig5-9)|UV-visible spectrum of concentrated fractions eluting under peak 2 following DEAE-Sepharose chromatography|
||\@ref(fig:fig5-10)|a) UV-visible spectrum of concentrated fractions eluting under peak 3 after DEAE-Sepharose chromatography, as isolated and (b) after the addition of excess sodium dithionite|
||\@ref(fig:fig5-11)|Difference spectra of (a) dithionite-reduced minus as-isolated spectra from peak 3 and (b) dithionite-reduced peak 3 minus peak 2|
||\@ref(fig:fig5-12)|Cytochrome _c_ oxidase and (b) nitrite reductase activities of the concentrated fractions from peak 3 following ion exchange chromatography|
|Chapter 6|\@ref(fig:fig6-1)|Construction of a kanamycin insertional mutant in the _nir_ region of the _T. pantotropha_ genome|
||\@ref(fig:fig6-2)|Construction of a _nirS_ deletion mutant strain in _T. pantotropha_|
||\@ref(fig:fig6-3)|Construction of the suicide plasmid pGRPN3K|
||\@ref(fig:fig6-4)|Restriction digest analysis of plasmid pGRPN3K|
||\@ref(fig:fig6-5)|Southern blot of genomic DNA from _T. pantotropha_ K7 using the kanamycin resistance gene as a probe|
||\@ref(fig:fig6-6)|Construction of the suicide plasmid pRVS$\Delta$nir|
||\@ref(fig:fig6-7)|Restriction digest analysis of plasmids pLN3N and pLN3N$\Delta$nir|
||\@ref(fig:fig6-8)|Restriction digest analysis of plasmids pRVS1 and pRVS$\Delta$nir|
||\@ref(fig:fig6-9)|PCR analysis of plasmid pRVS$\Delta$nir showing a 382 bp deletion in the cloned _nirS_ gene|
||\@ref(fig:fig6-10)|Southern blot of genomic DNA from wild-type _T. pantotropha_, _T. pantotropha_ K7 and the _nirS_ deletion mutants $\Delta5$, $\Delta7$ and $\Delta8$ using a _nirS_-derived probe|
||\@ref(fig:fig6-11)|Comparison of the anaerobic growth of _T. pantotropha_ with that of _T. pantotropha_ K7 and _T. pantotropha_ $\Delta7$|
||\@ref(fig:fig6-12)|Comparison of (a) the anaerobic growth and (b) nitrite accumulation of _T. pantotropha_ with that of _T. pantotropha_ K7 and _T. pantotropha_ $\Delta7$|
||\@ref(fig:fig6-13)|Western blot of total soluble extracts from wild-type _T. pantotropha_, _T. pantotropha_ K7, _T. pantotropha_ $\Delta7$, _T. pantotropha_ $\Delta7$ [pMMBSEK] and _T. pantotropha_ $\Delta7$ [pMMBY25F] using antibody raised against _P. denitrificans_ cytochrome _cd_$_1$|
||\@ref(fig:fig6-14)|Complementation of _T. pantotropha_ $\Delta7$ using plasmid pMMBSEK|
||\@ref(fig:fig6-15)|(a) Anaerobic growth of _T. pantotropha_ $\Delta7$ [pMMBSEK] and (b) nitrite concentration in the medium during growth|
||\@ref(fig:fig6-16)|Construction of the plasmid pVA191-Y25F containing the Y25F mutation in the _nirS_ gene of _T. pantotropha_|
||\@ref(fig:fig6-17)|Construction of the expression plasmid pMMBY25F containing the Y25F mutation in the _nirS_ gene of _T. pantotropha_|
||\@ref(fig:fig6-18)|Restriction digest analysis of plasmid pBY25FK|
||\@ref(fig:fig6-19)|Restriction digest analysis of plasmid pMMBY25F using _MluI_|
||\@ref(fig:fig6-20)|Comparison of (a) anaerobic growth and (b) nitrite accumulation in _T. pantotropha_ $\Delta7$ [pMMBY25F] compared with that of _T. pantotropha_, _T. pantotropha_ $\Delta7$ and _T. pantotropha_ $\Delta7$ [pMMBSEK]|
||\@ref(fig:fig6-21)|Elution profile during ion exchange chromatography of soluble extract from anaerobically grown _T. pantotropha_ $\Delta7$ [pMMBY25F]|
||\@ref(fig:fig6-22)|SDS-PAGE gel of fractions eluting from the DEAE-Sepharose column during chromatography of soluble extract from _T. pantotropha_ $\Delta7$ [pMMBY25F], stained for _c_-type cytochromes|
||\@ref(fig:fig6-23)|Visible spectra of an unidentified _c_-type cytochrome from _T. pantotropha_ [pMMBY25F]|
||\@ref(fig:fig6-24)|Visible spectra of (a) cytochrome _c_$_{550}$ and (b) cytochrome _c_' from _T. pantotropha_ [pMMBY25F]|
||\@ref(fig:fig6-25)|Visible spectra of cytochrome _c_ peroxidase from _T. pantotropha_ [pMMBY25F]|
||\@ref(fig:fig6-26)|Assay of cytochrome _c_ peroxidase activity in peak 4 following ion exchange chromatography|
||\@ref(fig:fig6-27)|Visible spectrum of pseudoazurin from _T. pantotropha_ [pMMBY25F]|
|Chapter 7|\@ref(fig:fig7-1)|Analysis of total RNA samples from human cells, _E. coli_ and _T. pantotropha_ using denaturing agarose gel electrophoresis|
||\@ref(fig:fig7-2)|Northern blot of total RNA from anaerobically grown _T. pantotropha_ using a _nirS_-derived probe|
||\@ref(fig:fig7-3)|Northern blot of total RNA from anaerobically grown _T. pantotropha_, using a _nirS_-derived probe together with chemiluminescent detection|
||\@ref(fig:fig7-4)|Location of primers used in primer extension and 5'-RACE experiments to determine the transcription start site of the _T. pantotropha nirS_ gene|
||\@ref(fig:fig7-5)|Primer extension product synthesised from _nirS_ mRNA, compared with a sequencing ladder generated from the _nirS_ gene using the same primer|
||\@ref(fig:fig7-6)|The upstream region of the _T. pantotropha nirS_ gene, showing the transcription start site as determined using primer extension and 5'-RACE|
||\@ref(fig:fig7-7)|Schematic diagram illustrating the principle of the 5'-RACE method|
||\@ref(fig:fig7-8)|Analysis using agarose gel electrophoresis of the PCR product generated from _nirS_ mRNA by 5'-RACE|
||\@ref(fig:fig7-9)|Restriction digest analysis of the cloned PCR products generated from _nirS_ mRNA by 5'-RACE|
||\@ref(fig:fig7-10)|Sequence of the cloned PCR product synthesised from _nirS_ mRNA by 5'-RACE|
||\@ref(fig:fig7-11)|Model for transcriptional termination of the _T. pantotropha nirS_ gene|
||\@ref(fig:fig7-12)|Alignments of the upstream regions of NNR-regulated genes from _P. denitrificans_ and _T. pantotropha_ showing regions of conserved sequence|
||\@ref(fig:fig7-13)|Putative factors influencing the transcription of nitrite and nitric oxide reductases in _T. pantotropha_|
|Appendix A|\@ref(fig:figA-1)|Schematic diagram of the $\beta$-propeller _d_$_1$-haem binding domain of _T. pantotropha_ cytochrome _cd_$_1$|


## List of Tables {-}

|Chapter|Table|Title|
|:-|:-|-:|
|Chapter 1|\@ref(tab:table11)|Reduction potentials and thermodynamics of the reactions of denitrification|
||\@ref(tab:table12)|Standard reduction potentials reported by other workers of the haem groups in cytochrome _cd_$_1$|
|Chapter 3|\@ref(tab:table31)|Primers used to sequence the _nirS_ gene from _T. pantotropha_|
||\@ref(tab:table32)|Comparison of inverted repeats in the DNA sequence of the _nir_ region from _T. pantotropha_ with those in the same region of _P. denitrificans_ PD1222|
||\@ref(tab:table33)|Matrix of similarities and identities between the cytochromes _cd_$_1$|
||\@ref(tab:table34)|Comparison of the physicochemical properties of cytochrome _cd_$_1$ from different sources|
|Chapter 4|\@ref(tab:table41)|Plasmids referred to in this chapter|
||\@ref(tab:table42)|Different growth conditions used to compare the growth of _E. coli_ JM83 with that of _E. coli_ containing either pMMB67EH, pMMBSEK or pMMBSE|
||\@ref(tab:table43)|Comparison of the ratios of the absorbance maxima in (left) the reduced and (right) the oxidised states of native holo- and chemically prepared semi-apo cytochrome _cd_$_1$ from _T. pantotropha_ with those of the recombinant protein from _E. coli_|
|Chapter 5|\@ref(tab:table51)|Different growth conditions used to compare the growth of _Ps. aeruginosa_ PAO1 with that of _Ps. aeruginosa_ [pMMBSE]|
||\@ref(tab:table52)|Comparison of the ratios of the absorbance maxima in the reduced state of native holo- and chemically-prepared semi-apo cytochrome _cd_$_1$ from _T. pantotropha_ with those of the partially purified recombinant protein from _Ps. aeruginosa_ or _E. coli_|
|Chapter 6|\@ref(tab:table61)|Bacterial strains and plasmids referred to in this chapter|
|Chapter 7|\@ref(tab:table71)|Proteins involved in regulating the expression of denitrification enzymes in _P. denitrificans_|


## List of Abbreviations {-}

|Abbreviation|Meaning|
|:-|-:|
|AMV|Avian myeloblastosis virus|
|ATP|Adenosine triphosphate|
|A$_{xxx}$|Absorbance at wavelength XXX nm|
|bp|Base pairs|
|BSA|Bovine serum albumin|
|DEAE|Diethylaminoethyl|
|$\Delta$_G_°'|Standard Gibbs free energy change at pH 7.0|
|DIG|Digoxigenin|
|DIG-11-dUTP|Digoxigenin-11-deoxyuridine triphosphate|
|DMSO|Dimethylsulphoxide|
|DNase|Deoxyribonuclease|
|dNTP|Deoxynucleotide triphosphate|
|DTT|Dithiothreitol|
|e$^-$|Electron|
|_E_°'|Standard redox potential at pH 7.0|
|EPR|Electron paramagnetic resonance|
|IPTG|Isopropylthio-$\beta$-D-galactoside|
|LB|Luria Bertani|
|MCD|Magnetic circular dichroism|
|MGD|Molybdopterin guanine dinucleotide|
|MMLV|Moloney murine leukaemia virus|
|Na$_2$EDTA/EDTA|Disodium ethylenediamine tetraacetic acid|
|NAD|Nicotinamide adenine dinucleotide|
|NMR|Nuclear magnetic resonance|
|O.D. (xxx)|Optical density at XXX nm|
|PBS|Phosphate buffered saline|
|PCR|Polymerase chain reaction|
|PEG|Polyethylene glycol|
|RNase|Ribonuclease|
|SDS-PAGE|Sodium dodecyl sulphate polyacrylamide gel electrophoresis|
|SSC|0.15 M NaCl, 0.15 mM sodium citrate|
|TAE|40 mM Tris-acetate pH 8.0, 1 mM EDTA|
|TBE|45 mM Tris-borate pH 8.3, 1 mM EDTA|
|TE|10 mM Tris-HCl, 1 mM EDTA |
|TEMED|N, N, N’, N’-tetramethylethylenediamine|
|UQ/UQH$_2$|Ubiquinone/ubiquinol |
|X-Gal|5-bromo-4-chloro-3-indolyl-$\beta$-D-galactoside|

## Quote {-}

>If we do not succeed, then we run the risk of failure”

_Former U.S. Vice-President Dan Quayle, 1990 (attrib.)_
