<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.23 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="2-chapter02.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t.-pantotropha.html"><a href="B-classification-of-t.-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter01" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<div id="denitrification-and-the-nitrogen-cycle" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Denitrification and the nitrogen cycle</h2>
<p>Many chemical elements of biological importance are cycled through the biosphere by a combination of physical and biological processes, in what are termed biogeochemical cycles.
One of these cycles is the nitrogen cycle shown schematically in Figure 1.1.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-1"></span>
<img src="figure/fig1-1.png" alt="The nitrogen cycle. The oxidation state of nitrogen in each intermediate is given in brackets. Denitrification is shown by black arrows, nitrogen fixation by a dark grey arrow and all other reactions by light grey arrows. Note that nitrification involves two reactions, oxidation of ammonia to nitrite and of nitrite to nitrate, but these have never been found together in the same organism. ANAMOX (anaerobic ammonia oxidation) is a poorly-characterised process in which nitrite and ammonia combine to give water and dinitrogen [@fry_nitrogen_1992]." width="50%" />
<p class="caption">
Figure 1.1: The nitrogen cycle. The oxidation state of nitrogen in each intermediate is given in brackets. Denitrification is shown by black arrows, nitrogen fixation by a dark grey arrow and all other reactions by light grey arrows. Note that nitrification involves two reactions, oxidation of ammonia to nitrite and of nitrite to nitrate, but these have never been found together in the same organism. ANAMOX (anaerobic ammonia oxidation) is a poorly-characterised process in which nitrite and ammonia combine to give water and dinitrogen <span class="citation">[<a href="#ref-fry_nitrogen_1992" role="doc-biblioref">1</a>]</span>.
</p>
</div>
<p>The cycle involves the interconversion between various oxidation states of nitrogen and is predominantly carried out by micro-organisms. A simplified view of the nitrogen cycle is one of a balance between two opposing processes. In nitrogen fixation atmospheric dinitrogen gas is reduced to ammonia, allowing the assimilation of nitrogen into biological molecules. Nitrogen is released back into the environment as ammonium during decay, oxidised to nitrate and converted to dinitrogen gas by the process of denitrification, the left hand branch of Figure 1.1. This involves the sequential reduction of nitrate to nitrite, nitric oxide, nitrous oxide and finally nitrogen, in a series of enzyme-catalysed reactions. The individual reactions of denitrification can be referred to as respiratory N-oxide reduction, because the process is used to generate ATP via a proton electrochemical gradient across the cytoplasmic membrane. The term dissimilatory N-oxide reduction is also used to indicate that the products are lost to the environment; this distinguishes the reactions from the assimilatory reductions used by plants and bacteria to incorporate nitrogen into biomass.</p>
<div id="the-importance-of-denitrification" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> The importance of denitrification</h3>
<p>Denitrification was first reported last century <span class="citation">[<a href="#ref-gayon_recherches_1886" role="doc-biblioref">2</a>]</span> and it has since been the focus of much research, spanning a variety of disciplines. Factors that affect the levels of N-oxides in the environment are of great ecological interest. For example, nitrous oxide, the penultimate compound in the pathway, has been implicated as a contributor to global warming and as an agent capable of destroying stratospheric ozone <span class="citation">[<a href="#ref-rasmussen_atmospheric_1986" role="doc-biblioref">3</a>]</span>. Denitrification is a route by which soil nitrogen may be lost, with implications for agricultural productivity <span class="citation">[<a href="#ref-hauck_nitrogen_1981" role="doc-biblioref">4</a>]</span>. The removal of nitrogenous compounds in waste water treatment by bioremediation is an important field, particularly in small countries such as the Netherlands <span class="citation">[<a href="#ref-hutchins_biodegradation_1991" role="doc-biblioref">5</a>]</span>, and the contribution of polluting nitrates in groundwater to eutrophication is another concern <span class="citation">[<a href="#ref-olson_fertilizers_1977" role="doc-biblioref">6</a>]</span>. Nitrate and nitrite levels in the environment may also have health consequences, through conditions such as methaemoglobinaemia <span class="citation">[<a href="#ref-levallois_contamination_1994" role="doc-biblioref">7</a>]</span> and certain stomach cancers (caused by conversion of nitrites to N-nitroso compounds in the stomach) <span class="citation">[<a href="#ref-calmels_characterization_1996" role="doc-biblioref">8</a>]</span>. Information about microbial N-oxide metabolism may therefore be relevant to many researchers.</p>
<p>In recent years denitrifying bacteria have proven to be extremely interesting systems for the study of fundamental prokaryotic biochemistry, physiology and molecular genetics, and this aspect of denitrification is the main concern of this thesis. As a bioenergetic process, the enzymes involved may be used to investigate electron transfer and proton translocation, both of which are poorly understood at present. Biogenesis of the enzymes, in terms of targeting and compartmentalisation, folding and assembly of their various metal centres is another fundamental process. The catalytic mechanisms of each enzyme also attract research, particularly as several of the proteins involved are soluble, easy to purify in large amounts and, in some cases, have had their crystal structures determined. Finally there is always the possibility that study of prokaryotic systems may provide structural models for analogous eukaryotic processes. As an example, the bacterial cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase (Section 1.5) generates bound nitric oxide which must be released from a haem group. In eukaryotes, nitric oxide participates in signal transduction by binding to a haem group on the enzyme guanyl cyclase, from which it must also be released <span class="citation">[<a href="#ref-bredt_nitric_1994" role="doc-biblioref">9</a>]</span>.</p>
</div>
</div>
<div id="the-denitrifying-micro-organisms" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> The denitrifying micro-organisms</h2>
<p>Denitrification research has traditionally focused on the Gram-negative bacteria (those containing a compartment between the cytoplasmic and outer membranes termed the periplasm) and until quite recently, respiratory N-oxide reduction was considered to be an exclusively prokaryotic attribute. This situation has altered somewhat with the discovery that at least two species of fungus, <em>Fusarium oxysporum</em> and <em>Cylindrocarpon tonkinense</em> can reduce nitrite to nitrous oxide <span class="citation">[<a href="#ref-shoun_denitrification_1991" role="doc-biblioref">10</a>,<a href="#ref-usuda_denitrification_1995" role="doc-biblioref">11</a>]</span>. In <em>F. oxysporum</em>, N-oxide reduction is localised to the mitochondrion <span class="citation">[<a href="#ref-kobayashi_denitrification_1996" role="doc-biblioref">12</a>]</span> and is comprised of distinct enzymes for the dissimilatory reduction of nitrate <span class="citation">[<a href="#ref-usuda_denitrification_1995" role="doc-biblioref">11</a>]</span>, nitrite <span class="citation">[<a href="#ref-kobayashi_copper-containing_1995" role="doc-biblioref">13</a>]</span> and nitric oxide <span class="citation">[<a href="#ref-nakahara_cytochrome_1993" role="doc-biblioref">14</a>]</span>. The system is significantly different to that found in bacteria, particularly the nitric oxide reductase which is a unique cytochrome P450-type enzyme.</p>
<p>The bacteria studied to date tend to fall into two groups; those in which at least one enzyme of the denitrification pathway has attracted interest, although the complete pathway has not been investigated (or may not be present), and those in which the complete denitrification pathway from nitrate through to dinitrogen has been characterised. <em>Escherichia coli</em> falls into the former category; although not a denitrifier, it possesses a membrane-bound nitrate reductase <span class="citation">[<a href="#ref-enoch_role_1974" role="doc-biblioref">15</a>]</span> and a putative periplasmic nitrate reductase <span class="citation">[<a href="#ref-berks_napedabc_1995" role="doc-biblioref">16</a>]</span>, encoded by the <em>nar</em> and <em>nap</em> gene clusters, respectively, which are essentially identical to those found in the denitrifying bacteria. The genetic amenability of this organism and its wider importance in biochemical and genetic research has encouraged investigation of these enzymes; for example extensive sequencing of the <em>E. coli</em> genome allowed the identification of the <em>E. coli</em> nap gene cluster during a DNA database search using the <em>Thiosphaera pantotropha nap</em> genes <span class="citation">[<a href="#ref-richterich_u00008_1993" role="doc-biblioref">17</a>]</span>.</p>
<p>Also in the first category are denitrifiers where one enzyme in the pathway has been singled out as being of specific interest. Examples include <em>Achromobacter cycloclastes</em> and <em>Alcaligenes faecalis</em> S-6; these organisms contain a copper nitrite reductase (Section 1.4.3), the X-ray crystal structures of which have been determined to high resolution <span class="citation">[<a href="#ref-godden_23_1991" role="doc-biblioref">18</a>,<a href="#ref-kukimoto_x-ray_1994" role="doc-biblioref">19</a>]</span>. This work has provided a great deal of information on copper centres and possible mechanisms of nitrite reduction.</p>
<p>However, the core of denitrification research concerns bacteria from the latter category; those in which the four enzymes required for complete denitrification have been identified and at least partially characterised. Many such species have been described <span class="citation">[<a href="#ref-zumft_denitrifying_1992" role="doc-biblioref">20</a>]</span> but the best studied are <em>Pseudomonas aeruginosa</em>, <em>Pseudomonas stutzeri</em> and <em>Paracoccus denitrificans</em> (and its close relatives, particularly <em>T. pantotropha</em>). <em>P. denitrificans</em> has long been of interest in the field of bioenergetics, due to the close similarity between its aerobic respiratory chain and that of the eukaryotic mitochondrion <span class="citation">[<a href="#ref-john_paracoccus_1975" role="doc-biblioref">21</a>]</span>. <em>P. denitrificans</em> has also been the subject of a great deal of metabolic research, due to its nutritional versatility. As well as growing anaerobically with nitrate as terminal electron acceptor, <em>P. denitrificans</em> can grow chemoautotrophically on inorganic substrates such as hydrogen or thiosulphate, chemoheterotrophically on organic acids such as acetate or succinate and methylotrophically, on methanol or methylamine. As with other bacteria denitrifying growth in <em>P. denitrificans</em> is an anaerobic process used as an alternative to aerobic respiration when oxygen, the preferred terminal electron acceptor, is in short supply.</p>
</div>
<div id="organisation-of-the-denitrifying-electron-transport-chain" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Organisation of the denitrifying electron transport chain</h2>
<p>Although denitrification is relatively widespread amongst the prokaryotes, detailed biochemical information is available for only a small number of organisms. However, the current model of the organisation of the denitrifying respiratory chain is shown in Figure 1.2.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-2"></span>
<img src="figure/fig1-2.png" alt="The denitrifying electron transport chain of _Paracoccus denitrificans_. The enzymes shown are: NarGHI, nitrate reductase, FbcFBC, cytochrome _bc_$_1$ complex, NorBC, nitric oxide reductase, NirS, cyochrome _cd_$_1$ nitrite reductase, NosZ, nitrous oxide reductase. 'Q' represents the ubiquinol/ubiquinone pool. Chemical reactions are indicated by solid arrows, proton translocation by dashed arrows and electron transfer by dotted arrows, with the stoichiometry shown in each case. Proton translocation at the cytochrome _bc_$_1$ complex is described by the proton-motive Q-cycle (see text for details). Not shown is the periplasmic nitrate reductase, which is not present during anaerobic denitrifying growth conditions." width="50%" />
<p class="caption">
Figure 1.2: The denitrifying electron transport chain of <em>Paracoccus denitrificans</em>. The enzymes shown are: NarGHI, nitrate reductase, FbcFBC, cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex, NorBC, nitric oxide reductase, NirS, cyochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, NosZ, nitrous oxide reductase. ‘Q’ represents the ubiquinol/ubiquinone pool. Chemical reactions are indicated by solid arrows, proton translocation by dashed arrows and electron transfer by dotted arrows, with the stoichiometry shown in each case. Proton translocation at the cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex is described by the proton-motive Q-cycle (see text for details). Not shown is the periplasmic nitrate reductase, which is not present during anaerobic denitrifying growth conditions.
</p>
</div>
<p>This diagram is based on the organism <em>Paracoccus denitrificans</em>, but is probably similar in other bacteria, particularly <em>Ps. aeruginosa</em> and <em>Ps. stutzeri</em>. During anaerobic denitrifying growth, a standard tricarboxylic acid cycle is employed, generating the reductants NADH and succinate. The cytoplasmic membrane thus contains prokaryotic homologues of the classical respiratory complexes: complex I (NADH-ubiquinone oxidoreductase), complex II (succinate-ubiquinone oxidoreductase) and complex III (ubiquinol-cytochrome <em>c</em> oxidoreductase), in addition to the integral membrane proteins nitrate and nitric oxide reductase. Nitrate reductase is believed to accept electrons from the ubiquinol pool. In contrast, the other three N-oxide reductases are believed to receive electrons from the cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex via small periplasmic electron transport proteins. Net proton translocation is achieved by the operation of the proton motive Q-cycle in the cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex <span class="citation">[<a href="#ref-trumpower_cytochrome_1990" role="doc-biblioref">22</a>]</span> and the activity of Complex I. The reactions of N-oxide reduction are therefore periplasmic, with the exception of nitrate reduction which occurs in the cytoplasm <span class="citation">[<a href="#ref-ballard_respiratory_1988" role="doc-biblioref">23</a>]</span>; this implies a transporter for entry of nitrate and export of nitrite to the periplasm (Section 1.4.1).</p>
<p>Denitrification is an energy conserving process. Consideration of the reduction potentials associated with each step (Table 1.1) suggests that each reaction is relatively exergonic, yielding sufficient free energy to drive the synthesis of ATP.</p>
<table>
<caption>
<span id="tab:table11">Table 1.1: </span>Reduction potentials and thermodynamics of the reactions of denitrification compared with those of the oxygen/water couple, assuming NADH as the reductant.
The four reactions of denitrification are shown for the reduction of two molecules of nitrate; this is to balance the overall equation as two nitric oxide molecules are required to dimerise, forming one molecule of nitrous oxide. The reduction potentials of each couple under biological standard state conditions (1 atm pressure for gases, solutions at 1 M, pH 7.0, at 298 K) are shown. Assuming NADH to be the reductant for each reaction allows the free energy change to be calculated as shown in column four, using the equation <span class="math inline">\(\Delta\)</span><em>G</em><span class="math inline">\(\unicode{x00b0}\)</span>’ = -nF<span class="math inline">\(\Delta\)</span><em>E</em><span class="math inline">\(\unicode{x00b0}\)</span>‘, where <span class="math inline">\(\Delta\)</span><em>G</em><span class="math inline">\(\unicode{x00b0}\)</span>’ = Gibbs free energy change, n = number of electrons transferred, F = 96 500 C mol<span class="math inline">\(^-1\)</span> and <span class="math inline">\(\Delta\)</span><em>E</em><span class="math inline">\(\unicode{x00b0}\)</span>’ = redox span between reductant and oxidant. This shows that the reduction of nitric and nitrous oxides is associated with a greater free energy change per reducing equivalent (two electrons) than that of oxygen to water. The overall denitrification reaction also has a comparable free energy change to that of oxygen reduction. However, as explained in the main text, standard state conditions are unlikely to occur in the cell and, additionally, the topology of the denitrifying electron transport chain results in a lower ATP:2e<span class="math inline">\(^-\)</span> stoichiometry than that of the aerobic respiratory chain.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Reaction
</th>
<th style="text-align:left;">
Couple
</th>
<th style="text-align:left;">
<span class="math inline">\(\Delta\)</span><em>E</em><span class="math inline">\(^{o&#39;}\)</span> (V)
</th>
<th style="text-align:left;">
<span class="math inline">\(\Delta\)</span><em>G</em><span class="math inline">\(^{o&#39;}\)</span> (kJ mol<span class="math inline">\(^{-1}\)</span>) per reaction
</th>
<th style="text-align:left;">
<span class="math inline">\(\Delta\)</span><em>G</em><span class="math inline">\(^o\)</span> (kJ mol<span class="math inline">\(^{-1}\)</span>) per NADH
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{2NO_3^- + 4H^+ + 4e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{2NO_2^- + 2H_2O}\)</span>
</td>
<td style="text-align:left;">
NO<span class="math inline">\(_3^-\)</span>/NO<span class="math inline">\(_2^-\)</span>
</td>
<td style="text-align:left;">
+0.420
</td>
<td style="text-align:left;">
-285.6
</td>
<td style="text-align:left;">
-142.8
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{2NO_2^- + 2H^+ + 2e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{2NO + 2H_2O}\)</span>
</td>
<td style="text-align:left;">
NO<span class="math inline">\(_2^-\)</span>/NO
</td>
<td style="text-align:left;">
+0.375
</td>
<td style="text-align:left;">
-134.1
</td>
<td style="text-align:left;">
-134.1
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{2NO + 2H^+ + 2e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{N_2O + H_2O}\)</span>
</td>
<td style="text-align:left;">
NO/N<span class="math inline">\(_2\)</span>O
</td>
<td style="text-align:left;">
+1.175
</td>
<td style="text-align:left;">
-288.5
</td>
<td style="text-align:left;">
-288.5
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{N_2O + 2H^+ + 2e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{N_2 + H_2O}\)</span>
</td>
<td style="text-align:left;">
N<span class="math inline">\(_2\)</span>O/N<span class="math inline">\(_2\)</span>
</td>
<td style="text-align:left;">
+1.355
</td>
<td style="text-align:left;">
-323.3
</td>
<td style="text-align:left;">
-323.3
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Overall</strong>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{2NO_3^- + 2H^+ + 5H_2}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{N_2 + 6H_2O}\)</span>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
-1031.5
</td>
<td style="text-align:left;">
-206.3
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{O_2 + 4H^+ + 4e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{2H_2O}\)</span>
</td>
<td style="text-align:left;">
O<span class="math inline">\(_2\)</span>/H<span class="math inline">\(_2\)</span>O
</td>
<td style="text-align:left;">
+0.800
</td>
<td style="text-align:left;">
-438.5
</td>
<td style="text-align:left;">
-219.2
</td>
</tr>
<tr>
<td style="text-align:left;">
<span class="math inline">\(\mathrm{NAD^+ + 2H^+ + 2e^-}\)</span> <span class="math inline">\(\rightarrow\)</span> <span class="math inline">\(\mathrm{NADH + H^+}\)</span>
</td>
<td style="text-align:left;">
NAD<span class="math inline">\(^+\)</span>/NADH
</td>
<td style="text-align:left;">
-0.320
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table>
<p>With NADH as electron donor (<i>E</i><span class="math inline">\(^o\)</span>’ for the NADH/NAD<span class="math inline">\(^+\)</span> couple is -320 mV), the redox span to nitric and nitrous oxide is actually greater than that for the oxygen/water couple. However, it must be born in mind that these values refer to standard state conditions and will be different in the cell; for instance the actual concentration of N<span class="math inline">\(_2\)</span>O will be rather lower than one atmosphere. Furthermore, the spatial arrangement of the components involved in the electron transport sequence influences the stoichiometry of net proton translocation per pair of electrons passing through the electron transport chain. Termination of the denitrifying electron transport chain with nitrous oxide reductase results in no net proton translocation across the redox span from cytochrome <em>c</em> to nitrous oxide, whereas oxygen reduction by the cytochrome <em>aa</em><span class="math inline">\(_3\)</span> oxidase results in the equivalent of four protons translocated per pair of electrons. Thus, assuming a constant H<span class="math inline">\(^+\)</span>:ATP ratio for the ATP synthase, nitrous oxide respiration gives a lower ATP:2e<span class="math inline">\(^-\)</span> ratio than does oxygen respiration. Nevertheless, denitrification represents an important alternative mode of respiration for those bacteria which have evolved the necessary enzymatic machinery, allowing them to exploit a wider range of environmental niches than obligate aerobes.</p>
<p>In the following sections, the enzymes of denitrification are discussed in more detail. The order of discussion is the same as that of each reaction in the pathway, with the exception of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase; as this enzyme is the main focus of the project it is treated in greater detail after the others.</p>
</div>
<div id="enzymes-of-the-denitrifying-electron-transport-chain" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Enzymes of the denitrifying electron transport chain</h2>
<div id="the-membrane-bound-nitrate-reductase" class="section level3" number="1.4.1">
<h3><span class="header-section-number">1.4.1</span> The membrane-bound nitrate reductase</h3>
<p>Membrane-bound nitrate reductase (Nar) is widely distributed, being found in many enteric bacteria as well as the denitrifying species. The most extensively characterised examples are from <em>E. coli</em> <span class="citation">[<a href="#ref-clegg_purification_1976" role="doc-biblioref">24</a>]</span> and <em>P. denitrificans</em> <span class="citation">[<a href="#ref-ballard_molecular_1987" role="doc-biblioref">25</a>]</span>. Nitrate reductase from these sources is composed of three subunits with molecular masses of around 140 kDa (<span class="math inline">\(\alpha\)</span>), 60 kDa (<span class="math inline">\(\beta\)</span>) and 25 kDa (<span class="math inline">\(\gamma\)</span>), as judged by SDS-PAGE. The <span class="math inline">\(\gamma\)</span> subunit anchors the other two subunits to the cytoplasmic side of the cytoplasmic membrane and an <span class="math inline">\(\alpha\beta\)</span> complex can be released from inside-out membrane vesicles by washing with EDTA or magnesium-free buffer <span class="citation">[<a href="#ref-ballard_respiratory_1988" role="doc-biblioref">23</a>]</span>. Loss of the <span class="math inline">\(\gamma\)</span> subunit during purification is accompanied by an apparent proteolytic cleavage at the C-terminus of the <span class="math inline">\(\beta\)</span> subunit <span class="citation">[<a href="#ref-demoss_characterization_1981" role="doc-biblioref">26</a>]</span>. The inference from these studies is that the a<span class="math inline">\(\beta\)</span> complex contacts the <span class="math inline">\(\gamma\)</span> subunit through a region near the carboxy end of the <span class="math inline">\(\beta\)</span> subunit polypeptide chain. Stoichiometry of the subunits is uncertain, but is thought most likely to be <span class="math inline">\(\alpha1\beta1\gamma1\)</span>; in <em>E. coli</em> the genes appear to have overlapping start and stop codons, which would result in translational coupling <span class="citation">[<a href="#ref-oppenheim_translational_1980" role="doc-biblioref">27</a>]</span> and a stoichiometric synthesis of the subunits.</p>
<p>Membrane-bound nitrate reductase is a quinol oxidase, receiving from the quinol pool the two electrons required to reduce a molecule of nitrate. The electrons are transported through the complex by a variety of cofactors. The <span class="math inline">\(\gamma\)</span> subunit contains two <em>b</em>-type haems <span class="citation">[<a href="#ref-ballard_respiratory_1988" role="doc-biblioref">23</a>]</span>; modelling studies suggest that these conduct electrons from ubiquinol across the cytoplasmic membrane, leaving two protons in the periplasm in a classical “redox loop” mechanism <span class="citation">[<a href="#ref-berks_sequence_1995" role="doc-biblioref">28</a>]</span>. The <span class="math inline">\(\alpha\beta\)</span> complex contains several iron-sulphur clusters <span class="citation">[<a href="#ref-johnson_iron-sulfur_1985" role="doc-biblioref">29</a>]</span> and the <span class="math inline">\(\alpha\)</span> subunit contains in addition a complex cofactor termed molybdopterin guanine dinucleotide (MGD) <span class="citation">[<a href="#ref-wootton_enzymes_1991" role="doc-biblioref">30</a>]</span>. All bacterial nitrate reductases contain this cofactor, a variant of molybdopterin which in this case is conjugated to guanosine monophosphate. MGD-dependent enzymes form a large family, whose members include soluble and membrane-bound nitrate reductases, formate dehydrogenases and S-oxide reductases. Amino acid similarities between these sequences have been used to establish a number of sub-families and to identify conserved regions involved in the binding of metal cofactors <span class="citation">[<a href="#ref-berks_enzymes_1995" role="doc-biblioref">31</a>]</span>.</p>
<p>A great deal of spectroscopic data has been amassed for the MGD cofactor, which receives electrons from the neighbouring <span class="math inline">\(\beta\)</span> subunit, binds nitrate and catalyses an oxo-transfer reaction between nitrate and water <span class="citation">[<a href="#ref-craig_reduction_1989" role="doc-biblioref">32</a>]</span>. EPR spectroscopy and mechanistic studies indicate that the molybdenum atom cycles between Mo(IV) and Mo(VI) states and is the site of the oxo-transfer reaction <span class="citation">[<a href="#ref-cramer_molybdenum_1983" role="doc-biblioref">33</a>]</span>. The complexity of the cofactor, in terms of the organic dithiolene moiety, is argued to provide a specific binding site for the Mo atom <span class="citation">[<a href="#ref-bellion_biological_1992" role="doc-biblioref">34</a>]</span>. This has been confirmed by two recent crystal structures of the enzymes dimethyl sulphoxide reductase <span class="citation">[<a href="#ref-schindelin_crystal_1996" role="doc-biblioref">35</a>]</span> and formate dehydrogenase H <span class="citation">[<a href="#ref-boyington_crystal_1997" role="doc-biblioref">36</a>]</span>. Both of these structures show that the molybdenum ion is co-ordinated by two MGD molecules through the dithiolene groups.</p>
<p>The structural genes for the three subunits of the complex, <em>narGHI</em>, have been cloned and sequenced from <em>E. coli</em>. A fourth gene, <em>narJ</em>, is found in this operon which encodes a cytoplasmic protein thought to be involved in biosynthesis of the complex <span class="citation">[<a href="#ref-sodergren_nari_1988" role="doc-biblioref">37</a>]</span>. This has been referred to as the <span class="math inline">\(\delta\)</span> subunit but does not copurify with the enzyme. The T. pantotropha <em>narJ</em> and <em>narHI</em> genes have also been sequenced <span class="citation">[<a href="#ref-berks_sequence_1995" role="doc-biblioref">28</a>]</span>. In <em>E. coli</em>, a second nitrate reductase operon, <em>narZYWV</em>, arising from gene duplication, is also found <span class="citation">[<a href="#ref-blasco_nitrate_1990" role="doc-biblioref">38</a>]</span>. The <em>E. coli narGHIJ</em> operon is also linked to three other genes nar genes, <em>narXL</em>, a two-component nitrate/nitrite sensing regulator <span class="citation">[<a href="#ref-nohno_narx_1989" role="doc-biblioref">39</a>]</span>, Section 1.7.3 and <em>narK</em>, a putative nitrite exporter <span class="citation">[<a href="#ref-rowe_nark_1994" role="doc-biblioref">40</a>]</span>, or nitrate/nitrite antiporter <span class="citation">[<a href="#ref-demoss_nark_1991" role="doc-biblioref">41</a>]</span>. Nitrate reduction is cytoplasmic; in <em>E. coli</em> the product nitrite may be reduced to ammonia by a cytoplasmic sirohaem-containing enzyme <span class="citation">[<a href="#ref-peakman_nucleotide_1990" role="doc-biblioref">42</a>]</span> or under some circumstances, exported to the periplasm by <em>narK</em> and reduced there by a <em>c</em> haem-containing nitrite reductase <span class="citation">[<a href="#ref-kajie_purification_1986" role="doc-biblioref">43</a>]</span>. A similar system for nitrite export must operate in the denitrifying bacteria and has often assumed to be a nitrate/nitrite antiporter. No such protein has yet been identified in denitrifying bacteria, though other transport systems have been identified in prokaryotes In the cyanobacteria these include an active transporter of the ABC type <span class="citation">[<a href="#ref-omata_structure_1995" role="doc-biblioref">44</a>]</span> and a sodium-linked symporter <span class="citation">[<a href="#ref-rodriguez_nitrate_1992" role="doc-biblioref">45</a>]</span>, both of which may transport nitrate and nitrite, although nitrate reduction in these organisms is assimilatory rather than respiratory.</p>
</div>
<div id="the-periplasmic-nitrate-reductase" class="section level3" number="1.4.2">
<h3><span class="header-section-number">1.4.2</span> The periplasmic nitrate reductase</h3>
<p>Periplasmic nitrate reductases have been purified from several bacteria, including <em>R. capsulatus</em> <span class="citation">[<a href="#ref-mcewan_periplasmic_1987" role="doc-biblioref">46</a>]</span>, <em>T. pantotropha</em> <span class="citation">[<a href="#ref-berks_purification_1994" role="doc-biblioref">47</a>]</span> and <em>A. eutrophus</em> <span class="citation">[<a href="#ref-siddiqui_structure_1993" role="doc-biblioref">48</a>]</span>. In addition, the complete gene locus <em>nap</em> has been mapped and sequenced from <em>T. pantotropha</em> <span class="citation">[<a href="#ref-berks_napedabc_1995" role="doc-biblioref">16</a>]</span>, <em>R. sphaeroides</em> <span class="citation">[<a href="#ref-reyes_isolation_1996" role="doc-biblioref">49</a>]</span> and <em>E. coli</em> <span class="citation">[<a href="#ref-richterich_u00008_1993" role="doc-biblioref">17</a>]</span>. Although purified from a relatively small number of species the enzyme is probably widespread amongst the <em>Proteobacteria</em>; probes for the gene have been used to isolate up to forty species of bacteria from soil and water environments in one locale <span class="citation">[<a href="#ref-carter_soil_1995" role="doc-biblioref">50</a>]</span>.</p>
<p>The enzyme is a heterodimer of two subunits, NapA (90 kDa) and NapB (16 kDa) <span class="citation">[<a href="#ref-berks_purification_1994" role="doc-biblioref">47</a>]</span>. NapB is a dihaem cytochrome <em>c</em><span class="math inline">\(_{552}\)</span>, which is believed to accept electrons from ubiquinol, via a quinol oxidase complex (possibly the <em>napE</em> gene product) and a tetrahaem cytochrome <em>c</em> (encoded by <em>napC</em>, <span class="citation">[<a href="#ref-berks_napedabc_1995" role="doc-biblioref">16</a>]</span>). NapA is the catalytic site, containing the MGD cofactor and, at least in <em>T. pantotropha</em>, an iron-sulphur cluster <span class="citation">[<a href="#ref-breton_characterization_1994" role="doc-biblioref">51</a>]</span>. The enzyme is catalytically distinct from the membrane-bound nitrate reductase, in that it is not competitively inhibited by azide, nor can it reduce chlorate or bromate <span class="citation">[<a href="#ref-berks_purification_1994" role="doc-biblioref">47</a>]</span>.</p>
<p>Extensive sequencing of the <em>nap</em> loci in <em>T. pantotropha</em>, <em>R. sphaeroides</em> and <em>E. coli</em> (the <em>aeg-46.5</em> locus) has revealed a number of open reading frames besides those of the structural genes. <em>T. pantotropha</em> contains the aforementioned <em>napE</em> and <em>napC</em>, as well as <em>napD</em>, coding for a cytoplasmic protein of unknown function. <em>E. coli</em> contains <em>napF</em>, <em>napG</em> and <em>napH</em>; these proteins have been suggested to bind iron-sulphur clusters and to be involved in electron transport to NapAB <span class="citation">[<a href="#ref-berks_enzymes_1995" role="doc-biblioref">31</a>]</span>. In addition, there are a number of genes downstream of the <em>nap</em> cluster involved in <em>c</em>-type cytochrome biogenesis <span class="citation">[<a href="#ref-thony-meyer_bacterial_1994" role="doc-biblioref">52</a>]</span> and <em>narP</em>, a nitrate/nitrite response regulator (Section 1.7.3).</p>
<p>The physiological role of the periplasmic nitrate reductase is not especially clear and may even vary in different organisms. In <em>T. pantotropha</em>, the enzyme was initially assigned a role in aerobic denitrification, a feature that originally distinguished this organism from the very closely related <em>P. denitrificans</em> <span class="citation">[<a href="#ref-stouthamer_metabolic_1992" role="doc-biblioref">53</a>]</span>. One rationale for this is that oxygen is known to inhibit nitrate transport to the cytoplasm <span class="citation">[<a href="#ref-noji_molecular_1987" role="doc-biblioref">54</a>]</span>, the site of reduction by the membrane-bound reductase, whereas a periplasmic location would effectively bypass this route of inhibition. However, more recent research seems to suggest that <em>T. pantotropha</em> does not denitrify aerobically to the extent previously reported <span class="citation">[<a href="#ref-robertson_confirmation_1995" role="doc-biblioref">55</a>]</span>, perhaps as a result of repeated sub-culturing in the laboratory. Furthermore, <em>P. denitrificans</em> also possesses a periplasmic nitrate reductase and synthesis of the other enzymes of denitrification appears to be quite strictly controlled by anaerobiosis. However, the periplasmic nitrate reductase may be expressed under conditions of stress (such as entering stationary phase, <span class="citation">[<a href="#ref-siddiqui_structure_1993" role="doc-biblioref">48</a>]</span>) and as at least pre-synthesised cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase appears to persist on switching from anaerobic to aerobic growth <span class="citation">[<a href="#ref-baumann_dynamics_1996" role="doc-biblioref">56</a>]</span>, the enzyme may play a role in aerobic denitrification as an adaptation to cope with rapidly changing levels of aeration in the environment.</p>
<p>Other roles have been suggested for the enzyme, involving the concepts of redox balance and redox poise. In photoheterotrophs and chemoheterotrophs, an optimum ratio of ubiquinol:ubiquinone is required to maintain turnover of enzyme complexes such as NADH dehydrogenase, the cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex and the photosynthetic reaction centre <span class="citation">[<a href="#ref-berks_enzymes_1995" role="doc-biblioref">31</a>]</span>. Certain growth conditions such as low light intensity or the provision of highly reduced carbon substrate can lead to an increase in the UQH<span class="math inline">\(_2\)</span>:UQ ratio and a fall in growth rate. The quinol oxidase activity of periplasmic nitrate reductase would alleviate this by consuming ubiquinol. There is good evidence for this model; in <em>T. pantotropha</em>, aerobic growth on the highly-reduced substrate butyrate results in a ten-fold increase in Nap expression compared to succinate <span class="citation">[<a href="#ref-richardson_influence_1992" role="doc-biblioref">57</a>]</span>, whilst nitrate reduction in photosynthetic <em>R. capsulatus</em> is markedly increased under low light intensity <span class="citation">[<a href="#ref-richardson_role_1988" role="doc-biblioref">58</a>]</span>.</p>
</div>
<div id="the-copper-nitrite-reductase" class="section level3" number="1.4.3">
<h3><span class="header-section-number">1.4.3</span> The copper nitrite reductase</h3>
<p>Copper nitrite reductase is one of two types of eubacterial dissimilatory nitrite reductase, the other being the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> enzyme discussed in Section 1.5. The two enzymes are distributed fairly evenly amongst bacteria with a slight preponderance of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> enzyme <span class="citation">[<a href="#ref-averill_dissimilatory_1996" role="doc-biblioref">59</a>]</span>. There is no correlation between genus (or even species) and the type of nitrite reductase present; for example <em>Alcaligenes eutrophus</em> H16 contains the <em>cd</em><span class="math inline">\(_1\)</span> type <span class="citation">[<a href="#ref-sann_cytochrome_1994" role="doc-biblioref">60</a>]</span> as does <em>A. faecalis</em> strain IAM 1015 <span class="citation">[<a href="#ref-iwasaki_cytochrome_1971" role="doc-biblioref">61</a>]</span>, whereas <em>A. faecalis</em> S-6 contains the copper enzyme <span class="citation">[<a href="#ref-kakutani_purification_1981" role="doc-biblioref">62</a>]</span>. The two enzymes are never found in the same organism but are functionally equivalent; expression of the copper nitrite reductase gene <em>nirK</em> in a <em>Ps. stutzeri</em> mutant lacking cytochrome <em>cd</em><span class="math inline">\(_1\)</span> restores nitrite reductase activity <span class="citation">[<a href="#ref-glockner_copper-containing_1993" role="doc-biblioref">63</a>]</span>.</p>
<p>Copper nitrite reductases have been purified from several Gram-negative bacteria, including <em>Achromobacter cycloclastes</em> <span class="citation">[<a href="#ref-iwasaki_nitrite_1972" role="doc-biblioref">64</a>]</span>, <em>Alcaligenes faecalis</em> S-6 <span class="citation">[<a href="#ref-kakutani_purification_1981" role="doc-biblioref">62</a>]</span> and <em>Pseudomonas aureofaciens</em> <span class="citation">[<a href="#ref-zumft_type_1987" role="doc-biblioref">65</a>]</span> and one Gram-positive bacterium, <em>Bacillus halodenitrificans</em> <span class="citation">[<a href="#ref-denariaz_denitrifying_1991" role="doc-biblioref">66</a>]</span>. It now seems clear that all the enzymes are homotrimeric, with each 37 kDa subunit containing a variant of a type I centre and one type II copper centre. Type I centres are mononuclear, cysteine-ligated, have high redox potentials and are an intense blue colour (but see below). They are always involved in electron transfer, rather than catalysis. Type II copper centres contain a single copper atom, with spectroscopic properties similar to those of inorganic copper complexes. They are ligated predominantly by histidine residues and are involved with catalysis at the active site, where substrate or exogenous ligands can displace a water-derived ligand. The type I centre in the copper nitrite reductase from <em>A. cycloclastes</em> is unusual in that it produces an intense green absorption at 458 nm; this is thought to be due to a short Cu-S-Met distance and substantial charge transfer through this bond <span class="citation">[<a href="#ref-han_resonance_1993" role="doc-biblioref">67</a>]</span>.</p>
<p>Copper nitrite reductases are relatively well understood due to the elucidation of two X-ray crystal structures from the organisms <em>A. cycloclastes</em> <span class="citation">[<a href="#ref-godden_23_1991" role="doc-biblioref">18</a>]</span> and <em>A. faecalis</em> S-6 <span class="citation">[<a href="#ref-kukimoto_x-ray_1994" role="doc-biblioref">19</a>]</span>. These have revealed that each monomer forms two <span class="math inline">\(\beta\)</span> barrel domains stabilised by a helical region. The histidine, cysteine and methionine ligands to the copper centres have also been identified; the type II copper centres are positioned at the subunit interfaces and share ligands from adjacent subunits. These centres can be removed and reconstituted, showing them to be the site of nitrite reduction. Some site directed mutagenesis of copper nitrite reductase has been carried out, which has identified the important role of the copper ligands <span class="citation">[<a href="#ref-kukimoto_x-ray_1994" role="doc-biblioref">19</a>]</span>, and the negatively-charged surface residues thought to interact with the electron donor pseudoazurin (Section 1.4.6 <span class="citation">[<a href="#ref-kukimoto_studies_1996" role="doc-biblioref">68</a>]</span>). A mechanism of nitrite reduction has been proposed which is quite similar to the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> enzyme (Section 1.5), involving protonation of the N-bound nitrite molecule by a neighbouring histidine residue, followed by dehydration to give a cuprous-NO+ species and followed by release of nitric oxide. The active site of copper nitrite reductase is comparable to that of carbonic anhydrase in that a number of similarly placed histidine residues act as proton donors <span class="citation">[<a href="#ref-strange_substrate-binding_1995" role="doc-biblioref">69</a>]</span>. However, there may be some mechanistic differences between the copper nitrite reductase and cytochrome <em>cd</em><span class="math inline">\(_1\)</span> -type nitrite reductase. No evidence for a Cu(I)-NO+ species has been obtained by H<span class="math inline">\({_2}^{18}\)</span>O exchange experiments using the copper nitrite reductase, <span class="citation">[<a href="#ref-hulse_evidence_1989" role="doc-biblioref">70</a>]</span> and the enzyme also seems to produce significant amounts of N<span class="math inline">\(_2\)</span>O as judged by isotope exchange experiments <span class="citation">[<a href="#ref-jackson_evidence_1991" role="doc-biblioref">71</a>]</span>. This occurs when nitric oxide levels are quite high, implying reaction between a labile Cu-NO species and NO or NO<span class="math inline">\(_2^-\)</span>. However, as with cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, this may not occur physiologically, as NO is normally rapidly removed in the cell by the activity of nitric oxide reductase (see below).</p>
</div>
<div id="the-nitric-oxide-reductase" class="section level3" number="1.4.4">
<h3><span class="header-section-number">1.4.4</span> The nitric oxide reductase</h3>
<p>For many years, it was unclear whether the product of nitrite reduction was nitric oxide or nitrous oxide and, if the former, whether nitric oxide was a free intermediate in denitrification. This situation has now been resolved by a number of methods: (1) trapping experiments using haemoglobin have shown nitric oxide to be freely diffusible <span class="citation">[<a href="#ref-goretski_trapping_1988" role="doc-biblioref">72</a>]</span>, (2) membranes of mutants lacking nitrite reductase still possess a nitric oxide reductase activity <span class="citation">[<a href="#ref-zumft_defects_1988" role="doc-biblioref">73</a>]</span> and most conclusively, (3) a distinct nitric oxide reductase enzyme has been identified, purified and shown to be essential for denitrification by gene disruption. Nitric oxide reductase has now been purified from several sources, including <em>P. denitrificans</em> <span class="citation">[<a href="#ref-carr_nitric_1990" role="doc-biblioref">74</a>]</span>, <em>A. cycloclastes</em> <span class="citation">[<a href="#ref-jones_nitric_1993" role="doc-biblioref">75</a>]</span> and <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>. On SDS-PAGE gels two subunits are seen; a cytochrome <em>c</em> of 17 kDa containing a single haem and a larger subunit (of apparent molecular mass 37 kDa, though actually 53 kDa from the gene sequence), binding a <em>b</em>-type haem. The enzyme also contains non-haem iron <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>. Biochemical studies of the protein solubilised in detergent vesicles have been carried out; cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> can donate electrons to the enzyme and the K<span class="math inline">\(_m\)</span> for nitric oxide under these conditions is sub-micromolar <span class="citation">[<a href="#ref-carr_nitric_1990" role="doc-biblioref">74</a>]</span>. This measurement seems reasonable as NO concentrations in whole cells are generally 10<span class="math inline">\(^{-7}\)</span> M or less <span class="citation">[<a href="#ref-goretski_steady-state_1990" role="doc-biblioref">77</a>]</span>.</p>
<p>Redox titrations have revealed only two redox components in the enzyme; the <em>c</em> haem component (+225 mV in <em>P. denitrificans</em>, +280 mV in <em>Ps. stutzeri</em>) and the <em>b</em> haem component (+305 mV or +322 mV in the same organisms respectively, Chan and Ferguson, unpublished work, <span class="citation">[<a href="#ref-kastrau_nitric_1994" role="doc-biblioref">78</a>]</span>). Early studies reported a stoichiometry of two haems per NorBC complex <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>, but more recent work indicates that there are two <em>b</em>-type haems (one high-spin that can bind carbon monoxide, the other low-spin), one <em>c</em> haem and one non-haem iron per enzyme complex <span class="citation">[<a href="#ref-fujiwara_cytochrome_1996" role="doc-biblioref">79</a>,<a href="#ref-girsch_purification_1997" role="doc-biblioref">80</a>]</span>. The membrane-bound nature of the enzyme makes purification quite difficult and it now seems that the earlier estimates of haem stoichiometry were erroneous.</p>
<p>The problems outlined above have made elucidation of a reaction mechanism for nitric oxide reductase quite problematic; in particular it was difficult to envisage how dimerisation of two NO molecules could be achieved at a single haem binding site. However, the cloning and sequencing of the <em>norBC</em> genes has led to several suggestions for a catalytic mechanism. The NorB subunit has structural similarity to subunit I of cytochrome <em>c</em> oxidase <span class="citation">[<a href="#ref-van_der_oost_heme-copper_1994" role="doc-biblioref">81</a>]</span>, in that both are predicted to have 12 transmembrane helices, with 6 conserved histidines that act as ligands (in the oxidase, to haems <em>a</em> and <em>a</em><span class="math inline">\(_3\)</span> and to the Cu<span class="math inline">\(_{A3}\)</span> site, <span class="citation">[<a href="#ref-iwata_structure_1995" role="doc-biblioref">82</a>]</span>. Furthermore, like nitric oxide reductase the more primitive cytochrome oxidase, cytochrome <em>cbb</em><span class="math inline">\(_3\)</span>, has a subunit II (FixN) possessing a cytochrome <em>c</em>, rather than a binuclear Cu<span class="math inline">\(_A\)</span> centre <span class="citation">[<a href="#ref-saraste_cytochrome_1994" role="doc-biblioref">83</a>]</span>. These analogies have strengthened the case for two <em>b</em>-type haems and the conserved histidine residues strongly suggest liganding to the non-haem iron identified in later preparations of nitric oxide reductase. This raises the intriguing possibility of NO reduction at a binuclear haem-non-haem iron centre. Modelling of this enzyme family has also been used to suggest an evolutionary progression from nitric oxide reductase, through cytochrome <em>cbb</em><span class="math inline">\(_3\)</span> to the cytochrome <em>c</em> oxidases, and to imply that the cytochromes <em>cbb</em><span class="math inline">\(_3\)</span> may have a different mechanism of proton pumping to the haem-copper enzymes, whilst nitric oxide reductase is probably not a proton pump <span class="citation">[<a href="#ref-van_der_oost_heme-copper_1994" role="doc-biblioref">81</a>]</span>. It is notable that cytochrome <em>c</em> oxidase is capable of nitric oxide reduction <span class="citation">[<a href="#ref-zhao_cytochrome_1995" role="doc-biblioref">84</a>]</span> and a recent paper describing nitric oxide reductase from <em>T. pantotropha</em> indicates that it has a cytochrome oxidase activity <span class="citation">[<a href="#ref-fujiwara_cytochrome_1996" role="doc-biblioref">79</a>]</span>.</p>
</div>
<div id="the-nitrous-oxide-reductase" class="section level3" number="1.4.5">
<h3><span class="header-section-number">1.4.5</span> The nitrous oxide reductase</h3>
<p>Nitrous oxide reductase, the final enzyme of the denitrification pathway is one of the most comprehensively studied of the denitrifying enzymes. It has been purified from numerous sources, including <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-coyle_nitrous_1985" role="doc-biblioref">85</a>]</span>, <em>P. denitrificans</em> <span class="citation">[<a href="#ref-snyder_purification_1987" role="doc-biblioref">86</a>]</span>, <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-soohoo_purification_1991" role="doc-biblioref">87</a>]</span> and <em>A. faecalis</em> <span class="citation">[<a href="#ref-matsubara_new-type_1972" role="doc-biblioref">88</a>]</span>. The enzyme appears essentially identical from all sources, being a homodimer of subunits each with a molecular mass of around 70 kDa. Nitrous oxide reductase is a soluble, periplasmic enzyme and contains four copper atoms per polypeptide <span class="citation">[<a href="#ref-coyle_nitrous_1985" role="doc-biblioref">85</a>]</span>. Spectroscopic models indicate that two types of site are present; a Cu<span class="math inline">\(_A\)</span> site (sometimes termed the “purple copper centre” due to its distinctive colour), related to that found in cytochrome <em>c</em> oxidase and which functions in electron transfer, and a catalytic site, variously termed Cu<span class="math inline">\(_B\)</span>, Cu<span class="math inline">\(_C\)</span> or Cu<span class="math inline">\(_Z\)</span>, the latter term being preferred. Of these the Cu<span class="math inline">\(_A\)</span> site is the better understood and has been extensively modelled on the basis of EPR and MCD data. It is similar to the site found in cytochrome <em>c</em> oxidase <span class="citation">[<a href="#ref-kelly_two_1993" role="doc-biblioref">89</a>]</span>; sequence similarity between the C-terminal of nitrous oxide reductase and cytochrome <em>c</em> oxidase subunit II (which binds Cu<span class="math inline">\(_A\)</span>) has been used to predict the ligands to the nitrous oxide reductase copper centre <span class="citation">[<a href="#ref-zumft_derived_1992" role="doc-biblioref">90</a>]</span>. In general, these predictions are supported by spectroscopic studies. The EPR spectrum of this centre is quite unusual and suggests that one electron is delocalised over both copper atoms, making the centre a one-electron acceptor <span class="citation">[<a href="#ref-farrar_model_1991" role="doc-biblioref">91</a>]</span>. The catalytic Cu<span class="math inline">\(_Z\)</span> site is rather less well understood, largely because it has fewer distinct spectroscopic properties. It has also been modelled by <span class="citation">[<a href="#ref-farrar_model_1991" role="doc-biblioref">91</a>]</span> and it is also thought to be dinuclear, containing 2 Cu(II) atoms. Reduction using dithionite is thought to reduce only one of these atoms to Cu(I), giving a dead-end product which is enzymically inactive. Ligands to the Cu<span class="math inline">\(_Z\)</span> site have been suggested by examining conserved residues known not to be involved with the Cu<span class="math inline">\(_A\)</span> site; 8 histidines and a single cysteine are possible candidates.</p>
<p>The enzymology of nitrous oxide reductase is rather complex. Two forms of the protein can be purified; the purple form (isolated anaerobically) and the pink form (isolated aerobically), with distinct spectroscopic features <span class="citation">[<a href="#ref-riester_nitrous_1989" role="doc-biblioref">92</a>]</span>, the anaerobic form having a higher specific activity. Reduction of the enzyme with dithionite followed by oxidation by molecular oxygen converts the purple form to the pink form. Dithionite reduction alone produces a third form of the enzyme, the “blue” form which still contains oxidised copper; this is associated with the Cu<span class="math inline">\(_Z\)</span> centre. The enzyme is inhibited by acetylene, azide, carbon monoxide and cyanide <span class="citation">[<a href="#ref-kristjansson_first_1980" role="doc-biblioref">93</a>]</span>, which presumably can act as copper ligands at one or both of the copper centres.</p>
<p>The purified enzyme shows the highest activities when assayed with reduced viologens as electron donors, activities using the presumed physiological donors are lower. However, turnover-dependent inactivation has been noted for some of these donors <em>e.g.</em> <span class="citation">[<a href="#ref-berks_purification_1993" role="doc-biblioref">94</a>]</span>. The activity of the enzyme from some species has been increased by incubation with reduced viologens <span class="citation">[<a href="#ref-snyder_purification_1987" role="doc-biblioref">86</a>]</span> and also by exposure to high pH buffer <span class="citation">[<a href="#ref-coyle_nitrous_1985" role="doc-biblioref">85</a>]</span>; the latter induces a permanent chemical change and it has been suggested that this reaction converts inactive forms of the Cu<span class="math inline">\(_Z\)</span> site to active species, perhaps by regenerating a thiolate ion <span class="citation">[<a href="#ref-berks_enzymes_1995" role="doc-biblioref">31</a>]</span>.</p>
<p>The complexity of the copper sites in nitrous oxide reductase has slowed progress in the understanding of its mechanism. However, the dinuclear nature of the Cu<span class="math inline">\(_Z\)</span> site suggests that nitrous oxide is bound as a bridging ligand between the two copper atoms; following electron transfer from the Cu<span class="math inline">\(_A\)</span> site, the N-O bond could be activated for cleavage, releasing dinitrogen. The intermolecular electron transfer is another intriguing aspect of this enzyme, in particular why the unusual purple copper centre is used in preference to the more normal type I blue copper centre. The electron pathway through the enzyme (whether through hydrogen bonds/through space or through covalent bonds) is another problem that remains to be elucidated.</p>
<p>The structural gene for nitrous oxide reductase, <em>nosZ</em>, has been cloned and sequenced from a number of sources, allowing the sequence analyses described previously. In <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-braun_structural_1992" role="doc-biblioref">95</a>]</span>, it is part of a large denitrification cluster containing a number of genes believed to be involved in biosynthesis of the enzyme. Similar clusters are thought to exist in <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-zumft_derived_1992" role="doc-biblioref">90</a>]</span> and <em>P. denitrificans</em> <span class="citation">[<a href="#ref-hoeren_sequence_1993" role="doc-biblioref">96</a>]</span>, though there is no evidence of the accessory genes in <em>A. eutrophus</em> <span class="citation">[<a href="#ref-zumft_derived_1992" role="doc-biblioref">90</a>]</span>. The <em>nos</em> gene cluster is discussed more fully in Section 1.6, in relation to the other genes of denitrification. An interesting feature of the <em>nosZ</em> genes is that the encoded NosZ protein appears to have an unusually long periplasmic targeting sequence (58-60 amino acids), compared to the more usual 26 or less <span class="citation">[<a href="#ref-von_heijne_signal_1985" role="doc-biblioref">97</a>]</span>. Another unusual feature is that the N-termini of the targeting sequences contain a quite highly conserved motif; these features have been suggested to point to a common export pathway for periplasmic proteins with complex metal cofactors <span class="citation">[<a href="#ref-berks_common_1996" role="doc-biblioref">98</a>]</span>.</p>
</div>
<div id="small-electron-transfer-proteins" class="section level3" number="1.4.6">
<h3><span class="header-section-number">1.4.6</span> Small electron transfer proteins</h3>
<p>The periplasms of Gram-negative bacteria contain a variety of small, soluble redox proteins. These are assumed to mediate electron transfer between membrane-bound redox complexes, such as the cytochrome <em>bc</em><span class="math inline">\(_1\)</span> complex and the periplasmic enzymes, including those of denitrification, although in few cases has an <em>in vivo</em> reaction been directly demonstrated. The small electron transfer proteins group into two types: the small cytochromes <em>c</em> and the cupredoxins.</p>
<p>The cytochromes <em>c</em> are a large family of proteins, that have been grouped into various families by consideration of size, sequence similarity and physicochemical properties <span class="citation">[<a href="#ref-ambler_structure_1980" role="doc-biblioref">99</a>]</span>. They have been popular subjects for the study of protein structure and function for many years and a vast literature exists on the subject, which can only be briefly summarised here. The periplasmic cytochromes <em>c</em> involved in electron transfer include cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> from <em>P. denitrificans</em>, cytochrome <em>c</em><span class="math inline">\(_{551}\)</span> from <em>Ps. aeruginosa</em> and cytochrome <em>c</em><span class="math inline">\(_2\)</span> from Rhodobacter capsulatus. These proteins are characterised as Class I cytochromes <em>c</em>: they are generally small (80-120 amino acids), and contain one covalently bound <em>c</em> haem. The haem group is low-spin, co-ordinated by histidine and methionine and found near the N terminus of the protein. Many crystal structures have been determined, revealing a mostly a helical structure, a crevice in which the haem group is buried and a positively-charged face around the haem edge, through which redox partners may interact by electrostatic interactions <span class="citation">[<a href="#ref-hall_reaction_1987" role="doc-biblioref">100</a>]</span>.</p>
<p>The cupredoxins are a family of small (around 13 kDa) copper-containing proteins containing what are known as type I or “blue copper” centres. They contain a single copper atom which is involved in electron transfer with donor and acceptor proteins. Cupredoxins absorb intensely near 600 nm giving them an intense blue colour. The copper atom is
liganded by two histidine residues, a cysteine and a weak methionine ligand; subtle variations in the ligand geometry give rise to four spectroscopic sub-classes of cupredoxin: (i) azurins, (ii) pseudoazurins, (iii) plastocyanin/amicyanin, (iv) stellacyanin and associated proteins. Examples of cupredoxins include pseudoazurin from <em>T. pantotropha</em> <span class="citation">[<a href="#ref-moir_purification_1993" role="doc-biblioref">101</a>]</span>, azurin from <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-parr_purification_1976" role="doc-biblioref">102</a>]</span> and amicyanin from <em>P. denitrificans</em> <span class="citation">[<a href="#ref-husain_properties_1986" role="doc-biblioref">103</a>]</span>. A number of X-ray crystal structures are known which show a high degree of similarity; cupredoxins contain a Greek key <span class="math inline">\(\beta\)</span>-barrel fold, consisting of 8 <span class="math inline">\(\beta\)</span> sheets. As with the cytochromes <em>c</em>, surface regions that govern the interactions between the cupredoxins and their redox partners have been defined; in azurin, His-117 is thought to transfer electrons to partners bound by a patch of hydrophobic residues around this residue <span class="citation">[<a href="#ref-van_de_kamp_involvement_1990" role="doc-biblioref">104</a>]</span>. However, recent analysis of several redox partners has introduced the concept of pseudospecific interactions, in which both hydrophobic patches and oppositely charged surface regions act to bring proteins together and promote electron transfer <span class="citation">[<a href="#ref-williams_pseudospecific_1995" role="doc-biblioref">105</a>]</span>.</p>
<p>Proteins are generally assumed to be physiological redox partners on the basis of their reactivity in <em>in vitro</em> assays, as it is difficult to prove an <em>in vivo</em> relationship. Where this has been attempted, mutant strains lacking the small redox protein under study have been used. A mutant of <em>P. denitrificans</em> lacking cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> has been shown to denitrify at a similar rate to the wild-type organism but anaerobic growth becomes sensitive to diethyldithiocarbamate (a copper chelator), suggesting a role for a copper protein (probably pseudoazurin) in electron transport <span class="citation">[<a href="#ref-moir_properties_1994" role="doc-biblioref">106</a>]</span>. It is often the case that other redox proteins can substitute for an absent protein in redox carrier mutants; for example a mutant of <em>R. capsulatus</em> lacking cytochrome <em>c</em><span class="math inline">\(_2\)</span> can still grow photosynthetically, because a membrane-bound cytochrome <em>c</em> takes over its role <span class="citation">[<a href="#ref-jenney_novel_1993" role="doc-biblioref">107</a>]</span>.</p>
<p>Redox partners may also be inferred from co-expression. In <em>P. denitrificans</em> grown methylotrophically, two additional cytochromes (<em>c</em><span class="math inline">\(_{551i}\)</span> and <em>c</em><span class="math inline">\(_{553i}\)</span>, <span class="citation">[<a href="#ref-husain_characterization_1986" role="doc-biblioref">108</a>]</span>) and the copper protein amicyanin <span class="citation">[<a href="#ref-husain_properties_1986" role="doc-biblioref">103</a>]</span> appear, suggesting that they are involved in electron transfer to or from the methanol dehydrogenase.</p>
</div>
</div>
<div id="the-cytochrome-cd_1-nitrite-reductase" class="section level2" number="1.5">
<h2><span class="header-section-number">1.5</span> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</h2>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase is the alternative dissimilatory nitrite reductase to the copper enzyme (Section 1.4.3). Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was first isolated from <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-horio_terminal_1958" role="doc-biblioref">109</a>]</span>. It reduces oxygen and has been referred to as soluble cytochrome oxidase, but it was later shown that the physiological role of the enzyme is nitrite reduction. This was based on the findings that the K<span class="math inline">\(_m\)</span> for nitrite is much lower than that for oxygen and that the enzyme is induced during anaerobic growth on nitrate <span class="citation">[<a href="#ref-yamanaka_biological_1963" role="doc-biblioref">110</a>]</span>. However oxygen reduction still attracts interest; it is known to be a four-electron reduction to water <span class="citation">[<a href="#ref-timkovich_evidence_1979" role="doc-biblioref">111</a>]</span> and may still prove relevant in the study of membrane bound cytochrome oxidases.</p>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> has since been identified, isolated and studied in detail from many sources; some of these include <em>P. denitrificans</em> <span class="citation">[<a href="#ref-lam_nitrite_1969" role="doc-biblioref">112</a>]</span>, <em>T. pantotropha</em> <span class="citation">[<a href="#ref-moir_purification_1993" role="doc-biblioref">101</a>]</span>, <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-zumft_defects_1988" role="doc-biblioref">73</a>]</span>, <em>A. faecalis</em> <span class="citation">[<a href="#ref-iwasaki_cytochrome_1971" role="doc-biblioref">61</a>]</span>, <em>A. eutrophus</em> <span class="citation">[<a href="#ref-sann_cytochrome_1994" role="doc-biblioref">60</a>]</span>, <em>Thiobacillus denitrificans</em> <span class="citation">[<a href="#ref-sawhney_sulphide-linked_1978" role="doc-biblioref">113</a>]</span>, <em>Magnetospirillum magnetotacticum</em> <span class="citation">[<a href="#ref-yamazaki_nitrite_1995" role="doc-biblioref">114</a>]</span> and <em>Roseobacter denitrificans</em> (formerly <em>Erythrobacter sp.</em> Och114 <span class="citation">[<a href="#ref-doi_two_1989" role="doc-biblioref">115</a>]</span>). Their gross properties are all similar, though as will be discussed later, there are important differences between them. The enzyme from <em>Ps. aeruginosa</em> is the most intensively studied of the family in terms of biochemical mechanistic data and most of the discussion below pertains to data from this enzyme.</p>
<div id="subunit-structure" class="section level3" number="1.5.1">
<h3><span class="header-section-number">1.5.1</span> Subunit structure</h3>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from all sources is a dimer of two identical subunits of molecular masses around 55-65 kDa, as judged by SDS-PAGE, size exclusion chromatography and sedimentation analysis <span class="citation">[<a href="#ref-kuronen_new_1972" role="doc-biblioref">116</a>]</span>. Each subunit contains two haem groups; a covalently-bound <em>c</em> haem identical to that found in mitochondrial cytochrome <em>c</em> and a non-covalently bound <em>d</em><span class="math inline">\(_1\)</span> haem, which is unique to this enzyme (Figure 1.3).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-3"></span>
<img src="figure/fig1-3.png" alt="(a) Structure of _d_$_1$ haem and (b) of protoporphyrin IX. The _d_$_1$ haem is unique to cytochrome _cd_$_1$. Protoporphyrin IX can occur as a non-covalently bound prosthetic group, where it is referred to as _b_-haem. In _c_-type cytochromes, protoporphyrin IX binds covalently via the vinyl side chains to two cysteine residues in the polypeptide at the consensus site CXXCH (X = any amino acid)." width="50%" />
<p class="caption">
Figure 1.3: (a) Structure of <em>d</em><span class="math inline">\(_1\)</span> haem and (b) of protoporphyrin IX. The <em>d</em><span class="math inline">\(_1\)</span> haem is unique to cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Protoporphyrin IX can occur as a non-covalently bound prosthetic group, where it is referred to as <em>b</em>-haem. In <em>c</em>-type cytochromes, protoporphyrin IX binds covalently via the vinyl side chains to two cysteine residues in the polypeptide at the consensus site CXXCH (X = any amino acid).
</p>
</div>
<p>The latter haem is distinguished by the presence of oxo and carboxylate moieties and partial saturation on two adjacent pyrroles (I and II) of the porphyrin ring. Its structure has been confirmed by NMR spectroscopy and total chemical synthesis <span class="citation">[<a href="#ref-chang_evidence_1986" role="doc-biblioref">117</a>]</span>. Haem <em>d</em><span class="math inline">\(_1\)</span> can be removed from the protein to give an inactive semi-apo protein; activity can be restored by reconstitution with the haem group, or to a much lesser extent, <em>a</em>-type haem <span class="citation">[<a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a>]</span>.</p>
</div>
<div id="properties-of-the-haem-groups-in-cytochrome-cd_1" class="section level3" number="1.5.2">
<h3><span class="header-section-number">1.5.2</span> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></h3>
<p>Haem groups and their ligands are fundamental determinants of the catalytic properties of enzymes like cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The ligands to the haems are amino acid residues from the polypeptide chain; they control access of the substrate and the electrochemistry of the reaction by modulating the reduction potential of the haem. Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> has been investigated by numerous spectroscopic methods, including UV-visible spectroscopy, nuclear magnetic resonance (NMR), electron paramagnetic resonance (EPR) and magnetic circular dichroism (MCD), from which models have been developed to explain the observed spin states and absorption features of the haem groups. In addition, a number of haem redox potentials have been measured for the enzyme.</p>
<p>In fully oxidised cytochrome <em>cd</em><span class="math inline">\(_1\)</span> both haems are in a low spin state, indicating that they are hexacoordinate. Reduction causes the <em>d</em><span class="math inline">\(_1\)</span> haem to become high spin indicating that it has lost an axial ligand <span class="citation">[<a href="#ref-walsh_magnetic_1979" role="doc-biblioref">119</a>]</span>. This implies that the <em>d</em><span class="math inline">\(_1\)</span> haem is the site of nitrite binding and a number of ligands have been used to probe this binding-site, including azide, imidazole <span class="citation">[<a href="#ref-walsh_characterization_1981" role="doc-biblioref">120</a>]</span>, cyanide <span class="citation">[<a href="#ref-barber_reactions_1978" role="doc-biblioref">121</a>]</span> and carbon monoxide <span class="citation">[<a href="#ref-parr_reaction_1975" role="doc-biblioref">122</a>]</span>. Similar behaviour has been noted for the periplasmic cytochrome <em>c</em> peroxidase enzyme of <em>P. denitrificans</em>, in which peroxide access at the catalytic site is thought to be blocked by a sixth axial ligand until the electron receiving haem group is fully reduced; this would prevent partial reduction of hydrogen peroxide to more reactive oxygen species <span class="citation">[<a href="#ref-gilmour_kinetics_1994" role="doc-biblioref">123</a>]</span>.</p>
<p>The identity of the ligands to the haems has taken a long time to resolve and apparently conflicting results have been reported for different cytochromes <em>cd</em><span class="math inline">\(_1\)</span>. Low temperature MCD and EPR spectra of the <em>Ps. aeruginosa</em> enzyme in the oxidised state suggest histidine-methionine ligation at the <em>c</em>-haem and possibly bis-histidine ligation at the <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-sutherland_investigation_1986" role="doc-biblioref">124</a>]</span>. This is also the case for <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-cheesman_two_1997" role="doc-biblioref">125</a>]</span>. In contrast, EPR and Mössbauer spectroscopy of the enzyme from <em>Th. denitrificans</em> suggested bis-histidine ligation at both haems <span class="citation">[<a href="#ref-huynh_mossbauer_1982" role="doc-biblioref">126</a>]</span>, whilst the <em>T. pantotropha</em> crystal structure (Section 1.5.3.1) has bis-histidine at the <em>c</em> haem and an unusual histidine-tyrosine at the <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>. Midpoint redox potentials of the haems have been reported for <em>Ps. aeruginosa</em> by several groups, with some disagreement. The values reported to date are summarised in Table 1.2.</p>
<table>
<caption>
<span id="tab:table12">Table 1.2: </span>Standard reduction potentials reported by other workers of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The figures for the difference between the midpoint redox potentials of the <em>c</em> and <em>d</em><span class="math inline">\(_1\)</span> haems are given where this measurement was the intention of the experiment, rather than a direct
measurement of the two values. The pH at which the reduction potential was measured is reported where it differs from pH 7.0.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Organism
</th>
<th style="text-align:left;">
<em>E</em><span class="math inline">\(^{o&#39;}\)</span> of <em>c</em> haem (mV)
</th>
<th style="text-align:left;">
<em>E</em><span class="math inline">\(^{o&#39;}\)</span> of <em>d</em><span class="math inline">\(_1\)</span> haem (mV)
</th>
<th style="text-align:left;">
<em>E</em><span class="math inline">\(^{o&#39;}\)</span> (<em>c</em> - <em>d</em><span class="math inline">\(_1\)</span>) (mV)
</th>
<th style="text-align:left;">
Reference
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<em>Pseudomonas aeruginosa</em>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
73
</td>
<td style="text-align:left;">
Horio <em>et al</em>. (1961) <span class="citation">[<a href="#ref-horio_relative_1961" role="doc-biblioref">128</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
288
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Kamen and Horio (1970) <span class="citation">[<a href="#ref-kamen_bacterial_1970" role="doc-biblioref">129</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
290
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
40-100
</td>
<td style="text-align:left;">
Wharton <em>et al</em>. (1973) <span class="citation">[<a href="#ref-wharton_cytochrome_1973" role="doc-biblioref">130</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
Shimada and Orii (1976) <span class="citation">[<a href="#ref-shimada_oxidation-reduction_1976" role="doc-biblioref">131</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
60
</td>
<td style="text-align:left;">
Blatt and Pecht (1979) <span class="citation">[<a href="#ref-blatt_allosteric_1979" role="doc-biblioref">132</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
294
</td>
<td style="text-align:left;">
287
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Schichman and Gray (1981) <span class="citation">[<a href="#ref-schichman_kinetics_1981" role="doc-biblioref">133</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
288
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Silvestrini <em>et al</em>. (1982) <span class="citation">[<a href="#ref-silvestrini_kinetics_1982" role="doc-biblioref">134</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
25 (pH 8)
</td>
<td style="text-align:left;">
Carson <em>et al</em>. (1986) <span class="citation">[<a href="#ref-carson_variations_1986" role="doc-biblioref">135</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pseudomonas nautica</em> 617
</td>
<td style="text-align:left;">
234 (pH 7.6)
</td>
<td style="text-align:left;">
199 (pH 7.6)
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Besson <em>et al</em>. (1995) <span class="citation">[<a href="#ref-besson_cytochrome_1995" role="doc-biblioref">136</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Magnetospirillum magnetotacticum</em>
</td>
<td style="text-align:left;">
191
</td>
<td style="text-align:left;">
180
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Yamazaki <em>et al</em>. (1995) <span class="citation">[<a href="#ref-yamazaki_nitrite_1995" role="doc-biblioref">114</a>]</span>
</td>
</tr>
</tbody>
</table>
<p>Measurement seems to be complicated by a strong cooperative redox reaction between the haems <span class="citation">[<a href="#ref-blatt_allosteric_1979" role="doc-biblioref">132</a>]</span> and by the possibility of <em>d</em><span class="math inline">\(_1\)</span> haem loss and protein denaturation during the experiment (Alrik Koppenhöfer, personal communication). The <em>c</em> haem value seems the more consistent of the two; for <em>Ps. aeruginosa</em> this is estimated at around +288 mV (pH 7.0). The <em>d</em><span class="math inline">\(_1\)</span> value has been extrapolated from data used to measure the difference between the two haems and has been variously estimated as being between 25 and 70 mV lower. For the enzyme from <em>M. magnetotacticum</em> (which may be an atypical member of the family), the <em>c</em> and <em>d</em><span class="math inline">\(_1\)</span> values are estimated as +191 and +180 mV respectively <span class="citation">[<a href="#ref-yamazaki_nitrite_1995" role="doc-biblioref">114</a>]</span>. The corresponding values reported for <em>Pseudomonas nautica</em> strain 617 cytochrome <em>cd</em><span class="math inline">\(_1\)</span> are +234 mV and +199 mV (pH 7.6) <span class="citation">[<a href="#ref-besson_cytochrome_1995" role="doc-biblioref">136</a>]</span>. The <em>c</em> haem in the semi-apo form of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> has been estimated as +243 mV (Koppenhöfer and Ferguson, unpublished work). The values are in the expected range given that the <em>c</em> haem is expected to receive electrons from donors of potentials around +250 mV, though the <em>d</em><span class="math inline">\(_1\)</span> values are somewhat lower than expected given that electrons are thought to flow from the <em>c</em> to the <em>d</em><span class="math inline">\(_1\)</span> haem.</p>
</div>
<div id="folding-and-conformation-of-cytochrome-cd_1" class="section level3" number="1.5.3">
<h3><span class="header-section-number">1.5.3</span> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></h3>
<p>At present seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span> are known, from the organisms <em>Pseudomonas aeruginosa</em> <span class="citation">[<a href="#ref-silvestrini_nitrite_1989" role="doc-biblioref">137</a>]</span>, <em>Pseudomonas stutzeri</em> Zobell <span class="citation">[<a href="#ref-jungst_nirstbm_1991" role="doc-biblioref">138</a>]</span>, <em>Pseudomonas stutzeri</em> JM300 <span class="citation">[<a href="#ref-smith_isolation_1992" role="doc-biblioref">139</a>]</span>, <em>Paracoccus denitrificans</em> IFO 12442 <span class="citation">[<a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a>]</span>, <em>Paracoccus denitrificans</em> PD1222 <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>, <em>Alcaligenes eutrophus</em> H16 <span class="citation">[<a href="#ref-rees_structural_1997" role="doc-biblioref">142</a>]</span> and <em>Thiosphaera pantotropha</em> (this work, Chapter 3). The first to be obtained was from <em>Ps. aeruginosa</em> and secondary structure predictions for this protein indicated that it contained two domains; a predominantly <span class="math inline">\(\alpha\)</span> helical N-terminal portion (including the <em>c</em>-haem region) and folds of largely <span class="math inline">\(\beta\)</span>-sheet towards the carboxy terminus <span class="citation">[<a href="#ref-silvestrini_nitrite_1989" role="doc-biblioref">137</a>]</span>. Further evidence of a two domain structure was obtained from partial digests with the alkaline serine protease subtilisin, which indicated that a dimer containing only <em>d</em><span class="math inline">\(_1\)</span> haem could be obtained <span class="citation">[<a href="#ref-horowitz_controlled_1982" role="doc-biblioref">143</a>]</span>. Circular dichroism spectroscopy using the <em>Ps. aeruginosa</em> enzyme indicated that the proportions of <span class="math inline">\(\alpha\)</span> helix and <span class="math inline">\(\beta\)</span>-sheet in the oxidised enzyme were 16% and 48% respectively, with a change on reduction to 35% <span class="math inline">\(\beta\)</span>-sheet <span class="citation">[<a href="#ref-tordi_circular-dichroic_1984" role="doc-biblioref">144</a>]</span>.</p>
<p>Conformational changes have been observed when the enzyme is reduced, although the precise character of these changes is unknown. In addition to a decrease in <span class="math inline">\(\beta\)</span>-sheet content the radius of gyration decreases as measured by small angle X-ray scattering <span class="citation">[<a href="#ref-berger_small-angle_1980" role="doc-biblioref">145</a>]</span>, the thermal stability of the protein decreases <span class="citation">[<a href="#ref-mitra_heat_1981" role="doc-biblioref">146</a>]</span> and crystals of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> crack on reduction <span class="citation">[<a href="#ref-akey_characterization_1980" role="doc-biblioref">147</a>]</span>. These observations point to some kind of rearrangement in quaternary structure with a change in redox state.</p>
<p>The orientation and relative position of the haem groups has long been a contentious issue. A slow electron transfer rate between the haems as observed by some groups suggested that they may be quite far apart. These data are contradicted by fluorescence quenching studies which indicated that the haems were grouped at one end of a subunit
<span class="citation">[<a href="#ref-mitra_location_1980" role="doc-biblioref">148</a>]</span> and by the recently obtained structure of the <em>T. pantotropha</em> enzyme (Section 1.5.3.1). In the latter case, the haems are at a relative angle of about 60<span class="math inline">\(\unicode{x00b0}\)</span> and the iron-iron distance between haems within a subunit is 20.6 <span class="math inline">\(\unicode{x212b}\)</span> <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>.</p>
<div id="crystal-structure-of-the-t.-pantotropha-enzyme" class="section level4" number="1.5.3.1">
<h4><span class="header-section-number">1.5.3.1</span> Crystal structure of the T. pantotropha enzyme</h4>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> was first isolated over thirty years ago. Crystallisation of the enzyme was reported almost immediately <span class="citation">[<a href="#ref-yamanaka_crystalline_1963" role="doc-biblioref">149</a>]</span> but although other groups have also obtained crystals <span class="citation">[<a href="#ref-akey_characterization_1980" role="doc-biblioref">147</a>,<a href="#ref-tegoni_crystallization_1994" role="doc-biblioref">150</a>]</span>, none of this work has led to a published X-ray structure. However, the crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the oxidised state from <em>T. pantotropha</em> has recently been determined (Figure 1.4).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-4"></span>
<img src="figure/fig1-4.png" alt="Crystal structure of cytochrome _cd_$_1$ in the oxidised form from _T. pantotropha_. Taken from Fülöp _et al_. (1995) [@fulop_anatomy_1995]. (a) shows the dimeric molecule with the $\alpha$-helical _c_-haem domain at the top and the $\beta$-propeller _d_$_1$ haem domain below. (b) shows the _d_$_1$ haem domains of each subunit from below, clearly illustrating how the eight-fold $\beta$-propeller structure forms a channel to bind the _d_$_1$ haem. The chain of each subunit is coloured from blue (N-terminus) to red (C-terminus)." width="50%" />
<p class="caption">
Figure 1.4: Crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the oxidised form from <em>T. pantotropha</em>. Taken from Fülöp <em>et al</em>. (1995) <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>. (a) shows the dimeric molecule with the <span class="math inline">\(\alpha\)</span>-helical <em>c</em>-haem domain at the top and the <span class="math inline">\(\beta\)</span>-propeller <em>d</em><span class="math inline">\(_1\)</span> haem domain below. (b) shows the <em>d</em><span class="math inline">\(_1\)</span> haem domains of each subunit from below, clearly illustrating how the eight-fold <span class="math inline">\(\beta\)</span>-propeller structure forms a channel to bind the <em>d</em><span class="math inline">\(_1\)</span> haem. The chain of each subunit is coloured from blue (N-terminus) to red (C-terminus).
</p>
</div>
<p>The <em>T. pantotropha</em> structure was initially determined at a resolution of 1.55 <span class="math inline">\(\unicode{x212b}\)</span> from diffraction data and using the primary sequence of the <em>P. denitrificans</em> PD1222 protein as a guide <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>. The latter sequence was assumed to be highly homologous to that of the protein from <em>T. pantotropha</em> and this has been shown subsequently to be the case (Chapter 3).</p>
<p>The crystal space group contains a dimer with each monomer organised in two domains, as predicted by earlier studies. The smaller domain is predominantly <span class="math inline">\(\alpha\)</span>-helical, binds the <em>c</em> haem and resembles a Class I fold to some degree, though with some significant differences. The ligands to this haem are His-17 and His-69; this is rather unexpected as bis-histidine ligation is normally associated with low redox potential <span class="citation">[<a href="#ref-moore_cytochromes_1990" role="doc-biblioref">151</a>]</span>, whereas the <em>c</em> haem is thought to have a redox potential in the +250 mV range. The
larger domain is an eight-bladed <span class="math inline">\(\beta\)</span>-propeller structure, a fold previously identified in methanol dehydrogenase <span class="citation">[<a href="#ref-xia_three-dimensional_1992" role="doc-biblioref">152</a>]</span>. Propeller structures with fewer blades are also seen in methylamine dehydrogenase (seven blades <span class="citation">[<a href="#ref-vellieux_structure_1989" role="doc-biblioref">153</a>]</span>) and neuraminidase (six blades <span class="citation">[<a href="#ref-varghese_structure_1983" role="doc-biblioref">154</a>]</span>). The central channel of this structure contains the <em>d</em><span class="math inline">\(_1\)</span> haem, which is ligated by His-200 and interestingly, by Tyr-25. Not only is this an unusual haem ligand, but it originates in the <em>c</em> haem domain; hence although separate, there is a distinct structural link between the domains. There are also a large number of important hydrogen bonds at the domain interface, and between subunits.</p>
<p>The structure answers some of the earlier questions about haem position and environment. The two haem types are at an angle of about 60<span class="math inline">\(\unicode{x00b0}\)</span> to each other within a monomer and separated by an iron-iron distance of 20.6 <span class="math inline">\(\unicode{x212b}\)</span>. The haem edge-edge distance in a monomer is 11.0 <span class="math inline">\(\unicode{x212b}\)</span>. Haem distances between subunits are even larger; <em>c</em> haem iron-iron is 42.4 <span class="math inline">\(\unicode{x212b}\)</span>. <em>c</em> haem-<em>d</em><span class="math inline">\(_1\)</span> haem is 47.0 <span class="math inline">\(\unicode{x212b}\)</span> and <em>d</em><span class="math inline">\(_1\)</span> haem iron-iron is 50.5 <span class="math inline">\(\unicode{x212b}\)</span>, with no edges closer than 38.5 <span class="math inline">\(\unicode{x212b}\)</span>. This would seem to rule out a simple iron-iron electron transfer mechanism and also discounts a binuclear haem binding site for oxygen reduction. The reported crystal structure, in combination with more recent work on reduced crystals of the enzyme and previous mechanistic studies, has allowed the development of a model for nitrite reduction which is discussed in the next section.</p>
</div>
</div>
<div id="kinetic-and-mechanistic-studies-of-the-enzyme" class="section level3" number="1.5.4">
<h3><span class="header-section-number">1.5.4</span> Kinetic and mechanistic studies of the enzyme</h3>
<p>The mechanism of nitrite reduction and electron transfer in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> has been investigated by a number of methods. Classical enzymological studies have been limited by the fact that the product of nitrite reduction, nitric oxide, is a potent inhibitor of the enzyme and may also bind to the haem groups of donor proteins used in the assay system; this leads to rather complex patterns of product inhibition and rate equations <span class="citation">[<a href="#ref-dhesi_patterns_1984" role="doc-biblioref">155</a>,<a href="#ref-robinson_implications_1979" role="doc-biblioref">156</a>]</span>. However, a considerable amount of information has been gained from isotopic labelling techniques, spectroscopy and fast kinetic methods and data so obtained, combined with structural information, have led to a coherent model for nitrite reduction (Figure 1.5).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-5"></span>
<img src="figure/fig1-5.png" alt="Proposed mechanism for nitrite reduction by cytochrome _cd_$_1$ from _Thiosphaera pantotropha_ (Fülöp _et al_., 1995 [@fulop_anatomy_1995]). (a) Nitrite binds at the reduced _d_$_1$ haem which is pentacoordinate, displacing a water molecule. (b) His-345 and His-388 act as proton donors, allowing the abstraction of an oxygen atom from bound nitrite. (c) This leaves nitric oxide bound at the oxidised _d_$_1$ haem. (d) Bound NO is then displaced by replacement with Tyr-25. The _d_$_1$ haem is re-reduced by an electron from the _c_-haem, displacing Tyr-25 and allowing the cycle to begin again." width="50%" />
<p class="caption">
Figure 1.5: Proposed mechanism for nitrite reduction by cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Thiosphaera pantotropha</em> (Fülöp <em>et al</em>., 1995 <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>). (a) Nitrite binds at the reduced <em>d</em><span class="math inline">\(_1\)</span> haem which is pentacoordinate, displacing a water molecule. (b) His-345 and His-388 act as proton donors, allowing the abstraction of an oxygen atom from bound nitrite. (c) This leaves nitric oxide bound at the oxidised <em>d</em><span class="math inline">\(_1\)</span> haem. (d) Bound NO is then displaced by replacement with Tyr-25. The <em>d</em><span class="math inline">\(_1\)</span> haem is re-reduced by an electron from the <em>c</em>-haem, displacing Tyr-25 and allowing the cycle to begin again.
</p>
</div>
<p>Electrons from donor proteins are believed to enter the enzyme at the <em>c</em> haem <span class="citation">[<a href="#ref-parr_electron-transfer_1977" role="doc-biblioref">157</a>]</span>. This is followed by an intramolecular electron transfer to the <em>d</em><span class="math inline">\(_1\)</span> haem. Reduction of this haem causes the loss of one of its axial ligands, allowing productive binding of nitrite. Isotope exchange studies have shown that the nitrite ion is protonated, then dehydrated to give an enzyme-bound nitrosyl intermediate <span class="citation">[<a href="#ref-kim_catalysis_1984" role="doc-biblioref">158</a>]</span>. Some earlier exchange studies also indicated that a substantial amount of reaction product was nitrous oxide <span class="citation">[<a href="#ref-kim_15n_1983" role="doc-biblioref">159</a>]</span>; however this has now been discounted as an artefact of the assay system, as <em>P. denitrificans</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> assayed using isoascorbate as electron donor generates stoichiometric amounts of nitric oxide from nitrite <span class="citation">[<a href="#ref-carr_energy-conserving_1989" role="doc-biblioref">160</a>]</span>. A bound Fe(III)-NO+ intermediate would not be detectable by EPR, but has recently been observed by FTIR spectroscopy <span class="citation">[<a href="#ref-wang_direct_1996" role="doc-biblioref">161</a>]</span>. Furthermore, stopped-flow kinetic studies show that the first observable species is enzyme-bound nitric oxide at the oxidised <em>d</em><span class="math inline">\(_1\)</span> haem, with the <em>c</em> haem reduced <span class="citation">[<a href="#ref-silvestrini_reaction_1990" role="doc-biblioref">162</a>]</span>. It is notable that in this assay system, a slow intramolecular electron transfer from the <em>c</em> haem to the <em>d</em><span class="math inline">\(_1\)</span> haem occurs, giving fully-reduced NO-bound enzyme as the final product. Bound NO has also been observed by EPR <span class="citation">[<a href="#ref-legall_purification_1979" role="doc-biblioref">163</a>]</span>. This implies that the protonation/dehydration step is very fast and is followed by rapid oxidation of the <em>d</em><span class="math inline">\(_1\)</span> haem. The <em>c</em> haem is also re-reduced rapidly by a second electron; this rate is increased by the presence of NO at the <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-silvestrini_reaction_1990" role="doc-biblioref">162</a>]</span>.</p>
<p>In <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, Tyr-25 is proposed to be the axial ligand which leaves the <em>d</em><span class="math inline">\(_1\)</span> haem on reduction. This ligand is then proposed to “swing back,” displacing the nitric oxide molecule from the oxidised <em>d</em><span class="math inline">\(_1\)</span> haem. Such a scheme would mean that NO had to be displaced before the <em>d</em><span class="math inline">\(_1\)</span> haem could be reduced by the second electron from the <em>c</em> haem (in contrast to <span class="citation">[<a href="#ref-silvestrini_reaction_1990" role="doc-biblioref">162</a>]</span>); this makes mechanistic sense as normally, Fe(II)-NO complexes are very stable, with a half-life for dissociation of the NO in the region of days <span class="citation">[<a href="#ref-traylor_why_1992" role="doc-biblioref">164</a>]</span>. This could still allow for the slow intramolecular electron transfer rate observed in some stop-flow experiments which has been assigned as the rate-limiting step <span class="citation">[<a href="#ref-silvestrini_reaction_1990" role="doc-biblioref">162</a>]</span>; the electron would have to “wait” at the <em>c</em> haem for ligand reorganisation around the haem <em>d</em><span class="math inline">\(_1\)</span> co-ordination sphere. However, the observed intramolecular transfer rate may vary according to the precise experimental method; more recent work using the technique of pulse radiolysis suggests that it may not be as slow as was previously thought <span class="citation">[<a href="#ref-kobayashi_pulse_1997" role="doc-biblioref">165</a>]</span>.</p>
<p>Although the scheme outlined above is very plausible and well-supported by evidence, there are still several issues to be resolved. One of these is the precise nature of the enzyme cycle; recently it has been shown that on reduction, the <em>c</em> haem of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> changes from bis-histidine to histidine-methionine ligation <span class="citation">[<a href="#ref-williams_time-resolved_1996" role="doc-biblioref">166</a>]</span>. Thus it may be that the fully-oxidised form of the enzyme is an artefact of purification and that during the catalytic cycle <em>in vivo</em>, the completely oxidised state is not regenerated. Another problem is relating mechanistic data concerning the enzyme from different sources to the crystal structure of the enzyme from <em>T. pantotropha</em>, particularly as mounting spectroscopic evidence indicates that axial haem ligation does seem genuinely to differ between cytochrome <em>cd</em><span class="math inline">\(_1\)</span> enzymes from different species; this point is discussed more fully in Chapter 3.</p>
</div>
</div>
<div id="genetic-analyses-of-denitrification" class="section level2" number="1.6">
<h2><span class="header-section-number">1.6</span> Genetic analyses of denitrification</h2>
<p>As outlined in the previous sections, a large number of genes involved in denitrification have been mapped and sequenced from several bacterial species. The first of these was the structural gene for nitrous oxide reductase, <em>nosZ</em>, from <em>Ps. stutzeri</em> Zobell; the gene was isolated using Tn5 transposon mutagenesis and analysis of mutants deficient in denitrifying growth <span class="citation">[<a href="#ref-zumft_isolation_1985" role="doc-biblioref">167</a>]</span>. Subsequent cloning and sequencing of the flanking DNA led to the discovery of a large cluster of over 30 denitrification genes in this organism, spanning over 30 kb <span class="citation">[<a href="#ref-braun_structural_1992" role="doc-biblioref">95</a>]</span>. Other workers have built on this approach by screening digested genomic DNA in different organisms using oligonucleotide probes for specific denitrification genes, isolating large hybridising fragments and sequencing. In this way, similar denitrification gene clusters have been identified in <em>Ps. aeruginosa</em> and <em>P. denitrificans</em>. Maps of the largest contiguous clusters as presently known are shown in Figure 1.6.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-6"></span>
<img src="figure/fig1-6.png" alt="Denitrification gene clusters in _Ps. stutzeri_ Zobell, _Ps. aeruginosa_ and _P. denitrificans_ PD1222. Loci within the clusters that contain genes for nitrite- (_nir_), nitric oxide- (_nor_) and nitrous oxide- (_nos_) reduction are indicated. Genes in these regions are labelled with single letters. Numbered genes are open reading frames that have not been designated a gene name. Homologous genes that are found in all three clusters are indicated by identical shading; the structural genes for nitrite reductase (_nirS_) and nitric oxide reductase (_norCB_) are shaded black. Note that a gene downstream of _norD_ designated _fnrD_ in _Ps. stutzeri_ Zobell is thought to be a homologue of _dnr_ (_Ps. aeruginosa_) and _nnr_ (_P. denitrificans_), Zumft and Körner (1997) [@zumft_enzyme_1997] but its sequence has not yet been reported. The figure was compiled from sequence data contained in the following GenBank accession numbers: _Ps. stutzeri_ Zobell Z69589, 250198, X56813, X53676, Z13988, S41769, X65277, Z17423, S51580, Z73914, Z28384; _Ps. aeruginosa_ D50473, D38133, D37883, X16452, D50019, D84475; _P. denitrificans_ U28078, U05002, U17435. Additional data for P. denitrificans was derived from de Boer (1996) [@de_boer_regulation_1996]." width="75%" />
<p class="caption">
Figure 1.6: Denitrification gene clusters in <em>Ps. stutzeri</em> Zobell, <em>Ps. aeruginosa</em> and <em>P. denitrificans</em> PD1222. Loci within the clusters that contain genes for nitrite- (<em>nir</em>), nitric oxide- (<em>nor</em>) and nitrous oxide- (<em>nos</em>) reduction are indicated. Genes in these regions are labelled with single letters. Numbered genes are open reading frames that have not been designated a gene name. Homologous genes that are found in all three clusters are indicated by identical shading; the structural genes for nitrite reductase (<em>nirS</em>) and nitric oxide reductase (<em>norCB</em>) are shaded black. Note that a gene downstream of <em>norD</em> designated <em>fnrD</em> in <em>Ps. stutzeri</em> Zobell is thought to be a homologue of <em>dnr</em> (<em>Ps. aeruginosa</em>) and <em>nnr</em> (<em>P. denitrificans</em>), Zumft and Körner (1997) <span class="citation">[<a href="#ref-zumft_enzyme_1997" role="doc-biblioref">168</a>]</span> but its sequence has not yet been reported. The figure was compiled from sequence data contained in the following GenBank accession numbers: <em>Ps. stutzeri</em> Zobell Z69589, 250198, X56813, X53676, Z13988, S41769, X65277, Z17423, S51580, Z73914, Z28384; <em>Ps. aeruginosa</em> D50473, D38133, D37883, X16452, D50019, D84475; <em>P. denitrificans</em> U28078, U05002, U17435. Additional data for P. denitrificans was derived from de Boer (1996) <span class="citation">[<a href="#ref-de_boer_regulation_1996" role="doc-biblioref">169</a>]</span>.
</p>
</div>
<p>Knowledge of these gene sequences has allowed analysis of the putative gene products using a range of methods. Some of these are theoretical; for instance the sequence can be analysed for known motifs (such as the CXXCH <em>c</em> haem binding motif), or subjected to computer-based methods which may predict properties such as sub-cellular location, the presence of a periplasmic targeting sequence or the number of transmembrane helices in a membrane protein. Database searching may also reveal possible homologues whose function is already known. All of these methods have been applied to the genes of denitrification.</p>
<p>In general, experimental verification of gene function lags behind the sequencing effort. However, some progress has been made in this area. The commonest approach is to inactivate the gene of interest, usually by inserting an antibiotic resistance cassette into the gene by homologous recombination, and then to look for phenotypic effects under denitrifying growth conditions. This approach has the disadvantage that it may have polar effects on genes downstream of the target gene, but such effects can normally be recognised by complementation experiments. Using the methods outlined above, a number of similarities in the denitrification gene clusters of <em>P. denitrificans</em>, <em>Ps. stutzeri</em> and <em>Ps. aeruginosa</em> are apparent, that can be summarised as follows.</p>
<ol style="list-style-type: lower-roman">
<li><p>All three clusters contain the structural genes for nitric oxide reductase, <em>norCB</em>, and nitrite reductase, <em>nirS</em>, quite close to each other. In addition, the larger <em>Ps. stutzeri</em> cluster contains the nitrous oxide reductase gene, <em>nosZ</em>, and associated genes <em>nosDFY</em>, which are thought to be involved in biosynthesis of the enzyme. The <em>nosZ</em> gene has also been sequenced in <em>P. denitrificans</em> and appears to be associated with other <em>nos</em> genes as in <em>Ps. stutzeri</em>.</p></li>
<li><p>Each organism contains a set of homologous genes that tend to be associated with the structural gene for a denitrification enzyme and appear to be involved with its biosynthesis. In <em>P. denitrificans</em>, the genes downstream of <em>nirS</em>, <em>nirECFD</em>, have all been inactivated by insertional mutagenesis and this results in a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> that lacks <em>d</em><span class="math inline">\(_1\)</span> haem; hence these genes are assigned a role in haem <em>d</em><span class="math inline">\(_1\)</span> biosynthesis <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>. Genes of this type that are conserved between the three clusters include <em>nirF</em> (a putative “scaffold” protein for haem <em>d</em><span class="math inline">\(_1\)</span> assembly), <em>nirC</em> (a small c-type cytochrome) and <em>nirE</em> (a S-adenosylmethionine uroporphyrinogen (III) C-methyl transferase).</p></li>
<li><p>A third class of genes encodes putative electron transport proteins and these tend to be less highly conserved between species. Examples include <em>nirM</em>, which encodes the Class I cytochrome <em>c</em><span class="math inline">\(_{551}\)</span> in <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-arai_cloning_1990" role="doc-biblioref">170</a>]</span> and <em>nirT</em>, encoding a membrane-bound tetrahaem cytochrome <em>c</em> in <em>Ps. stutzeri</em> <span class="citation">[<a href="#ref-jungst_nirstbm_1991" role="doc-biblioref">138</a>]</span>. The latter protein is not found in the other denitrification clusters, but has a homologue, <em>napC</em>, in the periplasmic nitrate reductase operon of <em>T. pantotropha</em>.</p></li>
<li><p>Each cluster contains several genes that appear to be involved in the regulation of denitrification. Predominant amongst these are members of the FNR family of proteins, discussed more fully in Section 1.7.1. Binding sites for these proteins are also detectable upstream of several denitrification operons. In addition, two other regulatory proteins are of note. One of these is <em>nirQ</em> <span class="citation">[<a href="#ref-jungst_interdependence_1992" role="doc-biblioref">171</a>]</span> (designated <em>norQ</em> in P. denitrificans <span class="citation">[<a href="#ref-de_boer_mutational_1996" role="doc-biblioref">172</a>]</span>). Inactivation of <em>nirQ</em> abolishes nitrite and nitric oxide reduction <em>in vivo</em> though the effects are complex; in <em>Ps. stutzeri</em> nitric oxide reductase becomes expressed to a two-fold higher degree and nitrite reductase retains <em>in vitro</em> activity. NirQ has some sequence similarity to NtrC, part of a two-component regulatory system which is linked to RNA polymerase containing the alternative sigma factor <span class="math inline">\(\sigma^{54}\)</span>. The other protein is <em>nosR</em> <span class="citation">[<a href="#ref-cuypers_nosr_1992" role="doc-biblioref">173</a>]</span> (with a second homologue, <em>nirI</em> in <em>P. denitrificans</em>, de Boer <em>et al</em>. 1994). This is predicted to be a membrane-bound protein and may possess an iron-sulphur cluster. Inactivation of <em>nosR</em> in <em>Ps. stutzeri</em> results in the loss of nitrous oxide reductase and in a <em>P. denitrificans</em> <em>nirI</em> mutant, the <em>nirS</em> promoter is inactivated and cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is absent (de Boer, 1996).</p></li>
<li><p>Finally, there are a number of genes to which no function can be assigned, or that do not appear to be involved in denitrification. One such example is <em>nirJ</em> from <em>P. denitrificans</em>, insertional mutagenesis of which has no effect on anaerobic growth <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>. Other examples include some recently sequenced open reading frames in <em>Ps. stutzeri</em>, such as <em>orf247</em> (a short-chain alcohol dehydrogenase) and <em>orf57</em>, which has homology to a hypothetical <em>E. coli</em> protein, YEBC <span class="citation">[<a href="#ref-glockner_sequence_1996" role="doc-biblioref">174</a>]</span>.</p></li>
</ol>
<p>The main feature of the denitrification gene clusters identified to date is that they contain homologous genes, but the gene order and content of the clusters are not conserved. For instance the <em>nor</em> genes are upstream of the <em>nir</em> operon in <em>Ps. aeruginosa</em> and <em>P. denitrificans</em>, but are downstream in <em>Ps. stutzeri</em>. Similar genes are also often transcribed in different directions amongst the three clusters. This has led to the description of a “mosaic” gene organisation in these clusters <span class="citation">[<a href="#ref-zumft_enzyme_1997" role="doc-biblioref">168</a>]</span> and supports a recent proposal that gene order is not conserved in evolution <span class="citation">[<a href="#ref-mushegian_gene_1996" role="doc-biblioref">175</a>]</span>. A final feature of note is that genes for the membrane-bound and periplasmic nitrate reductases, <em>nar</em> and <em>nap</em> respectively, are not found within the large denitrification gene clusters described above.</p>
</div>
<div id="regulation-of-denitrification" class="section level2" number="1.7">
<h2><span class="header-section-number">1.7</span> Regulation of denitrification</h2>
<p>Respiratory N-oxide reduction is an anaerobic process and its regulation might therefore be expected to be under the control of both oxygen tension and ambient N-oxide concentration. Both of these aspects have been investigated, particularly in <em>E. coli</em> due to its genetic amenability, though similar systems are becoming apparent in the denitrifying bacteria. It is now clear that the control systems involved act largely at the level of transcription. Several systems comprising DNA-binding transcriptional activators (or repressors), and the sequence motifs to which they bind, are now understood in some detail and are described in the following sections.</p>
<div id="anaerobic-response-regulators-the-fnr-family" class="section level3" number="1.7.1">
<h3><span class="header-section-number">1.7.1</span> Anaerobic response regulators: the FNR family</h3>
<p>The anaerobic regulator FNR was first identified in mutants of <em>E. coli</em>, which were deficient in anaerobic growth on fumarate and nitrite <span class="citation">[<a href="#ref-lambden_mutants_1976" role="doc-biblioref">176</a>]</span>. Complementation analysis of these mutants revealed an open reading frame encoding the FNR protein. The protein has a subunit mass of around 28 kDa and shows high homology to the cyclic AMP receptor protein <span class="citation">[<a href="#ref-shaw_homology_1983" role="doc-biblioref">177</a>]</span>. The most highly conserved regions are a DNA-binding helix-turn-helix motif, an N-terminal <span class="math inline">\(\beta\)</span>-roll and an RNA polymerase contact site. However, the FNR protein contains an N-terminal extension with four cysteine residues, and a further internal cysteine at position 122. The function of the cysteines is to bind iron; the activity of FNR can be correlated to its iron content, which varies with the method of purification, and can be reconstituted to some degree <span class="citation">[<a href="#ref-green_reconstitution_1996" role="doc-biblioref">178</a>]</span>. More recently, it has been suggested that the N-terminal region forms an iron-sulphur cluster, which is labile in the presence of oxygen <span class="citation">[<a href="#ref-khoroshilova_association_1995" role="doc-biblioref">179</a>]</span>.</p>
<p>FNR is a transcriptional regulator which can act under anaerobic conditions in either a positive or a negative fashion, depending on the target gene. It has been suggested that FNR senses oxygen directly, possibly through a different binding affinity for Fe(II) and Fe(III) <span class="citation">[<a href="#ref-becker_o2_1996" role="doc-biblioref">180</a>]</span>, but FNR-dependent promoter activity can also be stimulated by varying the redox potential in the cell <span class="citation">[<a href="#ref-unden_effect_1990" role="doc-biblioref">181</a>]</span>, suggesting it may react more indirectly to the redox state. However, currently the consensus of opinion supports the idea that the inactive form of FNR is generated as a result of the destruction of its iron-sulphur cluster in the presence of oxygen <span class="citation">[<a href="#ref-khoroshilova_association_1995" role="doc-biblioref">179</a>]</span>. Under anaerobic conditions, FNR is believed to dimerise and bind to a target sequence upstream of the FNR-dependent gene. Many such gene clusters have been identified in <em>E. coli</em> and other bacteria; the consensus target sequence (or “FNR box”) is an inverted repeat, TTGATN<span class="math inline">\(_4\)</span>ATCAA and in <em>E. coli</em>, this is generally centred at about -41.5 nucleotides relative to the transcription start point <span class="citation">[<a href="#ref-walker_promoter_1991" role="doc-biblioref">182</a>]</span> in genes that are activated by FNR. DNA-binding by FNR can then enhance transcription from the downstream promoter by contacting RNA polymerase. FNR can also negatively regulate transcription by binding at a different position and blocking RNA polymerase. The <em>fnr</em> gene is itself negatively auto-regulated in this way <span class="citation">[<a href="#ref-spiro_regulation_1987" role="doc-biblioref">183</a>]</span>. Binding to the FNR box has been demonstrated by a number of methods, including DNase I footprinting <span class="citation">[<a href="#ref-sharrocks_fnr_1991" role="doc-biblioref">184</a>]</span> and electrophoretic mobility shift assays <span class="citation">[<a href="#ref-ziegelhoffer_vitro_1995" role="doc-biblioref">185</a>]</span>. Having identified FNR-dependent promoters, reporter gene fusions have been employed to measure induction or repression at the promoter under different growth conditions <span class="citation">[<a href="#ref-park_regulation_1995" role="doc-biblioref">186</a>]</span>.</p>
<p>Site-directed mutagenesis has been used extensively to determine both the key amino acids in the FNR protein and the nucleotides at its binding site. This work has identified the conserved regions in the N-terminal cysteines of FNR, the RNA polymerase contact site and the DNA-binding domain. Mutations in residues 81-87 of the <em>E. coli</em> FNR protein can give a protein which is competent for DNA binding but not transcriptional activation, establishing the concept of a positive control “patch” which interacts with RNA polymerase <span class="citation">[<a href="#ref-williams_role_1991" role="doc-biblioref">187</a>]</span>. In this region, Gly-85 and Gln-88 are invariant but residues 86 and 87 vary in other members of the FNR family (see below). Site-directed mutagenesis has also established the residues thought to be critical in the DNA-binding domain; this is the most highly conserved region of the FNR family <span class="citation">[<a href="#ref-spiro_fnr_1994" role="doc-biblioref">188</a>]</span>. A number of FNR homologues have now been identified that fall into several distinct groups (see below) but intriguingly, the consensus FNR box appears to be very similar regardless of which FNR homologue it binds. There may be some promiscuous binding between FNR proteins and their target sequences, but on the whole the homologues do not appear to substitute for one another <em>in vivo</em>, suggesting rather subtle differences in the specificity of the different FNR-like proteins. However, only two amino acid substitutions in FNR allows binding to a CRP consensus site (and <em>vice-versa</em>, <span class="citation">[<a href="#ref-spiro_interconversion_1990" role="doc-biblioref">189</a>]</span>), and only three base changes in the FNR binding site allow binding by the CRP protein <span class="citation">[<a href="#ref-sawers_transcriptional_1997" role="doc-biblioref">190</a>]</span>.</p>
<p>Much of the detailed biochemical and genetic characterisation of FNR has been carried out in <em>E. coli</em>, the organism in which it was first identified. More recently, a number of FNR homologues have been identified in several other species of bacteria and the FNR binding consensus has been found upstream of a large number of open reading frames. Many of these FNR homologues have been cloned and sequenced. Phylogenetic analysis of the sequences has revealed a superfamily of proteins (Figure 1.7).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1-7"></span>
<img src="figure/fig1-7.png" alt="Unrooted phylogenetic tree of the FNR family of transcriptional activator proteins. Protein sequences that showed a high similarity to the FNR protein from _E. coli_ were retrieved from the GenBank database and aligned using the program Clustal W (Thompson _et al_. 1994 [@thompson_clustal_1994]). The phylogenetic tree was constructed by the method of neighbour-joining (Saitou and Nei, 1987 [@saitou_neighbor-joining_1987]) and analysed by bootstrapping (Felsenstein, 1985 [@felsenstein_confidence_1985]) using 1000 trials. The final tree was displayed using the program Treeview (Page, 1996 [@page_treeview_1996]). More details of these methods are given in Chapter 3. The tree shows that the FNR family falls into three main groups (van Spanning _et al_., 1997 [@van_spanning_fnrp_1997]): (A) the FNR subgroup, (B) the FixK subgroup and (C) the NNR subgroup. The FixK subgroup can be subdivided further into two groups, depending on the presence of an N-terminal cysteine cluster (van Spanning _et al_., 1997 [@van_spanning_fnrp_1997]). The NtcA proteins (Frías _et al_. 1993 [@frias_general_1993]) also cluster together within the NNR subgroup. Abbreviations: Syn1, _Synechococcus_ sp. WH7803, Psa, _Pseudomonas aeruginosa_, Ec, _Escherichia coli_, Sp, _Shewanella putrefaciens_, Bp, _Bordetella pertussis_, Av, _Azotobacter vinelandii_, Rh, _Rhizobium_ sp. IC3342, Ap, _Actinobacillus pleuropneumoniae_, Bj, _Bradyrhizobium japonicum_, Rp, _Rhodopseudomonas palustris_, Pd, _Paracoccus denitrificans_, St, _Salmonella typhimurium_, Rl, _Rhizobium leguminosarum_, Hi, _Haemophilus influenzae_, Rs, _Rhodobacter sphaeroides_, Tf, _Thiobacillus ferrooxidans_, Re, _Rhizobium etli_, Cy, _Cyanothece_ sp. ATCC 51142, Ac, _Azorhizobium caulinodans_, Syn2, _Synechococcus_ sp. PCC 7942, Syn3, _Synechocystis_ sp. PCC 6803, An, _Anabaena_ sp. PCC 7120, Rm, _Rhizobium meliloti_, Pss, _Pseudomonas stutzeri_ Zobell, Bs, _Bacillus subtilis_." width="50%" />
<p class="caption">
Figure 1.7: Unrooted phylogenetic tree of the FNR family of transcriptional activator proteins. Protein sequences that showed a high similarity to the FNR protein from <em>E. coli</em> were retrieved from the GenBank database and aligned using the program Clustal W (Thompson <em>et al</em>. 1994 <span class="citation">[<a href="#ref-thompson_clustal_1994" role="doc-biblioref">191</a>]</span>). The phylogenetic tree was constructed by the method of neighbour-joining (Saitou and Nei, 1987 <span class="citation">[<a href="#ref-saitou_neighbor-joining_1987" role="doc-biblioref">192</a>]</span>) and analysed by bootstrapping (Felsenstein, 1985 <span class="citation">[<a href="#ref-felsenstein_confidence_1985" role="doc-biblioref">193</a>]</span>) using 1000 trials. The final tree was displayed using the program Treeview (Page, 1996 <span class="citation">[<a href="#ref-page_treeview_1996" role="doc-biblioref">194</a>]</span>). More details of these methods are given in Chapter 3. The tree shows that the FNR family falls into three main groups (van Spanning <em>et al</em>., 1997 <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">195</a>]</span>): (A) the FNR subgroup, (B) the FixK subgroup and (C) the NNR subgroup. The FixK subgroup can be subdivided further into two groups, depending on the presence of an N-terminal cysteine cluster (van Spanning <em>et al</em>., 1997 <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">195</a>]</span>). The NtcA proteins (Frías <em>et al</em>. 1993 <span class="citation">[<a href="#ref-frias_general_1993" role="doc-biblioref">196</a>]</span>) also cluster together within the NNR subgroup. Abbreviations: Syn1, <em>Synechococcus</em> sp. WH7803, Psa, <em>Pseudomonas aeruginosa</em>, Ec, <em>Escherichia coli</em>, Sp, <em>Shewanella putrefaciens</em>, Bp, <em>Bordetella pertussis</em>, Av, <em>Azotobacter vinelandii</em>, Rh, <em>Rhizobium</em> sp. IC3342, Ap, <em>Actinobacillus pleuropneumoniae</em>, Bj, <em>Bradyrhizobium japonicum</em>, Rp, <em>Rhodopseudomonas palustris</em>, Pd, <em>Paracoccus denitrificans</em>, St, <em>Salmonella typhimurium</em>, Rl, <em>Rhizobium leguminosarum</em>, Hi, <em>Haemophilus influenzae</em>, Rs, <em>Rhodobacter sphaeroides</em>, Tf, <em>Thiobacillus ferrooxidans</em>, Re, <em>Rhizobium etli</em>, Cy, <em>Cyanothece</em> sp. ATCC 51142, Ac, <em>Azorhizobium caulinodans</em>, Syn2, <em>Synechococcus</em> sp. PCC 7942, Syn3, <em>Synechocystis</em> sp. PCC 6803, An, <em>Anabaena</em> sp. PCC 7120, Rm, <em>Rhizobium meliloti</em>, Pss, <em>Pseudomonas stutzeri</em> Zobell, Bs, <em>Bacillus subtilis</em>.
</p>
</div>
<p>Examples include NNR from <em>P. denitrificans</em> <span class="citation">[<a href="#ref-van_spanning_nitrite_1995" role="doc-biblioref">197</a>]</span>, ANR and DNR from <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-ye_anaerobic_1995" role="doc-biblioref">198</a>,<a href="#ref-arai_expression_1995" role="doc-biblioref">199</a>]</span> and FixK from <em>Rhizobium meliloti</em> <span class="citation">[<a href="#ref-batut_fixk_1989" role="doc-biblioref">200</a>]</span>. FNR homologues have also been identified in Gram-positive bacteria <span class="citation">[<a href="#ref-irvine_lactobacillus_1993" role="doc-biblioref">201</a>]</span> and the FNR box has been identified upstream of the CYP 55 gene in the fungus <em>F. oxysporum</em> <span class="citation">[<a href="#ref-tomura_nitric_1994" role="doc-biblioref">202</a>]</span>.</p>
<p>Figure 1.7 shows that the FNR family falls into three main groups, which van Spanning <em>et al</em>. <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">195</a>]</span> have labelled types A, B and C, with a further subdivision of group B into B1 and B2. This grouping reflects the different functional domains within the proteins. Group A are the prototype FNR proteins containing the N-terminal cysteine cluster, RNA polymerase contact site, dimerisation domain and DNA-binding helix-turn-helix motif. Group B1 also contains all of these domains, but in groups B2 and C (the FixK and NNR subfamilies), the cysteine cluster is not present and there are differences in the putative RNA polymerase contact site. The lack of an iron-binding cysteine cluster is intriguing, as it is this that is proposed to respond to anaerobiosis in the FNR proteins. FixK and NNR clearly do respond to anaerobiosis in some way, as shown by strains carrying mutations in these genes <span class="citation">[<a href="#ref-anthamatten_characterization_1992" role="doc-biblioref">203</a>,<a href="#ref-tosques_cloning_1996" role="doc-biblioref">204</a>]</span>, but the mechanism of the response is at present not clear. The differences in the RNA polymerase contact site have led some groups to suggest that the promoter regions of genes controlled in this manner bind an alternative sigma factor on RNA polymerase <span class="citation">[<a href="#ref-winteler_homologous_1996" role="doc-biblioref">205</a>]</span>. This is discussed further in Chapter 7. In many cases different FNR homologues have been found in the same organism; for instance in <em>P. denitrificans</em>, FnrP is thought to regulate transcription of membrane-bound nitrate reductase, cytochrome <em>c</em> peroxidase and <em>cbb</em><span class="math inline">\(_3\)</span> cytochrome oxidase, whilst NNR controls the nitrite and nitric oxide reductases <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">195</a>]</span>. Similarly in <em>Ps. aeruginosa</em>, the homologues ANR and DNR both seem to control denitrification genes <span class="citation">[<a href="#ref-arai_expression_1995" role="doc-biblioref">199</a>]</span>, whilst in <em>Ps. stutzeri</em>, mutation of FnrA (an ANR homologue) has no effect on denitrifying growth, suggesting the presence of a second FNR-like protein <span class="citation">[<a href="#ref-cuypers_anaerobic_1993" role="doc-biblioref">206</a>]</span>.</p>
</div>
<div id="the-arcab-system" class="section level3" number="1.7.2">
<h3><span class="header-section-number">1.7.2</span> The ArcA/B system</h3>
<p>In <em>E. coli</em> a second system for oxygen-regulated transcriptional control is known, the ArcA/B system. This is a two-component sensor/regulator signal transduction system, of a type commonly found where response to an environmental signal is required <span class="citation">[<a href="#ref-stock_protein_1989" role="doc-biblioref">207</a>]</span>. ArcB is a transmembrane protein, with a periplasmic N-terminal sensor domain and a cytoplasmic C-terminal transmitter domain <span class="citation">[<a href="#ref-iuchi_arcb_1990" role="doc-biblioref">208</a>]</span>. ArcA is a cytoplasmic protein, with an N-terminal receiver domain and a C-terminal DNA-binding domain <span class="citation">[<a href="#ref-iuchi_purification_1992" role="doc-biblioref">209</a>]</span>.</p>
<p>The action of the ArcA/B system has been recently reviewed in Unden <em>et al</em>. <span class="citation">[<a href="#ref-unden_oxygen_1994" role="doc-biblioref">210</a>]</span>. Under low oxygen tension, the transmembrane protein ArcB responds indirectly to a signal at the cytoplasmic membrane (possibly the UQH<span class="math inline">\(_2\)</span>/UQ ratio) and is autophosphorylated at His-292 by ATP. The phosphate is transferred intramolecularly to Asp-576, followed by a second phosphorylation at His-292. This double-phosphorylated species is then competent to phosphorylate the cytoplasmic ArcA protein at an N-terminal Asp residue. Once phosphorylated, ArcA becomes active and can bind DNA upstream of the genes to be regulated. ArcA-P acts for most genes as a repressor, blocking transcription under anaerobic conditions. Genes affected in this way include those encoding the citric acid cycle enzymes. ArcA-P can also act as an activator; the <em>E. coli</em> <em>cydAB</em> operon encoding cytochrome <em>d</em>
oxidase possesses two ArcA-P binding sites upstream of its promoter, which have been identified by DNase I footprinting <span class="citation">[<a href="#ref-lynch_transcriptional_1996" role="doc-biblioref">211</a>]</span>. This work has lead to the identification of a consensus ArcA-P binding site. LacZ reporter gene fusions have been used to show that the Arc A/B system exerts its effects in the range 10-20% air saturation, whereas Fnr functions in the 0-10% range <span class="citation">[<a href="#ref-tseng_effect_1996" role="doc-biblioref">212</a>]</span>. Hence in <em>E. coli</em> the two systems co-ordinate the control of respiratory gene expression to ensure optimal use of oxygen as environmental conditions change.</p>
<p>Unlike FNR the ArcA/B system is only well characterised in <em>E. coli</em> and to date, none of the denitrification genes have been shown to be under ArcA/B control. However, the system is important as a complementary response to the FNR system and it is possible that something similar will be found in denitrifying bacteria.</p>
</div>
<div id="sensors-of-nitrate-and-nitrite" class="section level3" number="1.7.3">
<h3><span class="header-section-number">1.7.3</span> Sensors of nitrate and nitrite</h3>
<p>In <em>E. coli</em> nitrate and nitrite are sensed by two, interacting, two-component regulatory systems known as NarX/L and NarQ/P <span class="citation">[<a href="#ref-stewart_nitrate_1993" role="doc-biblioref">213</a>,<a href="#ref-rabin_dual_1993" role="doc-biblioref">214</a>]</span>. These function in a similar fashion to the ArcA/B system described earlier. NarX and NarQ are transmembrane proteins, possessing a periplasmic domain which can sense nitrate or nitrite. A region within this domain termed the “P-box” has been shown by site-directed mutagenesis to be involved in nitrate/nitrite sensing <span class="citation">[<a href="#ref-cavicchioli_role_1996" role="doc-biblioref">215</a>]</span>. This results in an autophosphorylation at the C-terminal cytoplasmic domain, which in turn goes on to phosphorylate the cytoplasmic component, NarL or NarP. Activated NarL/P can then go on to bind to a target DNA sequence and enhance or repress transcription of nitrate/nitrite dependent genes. A consensus site, the “NarL heptamer,” has been proposed with the recognition sequence TACYNKT (where Y=C or T, N=any nucleotide and K=A or C) <span class="citation">[<a href="#ref-tyson_nitrite_1994" role="doc-biblioref">216</a>]</span>. The NarL heptamer appears as a single heptamer, a direct or inverted repeat, in either orientation and apparently between +20 and -220 nucleotides from the transcription start site. The interaction between the homologous NarX/L and NarQ/P
systems is quite complex; essentially it seems that nitrate stimulates NarX/Q to phosphorylate NarL/P, nitrite stimulates NarQ to phosphorylate both NarL and NarP and finally that NarX can act to dephosphorylate NarL-phosphate in the absence of nitrate <span class="citation">[<a href="#ref-rabin_dual_1993" role="doc-biblioref">214</a>]</span>.</p>
<p>No analogous system has yet been identified in the denitrifying bacteria, but all four N oxides have been reported to increase the expression of at least one reductase, particularly in <em>Ps. stutzeri</em> and <em>Ps. aeruginosa</em>. In the latter organism, expression of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase appears to rely more on the presence of nitrate or nitrite than on anaerobiosis <span class="citation">[<a href="#ref-zennaro_nitrite_1993" role="doc-biblioref">217</a>]</span>. Levels of nitrous oxide reductase in <em>Ps. stutzeri</em> are also regulated by nitrate, nitrous oxide and anaerobiosis <span class="citation">[<a href="#ref-korner_expression_1989" role="doc-biblioref">218</a>]</span>. There also appears to be a regulatory link between the nitrite and nitric oxide reductases. In <em>Ps. stutzeri</em>, mutation in nitric oxide reductase reduces nitrite reductase activity, though its expression level is similar. The converse is true for mutants in nitrite reductase <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>. This implies that nitric oxide may induce its reductase and that there is a functional coupling between these two enzymes.</p>
<p>It seems likely that control of denitrification will prove to be a complex process, with several hierarchical levels that act to “fine-tune” the response to changing environmental conditions. This is already becoming apparent in the systems described previously for <em>E. coli</em>. One example of a complex promoter system is the <em>focApfl</em> operon, encoding pyruvate formate lyase and the <em>focA</em> gene product. This system is regulated by both ArcA and FNR, which can act at several different promoters in the operon. One promoter, P6A, overlaps an FNR-dependent promoter, modulating its activity by competition for FNR- and RNA polymerase-binding <span class="citation">[<a href="#ref-kaiser_overlapping_1997" role="doc-biblioref">219</a>]</span>. There are also several examples of promoters that are regulated by the FNR, Arc A/B and NarX/L components <span class="citation">[<a href="#ref-bongaerts_transcriptional_1995" role="doc-biblioref">220</a>]</span>.</p>
<p>Although the regulation of denitrification is known to occur primarily at the level of transcription, transcriptional analysis of denitrification genes has been reported in relatively few cases. Northern blot analysis has been reported for the <em>norBC</em> genes of <em>Ps. stutzeri</em> which are co-transcribed <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>, and for the <em>nosZ</em> gene which is apparently under the control of 6 alternate promoters <span class="citation">[<a href="#ref-cuypers_multiple_1995" role="doc-biblioref">221</a>]</span>. Also in <em>Ps. stutzeri</em>, transcription of the <em>nirS</em> gene for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase occurs as two transcripts, <em>nirS</em> and <em>nirST</em>, and is controlled by anaerobiosis and nitrate <span class="citation">[<a href="#ref-zumft_diversity_1995" role="doc-biblioref">222</a>]</span>. The <em>fnrA</em> gene from this organism is also known to be transcribed as a single 0.85 kb species <span class="citation">[<a href="#ref-heiss_formation_1989" role="doc-biblioref">76</a>]</span>. Induction of the mRNAs for nitrate, nitrite and nitrous oxide reductases in <em>P. denitrificans</em> during the switch to anaerobiosis has also been reported <span class="citation">[<a href="#ref-baumann_dynamics_1996" role="doc-biblioref">56</a>]</span>. More commonly, transcriptional and translational features are inferred from inspection of the gene sequence; hence in <em>P. denitrificans</em>, the <em>nirE</em> and <em>nirC</em> genes have overlapping stop and start codons <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>, implying that they are co-translated, whilst stem-loop analysis of the <em>nir</em> operon in <em>Ps. aeruginosa</em> has been used to suggest that the 11 genes <em>nirSMCFDLGHJEN</em> are transcribed on a polycistronic mRNA <span class="citation">[<a href="#ref-kawasaki_gene_1997" role="doc-biblioref">223</a>]</span>.</p>
</div>
</div>
<div id="aims-of-the-present-work" class="section level2" number="1.8">
<h2><span class="header-section-number">1.8</span> Aims of the present work</h2>
<p>At the outset of this project the X-ray crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase from <em>T. pantotropha</em> had been determined from diffraction data and the <em>P. denitrificans</em> PD1222 sequence, with no complete protein sequence from the <em>T. pantotropha</em> protein. The initial aim of the project was to clone the structural gene, <em>nirS</em>, from <em>T. pantotropha</em>, so as to obtain the polypeptide sequence and refine this crystal structure. Mapping and sequencing of the <em>nirS</em> gene and surrounding regions was also essential for subsequent DNA manipulations. It was also of interest to determine whether the organisation of the <em>nir</em> cluster is similar to that of the closely-related <em>P. denitrificans</em>.</p>
<p>Having cloned the <em>nirS</em> gene, the next aim would be to investigate several expression systems for the enzyme. This work would involve cloning the gene into a suitableexpression vector and trying to transform a number of heterologous hosts, to determine which ones expressed the protein most efficiently. As the <em>d</em><span class="math inline">\(_1\)</span> haem group is unique to cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, it would also be advantageous to use heterologous hosts which are able to synthesise this group, so as to express the active holoprotein. Development of a suitable expression system is an essential prerequisite for future site-directed mutagenesis studies.</p>
<p>Finally, transcriptional regulation of the <em>nirS</em> and associated genes is of interest. Knowledge of this process may help to understand the expression of the protein, so a third aim of the project was to investigate the mRNA transcribed from the <em>nirS</em> gene, using methods such as Northern blotting and determination of the transcription start site. This work was expected to help to define novel promoter regions in the denitrification genes of <em>P. denitrificans</em> and <em>T. pantotropha</em>, and to provide insight into the transcriptional regulation of denitrification.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-fry_nitrogen_1992" class="csl-entry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Robertson LA, Kuenen JG. Nitrogen removal from water and waste. In: Fry S, editor. Microbial control of pollution. University of Cardiff: Society for General Microbiology; 1992. pp. 227–267. Available: <a href="https://repository.tudelft.nl/islandora/object/uuid%3A87a571e2-a46f-4472-92e0-11be9a6fce01">https://repository.tudelft.nl/islandora/object/uuid%3A87a571e2-a46f-4472-92e0-11be9a6fce01</a></div>
</div>
<div id="ref-gayon_recherches_1886" class="csl-entry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Gayon U, Dupetit G. Recherches sur la reduction des nitrates par les infiniment petits. Mem Soc Sci Phys Nat Bordeaux. 1886;2: 201–307. Available: <a href="https://gallica.bnf.fr/ark:/12148/bpt6k9366473/f12.item">https://gallica.bnf.fr/ark:/12148/bpt6k9366473/f12.item</a></div>
</div>
<div id="ref-rasmussen_atmospheric_1986" class="csl-entry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Rasmussen RA, Khalil MA. Atmospheric trace gases: Trends and distributions over the last decade. Science (New York, NY). 1986;232: 1623–1624. doi:<a href="https://doi.org/10.1126/science.232.4758.1623">10.1126/science.232.4758.1623</a></div>
</div>
<div id="ref-hauck_nitrogen_1981" class="csl-entry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Hauck RD. <span>NITROGEN</span> <span>FERTILIZER</span> <span>EFFECTS</span> <span>ON</span> <span>NITROGEN</span> <span>CYCLE</span> <span>PROCESSES</span>. Ecological Bulletins. 1981; 551–562. Available: <a href="https://www.jstor.org/stable/45128688">https://www.jstor.org/stable/45128688</a></div>
</div>
<div id="ref-hutchins_biodegradation_1991" class="csl-entry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Hutchins SR. Biodegradation of monoaromatic hydrocarbons by aquifer microorganisms using oxygen, nitrate, or nitrous oxide as the terminal electron acceptor. Applied and Environmental Microbiology. 1991;57: 2403–2407. doi:<a href="https://doi.org/10.1128/AEM.57.8.2403-2407.1991">10.1128/AEM.57.8.2403-2407.1991</a></div>
</div>
<div id="ref-olson_fertilizers_1977" class="csl-entry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Olson RA. Fertilizers for food production vs energy needs and environmental quality. Ecotoxicology and Environmental Safety. 1977;1: 311–326. doi:<a href="https://doi.org/10.1016/0147-6513(77)90023-9">10.1016/0147-6513(77)90023-9</a></div>
</div>
<div id="ref-levallois_contamination_1994" class="csl-entry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Levallois P, Phaneuf D. [<span>Contamination</span> of drinking water by nitrates: Analysis of health risks]. Canadian Journal of Public Health = Revue Canadienne De Sante Publique. 1994;85: 192–196. </div>
</div>
<div id="ref-calmels_characterization_1996" class="csl-entry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Calmels S, Ohshima H, Henry Y, Bartsch H. Characterization of bacterial cytochrome cd(1)-nitrite reductase as one enzyme responsible for catalysis of nitrosation of secondary amines. Carcinogenesis. 1996;17: 533–536. doi:<a href="https://doi.org/10.1093/carcin/17.3.533">10.1093/carcin/17.3.533</a></div>
</div>
<div id="ref-bredt_nitric_1994" class="csl-entry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Bredt DS, Snyder SH. Nitric oxide: A physiologic messenger molecule. Annual Review of Biochemistry. 1994;63: 175–195. doi:<a href="https://doi.org/10.1146/annurev.bi.63.070194.001135">10.1146/annurev.bi.63.070194.001135</a></div>
</div>
<div id="ref-shoun_denitrification_1991" class="csl-entry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Shoun H, Tanimoto T. Denitrification by the fungus <span>Fusarium</span> oxysporum and involvement of cytochrome <span>P</span>-450 in the respiratory nitrite reduction. The Journal of Biological Chemistry. 1991;266: 11078–11082. doi:<a href="https://doi.org/10.1016/S0021-9258(18)99130-1">10.1016/S0021-9258(18)99130-1</a></div>
</div>
<div id="ref-usuda_denitrification_1995" class="csl-entry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Usuda K, Toritsuka N, Matsuo Y, Kim DH, Shoun H. Denitrification by the fungus <span>Cylindrocarpon</span> tonkinense: Anaerobic cell growth and two isozyme forms of cytochrome <span>P</span>-450nor. Applied and Environmental Microbiology. 1995;61: 883–889. doi:<a href="https://doi.org/10.1128/AEM.61.3.883-889.1995">10.1128/AEM.61.3.883-889.1995</a></div>
</div>
<div id="ref-kobayashi_denitrification_1996" class="csl-entry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Kobayashi M, Matsuo Y, Takimoto A, Suzuki S, Maruo F, Shoun H. Denitrification, a novel type of respiratory metabolism in fungal mitochondrion. The Journal of Biological Chemistry. 1996;271: 16263–16267. doi:<a href="https://doi.org/10.1074/jbc.271.27.16263">10.1074/jbc.271.27.16263</a></div>
</div>
<div id="ref-kobayashi_copper-containing_1995" class="csl-entry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Kobayashi M, Shoun H. The copper-containing dissimilatory nitrite reductase involved in the denitrifying system of the fungus <span>Fusarium</span> oxysporum. The Journal of Biological Chemistry. 1995;270: 4146–4151. doi:<a href="https://doi.org/10.1074/jbc.270.8.4146">10.1074/jbc.270.8.4146</a></div>
</div>
<div id="ref-nakahara_cytochrome_1993" class="csl-entry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Nakahara K, Tanimoto T, Hatano K, Usuda K, Shoun H. Cytochrome <span>P</span>-450 <span>55A1</span> (<span>P</span>-<span>450dNIR</span>) acts as nitric oxide reductase employing <span>NADH</span> as the direct electron donor. The Journal of Biological Chemistry. 1993;268: 8350–8355. doi:<a href="https://doi.org/10.1016/S0021-9258(18)53102-1">10.1016/S0021-9258(18)53102-1</a></div>
</div>
<div id="ref-enoch_role_1974" class="csl-entry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Enoch HG, Lester RL. The role of a novel cytochrome b-containing nitrate reductase and quinone in the in vitro reconstruction of formate-nitrate reductase activity of <span>E</span>. coli. Biochemical and Biophysical Research Communications. 1974;61: 1234–1241. doi:<a href="https://doi.org/10.1016/s0006-291x(74)80416-x">10.1016/s0006-291x(74)80416-x</a></div>
</div>
<div id="ref-berks_napedabc_1995" class="csl-entry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Berks BC, Richardson DJ, Reilly A, Willis AC, Ferguson SJ. The <span class="nocase">napEDABC</span> gene cluster encoding the periplasmic nitrate reductase system of <span>Thiosphaera</span> pantotropha. The Biochemical Journal. 1995;309 ( Pt 3): 983–992. doi:<a href="https://doi.org/10.1042/bj3090983">10.1042/bj3090983</a></div>
</div>
<div id="ref-richterich_u00008_1993" class="csl-entry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Richterich P, Lakey N, Gryan G, Jaehn L, Mintz L, Robison K, et al. U00008. Genbank. 1993. Available: <a href="https://www.ncbi.nlm.nih.gov/nuccore/u00008">https://www.ncbi.nlm.nih.gov/nuccore/u00008</a></div>
</div>
<div id="ref-godden_23_1991" class="csl-entry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Godden JW, Turley S, Teller DC, Adman ET, Liu MY, Payne WJ, et al. The 2.3 angstrom <span>X</span>-ray structure of nitrite reductase from <span>Achromobacter</span> cycloclastes. Science (New York, NY). 1991;253: 438–442. doi:<a href="https://doi.org/10.1126/science.1862344">10.1126/science.1862344</a></div>
</div>
<div id="ref-kukimoto_x-ray_1994" class="csl-entry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Kukimoto M, Nishiyama M, Murphy ME, Turley S, Adman ET, Horinouchi S, et al. X-ray structure and site-directed mutagenesis of a nitrite reductase from <span>Alcaligenes</span> faecalis <span>S</span>-6: Roles of two copper atoms in nitrite reduction. Biochemistry. 1994;33: 5246–5252. doi:<a href="https://doi.org/10.1021/bi00183a030">10.1021/bi00183a030</a></div>
</div>
<div id="ref-zumft_denitrifying_1992" class="csl-entry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Zumft W. The denitrifying prokaryotes. 2nd ed. The <span>Prokaryotes</span>. 2nd ed. Springer-Verlag; 1992. pp. 554–582. Available: <a href="https://www.springer.com/gp/book/9781475721911">https://www.springer.com/gp/book/9781475721911</a></div>
</div>
<div id="ref-john_paracoccus_1975" class="csl-entry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">John P, Whatley FR. Paracoccus denitrificans and the evolutionary origin of the mitochondrion. Nature. 1975;254: 495–498. doi:<a href="https://doi.org/10.1038/254495a0">10.1038/254495a0</a></div>
</div>
<div id="ref-trumpower_cytochrome_1990" class="csl-entry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Trumpower BL. Cytochrome bc1 complexes of microorganisms. Microbiological Reviews. 1990;54: 101–129. doi:<a href="https://doi.org/10.1128/mr.54.2.101-129.1990">10.1128/mr.54.2.101-129.1990</a></div>
</div>
<div id="ref-ballard_respiratory_1988" class="csl-entry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Ballard AL, Ferguson SJ. Respiratory nitrate reductase from <span>Paracoccus</span> denitrificans. <span>Evidence</span> for two b-type haems in the gamma subunit and properties of a water-soluble active enzyme containing alpha and beta subunits. European Journal of Biochemistry. 1988;174: 207–212. doi:<a href="https://doi.org/10.1111/j.1432-1033.1988.tb14083.x">10.1111/j.1432-1033.1988.tb14083.x</a></div>
</div>
<div id="ref-clegg_purification_1976" class="csl-entry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Clegg RA. Purification and some properties of nitrate reductase (<span>EC</span> 1.7.99.4) from <span>Escherichia</span> coli <span>K12</span>. The Biochemical Journal. 1976;153: 533–541. doi:<a href="https://doi.org/10.1042/bj1530533">10.1042/bj1530533</a></div>
</div>
<div id="ref-ballard_molecular_1987" class="csl-entry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Ballard AL, Ferguson SJ. Molecular properties of the respiratory nitrate reductase of <span>Paracoccus</span> denitrificans. Biochemical Society Transactions. 1987;15: 937–938. doi:<a href="https://doi.org/10.1042/bst0150937">10.1042/bst0150937</a></div>
</div>
<div id="ref-demoss_characterization_1981" class="csl-entry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Demoss JA, Fan TY, Scott RH. Characterization of subunit structural alterations which occur during purification of nitrate reductase from <span>Escherichia</span> coli. Archives of Biochemistry and Biophysics. 1981;206: 54–64. doi:<a href="https://doi.org/10.1016/0003-9861(81)90065-5">10.1016/0003-9861(81)90065-5</a></div>
</div>
<div id="ref-oppenheim_translational_1980" class="csl-entry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Oppenheim DS, Yanofsky C. Translational coupling during expression of the tryptophan operon of <span>Escherichia</span> coli. Genetics. 1980;95: 785–795. doi:<a href="https://doi.org/10.1093/genetics/95.4.785">10.1093/genetics/95.4.785</a></div>
</div>
<div id="ref-berks_sequence_1995" class="csl-entry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Berks BC, Page MD, Richardson DJ, Reilly A, Cavill A, Outen F, et al. Sequence analysis of subunits of the membrane-bound nitrate reductase from a denitrifying bacterium: The integral membrane subunit provides a prototype for the dihaem electron-carrying arm of a redox loop. Molecular Microbiology. 1995;15: 319–331. doi:<a href="https://doi.org/10.1111/j.1365-2958.1995.tb02246.x">10.1111/j.1365-2958.1995.tb02246.x</a></div>
</div>
<div id="ref-johnson_iron-sulfur_1985" class="csl-entry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Johnson MK, Bennett DE, Morningstar JE, Adams MW, Mortenson LE. The iron-sulfur cluster composition of <span>Escherichia</span> coli nitrate reductase. The Journal of Biological Chemistry. 1985;260: 5456–5463. doi:<a href="https://doi.org/10.1016/S0021-9258(18)89044-5">10.1016/S0021-9258(18)89044-5</a></div>
</div>
<div id="ref-wootton_enzymes_1991" class="csl-entry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Wootton JC, Nicolson RE, Mark Cock J, Walters DE, Burke JF, Doyle WA, et al. Enzymes depending on the pterin molybdenum cofactor: Sequence families, spectroscopic properties of molybdenum and possible cofactor-binding domains. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1991;1057: 157–185. doi:<a href="https://doi.org/10.1016/S0005-2728(05)80100-8">10.1016/S0005-2728(05)80100-8</a></div>
</div>
<div id="ref-berks_enzymes_1995" class="csl-entry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Berks BC, Ferguson SJ, Moir JW, Richardson DJ. Enzymes and associated electron transport systems that catalyse the respiratory reduction of nitrogen oxides and oxyanions. Biochimica Et Biophysica Acta. 1995;1232: 97–173. doi:<a href="https://doi.org/10.1016/0005-2728(95)00092-5">10.1016/0005-2728(95)00092-5</a></div>
</div>
<div id="ref-craig_reduction_1989" class="csl-entry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Craig JA, Holm RH. Reduction of nitrate to nitrite by molybdenum-mediated atom transfer: A nitrate reductase analog reaction system. Journal of the American Chemical Society. 1989;111: 2111–2115. doi:<a href="https://doi.org/10.1021/ja00188a026">10.1021/ja00188a026</a></div>
</div>
<div id="ref-cramer_molybdenum_1983" class="csl-entry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Cramer S. Molybdenum enzymes: A survey of structural information from <span>EXAFS</span> and <span>EPR</span> spectroscopy. Adv Inorg Bioinorg Mech. 1983; 259–316. </div>
</div>
<div id="ref-bellion_biological_1992" class="csl-entry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Bellion E. The <span>Biological</span> <span>Chemistry</span> of the <span>Elements</span>: <span>The</span> <span>Inorganic</span> <span>Chemistry</span> of <span>Life</span> (da <span>Silva</span>, <span>J</span>. <span>J</span>. <span>R</span>. <span>Frausto</span>; <span>Williams</span>, <span>R</span>. <span>J</span>. <span>P</span>.). Journal of Chemical Education. 1992;69: A326. doi:<a href="https://doi.org/10.1021/ed069pA326.1">10.1021/ed069pA326.1</a></div>
</div>
<div id="ref-schindelin_crystal_1996" class="csl-entry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Schindelin H, Kisker C, Hilton J, Rajagopalan KV, Rees DC. Crystal structure of <span>DMSO</span> reductase: Redox-linked changes in molybdopterin coordination. Science (New York, NY). 1996;272: 1615–1621. doi:<a href="https://doi.org/10.1126/science.272.5268.1615">10.1126/science.272.5268.1615</a></div>
</div>
<div id="ref-boyington_crystal_1997" class="csl-entry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Boyington JC, Gladyshev VN, Khangulov SV, Stadtman TC, Sun PD. Crystal structure of formate dehydrogenase <span>H</span>: Catalysis involving <span>Mo</span>, molybdopterin, selenocysteine, and an <span>Fe4S4</span> cluster. Science (New York, NY). 1997;275: 1305–1308. doi:<a href="https://doi.org/10.1126/science.275.5304.1305">10.1126/science.275.5304.1305</a></div>
</div>
<div id="ref-sodergren_nari_1988" class="csl-entry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Sodergren EJ, DeMoss JA. <span class="nocase">narI</span> region of the <span>Escherichia</span> coli nitrate reductase (nar) operon contains two genes. Journal of Bacteriology. 1988;170: 1721–1729. doi:<a href="https://doi.org/10.1128/jb.170.4.1721-1729.1988">10.1128/jb.170.4.1721-1729.1988</a></div>
</div>
<div id="ref-blasco_nitrate_1990" class="csl-entry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Blasco F, Iobbi C, Ratouchniak J, Bonnefoy V, Chippaux M. Nitrate reductases of <span>Escherichia</span> coli: Sequence of the second nitrate reductase and comparison with that encoded by the <span class="nocase">narGHJI</span> operon. Molecular &amp; general genetics: MGG. 1990;222: 104–111. doi:<a href="https://doi.org/10.1007/BF00283030">10.1007/BF00283030</a></div>
</div>
<div id="ref-nohno_narx_1989" class="csl-entry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Nohno T, Noji S, Taniguchi S, Saito T. The <span class="nocase">narX</span> and <span class="nocase">narL</span> genes encoding the nitrate-sensing regulators of <span>Escherichia</span> coli are homologous to a family of prokaryotic two-component regulatory genes. Nucleic Acids Research. 1989;17: 2947–2957. doi:<a href="https://doi.org/10.1093/nar/17.8.2947">10.1093/nar/17.8.2947</a></div>
</div>
<div id="ref-rowe_nark_1994" class="csl-entry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Rowe JJ, Ubbink-Kok T, Molenaar D, Konings WN, Driessen AJ. <span>NarK</span> is a nitrite-extrusion system involved in anaerobic nitrate respiration by <span>Escherichia</span> coli. Molecular Microbiology. 1994;12: 579–586. doi:<a href="https://doi.org/10.1111/j.1365-2958.1994.tb01044.x">10.1111/j.1365-2958.1994.tb01044.x</a></div>
</div>
<div id="ref-demoss_nark_1991" class="csl-entry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">DeMoss JA, Hsu PY. <span>NarK</span> enhances nitrate uptake and nitrite excretion in <span>Escherichia</span> coli. Journal of Bacteriology. 1991;173: 3303–3310. doi:<a href="https://doi.org/10.1128/jb.173.11.3303-3310.1991">10.1128/jb.173.11.3303-3310.1991</a></div>
</div>
<div id="ref-peakman_nucleotide_1990" class="csl-entry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Peakman T, Crouzet J, Mayaux JF, Busby S, Mohan S, Harborne N, et al. Nucleotide sequence, organisation and structural analysis of the products of genes in the <span class="nocase">nirB</span>-<span class="nocase">cysG</span> region of the <span>Escherichia</span> coli <span>K</span>-12 chromosome. European Journal of Biochemistry. 1990;191: 315–323. doi:<a href="https://doi.org/10.1111/j.1432-1033.1990.tb19125.x">10.1111/j.1432-1033.1990.tb19125.x</a></div>
</div>
<div id="ref-kajie_purification_1986" class="csl-entry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Kajie S, Anraku Y. Purification of a hexaheme cytochrome c552 from <span>Escherichia</span> coli <span>K</span> 12 and its properties as a nitrite reductase. European Journal of Biochemistry. 1986;154: 457–463. doi:<a href="https://doi.org/10.1111/j.1432-1033.1986.tb09419.x">10.1111/j.1432-1033.1986.tb09419.x</a></div>
</div>
<div id="ref-omata_structure_1995" class="csl-entry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Omata T. Structure, function and regulation of the nitrate transport system of the cyanobacterium <span>Synechococcus</span> sp. <span>PCC7942</span>. Plant &amp; Cell Physiology. 1995;36: 207–213. doi:<a href="https://doi.org/10.1093/oxfordjournals.pcp.a078751">10.1093/oxfordjournals.pcp.a078751</a></div>
</div>
<div id="ref-rodriguez_nitrate_1992" class="csl-entry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Rodríguez R, Lara C, Guerrero MG. Nitrate transport in the cyanobacterium <span>Anacystis</span> nidulans <span>R2</span>. <span>Kinetic</span> and energetic aspects. The Biochemical Journal. 1992;282 ( Pt 3): 639–643. doi:<a href="https://doi.org/10.1042/bj2820639">10.1042/bj2820639</a></div>
</div>
<div id="ref-mcewan_periplasmic_1987" class="csl-entry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">McEwan AG, Wetzstein HG, Meyer O, Jackson JB, Ferguson SJ. The periplasmic nitrate reductase of <span>Rhodobacter</span> capsulatus; purification, characterisation and distinction from a single reductase for trimethylamine-<span>N</span>-oxide, dimethylsulphoxide and chlorate. Archives of Microbiology. 1987;147: 340–345. doi:<a href="https://doi.org/10.1007/BF00406130">10.1007/BF00406130</a></div>
</div>
<div id="ref-berks_purification_1994" class="csl-entry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Berks BC, Richardson DJ, Robinson C, Reilly A, Aplin RT, Ferguson SJ. Purification and characterization of the periplasmic nitrate reductase from <span>Thiosphaera</span> pantotropha. European Journal of Biochemistry. 1994;220: 117–124. doi:<a href="https://doi.org/10.1111/j.1432-1033.1994.tb18605.x">10.1111/j.1432-1033.1994.tb18605.x</a></div>
</div>
<div id="ref-siddiqui_structure_1993" class="csl-entry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Siddiqui RA, Warnecke-Eberz U, Hengsberger A, Schneider B, Kostka S, Friedrich B. Structure and function of a periplasmic nitrate reductase in <span>Alcaligenes</span> eutrophus <span>H16</span>. Journal of Bacteriology. 1993;175: 5867–5876. doi:<a href="https://doi.org/10.1128/jb.175.18.5867-5876.1993">10.1128/jb.175.18.5867-5876.1993</a></div>
</div>
<div id="ref-reyes_isolation_1996" class="csl-entry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Reyes F, Roldán MD, Klipp W, Castillo F, Moreno-Vivián C. Isolation of periplasmic nitrate reductase genes from <span>Rhodobacter</span> sphaeroides <span>DSM</span> 158: Structural and functional differences among prokaryotic nitrate reductases. Molecular Microbiology. 1996;19: 1307–1318. doi:<a href="https://doi.org/10.1111/j.1365-2958.1996.tb02475.x">10.1111/j.1365-2958.1996.tb02475.x</a></div>
</div>
<div id="ref-carter_soil_1995" class="csl-entry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Carter JP, Hsaio YH, Spiro S, Richardson DJ. Soil and sediment bacteria capable of aerobic nitrate respiration. Applied and Environmental Microbiology. 1995;61: 2852–2858. doi:<a href="https://doi.org/10.1128/AEM.61.8.2852-2858.1995">10.1128/AEM.61.8.2852-2858.1995</a></div>
</div>
<div id="ref-breton_characterization_1994" class="csl-entry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Breton J, Berks BC, Reilly A, Thomson AJ, Ferguson SJ, Richardson DJ. Characterization of the paramagnetic iron-containing redox centres of <span>Thiosphaera</span> pantotropha periplasmic nitrate reductase. FEBS letters. 1994;345: 76–80. doi:<a href="https://doi.org/10.1016/0014-5793(94)00445-5">10.1016/0014-5793(94)00445-5</a></div>
</div>
<div id="ref-thony-meyer_bacterial_1994" class="csl-entry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Thöny-Meyer L, Loferer H, Ritz D, Hennecke H. Bacterial genes and proteins involved in the biogenesis of c-type cytochromes and terminal oxidases. Biochimica Et Biophysica Acta. 1994;1187: 260–263. doi:<a href="https://doi.org/10.1016/0005-2728(94)90123-6">10.1016/0005-2728(94)90123-6</a></div>
</div>
<div id="ref-stouthamer_metabolic_1992" class="csl-entry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Stouthamer AH. Metabolic pathways in <span>Paracoccus</span> denitrificans and closely related bacteria in relation to the phylogeny of prokaryotes. Antonie Van Leeuwenhoek. 1992;61: 1–33. doi:<a href="https://doi.org/10.1007/BF00572119">10.1007/BF00572119</a></div>
</div>
<div id="ref-noji_molecular_1987" class="csl-entry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Noji S, Taniguchi S. Molecular oxygen controls nitrate transport of <span>Escherichia</span> coli nitrate-respiring cells. The Journal of Biological Chemistry. 1987;262: 9441–9443. Available: <a href="https://www.jbc.org/article/S0021-9258(18)47952-5/pdf">https://www.jbc.org/article/S0021-9258(18)47952-5/pdf</a></div>
</div>
<div id="ref-robertson_confirmation_1995" class="csl-entry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Robertson LA, Dalsgaard T, Revsbech N-P, Kuenen JG. Confirmation of <span>‘aerobic denitrification’</span> in batch cultures, using gas chromatography and <span>15N</span> mass spectrometry. FEMS Microbiology Ecology. 1995;18: 113–120. doi:<a href="https://doi.org/10.1111/j.1574-6941.1995.tb00168.x">10.1111/j.1574-6941.1995.tb00168.x</a></div>
</div>
<div id="ref-baumann_dynamics_1996" class="csl-entry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Baumann B, Snozzi M, Zehnder AJ, Van Der Meer JR. Dynamics of denitrification activity of <span>Paracoccus</span> denitrificans in continuous culture during aerobic-anaerobic changes. Journal of Bacteriology. 1996;178: 4367–4374. doi:<a href="https://doi.org/10.1128/jb.178.15.4367-4374.1996">10.1128/jb.178.15.4367-4374.1996</a></div>
</div>
<div id="ref-richardson_influence_1992" class="csl-entry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Richardson DJ, Ferguson SJ. The influence of carbon substrate on the activity of the periplasmic nitrate reductase in aerobically grown <span>Thiosphaera</span> pantotropha. Archives of Microbiology. 1992;157: 535–537. doi:<a href="https://doi.org/10.1007/BF00276774">10.1007/BF00276774</a></div>
</div>
<div id="ref-richardson_role_1988" class="csl-entry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">Richardson DJ, King GF, Kelly DJ, McEwan AG, Ferguson SJ, Jackson JB. The role of auxiliary oxidants in maintaining redox balance during phototrophic growth of <span>Rhodobacter</span> capsulatus on propionate or butyrate. Archives of Microbiology. 1988;150: 131–137. doi:<a href="https://doi.org/10.1007/BF00425152">10.1007/BF00425152</a></div>
</div>
<div id="ref-averill_dissimilatory_1996" class="csl-entry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">Averill BA. Dissimilatory <span>Nitrite</span> and <span>Nitric</span> <span>Oxide</span> <span>Reductases</span>. Chemical Reviews. 1996;96: 2951–2964. doi:<a href="https://doi.org/10.1021/cr950056p">10.1021/cr950056p</a></div>
</div>
<div id="ref-sann_cytochrome_1994" class="csl-entry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline">Sann R, Kostka S, Friedrich B. A cytochrome cd1-type nitrite reductase mediates the first step of denitrification in <span>Alcaligenes</span> eutrophus. Archives of Microbiology. 1994;161: 453–459. doi:<a href="https://doi.org/10.1007/BF00307765">10.1007/BF00307765</a></div>
</div>
<div id="ref-iwasaki_cytochrome_1971" class="csl-entry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">Iwasaki H, Matsubara T. Cytochrome c-557 (551) and cytochrome cd of <span>Alcaligenes</span> faecalis. Journal of Biochemistry. 1971;69: 847–857. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a129536">10.1093/oxfordjournals.jbchem.a129536</a></div>
</div>
<div id="ref-kakutani_purification_1981" class="csl-entry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Kakutani T, Watanabe H, Arima K, Beppu T. Purification and properties of a copper-containing nitrite reductase from a denitrifying bacterium, <span>Alcaligenes</span> faecalis strain <span>S</span>-6. Journal of Biochemistry. 1981;89: 453–461. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a133220">10.1093/oxfordjournals.jbchem.a133220</a></div>
</div>
<div id="ref-glockner_copper-containing_1993" class="csl-entry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Glockner AB, Jüngst A, Zumft WG. Copper-containing nitrite reductase from <span>Pseudomonas</span> aureofaciens is functional in a mutationally cytochrome cd1-free background (<span>NirS</span>-) of <span>Pseudomonas</span> stutzeri. Archives of Microbiology. 1993;160: 18–26. doi:<a href="https://doi.org/10.1007/BF00258141">10.1007/BF00258141</a></div>
</div>
<div id="ref-iwasaki_nitrite_1972" class="csl-entry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Iwasaki H, Matsubara T. A nitrite reductase from <span>Achromobacter</span> cycloclastes. Journal of Biochemistry. 1972;71: 645–652. Available: <a href="https://www.jstage.jst.go.jp/article/biochemistry1922/71/4/71_4_645/_article">https://www.jstage.jst.go.jp/article/biochemistry1922/71/4/71_4_645/_article</a></div>
</div>
<div id="ref-zumft_type_1987" class="csl-entry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">Zumft WG, Gotzmann DJ, Kroneck PM. Type 1, blue copper proteins constitute a respiratory nitrite-reducing system in <span>Pseudomonas</span> aureofaciens. European Journal of Biochemistry. 1987;168: 301–307. doi:<a href="https://doi.org/10.1111/j.1432-1033.1987.tb13421.x">10.1111/j.1432-1033.1987.tb13421.x</a></div>
</div>
<div id="ref-denariaz_denitrifying_1991" class="csl-entry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">Denariaz G, Payne WJ, LeGall J. The denitrifying nitrite reductase of <span>Bacillus</span> halodenitrificans. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1991;1056: 225–232. doi:<a href="https://doi.org/10.1016/S0005-2728(05)80053-2">10.1016/S0005-2728(05)80053-2</a></div>
</div>
<div id="ref-han_resonance_1993" class="csl-entry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Han J, Loehr TM, Lu Y, Valentine JS, Averill BA, Sanders-Loehr J. Resonance <span>Raman</span> excitation profiles indicate multiple <span>Cys</span>-<span>&gt;</span><span>Cu</span> charge transfer transitions in type 1 copper proteins. Journal of the American Chemical Society. 1993;115: 4256–4263. doi:<a href="https://doi.org/10.1021/ja00063a048">10.1021/ja00063a048</a></div>
</div>
<div id="ref-kukimoto_studies_1996" class="csl-entry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline">Kukimoto M, Nishiyama M, Tanokura M, Adman ET, Horinouchi S. Studies on protein-protein interaction between copper-containing nitrite reductase and pseudoazurin from <span>Alcaligenes</span> faecalis <span>S</span>-6. The Journal of Biological Chemistry. 1996;271: 13680–13683. doi:<a href="https://doi.org/10.1074/jbc.271.23.13680">10.1074/jbc.271.23.13680</a></div>
</div>
<div id="ref-strange_substrate-binding_1995" class="csl-entry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline">Strange RW, Dodd FE, Abraham ZH, Grossmann JG, Brüser T, Eady RR, et al. The substrate-binding site in <span>Cu</span> nitrite reductase and its similarity to <span>Zn</span> carbonic anhydrase. Nature Structural Biology. 1995;2: 287–292. doi:<a href="https://doi.org/10.1038/nsb0495-287">10.1038/nsb0495-287</a></div>
</div>
<div id="ref-hulse_evidence_1989" class="csl-entry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline">Hulse CL, Tiedje JM, Averill BA. Evidence for a copper-nitrosyl intermediate in denitrification by the copper-containing nitrite reductase of <span>Achromobacter</span> cycloclastes. Journal of the American Chemical Society. 1989;111: 2322–2323. doi:<a href="https://doi.org/10.1021/ja00188a067">10.1021/ja00188a067</a></div>
</div>
<div id="ref-jackson_evidence_1991" class="csl-entry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline">Jackson MA, Tiedje JM, Averill BA. Evidence for a <span>NO</span>-rebound mechanism for production of <span>N2O</span> from nitrite by the copper-containing nitrite reductase from <span>Achromobacter</span> cycloclastes. FEBS letters. 1991;291: 41–44. doi:<a href="https://doi.org/10.1016/0014-5793(91)81099-t">10.1016/0014-5793(91)81099-t</a></div>
</div>
<div id="ref-goretski_trapping_1988" class="csl-entry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline">Goretski J, Hollocher TC. Trapping of nitric oxide produced during denitrification by extracellular hemoglobin. The Journal of Biological Chemistry. 1988;263: 2316–2323. doi:<a href="https://doi.org/10.1016/S0021-9258(18)69208-7">10.1016/S0021-9258(18)69208-7</a></div>
</div>
<div id="ref-zumft_defects_1988" class="csl-entry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">Zumft WG, Döhler K, Körner H, Löchelt S, Viebrock A, Frunzke K. Defects in cytochrome cd1-dependent nitrite respiration of transposon <span>Tn5</span>-induced mutants from <span>Pseudomonas</span> stutzeri. Archives of Microbiology. 1988;149: 492–498. doi:<a href="https://doi.org/10.1007/BF00446750">10.1007/BF00446750</a></div>
</div>
<div id="ref-carr_nitric_1990" class="csl-entry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline">Carr GJ, Ferguson SJ. The nitric oxide reductase of <span>Paracoccus</span> denitrificans. The Biochemical Journal. 1990;269: 423–429. doi:<a href="https://doi.org/10.1042/bj2690423">10.1042/bj2690423</a></div>
</div>
<div id="ref-jones_nitric_1993" class="csl-entry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline">Jones AM, Hollocher TC. Nitric oxide reductase of <span>Achromobacter</span> cycloclastes. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1993;1144: 359–366. doi:<a href="https://doi.org/10.1016/0005-2728(93)90121-U">10.1016/0005-2728(93)90121-U</a></div>
</div>
<div id="ref-heiss_formation_1989" class="csl-entry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline">Heiss B, Frunzke K, Zumft WG. Formation of the <span>N</span>-<span>N</span> bond from nitric oxide by a membrane-bound cytochrome bc complex of nitrate-respiring (denitrifying) <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1989;171: 3288–3297. doi:<a href="https://doi.org/10.1128/jb.171.6.3288-3297.1989">10.1128/jb.171.6.3288-3297.1989</a></div>
</div>
<div id="ref-goretski_steady-state_1990" class="csl-entry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline">Goretski J, Zafiriou OC, Hollocher TC. Steady-state nitric oxide concentrations during denitrification. The Journal of Biological Chemistry. 1990;265: 11535–11538. doi:<a href="https://doi.org/10.1016/S0021-9258(19)38430-3">10.1016/S0021-9258(19)38430-3</a></div>
</div>
<div id="ref-kastrau_nitric_1994" class="csl-entry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline">Kastrau DH, Heiss B, Kroneck PM, Zumft WG. Nitric oxide reductase from <span>Pseudomonas</span> stutzeri, a novel cytochrome bc complex. <span>Phospholipid</span> requirement, electron paramagnetic resonance and redox properties. European Journal of Biochemistry. 1994;222: 293–303. doi:<a href="https://doi.org/10.1111/j.1432-1033.1994.tb18868.x">10.1111/j.1432-1033.1994.tb18868.x</a></div>
</div>
<div id="ref-fujiwara_cytochrome_1996" class="csl-entry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline">Fujiwara T, Fukumori Y. Cytochrome cb-type nitric oxide reductase with cytochrome c oxidase activity from <span>Paracoccus</span> denitrificans <span>ATCC</span> 35512. Journal of Bacteriology. 1996;178: 1866–1871. doi:<a href="https://doi.org/10.1128/jb.178.7.1866-1871.1996">10.1128/jb.178.7.1866-1871.1996</a></div>
</div>
<div id="ref-girsch_purification_1997" class="csl-entry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline">Girsch P, Vries S de. Purification and initial kinetic and spectroscopic characterization of <span>NO</span> reductase from <span>Paracoccus</span> denitrificans. Biochimica Et Biophysica Acta. 1997;1318: 202–216. doi:<a href="https://doi.org/10.1016/s0005-2728(96)00138-7">10.1016/s0005-2728(96)00138-7</a></div>
</div>
<div id="ref-van_der_oost_heme-copper_1994" class="csl-entry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline">Oost J van der, Boer AP de, Gier JW de, Zumft WG, Stouthamer AH, Spanning RJ van. The heme-copper oxidase family consists of three distinct types of terminal oxidases and is related to nitric oxide reductase. FEMS microbiology letters. 1994;121: 1–9. doi:<a href="https://doi.org/10.1111/j.1574-6968.1994.tb07067.x">10.1111/j.1574-6968.1994.tb07067.x</a></div>
</div>
<div id="ref-iwata_structure_1995" class="csl-entry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline">Iwata S, Ostermeier C, Ludwig B, Michel H. Structure at 2.8 <span>A</span> resolution of cytochrome c oxidase from <span>Paracoccus</span> denitrificans. Nature. 1995;376: 660–669. doi:<a href="https://doi.org/10.1038/376660a0">10.1038/376660a0</a></div>
</div>
<div id="ref-saraste_cytochrome_1994" class="csl-entry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline">Saraste M, Castresana J. Cytochrome oxidase evolved by tinkering with denitrification enzymes. FEBS letters. 1994;341: 1–4. doi:<a href="https://doi.org/10.1016/0014-5793(94)80228-9">10.1016/0014-5793(94)80228-9</a></div>
</div>
<div id="ref-zhao_cytochrome_1995" class="csl-entry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline">Zhao XJ, Sampath V, Caughey WS. Cytochrome c oxidase catalysis of the reduction of nitric oxide to nitrous oxide. Biochemical and Biophysical Research Communications. 1995;212: 1054–1060. doi:<a href="https://doi.org/10.1006/bbrc.1995.2076">10.1006/bbrc.1995.2076</a></div>
</div>
<div id="ref-coyle_nitrous_1985" class="csl-entry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline">Coyle CL, Zumft WG, Kroneck PM, Körner H, Jakob W. Nitrous oxide reductase from denitrifying <span>Pseudomonas</span> perfectomarina. <span>Purification</span> and properties of a novel multicopper enzyme. European Journal of Biochemistry. 1985;153: 459–467. doi:<a href="https://doi.org/10.1111/j.1432-1033.1985.tb09324.x">10.1111/j.1432-1033.1985.tb09324.x</a></div>
</div>
<div id="ref-snyder_purification_1987" class="csl-entry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline">Snyder SW, Hollocher TC. Purification and some characteristics of nitrous oxide reductase from <span>Paracoccus</span> denitrificans. The Journal of Biological Chemistry. 1987;262: 6515–6525. Available: <a href="https://www.jbc.org/article/S0021-9258(18)48272-5/pdf">https://www.jbc.org/article/S0021-9258(18)48272-5/pdf</a></div>
</div>
<div id="ref-soohoo_purification_1991" class="csl-entry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline">SooHoo CK, Hollocher TC. Purification and characterization of nitrous oxide reductase from <span>Pseudomonas</span> aeruginosa strain <span>P2</span>. The Journal of Biological Chemistry. 1991;266: 2203–2209. doi:<a href="https://doi.org/10.1016/S0021-9258(18)52229-8">10.1016/S0021-9258(18)52229-8</a></div>
</div>
<div id="ref-matsubara_new-type_1972" class="csl-entry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline">Matsubara T, Iwasaki H. A new-type of copper-protein from <span>Alcaligenes</span> faecalis. Journal of Biochemistry. 1972;71: 747–750. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a129822">10.1093/oxfordjournals.jbchem.a129822</a></div>
</div>
<div id="ref-kelly_two_1993" class="csl-entry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline">Kelly M, Lappalainen P, Talbo G, Haltia T, Oost J van der, Saraste M. Two cysteines, two histidines, and one methionine are ligands of a binuclear purple copper center. The Journal of Biological Chemistry. 1993;268: 16781–16787. Available: <a href="https://www.jbc.org/article/S0021-9258(19)85484-4/pdf">https://www.jbc.org/article/S0021-9258(19)85484-4/pdf</a></div>
</div>
<div id="ref-zumft_derived_1992" class="csl-entry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline">Zumft WG, Dreusch A, Löchelt S, Cuypers H, Friedrich B, Schneider B. Derived amino acid sequences of the <span class="nocase">nosZ</span> gene (respiratory <span>N2O</span> reductase) from <span>Alcaligenes</span> eutrophus, <span>Pseudomonas</span> aeruginosa and <span>Pseudomonas</span> stutzeri reveal potential copper-binding residues. <span>Implications</span> for the <span>CuA</span> site of <span>N2O</span> reductase and cytochrome-c oxidase. European Journal of Biochemistry. 1992;208: 31–40. doi:<a href="https://doi.org/10.1111/j.1432-1033.1992.tb17156.x">10.1111/j.1432-1033.1992.tb17156.x</a></div>
</div>
<div id="ref-farrar_model_1991" class="csl-entry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline">Farrar JA, Thomson AJ, Cheesman MR, Dooley DM, Zumft WG. A model of the copper centres of nitrous oxide reductase (<span>Pseudomonas</span> stutzeri). <span>Evidence</span> from optical, <span>EPR</span> and <span>MCD</span> spectroscopy. FEBS letters. 1991;294: 11–15. doi:<a href="https://doi.org/10.1016/0014-5793(91)81331-2">10.1016/0014-5793(91)81331-2</a></div>
</div>
<div id="ref-riester_nitrous_1989" class="csl-entry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline">Riester J, Zumft WG, Kroneck PM. Nitrous oxide reductase from <span>Pseudomonas</span> stutzeri. <span>Redox</span> properties and spectroscopic characterization of different forms of the multicopper enzyme. European Journal of Biochemistry. 1989;178: 751–762. doi:<a href="https://doi.org/10.1111/j.1432-1033.1989.tb14506.x">10.1111/j.1432-1033.1989.tb14506.x</a></div>
</div>
<div id="ref-kristjansson_first_1980" class="csl-entry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline">Kristjansson JK, Hollocher TC. First practical assay for soluble nitrous oxide reductase of denitrifying bacteria and a partial kinetic characterization. The Journal of Biological Chemistry. 1980;255: 704–707. Available: <a href="https://www.jbc.org/article/S0021-9258(19)86236-1/pdf">https://www.jbc.org/article/S0021-9258(19)86236-1/pdf</a></div>
</div>
<div id="ref-berks_purification_1993" class="csl-entry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline">Berks BC, Baratta D, Richardson J, Ferguson SJ. Purification and characterization of a nitrous oxide reductase from <span>Thiosphaera</span> pantotropha. <span>Implications</span> for the mechanism of aerobic nitrous oxide reduction. European Journal of Biochemistry. 1993;212: 467–476. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb17683.x">10.1111/j.1432-1033.1993.tb17683.x</a></div>
</div>
<div id="ref-braun_structural_1992" class="csl-entry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline">Braun C, Zumft WG. The structural genes of the nitric oxide reductase complex from <span>Pseudomonas</span> stutzeri are part of a 30-kilobase gene cluster for denitrification. Journal of Bacteriology. 1992;174: 2394–2397. doi:<a href="https://doi.org/10.1128/jb.174.7.2394-2397.1992">10.1128/jb.174.7.2394-2397.1992</a></div>
</div>
<div id="ref-hoeren_sequence_1993" class="csl-entry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Hoeren FU, Berks BC, Ferguson SJ, McCarthy JE. Sequence and expression of the gene encoding the respiratory nitrous-oxide reductase from <span>Paracoccus</span> denitrificans. <span>New</span> and conserved structural and regulatory motifs. European Journal of Biochemistry. 1993;218: 49–57. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb18350.x">10.1111/j.1432-1033.1993.tb18350.x</a></div>
</div>
<div id="ref-von_heijne_signal_1985" class="csl-entry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline">Heijne G von. Signal sequences. <span>The</span> limits of variation. Journal of Molecular Biology. 1985;184: 99–105. doi:<a href="https://doi.org/10.1016/0022-2836(85)90046-4">10.1016/0022-2836(85)90046-4</a></div>
</div>
<div id="ref-berks_common_1996" class="csl-entry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline">Berks BC. A common export pathway for proteins binding complex redox cofactors? Molecular Microbiology. 1996;22: 393–404. doi:<a href="https://doi.org/10.1046/j.1365-2958.1996.00114.x">10.1046/j.1365-2958.1996.00114.x</a></div>
</div>
<div id="ref-ambler_structure_1980" class="csl-entry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline">Ambler R. The structure and classification of cytochromes c. From <span>Cyclotrons</span> <span>To</span> <span>Cytochromes</span>: <span>Essays</span> in <span>Molecular</span> <span>Biology</span> and <span>Chemistry</span>. Saint Louis: Elsevier Science; 1980. Available: <a href="http://qut.eblib.com.au/patron/FullRecord.aspx?p=1183623">http://qut.eblib.com.au/patron/FullRecord.aspx?p=1183623</a></div>
</div>
<div id="ref-hall_reaction_1987" class="csl-entry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline">Hall J, Zha XH, Durham B, O’Brien P, Vieira B, Davis D, et al. Reaction of cytochromes c and c2 with the <span>Rhodobacter</span> sphaeroides reaction center involves the heme crevice domain. Biochemistry. 1987;26: 4494–4500. doi:<a href="https://doi.org/10.1021/bi00388a048">10.1021/bi00388a048</a></div>
</div>
<div id="ref-moir_purification_1993" class="csl-entry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Moir JW, Baratta D, Richardson DJ, Ferguson SJ. The purification of a cd1-type nitrite reductase from, and the absence of a copper-type nitrite reductase from, the aerobic denitrifier <span>Thiosphaera</span> pantotropha; the role of pseudoazurin as an electron donor. European Journal of Biochemistry. 1993;212: 377–385. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb17672.x">10.1111/j.1432-1033.1993.tb17672.x</a></div>
</div>
<div id="ref-parr_purification_1976" class="csl-entry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline">Parr SR, Barber D, Greenwood C. A purification procedure for the soluble cytochrome oxidase and some other respiratory proteins from <span>Pseudomonas</span> aeruginosa. The Biochemical Journal. 1976;157: 423–430. doi:<a href="https://doi.org/10.1042/bj1570423">10.1042/bj1570423</a></div>
</div>
<div id="ref-husain_properties_1986" class="csl-entry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline">Husain M, Davidson VL, Smith AJ. Properties of <span>Paracoccus</span> denitrificans amicyanin. Biochemistry. 1986;25: 2431–2436. doi:<a href="https://doi.org/10.1021/bi00357a020">10.1021/bi00357a020</a></div>
</div>
<div id="ref-van_de_kamp_involvement_1990" class="csl-entry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline">Kamp M van de, Silvestrini MC, Brunori M, Van Beeumen J, Hali FC, Canters GW. Involvement of the hydrophobic patch of azurin in the electron-transfer reactions with cytochrome <span>C551</span> and nitrite reductase. European Journal of Biochemistry. 1990;194: 109–118. doi:<a href="https://doi.org/10.1111/j.1432-1033.1990.tb19434.x">10.1111/j.1432-1033.1990.tb19434.x</a></div>
</div>
<div id="ref-williams_pseudospecific_1995" class="csl-entry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline">Williams PA, Fülöp V, Leung YC, Chan C, Moir JW, Howlett G, et al. Pseudospecific docking surfaces on electron transfer proteins as illustrated by pseudoazurin, cytochrome c550 and cytochrome cd1 nitrite reductase. Nature Structural Biology. 1995;2: 975–982. doi:<a href="https://doi.org/10.1038/nsb1195-975">10.1038/nsb1195-975</a></div>
</div>
<div id="ref-moir_properties_1994" class="csl-entry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline">Moir JWB, Ferguson SJ. Properties of a <span>Paracoccus</span> denitrificans mutant deleted in cytochrome c550 indicate that a copper protein can substitute for this cytochrome in electron transport to nitrite, nitric oxide and nitrous oxide. Microbiology. 1994;140: 389–397. doi:<a href="https://doi.org/10.1099/13500872-140-2-389">10.1099/13500872-140-2-389</a></div>
</div>
<div id="ref-jenney_novel_1993" class="csl-entry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline">Jenney FE, Daldal F. A novel membrane-associated c-type cytochrome, cyt cy, can mediate the photosynthetic growth of <span>Rhodobacter</span> capsulatus and <span>Rhodobacter</span> sphaeroides. The EMBO journal. 1993;12: 1283–1292. doi:<a href="https://doi.org/10.1002/j.1460-2075.1993.tb05773.x">10.1002/j.1460-2075.1993.tb05773.x</a></div>
</div>
<div id="ref-husain_characterization_1986" class="csl-entry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline">Husain M, Davidson VL. Characterization of two inducible periplasmic c-type cytochromes from <span>Paracoccus</span> denitrificans. The Journal of Biological Chemistry. 1986;261: 8577–8580. doi:<a href="https://doi.org/10.1016/S0021-9258(19)84415-0">10.1016/S0021-9258(19)84415-0</a></div>
</div>
<div id="ref-horio_terminal_1958" class="csl-entry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline">Horio T. <span>TERMINAL</span> <span>OXIDATION</span> <span>SYSTEM</span> <span>IN</span> <span>BACTERIA</span>. The Journal of Biochemistry. 1958;45: 195–205. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a126857">10.1093/oxfordjournals.jbchem.a126857</a></div>
</div>
<div id="ref-yamanaka_biological_1963" class="csl-entry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline">Yamanaka T, Kijimoto S, Okunuki K. Biological significance of <span>Pseudomonas</span> cytochrome oxidase in <span>Pseudomonas</span> aeruginosa. Journal of Biochemistry. 1963;53: 416–421. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a127716">10.1093/oxfordjournals.jbchem.a127716</a></div>
</div>
<div id="ref-timkovich_evidence_1979" class="csl-entry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline">Timkovich R, Robinson MK. Evidence for water as the product for oxygen reduction by cytochrome cd. Biochemical and Biophysical Research Communications. 1979;88: 649–655. doi:<a href="https://doi.org/10.1016/0006-291x(79)92097-7">10.1016/0006-291x(79)92097-7</a></div>
</div>
<div id="ref-lam_nitrite_1969" class="csl-entry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline">Lam Y, Nicholas DJ. A nitrite reductase with cytochrome oxidase activity from <span>Micrococcus</span> denitrificans. Biochimica Et Biophysica Acta. 1969;180: 459–472. doi:<a href="https://doi.org/10.1016/0005-2728(69)90025-5">10.1016/0005-2728(69)90025-5</a></div>
</div>
<div id="ref-sawhney_sulphide-linked_1978" class="csl-entry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline">Sawhney V, Nicholas DJD. Sulphide-linked <span>Nitrite</span> <span>Reductase</span> from <span>Thiobacillus</span> denitrificans with <span>Cytochrome</span> <span>Oxidase</span> <span>Activity</span>: <span>Purification</span> and <span>Properties</span>. Journal of General Microbiology. 1978;106: 119–128. doi:<a href="https://doi.org/10.1099/00221287-106-1-119">10.1099/00221287-106-1-119</a></div>
</div>
<div id="ref-yamazaki_nitrite_1995" class="csl-entry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline">Yamazaki T, Oyanagi H, Fujiwara T, Fukumori Y. Nitrite <span>Reductase</span> from the <span>Magnetotactic</span> <span>Bacterium</span> <span>Magnetospirillum</span> magnetotacticum. <span>A</span> <span>Novel</span> <span>Cytochrome</span> cd1 with <span>Fe</span>(<span>II</span>): <span>Nitrite</span> <span>Oxidoreductase</span> <span>Activity</span>. European Journal of Biochemistry. 1995;233: 665–671. doi:<a href="https://doi.org/10.1111/j.1432-1033.1995.665_2.x">10.1111/j.1432-1033.1995.665_2.x</a></div>
</div>
<div id="ref-doi_two_1989" class="csl-entry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline">Doi M, Shioi Y, Morita M, Takamiya K. Two types of cytochrome cd1 in the aerobic photosynthetic bacterium, <span>Erythrobacter</span> sp. <span>OCh</span> 114. European Journal of Biochemistry. 1989;184: 521–527. doi:<a href="https://doi.org/10.1111/j.1432-1033.1989.tb15045.x">10.1111/j.1432-1033.1989.tb15045.x</a></div>
</div>
<div id="ref-kuronen_new_1972" class="csl-entry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline">Kuronen T, Ellfolk N. A new purification procedure and molecular properties of <span>Pseudomonas</span> cytochrome oxidase. Biochimica Et Biophysica Acta. 1972;275: 308–318. doi:<a href="https://doi.org/10.1016/0005-2728(72)90212-5">10.1016/0005-2728(72)90212-5</a></div>
</div>
<div id="ref-chang_evidence_1986" class="csl-entry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline">Chang CK, Timkovich R, Wu W. Evidence that heme d1 is a 1,3-porphyrindione. Biochemistry. 1986;25: 8447–8453. doi:<a href="https://doi.org/10.1021/bi00374a019">10.1021/bi00374a019</a></div>
</div>
<div id="ref-hill_reconstitution_1978" class="csl-entry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline">Hill KE, Wharton DC. Reconstitution of the apoenzyme of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa with heme d1 and other heme groups. The Journal of Biological Chemistry. 1978;253: 489–495. doi:<a href="https://doi.org/10.1016/S0021-9258(17)38236-4">10.1016/S0021-9258(17)38236-4</a></div>
</div>
<div id="ref-walsh_magnetic_1979" class="csl-entry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline">Walsh TA, Johnson MK, Greenwood C, Barber D, Springall JP, Thomson AJ. Some magnetic properties of <span>Pseudomonas</span> cytochrome oxidase. The Biochemical Journal. 1979;177: 29–39. doi:<a href="https://doi.org/10.1042/bj1770029">10.1042/bj1770029</a></div>
</div>
<div id="ref-walsh_characterization_1981" class="csl-entry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline">Walsh TA, Johnson MK, Thomson AJ, Barber D, Greenwood C. The characterization and magnetic properties of the azide and imidazole derivatives of <span>Pseudomonas</span> nitrite reductase. Journal of Inorganic Biochemistry. 1981;14: 1–14. doi:<a href="https://doi.org/10.1016/s0162-0134(00)80010-0">10.1016/s0162-0134(00)80010-0</a></div>
</div>
<div id="ref-barber_reactions_1978" class="csl-entry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline">Barber D, Parr SR, Greenwood C. The reactions of <span>Pseudomonas</span> cytochrome c-551 oxidase with potassium cyanide. The Biochemical Journal. 1978;175: 239–249. doi:<a href="https://doi.org/10.1042/bj1750239">10.1042/bj1750239</a></div>
</div>
<div id="ref-parr_reaction_1975" class="csl-entry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline">Parr SR, Wilson MT, Greenwood C. The reaction of <span>Pseudomonas</span> aeruginosa cytochrome c oxidase with carbon monoxide. The Biochemical Journal. 1975;151: 51–59. doi:<a href="https://doi.org/10.1042/bj1510051">10.1042/bj1510051</a></div>
</div>
<div id="ref-gilmour_kinetics_1994" class="csl-entry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline">Gilmour R, Goodhew CF, Pettigrew GW, Prazeres S, Moura JJ, Moura I. The kinetics of the oxidation of cytochrome c by <span>Paracoccus</span> cytochrome c peroxidase. The Biochemical Journal. 1994;300 ( Pt 3): 907–914. doi:<a href="https://doi.org/10.1042/bj3000907">10.1042/bj3000907</a></div>
</div>
<div id="ref-sutherland_investigation_1986" class="csl-entry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Sutherland J, Greenwood C, Peterson J, Thomson AJ. An investigation of the ligand-binding properties of <span>Pseudomonas</span> aeruginosa nitrite reductase. The Biochemical Journal. 1986;233: 893–898. doi:<a href="https://doi.org/10.1042/bj2330893">10.1042/bj2330893</a></div>
</div>
<div id="ref-cheesman_two_1997" class="csl-entry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Cheesman MR, Ferguson SJ, Moir JW, Richardson DJ, Zumft WG, Thomson AJ. Two enzymes with a common function but different heme ligands in the forms as isolated. <span>Optical</span> and magnetic properties of the heme groups in the oxidized forms of nitrite reductase, cytochrome cd1, from <span>Pseudomonas</span> stutzeri and <span>Thiosphaera</span> pantotropha. Biochemistry. 1997;36: 16267–16276. doi:<a href="https://doi.org/10.1021/bi971677a">10.1021/bi971677a</a></div>
</div>
<div id="ref-huynh_mossbauer_1982" class="csl-entry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Huynh BH, Lui MC, Moura JJ, Moura I, Ljungdahl PO, Münck E, et al. Mössbauer and <span>EPR</span> studies on nitrite reductase from <span>Thiobacillus</span> denitrificans. The Journal of Biological Chemistry. 1982;257: 9576–9581. doi:<a href="https://doi.org/10.1016/S0021-9258(18)34110-3">10.1016/S0021-9258(18)34110-3</a></div>
</div>
<div id="ref-fulop_anatomy_1995" class="csl-entry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Fülöp V, Moir JW, Ferguson SJ, Hajdu J. The anatomy of a bifunctional enzyme: Structural basis for reduction of oxygen to water and synthesis of nitric oxide by cytochrome cd1. Cell. 1995;81: 369–377. doi:<a href="https://doi.org/10.1016/0092-8674(95)90390-9">10.1016/0092-8674(95)90390-9</a></div>
</div>
<div id="ref-horio_relative_1961" class="csl-entry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline">Horio T, Kamen MD, De Klerk H. Relative oxidation-reduction potentials of heme groups in two soluble doubleheme proteins. The Journal of Biological Chemistry. 1961;236: 2783–2787. doi:<a href="https://doi.org/10.1016/S0021-9258(19)61737-0">10.1016/S0021-9258(19)61737-0</a></div>
</div>
<div id="ref-kamen_bacterial_1970" class="csl-entry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline">Kamen MD, Horio T. Bacterial cytochromes. <span>I</span>. <span>Structural</span> aspects. Annual Review of Biochemistry. 1970;39: 673–700. doi:<a href="https://doi.org/10.1146/annurev.bi.39.070170.003325">10.1146/annurev.bi.39.070170.003325</a></div>
</div>
<div id="ref-wharton_cytochrome_1973" class="csl-entry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline">Wharton DC, Gudat JC, Gibson QH. Cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. <span>II</span>. <span>Reaction</span> with copper protein. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1973;292: 611–620. doi:<a href="https://doi.org/10.1016/0005-2728(73)90009-1">10.1016/0005-2728(73)90009-1</a></div>
</div>
<div id="ref-shimada_oxidation-reduction_1976" class="csl-entry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline">Shimada H, Orii Y. Oxidation-reduction behavior of the heme c and heme d moieties of <span>Pseudomonas</span> aeruginosa nitrite reductase and the formation of an oxygenated intermediate at heme d1. Journal of Biochemistry. 1976;80: 135–140. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a131245">10.1093/oxfordjournals.jbchem.a131245</a></div>
</div>
<div id="ref-blatt_allosteric_1979" class="csl-entry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline">Blatt Y, Pecht I. Allosteric cooperative interactions among redox sites of <span>Pseudomonas</span> cytochrome oxidase. Biochemistry. 1979;18: 2917–2922. doi:<a href="https://doi.org/10.1021/bi00580a037">10.1021/bi00580a037</a></div>
</div>
<div id="ref-schichman_kinetics_1981" class="csl-entry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline">Schichman SA, Gray HB. Kinetics of the anaerobic reduction of ferricytochrome cd1 by <span>Fe</span>(<span>EDTA</span>)2-. <span>Evidence</span> for bimolecular and intramolecular electron transfers to the d1 hemes. Journal of the American Chemical Society. 1981;103: 7794–7795. doi:<a href="https://doi.org/10.1021/ja00416a020">10.1021/ja00416a020</a></div>
</div>
<div id="ref-silvestrini_kinetics_1982" class="csl-entry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Silvestrini MC, Tordi MG, Colosimo A, Antonini E, Brunori M. The kinetics of electron transfer between pseudomonas aeruginosa cytochrome c-551 and its oxidase. The Biochemical Journal. 1982;203: 445–451. doi:<a href="https://doi.org/10.1042/bj2030445">10.1042/bj2030445</a></div>
</div>
<div id="ref-carson_variations_1986" class="csl-entry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline">Carson SD, Ching YC, Wells CA, Wharton DC, Ondrias MR. Variations in the oxidation-reduction behavior of liganded species of <span>Pseudomonas</span> cytochrome oxidase. Biochemistry. 1986;25: 787–790. doi:<a href="https://doi.org/10.1021/bi00352a008">10.1021/bi00352a008</a></div>
</div>
<div id="ref-besson_cytochrome_1995" class="csl-entry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline">Besson S, Carneiro C, Moura JJ, Moura I, Fauque G. A cytochrome cd1-type nitrite reductase isolated from the marine denitrifier <span>Pseudomonas</span> nautica 617: Purification and characterization. Anaerobe. 1995;1: 219–226. doi:<a href="https://doi.org/10.1006/anae.1995.1021">10.1006/anae.1995.1021</a></div>
</div>
<div id="ref-silvestrini_nitrite_1989" class="csl-entry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline">Silvestrini MC, Galeotti CL, Gervais M, Schininà E, Barra D, Bossa F, et al. Nitrite reductase from <span>Pseudomonas</span> aeruginosa: Sequence of the gene and the protein. FEBS letters. 1989;254: 33–38. doi:<a href="https://doi.org/10.1016/0014-5793(89)81004-x">10.1016/0014-5793(89)81004-x</a></div>
</div>
<div id="ref-jungst_nirstbm_1991" class="csl-entry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Jüngst A, Wakabayashi S, Matsubara H, Zumft WG. The <span class="nocase">nirSTBM</span> region coding for cytochrome cd1-dependent nitrite respiration of <span>Pseudomonas</span> stutzeri consists of a cluster of mono-, di-, and tetraheme proteins. FEBS letters. 1991;279: 205–209. doi:<a href="https://doi.org/10.1016/0014-5793(91)80150-2">10.1016/0014-5793(91)80150-2</a></div>
</div>
<div id="ref-smith_isolation_1992" class="csl-entry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">Smith GB, Tiedje JM. Isolation and characterization of a nitrite reductase gene and its use as a probe for denitrifying bacteria. Applied and Environmental Microbiology. 1992;58: 376–384. doi:<a href="https://doi.org/10.1128/AEM.58.1.376-384.1992">10.1128/AEM.58.1.376-384.1992</a></div>
</div>
<div id="ref-ohshima_cloning_1993" class="csl-entry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Ohshima T, Sugiyama M, Uozumi N, Iijima S, Kobayashi T. Cloning and sequencing of a gene encoding nitrite reductase from <span>Paracoccus</span> denitrificans and expression of the gene in <span>Escherichia</span> coli. Journal of Fermentation and Bioengineering. 1993;76: 82–88. doi:<a href="https://doi.org/10.1016/0922-338X(93)90061-C">10.1016/0922-338X(93)90061-C</a></div>
</div>
<div id="ref-de_boer_isolation_1994" class="csl-entry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Boer AP de, Reijnders WN, Kuenen JG, Stouthamer AH, Spanning RJ van. Isolation, sequencing and mutational analysis of a gene cluster involved in nitrite reduction in <span>Paracoccus</span> denitrificans. Antonie Van Leeuwenhoek. 1994;66: 111–127. doi:<a href="https://doi.org/10.1007/BF00871635">10.1007/BF00871635</a></div>
</div>
<div id="ref-rees_structural_1997" class="csl-entry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline">Rees E, Siddiqui RA, Köster F, Schneider B, Friedrich B. Structural gene (<span class="nocase">nirS</span>) for the cytochrome cd1 nitrite reductase of <span>Alcaligenes</span> eutrophus <span>H16</span>. Applied and Environmental Microbiology. 1997;63: 800–802. doi:<a href="https://doi.org/10.1128/AEM.63.2.800-802.1997">10.1128/AEM.63.2.800-802.1997</a></div>
</div>
<div id="ref-horowitz_controlled_1982" class="csl-entry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline">Horowitz PM, Muhoberac BB, Falksen K, Wharton DC. Controlled proteolysis by subtilisin as a probe for cyanide-induced conformational changes in <span>Pseudomonas</span> cytochrome oxidase. The Journal of Biological Chemistry. 1982;257: 2140–2143. Available: <a href="https://www.jbc.org/article/S0021-9258(18)34895-6/pdf">https://www.jbc.org/article/S0021-9258(18)34895-6/pdf</a></div>
</div>
<div id="ref-tordi_circular-dichroic_1984" class="csl-entry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline">Tordi MG, Silvestrini MC, Colosimo A, Provencher S, Brunori M. Circular-dichroic properties and secondary structure of <span>Pseudomonas</span> aeruginosa soluble cytochrome c oxidase. The Biochemical Journal. 1984;218: 907–912. doi:<a href="https://doi.org/10.1042/bj2180907">10.1042/bj2180907</a></div>
</div>
<div id="ref-berger_small-angle_1980" class="csl-entry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline">Berger H, Wharton DC. Small-angle <span>X</span>-ray scattering studies of oxidized and reduced cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. Biochimica Et Biophysica Acta. 1980;622: 355–359. doi:<a href="https://doi.org/10.1016/0005-2795(80)90047-1">10.1016/0005-2795(80)90047-1</a></div>
</div>
<div id="ref-mitra_heat_1981" class="csl-entry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline">Mitra S, Donovan JW, Bersohn R. Heat stabilization dependence on redox state of cytochrome cd1 oxidase from <span>Pseudomonas</span> aeruginosa. Biochemical and Biophysical Research Communications. 1981;98: 140–146. doi:<a href="https://doi.org/10.1016/0006-291x(81)91880-5">10.1016/0006-291x(81)91880-5</a></div>
</div>
<div id="ref-akey_characterization_1980" class="csl-entry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline">Akey CW, Moffat K, Wharton DC, Edelstein SJ. Characterization of crystals of a cytochrome oxidase (nitrite reductase) from <span>Pseudomonas</span> aeruginosa by x-ray diffraction and electron microscopy. Journal of Molecular Biology. 1980;136: 19–43. doi:<a href="https://doi.org/10.1016/0022-2836(80)90364-2">10.1016/0022-2836(80)90364-2</a></div>
</div>
<div id="ref-mitra_location_1980" class="csl-entry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline">Mitra S, Bersohn R. Location of the heme groups in cytochrome cd1 oxidase from <span>Pseudomonas</span> aeruginosa. Biochemistry. 1980;19: 3200–3203. doi:<a href="https://doi.org/10.1021/bi00555a015">10.1021/bi00555a015</a></div>
</div>
<div id="ref-yamanaka_crystalline_1963" class="csl-entry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline">Yamanaka T, Okunuki K. Crystalline <span>Pseudomonas</span> cytochrome oxidase. <span>I</span>. <span>Enzymic</span> properties with special reference to the biological specificity. Biochimica Et Biophysica Acta. 1963;67: 379–393. doi:<a href="https://doi.org/10.1016/0006-3002(63)91844-4">10.1016/0006-3002(63)91844-4</a></div>
</div>
<div id="ref-tegoni_crystallization_1994" class="csl-entry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline">Tegoni M, Silvestrini MC, Lamzin VS, Brunori M, Cambillau C. Crystallization and preliminary <span>X</span>-ray analysis of a new crystal form of nitrite reductase from <span>Pseudomonas</span> aeruginosa. Journal of Molecular Biology. 1994;243: 347–350. doi:<a href="https://doi.org/10.1006/jmbi.1994.1659">10.1006/jmbi.1994.1659</a></div>
</div>
<div id="ref-moore_cytochromes_1990" class="csl-entry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline">Moore GR, Pettigrew GW. Cytochromes <span>C</span> <span>Evolutionary</span>, <span>Structural</span> and <span>Physicochemical</span> <span>Aspects</span>. Berlin, Heidelberg: Springer Berlin / Heidelberg; 1990. Available: <a href="http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030">http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030</a></div>
</div>
<div id="ref-xia_three-dimensional_1992" class="csl-entry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline">Xia ZX, Dai WW, Xiong JP, Hao ZP, Davidson VL, White S, et al. The three-dimensional structures of methanol dehydrogenase from two methylotrophic bacteria at 2.6-<span>A</span> resolution. The Journal of Biological Chemistry. 1992;267: 22289–22297. Available: <a href="https://www.jbc.org/article/S0021-9258(18)41668-7/pdf">https://www.jbc.org/article/S0021-9258(18)41668-7/pdf</a></div>
</div>
<div id="ref-vellieux_structure_1989" class="csl-entry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline">Vellieux FM, Huitema F, Groendijk H, Kalk KH, Jzn JF, Jongejan JA, et al. Structure of quinoprotein methylamine dehydrogenase at 2.25 <span>A</span> resolution. The EMBO journal. 1989;8: 2171–2178. doi:<a href="https://doi.org/10.1002/j.1460-2075.1989.tb08339.x">10.1002/j.1460-2075.1989.tb08339.x</a></div>
</div>
<div id="ref-varghese_structure_1983" class="csl-entry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline">Varghese JN, Laver WG, Colman PM. Structure of the influenza virus glycoprotein antigen neuraminidase at 2.9 <span>A</span> resolution. Nature. 1983;303: 35–40. doi:<a href="https://doi.org/10.1038/303035a0">10.1038/303035a0</a></div>
</div>
<div id="ref-dhesi_patterns_1984" class="csl-entry">
<div class="csl-left-margin">155. </div><div class="csl-right-inline">Dhesi R, Timkovich R. Patterns of product inhibition for bacterial nitrite reductase. Biochemical and Biophysical Research Communications. 1984;123: 966–972. doi:<a href="https://doi.org/10.1016/s0006-291x(84)80228-4">10.1016/s0006-291x(84)80228-4</a></div>
</div>
<div id="ref-robinson_implications_1979" class="csl-entry">
<div class="csl-left-margin">156. </div><div class="csl-right-inline">Robinson MK, Martinkus K, Kennelly PJ, Timkovich R. Implications of the integrated rate law for the reactions of <span>Paracoccus</span> denitrificans nitrite reductase. Biochemistry. 1979;18: 3921–3926. doi:<a href="https://doi.org/10.1021/bi00585a012">10.1021/bi00585a012</a></div>
</div>
<div id="ref-parr_electron-transfer_1977" class="csl-entry">
<div class="csl-left-margin">157. </div><div class="csl-right-inline">Parr SR, Barber D, Greenwood C, Brunori M. The electron-transfer reaction between azurin and the cytochrome c oxidase from <span>Pseudomonas</span> aeruginosa. The Biochemical Journal. 1977;167: 447–455. doi:<a href="https://doi.org/10.1042/bj1670447">10.1042/bj1670447</a></div>
</div>
<div id="ref-kim_catalysis_1984" class="csl-entry">
<div class="csl-left-margin">158. </div><div class="csl-right-inline">Kim CH, Hollocher TC. Catalysis of nitrosyl transfer reactions by a dissimilatory nitrite reductase (cytochrome c,d1). The Journal of Biological Chemistry. 1984;259: 2092–2099. Available: <a href="https://www.jbc.org/article/S0021-9258(17)43321-7/pdf">https://www.jbc.org/article/S0021-9258(17)43321-7/pdf</a></div>
</div>
<div id="ref-kim_15n_1983" class="csl-entry">
<div class="csl-left-margin">159. </div><div class="csl-right-inline">Kim CH, Hollocher TC. <span>15N</span> tracer studies on the reduction of nitrite by the purified dissimilatory nitrite reductase of <span>Pseudomonas</span> aeruginosa. <span>Evidence</span> for direct production of <span>N2O</span> without free <span>NO</span> as an intermediate. The Journal of Biological Chemistry. 1983;258: 4861–4863. doi:<a href="https://doi.org/10.1016/S0021-9258(18)32505-5">10.1016/S0021-9258(18)32505-5</a></div>
</div>
<div id="ref-carr_energy-conserving_1989" class="csl-entry">
<div class="csl-left-margin">160. </div><div class="csl-right-inline">Carr GJ, Page MD, Ferguson SJ. The energy-conserving nitric-oxide-reductase system in <span>Paracoccus</span> denitrificans. <span>Distinction</span> from the nitrite reductase that catalyses synthesis of nitric oxide and evidence from trapping experiments for nitric oxide as a free intermediate during denitrification. European Journal of Biochemistry. 1989;179: 683–692. doi:<a href="https://doi.org/10.1111/j.1432-1033.1989.tb14601.x">10.1111/j.1432-1033.1989.tb14601.x</a></div>
</div>
<div id="ref-wang_direct_1996" class="csl-entry">
<div class="csl-left-margin">161. </div><div class="csl-right-inline">Wang Y, Averill BA. Direct <span>Observation</span> by <span>FTIR</span> <span>Spectroscopy</span> of the <span>Ferrous</span> <span>Heme</span>−<span>NO</span> <span class="math inline">\(^{\textrm{+}}\)</span> <span>Intermediate</span> in <span>Reduction</span> of <span>Nitrite</span> by a <span>Dissimilatory</span> <span>Heme</span> <em>cd</em> <span class="math inline">\(_{\textrm{1}}\)</span> <span>Nitrite</span> <span>Reductase</span>. Journal of the American Chemical Society. 1996;118: 3972–3973. doi:<a href="https://doi.org/10.1021/ja9538647">10.1021/ja9538647</a></div>
</div>
<div id="ref-silvestrini_reaction_1990" class="csl-entry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline">Silvestrini MC, Tordi MG, Musci G, Brunori M. The reaction of <span>Pseudomonas</span> nitrite reductase and nitrite. <span>A</span> stopped-flow and <span>EPR</span> study. The Journal of Biological Chemistry. 1990;265: 11783–11787. doi:<a href="https://doi.org/10.1016/S0021-9258(19)38466-2">10.1016/S0021-9258(19)38466-2</a></div>
</div>
<div id="ref-legall_purification_1979" class="csl-entry">
<div class="csl-left-margin">163. </div><div class="csl-right-inline">LeGall J, Payne WJ, Morgan TV, DerVartanian D. On the purification of nitrite reductase from <span>Thiobacillus</span> denitrificans and its reaction with nitrite under reducing conditions. Biochemical and Biophysical Research Communications. 1979;87: 355–362. doi:<a href="https://doi.org/10.1016/0006-291x(79)91804-7">10.1016/0006-291x(79)91804-7</a></div>
</div>
<div id="ref-traylor_why_1992" class="csl-entry">
<div class="csl-left-margin">164. </div><div class="csl-right-inline">Traylor TG, Sharma VS. Why <span>NO</span>? Biochemistry. 1992;31: 2847–2849. doi:<a href="https://doi.org/10.1021/bi00126a001">10.1021/bi00126a001</a></div>
</div>
<div id="ref-kobayashi_pulse_1997" class="csl-entry">
<div class="csl-left-margin">165. </div><div class="csl-right-inline">Kobayashi K, Koppenhöfer A, Ferguson SJ, Tagawa S. Pulse radiolysis studies on cytochrome cd1 nitrite reductase from <span>Thiosphaera</span> pantotropha: Evidence for a fast intramolecular electron transfer from c-heme to d1-heme. Biochemistry. 1997;36: 13611–13616. doi:<a href="https://doi.org/10.1021/bi971045o">10.1021/bi971045o</a></div>
</div>
<div id="ref-williams_time-resolved_1996" class="csl-entry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline">Williams P. Time-resolved structural studies on macromolecules. PhD thesis, University of Oxford. 1996. Available: <a href="https://www.worldcat.org/title/time-resolved-structural-studies-on-macromolecules/oclc/43530285">https://www.worldcat.org/title/time-resolved-structural-studies-on-macromolecules/oclc/43530285</a></div>
</div>
<div id="ref-zumft_isolation_1985" class="csl-entry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline">Zumft WG, Döhler K, Körner H. Isolation and characterization of transposon <span>Tn5</span>-induced mutants of <span>Pseudomonas</span> perfectomarina defective in nitrous oxide respiration. Journal of Bacteriology. 1985;163: 918–924. doi:<a href="https://doi.org/10.1128/JB.163.3.918-924.1985">10.1128/JB.163.3.918-924.1985</a></div>
</div>
<div id="ref-zumft_enzyme_1997" class="csl-entry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline">Zumft WG, Körner H. Enzyme diversity and mosaic gene organization in denitrification. Antonie Van Leeuwenhoek. 1997;71: 43–58. doi:<a href="https://doi.org/10.1023/a:1000112008026">10.1023/a:1000112008026</a></div>
</div>
<div id="ref-de_boer_regulation_1996" class="csl-entry">
<div class="csl-left-margin">169. </div><div class="csl-right-inline">Boer T de. Regulation of denitrification in <span>Paracoccus</span> denitrificans. PhD thesis, VU Amsterdam. 1996. Available: <a href="https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans">https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans</a></div>
</div>
<div id="ref-arai_cloning_1990" class="csl-entry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline">Arai H, Sanbongi Y, Igarashi Y, Kodama T. Cloning and sequencing of the gene encoding cytochrome c-551 from <span>Pseudomonas</span> aeruginosa. FEBS letters. 1990;261: 196–198. doi:<a href="https://doi.org/10.1016/0014-5793(90)80669-a">10.1016/0014-5793(90)80669-a</a></div>
</div>
<div id="ref-jungst_interdependence_1992" class="csl-entry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline">Jüngst A, Zumft WG. Interdependence of respiratory <span>NO</span> reduction and nitrite reduction revealed by mutagenesis of <span class="nocase">nirQ</span>, a novel gene in the denitrification gene cluster of <span>Pseudomonas</span> stutzeri. FEBS letters. 1992;314: 308–314. doi:<a href="https://doi.org/10.1016/0014-5793(92)81495-8">10.1016/0014-5793(92)81495-8</a></div>
</div>
<div id="ref-de_boer_mutational_1996" class="csl-entry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline">Boer AP de, Oost J van der, Reijnders WN, Westerhoff HV, Stouthamer AH, Spanning RJ van. Mutational analysis of the nor gene cluster which encodes nitric-oxide reductase from <span>Paracoccus</span> denitrificans. European Journal of Biochemistry. 1996;242: 592–600. doi:<a href="https://doi.org/10.1111/j.1432-1033.1996.0592r.x">10.1111/j.1432-1033.1996.0592r.x</a></div>
</div>
<div id="ref-cuypers_nosr_1992" class="csl-entry">
<div class="csl-left-margin">173. </div><div class="csl-right-inline">Cuypers H, Viebrock-Sambale A, Zumft WG. <span>NosR</span>, a membrane-bound regulatory component necessary for expression of nitrous oxide reductase in denitrifying <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1992;174: 5332–5339. doi:<a href="https://doi.org/10.1128/jb.174.16.5332-5339.1992">10.1128/jb.174.16.5332-5339.1992</a></div>
</div>
<div id="ref-glockner_sequence_1996" class="csl-entry">
<div class="csl-left-margin">174. </div><div class="csl-right-inline">Glockner AB, Zumft WG. Sequence analysis of an internal 9.72-kb segment from the 30-kb denitrification gene cluster of <span>Pseudomonas</span> stutzeri. Biochimica Et Biophysica Acta. 1996;1277: 6–12. doi:<a href="https://doi.org/10.1016/s0005-2728(96)00108-9">10.1016/s0005-2728(96)00108-9</a></div>
</div>
<div id="ref-mushegian_gene_1996" class="csl-entry">
<div class="csl-left-margin">175. </div><div class="csl-right-inline">Mushegian AR, Koonin EV. Gene order is not conserved in bacterial evolution. Trends in genetics: TIG. 1996;12: 289–290. doi:<a href="https://doi.org/10.1016/0168-9525(96)20006-x">10.1016/0168-9525(96)20006-x</a></div>
</div>
<div id="ref-lambden_mutants_1976" class="csl-entry">
<div class="csl-left-margin">176. </div><div class="csl-right-inline">Lambden PR, Guest JR. Mutants of <span>Escherichia</span> coli <span>K12</span> unable to use fumarate as an anaerobic electron acceptor. Journal of General Microbiology. 1976;97: 145–160. doi:<a href="https://doi.org/10.1099/00221287-97-2-145">10.1099/00221287-97-2-145</a></div>
</div>
<div id="ref-shaw_homology_1983" class="csl-entry">
<div class="csl-left-margin">177. </div><div class="csl-right-inline">Shaw DJ, Rice DW, Guest JR. Homology between <span>CAP</span> and <span>Fnr</span>, a regulator of anaerobic respiration in <span>Escherichia</span> coli. Journal of Molecular Biology. 1983;166: 241–247. doi:<a href="https://doi.org/10.1016/s0022-2836(83)80011-4">10.1016/s0022-2836(83)80011-4</a></div>
</div>
<div id="ref-green_reconstitution_1996" class="csl-entry">
<div class="csl-left-margin">178. </div><div class="csl-right-inline">Green J, Bennett B, Jordan P, Ralph ET, Thomson AJ, Guest JR. Reconstitution of the [<span>4Fe</span>-<span>4S</span>] cluster in <span>FNR</span> and demonstration of the aerobic-anaerobic transcription switch in vitro. The Biochemical Journal. 1996;316 ( Pt 3): 887–892. doi:<a href="https://doi.org/10.1042/bj3160887">10.1042/bj3160887</a></div>
</div>
<div id="ref-khoroshilova_association_1995" class="csl-entry">
<div class="csl-left-margin">179. </div><div class="csl-right-inline">Khoroshilova N, Beinert H, Kiley PJ. Association of a polynuclear iron-sulfur center with a mutant <span>FNR</span> protein enhances <span>DNA</span> binding. Proceedings of the National Academy of Sciences of the United States of America. 1995;92: 2499–2503. doi:<a href="https://doi.org/10.1073/pnas.92.7.2499">10.1073/pnas.92.7.2499</a></div>
</div>
<div id="ref-becker_o2_1996" class="csl-entry">
<div class="csl-left-margin">180. </div><div class="csl-right-inline">Becker S, Holighaus G, Gabrielczyk T, Unden G. O2 as the regulatory signal for <span>FNR</span>-dependent gene regulation in <span>Escherichia</span> coli. Journal of Bacteriology. 1996;178: 4515–4521. doi:<a href="https://doi.org/10.1128/jb.178.15.4515-4521.1996">10.1128/jb.178.15.4515-4521.1996</a></div>
</div>
<div id="ref-unden_effect_1990" class="csl-entry">
<div class="csl-left-margin">181. </div><div class="csl-right-inline">Unden G, Trageser M, Duchêne A. Effect of positive redox potentials (greater than +400 <span class="nocase">mV</span>) on the expression of anaerobic respiratory enzymes in <span>Escherichia</span> coli. Molecular Microbiology. 1990;4: 315–319. doi:<a href="https://doi.org/10.1111/j.1365-2958.1990.tb00598.x">10.1111/j.1365-2958.1990.tb00598.x</a></div>
</div>
<div id="ref-walker_promoter_1991" class="csl-entry">
<div class="csl-left-margin">182. </div><div class="csl-right-inline">Walker MS, DeMoss JA. Promoter sequence requirements for <span>Fnr</span>-dependent activation of transcription of the <span class="nocase">narGHJI</span> operon. Molecular Microbiology. 1991;5: 353–360. doi:<a href="https://doi.org/10.1111/j.1365-2958.1991.tb02116.x">10.1111/j.1365-2958.1991.tb02116.x</a></div>
</div>
<div id="ref-spiro_regulation_1987" class="csl-entry">
<div class="csl-left-margin">183. </div><div class="csl-right-inline">Spiro S, Guest JR. Regulation and over-expression of the fnr gene of <span>Escherichia</span> coli. Journal of General Microbiology. 1987;133: 3279–3288. doi:<a href="https://doi.org/10.1099/00221287-133-12-3279">10.1099/00221287-133-12-3279</a></div>
</div>
<div id="ref-sharrocks_fnr_1991" class="csl-entry">
<div class="csl-left-margin">184. </div><div class="csl-right-inline">Sharrocks AD, Green J, Guest JR. <span>FNR</span> activates and represses transcription in vitro. Proceedings Biological Sciences. 1991;245: 219–226. doi:<a href="https://doi.org/10.1098/rspb.1991.0113">10.1098/rspb.1991.0113</a></div>
</div>
<div id="ref-ziegelhoffer_vitro_1995" class="csl-entry">
<div class="csl-left-margin">185. </div><div class="csl-right-inline">Ziegelhoffer EC, Kiley PJ. In vitro analysis of a constitutively active mutant form of the <span>Escherichia</span> coli global transcription factor <span>FNR</span>. Journal of Molecular Biology. 1995;245: 351–361. doi:<a href="https://doi.org/10.1006/jmbi.1994.0029">10.1006/jmbi.1994.0029</a></div>
</div>
<div id="ref-park_regulation_1995" class="csl-entry">
<div class="csl-left-margin">186. </div><div class="csl-right-inline">Park SJ, Tseng CP, Gunsalus RP. Regulation of succinate dehydrogenase (<span class="nocase">sdhCDAB</span>) operon expression in <span>Escherichia</span> coli in response to carbon supply and anaerobiosis: Role of <span>ArcA</span> and <span>Fnr</span>. Molecular Microbiology. 1995;15: 473–482. doi:<a href="https://doi.org/10.1111/j.1365-2958.1995.tb02261.x">10.1111/j.1365-2958.1995.tb02261.x</a></div>
</div>
<div id="ref-williams_role_1991" class="csl-entry">
<div class="csl-left-margin">187. </div><div class="csl-right-inline">Williams R, Bell A, Sims G, Busby S. The role of two surface exposed loops in transcription activation by the <span>Escherichia</span> coli <span>CRP</span> and <span>FNR</span> proteins. Nucleic Acids Research. 1991;19: 6705–6712. doi:<a href="https://doi.org/10.1093/nar/19.24.6705">10.1093/nar/19.24.6705</a></div>
</div>
<div id="ref-spiro_fnr_1994" class="csl-entry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline">Spiro S. The <span>FNR</span> family of transcriptional regulators. Antonie Van Leeuwenhoek. 1994;66: 23–36. doi:<a href="https://doi.org/10.1007/BF00871630">10.1007/BF00871630</a></div>
</div>
<div id="ref-spiro_interconversion_1990" class="csl-entry">
<div class="csl-left-margin">189. </div><div class="csl-right-inline">Spiro S, Gaston KL, Bell AI, Roberts RE, Busby SJ, Guest JR. Interconversion of the <span>DNA</span>-binding specificities of two related transcription regulators, <span>CRP</span> and <span>FNR</span>. Molecular Microbiology. 1990;4: 1831–1838. doi:<a href="https://doi.org/10.1111/j.1365-2958.1990.tb02031.x">10.1111/j.1365-2958.1990.tb02031.x</a></div>
</div>
<div id="ref-sawers_transcriptional_1997" class="csl-entry">
<div class="csl-left-margin">190. </div><div class="csl-right-inline">Sawers G, Kaiser M, Sirko A, Freundlich M. Transcriptional activation by <span>FNR</span> and <span>CRP</span>: Reciprocity of binding-site recognition. Molecular Microbiology. 1997;23: 835–845. doi:<a href="https://doi.org/10.1046/j.1365-2958.1997.2811637.x">10.1046/j.1365-2958.1997.2811637.x</a></div>
</div>
<div id="ref-thompson_clustal_1994" class="csl-entry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline">Thompson JD, Higgins DG, Gibson TJ. <span>CLUSTAL</span> <span>W</span>: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research. 1994;22: 4673–4680. doi:<a href="https://doi.org/10.1093/nar/22.22.4673">10.1093/nar/22.22.4673</a></div>
</div>
<div id="ref-saitou_neighbor-joining_1987" class="csl-entry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline">Saitou N, Nei M. The neighbor-joining method: A new method for reconstructing phylogenetic trees. Molecular Biology and Evolution. 1987;4: 406–425. doi:<a href="https://doi.org/10.1093/oxfordjournals.molbev.a040454">10.1093/oxfordjournals.molbev.a040454</a></div>
</div>
<div id="ref-felsenstein_confidence_1985" class="csl-entry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline">Felsenstein J. <span>CONFIDENCE</span> <span>LIMITS</span> <span>ON</span> <span>PHYLOGENIES</span>: <span>AN</span> <span>APPROACH</span> <span>USING</span> <span>THE</span> <span>BOOTSTRAP</span>. Evolution; International Journal of Organic Evolution. 1985;39: 783–791. doi:<a href="https://doi.org/10.1111/j.1558-5646.1985.tb00420.x">10.1111/j.1558-5646.1985.tb00420.x</a></div>
</div>
<div id="ref-page_treeview_1996" class="csl-entry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline">Page RD. <span>TreeView</span>: An application to display phylogenetic trees on personal computers. Computer applications in the biosciences: CABIOS. 1996;12: 357–358. doi:<a href="https://doi.org/10.1093/bioinformatics/12.4.357">10.1093/bioinformatics/12.4.357</a></div>
</div>
<div id="ref-van_spanning_fnrp_1997" class="csl-entry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline">Van Spanning RJM, De Boer APN, Reijnders WNM, Westerhoff HV, Stouthamer AH, Van Der Oost J. <span>FnrP</span> and <span>NNR</span> of <em><span>Paracoccus</span> denitrificans</em> are both members of the <span>FNR</span> family of transcriptional activators but have distinct roles in respiratory adaptation in response to oxygen limitation. Molecular Microbiology. 1997;23: 893–907. doi:<a href="https://doi.org/10.1046/j.1365-2958.1997.2801638.x">10.1046/j.1365-2958.1997.2801638.x</a></div>
</div>
<div id="ref-frias_general_1993" class="csl-entry">
<div class="csl-left-margin">196. </div><div class="csl-right-inline">Frías JE, Mérida A, Herrero A, Martín-Nieto J, Flores E. General distribution of the nitrogen control gene <span class="nocase">ntcA</span> in cyanobacteria. Journal of Bacteriology. 1993;175: 5710–5713. doi:<a href="https://doi.org/10.1128/jb.175.17.5710-5713.1993">10.1128/jb.175.17.5710-5713.1993</a></div>
</div>
<div id="ref-van_spanning_nitrite_1995" class="csl-entry">
<div class="csl-left-margin">197. </div><div class="csl-right-inline">Van Spanning RJ, De Boer AP, Reijnders WN, Spiro S, Westerhoff HV, Stouthamer AH, et al. Nitrite and nitric oxide reduction in <span>Paracoccus</span> denitrificans is under the control of <span>NNR</span>, a regulatory protein that belongs to the <span>FNR</span> family of transcriptional activators. FEBS letters. 1995;360: 151–154. doi:<a href="https://doi.org/10.1016/0014-5793(95)00091-m">10.1016/0014-5793(95)00091-m</a></div>
</div>
<div id="ref-ye_anaerobic_1995" class="csl-entry">
<div class="csl-left-margin">198. </div><div class="csl-right-inline">Ye RW, Haas D, Ka JO, Krishnapillai V, Zimmermann A, Baird C, et al. Anaerobic activation of the entire denitrification pathway in <span>Pseudomonas</span> aeruginosa requires <span>Anr</span>, an analog of <span>Fnr</span>. Journal of Bacteriology. 1995;177: 3606–3609. doi:<a href="https://doi.org/10.1128/jb.177.12.3606-3609.1995">10.1128/jb.177.12.3606-3609.1995</a></div>
</div>
<div id="ref-arai_expression_1995" class="csl-entry">
<div class="csl-left-margin">199. </div><div class="csl-right-inline">Arai H, Igarashi Y, Kodama T. Expression of the nir and nor genes for denitrification of <span>Pseudomonas</span> aeruginosa requires a novel <span>CRP</span>/<span>FNR</span>-related transcriptional regulator, <span>DNR</span>, in addition to <span>ANR</span>. FEBS letters. 1995;371: 73–76. doi:<a href="https://doi.org/10.1016/0014-5793(95)00885-d">10.1016/0014-5793(95)00885-d</a></div>
</div>
<div id="ref-batut_fixk_1989" class="csl-entry">
<div class="csl-left-margin">200. </div><div class="csl-right-inline">Batut J, Daveran-Mingot ML, David M, Jacobs J, Garnerone AM, Kahn D. <span class="nocase">fixK</span>, a gene homologous with fnr and crp from <span>Escherichia</span> coli, regulates nitrogen fixation genes both positively and negatively in <span>Rhizobium</span> meliloti. The EMBO journal. 1989;8: 1279–1286. doi:<a href="https://doi.org/10.1002/j.1460-2075.1989.tb03502.x">10.1002/j.1460-2075.1989.tb03502.x</a></div>
</div>
<div id="ref-irvine_lactobacillus_1993" class="csl-entry">
<div class="csl-left-margin">201. </div><div class="csl-right-inline">Irvine AS, Guest JR. Lactobacillus casei contains a member of the <span>CRP</span>-<span>FNR</span> family. Nucleic Acids Research. 1993;21: 753. doi:<a href="https://doi.org/10.1093/nar/21.3.753">10.1093/nar/21.3.753</a></div>
</div>
<div id="ref-tomura_nitric_1994" class="csl-entry">
<div class="csl-left-margin">202. </div><div class="csl-right-inline">Tomura D, Obika K, Fukamizu A, Shoun H. Nitric oxide reductase cytochrome <span>P</span>-450 gene, <span>CYP</span> 55, of the fungus <span>Fusarium</span> oxysporum containing a potential binding-site for <span>FNR</span>, the transcription factor involved in the regulation of anaerobic growth of <span>Escherichia</span> coli. Journal of Biochemistry. 1994;116: 88–94. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a124508">10.1093/oxfordjournals.jbchem.a124508</a></div>
</div>
<div id="ref-anthamatten_characterization_1992" class="csl-entry">
<div class="csl-left-margin">203. </div><div class="csl-right-inline">Anthamatten D, Scherb B, Hennecke H. Characterization of a <span class="nocase">fixLJ</span>-regulated <span>Bradyrhizobium</span> japonicum gene sharing similarity with the <span>Escherichia</span> coli fnr and <span>Rhizobium</span> meliloti <span class="nocase">fixK</span> genes. Journal of Bacteriology. 1992;174: 2111–2120. doi:<a href="https://doi.org/10.1128/jb.174.7.2111-2120.1992">10.1128/jb.174.7.2111-2120.1992</a></div>
</div>
<div id="ref-tosques_cloning_1996" class="csl-entry">
<div class="csl-left-margin">204. </div><div class="csl-right-inline">Tosques IE, Shi J, Shapleigh JP. Cloning and characterization of <span class="nocase">nnrR</span>, whose product is required for the expression of proteins involved in nitric oxide metabolism in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1996;178: 4958–4964. doi:<a href="https://doi.org/10.1128/jb.178.16.4958-4964.1996">10.1128/jb.178.16.4958-4964.1996</a></div>
</div>
<div id="ref-winteler_homologous_1996" class="csl-entry">
<div class="csl-left-margin">205. </div><div class="csl-right-inline">Winteler HV, Haas D. The homologous regulators <span>ANR</span> of <span>Pseudomonas</span> aeruginosa and <span>FNR</span> of <span>Escherichia</span> coli have overlapping but distinct specificities for anaerobically inducible promoters. Microbiology (Reading, England). 1996;142 ( Pt 3): 685–693. doi:<a href="https://doi.org/10.1099/13500872-142-3-685">10.1099/13500872-142-3-685</a></div>
</div>
<div id="ref-cuypers_anaerobic_1993" class="csl-entry">
<div class="csl-left-margin">206. </div><div class="csl-right-inline">Cuypers H, Zumft WG. Anaerobic control of denitrification in <span>Pseudomonas</span> stutzeri escapes mutagenesis of an fnr-like gene. Journal of Bacteriology. 1993;175: 7236–7246. doi:<a href="https://doi.org/10.1128/jb.175.22.7236-7246.1993">10.1128/jb.175.22.7236-7246.1993</a></div>
</div>
<div id="ref-stock_protein_1989" class="csl-entry">
<div class="csl-left-margin">207. </div><div class="csl-right-inline">Stock JB, Ninfa AJ, Stock AM. Protein phosphorylation and regulation of adaptive responses in bacteria. Microbiological Reviews. 1989;53: 450–490. doi:<a href="https://doi.org/10.1128/mr.53.4.450-490.1989">10.1128/mr.53.4.450-490.1989</a></div>
</div>
<div id="ref-iuchi_arcb_1990" class="csl-entry">
<div class="csl-left-margin">208. </div><div class="csl-right-inline">Iuchi S, Matsuda Z, Fujiwara T, Lin EC. The <span class="nocase">arcB</span> gene of <span>Escherichia</span> coli encodes a sensor-regulator protein for anaerobic repression of the arc modulon. Molecular Microbiology. 1990;4: 715–727. doi:<a href="https://doi.org/10.1111/j.1365-2958.1990.tb00642.x">10.1111/j.1365-2958.1990.tb00642.x</a></div>
</div>
<div id="ref-iuchi_purification_1992" class="csl-entry">
<div class="csl-left-margin">209. </div><div class="csl-right-inline">Iuchi S, Lin EC. Purification and phosphorylation of the <span>Arc</span> regulatory components of <span>Escherichia</span> coli. Journal of Bacteriology. 1992;174: 5617–5623. doi:<a href="https://doi.org/10.1128/jb.174.17.5617-5623.1992">10.1128/jb.174.17.5617-5623.1992</a></div>
</div>
<div id="ref-unden_oxygen_1994" class="csl-entry">
<div class="csl-left-margin">210. </div><div class="csl-right-inline">Unden G, Becker S, Bongaerts J, Schirawski J, Six S. Oxygen regulated gene expression in facultatively anaerobic bacteria. Antonie Van Leeuwenhoek. 1994;66: 3–22. doi:<a href="https://doi.org/10.1007/BF00871629">10.1007/BF00871629</a></div>
</div>
<div id="ref-lynch_transcriptional_1996" class="csl-entry">
<div class="csl-left-margin">211. </div><div class="csl-right-inline">Lynch AS, Lin EC. Transcriptional control mediated by the <span>ArcA</span> two-component response regulator protein of <span>Escherichia</span> coli: Characterization of <span>DNA</span> binding at target promoters. Journal of Bacteriology. 1996;178: 6238–6249. doi:<a href="https://doi.org/10.1128/jb.178.21.6238-6249.1996">10.1128/jb.178.21.6238-6249.1996</a></div>
</div>
<div id="ref-tseng_effect_1996" class="csl-entry">
<div class="csl-left-margin">212. </div><div class="csl-right-inline">Tseng CP, Albrecht J, Gunsalus RP. Effect of microaerophilic cell growth conditions on expression of the aerobic (<span class="nocase">cyoABCDE</span> and <span class="nocase">cydAB</span>) and anaerobic (<span class="nocase">narGHJI</span>, <span class="nocase">frdABCD</span>, and <span class="nocase">dmsABC</span>) respiratory pathway genes in <span>Escherichia</span> coli. Journal of Bacteriology. 1996;178: 1094–1098. doi:<a href="https://doi.org/10.1128/jb.178.4.1094-1098.1996">10.1128/jb.178.4.1094-1098.1996</a></div>
</div>
<div id="ref-stewart_nitrate_1993" class="csl-entry">
<div class="csl-left-margin">213. </div><div class="csl-right-inline">Stewart V. Nitrate regulation of anaerobic respiratory gene expression in <span>Escherichia</span> coli. Molecular Microbiology. 1993;9: 425–434. doi:<a href="https://doi.org/10.1111/j.1365-2958.1993.tb01704.x">10.1111/j.1365-2958.1993.tb01704.x</a></div>
</div>
<div id="ref-rabin_dual_1993" class="csl-entry">
<div class="csl-left-margin">214. </div><div class="csl-right-inline">Rabin RS, Stewart V. Dual response regulators (<span>NarL</span> and <span>NarP</span>) interact with dual sensors (<span>NarX</span> and <span>NarQ</span>) to control nitrate- and nitrite-regulated gene expression in <span>Escherichia</span> coli <span>K</span>-12. Journal of Bacteriology. 1993;175: 3259–3268. doi:<a href="https://doi.org/10.1128/jb.175.11.3259-3268.1993">10.1128/jb.175.11.3259-3268.1993</a></div>
</div>
<div id="ref-cavicchioli_role_1996" class="csl-entry">
<div class="csl-left-margin">215. </div><div class="csl-right-inline">Cavicchioli R, Chiang RC, Kalman LV, Gunsalus RP. Role of the periplasmic domain of the <span>Escherichia</span> coli <span>NarX</span> sensor-transmitter protein in nitrate-dependent signal transduction and gene regulation. Molecular Microbiology. 1996;21: 901–911. doi:<a href="https://doi.org/10.1046/j.1365-2958.1996.491422.x">10.1046/j.1365-2958.1996.491422.x</a></div>
</div>
<div id="ref-tyson_nitrite_1994" class="csl-entry">
<div class="csl-left-margin">216. </div><div class="csl-right-inline">Tyson KL, Cole JA, Busby SJ. Nitrite and nitrate regulation at the promoters of two <span>Escherichia</span> coli operons encoding nitrite reductase: Identification of common target heptamers for both <span>NarP</span>- and <span>NarL</span>-dependent regulation. Molecular Microbiology. 1994;13: 1045–1055. doi:<a href="https://doi.org/10.1111/j.1365-2958.1994.tb00495.x">10.1111/j.1365-2958.1994.tb00495.x</a></div>
</div>
<div id="ref-zennaro_nitrite_1993" class="csl-entry">
<div class="csl-left-margin">217. </div><div class="csl-right-inline">Zennaro E, Ciabatti I, Cutruzzola F, D’Alessandro R, Silvestrini MC. The nitrite reductase gene of <span>Pseudomonas</span> aeruginosa: Effect of growth conditions on the expression and construction of a mutant by gene disruption. FEMS microbiology letters. 1993;109: 243–250. doi:<a href="https://doi.org/10.1016/0378-1097(93)90027-y">10.1016/0378-1097(93)90027-y</a></div>
</div>
<div id="ref-korner_expression_1989" class="csl-entry">
<div class="csl-left-margin">218. </div><div class="csl-right-inline">Körner H, Zumft WG. Expression of denitrification enzymes in response to the dissolved oxygen level and respiratory substrate in continuous culture of <span>Pseudomonas</span> stutzeri. Applied and Environmental Microbiology. 1989;55: 1670–1676. doi:<a href="https://doi.org/10.1128/AEM.55.7.1670-1676.1989">10.1128/AEM.55.7.1670-1676.1989</a></div>
</div>
<div id="ref-kaiser_overlapping_1997" class="csl-entry">
<div class="csl-left-margin">219. </div><div class="csl-right-inline">Kaiser M, Sawers G. Overlapping promoters modulate <span>Fnr</span>- and <span>ArcA</span>-dependent anaerobic transcriptional activation of the <span class="nocase">focApfl</span> operon in <span>Escherichia</span> coli. Microbiology (Reading, England). 1997;143 ( Pt 3): 775–783. doi:<a href="https://doi.org/10.1099/00221287-143-3-775">10.1099/00221287-143-3-775</a></div>
</div>
<div id="ref-bongaerts_transcriptional_1995" class="csl-entry">
<div class="csl-left-margin">220. </div><div class="csl-right-inline">Bongaerts J, Zoske S, Weidner U, Unden G. Transcriptional regulation of the proton translocating <span>NADH</span> dehydrogenase genes (<span class="nocase">nuoA</span>-<span>N</span>) of <span>Escherichia</span> coli by electron acceptors, electron donors and gene regulators. Molecular Microbiology. 1995;16: 521–534. doi:<a href="https://doi.org/10.1111/j.1365-2958.1995.tb02416.x">10.1111/j.1365-2958.1995.tb02416.x</a></div>
</div>
<div id="ref-cuypers_multiple_1995" class="csl-entry">
<div class="csl-left-margin">221. </div><div class="csl-right-inline">Cuypers H, Jürgen B, Zumft WG. Multiple <span class="nocase">nosZ</span> promoters and anaerobic expression of nos genes necessary for <span>Pseudomonas</span> stutzeri nitrous oxide reductase and assembly of its copper centers. Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression. 1995;1264: 183–190. doi:<a href="https://doi.org/10.1016/0167-4781(95)00128-4">10.1016/0167-4781(95)00128-4</a></div>
</div>
<div id="ref-zumft_diversity_1995" class="csl-entry">
<div class="csl-left-margin">222. </div><div class="csl-right-inline">Zumft W. Diversity of gene organization and enzymes of denitrification. Beijerinck <span>Centennial</span> <span>Microbial</span> <span>Physiology</span> and <span>Gene</span> <span>Regulation</span>: <span>Emerging</span> <span>Principles</span> and <span>Applications</span>. Den Haag: Delft University Press; 1995. pp. 122–123. Available: <a href="https://repository.tudelft.nl/islandora/object/uuid:e30b10c4-0fad-4517-8874-5b3208e18f6e">https://repository.tudelft.nl/islandora/object/uuid:e30b10c4-0fad-4517-8874-5b3208e18f6e</a></div>
</div>
<div id="ref-kawasaki_gene_1997" class="csl-entry">
<div class="csl-left-margin">223. </div><div class="csl-right-inline">Kawasaki S, Arai H, Kodama T, Igarashi Y. Gene cluster for dissimilatory nitrite reductase (nir) from <span>Pseudomonas</span> aeruginosa: Sequencing and identification of a locus for heme d1 biosynthesis. Journal of Bacteriology. 1997;179: 235–242. doi:<a href="https://doi.org/10.1128/jb.179.1.235-242.1997">10.1128/jb.179.1.235-242.1997</a></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-chapter02.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
