<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Materials and Methods | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Materials and Methods | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Materials and Methods | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="1-chapter01.html"/>
<link rel="next" href="3-chapter03.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter02" class="section level1" number="2">
<h1><span class="header-section-number">Chapter 2</span> Materials and Methods</h1>
<div id="bacterial-strains" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Bacterial strains</h2>
<p>The following bacterial strains were used in this study.</p>
<table>
<caption>
<span id="tab:table21">Table 2.1: </span>Bacterial strains
</caption>
<thead>
<tr>
<th style="text-align:left;">
Organism
</th>
<th style="text-align:left;">
Reference
</th>
<th style="text-align:left;">
Source
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<em>Thiosphaera pantotropha</em> L.M.D. 92.63
</td>
<td style="text-align:left;">
Thompson et al. (1997); Robertson and Kuenen (1983)<span class="citation"><sup><a href="#ref-thompson_variation_1997" role="doc-biblioref">222</a>,<a href="#ref-robertson_thiosphaera_1983" role="doc-biblioref">223</a></sup></span>
</td>
<td style="text-align:left;">
Lab stock
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pseudomonas stutzeri</em> A.T.C.C. 14405
</td>
<td style="text-align:left;">
Döhler et al. (1983)<span class="citation"><sup><a href="#ref-dohler_transfer_1987" role="doc-biblioref">224</a></sup></span>
</td>
<td style="text-align:left;">
NCIMB culture collection
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pseudomonas aeruginosa</em> PAO1
</td>
<td style="text-align:left;">
Holloway et al. (1979)<span class="citation"><sup><a href="#ref-holloway_chromosomal_1979" role="doc-biblioref">225</a></sup></span>
</td>
<td style="text-align:left;">
NCIMB culture collection
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Escherichia coli</em> JM83
</td>
<td style="text-align:left;">
Yanisch-Perron et al. (1985)<span class="citation"><sup><a href="#ref-yanisch-perron_improved_1985" role="doc-biblioref">226</a></sup></span>
</td>
<td style="text-align:left;">
Lab stock
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Escherichia coli</em> JM109
</td>
<td style="text-align:left;">
Yanisch-Perron et al. (1985)<span class="citation"><sup><a href="#ref-yanisch-perron_improved_1985" role="doc-biblioref">226</a></sup></span>
</td>
<td style="text-align:left;">
Lab stock
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Escherichia coli</em> DH5&amp;alpha;
</td>
<td style="text-align:left;">
Hanahan (1983)<span class="citation"><sup><a href="#ref-hanahan_studies_1983" role="doc-biblioref">227</a></sup></span>
</td>
<td style="text-align:left;">
Lab stock
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Escherichia coli</em> S17-1
</td>
<td style="text-align:left;">
Simon et al. (1983)<span class="citation"><sup><a href="#ref-simon_broad_1983" role="doc-biblioref">228</a></sup></span>
</td>
<td style="text-align:left;">
Lab stock
</td>
</tr>
</tbody>
</table>
<p>Tables of plasmids are given at the beginning of the chapter in which their construction is detailed.</p>
</div>
<div id="growth-media" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Growth media</h2>
<div id="luria-bertani-lb-medium" class="section level3" number="2.2.1">
<h3><span class="header-section-number">2.2.1</span> Luria-Bertani (LB) medium</h3>
<p>For routine aerobic growth, all strains were grown in LB medium.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">concentration (g l<span class="math inline">\(^{-1}\)</span>)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Tryptone</td>
<td align="right">10</td>
</tr>
<tr class="even">
<td align="left">Yeast extract</td>
<td align="right">5</td>
</tr>
<tr class="odd">
<td align="left">NaCl</td>
<td align="right">5</td>
</tr>
</tbody>
</table>
<p>50 ml cultures were grown in 250 ml flasks at 37 °C on a rotary shaker at 200-250 rpm.</p>
</div>
<div id="t.-pantotropha-anaerobic-minimal-medium" class="section level3" number="2.2.2">
<h3><span class="header-section-number">2.2.2</span> <em>T. pantotropha</em> anaerobic minimal medium</h3>
<p><em>T. pantotropha</em> was grown anaerobically, with nitrate (KNO<span class="math inline">\(_3\)</span>, 20 mM) as the terminal electron acceptor and acetate (potassium acetate, 20 mM) as the carbon source, using the medium of Robertson and Kuenen (1983).<span class="citation"><sup><a href="#ref-robertson_thiosphaera_1983" role="doc-biblioref">223</a></sup></span></p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Na<span class="math inline">\(_2\)</span>HPO<span class="math inline">\(_4\)</span></td>
<td align="right">55</td>
</tr>
<tr class="even">
<td align="left">KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span></td>
<td align="right">11</td>
</tr>
<tr class="odd">
<td align="left">NH<span class="math inline">\(_4\)</span>Cl</td>
<td align="right">6</td>
</tr>
<tr class="even">
<td align="left">MgSO<span class="math inline">\(_4\)</span>.7H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.4</td>
</tr>
</tbody>
</table>
<p>Trace elements solution (Vishniac and Santer, 1957)<span class="citation"><sup><a href="#ref-vishniac_thiobacilli_1957" role="doc-biblioref">229</a></sup></span> was added at 2 ml l<span class="math inline">\(^{-1}\)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration in stock trace elements solution (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Na<span class="math inline">\(_2\)</span>EDTA</td>
<td align="right">140</td>
</tr>
<tr class="even">
<td align="left">Zn<span class="math inline">\(_2\)</span>SO<span class="math inline">\(_4\)</span></td>
<td align="right">7.6</td>
</tr>
<tr class="odd">
<td align="left">CaCl<span class="math inline">\(_2\)</span>.2H<span class="math inline">\(_2\)</span>O</td>
<td align="right">37</td>
</tr>
<tr class="even">
<td align="left">MnCl<span class="math inline">\(_2\)</span>.4H<span class="math inline">\(_2\)</span>O</td>
<td align="right">25</td>
</tr>
<tr class="odd">
<td align="left">FeSO<span class="math inline">\(_4\)</span>.7H<span class="math inline">\(_2\)</span>O</td>
<td align="right">18</td>
</tr>
<tr class="even">
<td align="left">(NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_6\)</span>Mo<span class="math inline">\(_7\)</span>O<span class="math inline">\(_{24}\)</span>.4H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.9</td>
</tr>
<tr class="odd">
<td align="left">CuSO<span class="math inline">\(_4\)</span>.5H<span class="math inline">\(_2\)</span>O</td>
<td align="right">3.2</td>
</tr>
<tr class="even">
<td align="left">CoCl<span class="math inline">\(_2\)</span>.6H<span class="math inline">\(_2\)</span>O</td>
<td align="right">6.7</td>
</tr>
</tbody>
</table>
<p>Starter cultures of <em>T. pantotropha</em> were grown aerobically overnight at 37 °C. Anaerobic medium was then inoculated at a dilution of 1:100 and the cells were grown in completely filled bottles without shaking at 37 °C until the optical density at 650 nm (A<span class="math inline">\(_{650}\)</span>) was 0.8-0.9.</p>
</div>
<div id="minimal-succinate-medium" class="section level3" number="2.2.3">
<h3><span class="header-section-number">2.2.3</span> Minimal succinate medium</h3>
<p>For some experiments <em>T. pantotropha</em> was grown anaerobically in minimal succinate medium.<span class="citation"><sup><a href="#ref-burnell_reversibility_1975" role="doc-biblioref">230</a></sup></span></p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Sodium succinate</td>
<td align="right">20</td>
</tr>
<tr class="even">
<td align="left">KNO<span class="math inline">\(_3\)</span></td>
<td align="right">20</td>
</tr>
<tr class="odd">
<td align="left">KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span></td>
<td align="right">5</td>
</tr>
<tr class="even">
<td align="left">(NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_2\)</span>HPO<span class="math inline">\(_4\)</span></td>
<td align="right">4.5</td>
</tr>
<tr class="odd">
<td align="left">CaCl<span class="math inline">\(_2\)</span></td>
<td align="right">0.17</td>
</tr>
<tr class="even">
<td align="left">MgSO<span class="math inline">\(_4\)</span>.7H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.1</td>
</tr>
<tr class="odd">
<td align="left">FeNa.EDTA</td>
<td align="right">0.05</td>
</tr>
</tbody>
</table>
<p>Hoagland’s trace elements solution<span class="citation"><sup><a href="#ref-burnell_reversibility_1975" role="doc-biblioref">230</a></sup></span> was added at 100 <span class="math inline">\(\mu\)</span>l l<span class="math inline">\(^{-1}\)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration in stock trace elements solution (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">AlCl<span class="math inline">\(_3\)</span></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">KBr</td>
<td align="right">1.2</td>
</tr>
<tr class="odd">
<td align="left">LiCl</td>
<td align="right">3</td>
</tr>
<tr class="even">
<td align="left">MnCl<span class="math inline">\(_2\)</span>.2H<span class="math inline">\(_2\)</span>O</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">H<span class="math inline">\(_3\)</span>BO<span class="math inline">\(_3\)</span></td>
<td align="right">50</td>
</tr>
<tr class="even">
<td align="left">ZnCl<span class="math inline">\(_2\)</span></td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">CuCl<span class="math inline">\(_2\)</span></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">NiCl<span class="math inline">\(_2\)</span></td>
<td align="right">2</td>
</tr>
<tr class="odd">
<td align="left">CoCl<span class="math inline">\(_2\)</span></td>
<td align="right">2</td>
</tr>
<tr class="even">
<td align="left">KI</td>
<td align="right">0.84</td>
</tr>
<tr class="odd">
<td align="left">SnCl<span class="math inline">\(_2\)</span>.2H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.60</td>
</tr>
<tr class="even">
<td align="left">BaCl<span class="math inline">\(_2\)</span></td>
<td align="right">0.67</td>
</tr>
<tr class="odd">
<td align="left">Na<span class="math inline">\(_2\)</span>MoO<span class="math inline">\(_4\)</span></td>
<td align="right">0.67</td>
</tr>
<tr class="even">
<td align="left">NaVO<span class="math inline">\(_3\)</span>.H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.20</td>
</tr>
<tr class="odd">
<td align="left">Na<span class="math inline">\(_2\)</span>SeO<span class="math inline">\(_3\)</span></td>
<td align="right">0.80</td>
</tr>
</tbody>
</table>
<p>The medium was adjusted to pH 6.8. Starter cultures were grown aerobically overnight at 37 °C in minimal succinate medium with nitrate omitted. The anaerobic medium, with nitrate present, was then inoculated at a dilution of 1:100 and the cells grown in completely filled bottles without shaking at 37 °C until the A<span class="math inline">\(_{650}\)</span> was 1.2-1.3.</p>
</div>
<div id="ps.-aeruginosa-anaerobic-medium" class="section level3" number="2.2.4">
<h3><span class="header-section-number">2.2.4</span> <em>Ps. aeruginosa</em> anaerobic medium</h3>
<p><em>Ps. aeruginosa</em> PAO1 was grown anaerobically in LB medium (Section 2.2.1), supplemented with 50 mM KNO<span class="math inline">\(_3\)</span>. Overnight starter cultures were inoculated into fresh medium at a dilution of 1:100 and grown in completely filled bottles, without shaking, to an A<span class="math inline">\(_{650}\)</span> of 1.2.</p>
</div>
<div id="ps.-stutzeri-anaerobic-medium" class="section level3" number="2.2.5">
<h3><span class="header-section-number">2.2.5</span> <em>Ps. stutzeri</em> anaerobic medium</h3>
<p><em>Ps. stutzeri</em> ATCC 14405 was grown anaerobically using ACN medium.<span class="citation"><sup><a href="#ref-coyle_nitrous_1985" role="doc-biblioref">85</a></sup></span></p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">L-asparagine</td>
<td align="right">13.3</td>
</tr>
<tr class="even">
<td align="left">Tri-sodium citrate</td>
<td align="right">23.8</td>
</tr>
<tr class="odd">
<td align="left">KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span></td>
<td align="right">14.7</td>
</tr>
<tr class="even">
<td align="left">CaCl<span class="math inline">\(_2\)</span>.6H<span class="math inline">\(_2\)</span>O</td>
<td align="right">0.46</td>
</tr>
<tr class="odd">
<td align="left">MgSO<span class="math inline">\(_4\)</span></td>
<td align="right">8.2</td>
</tr>
<tr class="even">
<td align="left">NaCl</td>
<td align="right">345</td>
</tr>
<tr class="odd">
<td align="left">KNO<span class="math inline">\(_3\)</span></td>
<td align="right">9.9</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Trace metals</th>
<th align="right">Concentration (<span class="math inline">\(\mu\)</span>M)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">CuCl<span class="math inline">\(_2\)</span>:2H<span class="math inline">\(_2\)</span>O</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">FeCl<span class="math inline">\(_3\)</span></td>
<td align="right">70</td>
</tr>
</tbody>
</table>
<p>Small cultures (50 ml in 250 ml flasks) were inoculated with a starter culture and grown overnight at 30 °C, on a rotary shaker at 100 rpm. This lowered the oxygen tension sufficiently to allow expression of denitrifying enzymes. Large cultures (20 l) were inoculated with 1 l of late-exponential culture and grown for 5 hours at 30 °C, whilst
sparging with air at a rate of 2-3 1 min<span class="math inline">\(^{-1}\)</span>, then for 6 hours with no aeration. This cycle was repeated once, giving a total growth time of 22 hours, after which the cells were harvested at an A<span class="math inline">\(_{650}\)</span> of around 0.9.</p>
</div>
<div id="e.-coli-anaerobic-minimal-medium" class="section level3" number="2.2.6">
<h3><span class="header-section-number">2.2.6</span> <em>E. coli</em> anaerobic minimal medium</h3>
<p><em>E. coli</em> JM83 was grown anaerobically using a medium based on that described by Pope and Cole (1982).<span class="citation"><sup><a href="#ref-pope_generation_1982" role="doc-biblioref">231</a></sup></span></p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span></td>
<td align="right">33</td>
</tr>
<tr class="even">
<td align="left">K<span class="math inline">\(_2\)</span>HPO<span class="math inline">\(_4\)</span></td>
<td align="right">46</td>
</tr>
<tr class="odd">
<td align="left">trisodium citrate</td>
<td align="right">1.7</td>
</tr>
<tr class="even">
<td align="left">(NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_2\)</span>SO<span class="math inline">\(_4\)</span></td>
<td align="right">7.6</td>
</tr>
<tr class="odd">
<td align="left">Fumaric acid pH 7.0</td>
<td align="right">40</td>
</tr>
<tr class="even">
<td align="left">NaNO<span class="math inline">\(_2\)</span></td>
<td align="right">5</td>
</tr>
</tbody>
</table>
<table>
<thead>
<tr class="header">
<th align="left">Trace metals</th>
<th align="right">Concentration (<span class="math inline">\(\mu\)</span>M)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Na<span class="math inline">\(_2\)</span>SeO<span class="math inline">\(_4\)</span>.10H<span class="math inline">\(_2\)</span>O</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Na<span class="math inline">\(_2\)</span>MoO<span class="math inline">\(_4\)</span>.2H<span class="math inline">\(_2\)</span>O</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
<p>Sulphur-free trace metals solution<span class="citation"><sup><a href="#ref-cole_nitrite_1974" role="doc-biblioref">232</a></sup></span> was added at 1 ml l<span class="math inline">\(^{-1}\)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="left"></th>
<th align="right">Concentration in stock trace elements solution (mM)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">MgCl<span class="math inline">\(_2\)</span>:6H<span class="math inline">\(_2\)</span>O</td>
<td align="right">403</td>
</tr>
<tr class="even">
<td align="left">MnCl<span class="math inline">\(_2\)</span>:4H<span class="math inline">\(_2\)</span>O</td>
<td align="right">50.5</td>
</tr>
<tr class="odd">
<td align="left">FeCl<span class="math inline">\(_3\)</span>.6H<span class="math inline">\(_2\)</span>O</td>
<td align="right">14.8</td>
</tr>
<tr class="even">
<td align="left">CaCl<span class="math inline">\(_2\)</span>.6H<span class="math inline">\(_2\)</span>O</td>
<td align="right">4.6</td>
</tr>
</tbody>
</table>
<p>The medium also contained 0.4% v/v glycerol and 5% v/v Luria-Bertani medium (Section 2.2.1). Overnight starter cultures grown aerobically in LB medium were inoculated at a dilution of 1:100 and the cultures were incubated at 37 °C in completely filled bottles without shaking.</p>
</div>
<div id="growth-on-solid-medium" class="section level3" number="2.2.7">
<h3><span class="header-section-number">2.2.7</span> Growth on solid medium</h3>
<p>Strains were grown on plates containing the media described above using the addition of 1.5% w/v bacteriological agar.</p>
</div>
<div id="antibiotics-and-other-selective-additions-to-media" class="section level3" number="2.2.8">
<h3><span class="header-section-number">2.2.8</span> Antibiotics and other selective additions to media</h3>
<p>Antibiotics were used to select for cells carrying plasmids or to counter-select for one organism against another, at the following concentrations (in <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>): ampicillin (100), kanamycin (25 for <em>E. coli</em>, 100 for <em>T. pantotropha</em>), rifampicin (100), spectinomycin (25 for <em>E. coli</em>, 100 for T. pantotropha), streptomycin (50 for <em>E. coli</em>, 100 for <em>T. pantotropha</em>), carbenicillin (250). Lead nitrate was also used at a concentration of 1 mM to select for <em>T. pantotropha</em> following conjugative transfer of plasmids from <em>E. coli</em> S17-1 (Section 2.5.20). <em>Ps. aeruginosa</em> was selected for following the same procedure using <em>Pseudomonas</em> selective medium supplements (Oxoid). Plates used for this purpose contained 200 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> of cetrimide and 15 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> of sodium nalidixate.</p>
<p>For blue-white selection of plasmids by <span class="math inline">\(\alpha\)</span>-complementation in <em>E. coli</em>, isopropylthio-<span class="math inline">\(\beta\)</span>-D-galactoside (IPTG) and 5-bromo-4-chloro-3-indolyl-<span class="math inline">\(\beta\)</span>-D-galactoside (X-Gal) were used at concentrations of 1 mM and 100 mM, respectively. IPTG was also used at concentrations of 0.5-1 mM to induce transcription from plasmids containing the <em>tac</em> promoter.<span class="citation"><sup><a href="#ref-de_boer_tac_1983" role="doc-biblioref">233</a></sup></span></p>
</div>
</div>
<div id="protein-biochemistry-techniques" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Protein biochemistry techniques</h2>
<div id="preparation-of-total-soluble-extracts" class="section level3" number="2.3.1">
<h3><span class="header-section-number">2.3.1</span> Preparation of total soluble extracts</h3>
<p>Soluble extracts for protein analysis were prepared from cells by sonication or by a freeze-thaw lysozyme procedure.<span class="citation"><sup><a href="#ref-berks_purification_1993" role="doc-biblioref">94</a></sup></span> For sonication, cells were grown to mid-exponential phase, harvested by centrifugation and resuspended in 50 mM Tris-HCl pH 8.0. The cell suspension was sonicated in an ice bath by 12 rounds of sonication (MSE sonicator tuned to 7-8 <span class="math inline">\(\mu\)</span>m amplitude) for a duration of 30 seconds, separated by 30 second pauses to prevent overheating. Broken cells were then ultracentrifuged for 45 minutes at 183 000 g, 4 °C, to pellet membrane material. Soluble extracts were also prepared from <em>Ps. aeruginosa</em> using a freeze-thaw lysozyme procedure. Following centrifugation, the harvested cells were resuspended in 5 ml of Tris-HCl, pH 8.0, per litre of original culture and lysozyme was added to a concentration of 1 mg ml<span class="math inline">\(^{-1}\)</span>. The sample was incubated for 10 minutes at 37 °C, then frozen at -70 °C for 30 minutes. The cells were then thawed at room temperature and the cycle was repeated once more. The sample was incubated with a few crystals of DNase for 10 minutes at 37 °C, then centrifuged at 40 000 g for 30 minutes at 4 °C. The supernatant was filtered once through Whatman 3MM paper and used as the final total soluble extract.</p>
</div>
<div id="preparation-of-periplasmic-extracts" class="section level3" number="2.3.2">
<h3><span class="header-section-number">2.3.2</span> Preparation of periplasmic extracts</h3>
<p>Periplasmic fractions were prepared from cells of <em>T. pantotropha</em> by the method of spheroplasting. Cells were grown to late exponential phase, harvested by centrifugation and resuspended in 20 ml per litre of original culture of 0.5 M sucrose, 50 mM Tris-HCl pH 8.0, 3 mM EDTA. Lysozyme was added to a concentration of 1 mg ml<span class="math inline">\(^{-1}\)</span> and the mixture was incubated at 37 °C for 30 minutes. The mixture was then centrifuged at 18 000 g for 30 minutes, producing a pellet of spheroplasts and a supernatant containing periplasmic proteins. <em>Ps. aeruginosa</em> was spheroplasted in a similar way, except EDTA was omitted and the incubation at 37 °C was for 1 hour.</p>
</div>
<div id="protein-purification" class="section level3" number="2.3.3">
<h3><span class="header-section-number">2.3.3</span> Protein purification</h3>
<p>All protein purification was carried out at 4 °C. Column sizes, loading capacity and flow rates were determined according to information in Scopes (1987).<span class="citation"><sup><a href="#ref-scopes_protein_1987" role="doc-biblioref">234</a></sup></span> Flow rates were controlled with a peristaltic pump (P-1, Pharmacia Biotech) and fractions were collected using a RediFrac fraction collector (Pharmacia Biotech).</p>
<div id="ion-exchange-chromatography" class="section level4" number="2.3.3.1">
<h4><span class="header-section-number">2.3.3.1</span> Ion exchange chromatography</h4>
<p>Ion-exchange chromatography was employed not only as a first step in the purification of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Thiosphaera pantotropha</em> but also as a general first step in all protein purifications. Extracts for purification were prepared in 50 mM Tris-HCl buffer, pH 8.0 at 4 °C. Columns of sizes 1 x 10 cm and 2.6 x 40 cm were generally used. The columns were packed with DEAE-Sepharose CL6B (Pharmacia Biotech) and equilibrated with 50 mM Tris-HCl, pH 8.0 (start buffer). Extract was loaded onto the column and non-specifically bound material was washed through with 3 column volumes of start buffer. Fractions were eluted using a linear gradient of 0-400 mM NaCl in 50 mM Tris-HCl pH 8.0, using a volume and fraction size appropriate to the column.</p>
</div>
<div id="hydrophobic-interaction-chromatography" class="section level4" number="2.3.3.2">
<h4><span class="header-section-number">2.3.3.2</span> Hydrophobic interaction chromatography</h4>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was further purified using hydrophobic interaction chromatography on Phenyl-Sepharose CL-4B columns (Pharmacia Biotech). Protein extracts in 50 mM Tris HCl pH 8.0 were brought to 40% saturation with (NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_2\)</span>SO<span class="math inline">\(_4\)</span> using the formula of Scopes (1987) and gently stirred at 4 °C. Columns of 1 x 10 cm were packed and equilibrated in 50 mM Tris-HCl pH 8.0, 40% saturated (NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_2\)</span>SO<span class="math inline">\(_4\)</span> (start buffer). Samples were loaded and washed with 3 volumes of this start buffer. Proteins were eluted with a linear gradient of 40-0% saturation (NH<span class="math inline">\(_4\)</span>)<span class="math inline">\(_2\)</span>SO<span class="math inline">\(_4\)</span> in 50 mM Tris-HCl pH 8.0.</p>
</div>
<div id="size-exclusion-chromatography-gel-filtration" class="section level4" number="2.3.3.3">
<h4><span class="header-section-number">2.3.3.3</span> Size exclusion chromatography (gel filtration)</h4>
<p>Proteins were further purified by gel filtration using columns of Sephacryl S-200 HR (Pharmacia Biotech). Columns of 2.6 x 100 cm were packed and equilibrated with 20 mM Tris-HCl pH 8.0, 100 mM NaCl according to the manufacturer’s instructions and run at 10-20 ml h<span class="math inline">\(^{-1}\)</span>. Samples were concentrated to about 4 ml before application, using an Amicon ultrafiltration cell with a 10 kDa cut-off membrane.</p>
</div>
</div>
<div id="sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page" class="section level3" number="2.3.4">
<h3><span class="header-section-number">2.3.4</span> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</h3>
<p>Protein samples were analysed in whole cell extracts or during purification using size fractionation by SDS-PAGE. Gels of an appropriate percentage acrylamide (from a stock of acrylamide plus N, N’-methylenebisacrylamide in the ratio 30:0.8) were prepared according to Sambrook et al. (1989)<span class="citation"><sup><a href="#ref-sambrook_molecular_1989" role="doc-biblioref">235</a></sup></span> and cast using a Mini Protean II apparatus (BioRad). Stacking gels contained 5% acrylamide, 130 mM Tris-HCl pH 6.8 and 0.1% w/v SDS. Resolving gels contained 6-15% acrylamide (depending on the range of separation required), 375 mM Tris HCl pH 8.8 and 0.1% w/v SDS. Polymerisation of gels was initiated with 0.1% w/v ammonium persulphate and 0.1% v/v TEMED (N, N, N’, N’-tetramethylethylenediamine). Samples at an appropriate concentration were denatured by boiling for 5 minutes with 5 <span class="math inline">\(\mu\)</span>l of sample buffer (50 mM Tris-HCl pH 6.8, 100 mM dithiothreitol, 2% w/v SDS, 0.1% w/v bromophenol blue, 10% glycerol), in a total volume of 20 <span class="math inline">\(\mu\)</span>l, and separated at 180 V for 45 minutes. Electrophoresis buffer contained 25 mM Tris, 250 mM glycine pH 8.3, 0.1% w/v SDS.</p>
<div id="coomassie-blue-stain-for-proteins" class="section level4" number="2.3.4.1">
<h4><span class="header-section-number">2.3.4.1</span> Coomassie Blue stain for proteins</h4>
<p>After electrophoresis, total protein was detected on gels by staining with Coomassie Brilliant Blue R250 (0.5% w/v), in 50% v/v methanol, 10% v/v acetic acid, for 1 hour at room temperature. Gels were destained with 30% v/v methanol, 10% v/v acetic acid, until the background was clear. For storage, gels were either vacuum-dried on filter paper at 80 °C, after soaking in 10% v/v glycerol for several hours, or dried between cellulose sheets using the Promega gel drying system.</p>
</div>
<div id="haem-stain" class="section level4" number="2.3.4.2">
<h4><span class="header-section-number">2.3.4.2</span> Haem stain</h4>
<p>Following separation by SDS-PAGE, <em>c</em>-type cytochromes were stained for haem using the method of Goodhew et al. (1986).<span class="citation"><sup><a href="#ref-goodhew_cytochromes_1996" role="doc-biblioref">236</a></sup></span> Before electrophoresis, protein samples were denatured in sample loading buffer without dithiothreitol, as this reduces the intensity of the stain. The gel was soaked in 70 ml of 0.25 M sodium acetate, pH 5.2, for 30 minutes. 30 mg of 3, 3, 5, 5°-tetramethylbenzidine dissolved in 30 ml of methanol was added and the gel was incubated for a further 30 minutes. Staining was initiated by the addition of 0.3 ml of 30% v/v hydrogen peroxide and the gel was incubated with gentle shaking in the dark. After full development, the gel was washed twice in 50 ml of 70% v/v 0.25 M sodium acetate pH 5.2, 30% v/v isopropanol and stored in water in the dark.</p>
</div>
<div id="western-blotting" class="section level4" number="2.3.4.3">
<h4><span class="header-section-number">2.3.4.3</span> Western blotting</h4>
<p>Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Thiosphaera pantotropha</em> was detected immunologically after SDS PAGE by Western blotting. Protein extracts were separated on 6% acrylamide gels and transferred to nitrocellulose membranes using semi-dry transfer buffer (50 mM Tris, 38 mM glycine pH 8.3, 0.04% w/v SDS, 20% v/v methanol) as an electrolyte in a Pharmacia LKB Novablot electroblotting apparatus. After transfer, the gel was blocked for 1 hour in PBS-A-T buffer (8.1 mM Na<span class="math inline">\(_2\)</span>HPO<span class="math inline">\(_4\)</span>, 1.5 mM KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span> pH 7.4, 140 mM NaCl, 7.7 mM NaN<span class="math inline">\(_3\)</span>, 0.3% v/v Tween-20) plus 1% w/v skimmed milk powder (blocking buffer). The membrane was washed in PBS-A-T plus 0.3% w/v skimmed milk powder (wash buffer), then incubated for two hours in blocking buffer containing mouse anti-cytochrome <em>cd</em><span class="math inline">\(_1\)</span> antibody (obtained from Thon de Boer, Vrije Universiteit, Amsterdam), at a dilution of 1/1000. The membrane was washed three times for five minutes each time with wash buffer, then incubated for two hours with anti-rabbit antibody-alkaline phosphatase conjugate, also diluted to 1/1000. After three more five minute washes in wash buffer, the membrane was washed four times with PBS-A-T buffer and equilibrated for 1 minute in TBS buffer (100 mM Tris-HCl pH 9.5, 100 mM NaCl). Bound antibody was detected colorimetrically by incubating the membrane in TBS buffer containing 10 mM MgCl<span class="math inline">\(_2\)</span>, 200 <span class="math inline">\(\mu\)</span>M nitroblue tetrazolium and 20 <span class="math inline">\(\mu\)</span>M 5-bromo-4 chloro-3-indolyl phosphate. Development was stopped by several washes of distilled water and membranes were stored in TE buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA) in the dark.</p>
</div>
</div>
<div id="determination-of-protein-concentration" class="section level3" number="2.3.5">
<h3><span class="header-section-number">2.3.5</span> Determination of protein concentration</h3>
<p>Protein concentrations were estimated using the method of Bradford (1976)<span class="citation"><sup><a href="#ref-bradford_rapid_1976" role="doc-biblioref">237</a></sup></span> and the BioRad reagent kit. 20 <span class="math inline">\(\mu\)</span>l of diluted sample was mixed with 0.8 ml of distilled water and 0.2 ml of reagent and incubated at room temperature for 10 minutes. Absorbance at 595 nm was recorded and converted to protein concentration in mg ml<span class="math inline">\(^{-1}\)</span> using a standard curve prepared with known concentrations of bovine serum albumin.</p>
</div>
<div id="uv-visible-spectroscopy" class="section level3" number="2.3.6">
<h3><span class="header-section-number">2.3.6</span> UV-visible spectroscopy</h3>
<p>UV and visible spectra of protein samples were recorded on a Perkin Elmer Lambda 2 spectrophotometer, interfaced to a Dell 316SX PC running Perkin Elmer spectroscopy software.</p>
</div>
<div id="assays-for-enzyme-activity" class="section level3" number="2.3.7">
<h3><span class="header-section-number">2.3.7</span> Assays for enzyme activity</h3>
<div id="cytochrome-cd_1-nitrite-reductase-assay" class="section level4" number="2.3.7.1">
<h4><span class="header-section-number">2.3.7.1</span> Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase assay</h4>
<p>Nitrite reductase activity of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was routinely measured using reduced methyl viologen as an electron donor. Assays were performed in a 1 ml cuvette containing 50 mM Tris-HCl pH 8.0, 5 mM EDTA, 1 mM methyl viologen and an aliquot of sample. The cuvette was sealed with a rubber septum, evacuated and flushed with argon. 1-2 <span class="math inline">\(\mu\)</span>l of sodium dithionite (approximately 5 mg ml<span class="math inline">\(^{-1}\)</span>) was added to reduce the methyl viologen, giving a blue colour. After checking that the background rate of reduction was close to zero, the reaction was initiated by the addition of 10 mM KNO<span class="math inline">\(_2\)</span>. Oxidation of methyl viologen was followed by the decrease in absorbance at 600 nm over time. The rate was converted to a specific activity using the extinction coefficient for reduced methyl viologen at 600 nm (<span class="math inline">\(\epsilon_{600}\)</span>) of 13 mM<span class="math inline">\(^{-1}\)</span> cm<span class="math inline">\(^{-1}\)</span>.<span class="citation"><sup><a href="#ref-moir_aspects_1993" role="doc-biblioref">238</a></sup></span></p>
</div>
<div id="cytochrome-cd_1-cytochrome-oxidase-assay" class="section level4" number="2.3.7.2">
<h4><span class="header-section-number">2.3.7.2</span> Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> cytochrome oxidase assay</h4>
<p>The oxidase activity of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> was assayed using horse heart cytochrome <em>c</em> as an electron donor. To prepare pre-reduced cytochrome <em>c</em>, 35 mg of cytochrome <em>c</em> was dissolved in 2.5 ml of 200 mM KH<span class="math inline">\(_2\)</span>PO<span class="math inline">\(_4\)</span>, pH 7.4, that contained 1 mg ml<span class="math inline">\(^{-1}\)</span> of sodium dithionite. Excess sodium dithionite was removed by passage through a Sephadex PD-10 column, equilibrated with the same buffer. Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was assayed in a volume of 1 ml containing 50 mM Tris-HCl pH 8.0 and 25 <span class="math inline">\(\mu\)</span>M of reduced cytochrome <em>c</em>. The assay was started with the addition of an aliquot of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and the oxidation of the cytochrome <em>c</em> was followed by the decrease in its absorbance at 550 nm. A difference extinction coefficient of 18.5 mM<span class="math inline">\(^{-1}\)</span> cm<span class="math inline">\(^{-1}\)</span><span class="citation"><sup><a href="#ref-robinson_implications_1979" role="doc-biblioref">155</a></sup></span> was used to calculate the rate of activity.</p>
</div>
<div id="cytochrome-c-peroxidase-assay" class="section level4" number="2.3.7.3">
<h4><span class="header-section-number">2.3.7.3</span> Cytochrome <em>c</em> peroxidase assay</h4>
<p>Cytochrome <em>c</em> peroxidase from <em>T. pantotropha</em> was assayed using reduced horse heart cytochrome <em>c</em> as an electron donor, based on the method of Gilmour et al. (1994).<span class="citation"><sup><a href="#ref-gilmour_kinetics_1994" role="doc-biblioref">123</a></sup></span> Pre-reduced cytochrome <em>c</em> was prepared as described in the above section. Aliquots of the sample to be assayed were incubated in a 1 ml cuvette containing 50 mM Tris-HCl, pH 8.0, and 25 <span class="math inline">\(\mu\)</span>M reduced cytochrome <em>c</em>. The assay was initiated after 40-60 s by the addition of hydrogen peroxide to a final concentration of 18 <span class="math inline">\(\mu\)</span>M and the activity was followed by the decrease in absorbance at 550 nm, as described for the cytochrome oxidase assay.</p>
</div>
<div id="assay-of-nitrite-produced-by-in-vivo-nitrite-reductase-activity" class="section level4" number="2.3.7.4">
<h4><span class="header-section-number">2.3.7.4</span> Assay of nitrite produced by <em>in vivo</em> nitrite reductase activity</h4>
<p>Nitrite that had accumulated in the medium of <em>T. pantotropha</em> or <em>Ps. aeruginosa</em> cultures during denitrification was assayed colorimetrically using the method of Nicholas and Nason (1957).<span class="citation"><sup><a href="#ref-donald_nicholas_determination_1957" role="doc-biblioref">239</a></sup></span> 1 ml samples of the culture were centrifuged to pellet the cells and a 50 <span class="math inline">\(\mu\)</span>l aliquot of the supernatant (diluted where necessary) was mixed with 3.7 ml of 1% w/v sulphanilamide in 1 M HCl. 0.3 ml of 0.02% w/v N-(1-napthyl) ethylene diamine hydrochloride was added and the samples were incubated at room temperature for 20 minutes. The absorbance at 540 nm was recorded and the concentration of nitrite was calculated from a standard curve of known nitrite concentrations in the range 0-1 mM.</p>
</div>
</div>
<div id="chemical-preparation-of-semi-apo-cytochrome-cd_1" class="section level3" number="2.3.8">
<h3><span class="header-section-number">2.3.8</span> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></h3>
<p>Haem <em>d</em><span class="math inline">\(_1\)</span> was removed from cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, giving the <em>c</em>-haem containing semi-apo enzyme, by the method of Hill and Wharton (1978).<span class="citation"><sup><a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a></sup></span> 2 ml of purified protein (2 mg ml<span class="math inline">\(^{-1}\)</span>) in 50 mM Tris-HCl pH 8.0 was mixed with 18 ml of acetone containing 24 mM HCl. The solution was incubated at 37 °C for 20 minutes and centrifuged at 4000 g, 4 °C for 2 minutes. The red pellet of precipitated protein was washed twice with acetone/HCl and twice with acetone, then redissolved in 2 ml of 50 mM Tris-HCl pH 9.0, 6 M urea. This solution was dialysed overnight at 4 °C against 50 mM Tris-HCl pH 8.0. Samples were analysed by spectroscopy in the oxidised and reduced forms to confirm the presence of only <em>c</em>-type haem.</p>
</div>
<div id="purification-of-haem-d_1" class="section level3" number="2.3.9">
<h3><span class="header-section-number">2.3.9</span> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></h3>
<p>Haem <em>d</em><span class="math inline">\(_1\)</span> was purified during the preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, described above. After centrifugation of the semi-apo protein, 8 volumes of the green supernatant (containing haem <em>d</em><span class="math inline">\(_1\)</span>) was extracted with 1 volume of 1.2 M NaOH. Haem <em>d</em><span class="math inline">\(_1\)</span> dissolved in the lower aqueous layer and was removed with a pipette. The pH of the solution was adjusted to 7 with 1 M HCl and the solution was stored at 4 °C in the dark. The haem concentration was estimated from the absorbance of the pyridine haemochromogen at 620 nm using an extinction coefficient of 24 mM<span class="math inline">\(^{-1}\)</span> cm<span class="math inline">\(^{-1}\)</span>.<span class="citation"><sup><a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a></sup></span></p>
</div>
</div>
<div id="genetic-techniques" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Genetic techniques</h2>
<div id="dna-manipulation" class="section level3" number="2.4.1">
<h3><span class="header-section-number">2.4.1</span> DNA manipulation</h3>
<div id="organic-solvent-extraction-and-precipitation-of-nucleic-acids" class="section level4" number="2.4.1.1">
<h4><span class="header-section-number">2.4.1.1</span> Organic solvent extraction and precipitation of nucleic acids</h4>
<p>Nucleic acids were purified and concentrated after enzymatic manipulation by organic solvent extraction and ethanol precipitation. An equal volume of phenol:chloroform:isoamyl alcohol (25:24:1) was mixed with the sample by vortexing and the phases were separated by centrifugation at 15 000 g for two minutes. The upper aqueous phase was mixed with 0.1 volumes of 3 M sodium acetate pH 5.2 and 2.5 volumes of cold ethanol and incubated at -70 °C for 30 minutes. Nucleic acids were pelleted by centrifugation at 15 000 g for 15 minutes, then washed with 100 <span class="math inline">\(\mu\)</span>l of ice-cold 70% v/v ethanol by centrifuging for a further five minutes. The pellet was air-dried, then redissolved in TE buffer or sterile distilled water.</p>
</div>
<div id="purification-of-plasmid-dna" class="section level4" number="2.4.1.2">
<h4><span class="header-section-number">2.4.1.2</span> Purification of plasmid DNA</h4>
<p>Plasmid DNA was purified from bacteria by three methods, depending on the quality required for subsequent experiments. For restriction digests, analysis and cloning, the alkaline lysis miniprep procedure was used.<span class="citation"><sup><a href="#ref-birnboim_rapid_1979" role="doc-biblioref">240</a></sup></span> 1.5 ml of an overnight culture was harvested in a microcentrifuge and resuspended in 300 <span class="math inline">\(\mu\)</span>l of 25 mM Tris-HCl pH 8.0, 50 mM glucose, 10 mM EDTA. The cells were lysed by the addition of 600 <span class="math inline">\(\mu\)</span>l of 0.2 M NaOH, 1% w/v SDS. Protein and chromosomal DNA were then precipitated by the addition of 450 <span class="math inline">\(\mu\)</span>l of 3 M potassium acetate, 11.5 % v/v glacial acetic acid pH 4.8. The precipitate was removed by centrifugation at 15 000 g for 15 minutes. 800 <span class="math inline">\(\mu\)</span>l of supernatant was treated with 20 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> of DNase-free RNase A at 37 °C for 20 minutes. Plasmid DNA was precipitated with 0.8 volumes of isopropanol and pelleted by centrifugation at 15 000 g for 15 minutes. Finally, the pellet was washed with ice-cold 70% v/v ethanol by centrifugation for five minutes, drained, air-dried and redissolved in 50 <span class="math inline">\(\mu\)</span>l of TE buffer (10 mM Tris-HCl pH 8.0, 1 mM EDTA). Plasmid DNA was stored at -20 °C.</p>
<p>Plasmid DNA of higher purity for sequencing was purified by either PEG precipitation<span class="citation"><sup><a href="#ref-mierendorf_direct_1987" role="doc-biblioref">241</a></sup></span> or using the Wizard plasmid DNA purification kit (Promega). In the first method, the alkaline lysis procedure was followed up to the RNase A treatment, after which the sample was extracted once with 400 <span class="math inline">\(\mu\)</span>l of phenol:chloroform. After precipitation, washing and drying of the plasmid DNA, the pellet was redissolved in 16 <span class="math inline">\(\mu\)</span>l of sterile distilled water and thoroughly mixed with 4 <span class="math inline">\(\mu\)</span>l of 4 M NaCl and 20 <span class="math inline">\(\mu\)</span>l of 13% w/v polyethylene glycol (PEG 8000). The sample was incubated on ice for 30 minutes, then centrifuged at 15 000 g for 15 minutes to pellet the plasmid DNA. The pellet was then washed with 70% v/v ethanol, dried and redissolved in 20 <span class="math inline">\(\mu\)</span>l of sterile distilled water.</p>
<p>The Promega Wizard kit was used according to the manufacturer’s instructions. Cell lysis and the production of a cleared supernatant were performed much as described for the alkaline lysis miniprep. Plasmid DNA was then mixed with DNA binding resin, absorbed onto a minicolumn and washed with wash solution (80 mM potassium acetate, 8.3 mM Tris HCl pH 7.5, 40 <span class="math inline">\(\mu\)</span>M EDTA, 55% v/v ethanol). The purified plasmid DNA was eluted in 40-50 <span class="math inline">\(\mu\)</span>l of prewarmed (70 °C) TE buffer, by a brief centrifugation.</p>
</div>
<div id="restriction-digestion-of-dna" class="section level4" number="2.4.1.3">
<h4><span class="header-section-number">2.4.1.3</span> Restriction digestion of DNA</h4>
<p>Plasmid and chromosomal DNA was digested with restriction enzymes to allow cloning of DNA fragments and to analyse recombinant plasmids. Plasmid DNA was digested in a volume of 20 <span class="math inline">\(\mu\)</span>l (or 40 <span class="math inline">\(\mu\)</span>l for preparative digests), containing the appropriate restriction enzyme buffer (New England Biolabs), 5-10 units of restriction enzyme and 1 mg ml<span class="math inline">\(^{-1}\)</span> of bovine serum albumin where recommended, for one hour at the correct temperature (normally 37 °C, 25 °C for <em>SmaI</em>). Chromosomal DNA (usually 5-10 <span class="math inline">\(\mu\)</span>g) was digested for 4-8 hours. Digestion was stopped with 25 mM Na EDTA or by heating at the recommended inactivation temperature.</p>
</div>
<div id="conversion-of-cohesive-dna-ends-to-blunt-ends-using-t4-dna-polymerase" class="section level4" number="2.4.1.4">
<h4><span class="header-section-number">2.4.1.4</span> Conversion of cohesive DNA ends to blunt ends using T4 DNA polymerase</h4>
<p>Cohesive DNA ends produced by restriction digestion were converted to blunt ends for cloning into plasmids digested with “blunt cutters” such as <em>Smal</em> or <em>EcoRV</em>. After digestion and inactivation of the restriction enzyme, digests were extracted with an equal volume of phenol:chloroform and precipitated. The DNA was then redissolved in 20 <span class="math inline">\(\mu\)</span>l of sterile distilled water, containing T4 polymerase buffer (10 mM Tris-HCl pH 7.9, 50 mM NaCl, 10 mM MgCl<span class="math inline">\(_2\)</span>, 1 mM DTT, 50 <span class="math inline">\(\mu\)</span>g ml BSA), 100 <span class="math inline">\(\mu\)</span>M of each deoxynucleotide triphosphate and 3 units of T4 DNA polymerase (New England Biolabs). The mixture was incubated at 37 °C for five minutes, 15 °C for 20 minutes, then heat-inactivated at 75 °C for 10 minutes.</p>
</div>
<div id="dephosphorylation-of-plasmid-dna" class="section level4" number="2.4.1.5">
<h4><span class="header-section-number">2.4.1.5</span> Dephosphorylation of plasmid DNA</h4>
<p>Vector DNA was dephosphorylated before cloning to prevent religation of the vector in difficult clonings such as those without blue-white selection (Section 2.3.8), or into sites cut with a single blunt cutting enzyme. After digestion, the volume of the reaction mixture was increased to 200 <span class="math inline">\(\mu\)</span>l with sterile distilled water and shrimp alkaline phosphatase buffer (20 mM Tris-HCl pH 8.8, 10 mM MgCl<span class="math inline">\(_2\)</span>) and 2 units of shrimp alkaline phosphatase (United States Biochemical) was added. The mixture was incubated at 37 °C for 30 minutes, then at 65 °C for 20 minutes to inactivate the enzyme. DNA was recovered by extraction with an equal volume of phenol:chloroform, followed by ethanol precipitation.</p>
</div>
<div id="ligation-of-dna-fragments-into-plasmid-vectors" class="section level4" number="2.4.1.6">
<h4><span class="header-section-number">2.4.1.6</span> Ligation of DNA fragments into plasmid vectors</h4>
<p>DNA fragments with either blunt or cohesive termini were cloned into plasmid vectors using T4 DNA ligase. Ligations were performed in a volume of 20 <span class="math inline">\(\mu\)</span>l, containing ligase buffer (50 mM Tris-HCl pH 7.8, 10 mM MgCl<span class="math inline">\(_2\)</span>, 10 mM DTT, 1 mM ATP, 25 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> bovine serum albumin), 400 units of T4 DNA ligase and various proportions of vector and insert DNA, to generate the optimum number of clones containing an insert. Cohesive end ligations were usually incubated at 15 °C for one hour, then at 4 °C overnight. Blunt end ligations were usually incubated at 15 °C overnight. Ligation mixtures were stored at -20 °C until used for transformation.</p>
</div>
<div id="agarose-gel-electrophoresis" class="section level4" number="2.4.1.7">
<h4><span class="header-section-number">2.4.1.7</span> Agarose gel electrophoresis</h4>
<p>Restricted DNA was separated by size fractionation using agarose gel electrophoresis, for analysis of recombinant plasmids and separation of fragments for purification. After restriction, 0.1 volumes of sterile loading dye (10 mM EDTA, 50% v/v glycerol, 0.5% w/v bromophenol blue) was added. For analysis of digested plasmids, gels of 50 ml volume (106 x 78 x 6 mm) and 0.6-1.4% w/v agarose were most often used. These gels were run in TBE buffer (45 mM Tris-borate pH 8.3, 1 mM EDTA) at a constant current of 50-60 mA. For subsequent DNA purification, TAE buffer (40 mM Tris-acetate pH 8.0, 1 mM EDTA) was used instead. Ethidium bromide was added to the gel and the electrophoresis buffer at a concentration of 0.2 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> to stain the DNA. Stained gels were viewed and photographed under ultraviolet illumination.</p>
</div>
<div id="purification-of-dna-fragments-from-agarose-gels" class="section level4" number="2.4.1.8">
<h4><span class="header-section-number">2.4.1.8</span> Purification of DNA fragments from agarose gels</h4>
<p>DNA fragments were purified from TAE agarose gels for subsequent manipulation, using either the GeneClean II kit (BIO 101 Inc.) or the Wizard PCR prep kit (Promega). In the first procedure, the desired fragment was excised from the gel using a clean razor blade and dissolved in 1 ml of 6 M sodium iodide solution for five minutes at 50 °C. 5 <span class="math inline">\(\mu\)</span>l of Glassmilk (a DNA binding silica matrix) was added and the mixture was incubated on ice for 15 minutes. The Glassmilk was pelleted by a brief centrifugation, then washed three times in 1 ml of NEW Wash (an ethanol-based buffered salt solution) and pelleted again. The pellet was air-dried for 10 minutes, then the DNA was eluted in 10-20 <span class="math inline">\(\mu\)</span>l of TE buffer, by incubating at 50 °C for 2-3 minutes, pelleting the resin by centrifugation and retaining the supernatant.</p>
<p>Using the Promega Wizard PCR prep kit, the gel slice was dissolved in 1 ml of DNA binding resin by incubation at 50 °C for five minutes. The resin was absorbed onto a minicolumn and washed with 2 ml of 80 % v/v isopropanol. Finally, DNA was eluted with 40-50 <span class="math inline">\(\mu\)</span>l of prewarmed TE buffer (70 °C) by a brief centrifugation. The yield of purified
DNA fragment was analysed by agarose gel electrophoresis in TBE buffer.</p>
</div>
<div id="transformation-of-competent-e.-coli-with-plasmid-dna" class="section level4" number="2.4.1.9">
<h4><span class="header-section-number">2.4.1.9</span> Transformation of competent <em>E. coli</em> with plasmid DNA</h4>
<p><em>E. coli</em> strains were transformed with plasmid DNA by two methods: chemical transformation using CaCl<span class="math inline">\(_2\)</span><span class="citation"><sup><a href="#ref-cohen_nonchromosomal_1972" role="doc-biblioref">242</a></sup></span> and electroporation. For chemical transformations, an overnight culture of <em>E. coli</em> was reinoculated in 50 ml of LB medium at a dilution of 1/100 and grown to an absorbance at 650 nm of 0.3-0.5. The cells were harvested by centrifugation, resuspended in 10 ml of 0.1 M MgCl<span class="math inline">\(_2\)</span> and incubated on ice for 15 minutes. Cells were then harvested once more, resuspended in 1 ml of 0.1 M CaCl<span class="math inline">\(_2\)</span> and incubated on ice for one hour. At this stage, the competent cells could be stored at -70 °C in 25% v/v glycerol for several months. To transform the cells, a 100 <span class="math inline">\(\mu\)</span>l aliquot was mixed with 10 <span class="math inline">\(\mu\)</span>l of a ligation reaction, or 1-2 <span class="math inline">\(\mu\)</span>l of a plasmid purified by the alkaline lysis miniprep protocol, and incubated on ice for a further 45 minutes. The cells were then heat-shocked at 42 °C for 30 seconds in thin-walled polypropylene tubes, placed on ice and mixed with 1 ml of LB medium. Cells were left to recover for 90-120 minutes, with shaking at 37 °C, before plating out on a medium selective for the presence of the plasmid.</p>
<p>For electroporation, <em>E. coli</em> was grown to an A<span class="math inline">\(_{650}\)</span> of 0.5-1.0 and harvested by centrifugation. The pellet was then repeatedly resuspended and harvested in decreasing volumes of ice-cold sterile water in the following order: 1 volume, 0.5 volumes, 0.5 volumes and 0.02 volumes. After the final wash the cells were harvested once more and resuspended in 0.02 volumes of sterile 10% v/v ice-cold glycerol. 40 <span class="math inline">\(\mu\)</span>l aliquots of cells were mixed with 1 <span class="math inline">\(\mu\)</span>l of ligation mix (diluted 1:4 in sterile water) in a 1 ml electroporation cuvette (BioRad). The cells were given a pulse of 2500 V in a BioRad E. coli Pulser, mixed with 1 ml of LB and left to recover at 37 °C for one hour. Several dilutions were plated out on medium selective for the presence of the plasmid.</p>
</div>
<div id="transformation-of-gram-negative-bacteria-by-mating-with-e.-coli-s17-1" class="section level4" number="2.4.1.10">
<h4><span class="header-section-number">2.4.1.10</span> Transformation of Gram-negative bacteria by mating with <em>E. coli</em> S17-1</h4>
<p><em>E. coli</em> S17-1 has the <em>tra</em> genes of the RP4 plasmid integrated into its chromosome, allowing the transfer of plasmids that contain an origin of transfer (Mob site) to a recipient Gram-negative bacterium via a sex pilus.<span class="citation"><sup><a href="#ref-simon_broad_1983" role="doc-biblioref">228</a></sup></span> Donor <em>E. coli</em> cells containing the plasmid and the recipient strain were both grown to an A<span class="math inline">\(_{650}\)</span> of 0.5-1.0. The cells were harvested by centrifugation, washed twice with LB medium and harvested in 0.3 ml of LB. Harvested cells were then mixed, pipetted onto a sterile nitrocellulose filter on an LB plate and incubated overnight at 37 °C. The mated cells were harvested from the filter in 0.8 ml of LB. Serial dilutions of the mating mix were then plated onto medium which was selective for both the plasmid and the recipient strain and incubated at 37 °C until single colonies appeared. These were isolated and further analysed to confirm the presence of the plasmid and that the colonies were of the recipient bacterium.</p>
</div>
<div id="preparation-of-chromosomal-dna" class="section level4" number="2.4.1.11">
<h4><span class="header-section-number">2.4.1.11</span> Preparation of chromosomal DNA</h4>
<p>Chromosomal DNA from <em>T. pantotropha</em> was isolated by two methods, depending on the yield required. For large-scale isolation, 200 ml cultures were grown aerobically overnight on LB medium. The culture was divided into 50 ml portions and harvested by centrifugation. Each pellet was washed in 2.5 ml STE buffer (10 mM Tris-HCl pH 8.0,
100 mM NaCl, 1 mM EDTA) and resuspended in 15 ml STE buffer. SDS was added to a concentration of 0.15% w/w and the lysate was incubated at 37 °C for 90-180 minutes with 5 units each of proteases PS147 and P4880 (Sigma). The mixture was extracted with 20 ml of phenol:chloroform by gentle mixing for 10 minutes and centrifuged at 4000 g for 45 minutes. The aqueous phases were re-extracted twice with chloroform and pooled in a sterile 250 ml beaker. DNA was precipitated overnight at -20 °C with 10 ml of 3 M sodium acetate pH 5.2 and 100 ml ethanol. The DNA was then spooled onto a glass rod and left to dissolve in 2 ml of TE buffer at 4 °C for 2-3 days. DNA concentration and purity were determined by the absorbance ratio A<span class="math inline">\(_{260}\)</span>:A<span class="math inline">\(_{280}\)</span> and restriction of the DNA analysed by agarose gel electrophoresis.</p>
<p>Smaller scale purification was performed using the Promega Wizard genomic DNA purification kit, largely according to the manufacturer’s instructions. Using this procedure, 1 ml of overnight culture was lysed and treated with RNase, after which protein was precipitated and removed by centrifugation. Chromosomal DNA was then precipitated by
isopropanol, washed with 70% v/v ethanol, and dissolved in TE buffer overnight at 4 °C.</p>
</div>
<div id="polymerase-chain-reaction-methods" class="section level4" number="2.4.1.12">
<h4><span class="header-section-number">2.4.1.12</span> Polymerase chain reaction methods</h4>
<p>The polymerase chain reaction (PCR) was routinely used to generate DNA fragments for cloning and sequencing. Many variations of the technique were used; however, a typical reaction (50 <span class="math inline">\(\mu\)</span>l) contained 10-500 ng of template genomic DNA, PCR buffer (10 mM Tris HCl pH 9.0, 50 mM KCl, 0.1% Triton X-100), 1.5 mM MgCl<span class="math inline">\(_2\)</span>, 0.2 mM each dNTP, 25 pmol of forward and reverse primers and 2.5 units of Taq DNA polymerase. Amplification was performed in 0.5 ml thin-walled Eppendorf tubes, in a programmable thermocycler (MJ Research Inc.). A typical cycle would consist of denaturation (97 °C, 3 minutes), followed by 30 cycles of denaturation (94 °C, 1 minute), primer annealing (55 °C, 2 minutes) and extension (72 °C, 2 minutes), with a final extension step of 72 °C for 10 minutes. PCR products were analysed by electrophoresis on agarose gels in TBE buffer.</p>
<p>Taq polymerase catalyses the addition of a single dNTP residue (normally dATP, Clark 1988)<span class="citation"><sup><a href="#ref-clark_novel_1988" role="doc-biblioref">243</a></sup></span> to the 3’ end of PCR products, which allows them to be cloned into linearised vectors with a 5’ dTTP overhang. The pGEM-T cloning kit (Promega) was used for this purpose. Typically, 5 <span class="math inline">\(\mu\)</span>l of a PCR product purified by the GeneClean method was ligated to 1 <span class="math inline">\(\mu\)</span>l of PGEM-T vector in a 10 <span class="math inline">\(\mu\)</span>l reaction, as described previously. The ligation mix was used to transform competent cells as described in Section 2.5.19.</p>
</div>
<div id="southern-blotting-of-dna-using-the-digoxigenin-detection-system" class="section level4" number="2.4.1.13">
<h4><span class="header-section-number">2.4.1.13</span> Southern blotting of DNA using the digoxigenin detection system</h4>
<p>Probes for Southern blotting were generated by random priming of DNA fragments using hexanucleotides, Klenow DNA polymerase and a deoxynucleotide triphosphate mixture containing dUTP coupled to digoxigenin (DIG-11-UTP). The latter was detected on a membrane using antibodies to digoxigenin, which were coupled to alkaline phosphatase. To
generate the probe, approximately 1 <span class="math inline">\(\mu\)</span>g of DNA template (a restriction fragment or PCR product homologous to the region to be probed) was denatured by boiling and chilled in an ice-brine mixture. 10 <span class="math inline">\(\mu\)</span>l of denatured DNA was mixed with 2 <span class="math inline">\(\mu\)</span>l of DIG labelling mix (1 mM DATP, SCTP and DGTP, 6.5 mM UTP, 0.35 mM HTTP, 0.35 mM DIG-11-UTP), 2 <span class="math inline">\(\mu\)</span>l random hexanucleotides (62.5 A<span class="math inline">\(_{260}\)</span> units ml<span class="math inline">\(^{-1}\)</span>) and 5 units of Klenow fragment in a total volume of 20 <span class="math inline">\(\mu\)</span>l and incubated at 30 °C overnight. Probe DNA was precipitated with 0.1 volumes of 4 M LiCl and 2.5 volumes of ethanol, washed with ice-cold 70% v/v ethanol, air dried and redissolved in 20 <span class="math inline">\(\mu\)</span>l of TE buffer.</p>
<p>The DNA to be probed was separated by agarose gel electrophoresis and transferred overnight at room temperature to a positive nylon membrane by capillary transfer with 0.4 M NaOH. The membrane was then prehybridised at 68 °C for one hour in 20 ml of hybridisation buffer (5x SSC, 1% w/v blocking reagent, 0.1% w/v N-laurylsarcosine, 0.02%
w/v SDS). Approximately 0.5 <span class="math inline">\(\mu\)</span>g of probe DNA was heat-denatured, chilled and added to 5 ml of prehybridisation buffer. The membrane was hybridised at 68 °C overnight.</p>
<p>For washing and probe detection, the membrane was washed twice in 50 ml 2x SSC, 0.1% SDS at room temperature for five minutes, then twice in 50 ml 0.1x SSC, 0.1% SDS at 68 °C for 15 minutes. The membrane was equilibrated for one minute in TS buffer (100 mM Tris HCl pH 7.5, 150 mM NaCl), then blocked for 30 minutes in TSB buffer (TS buffer plus 1% w/v blocking reagent). Anti-DIG-alkaline phosphatase antibody conjugate was diluted 1:5000 in TSB buffer and the membrane was incubated in this solution for 30 minutes. The membrane was washed twice for 15 minutes with TS buffer and equilibrated for two minutes in substrate buffer (100 mM Tris-HCl pH 9.5, 100 mM NaCl, 50 mM MgCl<span class="math inline">\(_2\)</span>). The bound probe was detected colorimetrically by incubating the membrane in the dark without shaking in 20 ml of substrate buffer containing 400 <span class="math inline">\(\mu\)</span>M nitroblue tetrazolium and 400 <span class="math inline">\(\mu\)</span>M 5-bromo-4-chloro-3-indolyl phosphate. Colour development was stopped by washing the membrane in TE buffer.</p>
</div>
<div id="dna-sequencing-methods" class="section level4" number="2.4.1.14">
<h4><span class="header-section-number">2.4.1.14</span> DNA sequencing methods</h4>
<p>Plasmid DNA was sequenced by the dideoxy-chain termination method,<span class="citation"><sup><a href="#ref-sanger_dna_1977" role="doc-biblioref">244</a></sup></span> using the Sequenase 2.0 kit (Amersham). Sequencing was divided into three stages: template denaturation plus primer annealing, labelling and termination.</p>
<p>1 <span class="math inline">\(\mu\)</span>g of plasmid DNA was denatured either by boiling for five minutes then chilling on ice, or by incubation in 0.2 M NaOH for 5 minutes at 37 °C. In the latter case, the NaOH was neutralised with 0.2 M HCl after denaturation. Approximately 1 pmol of sequencing primer was annealed to the template at 37 °C for 10 minutes, in a volume of 10 <span class="math inline">\(\mu\)</span>l. The DNA labelling reaction comprised the 10 <span class="math inline">\(\mu\)</span>l solution of annealed DNA template plus primer, which was made up to a final volume of 16 <span class="math inline">\(\mu\)</span>l containing the following components: <span class="math inline">\(^{35}\)</span>S-dATP (6.25 <span class="math inline">\(\mu\)</span>Ci), DTT (3.2 mM), Sequenase reaction buffer (40 mM Tris-HCl pH 7.5, 50 mM NaCl, 20 mM MgCl<span class="math inline">\(_2\)</span>), 0.25 <span class="math inline">\(\mu\)</span>M each of dCTP, dGTP and dTTP, and 1.6 units of Sequenase enzyme. In some reactions two optional additions were made: (1) 3.2 mM MnCl<span class="math inline">\(_2\)</span> plus 5 mM sodium isocitrate, which permitted reading of the sequence closer to the primer, and/or (2) 7% v/v DMSO, which alleviated the problem of secondary structure formation during extension of the labelled DNA fragments. The labelling reaction was incubated at room temperature for five minutes. Following labelling, the reaction mix was divided into four aliquots and each of these was mixed with 2.5 <span class="math inline">\(\mu\)</span>l of termination mixture (which contained 80 <span class="math inline">\(\mu\)</span>M of each dNTP plus 8 <span class="math inline">\(\mu\)</span>M of the appropriate dideoxy-NTP, in 50 mM NaCl). The termination reactions were performed at 42 °C for five minutes. In some reactions DMSO was added to a concentration of 7% v/v, to eliminate the formation of secondary structure as described above. Finally, each reaction was inactivated with the addition of 4 <span class="math inline">\(\mu\)</span>l of stop solution (95% v/v formamide, 20 mM Na<span class="math inline">\(_2\)</span>EDTA, 0.05% w/v each of bromophenol blue and xylene cyanol FF). 3.5 <span class="math inline">\(\mu\)</span>l of the A, C, G and T termination reactions were electrophoresed side by side on a 6% acrylamide sequencing gel, in TBE buffer at a constant power of 70 W. Following separation the gel was dried under vacuum at 80 °C and exposed to X-ray film.</p>
<p>PCR products were sometimes sequenced directly from low melting point agarose gels.<span class="citation"><sup><a href="#ref-kretz_direct_1993" role="doc-biblioref">245</a></sup></span> Following electrophoresis on 1% agarose gels (NuSieve GTG), a gel slice containing the PCR product was melted at 68 °C. The DNA was denatured by boiling, chilled on ice and annealed to approximately 1 pmol of primer in a volume of 10 <span class="math inline">\(\mu\)</span>l as described for plasmid DNA. Subsequent sequencing reactions were carried out as described above, except that labelling was at 37 °C, to prevent solidification of the agarose.</p>
</div>
<div id="computer-analysis-of-dna-data" class="section level4" number="2.4.1.15">
<h4><span class="header-section-number">2.4.1.15</span> Computer analysis of DNA data</h4>
<p>Most DNA data analysis was performed using the GCG8-Open VMS and UNIX software suite,<span class="citation"><sup><a href="#ref-devereux_comprehensive_1984" role="doc-biblioref">246</a></sup></span> Genetics Computer Group, Wisconsin, USA, maintained at the William Dunn School of Pathology, University of Oxford. Freely-available software on the World Wide Web was also used widely and is referred to in the relevant chapters.</p>
</div>
<div id="oligonucleotide-synthesis-and-preparation" class="section level4" number="2.4.1.16">
<h4><span class="header-section-number">2.4.1.16</span> Oligonucleotide synthesis and preparation</h4>
<p>Oligonucleotides for sequencing, PCR and primer extension were synthesised by Val Cooper at the Oligonucleotide Synthesis Service, Dyson Perrins Organic Chemistry Laboratory, University of Oxford. Before use, oligonucleotides were incubated at 55 °C overnight to evaporate residual ammonia. They were then recovered by ethanol precipitation and redissolved in sterile distilled water.</p>
</div>
</div>
<div id="rna-manipulation" class="section level3" number="2.4.2">
<h3><span class="header-section-number">2.4.2</span> RNA manipulation</h3>
<div id="extraction-of-total-rna-from-thiosphaera-pantotropha" class="section level4" number="2.4.2.1">
<h4><span class="header-section-number">2.4.2.1</span> Extraction of total RNA from <em>Thiosphaera pantotropha</em></h4>
<p>No single published protocol was used to isolate total RNA from cells of <em>Thiosphaera pantotropha</em>; the protocol described below was based on the methods described by Chattopadhyay et al. (1993)<span class="citation"><sup><a href="#ref-chattopadhyay_inexpensive_1993" role="doc-biblioref">247</a></sup></span> and Majumdar et al. (1991).<span class="citation"><sup><a href="#ref-majumdar_simultaneous_1991" role="doc-biblioref">248</a></sup></span> 50 ml cultures were grown to mid-exponential phase, harvested by centrifugation and lysed in 4 ml of lysis buffer (1% SDS, 1 mM EDTA pH 8.0). The lysate was extracted with 4 ml of phenol:chloroform, saturated with 2 M sodium acetate pH 4.0 and centrifuged at 4000 g for 10 minutes, 4 °C. The aqueous phase was then re-extracted with 4 ml of chloroform and recovered by centrifugation. 0.8 volumes of isopropanol was added and the mixture was incubated on ice for 30 minutes, then centrifuged at 15 000 g for 15 minutes at 4 °C. The precipitated nucleic acids were redissolved in 200 <span class="math inline">\(\mu\)</span>l of DNase buffer (40 mM Tris-HCl pH 8.0, 10 mM NaCl, 6 mM CaCl<span class="math inline">\(_2\)</span>), containing 100 units of RNase-free DNase and incubated at 37 °C for 30 minutes. After DNase treatment, the mixture was extracted once more with an equal volume of phenol:chloroform and the RNA was ethanol-precipitated by incubating at -70 °C, followed by centrifugation at 15 000 g for 15 minutes, 4 °C. The pellet was washed with 100 <span class="math inline">\(\mu\)</span>l of ice-cold 70 % v/v ethanol, air-dried and redissolved in 100 <span class="math inline">\(\mu\)</span>l of RNase-free water.</p>
<p>RNA quality was analysed by the absorbance ratio A<span class="math inline">\(_{260}\)</span>:A<span class="math inline">\(_{280}\)</span> and by denaturing gel electrophoresis (Section 2.5.32).</p>
</div>
<div id="formaldehyde-agarose-gel-electrophoresis" class="section level4" number="2.4.2.2">
<h4><span class="header-section-number">2.4.2.2</span> Formaldehyde-agarose gel electrophoresis</h4>
<p>Denaturing agarose gel electrophoresis in the presence of formamide was used to examine the integrity of RNA preparations and to separate RNA molecules by size before transfer to a membrane for Northern blotting. Up to 20 <span class="math inline">\(\mu\)</span>g of total RNA in 4.5 <span class="math inline">\(\mu\)</span>l was made up to a volume of 20 <span class="math inline">\(\mu\)</span>l, in MOPS buffer (20 mM morpholinopropanesulphonic acid pH 7.0, 5 mM sodium acetate, 0.5 mM EDTA) containing 50% v/v freshly deionised formamide and 2.2 M formaldehyde. The RNA was denatured by heating at 80 °C for 3 minutes, then placed on
ice. 1% formaldehyde-agarose gels were prepared by dissolving 1 g of agarose in a 50 ml solution containing MOPS buffer and 2.2 M formaldehyde. 4 <span class="math inline">\(\mu\)</span>l of loading dye (50% v/v glycerol, 1 mM EDTA, 0.4 % w/v bromophenol blue, 33 ng <span class="math inline">\(\mu\)</span>l<span class="math inline">\(^{-1}\)</span> ethidium bromide) was added to the RNA samples before loading and the gel was run at a constant voltage of 100 V for 5-6 hours. Gels were examined and photographed under ultraviolet light.</p>
</div>
<div id="northern-blotting" class="section level4" number="2.4.2.3">
<h4><span class="header-section-number">2.4.2.3</span> Northern blotting</h4>
<p>Northern blotting was used to detect specific mRNA transcripts from denitrification genes in <em>T. pantotropha</em>. The method used closely followed that of Engler-Blum et al. (1993).<span class="citation"><sup><a href="#ref-engler-blum_reduction_1993" role="doc-biblioref">249</a></sup></span> Following formaldehyde-agarose gel electrophoresis, the gel was washed with 20x SSC for 20 minutes. RNA was then transferred to a positive nylon membrane (Boehringer Mannheim) by capillary transfer with 20x SSC at 4 °C and crosslinked to the membrane using UV illumination. A lane containing molecular size markers was cut from the membrane and stained with methylene blue (0.2% w/v in 0.2 M sodium acetate pH 4.7). The membrane was prehybridised for one hour at 68 °C in a high SDS phosphate buffer (250 mM sodium phosphate pH 7.2, 1 mM EDTA, 20% w/v SDS). A DIG labelled dsDNA probe (Section 2.5.22) was hybridised to the RNA in 10 ml of the same buffer at 68 °C overnight. The membrane was washed three times for 20 minutes each at 65 °C in 50 ml of H-wash buffer (20 mM sodium phosphate pH 7.2, 1% w/v SDS, 1 mM EDTA), then once for five minutes in W-wash buffer (100 mM maleic acid pH 8.0, 3 M NaCl, 0.3% v/v Tween-20) and blocked for one hour in blocking buffer (W-wash buffer plus 0.5% w/v blocking reagent, Boehringer Mannheim). DIG antibody-alkaline phosphatase conjugate was diluted 1:10 000 in blocking buffer and the membrane was incubated with 20 ml of this buffer for 30 minutes. The membrane was then washed four times for 10 minutes each with W-wash buffer and equilibrated for five minutes in substrate buffer (100 mM Tris-HCl pH 9.5, 100 mM NaCl, 50 mM MgCl<span class="math inline">\(_2\)</span>).</p>
<p>The hybridised probe was detected either colorimetrically or by chemiluminescence. For colorimetric detection, the membrane was incubated with 20 ml of colour solution (Section 2.5.22) in the dark without shaking until bands appeared. Development was stopped by washing with TE buffer. For chemiluminescent detection, the alkaline
phosphatase substrate CSPD (Boehringer Mannheim) was diluted 1:100 in 1 ml of substrate buffer and pipetted onto the membrane. Following a five minute incubation in the dark, excess solution was removed and the membrane was incubated in a sealed hybridisation bag at 37 °C for 15 minutes. The membrane was then exposed to X-ray film (Kodak), initially for one hour, until exposed bands appeared on the film.</p>
</div>
<div id="primer-extension-analysis" class="section level4" number="2.4.2.4">
<h4><span class="header-section-number">2.4.2.4</span> Primer extension analysis</h4>
<p>Primer extension was used to determine the 5’-end of the <em>nirS</em> gene mRNA transcript. The procedure was divided into 3 stages: oligonucleotide labelling, primer-RNA annealing and cDNA extension. 10 pmol of oligonucleotide was end-labelled in a 20 <span class="math inline">\(\mu\)</span>l reaction containing kinase buffer (70 mM Tris-HCl pH 7.6, 10 mM MgCl<span class="math inline">\(_2\)</span>, 5 mM DTT), 50 <span class="math inline">\(\mu\)</span>Ci of <span class="math inline">\(\gamma^{32}\)</span>P-ATP and 8 units of polynucleotide kinase, exactly as described in Sambrook (1989). The labelled oligonucleotide was redissolved in 100 <span class="math inline">\(\mu\)</span>l of TE buffer and could be stored at -20 °C for up to 1 week.</p>
<p>1 <span class="math inline">\(\mu\)</span>l (approximately 109 cpm) of labelled primer was annealed to 40 <span class="math inline">\(\mu\)</span>g of heat denatured RNA in 20 <span class="math inline">\(\mu\)</span>l of annealing buffer (2 mM Tris-HCl pH 7.9, 250 mM KCl, 0.2 mM EDTA) at 50 °C for 90 minutes. The annealed primer-RNA was then ethanol-precipitated, redissolved in 20 <span class="math inline">\(\mu\)</span>l of reverse transcription mix (50 mM Tris-HCl pH 8.3, 75 mM KCl, 3 mM MgCl<span class="math inline">\(_2\)</span>, 10 mM DTT, 1.3 mM each dNTP, 20 units RNasin inhibitor, 50 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> actinomycin D, 33 units AMV reverse transcriptase) and incubated at 37 °C for 2 hours. DNA synthesis was stopped with 25 mM EDTA, pH 8.0. The mixture was treated with RNase at 37 °C for 20 minutes, extracted with phenol-chloroform, ethanol-precipitated and redissolved in 4 <span class="math inline">\(\mu\)</span>l of TE buffer. 6 <span class="math inline">\(\mu\)</span>l of sequencing stop solution was added and the labelled cDNA was run on a 6% acrylamide sequencing gel alongside a reference sequencing ladder generated using the same primer.</p>
</div>
<div id="race-rapid-amplification-of-cdna-5-ends" class="section level4" number="2.4.2.5">
<h4><span class="header-section-number">2.4.2.5</span> 5’-RACE (Rapid amplification of cDNA 5’ ends)</h4>
<p>5’-RACE was used as an alternative method to determine the 5’-end of mRNA transcripts. The procedure used closely followed that of Frohman (1993).<span class="citation"><sup><a href="#ref-frohman_rapid_1993" role="doc-biblioref">250</a></sup></span> 10 <span class="math inline">\(\mu\)</span>g of total RNA was reverse-transcribed using the <em>nirS</em> sequence specific primer 284R (5’-GACATCCTGCTGCGCAAGGT-3’). The mixture was then diluted with TE buffer, excess primer and nucleotides were removed by centrifugation through a Centricon-100 filter and the cDNA was recovered on a Millipore spin filter in 10 <span class="math inline">\(\mu\)</span>l of TE buffer. Addition of the poly-dATP tail and the two subsequent nested PCR amplifications were performed exactly as described by Frohman (1993), using the forward primers Q<span class="math inline">\(_T\)</span>, Q<span class="math inline">\(_0\)</span> and Q<span class="math inline">\(_1\)</span> (detailed in Frohman, 1993) and the sequence specific reverse primers 242R (5’-ATAGTCGGTGCGCGTCTT-3’) and 228R (5P-TCTTGTGGTCCTCCAGCG-3’). PCR products were analysed by electrophoresis on 1.2% agarose gels, cloned into the pGEM-T vector and sequenced using M13 universal primers.</p>

</div>
</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-coyle_nitrous_1985" class="csl-entry">
(85) Coyle, C. L., Zumft, W. G., Kroneck, P. M., Körner, H., and Jakob, W. (1985) Nitrous oxide reductase from denitrifying <span>Pseudomonas</span> perfectomarina. <span>Purification</span> and properties of a novel multicopper enzyme. <em>European Journal of Biochemistry</em> <em>153</em>, 459–467.
</div>
<div id="ref-berks_purification_1993" class="csl-entry">
(94) Berks, B. C., Baratta, D., Richardson, J., and Ferguson, S. J. (1993) Purification and characterization of a nitrous oxide reductase from <span>Thiosphaera</span> pantotropha. <span>Implications</span> for the mechanism of aerobic nitrous oxide reduction. <em>European Journal of Biochemistry</em> <em>212</em>, 467–476.
</div>
<div id="ref-hill_reconstitution_1978" class="csl-entry">
(118) Hill, K. E., and Wharton, D. C. (1978) Reconstitution of the apoenzyme of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa with heme d1 and other heme groups. <em>The Journal of Biological Chemistry</em> <em>253</em>, 489–495.
</div>
<div id="ref-gilmour_kinetics_1994" class="csl-entry">
(123) Gilmour, R., Goodhew, C. F., Pettigrew, G. W., Prazeres, S., Moura, J. J., and Moura, I. (1994) The kinetics of the oxidation of cytochrome c by <span>Paracoccus</span> cytochrome c peroxidase. <em>The Biochemical Journal</em> <em>300 ( Pt 3)</em>, 907–914.
</div>
<div id="ref-robinson_implications_1979" class="csl-entry">
(155) Robinson, M. K., Martinkus, K., Kennelly, P. J., and Timkovich, R. (1979) Implications of the integrated rate law for the reactions of <span>Paracoccus</span> denitrificans nitrite reductase. <em>Biochemistry</em> <em>18</em>, 3921–3926.
</div>
<div id="ref-thompson_variation_1997" class="csl-entry">
(222) Thompson, I., Ferguson, S., and Baker, S. (1997) Variation within <span>Paracoccus</span> denitrificans and revival of the species <span>Thiosphaera</span> pantotropha as <span>Paracoccus</span> pantotrophus comb. nov.
</div>
<div id="ref-robertson_thiosphaera_1983" class="csl-entry">
(223) Robertson, L. A., and Kuenen, J. (1983) Thiosphaera pantotropha gen. Nov. Sp. Nov., A <span>Facultatively</span> <span>Anaerobic</span>, <span>Facultatively</span> <span>Autotrophic</span> <span>Sulphur</span> <span>Bacterium</span>. <em>Microbiology,</em> <em>129</em>, 2847–2855.
</div>
<div id="ref-dohler_transfer_1987" class="csl-entry">
(224) Dohler, K., Huss, V. A. R., and Zumft, W. G. (1987) Transfer of <span>Pseudomonas</span> perfectomarina <span>Baumann</span>, <span>Bowditch</span>, <span>Baumann</span>, and <span>Beaman</span> 1983 to <span>Pseudomonas</span> stutzeri (<span>Lehmann</span> and <span>Neumann</span> 1896) <span>Sijderius</span> 1946. <em>International Journal of Systematic Bacteriology</em> <em>37</em>, 1–3.
</div>
<div id="ref-holloway_chromosomal_1979" class="csl-entry">
(225) Holloway, B. W., Krishnapillai, V., and Morgan, A. F. (1979) Chromosomal genetics of <span>Pseudomonas</span>. <em>Microbiological Reviews</em> <em>43</em>, 73–102.
</div>
<div id="ref-yanisch-perron_improved_1985" class="csl-entry">
(226) Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Improved <span>M13</span> phage cloning vectors and host strains: Nucleotide sequences of the <span>M13mp18</span> and <span class="nocase">pUC19</span> vectors. <em>Gene</em> <em>33</em>, 103–119.
</div>
<div id="ref-hanahan_studies_1983" class="csl-entry">
(227) Hanahan, D. (1983) Studies on transformation of <span>Escherichia</span> coli with plasmids. <em>Journal of Molecular Biology</em> <em>166</em>, 557–580.
</div>
<div id="ref-simon_broad_1983" class="csl-entry">
(228) Simon, R., Priefer, U., and Pühler, A. (1983) A <span>Broad</span> <span>Host</span> <span>Range</span> <span>Mobilization</span> <span>System</span> for <span>In</span> <span>Vivo</span> <span>Genetic</span> <span>Engineering</span>: <span>Transposon</span> <span>Mutagenesis</span> in <span>Gram</span> <span>Negative</span> <span>Bacteria</span>. <em>Bio/Technology</em> <em>1</em>, 784–791.
</div>
<div id="ref-vishniac_thiobacilli_1957" class="csl-entry">
(229) Vishniac, W., and Santer, M. (1957) The thiobacilli. <em>Bacteriological Reviews</em> <em>21</em>, 195–213.
</div>
<div id="ref-burnell_reversibility_1975" class="csl-entry">
(230) Burnell, J. N., John, P., and Whatley, F. R. (1975) The reversibility of active sulphate transport in membrane vesicles of <span>Paracoccus</span> denitrificans. <em>The Biochemical Journal</em> <em>150</em>, 527–536.
</div>
<div id="ref-pope_generation_1982" class="csl-entry">
(231) Pope, N. R., and Cole, J. A. (1982) Generation of a membrane potential by one of two independent pathways for nitrite reduction by <span>Escherichia</span> coli. <em>Journal of General Microbiology</em> <em>128</em>, 219–222.
</div>
<div id="ref-cole_nitrite_1974" class="csl-entry">
(232) Cole, J. A., Coleman, K. J., Compton, B. E., Kavanagh, B. M., and Keevil, C. W. (1974) Nitrite and ammonia assimilation by anaerobic continuous cultures of <span>Escherichia</span> coli. <em>Journal of General Microbiology</em> <em>85</em>, 11–22.
</div>
<div id="ref-de_boer_tac_1983" class="csl-entry">
(233) Boer, H. A. de, Comstock, L. J., and Vasser, M. (1983) The tac promoter: A functional hybrid derived from the trp and lac promoters. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <em>80</em>, 21–25.
</div>
<div id="ref-scopes_protein_1987" class="csl-entry">
(234) Scopes, R. K. (1987) Protein <span>Purification</span>: <span>Principles</span> and <span>Practice</span>.
</div>
<div id="ref-sambrook_molecular_1989" class="csl-entry">
(235) Sambrook, J., Fritsch, E., and Maniatis, T. (1989) Molecular cloning: <span>A</span> laboratory manual: <span>Vol</span>. 2 2. ed. Cold Spring Harbor, S.l.
</div>
<div id="ref-goodhew_cytochromes_1996" class="csl-entry">
(236) Goodhew, C. F., Pettigrew, G. W., Devreese, B., Beeumen, jozef, Spanning, R. J. M., Baker, S. C., Saunders, N., Ferguson, stuart J., and Thompson, I. P. (1996) The cytochromes <em>c</em> -550 of <em><span>Paracoccus</span> denitrificans</em> and <em><span>Thiosphaera</span> pantotropha</em> : A need for re-evaluation of the history of <em>paracoccus</em> cultures. <em>FEMS Microbiology Letters</em> <em>137</em>, 95–101.
</div>
<div id="ref-bradford_rapid_1976" class="csl-entry">
(237) Bradford, M. M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. <em>Analytical Biochemistry</em> <em>72</em>, 248–254.
</div>
<div id="ref-moir_aspects_1993" class="csl-entry">
(238) Moir, J. (1993) Aspects of electron transport in <span>Thiosphaera</span> pantotropha and <span>Paracoccus</span> denitrificans. PhD thesis, University of Oxford.
</div>
<div id="ref-donald_nicholas_determination_1957" class="csl-entry">
(239) Donald Nicholas, D. J., and Nason, A. (1957) Determination of nitrate and nitrite, in <em>Methods in <span>Enzymology</span></em>, pp 981–984. Elsevier.
</div>
<div id="ref-birnboim_rapid_1979" class="csl-entry">
(240) Birnboim, H. C., and Doly, J. (1979) A rapid alkaline extraction procedure for screening recombinant plasmid <span>DNA</span>. <em>Nucleic Acids Research</em> <em>7</em>, 1513–1523.
</div>
<div id="ref-mierendorf_direct_1987" class="csl-entry">
(241) Mierendorf, R. C., and Pfeffer, D. (1987) Direct sequencing of denatured plasmid <span>DNA</span>. <em>Methods in Enzymology</em> <em>152</em>, 556–562.
</div>
<div id="ref-cohen_nonchromosomal_1972" class="csl-entry">
(242) Cohen, S. N., Chang, A. C., and Hsu, L. (1972) Nonchromosomal antibiotic resistance in bacteria: Genetic transformation of <span>Escherichia</span> coli by <span>R</span>-factor <span>DNA</span>. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <em>69</em>, 2110–2114.
</div>
<div id="ref-clark_novel_1988" class="csl-entry">
(243) Clark, J. M. (1988) Novel non-templated nucleotide addition reactions catalyzed by procaryotic and eucaryotic <span>DNA</span> polymerases. <em>Nucleic Acids Research</em> <em>16</em>, 9677–9686.
</div>
<div id="ref-sanger_dna_1977" class="csl-entry">
(244) Sanger, F., Nicklen, S., and Coulson, A. R. (1977) <span>DNA</span> sequencing with chain-terminating inhibitors. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <em>74</em>, 5463–5467.
</div>
<div id="ref-kretz_direct_1993" class="csl-entry">
(245) Kretz, K. A., and O’Brien, J. S. (1993) Direct sequencing of polymerase chain reaction products from low melting temperature agarose. <em>Methods in Enzymology</em> <em>218</em>, 72–79.
</div>
<div id="ref-devereux_comprehensive_1984" class="csl-entry">
(246) Devereux, J., Haeberli, P., and Smithies, O. (1984) A comprehensive set of sequence analysis programs for the <span>VAX</span>. <em>Nucleic Acids Research</em> <em>12</em>, 387–395.
</div>
<div id="ref-chattopadhyay_inexpensive_1993" class="csl-entry">
(247) Chattopadhyay, N., Kher, R., and Godbole, M. (1993) Inexpensive <span>SDS</span>/phenol method for <span>RNA</span> extraction from tissues. <em>BioTechniques</em> <em>15</em>, 24–26.
</div>
<div id="ref-majumdar_simultaneous_1991" class="csl-entry">
(248) Majumdar, D., Avissar, Y. J., and Wyche, J. H. (1991) Simultaneous and rapid isolation of bacterial and eukaryotic <span>DNA</span> and <span>RNA</span>: A new approach for isolating <span>DNA</span>. <em>BioTechniques</em> <em>11</em>, 94–101.
</div>
<div id="ref-engler-blum_reduction_1993" class="csl-entry">
(249) Engler-Blum, G., Meier, M., Frank, J., and Müller, G. A. (1993) Reduction of background problems in nonradioactive northern and <span>Southern</span> blot analyses enables higher sensitivity than <span>32P</span>-based hybridizations. <em>Analytical Biochemistry</em> <em>210</em>, 235–244.
</div>
<div id="ref-frohman_rapid_1993" class="csl-entry">
(250) Frohman, M. A. (1993) Rapid amplification of complementary <span>DNA</span> ends for generation of full-length complementary <span>DNAs</span>: Thermal <span>RACE</span>. <em>Methods in Enzymology</em> <em>218</em>, 340–356.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="1-chapter01.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="3-chapter03.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
