<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Sequencing and analysis of the nirS gene from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.34.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Sequencing and analysis of the nirS gene from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Sequencing and analysis of the nirS gene from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="2-chapter02.html"/>
<link rel="next" href="4-chapter04.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t.-pantotropha.html"><a href="B-classification-of-t.-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter03" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em><a href="3-chapter03.html#chapter03" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="introduction" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> Introduction<a href="3-chapter03.html#introduction" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To date the primary sequences of six cytochrome <em>cd</em><span class="math inline">\(_1\)</span> proteins, obtained by DNA sequencing of the <em>nirS</em> gene, have been published. These are from the organisms <em>Ps. aeruginosa</em> NTCC 6750 <span class="citation">[<a href="#ref-silvestrini_kinetics_1982">134</a>]</span>, <em>Ps. stutzeri</em> JM300 <span class="citation">[<a href="#ref-smith_isolation_1992">139</a>]</span>, <em>Ps. stutzeri</em> Zobell <span class="citation">[<a href="#ref-jungst_nirstbm_1991">138</a>]</span>, <em>P. denitrificans</em> PD1222 <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>, <em>P. denitrificans</em> IFO 12442 <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span> and <em>A. eutrophus</em> H16 <span class="citation">[<a href="#ref-rees_structural_1997">142</a>]</span>. Additionally, small lengths of peptide sequence have been determined from several cytochromes <em>cd</em><span class="math inline">\(_1\)</span> as a result of either N-terminal or internal protein sequencing, <em>e.g.</em> from <em>Pseudomonas stutzeri</em> JM300 <span class="citation">[<a href="#ref-weeg-aerssens_purification_1991">253</a>]</span> and <em>Thiobacillus denitrificans</em> <span class="citation">[<a href="#ref-hole_characterization_1996">254</a>]</span>. The initial aim of the present work, the sequencing of the structural gene for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (nirS) from <em>T. pantotropha</em>, was important for several reasons. First, the published crystal structure of the enzyme was determined from the electron density map using comparison to the sequence of the protein from <em>P. denitrificans</em> PD1222 <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span>. Although this procedure fitted the diffraction data well, the authentic protein sequence was required to resolve any ambiguities, for instance in the N-terminus of the protein which was disordered in the crystal structure, and to refine the structure. Second, knowledge of the <em>nirS</em> nucleotide sequence was essential for future work involving manipulation of the gene; for example restriction mapping, cloning procedures and mutagenesis. Mapping of the regions either side of the <em>nirS</em> gene would also facilitate cloning and allow comparison of the nir region in <em>T. pantotropha</em> to the closely-related <em>P. denitrificans</em> PD1222. Third, knowledge of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> peptide sequence allows comparison with other cytochromes <em>cd</em><span class="math inline">\(_1\)</span> and analysis of the similarities and differences within this group of enzymes.</p>
<p>This chapter describes the strategy used to sequence the <em>nirS</em> gene from <em>T. pantotropha</em>. The DNA sequence is reported and analysed, including comparison with the sequences from two strains of <em>P. denitrificans</em>. The translated cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequence is reported and compared with other known cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences, using several computer-based methods. This work reveals that there are important differences between the proteins and the comparison with cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> Zobell is explored in more detail.</p>
<p>Finally the relationship between the proposed haem ligands and the mechanism of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is discussed, in the light of recent spectroscopic and crystallographic data.</p>
</div>
<div id="sequencing-strategy" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Sequencing strategy<a href="3-chapter03.html#sequencing-strategy" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The <em>nirS</em> gene from <em>T. pantotropha</em> was sequenced in collaboration with Dr. Simon Baker, from a series of overlapping PCR products. Initial studies begun before the present project by Dr. Simon Baker had generated a PCR product using degenerate primers that were designed using short lengths of peptide sequence obtained from <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <span class="citation">[<a href="#ref-moir_aspects_1993">240</a>]</span>. The sequence of the PCR product indicated that the <em>nirS</em> DNA sequence of <em>T. pantotropha</em> was very similar to that from <em>P. denitrificans</em> PD1222, and so further primers for PCR and sequencing were designed using the sequence of the <em>nirS</em> gene from <em>P. denitrificans</em> PD1222 <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>. Table 3.1 summarises the primers used.</p>
<table>
<caption>
<span id="tab:table31">Table 3.1: </span>Primers used to sequence the <em>nirS</em> gene from <em>T. pantotropha</em>. The equivalent positions in the GenBank sequence accession number U05002 (the sequence of the genes <em>nirI</em>, <em>nirS</em>, <em>nirE</em>, <em>nirC</em> and <em>nirF</em> from <em>P. denitrificans</em> PD1222) are shown. Primer sequences are in the 5’ to 3’ direction for both forward and reverse primers.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Primer direction
</th>
<th style="text-align:left;">
Primer name
</th>
<th style="text-align:left;">
Equivalent bases in <em>P. denitrificans</em> <em>nirS</em> sequence(GenBank U05002)
</th>
<th style="text-align:left;">
Sequence (5’-3’)
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Forward
</td>
<td style="text-align:left;">
460F
</td>
<td style="text-align:left;">
473-492
</td>
<td style="text-align:left;">
ACCTCGGCCTTAACAAAGGT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
610F
</td>
<td style="text-align:left;">
610-629
</td>
<td style="text-align:left;">
CTGGCCCTTGTCCTTGGGCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MIDF
</td>
<td style="text-align:left;">
913-932
</td>
<td style="text-align:left;">
GGCTTCGACTACCTGCAAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1053F
</td>
<td style="text-align:left;">
1053-1072
</td>
<td style="text-align:left;">
ATTCGGCATGAAGGAGATGCGC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1000F
</td>
<td style="text-align:left;">
1192-1211
</td>
<td style="text-align:left;">
GACGGCACCACCTATGAGAT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CT2F
</td>
<td style="text-align:left;">
1346-1364
</td>
<td style="text-align:left;">
CCGAGATCAAGATCGGCTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CTIF
</td>
<td style="text-align:left;">
1352-1369
</td>
<td style="text-align:left;">
TCAAGATCGGCTCGGAAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1572F
</td>
<td style="text-align:left;">
1572-1590
</td>
<td style="text-align:left;">
GCCCGAGTTCATCGTGAAC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1631F
</td>
<td style="text-align:left;">
1630-1650
</td>
<td style="text-align:left;">
GACCTCAAGAACCTCAAGACC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1918F
</td>
<td style="text-align:left;">
1918-1940
</td>
<td style="text-align:left;">
CCCGACAATGCCTGGAAGATCCT
</td>
</tr>
<tr>
<td style="text-align:left;">
Reverse
</td>
<td style="text-align:left;">
228R
</td>
<td style="text-align:left;">
756-737
</td>
<td style="text-align:left;">
TACGTCCTGCTGTGCAAGGT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
242R
</td>
<td style="text-align:left;">
714-697
</td>
<td style="text-align:left;">
GTTGTCCGTCTTGGTCTT
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
284R
</td>
<td style="text-align:left;">
700-682
</td>
<td style="text-align:left;">
TCTTGTGATCCTCGAGTG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
811R
</td>
<td style="text-align:left;">
810-790
</td>
<td style="text-align:left;">
GTTGTATTGGGCGTCGGACAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
460R
</td>
<td style="text-align:left;">
932-913
</td>
<td style="text-align:left;">
CTTTGCAGGTAGTCGAAGCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1073R
</td>
<td style="text-align:left;">
1072-1051
</td>
<td style="text-align:left;">
GCATCTCCTTCATGCCGAATTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
300R
</td>
<td style="text-align:left;">
1211-1192
</td>
<td style="text-align:left;">
ATCTCATAGGTGGTGCCGTC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
MIDR
</td>
<td style="text-align:left;">
1461-1442
</td>
<td style="text-align:left;">
GTCCATGATGACGTATTGCG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1572R
</td>
<td style="text-align:left;">
1592-1574
</td>
<td style="text-align:left;">
ACGTTCACGATGAACTCGG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1661R
</td>
<td style="text-align:left;">
1660-1642
</td>
<td style="text-align:left;">
CGATCTCGGTGGTCTTGAG
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1982R
</td>
<td style="text-align:left;">
1982-1964
</td>
<td style="text-align:left;">
TTGATGAAGAGCGAGCCACC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CT2R
</td>
<td style="text-align:left;">
2266-2248
</td>
<td style="text-align:left;">
GTTCGAGGGTCTTGTCGTCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
CTIR
</td>
<td style="text-align:left;">
2280-2261
</td>
<td style="text-align:left;">
GATGACGTGCTTCAGTTCGA
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
nirS2R
</td>
<td style="text-align:left;">
2444-2427
</td>
<td style="text-align:left;">
GTTCGTCACCGTCTTGCC
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
nirSR
</td>
<td style="text-align:left;">
2447-2430
</td>
<td style="text-align:left;">
CCCGTTCGTCACCGTCTT
</td>
</tr>
</tbody>
</table>
<p>The <em>nirS</em> gene was cloned as four overlapping PCR products, generating the plasmids pAP4. pMID1, pCU2p1 and pNDp1. Figure 3.1 shows the sizes of these four PCR products, the primers used to generate them and their relation to the <em>nirS</em> gene from <em>P. denitrificans</em> PD1222.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-1"></span>
<img src="figure/fig3-1.png" alt="PCR products used to sequence the _nirS_ gene of _T. pantotropha_. The _nir_ region of _P. denitrificans_ PD1222 is shown, numbered according to the GenBank sequence U05002. Below is a magnified view of the _P. denitrificans_ PD1222 _nirS_ gene, showing its relationship to the four overlapping PCR products AP4, MID1, CU2p2 and NDp1. The lengths of the PCR products and the primer pairs (Table 3.1) used to generate them are shown." width="75%" />
<p class="caption">
Figure 3.1: PCR products used to sequence the <em>nirS</em> gene of <em>T. pantotropha</em>. The <em>nir</em> region of <em>P. denitrificans</em> PD1222 is shown, numbered according to the GenBank sequence U05002. Below is a magnified view of the <em>P. denitrificans</em> PD1222 <em>nirS</em> gene, showing its relationship to the four overlapping PCR products AP4, MID1, CU2p2 and NDp1. The lengths of the PCR products and the primer pairs (Table 3.1) used to generate them are shown.
</p>
</div>
<p>Three different clones of each product were used for sequencing, and each clone was sequenced at least twice in both directions. This procedure resolved any ambiguities at each base. Additional sequence information from a genomic clone (plasmid pTNIR3, described in Chapter 4) is completely consistent with the sequence derived from these PCR
products.</p>
</div>
<div id="results" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Results<a href="3-chapter03.html#results" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Altogether, 2008 bp of DNA sequence was obtained, including that from the plasmid pTNIR3 (which contributed 139 bp of upstream sequence, Chapters 4 and 7). This covers the <em>nirS</em> open reading frame (bases 139-1926) and short flanking regions at the 5’ and 3’ ends. The complete DNA sequence is presented in Figure 3.2, with the translated cytochrome <em>cd</em><span class="math inline">\(_1\)</span> polypeptide sequence aligned below. Also indicated in this figure are features of interest within the non-coding regions of the DNA.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-2"></span>
<img src="figure/fig3-2.png" alt="Sequence of the _nirS_ gene and its product, cytochrome _cd_$_1$, from _T. pantotropha_. The ATG start codons of open reading frames are labelled with the gene name and an arrow indicating the direction of transcription. Inverted repeats in the DNA sequence are shown by inward facing pairs of arrows. Putative Shine-Delgarno ribosome binding sites are dash-underlined. The _c_-haem binding site in cytochrome _cd_$_1$ is single-underlined. The putative NNR binding site upstream of _nirS_ is boxed. The vertical downward-facing arrow indicates the predicted cleavage site of the periplasmic targeting sequence." width="75%" />
<p class="caption">
Figure 3.2: Sequence of the <em>nirS</em> gene and its product, cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, from <em>T. pantotropha</em>. The ATG start codons of open reading frames are labelled with the gene name and an arrow indicating the direction of transcription. Inverted repeats in the DNA sequence are shown by inward facing pairs of arrows. Putative Shine-Delgarno ribosome binding sites are dash-underlined. The <em>c</em>-haem binding site in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is single-underlined. The putative NNR binding site upstream of <em>nirS</em> is boxed. The vertical downward-facing arrow indicates the predicted cleavage site of the periplasmic targeting sequence.
</p>
</div>
<div id="dna-sequence-features" class="section level3 hasAnchor" number="3.3.1">
<h3><span class="header-section-number">3.3.1</span> DNA sequence features<a href="3-chapter03.html#dna-sequence-features" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Three imperfect inverted repeats are prominent in the DNA sequence, at positions 58-79, 1937-1946 and 1951-1968, and one perfect inverted repeat is found at positions 80-89. Similar repeats are found in the sequences from <em>P. denitrificans</em> PD1222 <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span> and <em>P. denitrificans</em> IFO 12442 <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span>. The positions, sequences and previously assigned functions of these repeats are summarised in Table 3.2.</p>
<table>
<caption>
<span id="tab:table32">Table 3.2: </span>Comparison of inverted repeats in the DNA sequence of the <em>nir</em> region from <em>T. pantotropha</em> with those in the same region of <em>P. denitrificans</em> PD1222. The sequences are numbered according to the GenBank accession numbers U75413 (<em>T. pantotropha</em>) and U05002 (<em>P. denitrificans</em> PD1222). Previously assigned functions of the repeats in the <em>P. denitrificans</em> PD1222 sequence are taken from the GenBank sequence annotation and from de Boer <em>et al</em>. (1994) <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>.
</caption>
<thead>
<tr>
<th style="text-align:left;">
DNA sequence of repeat in <em>T. pantotropha</em>
</th>
<th style="text-align:left;">
DNA sequence of repeat in <em>P. denitrificans</em> PD1222
</th>
<th style="text-align:left;">
Previously assigned function of repeat
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;font-family: monospace;">
Repeat 1: 58-79<br>5’-GGCCTTAACAATGGTCAAAGCC-3’
</td>
<td style="text-align:left;font-family: monospace;">
478-499<br>5’-GGCCTTAACAAAGTCAAAGCC-3’
</td>
<td style="text-align:left;width: 25%; ">
Binding site for NNR, an anaerobic transcriptional activator protein of the FNR family
</td>
</tr>
<tr>
<td style="text-align:left;font-family: monospace;">
Repeat 2: 80-89<br>5’-TTGCGCGCAA-3’
</td>
<td style="text-align:left;font-family: monospace;">
500-509<br>5’-CCGCGCGCGG-3’
</td>
<td style="text-align:left;width: 25%; ">
No function assigned
</td>
</tr>
<tr>
<td style="text-align:left;font-family: monospace;">
Repeat 3: 1937-1946<br>5’-GGTTACGGACC-3’
</td>
<td style="text-align:left;font-family: monospace;">
2357-2366<br>5’-GGTTGCAACC-3’
</td>
<td style="text-align:left;width: 25%; ">
No function assigned
</td>
</tr>
<tr>
<td style="text-align:left;font-family: monospace;">
Repeat 4: 1952-1969<br>5’-GGGGGCGTTCGCGCCCCC-3’
</td>
<td style="text-align:left;font-family: monospace;">
2372-2389<br>5’-GGGGGCGTTCGCGCCCCC-3’
</td>
<td style="text-align:left;width: 25%; ">
Repeat structure assigned as binding site for an oxygen responsive element (ORE). 5’ GGGGGC assigned as a -10 RNA polymerase binding site
</td>
</tr>
</tbody>
</table>
<p>Two partial open reading frames can be identified in the <em>T. pantotropha</em> DNA sequence (Figure 3.2). The first of these begins at position 17 and reads in the opposite orientation to the <em>nirS</em> gene. The first four amino acids are Met-Ala-Met-Arg. This appears to correspond to the start of the <em>nirI</em> gene (Section 3.4.1), which in <em>P. denitrificans</em> PD1222 begins Met-Ala-Met-Gly <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>. The second ORF at position 2001 begins Met-Ala-Gly; this seems to be the start of the <em>nirE</em> gene (Section 3.4.1), which begins with an identical sequence in P.denitrificans PD1222 <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>. The <em>nirS</em> genes of <em>T. pantotropha</em> and <em>P. denitrificans</em> PD1222 are 94.1% identical and the GC content of the <em>nirS</em> gene is 64%.</p>
</div>
<div id="primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha" class="section level3 hasAnchor" number="3.3.2">
<h3><span class="header-section-number">3.3.2</span> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The <em>nirS</em> gene codes for a protein of 596 amino acids, the first 29 of which are predicted to be a cleavable periplasmic targeting sequence (see Discussion, Section 3.4.3 and Figure 3.3). The six previously published cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences were downloaded from the GenBank and Swissprot databases and processed using the program SignalP, which predicts targeting sequences according to the rules developed by von Heijne <span class="citation">[<a href="#ref-von_heijne_new_1986">255</a>]</span> and uses algorithms specifically tested on protein sequences from Gram-negative bacteria <span class="citation">[<a href="#ref-nielsen_identification_1997">256</a>]</span>. The sequences were edited to remove the predicted targeting sequence and aligned using the ClustalW program <span class="citation">[<a href="#ref-thompson_clustal_1994">191</a>]</span>. The multiple sequence alignment was then formatted using the program ALSCRIPT <span class="citation">[<a href="#ref-barton_alscript_1993">257</a>]</span> to generate Figure 3.4. In this figure, identity between four or more of the sequences is indicated by black shading and similarity is indicated by various grades of grey shading, using amino acid similarity groups based on physicochemical properties. Other shadings are described in the figure legend.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-3"></span>
<img src="figure/fig3-3.png" alt="Predicted cleavage site of the periplasmic targeting sequence in cytochrome _cd_$_1$ from _T. pantotropha_. The SignalP program (Nielsen _et al_., 1997 [@nielsen_identification_1997]) predicts the cleavage site on the basis of three scores, termed C, S and Y. The residue lying immediately after the cleavage site has high C and Y scores and a sharp gradient in the S score. In the above figure it can be seen that the residue Q30 is predicted to be this residue. As described in the text, an N-terminal glutamine as the first residue of the mature protein is fully consistent with experimental data." width="75%" />
<p class="caption">
Figure 3.3: Predicted cleavage site of the periplasmic targeting sequence in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>. The SignalP program (Nielsen <em>et al</em>., 1997 <span class="citation">[<a href="#ref-nielsen_identification_1997">256</a>]</span>) predicts the cleavage site on the basis of three scores, termed C, S and Y. The residue lying immediately after the cleavage site has high C and Y scores and a sharp gradient in the S score. In the above figure it can be seen that the residue Q30 is predicted to be this residue. As described in the text, an N-terminal glutamine as the first residue of the mature protein is fully consistent with experimental data.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-4"></span>
<img src="figure/fig3-4.png" alt="Multiple alignment of cytochrome _cd_$_1$ sequences. Six previously published cytochrome _cd_$_1$ sequences (see main text for details) were downloaded from the GenBank and SwissProt databases. The sequence of cytochrome _cd_$_1$ from _T. pantotropha_ was added to the group. All sequences were edited to remove the predicted periplasmic targeting sequence and aligned using the Program ClustalW [@thompson_clustal_1994]. The multiple alignment was then processed using the program ALSCRIPT [@barton_alscript_1993], to shade homologous regions and other features of interest. Identity within four or more sequences is indicated by black shading; similarity is indicated by levels of grey shading based on the physicochemical properties of the amino acids. Conserved methionine and histidine residues which could be potential haem ligands are indicated in white on dark-grey and conserved aspartate residues at the end of the third strand in each $\beta$ propeller (see Appendix A) are shown in white on light-grey. The haem ligands His-17, His-69 (_c_-haem), His-200 and Tyr-25 (_d_$_1$ haem) in the oxidised _T. pantotropha_ enzyme are shown by open boxes. The secondary structure elements of the _T. pantotropha_ enzyme  [@fulop_anatomy_1995] are shown above the alignment. Abbreviations: tp9263, _T. pantotropha_ LMD 92.63; pd1222, _P. denitrificans_ PD1222; pd12442, _P. denitrificans_ IFO 12442; pss14405, _Ps. stutzeri_ Zobell; pss300, _Ps. stutzeri_ JM300; psa6750, _Ps. aeruginosa_ NTCC 6750; aeh16, _A. eutrophus_ H16." width="75%" />
<p class="caption">
Figure 3.4: Multiple alignment of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences. Six previously published cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences (see main text for details) were downloaded from the GenBank and SwissProt databases. The sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> was added to the group. All sequences were edited to remove the predicted periplasmic targeting sequence and aligned using the Program ClustalW <span class="citation">[<a href="#ref-thompson_clustal_1994">191</a>]</span>. The multiple alignment was then processed using the program ALSCRIPT <span class="citation">[<a href="#ref-barton_alscript_1993">257</a>]</span>, to shade homologous regions and other features of interest. Identity within four or more sequences is indicated by black shading; similarity is indicated by levels of grey shading based on the physicochemical properties of the amino acids. Conserved methionine and histidine residues which could be potential haem ligands are indicated in white on dark-grey and conserved aspartate residues at the end of the third strand in each <span class="math inline">\(\beta\)</span> propeller (see Appendix A) are shown in white on light-grey. The haem ligands His-17, His-69 (<em>c</em>-haem), His-200 and Tyr-25 (<em>d</em><span class="math inline">\(_1\)</span> haem) in the oxidised <em>T. pantotropha</em> enzyme are shown by open boxes. The secondary structure elements of the <em>T. pantotropha</em> enzyme <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span> are shown above the alignment. Abbreviations: tp9263, <em>T. pantotropha</em> LMD 92.63; pd1222, <em>P. denitrificans</em> PD1222; pd12442, <em>P. denitrificans</em> IFO 12442; pss14405, <em>Ps. stutzeri</em> Zobell; pss300, <em>Ps. stutzeri</em> JM300; psa6750, <em>Ps. aeruginosa</em> NTCC 6750; aeh16, <em>A. eutrophus</em> H16.
</p>
</div>
<p>The multiple sequence file was processed using three other computational methods in order to compare the seven sequences. Each cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequence was compared to the other six using the GCG program BESTFIT <span class="citation">[<a href="#ref-devereux_comprehensive_1984">248</a>]</span>. This program generates a percentage identity and similarity for each pair, the results of which are summarised in Table 3.3.</p>
<table>
<caption>
<span id="tab:table33">Table 3.3: </span>Matrix of similarities and identities between the cytochromes <em>cd</em><span class="math inline">\(_1\)</span>.Pairwise comparisons were determined using the program BESTFIT, part of the GCG 8.0 package. In each box the upper bold figure is the percentage similarity for the pair, the lower is percentage identity. Sequence abbreviations are: <em>Tp9263</em>, <em>T. pantotropha</em> LMD 92.63, <em>Pd1222</em>, <em>P. denitrificans</em> PD1222, <em>Pd12442</em>, <em>P. denitrificans</em> IFO 12442, <em>Pss14405</em>, <em>Ps. stutzeri</em> ATCC 14405, <em>Pss300</em>, <em>Ps. stutzeri</em> JM300, <em>Psa6750</em>, <em>Ps. aeruginosa</em> NTCC 6750, <em>AeH16</em>, <em>A. eutrophus</em> H16.
</caption>
<tbody>
<tr>
<td style="text-align:left;">
<em>Pd1222</em>
</td>
<td style="text-align:left;">
<strong>97.8</strong><br />97.1
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pd12442</em>
</td>
<td style="text-align:left;">
<strong>95.0</strong><br />91.6
</td>
<td style="text-align:left;">
<strong>96.7</strong><br />94.0
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pss14405</em>
</td>
<td style="text-align:left;">
<strong>73.5</strong><br />55.0
</td>
<td style="text-align:left;">
<strong>78.5</strong><br />66.1
</td>
<td style="text-align:left;">
<strong>73.7</strong><br />55.4
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Pss300</em>
</td>
<td style="text-align:left;">
<strong>71.8</strong><br />53.8
</td>
<td style="text-align:left;">
<strong>72.4</strong><br />54.4
</td>
<td style="text-align:left;">
<strong>71.6</strong><br />52.7
</td>
<td style="text-align:left;">
<strong>93.7</strong><br />89.2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>Psa6750</em>
</td>
<td style="text-align:left;">
<strong>78.5</strong><br />65.8
</td>
<td style="text-align:left;">
<strong>78.5</strong><br />66.1
</td>
<td style="text-align:left;">
<strong>77.5</strong><br />64.1
</td>
<td style="text-align:left;">
<strong>75.2</strong><br />58.9
</td>
<td style="text-align:left;">
<strong>73.7</strong><br />57.2
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>AeH16</em>
</td>
<td style="text-align:left;">
<strong>78.6</strong><br />61.8
</td>
<td style="text-align:left;">
<strong>78.8</strong><br />62.4
</td>
<td style="text-align:left;">
<strong>77.6</strong><br />60.8
</td>
<td style="text-align:left;">
<strong>74.1</strong><br />59.2
</td>
<td style="text-align:left;">
<strong>73.2</strong><br />59.2
</td>
<td style="text-align:left;">
<strong>80.3</strong><br />65.5
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
<em>Tp9263</em>
</td>
<td style="text-align:left;">
<em>Pd1222</em>
</td>
<td style="text-align:left;">
<em>Pd12442</em>
</td>
<td style="text-align:left;">
<em>Pss14405</em>
</td>
<td style="text-align:left;">
<em>Pss300</em>
</td>
<td style="text-align:left;">
<em>Psa6750</em>
</td>
</tr>
</tbody>
</table>
<p>Phylogenetic relationships between the aligned sequences were inferred using ClustalW. To generate the unrooted phylogram shown in Figure 3.5a, the multiple alignment was processed using a neighbour-joining method <span class="citation">[<a href="#ref-saitou_neighbor-joining_1987">192</a>]</span> to calculate divergence distance between the sequences. The statistical significance of the tree was analysed by the method of bootstrapping <span class="citation">[<a href="#ref-felsenstein_confidence_1985">193</a>]</span> with 1000 trials and the output was visualised with the program TreeView <span class="citation">[<a href="#ref-page_treeview_1996">194</a>]</span>. Figure 3.5b is a phylogenetic tree inferred from 16S ribosomal RNA sequences and used for comparison with the protein sequence tree.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-5"></span>
<img src="figure/fig3-5.png" alt="Phylogenetic tree of cytochrome _cd_$_1$ sequences compared with a phylogenetic tree of 16S rRNA sequences from the same organisms. (a) The sequences were aligned using the ClustalW program and the tree was calculated using the neighbour-joining method of Saitou and Nei [@saitou_neighbor-joining_1987]. The tree was then analysed using the method of bootstrapping, with 1000 trials and visualised with the program TreeView. Sequence abbreviations are as in Table 3.3. (b) The 16S rRNA tree was calculated in the same way as for the cytochromes _cd_$_1$, except that gaps were ignored due to the partial nature of the _T. pantotropha_ LMD 92.63 and _P. denitrificans_ PD1222 sequences (obtained from Dr. D.J. Richardson, University of East Anglia). Complete 16S rRNA gene sequences for _Ps. stutzeri_ ATCC 14405, _Ps. stutzeri_ JM300, _Ps. aeruginosa_ DSM 50071 (Psa50071) and _P. denitrificans_ LMG 4218 (Pd4218) were taken from the GenBank database." width="60%" />
<p class="caption">
Figure 3.5: Phylogenetic tree of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences compared with a phylogenetic tree of 16S rRNA sequences from the same organisms. (a) The sequences were aligned using the ClustalW program and the tree was calculated using the neighbour-joining method of Saitou and Nei <span class="citation">[<a href="#ref-saitou_neighbor-joining_1987">192</a>]</span>. The tree was then analysed using the method of bootstrapping, with 1000 trials and visualised with the program TreeView. Sequence abbreviations are as in Table 3.3. (b) The 16S rRNA tree was calculated in the same way as for the cytochromes <em>cd</em><span class="math inline">\(_1\)</span>, except that gaps were ignored due to the partial nature of the <em>T. pantotropha</em> LMD 92.63 and <em>P. denitrificans</em> PD1222 sequences (obtained from Dr. D.J. Richardson, University of East Anglia). Complete 16S rRNA gene sequences for <em>Ps. stutzeri</em> ATCC 14405, <em>Ps. stutzeri</em> JM300, <em>Ps. aeruginosa</em> DSM 50071 (Psa50071) and <em>P. denitrificans</em> LMG 4218 (Pd4218) were taken from the GenBank database.
</p>
</div>
<p>Complete or partial rRNA sequence is known for four of the organisms under study as indicated in the figure legend; a different strain of <em>Ps. aeruginosa</em> and a third <em>P. denitrificans</em> strain were added to this group for comparison. The incomplete 16S rRNA sequences of <em>P. denitrificans</em> PD1222 and <em>T. pantotropha</em> were obtained from Dr. D.J. Richardson, University of East Anglia (J.P. Carter, S. Spiro and D.J. Richardson, unpublished results). The tree was produced in a similar manner to that for the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences, except that gaps in the alignment were ignored owing to the partial nature of some of the 16S rRNA sequences. Finally the amino acid composition, molecular weight and predicted isoelectric point of each sequence was analysed using the program ProtParam <span class="citation">[<a href="#ref-appel_new_1994">258</a>]</span>. These data are presented in Table 3.4.</p>
<table>
<caption>
<span id="tab:table34">Table 3.4: </span>Comparison of the physicochemical properties of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from different sources. Sequence abbreviations are as in Table 3.3 and Figure 3.4. Cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences were retrieved from the GenBank and SwissProt databases. The data presented in the table was obtained using the program ProtParam <span class="citation">[<a href="#ref-appel_new_1994">258</a>]</span>.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Sequence ID
</th>
<th style="text-align:left;">
Tp9263
</th>
<th style="text-align:left;">
Pd1222
</th>
<th style="text-align:left;">
Pd12442
</th>
<th style="text-align:left;">
Ps14405
</th>
<th style="text-align:left;">
Ps300
</th>
<th style="text-align:left;">
Pa6750
</th>
<th style="text-align:left;">
Aeh16
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
<strong>Length</strong>
</td>
<td style="text-align:left;">
567
</td>
<td style="text-align:left;">
568
</td>
<td style="text-align:left;">
568
</td>
<td style="text-align:left;">
534
</td>
<td style="text-align:left;">
527
</td>
<td style="text-align:left;">
543
</td>
<td style="text-align:left;">
529
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Molecular weight (Da)</strong>
</td>
<td style="text-align:left;">
62 538.9
</td>
<td style="text-align:left;">
62 596.1
</td>
<td style="text-align:left;">
62 687.5
</td>
<td style="text-align:left;">
59 532.7
</td>
<td style="text-align:left;">
58 862.8
</td>
<td style="text-align:left;">
60 178.2
</td>
<td style="text-align:left;">
58 000.0
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Theoretical pI</strong>
</td>
<td style="text-align:left;">
4.75
</td>
<td style="text-align:left;">
4.73
</td>
<td style="text-align:left;">
5.02
</td>
<td style="text-align:left;">
6.44
</td>
<td style="text-align:left;">
6.59
</td>
<td style="text-align:left;">
7.39
</td>
<td style="text-align:left;">
8.44
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Amino acid (%)</strong>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Ala</strong>
</td>
<td style="text-align:left;">
8.8
</td>
<td style="text-align:left;">
9.3
</td>
<td style="text-align:left;">
9.3
</td>
<td style="text-align:left;">
7.3
</td>
<td style="text-align:left;">
8.0
</td>
<td style="text-align:left;">
7.9
</td>
<td style="text-align:left;">
9.8
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Arg</strong>
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
3.5
</td>
<td style="text-align:left;">
4.0
</td>
<td style="text-align:left;">
3.6
</td>
<td style="text-align:left;">
4.4
</td>
<td style="text-align:left;">
4.2
</td>
<td style="text-align:left;">
4.3
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Asn</strong>
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
4.0
</td>
<td style="text-align:left;">
4.3
</td>
<td style="text-align:left;">
4.2
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
3.4
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Asp</strong>
</td>
<td style="text-align:left;">
7.8
</td>
<td style="text-align:left;">
7.6
</td>
<td style="text-align:left;">
6.9
</td>
<td style="text-align:left;">
6.4
</td>
<td style="text-align:left;">
6.8
</td>
<td style="text-align:left;">
7.0
</td>
<td style="text-align:left;">
7.4
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Cys</strong>
</td>
<td style="text-align:left;">
0.4
</td>
<td style="text-align:left;">
0.4
</td>
<td style="text-align:left;">
0.5
</td>
<td style="text-align:left;">
0.4
</td>
<td style="text-align:left;">
0.4
</td>
<td style="text-align:left;">
0.4
</td>
<td style="text-align:left;">
0.4
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Gln</strong>
</td>
<td style="text-align:left;">
3.2
</td>
<td style="text-align:left;">
3.3
</td>
<td style="text-align:left;">
3.2
</td>
<td style="text-align:left;">
2.4
</td>
<td style="text-align:left;">
2.3
</td>
<td style="text-align:left;">
4.2
</td>
<td style="text-align:left;">
2.3
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Glu</strong>
</td>
<td style="text-align:left;">
7.6
</td>
<td style="text-align:left;">
7.7
</td>
<td style="text-align:left;">
7.2
</td>
<td style="text-align:left;">
6.6
</td>
<td style="text-align:left;">
6.3
</td>
<td style="text-align:left;">
5.0
</td>
<td style="text-align:left;">
4.5
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Gly</strong>
</td>
<td style="text-align:left;">
8.3
</td>
<td style="text-align:left;">
8.5
</td>
<td style="text-align:left;">
8.5
</td>
<td style="text-align:left;">
7.5
</td>
<td style="text-align:left;">
7.0
</td>
<td style="text-align:left;">
7.7
</td>
<td style="text-align:left;">
7.9
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>His</strong>
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
2.6
</td>
<td style="text-align:left;">
2.7
</td>
<td style="text-align:left;">
2.9
</td>
<td style="text-align:left;">
2.5
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Ile</strong>
</td>
<td style="text-align:left;">
5.1
</td>
<td style="text-align:left;">
5.3
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
5.8
</td>
<td style="text-align:left;">
5.9
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
5.5
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Leu</strong>
</td>
<td style="text-align:left;">
7.6
</td>
<td style="text-align:left;">
7.6
</td>
<td style="text-align:left;">
7.4
</td>
<td style="text-align:left;">
6.4
</td>
<td style="text-align:left;">
6.1
</td>
<td style="text-align:left;">
6.8
</td>
<td style="text-align:left;">
7.6
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Lys</strong>
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
5.6
</td>
<td style="text-align:left;">
8.6
</td>
<td style="text-align:left;">
8.2
</td>
<td style="text-align:left;">
7.7
</td>
<td style="text-align:left;">
8.1
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Met</strong>
</td>
<td style="text-align:left;">
2.1
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
2.3
</td>
<td style="text-align:left;">
2.6
</td>
<td style="text-align:left;">
2.8
</td>
<td style="text-align:left;">
2.0
</td>
<td style="text-align:left;">
1.7
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Phe</strong>
</td>
<td style="text-align:left;">
3.2
</td>
<td style="text-align:left;">
3.2
</td>
<td style="text-align:left;">
3.2
</td>
<td style="text-align:left;">
2.6
</td>
<td style="text-align:left;">
2.7
</td>
<td style="text-align:left;">
2.6
</td>
<td style="text-align:left;">
2.6
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Pro</strong>
</td>
<td style="text-align:left;">
6.0
</td>
<td style="text-align:left;">
6.0
</td>
<td style="text-align:left;">
6.3
</td>
<td style="text-align:left;">
5.2
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
6.3
</td>
<td style="text-align:left;">
6.0
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Ser</strong>
</td>
<td style="text-align:left;">
5.1
</td>
<td style="text-align:left;">
4.8
</td>
<td style="text-align:left;">
4.6
</td>
<td style="text-align:left;">
5.6
</td>
<td style="text-align:left;">
5.5
</td>
<td style="text-align:left;">
6.4
</td>
<td style="text-align:left;">
5.1
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Thr</strong>
</td>
<td style="text-align:left;">
7.4
</td>
<td style="text-align:left;">
7.0
</td>
<td style="text-align:left;">
7.0
</td>
<td style="text-align:left;">
7.3
</td>
<td style="text-align:left;">
7.8
</td>
<td style="text-align:left;">
5.7
</td>
<td style="text-align:left;">
7.4
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Trp</strong>
</td>
<td style="text-align:left;">
1.9
</td>
<td style="text-align:left;">
1.9
</td>
<td style="text-align:left;">
1.9
</td>
<td style="text-align:left;">
2.2
</td>
<td style="text-align:left;">
2.5
</td>
<td style="text-align:left;">
2.0
</td>
<td style="text-align:left;">
1.7
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Tyr</strong>
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
3.9
</td>
<td style="text-align:left;">
3.9
</td>
<td style="text-align:left;">
3.4
</td>
<td style="text-align:left;">
3.7
</td>
<td style="text-align:left;">
4.0
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Val</strong>
</td>
<td style="text-align:left;">
6.5
</td>
<td style="text-align:left;">
6.2
</td>
<td style="text-align:left;">
6.2
</td>
<td style="text-align:left;">
8.6
</td>
<td style="text-align:left;">
7.8
</td>
<td style="text-align:left;">
8.1
</td>
<td style="text-align:left;">
8.0
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="discussion" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Discussion<a href="3-chapter03.html#discussion" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="features-of-the-dna-sequence" class="section level3 hasAnchor" number="3.4.1">
<h3><span class="header-section-number">3.4.1</span> Features of the DNA sequence<a href="3-chapter03.html#features-of-the-dna-sequence" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The <em>nirS</em> gene and its immediate flanking regions are very similar in both <em>T. pantotropha</em> and the two previously published <em>P. denitrificans</em> sequences; in particular the <em>nirS</em> open reading frame from <em>T. pantotropha</em> is 94.1% identical to that of <em>P. denitrificans</em> PD1222. The GC content of the two genes is 64%, a typical figure for genes from <em>Paracoccus</em> species, reflecting the high GC content of the genome (68% <span class="citation">[<a href="#ref-steinrucke_genetics_1993">259</a>]</span>).</p>
<p>In <em>P. denitrificans</em> PD1222, an open reading frame was identified both upstream and downstream of the <em>nirS</em> gene. The upstream ORF was designated <em>nirI</em> and has some sequence homology to the regulatory protein <em>nosR</em> <span class="citation">[<a href="#ref-hoeren_sequence_1993">96</a>,<a href="#ref-de_boer_isolation_1994">141</a>]</span>, in that it is predicted to be an integral membrane protein with a putative iron-sulphur cluster. It is transcribed divergently from the <em>nirS</em> gene. Insertional mutation in <em>nirI</em> results in the loss of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>, Chapter 7. The <em>nirI</em> ORF is also present in the sequence from <em>P. denitrificans</em> IFO 12442, although Ohshima <em>et al</em>. (1993) <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span> failed to identify it as such. Translation of the first 37 codons from the latter sequence shows 97% identity to the <em>P. denitrificans</em> PD1222 NirI protein, with a substitution of Phe for Leu at position 33. In the <em>T. pantotropha</em> sequence, the start of the <em>nirI</em> gene can also be discerned, with a substitution of Arg for Gly at position 4.</p>
<p>Downstream of <em>nirs</em>, de Boer <em>et al</em>. (1994) <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span> identified the gene <em>nirE</em>. The product of this gene shows similarity to a number of S-adenosyl-L-methionine uroporphyrinogen (III) methyltransferases such as the sirohaem biosynthetic gene <em>cysG</em> of <em>E. coli</em> <span class="citation">[<a href="#ref-peakman_nucleotide_1990">42</a>]</span>, and <em>cobA</em>, a cobalamin biosynthesis gene of <em>S. typhimurium</em> <span class="citation">[<a href="#ref-roth_characterization_1993">260</a>]</span>, and is thought to be involved in methyl transfer to the isobacteriochlorin ring during the biosynthesis of haem <em>d</em><span class="math inline">\(_1\)</span>. Insertional inactivation of <em>nirE</em> results in the formation of inactive cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, lacking the <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>. This open reading frame was identified (but not characterised) in <em>P. denitrificans</em> IFO 12442 <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span>.</p>
<p>Translation of the first 27 codons from the <em>P. denitrificans</em> IFO 12242 gene shows 81% identity to the <em>P. denitrificans</em> PD1222 NirE product. Once again, the start of the <em>nirE</em> gene can be identified in the <em>T. pantotropha</em> sequence. These sequence data indicate that the structure of the <em>nir</em> gene cluster in the immediate vicinity of <em>nirS</em> is highly conserved in <em>T. pantotropha</em> and the two strains of <em>Paracoccus denitrificans</em> characterised to date. Further mapping of this region, described in Chapter 4, confirms this observation.</p>
<p>The non-coding regions flanking the <em>nirS</em> gene contain a number of inverted repeats which are also quite highly conserved. In the <em>T. pantotropha</em> sequence, the first of these (repeat 1, Table 3.1) begins 81 bp upstream of the <em>nirS</em> initiation codon. In <em>P. denitrificans</em> PD1222 this sequence was identified as containing a putative binding site for NNR, a transcriptional activator protein belonging to the FNR family <span class="citation">[<a href="#ref-spiro_fnr_1994">188</a>]</span>, Chapter 7. The NNR gene has been cloned and sequenced in this organism <span class="citation">[<a href="#ref-van_spanning_nitrite_1995">197</a>]</span> and insertional mutagenesis results in the loss of nitrite and nitric oxide reductases. The putative NNR binding sequences are very similar, with a single substitution of T for A at position 12 in the <em>T. pantotropha</em> sequence. This lies in the non-conserved region of the NNR box consensus and is unlikely to be important. An identical sequence to the <em>P. denitrificans</em> PD1222 NNR box is found in the <em>P. denitrificans</em> IFO 12442 sequence, although Ohshima <em>et al</em>. (1993) <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span> did not identify it as such. Instead, they assigned the TAA trinucleotide as a stop sequence for a putative upstream open reading frame and included part of the NNR box sequence in a larger, downstream repeat. Both of these assignments appear to be erroneous in the light of other data. Expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in all three organisms is, therefore, concluded to be transcriptionally regulated by the NNR protein. An alignment of these three NNR boxes with the FNR consensus sequence is shown in Figure 3.6.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-6"></span>
<img src="figure/fig3-6.png" alt="Putative NNR binding sites in the upstream regions of the _nirS_ gene aligned with the FNR consensus binding site. The NNR boxes upstream of _nirS_ in _T. pantotropha_ LMD 92.63 (Tp9263), _P. denitrificans_ PD1222 (Pd1222) and _P. denitrificans_ IFO 12442 (Pd12442) are shown as part of a larger inverted repeat, aligned to the consensus FNR binding site derived from anaerobically-regulated _E. coli_ genes [@spiro_fnr_1994]." width="60%" />
<p class="caption">
Figure 3.6: Putative NNR binding sites in the upstream regions of the <em>nirS</em> gene aligned with the FNR consensus binding site. The NNR boxes upstream of <em>nirS</em> in <em>T. pantotropha</em> LMD 92.63 (Tp9263), <em>P. denitrificans</em> PD1222 (Pd1222) and <em>P. denitrificans</em> IFO 12442 (Pd12442) are shown as part of a larger inverted repeat, aligned to the consensus FNR binding site derived from anaerobically-regulated <em>E. coli</em> genes <span class="citation">[<a href="#ref-spiro_fnr_1994">188</a>]</span>.
</p>
</div>
<p>Immediately downstream of the NNR box is a small, perfect inverted repeat (repeat 2, Table 3.2). This structure is also seen in the two <em>P. denitrificans</em> sequences but interestingly, in both cases, the first and last dinucleotides are substituted by CC and GG respectively. Hence the repeat structure is preserved, although the sequences differ. No putative function has been assigned to this repeat, but its structural conservation implies that it has some importance. A possible function is the binding of a transcriptional repressor protein; this possibility is explored more fully in Chapter 7.</p>
<p>Two more repeat sequences are found between the <em>nirS</em> and nirE genes in <em>T. pantotropha</em>. The first of these (repeat 3, Table 3.2) was identified in the <em>P. denitrificans</em> PD1222 sequence as a perfect 10 bp inverted repeat and is also found in the <em>P. denitrificans</em> IFO 12442 sequence. However, in <em>T. pantotropha</em> the sequence differs at positions five and seven and so is an imperfect repeat. The significance of this repeat is unclear.</p>
<p>Finally, a large imperfect repeat (repeat 4, Table 3.2) is found 6 bp downstream of repeat 3 in <em>T. pantotropha</em>. The sequence of this repeat is perfectly conserved in the <em>P. denitrificans</em> PD1222 sequence, but less so in <em>P. denitrificans</em> IFO 12442 where the corresponding sequence differs at positions one and five. In <em>P. denitrificans</em> PD1222 two assignments were made to this sequence; the first GGGGGC hexanucleotide was suggested to be a putative -10 promoter sequence for the <em>nirE</em> gene <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span>, based on the promoter consensus for purple non-sulphur bacteria of Steinrücke and Ludwig (1993) <span class="citation">[<a href="#ref-steinrucke_genetics_1993">259</a>]</span>, whilst the structure as a whole was suggested by the same authors to be a putative oxygen responsive element (ORE). An ORE is a region of dyad symmetry found upstream of genes that are regulated by oxygen; it was first identified in photosynthetic genes from the genus <em>Rhodobacter</em> <span class="citation">[<a href="#ref-klug_dna_1991">261</a>]</span> and is thought to bind a protein that represses transcription under aerobic conditions. Steinrücke and Ludwig (1993) <span class="citation">[<a href="#ref-steinrucke_genetics_1993">259</a>]</span> have proposed that some <em>Paracoccus</em> are also ORE-regulated. Both of the assignments proposed by de Boer <em>et al</em>. (1994) <span class="citation">[<a href="#ref-de_boer_isolation_1994">141</a>]</span> were tentative and both now appear unlikely. Based on the finding that <em>nirS</em> is transcribed as a single mRNA species (see Chapter 7 for data and discussion), this feature could be involved in transcriptional termination.</p>
</div>
<div id="analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1" class="section level3 hasAnchor" number="3.4.2">
<h3><span class="header-section-number">3.4.2</span> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>A ClustalW multiple alignment of the six previously published cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences plus the <em>T. pantotropha</em> sequence is shown in Figure 3.4. It is clear that as a group, the cytochromes <em>cd</em><span class="math inline">\(_1\)</span> are extremely highly conserved. In this alignment identity between four or more sequences is indicated by black shading. A large number of identical regions are apparent, particularly around the <em>c</em>-haem binding region and in the haem <em>d</em><span class="math inline">\(_1\)</span> binding domain (which corresponds to residues 135-567 in the mature <em>T. pantotropha</em> sequence). The <em>T. pantotropha</em> enzyme shows most homology to the two <em>P. denitrificans</em> sequences (97.1% identity to <em>P. denitrificans</em> PD1222); of all the pairwise comparisons, the lowest identity is still considerable (52.7% between <em>Ps. stutzeri</em> JM300 and <em>P. denitrificans</em> IFO 12442). However, variation is apparent within the N-terminal region, a point which is returned to presently.</p>
<p>The ClustalW alignment was used as the basis for construction of a phylogenetic tree. To generate this tree, the ClustalW program uses the neighbour-joining method of Saitou and Nei <span class="citation">[<a href="#ref-saitou_neighbor-joining_1987">192</a>]</span>. Briefly, this algorithm compares all pairs of sequences in an alignment and computes a percentage divergence between them. A correction is included to take account of the possibility that multiple mutations can occur at one site. Pairs of sequences are then joined and connected to the other sequences by an internal branch, such that the total branch length is minimised. The joined sequences are merged and the process is repeated, so an alignment of N sequences allows an unrooted tree with N-3 internal branches. To place confidence limits on the tree, the method of bootstrapping <span class="citation">[<a href="#ref-felsenstein_confidence_1985">193</a>]</span> was employed. This process is quite complex, but essentially for an alignment of N residues in length, a random sample of N sites is taken, from which a distance matrix and tree can be calculated. This process is repeated a large number of times (1000 is a typical figure) and a site may be selected several times, or not at all, in any given sample. If the sample trees are similar to one another and the tree from the complete data set, the tree is strongly supported. A particular grouping of sequences is considered significant at the 95% confidence level if it occurs in 95% or more of the bootstrap samples. The biological significance of the tree, in terms of phylogenetic relationships will always be a rather subjective judgement, based on factors such as agreement between trees from different data sources (e.g. protein and rRNA sequences) and other taxonomic schemes for grouping organisms.</p>
<p>Figure 3.5a shows that the <em>T. pantotropha</em> sequence lies closest to that from <em>P. denitrificans</em> PD1222 and that these two form a group with <em>P. denitrificans</em> IFO 12442. This group is most distant from the two strains of <em>Ps. stutzeri</em> which group together. The proteins from <em>Ps. aeruginosa</em> and <em>A. eutrophus</em> are approximately equidistant from these two groups and closer to each other than the other enzymes. This grouping is what would be expected from the current consensus opinion of the phylogeny of the <em>Proteobacteria</em> <span class="citation">[<a href="#ref-olsen_winds_1994">262</a>]</span>. For comparison, a phylogenetic tree constructed from 16S ribosomal RNA sequences is shown in Figure 3.5b. 16S rRNA sequences are widely used to infer phylogeny (the advantages and limitations of the method are discussed in Fox <em>et al</em>. (1992) <span class="citation">[<a href="#ref-fox_how_1992">263</a>]</span>, and the results agree closely with that obtained from the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences. The main results of this analysis can be summarised as follows: (1) comparison of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and 16S rRNA sequences between the organisms under consideration gives estimates of relatedness which agree well with one another and with current ideas about bacterial taxonomy and (2) the high similarity of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences indicates an interspecies transfer of the <em>nirS</em> gene in the evolutionary past and/or a strong relationship between structure and function in the protein.</p>
</div>
<div id="relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1" class="section level3 hasAnchor" number="3.4.3">
<h3><span class="header-section-number">3.4.3</span> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Sequencing of the <em>nirS</em> gene from <em>T. pantotropha</em> has allowed the crystal structure of the enzyme, originally deduced from electron density by comparison with the <em>P. denitrificans</em> PD1222 sequence <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span>, to be checked for errors and refined <span class="citation">[<a href="#ref-baker_cytochrome_1997">264</a>]</span>. There were 14 discrepancies between the density map and the <em>P. denitrificans</em> PD1222 sequence in the original publication of the oxidised crystal structure; the translated protein sequence has confirmed 11 of these (three were shown to be the same as the <em>P. denitrificans</em> sequence), as well as revealing four others that were not apparent earlier (counting from the N-terminus of the mature protein these are K2OR, G101A, E339S and A499T, the corrected amino acid second). The oxidised crystal structure has now been determined at the higher resolution of 1.28 <span class="math inline">\(\unicode{x212b}\)</span> and is fully consistent with the protein sequence presented in this chapter.</p>
<p>The crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> contains 559 residues, giving a calculated molecular weight of 62 249 Da per monomer including the covalently-bound <em>c</em> haem moiety. The monomeric molecular weight as measured by electrospray mass spectrometry was 63 091 +/- 8 Da <span class="citation">[<a href="#ref-moir_purification_1993">101</a>]</span>, a discrepancy of 842 Da (when using the <em>P. denitrificans</em> sequence for comparison). From this, it was argued that eight N-terminal residues of the mature protein in the crystal structure were not visible, due to disorder in the crystal packing <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span>. This has been confirmed by sequencing of the <em>nirS</em> gene. The <em>T. pantotropha</em> <em>nirS</em> gene encodes a protein of 596 amino acids, with a calculated molecular weight of 66 045 Da, including the <em>c</em> haem. The first 29 residues are predicted to be a cleavable periplasmic targeting sequence (Figure 3.3), giving a molecular weight for the mature protein of 63 141 Da. The discrepancy between the latter figure and the molecular weight calculated from the crystal structure is 892 Da, which is accounted for by the first eight residues in the mature <em>T. pantotropha</em> sequence. These residues are QEQVAPPK, an additional molecular weight of 896 Da. The N-terminus of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> is known to be blocked, as it is resistant to Edman degradation <span class="citation">[<a href="#ref-moir_aspects_1993">240</a>]</span>. An N-terminal glutamine, as postulated above, can account for this by cyclisation to pyroglutamate. The N-terminus of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>P. denitrificans</em> IFO 12442 has also been reported to be blocked <span class="citation">[<a href="#ref-ohshima_cloning_1993">140</a>]</span>. However, the equivalent eight residue sequence in the two Paracoccus sequences is QEQAAPPK and a further difference (counting from the N-terminus of the uncleaved protein, residue 27 is alanine in the <em>Paracoccus</em> sequences, valine in <em>T. pantotropha</em>) makes the predicted signal cleavage site one residue earlier in both <em>Paracoccus</em> sequences, giving AQEQAAPPK as the mature N-terminus. This would not be expected to show blockage, but in the absence of further experimental data, the predicted mature sequences have been used in this chapter and the effect on the alignment is insignificant.</p>
<p>The crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> showed that the <em>c</em> haem was ligated by His-17 and His-69 (numbering is with the targeting sequence removed) and the <em>d</em><span class="math inline">\(_1\)</span> haem by Tyr-25 and His-200 <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span>. Both of these arrangements were somewhat unexpected; first, bis-histidyl ligation is normally associated with low-potential haems <span class="citation">[<a href="#ref-moore_cytochromes_1990">151</a>,<a href="#ref-wallace_functional_1992">265</a>]</span>, whereas the <em>c</em>-haem in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> has a midpoint redox potential of around 243 mV (A. Koppenhöfer, unpublished data). Second, tyrosyl ligation to haem groups, which is most common in bacterial catalases <span class="citation">[<a href="#ref-dawson_active_1991">266</a>]</span> also results in a low redox potential, as illustrated by mutant forms of haemoglobin <span class="citation">[<a href="#ref-nagai_reduction_1983">267</a>]</span> and myoglobin <span class="citation">[<a href="#ref-hildebrand_proximal_1995">268</a>]</span>. The midpoint redox potential of the <em>d</em><span class="math inline">\(_1\)</span> haem in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> is not known but is likely to be a few tens of millivolts lower than that of the <em>c</em> haem (Chapter 1, Table 1.2). However, the protein sequence confirms the presence of all of these residues.</p>
<p>His-17 and Tyr-25 are located towards the N-terminus of the protein, which Figure 3.4 shows to be the least conserved region of the cytochromes <em>cd</em><span class="math inline">\(_1\)</span>. Intriguingly, inspection of Figure 3.4 shows that these residues are conserved only in <em>T. pantotropha</em> and the two strains of <em>Paracoccus</em>. The other proteins lack the N-terminal extension seen in these sequences which contains the two haem ligands. Of the other proteins, only that from <em>Ps. aeruginosa</em> contains a histidine or tyrosine residue at even roughly similar positions in the N-terminus before the <em>c</em> haem binding site. This finding was extremely unexpected. Ligands to the haem groups control several important factors such as the redox potential, substrate accessibility and the reaction mechanism; in <em>T. pantotropha</em>, Tyr-25 has been postulated to play a crucial role in the release of nitric oxide from the haem group <span class="citation">[<a href="#ref-fulop_anatomy_1995">127</a>]</span>. The importance of these residues and the overall high similarity of the proteins suggests that functional amino acids should be highly conserved. For this reason, it was decided to re-sequence the relevant segment of the <em>nirS</em> sequence from <em>Ps. stutzeri</em> Zobell so as to examine the possibility that the sequence had been mis-reported, or that errors in the sequence (such as base changes or frame shifts) could have resulted in the apparent absence of at least Tyr 25. Two sets of forward and reverse primers, flanking the N-terminal region of the <em>nirS</em> protein, were designed and used to amplify genomic DNA from <em>Ps. stutzeri</em> Zobell, as shown in Figure 3.7.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-7"></span>
<img src="figure/fig3-7.png" alt="Early region of the _nirS_ gene from _Ps. stutzeri_ ATCC 14405, showing primers used for re-sequencing and the sequence obtained. The downward arrow indicates a deletion of about 40 amino acids seen when the cytochrome _cd_$_1$ sequence is aligned to that of _T. pantotropha_. Primers designed to flank this region (FOR1, FOR2, REV1 and REV2) are indicated by horizontal arrows. The sequence obtained is annotated as follows; dotted underline = sequenced once in one direction, single underline = sequenced twice in forward direction, double underline = sequenced in both directions." width="75%" />
<p class="caption">
Figure 3.7: Early region of the <em>nirS</em> gene from <em>Ps. stutzeri</em> ATCC 14405, showing primers used for re-sequencing and the sequence obtained. The downward arrow indicates a deletion of about 40 amino acids seen when the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequence is aligned to that of <em>T. pantotropha</em>. Primers designed to flank this region (FOR1, FOR2, REV1 and REV2) are indicated by horizontal arrows. The sequence obtained is annotated as follows; dotted underline = sequenced once in one direction, single underline = sequenced twice in forward direction, double underline = sequenced in both directions.
</p>
</div>
<p>The four resulting PCR products were directly sequenced from low-melting point agarose using these primers. Figure 3.7 indicates the region, direction and strand coverage (forward, reverse or both) of the sequence obtained. This sequence was 100% identical to that deposited in the GenBank database. It is thus clear that the gene sequence is correct and that the tyrosyl and histidyl ligands in the protein from <em>T. pantotropha</em> really are absent from at least the <em>Ps. stutzeri</em> Zobell enzyme.</p>
<p>This result raises a number of possibilities. First, the <em>c</em>- and <em>d</em><span class="math inline">\(_1\)</span> haems in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the oxidised state from <em>Ps. stutzeri</em> (and from other species) may be liganded by tyrosyl and histidyl residues lying elsewhere in the polypeptide chain. This would result in a substantially different tertiary structure in these enzymes and given the overall high sequence homology, does not seem to be a credible proposal. Second, the other cytochromes <em>cd</em><span class="math inline">\(_1\)</span> may have different haem ligands (such as the conserved methionines and histidines seen in Figure 3.4), which could preserve a similar structure to the <em>T. pantotropha</em> protein, but might be expected to result in different spectroscopic and biochemical properties. Third, the oxidised <em>T. pantotropha</em> enzyme structure may not be representative of the physiologically-active structure found <em>in vivo</em>; the latter, hypothetical, structure may have the haem ligands that are conserved between all cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences.</p>
<p>In order to examine the optical spectrum of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> Zobell more closely, it was purified using the same procedure as that for the <em>T. pantotropha</em> protein. This protocol gave a product that was approximately 90-95% pure as judged by SDS-PAGE. The spectra of the oxidised and reduced forms of the enzyme are shown in Figure 3.8.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-8"></span>
<img src="figure/fig3-8.png" alt="Visible spectra of cytochrome _cd_$_1$ (a) in the oxidised and (b) in the reduced forms from _Ps. stutzeri_ Zobell. As isolated, cytochrome _cd_$_1$ from _Ps. stutzeri_ Zobell exhibited absorbance maxima at 416, 526, 554 and 645 nm. Following reduction with dithionite maxima were observed at 421 (not shown), 464, 526, 553, 557 and 660 nm. The reversal in the relative amplitudes of the peaks at 548 and 554 nm referred to in the text is clearly visible in (b)." width="60%" />
<p class="caption">
Figure 3.8: Visible spectra of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (a) in the oxidised and (b) in the reduced forms from <em>Ps. stutzeri</em> Zobell. As isolated, cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> Zobell exhibited absorbance maxima at 416, 526, 554 and 645 nm. Following reduction with dithionite maxima were observed at 421 (not shown), 464, 526, 553, 557 and 660 nm. The reversal in the relative amplitudes of the peaks at 548 and 554 nm referred to in the text is clearly visible in (b).
</p>
</div>
<p>Overall, the spectra of the seven cytochromes <em>cd</em><span class="math inline">\(_1\)</span> that have been sequenced are very similar. In the oxidised form there are peaks at or near 411 and 640 nm, with shoulders around 561, 325 and 360 nm. In the reduced form, peaks at 418, 460, 521, 548, 554 and 625 nm (dithionite-reduced) or 655 nm (ascorbate-reduced) are seen. The absorbances at 655/625 and 460 nm are assigned to the <em>d</em><span class="math inline">\(_1\)</span> haem, whereas the 548-554, 521/525 and 418/411 nm peaks are the <span class="math inline">\(\alpha\)</span>, <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span> (Soret) peaks of the <em>c</em>-type haem. However, the enzyme from <em>Ps. stutzeri</em> Zobell shows a small, but interesting difference. In the split <span class="math inline">\(\alpha\)</span>-peak of the reduced <em>c</em> haem at 548-554 nm, the first peak of the pair is slightly higher for the enzymes from <em>T. pantotropha</em>, <em>P. denitrificans</em> and <em>Ps. aeruginosa</em>. In the purified enzyme from <em>Ps. stutzeri</em> Zobell, this situation is reversed. This feature can be seen in published spectra by other workers <span class="citation">[<a href="#ref-zumft_defects_1988">73</a>]</span> and was also specifically noted for the enzyme from <em>Ps. stutzeri</em> JM300 <span class="citation">[<a href="#ref-weeg-aerssens_purification_1991">253</a>]</span>. Interestingly, if the <em>d</em><span class="math inline">\(_1\)</span> haem is removed from the <em>T. pantotropha</em> protein to give the semi-apo enzyme, the relative heights of the split <span class="math inline">\(\alpha\)</span>-peaks in the reduced state become reversed, so as to resemble the <em>Ps. stutzeri</em> enzymes (Figure 3.9).</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-9"></span>
<img src="figure/fig3-9.png" alt="Visible spectra of (a) holocytochrome _cd_$_1$ and (b) semi-apo cytochrome _cd_$_1$ in the reduced forms from _T. pantotropha_. The reduced holoprotein exhibited absorbance maxima at 418, 460 (not shown), 525, 548, 554 and 650 nm (a). Following removal of the _d_$_1$ haem the maxima at 460 and 650 nm (not shown) are lost and the amplitude of the peaks at 548 and 554 nm is reversed as compared to the holoprotein, as shown in (b)." width="60%" />
<p class="caption">
Figure 3.9: Visible spectra of (a) holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> and (b) semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the reduced forms from <em>T. pantotropha</em>. The reduced holoprotein exhibited absorbance maxima at 418, 460 (not shown), 525, 548, 554 and 650 nm (a). Following removal of the <em>d</em><span class="math inline">\(_1\)</span> haem the maxima at 460 and 650 nm (not shown) are lost and the amplitude of the peaks at 548 and 554 nm is reversed as compared to the holoprotein, as shown in (b).
</p>
</div>
<p>From the <em>T. pantotropha</em> crystal structure, we know that the Tyr-25 ligand originates in the <em>c</em> haem domain. Hence this result could be interpreted as showing that the <em>d</em><span class="math inline">\(_1\)</span> haem domain has a subtle effect on the absorption spectrum of the <em>c</em> haem, which is altered when the <em>d</em><span class="math inline">\(_1\)</span> haem is removed. The apparent lack of this effect in the <em>Ps. stutzeri</em> enzymes might be taken as evidence that a similar link between the <em>c</em> and <em>d</em><span class="math inline">\(_1\)</span> haem domains does not exist in these proteins. A second interpretation is that <em>d</em><span class="math inline">\(_1\)</span> haem loss is more extensive during the purification of the <em>Ps. stutzeri</em> enzymes, yielding a species more like the semi-apo enzyme. However, the observation of this spectrum by other groups and previously reported activity measurements <span class="citation">[<a href="#ref-weeg-aerssens_purification_1991">253</a>]</span> argue against this idea.</p>
<p>In fact, mounting spectroscopic evidence indicates that haem ligation is genuinely not identical in different cytochromes <em>cd</em><span class="math inline">\(_1\)</span>. Magnetic circular dichroism (MCD) and electron paramagnetic resonance (EPR) spectroscopy of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> have indicated Met-His ligation at the <em>c</em> haem and His-His ligation at the <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-sutherland_investigation_1986">124</a>]</span>. EPR and Mössbauer spectroscopy of the enzyme from <em>Thiobacillus denitrificans</em> suggested His-His ligation at both haems <span class="citation">[<a href="#ref-huynh_mossbauer_1982">126</a>]</span>. More recently, the <em>T. pantotropha</em> and <em>Ps. stutzeri</em> proteins in the oxidised form have been compared using EPR and MCD spectroscopy at the University of East Anglia <span class="citation">[<a href="#ref-cheesman_two_1997">125</a>]</span>. The latter method in particular is regarded as the most accurate diagnostic of haem ligation <span class="citation">[<a href="#ref-andersson_comparative_1995">269</a>]</span>. The EPR and MCD data are interpreted as being consistent with His-His ligation at the <em>c</em> haem in the <em>T. pantotropha</em> enzyme although, interestingly, a very small proportion of the sample appears to have Met-His ligation. The <em>d</em><span class="math inline">\(_1\)</span> haem ligands are more difficult to assign in the absence of data from model compounds, but are thought to be consistent with His-Tyr or possibly His-OH ligation in both cases. However, the <em>c</em> haem in the oxidised <em>Ps. stutzeri</em> enzyme is clearly Met-His ligated, as was previously reported for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-sutherland_investigation_1986">124</a>]</span>.</p>
<p>In summary, it is apparent that there are significant differences in haem ligation amongst the cytochromes <em>cd</em><span class="math inline">\(_1\)</span>, though bis-histidyl ligation at the <em>d</em><span class="math inline">\(_1\)</span> haem and methionyl histidyl ligation at the <em>c</em> haem are commonly assigned to several members of the group. This brings us to the question of whether the <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> crystal structure is anomalous or in some way misleading. There can be no doubt that the structure is “correct” in terms of the integrity of the diffraction data, its interpretation, the chain tracing methods used and correlation with the protein sequence; the question is whether the structure is physiologically relevant.</p>
<p>As described above, MCD spectra of the <em>T. pantotropha</em> enzyme confirm the bis histidyl ligation seen in the oxidised crystal structure, whereas the enzymes from <em>Ps. stutzeri</em> and <em>Ps. aeruginosa</em> are thought to have Met-His ligation to the <em>c</em> haem in the oxidised state. Recent work <span class="citation">[<a href="#ref-williams_time-resolved_1996">166</a>]</span> has shown that in reduced crystals of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> (which do not crack when reduced by sodium dithionite, unlike crystals of the <em>Ps. aeruginosa</em> enzyme <span class="citation">[<a href="#ref-akey_characterization_1980">147</a>]</span>, dramatic changes in haem ligation are observed. Tyr-25 is seen to move away from the <em>d</em><span class="math inline">\(_1\)</span> haem, as predicted by spectroscopic and mechanistic studies (Chapter 1, Section 1.5.4), but at the <em>c</em> haem His-17 is no longer a ligand. Instead, a loop of residues (100-114) moves upwards in the structure and Met-106 becomes a new <em>c</em> haem ligand. This arrangement means that the <em>T. pantotropha</em> enzyme in the reduced state more closely resembles the predicted ligation of the enzymes from <em>Ps. aeruginosa</em> and <em>Ps. stutzeri</em>; additionally Met-106 is a conserved residue in all cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences.</p>
<p>These data raise the possibility that the physiologically active form of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is Met-His ligated at the <em>c</em> haem at all times, and that the fully-oxidised His His ligated species is a relaxed or resting artefact which is not returned to in the catalytic cycle. Three alternative reaction schemes are outlined in Figure 3.10.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3-10"></span>
<img src="figure/fig3-10.png" alt="Alternative reaction schemes for the reduction of nitrite by cytochrome _cd_$_1$ starting from the fully oxidised enzyme. (A) The _c_-haem switches to Met-His ligation on reduction. After intramolecular electron transfer to the _d_$_1$ haem, the re-oxidised _c_-haem switches back to His-His ligation. The fully oxidised enzyme is regenerated. (B) The _c_-haem becomes Met-His ligated on reduction and remains in this state even after reoxidisation. The fully oxidised state is regenerated, but His-His ligation is not. (C) The _c_-haem becomes His-Met ligated on reduction. After internal electron transfer to the _d_$_1$ haem, the _c_-haem is instantly re-reduced by an external electron donor protein and stays in the Met-His ligation state. Neither His-His ligation nor the fully oxidised enzyme are regenerated. Note that all schemes assume Tyr-25 returns to the _d_$_1$ haem after NO release. Dashed arrows indicate intramolecular electron transfer. M = methionine, H = histidine, Y = tyrosine." width="75%" />
<p class="caption">
Figure 3.10: Alternative reaction schemes for the reduction of nitrite by cytochrome <em>cd</em><span class="math inline">\(_1\)</span> starting from the fully oxidised enzyme. (A) The <em>c</em>-haem switches to Met-His ligation on reduction. After intramolecular electron transfer to the <em>d</em><span class="math inline">\(_1\)</span> haem, the re-oxidised <em>c</em>-haem switches back to His-His ligation. The fully oxidised enzyme is regenerated. (B) The <em>c</em>-haem becomes Met-His ligated on reduction and remains in this state even after reoxidisation. The fully oxidised state is regenerated, but His-His ligation is not. (C) The <em>c</em>-haem becomes His-Met ligated on reduction. After internal electron transfer to the <em>d</em><span class="math inline">\(_1\)</span> haem, the <em>c</em>-haem is instantly re-reduced by an external electron donor protein and stays in the Met-His ligation state. Neither His-His ligation nor the fully oxidised enzyme are regenerated. Note that all schemes assume Tyr-25 returns to the <em>d</em><span class="math inline">\(_1\)</span> haem after NO release. Dashed arrows indicate intramolecular electron transfer. M = methionine, H = histidine, Y = tyrosine.
</p>
</div>
<p>In the first, the <em>c</em> haem returns to His-His ligation when it becomes reoxidised following intramolecular electron transfer to the <em>d</em><span class="math inline">\(_1\)</span> haem. In the second, the <em>c</em> haem remains Met-His ligated after reoxidation and a fully-oxidised His-His species is not regenerated. In the third, a fully oxidised enzyme is not regenerated because re-reduction of the <em>c</em> haem following the first intramolecular electron transfer to the <em>d</em><span class="math inline">\(_1\)</span> haem is very rapid, preventing the formation of the oxidised His-His ligation state. Further data using reduced crystals of the <em>T. pantotropha</em> enzyme <span class="citation">[<a href="#ref-williams_time-resolved_1996">166</a>]</span> has provided some insight into which of these schemes may be correct. It has proved possible to soak the reduced crystals in potassium nitrite and then rapidly freeze them, capturing what are proposed to be structural intermediates in the catalytic cycle. The fully-reduced enzyme molecules in the crystal carry two electrons, which allows two turnovers of the enzyme. The freeze-quenched structures contain molecules captured after the binding of the second nitrite molecule and different structures show either nitrite or nitric oxide bound at the <em>d</em><span class="math inline">\(_1\)</span> haem. It is clear from the diffraction data that following the reduction of nitrite to nitric oxide at the <em>d</em><span class="math inline">\(_1\)</span> haem, the <em>c</em> haem has returned to His-His ligation, as in Figure 3.10a. Such a rearrangement of the haem ligands during catalysis was unexpected and it has been suggested that this is the rate-limiting step of the reaction <span class="citation">[<a href="#ref-williams_time-resolved_1996">166</a>]</span>. It is also suggested that the mechanism prevents reduction of the <em>d</em><span class="math inline">\(_1\)</span> haem whilst NO is still bound; as His-17 and Tyr 25 are on the same stretch of the polypeptide chain, return of His-17 to the <em>c</em> haem would facilitate the re-ligation of the <em>d</em><span class="math inline">\(_1\)</span> haem by Tyr-25, displacing the bound NO. The <em>c</em> haem would then be re-reduced, change to Met-His ligation and transfer its electron to the <em>d</em><span class="math inline">\(_1\)</span> haem where Tyr-25 would be displaced to begin a new catalytic cycle.</p>
<p>However, the data from the experiments of Williams <span class="citation">[<a href="#ref-williams_time-resolved_1996">166</a>]</span> are not complete. Crystalline <em>cd</em><span class="math inline">\(_1\)</span> molecules can undergo only two turnovers whereas in the periplasm it is possible that a constant supply of electrons from donor proteins would act to keep the <em>c</em> haem reduced and in the Met-His state. Notably, stopped flow experiments using the enzyme from <em>Ps. aeruginosa</em> showed that the <em>c</em> haem was more than 90% reduced at all times <span class="citation">[<a href="#ref-silvestrini_reaction_1990">162</a>]</span>. The conservation of Met-106 in all cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences is also a strong argument for this residue to be more functionally important than His-17 in <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The issue will ultimately be resolved by further kinetic studies of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, in conjunction with site-directed mutagenesis of His-17 and Met-106.</p>
<p>The role of the non-conserved Tyr-25 <em>d</em><span class="math inline">\(_1\)</span> haem ligand remains more problematic. It is clearly a ligand to the <em>c</em> haem in the oxidised crystal structure and can also be seen near the haem, having moved away, in the reduced state. Notably however, it has not yet been observed to return to the <em>d</em><span class="math inline">\(_1\)</span> haem in the crystal structure after NO release; this step remains hypothetical. Therefore at present it is not possible to say with certainty whether Tyr-25 plays a crucial role in the mechanism of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The lack of sequence conservation in an otherwise highly homologous group of proteins does suggest that Tyr-25 may not be an obligatory residue. Alternatively we have to accept that for some reason, within a set of enzymes that are so strongly conserved as to suggest high structural homology, different functional residues are used for catalysis.</p>
<p>The clearest way to resolve this issue, as with the problem of haem ligation in the <em>c</em> domain, will be using site-directed mutagenesis to generate changes at Tyr-25 and analysis of the resultant proteins. As with all site-directed mutagenesis experiments, care will have to be taken to distinguish effects arising due to gross perturbation of the protein tertiary structure from those that involve only alteration around the haem ligation site. Bearing this in mind, obvious candidates for substitution at Tyr-25 are (1) phenylalanine, the same basic structure but lacking the liganding -OH moiety, (2) serine, of small size with an -OH group and (3) histidine, of similar size with the possibility of acting as a ligand, having been tentatively identified in other cytochromes <em>cd</em><span class="math inline">\(_1\)</span>. This work is currently underway in our laboratory and some early data are presented in Chapter 6. Further crystal structures of other cytochromes <em>cd</em><span class="math inline">\(_1\)</span> will also be of great interest. The purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> Zobell, described in Section 3.4.3, was used for crystallisation trials by Dr. Vilmos Fülöp in the Laboratory of Molecular Biophysics and gave small crystals, though not of diffraction quality. However, we have since learned that the group of Dr. E.N. Baker (Massey University, New Zealand) is close to solving this structure, which should be available in the near future. Very recently the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> has been determined (V. Fülöp, personal communication), although the details have not yet been published. Some aspects of this structure are discussed in Chapter 8.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-peakman_nucleotide_1990" class="csl-entry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Peakman T, Crouzet J, Mayaux JF, Busby S, Mohan S, Harborne N, et al. Nucleotide sequence, organisation and structural analysis of the products of genes in the <span class="nocase">nirB</span>-<span class="nocase">cysG</span> region of the <span>Escherichia</span> coli <span>K</span>-12 chromosome. European Journal of Biochemistry. 1990;191: 315–323. doi:<a href="https://doi.org/10.1111/j.1432-1033.1990.tb19125.x">10.1111/j.1432-1033.1990.tb19125.x</a></div>
</div>
<div id="ref-zumft_defects_1988" class="csl-entry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">Zumft WG, Döhler K, Körner H, Löchelt S, Viebrock A, Frunzke K. Defects in cytochrome cd1-dependent nitrite respiration of transposon <span>Tn5</span>-induced mutants from <span>Pseudomonas</span> stutzeri. Archives of Microbiology. 1988;149: 492–498. doi:<a href="https://doi.org/10.1007/BF00446750">10.1007/BF00446750</a></div>
</div>
<div id="ref-hoeren_sequence_1993" class="csl-entry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Hoeren FU, Berks BC, Ferguson SJ, McCarthy JE. Sequence and expression of the gene encoding the respiratory nitrous-oxide reductase from <span>Paracoccus</span> denitrificans. <span>New</span> and conserved structural and regulatory motifs. European Journal of Biochemistry. 1993;218: 49–57. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb18350.x">10.1111/j.1432-1033.1993.tb18350.x</a></div>
</div>
<div id="ref-moir_purification_1993" class="csl-entry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Moir JW, Baratta D, Richardson DJ, Ferguson SJ. The purification of a cd1-type nitrite reductase from, and the absence of a copper-type nitrite reductase from, the aerobic denitrifier <span>Thiosphaera</span> pantotropha; the role of pseudoazurin as an electron donor. European Journal of Biochemistry. 1993;212: 377–385. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb17672.x">10.1111/j.1432-1033.1993.tb17672.x</a></div>
</div>
<div id="ref-sutherland_investigation_1986" class="csl-entry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Sutherland J, Greenwood C, Peterson J, Thomson AJ. An investigation of the ligand-binding properties of <span>Pseudomonas</span> aeruginosa nitrite reductase. The Biochemical Journal. 1986;233: 893–898. doi:<a href="https://doi.org/10.1042/bj2330893">10.1042/bj2330893</a></div>
</div>
<div id="ref-cheesman_two_1997" class="csl-entry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Cheesman MR, Ferguson SJ, Moir JW, Richardson DJ, Zumft WG, Thomson AJ. Two enzymes with a common function but different heme ligands in the forms as isolated. <span>Optical</span> and magnetic properties of the heme groups in the oxidized forms of nitrite reductase, cytochrome cd1, from <span>Pseudomonas</span> stutzeri and <span>Thiosphaera</span> pantotropha. Biochemistry. 1997;36: 16267–16276. doi:<a href="https://doi.org/10.1021/bi971677a">10.1021/bi971677a</a></div>
</div>
<div id="ref-huynh_mossbauer_1982" class="csl-entry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Huynh BH, Lui MC, Moura JJ, Moura I, Ljungdahl PO, Münck E, et al. Mössbauer and <span>EPR</span> studies on nitrite reductase from <span>Thiobacillus</span> denitrificans. The Journal of Biological Chemistry. 1982;257: 9576–9581. doi:<a href="https://doi.org/10.1016/S0021-9258(18)34110-3">10.1016/S0021-9258(18)34110-3</a></div>
</div>
<div id="ref-fulop_anatomy_1995" class="csl-entry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Fülöp V, Moir JW, Ferguson SJ, Hajdu J. The anatomy of a bifunctional enzyme: Structural basis for reduction of oxygen to water and synthesis of nitric oxide by cytochrome cd1. Cell. 1995;81: 369–377. doi:<a href="https://doi.org/10.1016/0092-8674(95)90390-9">10.1016/0092-8674(95)90390-9</a></div>
</div>
<div id="ref-silvestrini_kinetics_1982" class="csl-entry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Silvestrini MC, Tordi MG, Colosimo A, Antonini E, Brunori M. The kinetics of electron transfer between pseudomonas aeruginosa cytochrome c-551 and its oxidase. The Biochemical Journal. 1982;203: 445–451. doi:<a href="https://doi.org/10.1042/bj2030445">10.1042/bj2030445</a></div>
</div>
<div id="ref-jungst_nirstbm_1991" class="csl-entry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Jüngst A, Wakabayashi S, Matsubara H, Zumft WG. The <span class="nocase">nirSTBM</span> region coding for cytochrome cd1-dependent nitrite respiration of <span>Pseudomonas</span> stutzeri consists of a cluster of mono-, di-, and tetraheme proteins. FEBS letters. 1991;279: 205–209. doi:<a href="https://doi.org/10.1016/0014-5793(91)80150-2">10.1016/0014-5793(91)80150-2</a></div>
</div>
<div id="ref-smith_isolation_1992" class="csl-entry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">Smith GB, Tiedje JM. Isolation and characterization of a nitrite reductase gene and its use as a probe for denitrifying bacteria. Applied and Environmental Microbiology. 1992;58: 376–384. doi:<a href="https://doi.org/10.1128/AEM.58.1.376-384.1992">10.1128/AEM.58.1.376-384.1992</a></div>
</div>
<div id="ref-ohshima_cloning_1993" class="csl-entry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Ohshima T, Sugiyama M, Uozumi N, Iijima S, Kobayashi T. Cloning and sequencing of a gene encoding nitrite reductase from <span>Paracoccus</span> denitrificans and expression of the gene in <span>Escherichia</span> coli. Journal of Fermentation and Bioengineering. 1993;76: 82–88. doi:<a href="https://doi.org/10.1016/0922-338X(93)90061-C">10.1016/0922-338X(93)90061-C</a></div>
</div>
<div id="ref-de_boer_isolation_1994" class="csl-entry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Boer AP de, Reijnders WN, Kuenen JG, Stouthamer AH, Spanning RJ van. Isolation, sequencing and mutational analysis of a gene cluster involved in nitrite reduction in <span>Paracoccus</span> denitrificans. Antonie Van Leeuwenhoek. 1994;66: 111–127. doi:<a href="https://doi.org/10.1007/BF00871635">10.1007/BF00871635</a></div>
</div>
<div id="ref-rees_structural_1997" class="csl-entry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline">Rees E, Siddiqui RA, Köster F, Schneider B, Friedrich B. Structural gene (<span class="nocase">nirS</span>) for the cytochrome cd1 nitrite reductase of <span>Alcaligenes</span> eutrophus <span>H16</span>. Applied and Environmental Microbiology. 1997;63: 800–802. doi:<a href="https://doi.org/10.1128/AEM.63.2.800-802.1997">10.1128/AEM.63.2.800-802.1997</a></div>
</div>
<div id="ref-akey_characterization_1980" class="csl-entry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline">Akey CW, Moffat K, Wharton DC, Edelstein SJ. Characterization of crystals of a cytochrome oxidase (nitrite reductase) from <span>Pseudomonas</span> aeruginosa by x-ray diffraction and electron microscopy. Journal of Molecular Biology. 1980;136: 19–43. doi:<a href="https://doi.org/10.1016/0022-2836(80)90364-2">10.1016/0022-2836(80)90364-2</a></div>
</div>
<div id="ref-moore_cytochromes_1990" class="csl-entry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline">Moore GR, Pettigrew GW. Cytochromes <span>C</span> <span>Evolutionary</span>, <span>Structural</span> and <span>Physicochemical</span> <span>Aspects</span>. Berlin, Heidelberg: Springer Berlin / Heidelberg; 1990. Available: <a href="http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030">http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030</a></div>
</div>
<div id="ref-silvestrini_reaction_1990" class="csl-entry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline">Silvestrini MC, Tordi MG, Musci G, Brunori M. The reaction of <span>Pseudomonas</span> nitrite reductase and nitrite. <span>A</span> stopped-flow and <span>EPR</span> study. The Journal of Biological Chemistry. 1990;265: 11783–11787. doi:<a href="https://doi.org/10.1016/S0021-9258(19)38466-2">10.1016/S0021-9258(19)38466-2</a></div>
</div>
<div id="ref-williams_time-resolved_1996" class="csl-entry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline">Williams P. Time-resolved structural studies on macromolecules. PhD thesis, University of Oxford. 1996. Available: <a href="https://www.worldcat.org/title/time-resolved-structural-studies-on-macromolecules/oclc/43530285">https://www.worldcat.org/title/time-resolved-structural-studies-on-macromolecules/oclc/43530285</a></div>
</div>
<div id="ref-spiro_fnr_1994" class="csl-entry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline">Spiro S. The <span>FNR</span> family of transcriptional regulators. Antonie Van Leeuwenhoek. 1994;66: 23–36. doi:<a href="https://doi.org/10.1007/BF00871630">10.1007/BF00871630</a></div>
</div>
<div id="ref-thompson_clustal_1994" class="csl-entry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline">Thompson JD, Higgins DG, Gibson TJ. <span>CLUSTAL</span> <span>W</span>: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research. 1994;22: 4673–4680. doi:<a href="https://doi.org/10.1093/nar/22.22.4673">10.1093/nar/22.22.4673</a></div>
</div>
<div id="ref-saitou_neighbor-joining_1987" class="csl-entry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline">Saitou N, Nei M. The neighbor-joining method: A new method for reconstructing phylogenetic trees. Molecular Biology and Evolution. 1987;4: 406–425. doi:<a href="https://doi.org/10.1093/oxfordjournals.molbev.a040454">10.1093/oxfordjournals.molbev.a040454</a></div>
</div>
<div id="ref-felsenstein_confidence_1985" class="csl-entry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline">Felsenstein J. <span>CONFIDENCE</span> <span>LIMITS</span> <span>ON</span> <span>PHYLOGENIES</span>: <span>AN</span> <span>APPROACH</span> <span>USING</span> <span>THE</span> <span>BOOTSTRAP</span>. Evolution; International Journal of Organic Evolution. 1985;39: 783–791. doi:<a href="https://doi.org/10.1111/j.1558-5646.1985.tb00420.x">10.1111/j.1558-5646.1985.tb00420.x</a></div>
</div>
<div id="ref-page_treeview_1996" class="csl-entry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline">Page RD. <span>TreeView</span>: An application to display phylogenetic trees on personal computers. Computer applications in the biosciences: CABIOS. 1996;12: 357–358. doi:<a href="https://doi.org/10.1093/bioinformatics/12.4.357">10.1093/bioinformatics/12.4.357</a></div>
</div>
<div id="ref-van_spanning_nitrite_1995" class="csl-entry">
<div class="csl-left-margin">197. </div><div class="csl-right-inline">Van Spanning RJ, De Boer AP, Reijnders WN, Spiro S, Westerhoff HV, Stouthamer AH, et al. Nitrite and nitric oxide reduction in <span>Paracoccus</span> denitrificans is under the control of <span>NNR</span>, a regulatory protein that belongs to the <span>FNR</span> family of transcriptional activators. FEBS letters. 1995;360: 151–154. doi:<a href="https://doi.org/10.1016/0014-5793(95)00091-m">10.1016/0014-5793(95)00091-m</a></div>
</div>
<div id="ref-moir_aspects_1993" class="csl-entry">
<div class="csl-left-margin">240. </div><div class="csl-right-inline">Moir J. Aspects of electron transport in <span>Thiosphaera</span> pantotropha and <span>Paracoccus</span> denitrificans. PhD thesis, University of Oxford. 1993. Available: <a href="https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635">https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635</a></div>
</div>
<div id="ref-devereux_comprehensive_1984" class="csl-entry">
<div class="csl-left-margin">248. </div><div class="csl-right-inline">Devereux J, Haeberli P, Smithies O. A comprehensive set of sequence analysis programs for the <span>VAX</span>. Nucleic Acids Research. 1984;12: 387–395. doi:<a href="https://doi.org/10.1093/nar/12.1part1.387">10.1093/nar/12.1part1.387</a></div>
</div>
<div id="ref-weeg-aerssens_purification_1991" class="csl-entry">
<div class="csl-left-margin">253. </div><div class="csl-right-inline">Weeg-Aerssens E, Wu WS, Ye RW, Tiedje JM, Chang CK. Purification of cytochrome cd1 nitrite reductase from <span>Pseudomonas</span> stutzeri <span>JM300</span> and reconstitution with native and synthetic heme d1. The Journal of Biological Chemistry. 1991;266: 7496–7502. Available: <a href="https://www.jbc.org/article/S0021-9258(20)89474-5/pdf">https://www.jbc.org/article/S0021-9258(20)89474-5/pdf</a></div>
</div>
<div id="ref-hole_characterization_1996" class="csl-entry">
<div class="csl-left-margin">254. </div><div class="csl-right-inline">Hole UH, Vollack K-U, Zumft WG, Eisenmann E, Siddiqui RA, Friedrich B, et al. Characterization of the membranous denitrification enzymes nitrite reductase (cytochrome cd 1 ) and copper-containing nitrous oxide reductase from <span>Thiobacillus</span> denitrificans. Archives of Microbiology. 1996;165: 55–61. doi:<a href="https://doi.org/10.1007/s002030050296">10.1007/s002030050296</a></div>
</div>
<div id="ref-von_heijne_new_1986" class="csl-entry">
<div class="csl-left-margin">255. </div><div class="csl-right-inline">Heijne G von. A new method for predicting signal sequence cleavage sites. Nucleic Acids Research. 1986;14: 4683–4690. doi:<a href="https://doi.org/10.1093/nar/14.11.4683">10.1093/nar/14.11.4683</a></div>
</div>
<div id="ref-nielsen_identification_1997" class="csl-entry">
<div class="csl-left-margin">256. </div><div class="csl-right-inline">Nielsen H, Engelbrecht J, Brunak S, Heijne G von. Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. Protein Engineering. 1997;10: 1–6. doi:<a href="https://doi.org/10.1093/protein/10.1.1">10.1093/protein/10.1.1</a></div>
</div>
<div id="ref-barton_alscript_1993" class="csl-entry">
<div class="csl-left-margin">257. </div><div class="csl-right-inline">Barton GJ. <span>ALSCRIPT</span>: A tool to format multiple sequence alignments. Protein Engineering. 1993;6: 37–40. doi:<a href="https://doi.org/10.1093/protein/6.1.37">10.1093/protein/6.1.37</a></div>
</div>
<div id="ref-appel_new_1994" class="csl-entry">
<div class="csl-left-margin">258. </div><div class="csl-right-inline">Appel RD, Bairoch A, Hochstrasser DF. A new generation of information retrieval tools for biologists: The example of the <span>ExPASy</span> <span>WWW</span> server. Trends in Biochemical Sciences. 1994;19: 258–260. doi:<a href="https://doi.org/10.1016/0968-0004(94)90153-8">10.1016/0968-0004(94)90153-8</a></div>
</div>
<div id="ref-steinrucke_genetics_1993" class="csl-entry">
<div class="csl-left-margin">259. </div><div class="csl-right-inline">Steinrücke P, Ludwig B. Genetics of <span>Paracoccus</span> denitrificans. FEMS microbiology reviews. 1993;10: 83–117. doi:<a href="https://doi.org/10.1016/0378-1097(93)90505-v">10.1016/0378-1097(93)90505-v</a></div>
</div>
<div id="ref-roth_characterization_1993" class="csl-entry">
<div class="csl-left-margin">260. </div><div class="csl-right-inline">Roth JR, Lawrence JG, Rubenfield M, Kieffer-Higgins S, Church GM. Characterization of the cobalamin (vitamin <span>B12</span>) biosynthetic genes of <span>Salmonella</span> typhimurium. Journal of Bacteriology. 1993;175: 3303–3316. doi:<a href="https://doi.org/10.1128/jb.175.11.3303-3316.1993">10.1128/jb.175.11.3303-3316.1993</a></div>
</div>
<div id="ref-klug_dna_1991" class="csl-entry">
<div class="csl-left-margin">261. </div><div class="csl-right-inline">Klug G. A <span>DNA</span> sequence upstream of the puf operon of <span>Rhodobacter</span> capsulatus is involved in its oxygen-dependent regulation and functions as a protein binding site. Molecular &amp; general genetics: MGG. 1991;226: 167–176. doi:<a href="https://doi.org/10.1007/BF00273600">10.1007/BF00273600</a></div>
</div>
<div id="ref-olsen_winds_1994" class="csl-entry">
<div class="csl-left-margin">262. </div><div class="csl-right-inline">Olsen GJ, Woese CR, Overbeek R. The winds of (evolutionary) change: Breathing new life into microbiology. Journal of Bacteriology. 1994;176: 1–6. doi:<a href="https://doi.org/10.1128/jb.176.1.1-6.1994">10.1128/jb.176.1.1-6.1994</a></div>
</div>
<div id="ref-fox_how_1992" class="csl-entry">
<div class="csl-left-margin">263. </div><div class="csl-right-inline">Fox GE, Wisotzkey JD, Jurtshuk P. How close is close: <span>16S</span> <span class="nocase">rRNA</span> sequence identity may not be sufficient to guarantee species identity. International Journal of Systematic Bacteriology. 1992;42: 166–170. doi:<a href="https://doi.org/10.1099/00207713-42-1-166">10.1099/00207713-42-1-166</a></div>
</div>
<div id="ref-baker_cytochrome_1997" class="csl-entry">
<div class="csl-left-margin">264. </div><div class="csl-right-inline">Baker SC, Saunders NF, Willis AC, Ferguson SJ, Hajdu J, Fülöp V. Cytochrome cd1 structure: Unusual haem environments in a nitrite reductase and analysis of factors contributing to beta-propeller folds. Journal of Molecular Biology. 1997;269: 440–455. doi:<a href="https://doi.org/10.1006/jmbi.1997.1070">10.1006/jmbi.1997.1070</a></div>
</div>
<div id="ref-wallace_functional_1992" class="csl-entry">
<div class="csl-left-margin">265. </div><div class="csl-right-inline">Wallace CJ, Clark-Lewis I. Functional role of heme ligation in cytochrome c. <span>Effects</span> of replacement of methionine 80 with natural and non-natural residues by semisynthesis. The Journal of Biological Chemistry. 1992;267: 3852–3861. Available: <a href="https://www.jbc.org/article/S0021-9258(19)50604-4/pdf">https://www.jbc.org/article/S0021-9258(19)50604-4/pdf</a></div>
</div>
<div id="ref-dawson_active_1991" class="csl-entry">
<div class="csl-left-margin">266. </div><div class="csl-right-inline">Dawson JH, Bracete AM, Huff AM, Kadkhodayan S, Zeitler CM, Sono M, et al. The active site structure of <span>E</span>. Coli <span>HPII</span> catalase. <span>Evidence</span> favoring coordination of a tyrosinate proximal ligand to the chlorin iron. FEBS letters. 1991;295: 123–126. doi:<a href="https://doi.org/10.1016/0014-5793(91)81401-s">10.1016/0014-5793(91)81401-s</a></div>
</div>
<div id="ref-nagai_reduction_1983" class="csl-entry">
<div class="csl-left-margin">267. </div><div class="csl-right-inline">Nagai M, Yoneyama Y. Reduction of methemoglobins <span>M</span> <span>Hyde</span> <span>Park</span>, <span>M</span> <span>Saskatoon</span>, and <span>M</span> <span>Milwaukee</span> by ferredoxin and ferredoxin-nicotinamide adenine dinucleotide phosphate reductase system. The Journal of Biological Chemistry. 1983;258: 14379–14384. doi:<a href="https://doi.org/10.1016/S0021-9258(17)43872-5">10.1016/S0021-9258(17)43872-5</a></div>
</div>
<div id="ref-hildebrand_proximal_1995" class="csl-entry">
<div class="csl-left-margin">268. </div><div class="csl-right-inline">Hildebrand DP, Burk DL, Maurus R, Ferrer JC, Brayer GD, Mauk AG. The proximal ligand variant <span>His93Tyr</span> of horse heart myoglobin. Biochemistry. 1995;34: 1997–2005. doi:<a href="https://doi.org/10.1021/bi00006a021">10.1021/bi00006a021</a></div>
</div>
<div id="ref-andersson_comparative_1995" class="csl-entry">
<div class="csl-left-margin">269. </div><div class="csl-right-inline">Andersson LA, Johnson AK, Simms MD, Willingham TR. Comparative analysis of catalases: Spectral evidence against heme-bound water for the solution enzymes. FEBS letters. 1995;370: 97–100. doi:<a href="https://doi.org/10.1016/0014-5793(95)00651-o">10.1016/0014-5793(95)00651-o</a></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="2-chapter02.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="4-chapter04.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
