<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Cloning and expression of the nirS gene from T. pantotropha in Escherichia coli | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Cloning and expression of the nirS gene from T. pantotropha in Escherichia coli | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Cloning and expression of the nirS gene from T. pantotropha in Escherichia coli | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="3-chapter03.html"/>
<link rel="next" href="5-chapter05.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter04" class="section level1" number="4">
<h1><span class="header-section-number">Chapter 4</span> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></h1>
<div id="introduction-1" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Introduction</h2>
<p>The aim of the work described in this chapter was to investigate the expression of the cloned <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em>. Although <em>E. coli</em> does not synthesise the <em>d</em><span class="math inline">\(_1\)</span> haem group, and so cannot express holocytochrome <em>cd</em><span class="math inline">\(_1\)</span>, expression studies in this organism are still valuable for two reasons. First, <em>E. coli</em> provides a convenient experimental system to test gene expression, due to its widespread use in molecular biology, the consequent large technical literature, number of methods and range of plasmids available. This means that experiments using <em>E. coli</em> can be carried out with relative ease and efficiency. Second it is probable that in expression systems that can synthesise the holocytochrome (see Chapters 5 and 6), certain site-directed mutants of the nitrite reductase, the ultimate aim of this work, will be inactive and the host cells will be unable to grow under denitrifying conditions because of the accumulation of toxic nitrite. However, using <em>E. coli</em><span class="citation"><sup><a href="#ref-glockner_copper-containing_1993" role="doc-biblioref">63</a></sup></span> and <em>Ps. putida</em><span class="citation"><sup><a href="#ref-silvestrini_expression_1992" role="doc-biblioref">268</a></sup></span> it has already proved possible to express the semi-apo enzyme containing only <em>c</em> haem and then to reconstitute the enzyme with purified <em>d</em><span class="math inline">\(_1\)</span> haem. The latter approach may be a way of producing sufficient quantities of mutant enzyme for kinetic studies. Previously published data concerning the expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in <em>E. coli</em> and the expression of other <em>c</em> type cytochromes, both exogenous and endogenous<span class="citation"><sup><a href="#ref-iobbi-nivol_reassessment_1994" role="doc-biblioref">269</a></sup></span> indicated that <em>c</em> haem attachment to the <em>T.pantotropha</em> enzyme should not pose a problem. However, a potential problem was that <em>E. coli</em> is quite distantly related to <em>T. pantotropha</em> and so aspects of its genetics such as promoter sequences and codon usage may be quite different.</p>
<p>Although seven <em>nirS</em> genes have now been sequenced (including that from <em>T. pantotropha</em>, Chapter 3), expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from the cloned gene has only been studied in any detail for <em>Ps. aeruginosa</em>, <em>Ps. stutzeri</em> Zobell and <em>P. denitrificans</em>. Three organisms have been used as heterologous hosts for expression. Silvestrini et al. (1992)<span class="citation"><sup><a href="#ref-silvestrini_expression_1992" role="doc-biblioref">268</a></sup></span> have expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> in <em>Ps. putida</em>. In common with <em>E. coli</em> this organism does not make <em>d</em><span class="math inline">\(_1\)</span> haem and thus the semi-apoprotein, containing <em>c</em>-haem was produced. Glockner et al. (1993)<span class="citation"><sup><a href="#ref-glockner_copper-containing_1993" role="doc-biblioref">63</a></sup></span> have expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> Zobell in <em>Ps. aureofaciens</em> and <em>E. coli</em> with similar results. Ohshima et al. (1993)<span class="citation"><sup><a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a></sup></span> have expressed the <em>P. denitrificans</em> IFO 12442 <em>nirS</em> gene in <em>E. coli</em>, but in this case it was claimed that active holoprotein was synthesised. Nitric oxide reductase activity was also apparently detectable in this system. An alternative approach to expression of the <em>nirS</em> gene has been described by Zennaro et al. (1993),<span class="citation"><sup><a href="#ref-zennaro_nitrite_1993" role="doc-biblioref">215</a></sup></span> who inactivated the <em>nirS</em> gene in <em>Ps. aeruginosa</em> using a tetracycline cassette and then complemented the mutants with the wild-type gene on a plasmid. This met with some success and is discussed further in Chapter 6.</p>
<p>In all cases where semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is produced, it has proved possible to obtain holoprotein by reconstitution with purified haem <em>d</em><span class="math inline">\(_1\)</span>. This has been performed for <em>Ps. aeruginosa</em><span class="citation"><sup><a href="#ref-silvestrini_expression_1992" role="doc-biblioref">268</a></sup></span> and <em>Ps. stutzeri</em> Zobell.<span class="citation"><sup><a href="#ref-glockner_copper-containing_1993" role="doc-biblioref">63</a></sup></span> It is also possible to remove haem <em>d</em><span class="math inline">\(_1\)</span> chemically from the purified native holoprotein and reconstitute the semi-apo enzyme prepared in this manner to recover most of the activity; this has been described for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em><span class="citation"><sup><a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a></sup></span> and <em>Ps. stutzeri</em> M300.<span class="citation"><sup><a href="#ref-weeg-aerssens_purification_1991" role="doc-biblioref">251</a></sup></span></p>
<p>This chapter describes a series of experiments carried out to express cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> in <em>E. coli</em>. First, mapping of the <em>nir</em> gene locus, which was performed to identify a suitable fragment that contained the full-length <em>nirS</em> gene for sub-cloning, is discussed. Cloning of the <em>nirS</em> gene into a broad-host range vector is then described. Expression of the gene in <em>E. coli</em> under aerobic and anaerobic conditions is analysed. The final section describes the partial purification of the expressed semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and a first attempt to reconstitute this material with purified haem <em>d</em><span class="math inline">\(_1\)</span>. Throughout the chapter comparisons with the results of similar experiments from other laboratories are made.</p>
</div>
<div id="results-1" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Results</h2>
<div id="sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha" class="section level3" number="4.2.1">
<h3><span class="header-section-number">4.2.1</span> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></h3>
<p>The <em>T. pantotropha</em> <em>nirS</em> gene was cloned initially as a set of four overlapping PCR products for sequencing (Chapter 3). For further work, a full length copy of the gene was required. In theory this could be produced by PCR using primers flanking the 5’- and 3’ ends of the gene. PCR reactions using such primers (460F, nirSF, CT3R and nirS2R, Table 3.1, Chapter 3) were performed and appeared to produce fragments of the expected lengths (results not shown). However, extreme difficulty was experienced when attempting to clone the full length products. A large number of remedies were tried, including proteinase K treatment of the PCR products,<span class="citation"><sup><a href="#ref-hitti_proteinase_1994" role="doc-biblioref">270</a></sup></span> the use of TA cloning vectors and different methods of PCR product purification, but none of the putative clones obtained after blue-white selection contained the desired insert when analysed by restriction mapping. In all cases, it appeared that rearrangement of the DNA had occurred. At present, the best explanation for this is that a recurrent error during synthesis of the longer PCR products prevented successful cloning, perhaps because of the expression of a toxic product from the DNA.</p>
<p>It therefore became necessary to obtain a genomic clone of the <em>nirS</em> gene. Probing a cosmid library prepared from <em>T. pantotropha</em> genomic DNA by Southern blotting with a <em>nirS</em> gene probe (the MID1 PCR product, Figure 3.1, Chapter 3) failed to identify a positive clone. However, blots of total genomic DNA, digested with <em>PstI</em>, <em>NotI</em> and <em>DdeI</em>, gave single bands in the size range 4.5-6 kbp (results not shown). Attempts to clone these fragments were in progress, when a plasmid containing the <em>T. pantotropha</em> <em>nirS</em> gene was obtained from Thon de Boer, Vrije Universiteit, Amsterdam. The plasmid, pTNIR3, contains the <em>nirS</em> gene on a 7.5 kbp <em>SphI</em> fragment cloned in the vector pGEM7zf(+).</p>
<p>Plasmid pTNIR3 was analysed by restriction mapping so as to determine the best way of generating a fragment containing the <em>nirS</em> gene for further work. Figure 4.1 shows a restriction map of pTNIR3 obtained using six enzymes.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-1"></span>
<img src="figure/fig4-1.png" alt="Physical map of the plasmid pTNIR3 produced using six restriction enzymes. Plasmid pTNIR3, containing the _nirS_ gene from _T. pantotropha_, was obtained from Thon de Boer, Vrije Universiteit, Amsterdam. It consists of a 7.5 kbp _SphI_ fragment of genomic DNA cloned in pGEM7zf (+). The plasmid was digested with 6 enzymes, both singly and in combination, and the fragments sized by agarose gel electrophoresis to generate the map. Enzyme abbreviations: Sp, _SphI_; B, _BamHI_; E, _EcoRI_; H, _HindIII_; P, _PstI_; S, _SalI_" width="75%" />
<p class="caption">
Figure 4.1: Physical map of the plasmid pTNIR3 produced using six restriction enzymes. Plasmid pTNIR3, containing the <em>nirS</em> gene from <em>T. pantotropha</em>, was obtained from Thon de Boer, Vrije Universiteit, Amsterdam. It consists of a 7.5 kbp <em>SphI</em> fragment of genomic DNA cloned in pGEM7zf (+). The plasmid was digested with 6 enzymes, both singly and in combination, and the fragments sized by agarose gel electrophoresis to generate the map. Enzyme abbreviations: Sp, <em>SphI</em>; B, <em>BamHI</em>; E, <em>EcoRI</em>; H, <em>HindIII</em>; P, <em>PstI</em>; S, <em>SalI</em>
</p>
</div>
<p>This indicated that the 7.5 kbp insert could be conveniently cut into two fragments of 5.1 kbp and 2.4 kbp using <em>PstI</em> and <em>SphI</em>. These fragments were cloned into the pBS(-) plasmid and the ends of the inserts were sequenced using universal M13 primers. The restriction mapping data in conjunction with the partial DNA sequences were used to align the plasmid pTNIR3 and its derivatives with the sequence of the <em>nir</em> region from <em>P. denitrificans</em> PD1222,<span class="citation"><sup><a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a></sup></span> as shown in Figure 4.2.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-2"></span>
<img src="figure/fig4-2.png" alt="Location of denitrification genes of _T. pantotropha_ on the plasmid pTNIR3 as determined by physical mapping and limited sequencing. Plasmid pTNIR3 was digested with _PstI_ and _SphI_ and the fragments were sub-cloned into pBS(-). The ends of the fragments were then sequenced using M13 universal primers. The plasmids pTNIR3, pB5K and pBNIR are aligned to previously published sequence from _P. denitrificans_ PD1222, covering the _nir_ and _nor_ loci (U05002 and U28078 respectively). No direct evidence was obtained for the presence of the _nirJ_ gene in _pTNIR3_, although it is assumed to be present by analogy to _P. denitrificans_ PD1222 [@de_boer_regulation_1996]." width="75%" />
<p class="caption">
Figure 4.2: Location of denitrification genes of <em>T. pantotropha</em> on the plasmid pTNIR3 as determined by physical mapping and limited sequencing. Plasmid pTNIR3 was digested with <em>PstI</em> and <em>SphI</em> and the fragments were sub-cloned into pBS(-). The ends of the fragments were then sequenced using M13 universal primers. The plasmids pTNIR3, pB5K and pBNIR are aligned to previously published sequence from <em>P. denitrificans</em> PD1222, covering the <em>nir</em> and <em>nor</em> loci (U05002 and U28078 respectively). No direct evidence was obtained for the presence of the <em>nirJ</em> gene in <em>pTNIR3</em>, although it is assumed to be present by analogy to <em>P. denitrificans</em> PD1222.<span class="citation"><sup><a href="#ref-de_boer_regulation_1996" role="doc-biblioref">168</a></sup></span>
</p>
</div>
<p>The 5.1 kbp fragment lies upstream of the <em>nirS</em> gene and by analogy to <em>P. denitrificans</em> PD1222 would contain the genes <em>nirI</em>, <em>nirJ</em>, <em>norС</em> and part of the <em>norB</em> gene, in which lies the <em>SphI</em> site. The plasmid containing this fragment is designated pB5K. The 2.4 kbp fragment contains the <em>nirS</em> gene, with the <em>PstI</em> site at the 5’-end of the gene, just upstream of the NNR box, and the <em>SphI</em> site lying in the early region of the downstream <em>nirE</em> gene. This plasmid was designated pBNIR and was selected for further sub-cloning and expression experiments.</p>
</div>
<div id="cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh" class="section level3" number="4.2.2">
<h3><span class="header-section-number">4.2.2</span> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</h3>
<p>The broad host range expression plasmid pMMB67EH<span class="citation"><sup><a href="#ref-furste_molecular_1986" role="doc-biblioref">271</a></sup></span> was chosen as a suitable vector for expression experiments. A diagram of the vector is shown in Figure 4.3.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-3"></span>
<img src="figure/fig4-3.png" alt="Diagram of the broad host range expression vector pMMB67EH, showing relevant features. pMMB67EH [@furste_molecular_1986] was used for cloning and expression of the _nirS_ gene from _T. pantotropha_. Features are described fully in the main text and are as Tollows: _bla_, $eta$-lactamase gene conferring ampicillin resistance, P$_{tac}$, _tac_ promoter, _rrnB_, terminator derived from the _E. coli_ 5S rRNA gene, _oriV_, vegetative origin of replication, _oriT_, origin of transfer, _lacI_$^q$, over-produced form of the _lacI_ repressor protein. The vector contains an mp18 multiple cloning site and 6 of the restriction enzyme sites that cut in this region are indicated." width="75%" />
<p class="caption">
Figure 4.3: Diagram of the broad host range expression vector pMMB67EH, showing relevant features. pMMB67EH<span class="citation"><sup><a href="#ref-furste_molecular_1986" role="doc-biblioref">271</a></sup></span> was used for cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em>. Features are described fully in the main text and are as Tollows: <em>bla</em>, <span class="math inline">\(eta\)</span>-lactamase gene conferring ampicillin resistance, P<span class="math inline">\(_{tac}\)</span>, <em>tac</em> promoter, <em>rrnB</em>, terminator derived from the <em>E. coli</em> 5S rRNA gene, <em>oriV</em>, vegetative origin of replication, <em>oriT</em>, origin of transfer, <em>lacI</em><span class="math inline">\(^q\)</span>, over-produced form of the <em>lacI</em> repressor protein. The vector contains an mp18 multiple cloning site and 6 of the restriction enzyme sites that cut in this region are indicated.
</p>
</div>
<p>It contains a number of useful features, which are as follows: (1) A <em>tac</em> promoter, which is a strong hybrid promoter constructed from the <em>trp</em> and <em>lac</em> promoters<span class="citation"><sup><a href="#ref-de_boer_tac_1983" role="doc-biblioref">233</a></sup></span> for driving transcription, (2) the mp18 multiple cloning site which is compatible with the pBS(-) plasmid, in two possible orientations (these are designated pMMB67EH and HE), (3) a transcriptional terminator, derived from the <em>rrnB</em> gene, (4) the <span class="math inline">\(\beta\)</span>-lactamase gene conferring ampicillin resistance, (5) the <em>lacI</em><span class="math inline">\(^q\)</span> gene, encoding a mutant form of the <em>lacI</em> repressor<span class="citation"><sup><a href="#ref-gronenborn_overproduction_1976" role="doc-biblioref">272</a></sup></span> which is overproduced to prevent transcription in the absence of IPTG and (6) an origin of transfer, <em>oriT</em>, allowing transfer of the plasmid to other Gram-negative species by conjugation.<span class="citation"><sup><a href="#ref-derbyshire_mobilization_1987" role="doc-biblioref">273</a></sup></span> This last feature allows the use of the same construct to investigate expression in other bacteria, which is described in Chapters 5 and 6.</p>
<p>In theory the 2.4 kbp <em>PstI-SphI</em> fragment derived from the plasmid pBNIR should be easily cloned into the mp18 cloning site of pMMB67EH. However, difficulty was experienced due to the problem of vector carryover as shown in Figure 4.4.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-4"></span>
<img src="figure/fig4-4.png" alt="Restriction digests illustrating the problem of vector carryover during cloning of the _T. pantotropha_ _nirS_ gene into pMMB67EH. A 2.4 kbp _PstI-SphI_ fragment of pBNIR was isolated and used in a ligation reaction with pMMB67EH, also digested with _PstI-SphI_. _E. coli_ JM83 was then transformed with the ligation mix and ampicillin resistant clones screened for the presence of the desired plasmid by restriction digests using _PstI_ plus _SphI_ and electrophoresis on a 0.8% agarose gel. However, in this case, all the plasmids recovered are either the original pBNIR plasmid (lanes 3, 6 and 7) or the pMMB67EH plasmid (lanes 1, 2, 4, 5 and 8) that has re-ligated due to incomplete digestion. Solutions to the problem are described in the text. The gel is displayed as a negative image to show faint bands more clearly. Lanes labelled III and II contain DNA size standards, the sizes of some of which are indicated at the left of the gel." width="75%" />
<p class="caption">
Figure 4.4: Restriction digests illustrating the problem of vector carryover during cloning of the <em>T. pantotropha</em> <em>nirS</em> gene into pMMB67EH. A 2.4 kbp <em>PstI-SphI</em> fragment of pBNIR was isolated and used in a ligation reaction with pMMB67EH, also digested with <em>PstI-SphI</em>. <em>E. coli</em> JM83 was then transformed with the ligation mix and ampicillin resistant clones screened for the presence of the desired plasmid by restriction digests using <em>PstI</em> plus <em>SphI</em> and electrophoresis on a 0.8% agarose gel. However, in this case, all the plasmids recovered are either the original pBNIR plasmid (lanes 3, 6 and 7) or the pMMB67EH plasmid (lanes 1, 2, 4, 5 and 8) that has re-ligated due to incomplete digestion. Solutions to the problem are described in the text. The gel is displayed as a negative image to show faint bands more clearly. Lanes labelled III and II contain DNA size standards, the sizes of some of which are indicated at the left of the gel.
</p>
</div>
<p>This problem occurs when a fragment of similar size to the vector fragment is isolated from an agarose gel. A small amount of vector can be carried over with the desired fragment and, as pBS(-) is smaller than pMMB67EH, religation of the original plasmid occurs with higher efficiency producing clones containing the original vector. This is a particular problem when the two vectors contain the same antibiotic resistance determinant (in this case ampicillin). To circumvent this problem, a kanamycin resistance cassette was inserted in the <em>PstI</em> site of pBNIR to give the plasmid pBNIRK. The cassette and <em>nirS</em> gene were then isolated together as an <em>XbaI-SphI</em> fragment and cloned into pMMB67EH. This step introduces a new antibiotic resistance determinant and also increases the size of the insert by 1.2 kbp, thus improving its separation from the vector fragment. Several positive clones were selected by kanamycin resistance. The isolated plasmid, pMMBSEK, was then cut with <em>PstI</em> and subjected to partial religation to remove the kanamycin cassette. Cells of <em>E. coli</em> JM83 were re-transformed and ampicillin resistant clones were streaked onto plates containing either ampicillin or ampicillin plus kanamycin. Kanamycin sensitive clones, which had lost the cassette, were re-isolated and the construct was confirmed by restriction digests (Figure 4.5).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-5"></span>
<img src="figure/fig4-5.png" alt="Restriction digest analysis of the plasmids pMMBSE and pMMBSEK. Digests of pMMBSE were performed using _SalI_ (S), _BamHI_ (B), _HindIII_ (H), _BglII_ (Bg), _EcoRI_ (E) and _PstI_ (P) and the fragments were separated by electrophoresis on a 0.8% agarose gel. The fragments shown above are of the expected sizes from the 2.4 kbp fragment of pBNIR, containing the _nirS_ gene, in pMMB67EH. Lane 7 contains a _HindIII_ digest of the plasmid pMMBSEK. This clearly shows an extra fragment of 1.38 kbp from pMMBSEK, which arises due to an extra _HindIII_ site in the centre of the kanamycin cassette. The lane labelled III contains DNA size markers, the sizes of some of which are indicated at the left of the gel." width="75%" />
<p class="caption">
Figure 4.5: Restriction digest analysis of the plasmids pMMBSE and pMMBSEK. Digests of pMMBSE were performed using <em>SalI</em> (S), <em>BamHI</em> (B), <em>HindIII</em> (H), <em>BglII</em> (Bg), <em>EcoRI</em> (E) and <em>PstI</em> (P) and the fragments were separated by electrophoresis on a 0.8% agarose gel. The fragments shown above are of the expected sizes from the 2.4 kbp fragment of pBNIR, containing the <em>nirS</em> gene, in pMMB67EH. Lane 7 contains a <em>HindIII</em> digest of the plasmid pMMBSEK. This clearly shows an extra fragment of 1.38 kbp from pMMBSEK, which arises due to an extra <em>HindIII</em> site in the centre of the kanamycin cassette. The lane labelled III contains DNA size markers, the sizes of some of which are indicated at the left of the gel.
</p>
</div>
<p>This procedure yielded two plasmids for further work: (1) pMMBSE, containing the <em>nirS</em> gene together with its upstream region (the NNR box and promoter region) and part of the downstream <em>nirE</em> gene, and (2) pMMBSEK, in which the <em>tac</em> promoter and the NNR box are separated by a kanamycin resistance cassette, transcribed in the same direction as the <em>nirS</em> gene. A list of the plasmids referred to in this chapter is given in Table 4.1.</p>
<table>
<caption>
<span id="tab:table41">Table 4.1: </span>Plasmids referred to in this chapter
</caption>
<thead>
<tr>
<th style="text-align:left;">
Plasmid name
</th>
<th style="text-align:left;">
Description
</th>
<th style="text-align:left;">
Source or reference
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
pBS(-)
</td>
<td style="text-align:left;">
General purpose cloning vector
</td>
<td style="text-align:left;">
Stratagene
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMB67EH
</td>
<td style="text-align:left;">
Broad host range expression vector
</td>
<td style="text-align:left;">
Fürste et al. (1986)<span class="citation"><sup><a href="#ref-furste_molecular_1986" role="doc-biblioref">271</a></sup></span>
</td>
</tr>
<tr>
<td style="text-align:left;">
pTNIR3
</td>
<td style="text-align:left;">
7.5 kbp <em>SphI</em> fragment of genomic DNA from <em>T. pantotropha</em> containing <em>nirS</em> and flanking DNA
</td>
<td style="text-align:left;">
Obtained from Thon de Boer, Vrije Universiteit, Amsterdam
</td>
</tr>
<tr>
<td style="text-align:left;">
pBNIR
</td>
<td style="text-align:left;">
2.4 kbp Pstl-Sphl fragment of pTNIR3, containing the <em>nirS</em> gene, cloned in pBS(-)
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pB5K
</td>
<td style="text-align:left;">
5.1 kbp <em>PstI-SphI</em> fragment of pTNIR3, containing the genes <em>nirI-norB</em>, cloned in pBS(-)
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
2.4 kbp <em>Xba-SphI</em> fragment of pBNIR, containing the <em>kan</em> gene (kanamycin resistance) in the <em>PstI</em> site, and the <em>nirS</em> gene, cloned in pMMB67EH
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMBSE
</td>
<td style="text-align:left;">
pMMBSEK cut with <em>PstI</em> to remove kan gene and religated
</td>
<td style="text-align:left;">
This work
</td>
</tr>
</tbody>
</table>
</div>
<div id="expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions" class="section level3" number="4.2.3">
<h3><span class="header-section-number">4.2.3</span> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</h3>
<p>In order to obtain a complete analysis of expression from the <em>nirS</em> gene in <em>E. coli</em> JM83, cells of wild-type <em>E. coli</em> JM83, <em>E. coli</em> JM83 [pMMB67EH], <em>E. coli</em> JM83 [pMMBSEK] and <em>E. coli</em> JM83 [pMMBSE] were grown under both aerobic (in LB medium) anaerobic (minimal medium containing fumarate plus nitrite) conditions. Additionally, cultures of <em>E. coli</em> JM83 [pMMBSEK] and <em>E. coli</em> [pMMBSE] were induced with 1 mM IPTG during the early exponential phase of growth to activate transcription from the <em>tac</em> promoter. Twelve different cultures were grown as outlined in Table 4.2.</p>
<table>
<caption>
<span id="tab:table42">Table 4.2: </span>Different growth conditions used to compare the growth of <em>E. coli</em> JM83 with that of <em>E. coli</em> containing either pMMB67EH, pMMBSEK or pMMBSE. Wild-type <em>E. coli</em> JM83 and <em>E. coli</em> JM83 [pMMB67EH] were grown either aerobically or anaerobically. <em>E. coli</em> JM83 containing either pMMBSEK or pMMBSE were also grown under these conditions; additionally, these strains could be induced with IPTG during growth under either condition. This gave rise to the 12 different cultures indicated in the table.
</caption>
<thead>
<tr>
<th style="text-align:right;">
Culture number
</th>
<th style="text-align:left;">
Growth conditions
</th>
<th style="text-align:left;">
Plasmid present
</th>
<th style="text-align:left;">
IPTG added
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
pMMB67EH
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
pMMB67EH
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
yes
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
yes
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
pMMBSE
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
pMMBSE
</td>
<td style="text-align:left;">
yes
</td>
</tr>
<tr>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
pMMBSE
</td>
<td style="text-align:left;">
no
</td>
</tr>
<tr>
<td style="text-align:right;">
12
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
pMMBSE
</td>
<td style="text-align:left;">
yes
</td>
</tr>
</tbody>
</table>
<p>50 ml of each medium was inoculated with a 1:100 dilution of overnight culture that had been grown in 1.5 ml LB medium plus appropriate antibiotics. Aerobic cultures were sampled every 30 minutes and the absorbance at 650 nm was recorded to give the growth curves shown in Figure 4.6.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-6"></span>
<img src="figure/fig4-6.png" alt="Comparison of the aerobic growth of _E. coli_, E. coli [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]. Overnight cultures of wild-type cells and those carrying plasmid were inoculated into LB medium at a dilution of 1/100 and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). Symbols: (&amp;#9670;) _E. coli_ JM83, (&amp;#9632;) _E. coli_ [pMMB67EH], (&amp;#9650;) _E. coli_ [pMMBSEK], (x) _E. coli_ [pMMBSEK] induced with 1 mM IPTG, (&amp;#9633;) _E. coli_ [pMMBSE], (&amp;#9679;) _E. coli_ [pMMBSE] induced with 1 mM IPTG." width="75%" />
<p class="caption">
Figure 4.6: Comparison of the aerobic growth of <em>E. coli</em>, E. coli [pMMB67EH], <em>E. coli</em> [pMMBSEK] and <em>E. coli</em> [pMMBSE]. Overnight cultures of wild-type cells and those carrying plasmid were inoculated into LB medium at a dilution of 1/100 and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). Symbols: (◆) <em>E. coli</em> JM83, (■) <em>E. coli</em> [pMMB67EH], (▲) <em>E. coli</em> [pMMBSEK], (x) <em>E. coli</em> [pMMBSEK] induced with 1 mM IPTG, (□) <em>E. coli</em> [pMMBSE], (●) <em>E. coli</em> [pMMBSE] induced with 1 mM IPTG.
</p>
</div>
<p>Reliable growth curves were not obtained for the anaerobic cultures, because in the cultures containing the <em>nirS</em> gene the cells tended to sediment and adhere quite strongly to the lower wall of the culture tube. The cells could be resuspended by vigorous vortexing, but this was avoided to maintain anaerobic conditions in the tubes. Instead, anaerobic cultures were vortexed once after 12 hours to ascertain that they had entered early exponential phase. At this point, cultures containing <em>E. coli</em> [pMMBSE] and E. coli [pMMBSEK] were induced with 1 mM IPTG and the cultures were left overnight for a total of 24 hours. Comparison with the early growth curve of the anaerobic culture of <em>E. coli</em> [pMMB67EH] indicated that this growth period was sufficient to reach stationary phase without over-growth (Figure 4.7).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-7"></span>
<img src="figure/fig4-7.png" alt="Comparison of the anaerobic growth of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH]. Overnight cultures of wild-type cells and those carrying pMMB67EH were inoculated into _E. coli_ anaerobic medium (Materials and Methods) and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). As explained in the main text, growth curves for cells containing the other plasmids, pMMBSE and pMMBSEK, could not be obtained. Symbols: (&amp;#9670;) _E. coli_ JM83, (&amp;#9632;) _E. coli_ [pMMB67EH]." width="75%" />
<p class="caption">
Figure 4.7: Comparison of the anaerobic growth of <em>E. coli</em> JM83 with that of <em>E. coli</em> [pMMB67EH]. Overnight cultures of wild-type cells and those carrying pMMB67EH were inoculated into <em>E. coli</em> anaerobic medium (Materials and Methods) and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). As explained in the main text, growth curves for cells containing the other plasmids, pMMBSE and pMMBSEK, could not be obtained. Symbols: (◆) <em>E. coli</em> JM83, (■) <em>E. coli</em> [pMMB67EH].
</p>
</div>
<p>After 24 hours, the cultures were vortexed to record the final optical density and then harvested. Aerobic cultures were harvested when judged to have just entered stationary phase. Total soluble extracts were prepared from each culture by sonication and ultracentrifugation of the lysate.</p>
<p>The protein profile in the extracts from each of the 12 cultures was first investigated using SDS-PAGE and staining for protein using Coomassie Blue (Figure 4.8).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-8"></span>
<img src="figure/fig4-8.png" alt="Comparison of the protein content of _E. coli_ JM83 with that of _E. coli_ [pMMB67EH], _E. coli_ [pMMBSEK] and _E. coli_ [pMMBSE]. 40 $\mu$g of total protein per lane was separated by SDS-PAGE on an 8% acrylamide gel and the gel was stained for protein using Coomassie Brilliant Blue R-250. Lane M contains molecular weight markers, the sizes of some of which are indicated at the left of the gel. The lanes are labelled to indicate the growth conditions, plasmid present and presence or absence of IPTG. A 63 kDa band, corresponding to expressed _T. pantotropha_ cytochrome _cd_$_1$, is visible in lanes 4 and 8." width="75%" />
<p class="caption">
Figure 4.8: Comparison of the protein content of <em>E. coli</em> JM83 with that of <em>E. coli</em> [pMMB67EH], <em>E. coli</em> [pMMBSEK] and <em>E. coli</em> [pMMBSE]. 40 <span class="math inline">\(\mu\)</span>g of total protein per lane was separated by SDS-PAGE on an 8% acrylamide gel and the gel was stained for protein using Coomassie Brilliant Blue R-250. Lane M contains molecular weight markers, the sizes of some of which are indicated at the left of the gel. The lanes are labelled to indicate the growth conditions, plasmid present and presence or absence of IPTG. A 63 kDa band, corresponding to expressed <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, is visible in lanes 4 and 8.
</p>
</div>
<p>Equal amounts of protein from each sample were loaded in each lane and separated on 8% acrylamide gels. A band corresponding to the size of mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (63 kDa) was apparent only in the samples induced using IPTG from <em>E. coli</em> [pMMBSE], grown either aerobically or anaerobically, and is more prominent in the latter sample.</p>
<p>Next, an identical set of samples was separated by SDS-PAGE, transferred to nitrocellulose by semi-dry electroblotting and analysed by Western blotting using a mouse antibody to <em>P. denitrificans</em> PD1222 cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (obtained from Thon de Boer, Vrije Universiteit, Amsterdam). For comparison, a soluble extract from <em>T. pantotropha</em> grown anaerobically with nitrate was run on the same gel. Only the aerobic and anaerobic samples from <em>E. coli</em> [pMMBSE] that had been induced using IPTG gave a positive reaction for the presence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (Figure 4.9).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-9"></span>
<img src="figure/fig4-9.png" alt="Western blot of protein samples from _E. coli_ JM83, _E. coli_ [pMMB67EH], E. coli [pMMBSEK] and _E. coli_ [pMMBSE], probed using antibody to cytochrome _cd_$_1$. 40 $\mu$g of protein from the cultures indicated above the lanes was separated by electrophoresis on an 8% SDS-PAGE gel and transferred to nitrocellulose by semi-dry electroblotting. The membrane was then probed a cytochrome _cd_$_1$ antibody, detected colorimetrically with a secondary antibody coupled to alkaline phosphatase, as detailed in Materials and Methods. The lane labelled Tp contained 20 $\mu$g of protein from a soluble extract of wild-type _T. pantotropha_ grown anaerobically with nitrate." width="75%" />
<p class="caption">
Figure 4.9: Western blot of protein samples from <em>E. coli</em> JM83, <em>E. coli</em> [pMMB67EH], E. coli [pMMBSEK] and <em>E. coli</em> [pMMBSE], probed using antibody to cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. 40 <span class="math inline">\(\mu\)</span>g of protein from the cultures indicated above the lanes was separated by electrophoresis on an 8% SDS-PAGE gel and transferred to nitrocellulose by semi-dry electroblotting. The membrane was then probed a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> antibody, detected colorimetrically with a secondary antibody coupled to alkaline phosphatase, as detailed in Materials and Methods. The lane labelled Tp contained 20 <span class="math inline">\(\mu\)</span>g of protein from a soluble extract of wild-type <em>T. pantotropha</em> grown anaerobically with nitrate.
</p>
</div>
<p>The band in the anaerobic sample was somewhat stronger than that in the aerobic lane, and both bands ran at the same position as the wild type protein.</p>
<p>Finally, the samples were analysed for the presence of <em>c</em>-type cytochromes by SDS PAGE followed by haem staining. A sample of purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> was run on this gel for comparison (Figure 4.10).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-10"></span>
<img src="figure/fig4-10.png" alt="Comparison of the _c_-type cytochrome content of _E. coli_ [pMMBSE] grown under aerobic or anaerobic conditions. 100 $\mu$g of protein from the soluble extracts indicated was separated by electrophoresis on an 8% SDS-PAGE gel and stained for the presence of covalently-attached haem using tetramethylbenzidine and H$_2$O$_2$, as described in Materials and Methods. The lane labelled Tp contained 5 $\mu$g of purified cytochrome _cd_$_1$ for comparison." width="75%" />
<p class="caption">
Figure 4.10: Comparison of the <em>c</em>-type cytochrome content of <em>E. coli</em> [pMMBSE] grown under aerobic or anaerobic conditions. 100 <span class="math inline">\(\mu\)</span>g of protein from the soluble extracts indicated was separated by electrophoresis on an 8% SDS-PAGE gel and stained for the presence of covalently-attached haem using tetramethylbenzidine and H<span class="math inline">\(_2\)</span>O<span class="math inline">\(_2\)</span>, as described in Materials and Methods. The lane labelled Tp contained 5 <span class="math inline">\(\mu\)</span>g of purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> for comparison.
</p>
</div>
<p>Once again, the only positive lanes were those containing protein samples from <em>E. coli</em> [pMMBSE] grown either aerobically or anaerobically. The anaerobic sample stained appreciably more strongly than the aerobic sample, and the mobilities of the haem stained bands were identical to that of the wild-type cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.</p>
<p>The soluble extract from anaerobically-grown <em>E. coli</em> [pMMBSE] was pink. A sample of the extract was analysed using UV-visible spectroscopy (Figure 4.11).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-11"></span>
<img src="figure/fig4-11.png" alt="Visible spectrum of soluble extract from _E. coli_ [pMMBSE] grown anaerobically and induced with 1 mM IPTG. 1 ml of soluble extract was prepared from 50 ml of cells and used directly for spectroscopy in 50mM Tris-HCl, pH 8.0. Absorbance maxima are at 416 (not shown), 522, 548 and 554 nm." width="75%" />
<p class="caption">
Figure 4.11: Visible spectrum of soluble extract from <em>E. coli</em> [pMMBSE] grown anaerobically and induced with 1 mM IPTG. 1 ml of soluble extract was prepared from 50 ml of cells and used directly for spectroscopy in 50mM Tris-HCl, pH 8.0. Absorbance maxima are at 416 (not shown), 522, 548 and 554 nm.
</p>
</div>
<p>The spectrum obtained was extremely similar to that of chemically-prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the reduced form and showed absorbance maxima at 416, 521, 548 and The peaks at 548 and 554 nm were reversed in magnitude when compared to the spectrum of the holoprotein, as was observed with the chemically-prepared semi-apoprotein (Chapter 3, Figure 3.9).</p>
<p>To partially purify the expressed semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, a 1 l culture of <em>E. coli</em> [pMMBSE] was grown anaerobically, induced with 1 mM IPTG in early exponential phase and harvested at stationary phase, the final A<span class="math inline">\(_{650}\)</span> being around 0.7. A soluble extract was prepared and subjected to ion exchange chromatography on a small DEAE-Sepharose column. The elution profile is shown in Figure 4.12.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-12"></span>
<img src="figure/fig4-12.png" alt="Elution profile during ion exchange chromatography of soluble extract from anaerobically-grown _E. coli_ [pMMBSE] induced using IPTG. 1 l of cells was grown anaerobically to stationary phase and a soluble extract was prepared by sonication in 50 mM Tris-HCl pH 8.0 followed by ultracentrifugation. The extract was bound to a DEAE-Sepharose CL6B column (1 x 10 cm) equilibrated in the same buffer and eluted with a 200 ml gradient (0-400 mM), also in 50 mM Tris-HCl pH 8.0. Symbols: Elution of haem proteins was followed by absorption at 410 nm (&amp;#9632;), NaCl gradient (&amp;#9670;)." width="75%" />
<p class="caption">
Figure 4.12: Elution profile during ion exchange chromatography of soluble extract from anaerobically-grown <em>E. coli</em> [pMMBSE] induced using IPTG. 1 l of cells was grown anaerobically to stationary phase and a soluble extract was prepared by sonication in 50 mM Tris-HCl pH 8.0 followed by ultracentrifugation. The extract was bound to a DEAE-Sepharose CL6B column (1 x 10 cm) equilibrated in the same buffer and eluted with a 200 ml gradient (0-400 mM), also in 50 mM Tris-HCl pH 8.0. Symbols: Elution of haem proteins was followed by absorption at 410 nm (■), NaCl gradient (◆).
</p>
</div>
<p>Three major peaks absorbing at 410 nm were apparent, the last which eluted at around 250 mM NaCl which is the expected position for cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Fractions from this peak were pooled and concentrated, and the spectrum of the pooled fractions was recorded (Figure 4.13).</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-13"></span>
<img src="figure/fig4-13.png" alt="UV-visible spectrum of recombinant semi-apo cytochrome _cd_$_1$ following ion-exchange chromatography. The peak fractions (28-31) in Figure 4.12 were pooled and concentrated to approximately 200 $\mu$l. 40 $\mu$l of the concentrated material was then diluted 5-fold in 50 mM Tris-HCl pH 8.0 and the spectrum was recorded. Absorbance maxima are at 255, 294, 317, 416, 522,548 and 554 nm." width="75%" />
<p class="caption">
Figure 4.13: UV-visible spectrum of recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following ion-exchange chromatography. The peak fractions (28-31) in Figure 4.12 were pooled and concentrated to approximately 200 <span class="math inline">\(\mu\)</span>l. 40 <span class="math inline">\(\mu\)</span>l of the concentrated material was then diluted 5-fold in 50 mM Tris-HCl pH 8.0 and the spectrum was recorded. Absorbance maxima are at 255, 294, 317, 416, 522,548 and 554 nm.
</p>
</div>
<p>Reconstitution of the semi-apoprotein was attempted using haem <em>d</em><span class="math inline">\(_1\)</span> purified from a stock of native cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. In the reconstitution experiment 130 <span class="math inline">\(\mu\)</span>l of concentrated material from the ion exchange column, containing approximately 18.8 <span class="math inline">\(\mu\)</span>M semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was mixed with 150 <span class="math inline">\(\mu\)</span>l of haem <em>d</em><span class="math inline">\(_1\)</span>, prepared from approximately 40 mg of partially purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, in a total volume of 1 ml containing 50 mM Tris-HCl, pH 8.0. The solution was incubated at 4 °C for 30 minutes, with gentle mixing by inversion. On mixing the colour of the solution changed from pink to green. The sample was then diluted to 15 ml with 50 mM Tris-HCl pH 8.0, concentrated in a 50 kDa cut-off centrifugal filtration unit, diluted once more to 15 ml and reconcentrated to approximately 300 <span class="math inline">\(\mu\)</span>l, with the aim of retaining cytochrome <em>cd</em><span class="math inline">\(_1\)</span> plus bound haem and removing unbound haem <em>d</em><span class="math inline">\(_1\)</span>. After concentration, the spectrum of the sample was recorded as isolated and after the addition of sodium dithionite. These spectra are shown in Figure 4.14.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-14"></span>
<img src="figure/fig4-14.png" alt="Visible spectra of recombinant cytochrome _cd_$_1$ following reconstitution with purified haem _d_$_1$ (a) as prepared, (b) following reduction using sodium dithionite, compared with (c) native cytochrome _cd_$_1$ from _T. pantotropha_. Approximately 2.4 nmol of recombinant semi-apo cytochrome _cd_$_1$ was reconstituted with haem _d_$_1$ as described in the main text and concentrated to approximately 300 $\mu$l. A 200 $\mu$l sample was used to record spectrum (a) and was then reduced using sodium dithionite to record spectrum (b). Spectrum (c) was recorded from 1.5 $\mu$M of purified native cytochrome _cd_$_1$ in 50 mM Tris-HCl pH 8.0, also reduced using sodium dithionite." width="75%" />
<p class="caption">
Figure 4.14: Visible spectra of recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following reconstitution with purified haem <em>d</em><span class="math inline">\(_1\)</span> (a) as prepared, (b) following reduction using sodium dithionite, compared with (c) native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>. Approximately 2.4 nmol of recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was reconstituted with haem <em>d</em><span class="math inline">\(_1\)</span> as described in the main text and concentrated to approximately 300 <span class="math inline">\(\mu\)</span>l. A 200 <span class="math inline">\(\mu\)</span>l sample was used to record spectrum (a) and was then reduced using sodium dithionite to record spectrum (b). Spectrum (c) was recorded from 1.5 <span class="math inline">\(\mu\)</span>M of purified native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in 50 mM Tris-HCl pH 8.0, also reduced using sodium dithionite.
</p>
</div>
<p>In the reduced form, the reconstituted protein exhibited a prominent shoulder at 460 nm and a broad peak at around 625 nm, both of which are diagnostic for the presence of bound haem <em>d</em><span class="math inline">\(_1\)</span>. The activity of the expressed semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, reconstituted cytochrome <em>cd</em><span class="math inline">\(_1\)</span> and native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (purified from <em>T. pantotropha</em>) was assayed using reduced methyl viologen as an electron donor. Assay traces for the three samples are shown in Figure 4.15 and estimates of turnover number are summarised below the traces.</p>
<div class="figure" style="text-align: center"><span id="fig:fig4-15"></span>
<img src="figure/fig4-15.png" alt="Assays for nitrite reductase activity using reduced methyl viologen as an electron donor: (a) recombinant semi-apo cytochrome _cd_$_1$, (b) recombinant cytochrome _cd_$_1$ following reconstitution with _d_$_1$ haem, (c) native cytochrome _cd_$_1$ purified from _T. pantotropha_. All assays were performed anaerobically in a volume of 1 ml as described in Materials and Methods. The traces show the decrease in absorbance at 600 nm as the methyl viologen becomes oxidised. The reaction rates used for the calculations described in the main text were obtained by subtraction of the drift rate (before nitrite addition) from the initial rate (after nitrite addition)" width="75%" />
<p class="caption">
Figure 4.15: Assays for nitrite reductase activity using reduced methyl viologen as an electron donor: (a) recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, (b) recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following reconstitution with <em>d</em><span class="math inline">\(_1\)</span> haem, (c) native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> purified from <em>T. pantotropha</em>. All assays were performed anaerobically in a volume of 1 ml as described in Materials and Methods. The traces show the decrease in absorbance at 600 nm as the methyl viologen becomes oxidised. The reaction rates used for the calculations described in the main text were obtained by subtraction of the drift rate (before nitrite addition) from the initial rate (after nitrite addition)
</p>
</div>
</div>
</div>
<div id="discussion-1" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Discussion</h2>
<p>When grown aerobically all of the <em>E. coli</em> strains exhibited similar growth rates (Figure 4.6). The final optical density reached was similar for all strains carrying a plasmid selected by ampicillin resistance (pMMB67EH or pMMBSE), but slightly lower for plasmids selected by kanamycin resistance (pMMBSEK). As no cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was expressed from plasmid pMMBSEK (Figure 4.9), this effect seems likely to result from the stress of countering the antibiotic. Thus it is concluded that, aerobically, the <em>nirS</em> gene exerts no toxic effects on <em>E. coli</em> JM83.</p>
<p>Under anaerobic conditions, a greater disparity was observed between cultures of <em>E. coli</em> when compared with cultures carrying the different plasmids. Cells carrying the parent plasmid pMMB67EH, selected by ampicillin resistance, showed a pronounced lag when compared to the wild-type strain, taking around 24 hours to reach stationary phase as compared to around 16 hours for the wild-type (Figure 4.7) and the final optical density reached was slightly lower. This again suggests that the combined stresses of growing anaerobically in the minimal medium and countering the effects of the antibiotic acted together to slow the growth of the cells. Optical densities of cultures containing anaerobically grown <em>E. coli</em> [pMMBSE] and <em>E. coli</em> [pMMBSEK] could not be accurately determined because the cells in these cultures tended to sediment and adhere strongly to the walls of the tube. However, the final density of these cultures did appear to be somewhat lower than that of <em>E. coli</em> [pMMB67EH] grown anaerobically. The reduced growth rate and adherence to the walls of the tube occurred in <em>E. coli</em> [pMMBSE] and <em>E. coli</em> [pMMBSEK] irrespective of the presence of IPTG and so do not appear to be a consequence of expression of the <em>nirS</em> gene. It seems that cells in the latter two cultures were experiencing stress arising from an as-yet unidentified factor. One possible cause might be expression of the truncated <em>nirE</em> gene during anaerobic growth. The first 648 bp (encoding 216 amino acids) of the <em>nirE</em> gene are present on the plasmids pMMBSE and pMMBSEK and it is possible that the truncated gene product is toxic to the cells. Plasmids containing the upstream region of the <em>nirE</em> gene have also been observed to reduce yields of cells and DNA during routine cloning procedures in <em>E. coli</em> (Dr. Simon Baker, personal communication).</p>
<p>A Western blot of extracts from cells grown under various conditions (Figure 4.9) showed that cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was expressed only from pMMBSE when induced using IPTG. No cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was detectable without induction, indicating that the endogenous <em>nirS</em> promoter is not functional in <em>E. coli</em> under either aerobic or anaerobic conditions. Further evidence for this assertion is supplied by strains carrying the plasmid pMMBSEK, a construct where a kanamycin cassette lies between the <em>tac</em> promoter and the NNR box. In this construct, transcription from the kanamycin gene promoter would be expected to interfere with transcription from the upstream <em>tac</em> promoter, but not with the downstream endogenous promoter region, were it to be active. This assumes that the NNR box and downstream regions contain all the necessary information for transcriptional activation of the <em>nirS</em> gene; Chapter 7 discusses the region in more detail, but by analogy to FNR-activated genes it is believed that the above assumption is true. In cultures of <em>E. coli</em> [pMMBSEK] no expression was observed under any growth condition, even when induced with IPTG.</p>
<p>Under anaerobic growth conditions expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in <em>E. coli</em> was enhanced, as judged by Western blotting. <em>E. coli</em> has only been shown quite recently to synthesise a range of previously uncharacterised <em>c</em>-type cytochromes and it has been demonstrated that these are most abundant during anaerobic growth.<span class="citation"><sup><a href="#ref-iobbi-nivol_reassessment_1994" role="doc-biblioref">269</a></sup></span> In the case of the endogenous <em>c</em>-type cytochromes the increase under anaerobic growth conditions is the result of several factors, including activation by FNR and other transcriptional activators, such as the NarX/L family.<span class="citation"><sup><a href="#ref-darwin_nitrate_1995" role="doc-biblioref">274</a></sup></span> However, for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> these factors are unlikely to be operative as the cloned fragment contains only the NNR box, which is probably not recognised by the <em>E. coli</em> FNR protein.<span class="citation"><sup><a href="#ref-spiro_fnr_1994" role="doc-biblioref">186</a></sup></span> The increased amounts of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> under anaerobic growth conditions are due in large part to the increased expression levels of the cytochrome <em>c</em> biogenesis genes under these conditions.<span class="citation"><sup><a href="#ref-thony-meyer_escherichia_1995" role="doc-biblioref">275</a></sup></span> The amount of covalently-attached <em>c</em> haem increased in parallel with the increased levels of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> polypeptide under anaerobic conditions (Figure 4.10), indicating that haem supply is increased anaerobically in <em>E. coli</em>. Previous studies have shown that expression of the <em>hemA</em> gene in <em>E. coli</em>, which encodes glutamyl-tRNA dehydrogenase, a haem precursor biosynthetic enzyme, is enhanced 2.5-fold under anaerobic as compared to aerobic growth conditions.<span class="citation"><sup><a href="#ref-darie_effect_1994" role="doc-biblioref">276</a></sup></span> It has also been demonstrated that expression of cytochrome b<span class="math inline">\(_5\)</span> in <em>E. coli</em> provides a “haem sink,” which increases the cellular haem content,<span class="citation"><sup><a href="#ref-woodard_regulation_1995" role="doc-biblioref">277</a></sup></span> and a similar effect may result from cytochrome <em>cd</em><span class="math inline">\(_1\)</span> expression.</p>
<p>Western blotting and protein staining of the protein extracts showed that the expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was the same size as native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> (Figures 4.8, 4.9). This indicates that the targeting sequence was recognised by the periplasmic export complex of <em>E. coli</em> and efficiently cleaved. It also suggests that the protein is exported to the periplasm. The fact that <em>c</em>-type haem is covalently attached also suggests correct targeting, as attachment of <em>c</em>-type haem is believed to take place in the periplasm.<span class="citation"><sup><a href="#ref-page_apo_1990" role="doc-biblioref">278</a></sup></span> Thus it appears that recognition of the foreign <em>c</em>-type haem attachment site, processing and export to the periplasm can all occur in <em>E. coli</em>, implying that the steps are fundamentally similar to those in <em>P. denitrificans</em> and other Gram-negative bacteria. Similar observations have been made for other <em>c</em>-type cytochromes although the presence or absence of the <em>c</em>-type haem seems to depend very much on the particular protein. Examples of <em>c</em>-type cytochromes that have been successfully expressed as the holoprotein in <em>E. coli</em> include cytochrome <em>c</em><span class="math inline">\(_2\)</span> from <em>Rhodobacter sphaeroides</em>,<span class="citation"><sup><a href="#ref-grisshammer_expression_1991" role="doc-biblioref">279</a></sup></span> though not the same protein from <em>R. viridis</em>, cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> from <em>P. denitrificans</em>,<span class="citation"><sup><a href="#ref-sambongi_alteration_1996" role="doc-biblioref">280</a></sup></span> cytochrome <em>c</em><span class="math inline">\(_{553}\)</span><span class="citation"><sup><a href="#ref-pollock_aerobic_1994" role="doc-biblioref">281</a></sup></span> and DcrA<span class="citation"><sup><a href="#ref-fu_dcra_1994" role="doc-biblioref">282</a></sup></span> from <em>Desulfovibrio vulgaris</em>, mitochondrial cytochrome <em>c</em> from <em>Saccharomyces cerevisiae</em><span class="citation"><sup><a href="#ref-sambongi_alteration_1996" role="doc-biblioref">280</a></sup></span> and cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> from <em>Thiobacillus versutus</em>.<span class="citation"><sup><a href="#ref-ubbink_cytochrome_1992" role="doc-biblioref">283</a></sup></span> The optimal growth conditions also seem to be specific to each case; cytochrome <em>c</em><span class="math inline">\(_2\)</span> required anaerobic growth, whilst <em>T. versutus</em> cytochrome <em>c</em><span class="math inline">\(_550\)</span> was optimally expressed semi-anaerobically and the <em>D. vulgaris</em> cytochrome was optimally expressed under aerobic growth conditions. These observations have led to the suggestion that successful covalent haem attachment in <em>E. coli</em> may depend on the physical properties of the apoprotein. Anomalously, cytochrome <em>c</em><span class="math inline">\(_{552}\)</span> from the thermophilic bacterium <em>Hydrogenobacter thermophilus</em> can apparently attach <em>c</em>-type haem in the cytoplasm of <em>E. coli</em>, as demonstrated using expression from a gene in which the periplasmic targeting sequence has been deleted.<span class="citation"><sup><a href="#ref-sambongi_synthesis_1994" role="doc-biblioref">284</a></sup></span> This is believed to result from spontaneous covalent attachment to the apoprotein, which possesses a highly ordered structure due to its thermostability.</p>
<p>Heterologous expression of <em>c</em>-type cytochromes is thus similar to that of other metalloproteins which have been expressed in <em>E. coli</em>, in that the complexity of the both the protein and its endogenous biosynthetic pathway both come to bear on the ease of expression. To give some examples, a number of proteins possessing the molybdopterin guanine dinucleotide cofactor (found in the dissimilatory nitrate reductases) have been successfully expressed,<span class="citation"><sup><a href="#ref-garrett_molecular_1994" role="doc-biblioref">285</a></sup></span> as <em>E. coli</em> possesses several enzymes that contain the cofactor (and therefore the biosynthetic machinery to process such proteins) Iron-sulphur proteins have also been expressed relatively easily,<span class="citation"><sup><a href="#ref-vidakovic_expression_1996" role="doc-biblioref">286</a></sup></span> as have simple copper proteins such as pseudoazurin,<span class="citation"><sup><a href="#ref-leung_pseudoazurin_1997" role="doc-biblioref">287</a></sup></span> but more complex copper proteins such as nitrous oxide reductase are expressed as the apoprotein<span class="citation"><sup><a href="#ref-viebrock_molecular_1988" role="doc-biblioref">288</a></sup></span> because of the absence of endogenous accessory gene products.</p>
<p>A spectrum of the soluble extract from anaerobically grown <em>E. coli</em> [pMMBSE] that had been induced using IPTG is shown in Figure 4.11. The quality of the spectrum was quite surprising, as it was very similar to that of pure semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, suggesting that there was very little contamination by endogenous <em>c</em>-type cytochromes and that the expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was the major cytochrome <em>c</em> in the cells. In <em>E. coli</em> 712, a strain of <em>E. coli</em> that over-produces cytochromes <em>c</em>, five endogenous <em>c</em>-type cytochromes can be detected by haem staining after SDS-PAGE.<span class="citation"><sup><a href="#ref-iobbi-nivol_reassessment_1994" role="doc-biblioref">269</a></sup></span> It appears that these cytochromes are less abundant in <em>E. coli</em> JM83, though a faint band corresponding to the <em>nrfA</em> gene product (a tetra- or possibly pentahaem nitrite reductase)<span class="citation"><sup><a href="#ref-darwin_regulation_1993" role="doc-biblioref">289</a></sup></span> could be seen in some haem-stained extracts from <em>E. coli</em> [pMMBSE]. The expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> shows the same characteristic absorption spectrum as the reduced chemically prepared form, with peaks at 416, 521, 548 and 554 nm, all originating from the <em>c</em>-haem, and the reversal relative to the holoprotein in the amplitude of the split peak at 548-554 nm noted in Chapter 3. Additionally, a peak can be seen at 317 nm. This peak is ascribed to low-spin reduced <em>c</em>-haem<span class="citation"><sup><a href="#ref-barber_spectral_1976" role="doc-biblioref">290</a></sup></span> and cannot normally be seen in the chemically-prepared semi-apo protein, as the addition of reducing agents such as sodium dithionite masks absorbance in this region. The expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> protein is reduced as isolated and remains stable in this form for several days at -20 °C. The same property and the 317 nm absorbance band have also been noted for the recombinant enzyme from <em>Ps. aeruginosa</em> when expressed in <em>Ps. putida</em>.<span class="citation"><sup><a href="#ref-silvestrini_expression_1992" role="doc-biblioref">268</a></sup></span></p>
<p>When assayed with reduced methyl viologen, expressed semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is devoid of detectable nitrite reductase activity (Figure 4.15). This is unsurprising, as the <em>d</em><span class="math inline">\(_1\)</span> haem has previously been shown to be the site of this reaction,<span class="citation"><sup><a href="#ref-silvestrini_reaction_1990" role="doc-biblioref">161</a></sup></span> although it has been reported that about 5% of the cytochrome <em>c</em> oxidase activity of native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> can be restored to the chemically prepared semi-apo protein by reconstitution using <em>a</em>-type haem.<span class="citation"><sup><a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a></sup></span> However, in the case of the expressed protein in <em>E. coli</em> it seems that alternative haems are not inserted <em>in vivo</em>, as no activity was present and the only spectroscopic features were those of <em>c</em>-haem. In contrast to this result, Ohshima et al. (1993)<span class="citation"><sup><a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a></sup></span> reported that expression of the <em>nirS</em> gene from <em>P. denitrificans</em> IFO 12442 in <em>E. coli</em> gave measurable nitrite reductase and nitric oxide activities. However, the construct used in these experiments contained 10 kbp of downstream DNA in addition to the <em>nirS</em> gene and as the same downstream region in <em>P. denitrificans</em> PD1222 contains the genes for haem <em>d</em><span class="math inline">\(_1\)</span> biosynthesis,<span class="citation"><sup><a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a></sup></span> it may be that the same genes are present in <em>P. denitrificans</em> IFO 12242, thus allowing the formation of some active cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in <em>E. coli</em>. The nitric oxide reductase activity is harder to explain, as the genes for this enzyme are upstream of the <em>nirS</em> gene in <em>P. denitrificans</em> PD1222<span class="citation"><sup><a href="#ref-de_boer_mutational_1996" role="doc-biblioref">171</a></sup></span> (and in <em>T. pantotropha</em>, as shown by mapping the plasmid pTNIR3, Figure 4.2). It should be noted that the activities reported by Ohshima et al. (1993)<span class="citation"><sup><a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a></sup></span> are extremely low (in the order of units per gram of protein) and are barely above background level. Furthermore, the expressed protein was detected only by Western blotting and all work was performed with crude extracts, with no direct demonstration that holoprotein was formed (<em>e.g.</em> by partial purification and spectroscopy). The results of Ohshima et al. (1993)<span class="citation"><sup><a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a></sup></span> should therefore be viewed with some scepticism.</p>
<p>To reconstitute expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> using haem <em>d</em><span class="math inline">\(_1\)</span> a soluble extract was prepared from 1 l of cells and partially purified on a small ion exchange column, using the same elution conditions as those employed during the purification of native cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Figure 4.12 shows the elution profile from this column, which confirms that semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, eluting at around 250 mM NaCl, was the major cytochrome <em>c</em> in the extract. SDS-PAGE of the peak fractions followed by staining for protein and haem showed the presence of a number of other proteins in the sample, although the expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was the only haem protein in the peak (data not shown). Further purification of the sample was not considered to be a sensible option because of the relatively low amounts of expressed protein obtained (about 1 mg from 1 l). Other groups have reported reconstitution of whole cell extracts,<span class="citation"><sup><a href="#ref-glockner_copper-containing_1993" role="doc-biblioref">63</a></sup></span> so purity of the sample was not considered to be of paramount importance.</p>
<p>On mixing purified haem <em>d</em><span class="math inline">\(_1\)</span> and expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> the pink colour was lost from the semi-apoprotein as the reduced <em>c</em>-haem was oxidised by the added haem <em>d</em><span class="math inline">\(_1\)</span>. After removal of excess haem <em>d</em><span class="math inline">\(_1\)</span>, the spectrum of the now oxidised protein was recorded (Figure 4.14a). The absorbance maxima were at similar positions to those in the native oxidised protein (407, 525 and 640 nm), although the peak at 407 nm was somewhat broader. The rise at around 640 nm due to the <em>d</em><span class="math inline">\(_1\)</span> haem was not as pronounced as that seen in the native enzyme, although there was clearly some absorption in this region. However, there was a clear shoulder at around 440 nm, indicative of absorption by the <em>d</em><span class="math inline">\(_1\)</span> haem.</p>
<p>The spectroscopic features became much clearer when the sample was reduced with sodium dithionite (Figure 4.14b). The double peak of the <em>c</em>-type haem was apparent, and the split peaks were of similar magnitude, which is indicative of holo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. There was also a peak at around 625 nm and the shoulder at 440 nm moved to 460 nm and increased in size. All of these features indicate that at least some of the protein has bound the <em>d</em><span class="math inline">\(_1\)</span> haem to yield holoenzyme, with the 625 nm peak being shifted due to the binding of breakdown products from the sodium dithionite.<span class="citation"><sup><a href="#ref-parr_reaction_1974" role="doc-biblioref">291</a></sup></span> However, the attenuated size of the 460 nm shoulder and 625 nm peak as compared to the native reduced protein (Figure 4.14c) suggests that not all of the enzyme has been reconstituted, and the spectrum is therefore a composite of the holo- and semi-apo forms.</p>
<p>The holo- and semi-apo forms of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> can be further separated using ion exchange chromatography, as the holo- form is more negatively charged and so binds to a DEAE-Sepharose column more tightly. However, this step was not performed, again due to the low amounts of material obtained. The sample was assayed for nitrite reductase activity, using reduced methyl viologen as electron donor and a trace from this assay is shown in Figure 4.15. The reconstituted sample is clearly active as a nitrite reductase. Estimates of turnover number are quite difficult to obtain from this experiment, because the mixture of reconstituted holoprotein and remaining semi-apoprotein means that the amount of reconstituted material cannot be estimated spectrophotometrically, using previously determined extinction coefficients for cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. However, a rough calculation was made as follows. 50 <span class="math inline">\(\mu\)</span>l of reconstituted material was assayed in a volume of 1 ml, and a methyl viologen oxidation rate of 0.109 <span class="math inline">\(\mu\)</span>mol min<span class="math inline">\(^{-1}\)</span> was recorded. Assuming that the absorbance of the reduced semi-apoprotein is not greatly different at 548 nm to that of the holoprotein, the extinction coefficient at 548 nm is 48.95 mM<span class="math inline">\(^{-1}\)</span> cm<span class="math inline">\(^{-1}\)</span>, indicating that the material used in the 1 ml reconstitution reaction contained approximately 2.4 nmol of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. As 50 <span class="math inline">\(\mu\)</span>l of this (= 0.4 nmol) was used in the assay, the turnover number is 109/0.4 giving approximately 273 <span class="math inline">\(\mu\)</span>mol of methyl viologen oxidised per minute per <span class="math inline">\(\mu\)</span>mol of total cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (<em>i.e.</em> reconstituted plus semi-apo protein). This is a similar figure to a typical assay of native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>. However, as already noted the spectrum of the reconstituted material (Figure 4.14b) differs to that of the native protein, indicating that not all of the protein has been reconstituted. Table 4.3 shows the ratios of some of the peak absorbances of the semi-apo, reconstituted and native proteins in the reduced and oxidised states.</p>
<table class="kable_wrapper">
<caption>
<span id="tab:table43">Table 4.3: </span>Comparison of the ratios of the absorbance maxima in (left) the reduced and (right) the oxidised states of native holo- and chemically prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> with those of the recombinant protein from <em>E. coli</em>. Data for native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was calculated using extinction coefficients obtained from Alrik Koppenhöfer; the absorbances for the recombinant protein were obtained from the spectra presented in this chapter.
</caption>
<tbody>
<tr>
<td>
<table>
<thead>
<tr>
<th style="text-align:left;">
Absorbance ratio
</th>
<th style="text-align:left;">
native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> reduced
</th>
<th style="text-align:left;">
expressed semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> reduced
</th>
<th style="text-align:left;">
reconstituted expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> reduced
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
418:522
</td>
<td style="text-align:left;">
7.48
</td>
<td style="text-align:left;">
5.73
</td>
<td style="text-align:left;">
3.36
</td>
</tr>
<tr>
<td style="text-align:left;">
418:548
</td>
<td style="text-align:left;">
6.60
</td>
<td style="text-align:left;">
5.72
</td>
<td style="text-align:left;">
3.32
</td>
</tr>
<tr>
<td style="text-align:left;">
418:554
</td>
<td style="text-align:left;">
6.68
</td>
<td style="text-align:left;">
5.62
</td>
<td style="text-align:left;">
3.32
</td>
</tr>
<tr>
<td style="text-align:left;">
418:650
</td>
<td style="text-align:left;">
8.06
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
4.25
</td>
</tr>
<tr>
<td style="text-align:left;">
548:650
</td>
<td style="text-align:left;">
1.22
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1.28
</td>
</tr>
<tr>
<td style="text-align:left;">
554:650
</td>
<td style="text-align:left;">
1.21
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
1.28
</td>
</tr>
</tbody>
</table>
</td>
<td>
<table>
<thead>
<tr>
<th style="text-align:left;">
Absorbance ratio
</th>
<th style="text-align:left;">
native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> oxidised
</th>
<th style="text-align:left;">
reconstituted expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span> reduced
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
407:525
</td>
<td style="text-align:left;">
6.84
</td>
<td style="text-align:left;">
2.59
</td>
</tr>
<tr>
<td style="text-align:left;">
406:645
</td>
<td style="text-align:left;">
14.44
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
526:645
</td>
<td style="text-align:left;">
15.31
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table>
</td>
</tr>
</tbody>
</table>
<p>The figures indicate that the ratio of the Soret peak (the absorbance of which is contributed to by both <em>c</em> and <em>d</em><span class="math inline">\(_1\)</span> haems) to peaks composed primarily of absorbance from the <em>c</em> haem is lower by a factor of about two in the reconstituted protein compared to native cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The spectroscopic features of the reduced reconstituted enzyme thus allow the crude estimate that the reconstitution was about 50% efficient.</p>
<p>A substantial amount of the maximum activity was, therefore, recovered following reconstitution of the expressed semi-apo enzyme, although it is considerably lower than that reported for reconstitutions of the chemically-prepared semi-apo enzyme from <em>Ps. aeruginosa</em><span class="citation"><sup><a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a></sup></span> and <em>Ps. stutzeri</em> JM300,<span class="citation"><sup><a href="#ref-weeg-aerssens_purification_1991" role="doc-biblioref">251</a></sup></span> in which almost 100% recovery was reported. There are a number of reasons why this might be the case. First, no attempt was made to optimise the procedure. Partial reconstitution of the recombinant semi-apoprotein was achieved quite easily in the present work, but attempts to reconstitute the chemically-prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> in this laboratory have proved to be technically challenging and success requires quite specific salt concentrations, buffer, haem <em>d</em><span class="math inline">\(_1\)</span>:protein ratios and pH (A. Koppenhöfer, personal communication). It is probable that the procedure for reconstituting the recombinant semi-apo enzyme could be improved considerably by varying the conditions of the experiment. Secondly, the partial purity of the enzyme sample from <em>E. coli</em> may mean that some of the haem <em>d</em><span class="math inline">\(_1\)</span> bound non-specifically to other proteins and so was not available for reconstitution.</p>
<p>Bearing the above points in mind, it is concluded that expression in <em>E. coli</em> is a viable system for producing recombinant holo-cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the absence of an alternative system which can express the holoenzyme. The value of using <em>E. coli</em> is in the ability to express enzymically-inactive mutants of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> which could then be reconstituted, as outlined in the introduction to this chapter. There are a few disadvantages to the approach, one of which is that large quantities of native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> must be purified in order to extract haem <em>d</em><span class="math inline">\(_1\)</span> for reconstitution (though in the above experiments, relatively small quantities were used due to the scale of the experiment). A second problem is the possibility of heterogeneity between reconstituted samples which may make it difficult to interpret the results of some experiments, such as enzyme kinetics. This could probably be overcome to some extent by optimisation and standardisation of the procedure, though it is unlikely that reconstituted samples will be suitable for crystallisation unless the reconstitution is very uniform and efficient. The yield of expressed protein in the present work was quite low (around 1 mg l<span class="math inline">\(^{-1}\)</span>) and could probably be improved given time to optimise the conditions. Increased amounts of expressed protein would allow further purification of the sample, to reduce the possibility of non-specific binding of haem <em>d</em><span class="math inline">\(_1\)</span> outlined above. In the only other quantitative study of heterologous <em>nirS</em> expression, around 3 mg l<span class="math inline">\(^{-1}\)</span> of recombinant protein was obtained from cells of <em>Ps. putida</em>.<span class="citation"><sup><a href="#ref-silvestrini_expression_1992" role="doc-biblioref">268</a></sup></span> Improvements in yield using the present construct may be achieved by varying the time of induction with IPTG or possibly varying the IPTG concentration. Investigation of expression levels in other strains of <em>E. coli</em> which overproduce <em>c</em>-type cytochromes, such as <em>E. coli</em> 7120 and its derivatives<span class="citation"><sup><a href="#ref-darwin_regulation_1993" role="doc-biblioref">289</a></sup></span> might also be productive. The present construct, pMMBSE, is somewhat inconvenient; it would be better to try and clone only the <em>nirS</em> structural gene under the <em>tac</em> promoter, without the upstream region. This region contains an inverted repeat adjacent to the NNR box and removal of such structures has been demonstrated to improve in other cases.<span class="citation"><sup><a href="#ref-yamamoto_blue_1987" role="doc-biblioref">292</a></sup></span> Removal of the truncated <em>nirE</em> gene may also be advantageous in view of the possible toxicity effects discussed previously. In addition to these experiments, a different vector system might be used. The pMMB67EH vector is quite large and cumbersome and several newer alternatives are now available, such as the pET vector systems,<span class="citation"><sup><a href="#ref-studier_use_1990" role="doc-biblioref">293</a></sup></span> in which transcription is driven from a T7 RNA polymerase promoter, using modified <em>E. coli</em> strains that have the gene for this polymerase integrated into their genomic DNA. Figures for protein expression in the range of tens of milligrams per litre are frequently quoted for the latter system (though success always depends on the specific nature of the protein under investigation) and it would be of interest to investigate the expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in such a system.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-glockner_copper-containing_1993" class="csl-entry">
(63) Glockner, A. B., Jüngst, A., and Zumft, W. G. (1993) Copper-containing nitrite reductase from <span>Pseudomonas</span> aureofaciens is functional in a mutationally cytochrome cd1-free background (<span>NirS</span>-) of <span>Pseudomonas</span> stutzeri. <em>Archives of Microbiology</em> <em>160</em>, 18–26.
</div>
<div id="ref-hill_reconstitution_1978" class="csl-entry">
(118) Hill, K. E., and Wharton, D. C. (1978) Reconstitution of the apoenzyme of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa with heme d1 and other heme groups. <em>The Journal of Biological Chemistry</em> <em>253</em>, 489–495.
</div>
<div id="ref-ohshima_cloning_1993" class="csl-entry">
(140) Ohshima, T., Sugiyama, M., Uozumi, N., Iijima, S., and Kobayashi, T. (1993) Cloning and sequencing of a gene encoding nitrite reductase from <span>Paracoccus</span> denitrificans and expression of the gene in <span>Escherichia</span> coli. <em>Journal of Fermentation and Bioengineering</em> <em>76</em>, 82–88.
</div>
<div id="ref-de_boer_isolation_1994" class="csl-entry">
(141) Boer, A. P. de, Reijnders, W. N., Kuenen, J. G., Stouthamer, A. H., and Spanning, R. J. van. (1994) Isolation, sequencing and mutational analysis of a gene cluster involved in nitrite reduction in <span>Paracoccus</span> denitrificans. <em>Antonie Van Leeuwenhoek</em> <em>66</em>, 111–127.
</div>
<div id="ref-silvestrini_reaction_1990" class="csl-entry">
(161) Silvestrini, M. C., Tordi, M. G., Musci, G., and Brunori, M. (1990) The reaction of <span>Pseudomonas</span> nitrite reductase and nitrite. <span>A</span> stopped-flow and <span>EPR</span> study. <em>The Journal of Biological Chemistry</em> <em>265</em>, 11783–11787.
</div>
<div id="ref-de_boer_regulation_1996" class="csl-entry">
(168) Boer, T. de. (1996, October) Regulation of denitrification in <span>Paracoccus</span> denitrificans. PhD thesis, VU Amsterdam.
</div>
<div id="ref-de_boer_mutational_1996" class="csl-entry">
(171) Boer, A. P. de, Oost, J. van der, Reijnders, W. N., Westerhoff, H. V., Stouthamer, A. H., and Spanning, R. J. van. (1996) Mutational analysis of the nor gene cluster which encodes nitric-oxide reductase from <span>Paracoccus</span> denitrificans. <em>European Journal of Biochemistry</em> <em>242</em>, 592–600.
</div>
<div id="ref-spiro_fnr_1994" class="csl-entry">
(186) Spiro, S. (1994) The <span>FNR</span> family of transcriptional regulators. <em>Antonie Van Leeuwenhoek</em> <em>66</em>, 23–36.
</div>
<div id="ref-zennaro_nitrite_1993" class="csl-entry">
(215) Zennaro, E., Ciabatti, I., Cutruzzola, F., D’Alessandro, R., and Silvestrini, M. C. (1993) The nitrite reductase gene of <span>Pseudomonas</span> aeruginosa: Effect of growth conditions on the expression and construction of a mutant by gene disruption. <em>FEMS microbiology letters</em> <em>109</em>, 243–250.
</div>
<div id="ref-de_boer_tac_1983" class="csl-entry">
(233) Boer, H. A. de, Comstock, L. J., and Vasser, M. (1983) The tac promoter: A functional hybrid derived from the trp and lac promoters. <em>Proceedings of the National Academy of Sciences of the United States of America</em> <em>80</em>, 21–25.
</div>
<div id="ref-weeg-aerssens_purification_1991" class="csl-entry">
(251) Weeg-Aerssens, E., Wu, W. S., Ye, R. W., Tiedje, J. M., and Chang, C. K. (1991) Purification of cytochrome cd1 nitrite reductase from <span>Pseudomonas</span> stutzeri <span>JM300</span> and reconstitution with native and synthetic heme d1. <em>The Journal of Biological Chemistry</em> <em>266</em>, 7496–7502.
</div>
<div id="ref-silvestrini_expression_1992" class="csl-entry">
(268) Silvestrini, M. C., Cutruzzolà, F., D’Alessandro, R., Brunori, M., Fochesato, N., and Zennaro, E. (1992) Expression of <span>Pseudomonas</span> aeruginosa nitrite reductase in <span>Pseudomonas</span> putida and characterization of the recombinant protein. <em>The Biochemical Journal</em> <em>285 ( Pt 2)</em>, 661–666.
</div>
<div id="ref-iobbi-nivol_reassessment_1994" class="csl-entry">
(269) Iobbi-Nivol, C., Crooke, H., Griffiths, L., Grove, J., Hussain, H., Pommier, J., Mejean, V., and Cole, J. A. (1994) A reassessment of the range of c-type cytochromes synthesized by <span>Escherichia</span> coli <span>K</span>-12. <em>FEMS microbiology letters</em> <em>119</em>, 89–94.
</div>
<div id="ref-hitti_proteinase_1994" class="csl-entry">
(270) Hitti, Y. S., and Bertino, A. M. (1994) Proteinase <span>K</span> and <span>T4</span> <span>DNA</span> polymerase facilitate the blunt-end subcloning of <span>PCR</span> products. <em>BioTechniques</em> <em>16</em>, 802–805.
</div>
<div id="ref-furste_molecular_1986" class="csl-entry">
(271) Fürste, J. P., Pansegrau, W., Frank, R., Blöcker, H., Scholz, P., Bagdasarian, M., and Lanka, E. (1986) Molecular cloning of the plasmid <span>RP4</span> primase region in a multi-host-range <span class="nocase">tacP</span> expression vector. <em>Gene</em> <em>48</em>, 119–131.
</div>
<div id="ref-gronenborn_overproduction_1976" class="csl-entry">
(272) Gronenborn, B. (1976) Overproduction of phage lambda repressor under control of the lac promotor of <span>Escherichia</span> coli. <em>Molecular &amp; general genetics: MGG</em> <em>148</em>, 243–250.
</div>
<div id="ref-derbyshire_mobilization_1987" class="csl-entry">
(273) Derbyshire, K. M., and Willetts, N. S. (1987) Mobilization of the non-conjugative plasmid <span>RSF1010</span>: A genetic analysis of its origin of transfer. <em>Molecular &amp; general genetics: MGG</em> <em>206</em>, 154–160.
</div>
<div id="ref-darwin_nitrate_1995" class="csl-entry">
(274) Darwin, A. J., and Stewart, V. (1995) Nitrate and nitrite regulation of the <span>Fnr</span>-dependent aeg-46.5 promoter of <span>Escherichia</span> coli <span>K</span>-12 is mediated by competition between homologous response regulators (<span>NarL</span> and <span>NarP</span>) for a common <span>DNA</span>-binding site. <em>Journal of Molecular Biology</em> <em>251</em>, 15–29.
</div>
<div id="ref-thony-meyer_escherichia_1995" class="csl-entry">
(275) Thöny-Meyer, L., Fischer, F., Künzler, P., Ritz, D., and Hennecke, H. (1995) Escherichia coli genes required for cytochrome c maturation. <em>Journal of Bacteriology</em> <em>177</em>, 4321–4326.
</div>
<div id="ref-darie_effect_1994" class="csl-entry">
(276) Darie, S., and Gunsalus, R. P. (1994) Effect of heme and oxygen availability on <span class="nocase">hemA</span> gene expression in <span>Escherichia</span> coli: Role of the fnr, <span class="nocase">arcA</span>, and <span class="nocase">himA</span> gene products. <em>Journal of Bacteriology</em> <em>176</em>, 5270–5276.
</div>
<div id="ref-woodard_regulation_1995" class="csl-entry">
(277) Woodard, S. I., and Dailey, H. A. (1995) Regulation of heme biosynthesis in <span>Escherichia</span> coli. <em>Archives of Biochemistry and Biophysics</em> <em>316</em>, 110–115.
</div>
<div id="ref-page_apo_1990" class="csl-entry">
(278) Page, M. D., and Ferguson, S. J. (1990) Apo forms of cytochrome c550 and cytochrome cd1 are translocated to the periplasm of <span>Paracoccus</span> denitrificans in the absence of haem incorporation caused either mutation or inhibition of haem synthesis. <em>Molecular Microbiology</em> <em>4</em>, 1181–1192.
</div>
<div id="ref-grisshammer_expression_1991" class="csl-entry">
(279) Grisshammer, R., Oeckl, C., and Michel, H. (1991) Expression in <span>Escherichia</span> coli of c-type cytochrome genes from <span>Rhodopseudomonas</span> viridis. <em>Biochimica Et Biophysica Acta</em> <em>1088</em>, 183–190.
</div>
<div id="ref-sambongi_alteration_1996" class="csl-entry">
(280) Sambongi, Y., Stoll, R., and Ferguson, S. J. (1996) Alteration of haem-attachment and signal-cleavage sites for <span>Paracoccus</span> denitrificans cytochrome <span>C550</span> probes pathway of c-type cytochrome biogenesis in <span>Escherichia</span> coli. <em>Molecular Microbiology</em> <em>19</em>, 1193–1204.
</div>
<div id="ref-pollock_aerobic_1994" class="csl-entry">
(281) Pollock, W. B., and Voordouw, G. (1994) Aerobic expression of the cyf gene encoding cytochrome c-553 from <span>Desulfovibrio</span> vulgaris <span>Hildenborough</span> in <span>Escherichia</span> coli. <em>Microbiology (Reading, England)</em> <em>140 ( Pt 4)</em>, 879–887.
</div>
<div id="ref-fu_dcra_1994" class="csl-entry">
(282) Fu, R., Wall, J. D., and Voordouw, G. (1994) <span>DcrA</span>, a c-type heme-containing methyl-accepting protein from <span>Desulfovibrio</span> vulgaris <span>Hildenborough</span>, senses the oxygen concentration or redox potential of the environment. <em>Journal of Bacteriology</em> <em>176</em>, 344–350.
</div>
<div id="ref-ubbink_cytochrome_1992" class="csl-entry">
(283) Ubbink, M., Van Beeumen, J., and Canters, G. W. (1992) Cytochrome c550 from <span>Thiobacillus</span> versutus: Cloning, expression in <span>Escherichia</span> coli, and purification of the heterologous holoprotein. <em>Journal of Bacteriology</em> <em>174</em>, 3707–3714.
</div>
<div id="ref-sambongi_synthesis_1994" class="csl-entry">
(284) Sambongi, Y., and Ferguson, S. J. (1994) Synthesis of holo <span>Paracoccus</span> denitrificans cytochrome c550 requires targeting to the periplasm whereas that of holo <span>Hydrogenobacter</span> thermophilus cytochrome c552 does not. <span>Implications</span> for c-type cytochrome biogenesis. <em>FEBS letters</em> <em>340</em>, 65–70.
</div>
<div id="ref-garrett_molecular_1994" class="csl-entry">
(285) Garrett, R. M., and Rajagopalan, K. V. (1994) Molecular cloning of rat liver sulfite oxidase. <span>Expression</span> of a eukaryotic <span>Mo</span>-pterin-containing enzyme in <span>Escherichia</span> coli. <em>The Journal of Biological Chemistry</em> <em>269</em>, 272–276.
</div>
<div id="ref-vidakovic_expression_1996" class="csl-entry">
(286) Vidakovic, M. S., Fraczkiewicz, G., and Germanas, J. P. (1996) Expression and spectroscopic characterization of the hydrogenosomal [<span>2Fe</span>-<span>2S</span>] ferredoxin from the protozoan <span>Trichomonas</span> vaginalis. <em>The Journal of Biological Chemistry</em> <em>271</em>, 14734–14739.
</div>
<div id="ref-leung_pseudoazurin_1997" class="csl-entry">
(287) Leung, Y. C., Chan, C., Reader, J. S., Willis, A. C., Spanning, R. J. van, Ferguson, S. J., and Radford, S. E. (1997) The pseudoazurin gene from <span>Thiosphaera</span> pantotropha: Analysis of upstream putative regulatory sequences and overexpression in <span>Escherichia</span> coli. <em>The Biochemical Journal</em> <em>321 ( Pt 3)</em>, 699–705.
</div>
<div id="ref-viebrock_molecular_1988" class="csl-entry">
(288) Viebrock, A., and Zumft, W. G. (1988) Molecular cloning, heterologous expression, and primary structure of the structural gene for the copper enzyme nitrous oxide reductase from denitrifying <span>Pseudomonas</span> stutzeri. <em>Journal of Bacteriology</em> <em>170</em>, 4658–4668.
</div>
<div id="ref-darwin_regulation_1993" class="csl-entry">
(289) Darwin, A., Hussain, H., Griffiths, L., Grove, J., Sambongi, Y., Busby, S., and Cole, J. (1993) Regulation and sequence of the structural gene for cytochrome c552 from <span>Escherichia</span> coli: Not a hexahaem but a 50 <span class="nocase">kDa</span> tetrahaem nitrite reductase. <em>Molecular Microbiology</em> <em>9</em>, 1255–1265.
</div>
<div id="ref-barber_spectral_1976" class="csl-entry">
(290) Barber, D., Parr, S. R., and Greenwood, C. (1976) Some spectral and steady-state kinetic properties of <span>Pseudomonas</span> cytochrome oxidase. <em>The Biochemical Journal</em> <em>157</em>, 431–438.
</div>
<div id="ref-parr_reaction_1974" class="csl-entry">
(291) Parr, S. R., Wilson, M. T., and Greenwood, C. (1974) The reaction of <span>Pseudomonas</span> aeurginosa cytochrome c oxidase with sodium metabisulphite. <em>The Biochemical Journal</em> <em>139</em>, 273–276.
</div>
<div id="ref-yamamoto_blue_1987" class="csl-entry">
(292) Yamamoto, K., Uozumi, T., and Beppu, T. (1987) The blue copper protein gene of <span>Alcaligenes</span> faecalis <span>S</span>-6 directs secretion of blue copper protein from <span>Escherichia</span> coli cells. <em>Journal of Bacteriology</em> <em>169</em>, 5648–5652.
</div>
<div id="ref-studier_use_1990" class="csl-entry">
(293) Studier, F. W., Rosenberg, A. H., Dunn, J. J., and Dubendorff, J. W. (1990) Use of <span>T7</span> <span>RNA</span> polymerase to direct expression of cloned genes. <em>Methods in Enzymology</em> <em>185</em>, 60–89.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-chapter03.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-chapter05.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
