<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Heterologous expression of the nirS gene from T. pantotropha in the denitrifying host Ps. aeruginosa | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Heterologous expression of the nirS gene from T. pantotropha in the denitrifying host Ps. aeruginosa | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Heterologous expression of the nirS gene from T. pantotropha in the denitrifying host Ps. aeruginosa | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St.Â Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="4-chapter04.html"/>
<link rel="next" href="6-chapter06.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter05" class="section level1" number="5">
<h1><span class="header-section-number">Chapter 5</span> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></h1>
<div id="introduction-2" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Introduction</h2>
<p>In the previous chapter it was demonstrated that the <em>nirS</em> gene from <em>T. pantotropha</em> can be
expressed in <em>E. coli</em> to give a semi-apo protein containing only <em>c</em>-type haem, and that this
protein can be reconstituted with purified <em>d</em><span class="math inline">\(_1\)</span> haem to restore some nitrite reductase activity.
There are a number of disadvantages to this approach, in particular the requirement for large
quantities of purified <em>d</em><span class="math inline">\(_1\)</span> haem which can only be obtained from several tens of milligrams of
purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Additionally, the expression of the protein and the reconstitution
protocol require substantial optimisation; heterogeneity in the reconstituted material may
thus be a problem during crystallisation or attempts to compare thoroughly different samples.</p>
<p>An ideal expression system would, therefore, synthesise holo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> that
contained both the <em>c</em>- and <em>d</em><span class="math inline">\(_1\)</span> haems, in a form that could be easily purified, assayed for
activity and crystallised for structural studies so as to compare site-directed mutants of the
enzyme with the native protein. One approach to this may be to express the cloned <em>nirS</em> gene
in a heterologous host which also contains an endogenous cytochrome <em>cd</em><span class="math inline">\(_1\)</span>-type nitrite
reductase. This requires a denitrifying organism that is genetically amenable. It would be
hoped that the cloned gene would be expressed at a sufficiently high level and that the
recombinant protein would be translocated to the periplasm, where <em>c</em>-haem could be
covalently attached. Additionally, a sufficient pool of free <em>d</em><span class="math inline">\(_1\)</span> haem would exist that could be
sequestrated and incorporated into the recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. Finally the process
should not compromise the growth of the host organism and the recombinant protein should
be physically different to the endogenous nitrite reductase, so that it can be separated and
purified for further analysis.</p>
<p>The experiments described in this chapter were designed to investigate the expression
of the <em>nirS</em> gene from <em>T. pantotropha</em> in the host <em>Ps. aeruginosa</em> PAO1. The latter organism
was selected using the criteria outlined above; it contains a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase
<span class="citation">[<a href="#ref-horio_purification_1961" role="doc-biblioref">294</a>]</span> and systems for genetic manipulation have been developed in the
organism <span class="citation">[<a href="#ref-rothmel_genetic_1991" role="doc-biblioref">295</a>]</span>. Additionally, the isoelectric point of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span>
from this organism (approximately 7.0 <span class="citation">[<a href="#ref-hull_isoelectrophoretic_1993" role="doc-biblioref">296</a>]</span>) is much more basic than
that of the enzyme from <em>T. pantotropha</em> (predicted from the protein sequence to be 4.75), and
so the expressed protein should be separable from the host cytochrome <em>cd</em><span class="math inline">\(_1\)</span> using ion
exchange chromatography. The transformation of <em>Ps. aeruginosa</em> PAO1 with the expression
plasmid pMMBSE is described and the remainder of the chapter focuses on the expression of
the cloned <em>nirS</em> gene under different growth conditions, attempts to partially purify the
expressed protein and initial analyses of the spectroscopic features and activity of the
enzyme obtained.</p>
</div>
<div id="results-2" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> Results</h2>
<div id="transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse" class="section level3" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</h3>
<p>The plasmid used to investigate expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in
<em>Pseudomonas aeruginosa</em> PAO1 was pMMBSE, construction and details of which were
described in Chapter 4. The plasmid was transferred to <em>Ps. aeruginosa</em> by mobilisation from
the donor <em>E. coli</em> strain S17-1. Mated cells were plated on <em>Pseudomonas</em> selective medium
(Materials and Methods), containing 250 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> carbenicillin, to obtain colonies of <em>Ps. aeruginosa</em> containing the plasmid. Single colonies so selected were picked, grown up in LB medium plus 250 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> carbenicillin and tested for the presence of plasmid pMMBSE
using plasmid purification (Promega Wizard miniprep) followed by restriction analysis with
the enzyme <em>HindIII</em>. Figure 5.1 shows agarose gels of the digests from several colonies of <em>Ps. aeruginosa</em>.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-1"></span>
<img src="figure/fig5-1.png" alt="Re-isolation of intact plasmid pMMBSE from _Ps. aeruginosa_ PAO1. Plasmid pMMBSE was introduced into _Ps. aeruginosa_ PAO1 by conjugal transfer from _E. coli_ S17-1. Following selection, the plasmid was isolated using the Promega Wizard miniprep kit and digested with _HindIII_. The fragments were separated by electrophoresis on a 0.8% agarose gel. This digest produces two fragments of approximately 9.3 and 1.9 kbp, seen clearly in lanes 1-3, 5 and 6. Lanes III and II contain DNA size standards, the sizes of some of which are indicated at the left of the gel." width="75%" />
<p class="caption">
Figure 5.1: Re-isolation of intact plasmid pMMBSE from <em>Ps. aeruginosa</em> PAO1. Plasmid pMMBSE was introduced into <em>Ps. aeruginosa</em> PAO1 by conjugal transfer from <em>E. coli</em> S17-1. Following selection, the plasmid was isolated using the Promega Wizard miniprep kit and digested with <em>HindIII</em>. The fragments were separated by electrophoresis on a 0.8% agarose gel. This digest produces two fragments of approximately 9.3 and 1.9 kbp, seen clearly in lanes 1-3, 5 and 6. Lanes III and II contain DNA size standards, the sizes of some of which are indicated at the left of the gel.
</p>
</div>
<p>Wizard minipreps from <em>Ps. aeruginosa</em> PAO1 were of consistently good quality and always yielded the expected fragments from the pMMBSE digest.</p>
</div>
<div id="analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions" class="section level3" number="5.2.2">
<h3><span class="header-section-number">5.2.2</span> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</h3>
<p><em>Ps. aeruginosa</em> was found to be a convenient organism for the study of expression
under different growth conditions, as it grew vigorously both aerobically in LB medium and
anaerobically in LB plus 50 mM potassium nitrate. To analyse the growth of transformed
cells, 50 ml cultures of <em>Ps. aeruginosa</em> PAO1 and <em>Ps. aeruginosa</em> PAO1 [pMMBSE] were
grown either aerobically, aerobically plus 50 mM KNO<span class="math inline">\(_3\)</span> or anaerobically. Additionally,
cultures carrying the pMMBSE plasmid under each condition could be induced using 1 mM
IPTG during the early exponential growth phase <span class="citation">[<a href="#ref-rothmel_genetic_1991" role="doc-biblioref">295</a>]</span>. This set of
conditions gave rise to nine different cultures, detailed in Table 5.1.</p>
<table>
<caption>
<span id="tab:table51">Table 5.1: </span>Different growth conditions used to compare the growth of <em>Ps. aeruginosa</em> PAO1 with that of <em>Ps. aeruginosa</em> [pMMBSE]
Cultures contained <em>Ps. aeruginosa</em> PAO1, <em>Ps. aeruginosa</em> [pMMBSE] or <em>Ps. aeruginosa</em> [pMMBSE] induced using 1 mM IPTG. Each of these were grown aerobically in LB medium, aerobically in LB medium plus 50 mM KNO<span class="math inline">\(_3\)</span> or anaerobically in LB medium plus 50 mM KNO<span class="math inline">\(_3\)</span>, giving a total of nine different cultures as shown.
</caption>
<thead>
<tr>
<th style="text-align:right;">
Culture number
</th>
<th style="text-align:left;">
Growth conditions
</th>
<th style="text-align:left;">
pMMBSE present
</th>
<th style="text-align:left;">
IPTG added
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:left;">
aerobic
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
â
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
aerobic + 50 mM KNO<span class="math inline">\(_3\)</span>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
â
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:left;">
anaerobic
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
â
</td>
<td style="text-align:left;">
â
</td>
</tr>
</tbody>
</table>
<p>Growth was recorded by the increase in absorbance at 650 nm until stationary phase was reached and the resulting curves are shown in Figure 5.2.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-2"></span>
<img src="figure/fig5-2.png" alt="Comparison of the growth curves of _Ps. aeruginosa_ and _Ps. aeruginosa_ [pMMBSE] under different growth conditions. Overnight cultures were inoculated at a dilution of 1/100 into 50 ml of medium and growth was followed by the change in absorbance at 650 nm (logarithmic scale). Symbols: (&amp;#9670;) _Ps. aeruginosa_ aerobic, (&amp;#9632;) _Ps. aeruginosa_ [pMMBSE] aerobic, (&amp;#9650;) _Ps. aeruginosa_ [pMMBSE] aerobic + induced, (&amp;#9671;) _Ps. aeruginosa_ aerobic + nitrate, (&amp;#9633;) _Ps. aeruginosa_ [pMMBSE] aerobic + nitrate, (&amp;#9651;) _Ps. aeruginosa_ [pMMBSE] aerobic + nitrate + induced, (diamond/strikethrough) _Ps. aeruginosa_ anaerobic. (square/strikethrough) Ps, aeruginosa [pMMBSE] anaerobic, (triangle/strikethrough) _Ps. aeruginosa_ [pMMBSE] anaerobic + induced." width="75%" />
<p class="caption">
Figure 5.2: Comparison of the growth curves of <em>Ps. aeruginosa</em> and <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions. Overnight cultures were inoculated at a dilution of 1/100 into 50 ml of medium and growth was followed by the change in absorbance at 650 nm (logarithmic scale). Symbols: (â) <em>Ps. aeruginosa</em> aerobic, (â ) <em>Ps. aeruginosa</em> [pMMBSE] aerobic, (â²) <em>Ps. aeruginosa</em> [pMMBSE] aerobic + induced, (â) <em>Ps. aeruginosa</em> aerobic + nitrate, (â¡) <em>Ps. aeruginosa</em> [pMMBSE] aerobic + nitrate, (â³) <em>Ps. aeruginosa</em> [pMMBSE] aerobic + nitrate + induced, (diamond/strikethrough) <em>Ps. aeruginosa</em> anaerobic. (square/strikethrough) Ps, aeruginosa [pMMBSE] anaerobic, (triangle/strikethrough) <em>Ps. aeruginosa</em> [pMMBSE] anaerobic + induced.
</p>
</div>
<p>Aerobically, the growth of all cultures was the same, regardless of the presence of IPTG or nitrate. However,
anaerobically, <em>Ps. aeruginosa</em> [pMMBSE] showed a marked growth defect, entered
stationary phase much earlier than the wild-type and reached a lower final optical density of
around 0.6. This occurred even in the absence of induction with IPTG. To investigate the
defect further, cultures of <em>Ps. aeruginosa</em> and <em>Ps. aeruginosa</em> [pMMBSE] plus or minus
IPTG were again grown anaerobically over a 13 hour period and their growth curves
recorded. To estimate the level of nitrite reductase activity in each culture, 1 ml samples
taken at each time point on the growth curve were harvested and the supernatant was assayed
colorimetrically for the presence of nitrite. These data are shown in Figure 5.3.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-3"></span>
<img src="figure/fig5-3.png" alt="Comparison of (a) the growth curve and (b) nitrite accumulation of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE]. Growth is shown as absorbance at 650 nm (logarithmic scale) against time after inoculation. For nitrite determination, 1 ml samples were harvested at each time point and the supernatant was assayed for nitrite colorimetrically (Materials and Methods). Symbols: (&amp;#9670;) _Ps. aeruginosa_, (&amp;#9632;) _Ps. aeruginosa_ [pMMBSE], (&amp;#9650;) _Ps. aeruginosa_ [pMMBSE] induced using 1 mM IPTG." width="75%" />
<p class="caption">
Figure 5.3: Comparison of (a) the growth curve and (b) nitrite accumulation of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE]. Growth is shown as absorbance at 650 nm (logarithmic scale) against time after inoculation. For nitrite determination, 1 ml samples were harvested at each time point and the supernatant was assayed for nitrite colorimetrically (Materials and Methods). Symbols: (â) <em>Ps. aeruginosa</em>, (â ) <em>Ps. aeruginosa</em> [pMMBSE], (â²) <em>Ps. aeruginosa</em> [pMMBSE] induced using 1 mM IPTG.
</p>
</div>
<p>In all samples, the nitrite concentration rose to a maximum of around 3.4 mM near the end of the
exponential growth phase. In wild-type cells, it then declined very rapidly, to undetectable
levels within two hours. In contrast nitrite concentration remained near the maximum level
reached in <em>Ps. aeruginosa</em> [pMMBSE], regardless of induction, throughout stationary phase.
Samples of these cultures assayed after a further 12 hours contained similar levels of nitrite
(not shown).</p>
<p>The nine different cultures used to obtain the first set of growth curves were
harvested and total soluble extracts were prepared by sonication and ultracentrifugation.
Samples from each extract were separated by SDS-PAGE and stained for protein using
Coomassie Blue (Figure 5.4).</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-4"></span>
<img src="figure/fig5-4.png" alt="Comparison of the protein content of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE] under different growth conditions. 40 $\mu$g of protein from the soluble extracts indicated was separated by electrophoresis on an 8% SDS-PAGE gel and stained for protein using Coomassie Brilliant Blue R250. The lane labelled M contained molecular weight markers, the sizes of which are indicated at the left of the gel. A 63 kDa band, corresponding to expressed _T. pantotropha_ cytochrome _cd_$_1$ is visible in lanes 4 and 7." width="75%" />
<p class="caption">
Figure 5.4: Comparison of the protein content of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions. 40 <span class="math inline">\(\mu\)</span>g of protein from the soluble extracts indicated was separated by electrophoresis on an 8% SDS-PAGE gel and stained for protein using Coomassie Brilliant Blue R250. The lane labelled M contained molecular weight markers, the sizes of which are indicated at the left of the gel. A 63 kDa band, corresponding to expressed <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is visible in lanes 4 and 7.
</p>
</div>
<p>The region in which cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is found stained quite
densely, but a clear band of 63 kDa, (which corresponds to the size of mature cytochrome
<em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>) was visible in the induced sample grown aerobically and
anaerobically with nitrate. The same band was not clearly seen in induced, anaerobically
grown samples due to their poor growth (data not shown). A band running with slightly
lower mobility was observed in the samples from wild-type cells grown aerobically plus
nitrate and anaerobically; this band corresponded to the endogenous cytochrome <em>cd</em><span class="math inline">\(_1\)</span> which
has a slightly lower molecular weight than its counterpart from <em>T. pantotropha</em> (Table 3.4.
Chapter 3). To compare the effect on expression of growth under either aerobic or anaerobic
conditions an identical set of samples, except for <em>Ps. aeruginosa</em> [pMMBSE] grown
aerobically with nitrate, were again separated by SDS-PAGE under the same conditions,
transferred to a nitrocellulose membrane by semi-dry electroblotting and probed with the <em>P. denitrificans</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> antibody. The Western blot is shown in Figure 5.5.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-5"></span>
<img src="figure/fig5-5.png" alt="Western blot of protein samples from _Ps. aeruginosa_ or _Ps. aeruginosa_ [pMMBSE] grown under different growth conditions, probed using antibody to cytochrome _cd_$_1$. 40 $\mu$g of protein from the cultures indicated was loaded in each lane and separated by SDS-PAGE in an 8% acrylamide gel. The samples were then transferred to a nitrocellulose membrane by semi-dry electroblotting and probed with antibody as detailed in Materials and Methods." width="75%" />
<p class="caption">
Figure 5.5: Western blot of protein samples from <em>Ps. aeruginosa</em> or <em>Ps. aeruginosa</em> [pMMBSE] grown under different growth conditions, probed using antibody to cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. 40 <span class="math inline">\(\mu\)</span>g of protein from the cultures indicated was loaded in each lane and separated by SDS-PAGE in an 8% acrylamide gel. The samples were then transferred to a nitrocellulose membrane by semi-dry electroblotting and probed with antibody as detailed in Materials and Methods.
</p>
</div>
<p>Only two samples tested positive with this antibody, those from <em>Ps. aeruginosa</em> [pMMBSE] grown
aerobically or anaerobically and induced with IPTG. Each sample showed a band of the
same mobility as native purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>, with a more intense
staining reaction in the anaerobic sample. The latter sample also showed a smeared high
molecular weight band, near the top of the gel.</p>
<p>Next, the <em>c</em>-type cytochrome content of each sample was analysed by SDS-PAGE
followed by staining for covalently attached haem. In all samples grown aerobically the only
detectable band was that of the low molecular weight cytochrome <em>c</em><span class="math inline">\(_{551}\)</span> (9 kDa); this was seen
only on gels of 10% acrylamide or more (not shown). 100 <span class="math inline">\(\mu\)</span>g of protein from the
anaerobically and aerobically plus nitrate grown samples was separated on 8% acrylamide
gels and stained for haem (Figure 5.6).</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-6"></span>
<img src="figure/fig5-6.png" alt="Comparison of the _c_-type cytochrome content of _Ps. aeruginosa_ with that of _Ps. aeruginosa_ [pMMBSE] under different growth conditions. 100 $\mu$g of protein from the cultures indicated was loaded in each lane and separated by SDS-PAGE in a 6% acrylamide gel. The gel was stained for covalently-attached haem as described in Materials and Methods. Lane Tp contained 5 $\mu$g of purified cytochrome _cd_$_1$ from _T. pantotropha_." width="75%" />
<p class="caption">
Figure 5.6: Comparison of the <em>c</em>-type cytochrome content of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions. 100 <span class="math inline">\(\mu\)</span>g of protein from the cultures indicated was loaded in each lane and separated by SDS-PAGE in a 6% acrylamide gel. The gel was stained for covalently-attached haem as described in Materials and Methods. Lane Tp contained 5 <span class="math inline">\(\mu\)</span>g of purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>.
</p>
</div>
<p>A doublet of around 60 kDa was seen in the lane
containing extract from anaerobically-grown <em>Ps. aeruginosa</em> and in the extract from cells
grown aerobically with nitrate; this doublet corresponded to the endogenous cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.
In samples from cells containing plasmid pMMBSE but not induced with IPTG, the intensity
of this band was greatly attenuated (lane 2). When the cultures were induced (lane 3), the
band was again attenuated and a new doublet of slightly higher molecular weight appeared.
The latter doublet stained most strongly in the aerobically grown cultures plus nitrate (lane
6) and had the same mobility as purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> (lane 7).
These experiments showed that the <em>nirS</em> gene from <em>T. pantotropha</em> expressed to a
high level in <em>Ps. aeruginosa</em> PAO1 when induced with IPTG, particularly under anaerobic
denitrifying growth conditions. The next step was to attempt a partial purification of the
recombinant protein. A 1 l culture of <em>Ps. aeruginosa</em> [pMMBSE] was grown anaerobically
to early exponential phase, induced with 1 mM IPTG and grown for a further 10 hours to a
final A<span class="math inline">\(_{650}\)</span> of about 0.8. The cells were harvested and an attempt to prepare a periplasmic
extract was made by spheroplasting in the normal way, except for the omission of EDTA
which was found to cause cell lysis. The extract obtained was extremely pale brown in
colour, and so it was decided to improve the extraction at the expense of initial purity by
using a total soluble extract, prepared as before by sonication and ultracentrifugation. This
extract was applied to a small DEAE-Sepharose column (1 x 10 cm), previously equilibrated
with 50 mM Tris-HCl pH 8.0, washed with several column volumes of the same buffer and
eluted with a 250 ml linear gradient of 0-400 mM NaCl in the same buffer. However, it was
noticed that after application of the sample to the column the flow rate became very
irregular. The elution profile, obtained by measuring the A<span class="math inline">\(_{410}\)</span> of the fractions was also very
irregular and no peaks corresponding to the elution of haem proteins could be resolved.</p>
<p>The problem of poor column flow may have occurred for a number of reasons. The
initial hypothesis was that the aggregated material observed on the Western blot of
anaerobically grown induced cell extracts may have been causing a blockage in the column.
However, the aggregate was not observed in aerobic samples and as it is known that in <em>Ps. aeruginosa</em> holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> is expressed aerobically if nitrate is present <span class="citation">[<a href="#ref-zennaro_nitrite_1993" role="doc-biblioref">215</a>]</span>, it was thought possible that some recombinant holoprotein might be produced under
the same conditions, but without the risk of aggregation. To achieve this a 1 l culture was
grown in a 5 1 flask with low shaking (100 rpm), induced as before and harvested after 10
hours. Additional steps were taken to reduce the risk of poor performance during the ion
exchange chromatography. First, as lysed cells of <em>Ps. aeruginosa</em> produce substantial
amounts of polysaccharides and DNA, the harvested cells were broken by a freeze-thaw
lysozyme treatment, followed by incubation with 100 U of DNase. This material was
filtered before application to a slightly larger DEAE-Sepharose column (2 x 12 cm) in order
to reduce irregularities in the flow due to surface tension at the column walls. The material
was then eluted using a 250 ml linear gradient of 0-400 mM NaCl as before. This time a
clean, well-resolved elution profile was obtained, as shown in Figure 5.7.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-7"></span>
<img src="figure/fig5-7.png" alt="Elution profile during ion-exchange chromatography of soluble extract from _Ps. aeruginosa_ [pMMBSE] grown semi-aerobically with nitrate and induced using IPTG. A 1 l culture of _Ps. aeruginosa_ [pMMBSE] was grown semi-aerobically with 50 mM KNO$_3$ and induced with 1 mM IPTG. A total soluble extract was prepared and bound to a DEAE-Sepharose column (2 x 12 cm) previously equilibrated with 50 mM Tris-HCI pH 8.0, washed with several column volumes of the same buffer and eluted using a 250 ml linear gradient of 0-400 mM NaCl. Symbols: (&amp;#9632;) absorbance at 410 nm, (&amp;#9670;) NaCl concentration" width="75%" />
<p class="caption">
Figure 5.7: Elution profile during ion-exchange chromatography of soluble extract from <em>Ps. aeruginosa</em> [pMMBSE] grown semi-aerobically with nitrate and induced using IPTG. A 1 l culture of <em>Ps. aeruginosa</em> [pMMBSE] was grown semi-aerobically with 50 mM KNO<span class="math inline">\(_3\)</span> and induced with 1 mM IPTG. A total soluble extract was prepared and bound to a DEAE-Sepharose column (2 x 12 cm) previously equilibrated with 50 mM Tris-HCI pH 8.0, washed with several column volumes of the same buffer and eluted using a 250 ml linear gradient of 0-400 mM NaCl. Symbols: (â ) absorbance at 410 nm, (â) NaCl concentration
</p>
</div>
<p>Three peaks of absorbance at 410 nm were seen, eluting at approximately 125, 225 and 250 mM NaCl.</p>
<p>Fractions from around the top of peaks 2 and 3 were run on an 8% acrylamide SDS
PAGE gel, along with a sample of total soluble extract from <em>Ps. aeruginosa</em> [pMMBSE]
grown anaerobically and induced with 1 mM IPTG, and stained for covalently-bound haem (Figure 5.8).</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-8"></span>
<img src="figure/fig5-8.png" alt="Analysis of the _c_-type cytochrome content in the elution profile of soluble extract from _Ps. aeruginosa_ [pMMBSE] following ion exchange chromatography. 15 $\mu$l of each fraction was loaded in each lane and separated by SDS-PAGE in an 8% acrylamide gel. Lanes are numbered according to the fraction loaded and the peak from which the fractions were taken is indicated. Lane S contained 100 $\mu$g of total soluble extract from _Ps. aeruginosa_ [pMMBSE] grown anaerobically and induced with 1 mM IPTG. In lane S the endogenous (60 kDa) and recombinant (63 kDa) cytochrome _cd_$_1$ are indicated." width="75%" />
<p class="caption">
Figure 5.8: Analysis of the <em>c</em>-type cytochrome content in the elution profile of soluble extract from <em>Ps. aeruginosa</em> [pMMBSE] following ion exchange chromatography. 15 <span class="math inline">\(\mu\)</span>l of each fraction was loaded in each lane and separated by SDS-PAGE in an 8% acrylamide gel. Lanes are numbered according to the fraction loaded and the peak from which the fractions were taken is indicated. Lane S contained 100 <span class="math inline">\(\mu\)</span>g of total soluble extract from <em>Ps. aeruginosa</em> [pMMBSE] grown anaerobically and induced with 1 mM IPTG. In lane S the endogenous (60 kDa) and recombinant (63 kDa) cytochrome <em>cd</em><span class="math inline">\(_1\)</span> are indicated.
</p>
</div>
<p>Fractions from both peaks showed a single haem-staining doublet with
identical mobility. The doublet was also seen in the total soluble extract and appeared
slightly higher in the gel than a second band which corresponded to the endogenous
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> at about 60 kDa. The doublet in the samples from both peak 2 and peak 3
therefore corresponded to expressed <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.</p>
<p>The eluted fractions from the DEAE-Sepharose column were very dilute and so the
top fractions from peaks 2 and 3 were pooled and each pool was concentrated to about 1 ml
by centrifugation in an Amicon-30 microconcentrator. Following concentration, the material
from peak 2 was pale pink, whereas that from peak 3 was orange-brown. The spectra of the
concentrated fractions were recorded. Figure 5.9a and 5.9b shows a spectrum of the material
from peak 2; absorbance maxima were observed at 289, 415, 522, 548 and 554 nm.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-9"></span>
<img src="figure/fig5-9.png" alt="UV-visible spectrum of concentrated fractions eluting under peak 2 following DEAE-Sepharose chromatography. Five fractions from the top of peak 2 (Figure 5.7) were pooled and concentrated to approximately 1 ml by centrifugation in an Amicon-30 microconcentrator Absorbance maxima were observed at 289 (not shown), 415, 522 548 and 554 nm. (b) is an expansion of the region from 440-800 nm in (a), showing the split $\alpha$-peak and the absence of _d_$_1$ haem absorbance features. No change in the spectrum was observed on the addition of sodium dithionite (not shown)." width="75%" />
<p class="caption">
Figure 5.9: UV-visible spectrum of concentrated fractions eluting under peak 2 following DEAE-Sepharose chromatography. Five fractions from the top of peak 2 (Figure 5.7) were pooled and concentrated to approximately 1 ml by centrifugation in an Amicon-30 microconcentrator Absorbance maxima were observed at 289 (not shown), 415, 522 548 and 554 nm. (b) is an expansion of the region from 440-800 nm in (a), showing the split <span class="math inline">\(\alpha\)</span>-peak and the absence of <em>d</em><span class="math inline">\(_1\)</span> haem absorbance features. No change in the spectrum was observed on the addition of sodium dithionite (not shown).
</p>
</div>
<p>The spectrum was virtually identical to that of the <em>T. pantotropha</em> semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>
expressed in <em>E. coli</em> (Chapter 4, Figure 4.13). No change in this spectrum was observed on
the addition of sodium dithionite. Figure 5.10a shows the spectrum from peak 3, with
288 409 and 523 nm, and smaller peaks at 547 and 628 nm.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-10"></span>
<img src="figure/fig5-10.png" alt="(a) UV-visible spectrum of concentrated fractions eluting under peak 3 after DEAE-Sepharose chromatography, as isolated and (b) after the addition of excess sodium dithionite. Five fractions from the top of peak 3 (Figure 5.7) were pooled and concentrated to approximately 1 ml by centrifugation in an Amicon-10 microconcentrator. Absorbance maxima were observed at 286, 409, 523 and 547 nm in the as-isolated form, and 418, 522, 548, 554 and 607 nm in the dithionite-reduced form." width="75%" />
<p class="caption">
Figure 5.10: (a) UV-visible spectrum of concentrated fractions eluting under peak 3 after DEAE-Sepharose chromatography, as isolated and (b) after the addition of excess sodium dithionite. Five fractions from the top of peak 3 (Figure 5.7) were pooled and concentrated to approximately 1 ml by centrifugation in an Amicon-10 microconcentrator. Absorbance maxima were observed at 286, 409, 523 and 547 nm in the as-isolated form, and 418, 522, 548, 554 and 607 nm in the dithionite-reduced form.
</p>
</div>
<p>Sodium dithionite was gradually added to a 200 <span class="math inline">\(\mu\)</span>l sample of the concentrated peak 3 fractions and
the spectrum was recorded until no further changes were observed. Figure 5.10b shows the
final reduced spectrum, with absorbance peaks at 418, 522, 548, 554 and 607 nm, and a
shoulder at 460 nm. The spectrum was similar to that of holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> but not identical; this point is discussed further in the following section.
Changes in the spectrum between the as-isolated and dithionite reduced forms are illustrated by a difference spectrum, shown in Figure 5.11a.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-11"></span>
<img src="figure/fig5-11.png" alt="Difference spectra of (a) dithionite-reduced minus as-isolated spectra from peak 3 and (b) dithionite-reduced peak 3 minus peak 2. The absolute difference spectra shown in figures 5.9 and 5.10 were subtracted from one another to highlight differences between the reduced and oxidised holo cytochrome (a) and the holo- and semi-apo cytochromes (b). In both cases, difference absorbance maxima appear at 423, 522, 548 and 610 nm. In addition, spectrum (a) shows a peak at 460 nm, which is a pronounced shoulder in spectrum (b)." width="75%" />
<p class="caption">
Figure 5.11: Difference spectra of (a) dithionite-reduced minus as-isolated spectra from peak 3 and (b) dithionite-reduced peak 3 minus peak 2. The absolute difference spectra shown in figures 5.9 and 5.10 were subtracted from one another to highlight differences between the reduced and oxidised holo cytochrome (a) and the holo- and semi-apo cytochromes (b). In both cases, difference absorbance maxima appear at 423, 522, 548 and 610 nm. In addition, spectrum (a) shows a peak at 460 nm, which is a pronounced shoulder in spectrum (b).
</p>
</div>
<p>Major changes in absorbance were noted at 423, 460, 522, 554 and 608 nm. A second difference spectrum of dithionite-reduced peak
3 minus peak 2 is shown in Figure 5.11b. Similar changes were noted, at 423, 522, 548 and 610 nm.</p>
<p>The concentrated material from peaks 2 and 3 was assayed for oxidase activity using
horse heart cytochrome <em>c</em> as an electron donor, and for nitrite reductase activity using
reduced methyl viologen. Assay traces from these experiments are shown in Figure 5.12.</p>
<div class="figure" style="text-align: center"><span id="fig:fig5-12"></span>
<img src="figure/fig5-12.png" alt="(a) Cytochrome _c_ oxidase and (b) nitrite reductase activities of the concentrated fractions from peak 3 following ion exchange chromatography. Samples from peak 3 were assayed for oxidase activity using reduced horse heart cytochrome _c_ as electron donor, and for nitrite reductase activity using reduced methyl viologen as electron donor, as described in Materials and Methods. Beneath each trace are estimates of the activity expressed as nmol of electron donor oxidised per minute per mg of protein, which are the means of three separate assays." width="75%" />
<p class="caption">
Figure 5.12: (a) Cytochrome <em>c</em> oxidase and (b) nitrite reductase activities of the concentrated fractions from peak 3 following ion exchange chromatography. Samples from peak 3 were assayed for oxidase activity using reduced horse heart cytochrome <em>c</em> as electron donor, and for nitrite reductase activity using reduced methyl viologen as electron donor, as described in Materials and Methods. Beneath each trace are estimates of the activity expressed as nmol of electron donor oxidised per minute per mg of protein, which are the means of three separate assays.
</p>
</div>
<p>Peak 2 had no activity as either an oxidase or a nitrite reductase. Peak 3 showed a very low
oxidase activity of 4.32 +/- 0.6 nmol cytochrome <em>c</em> oxidised per minute per mg of protein.
The nitrite reductase activity was higher but was more variable; approximately 290 +/- 90
nmol of methyl viologen was oxidised per minute per mg of protein. Attempts to relate these
figures to the turnover number of purified cytochrome <em>cd</em><span class="math inline">\(_1\)</span> are discussed in the following
section.</p>
</div>
</div>
<div id="discussion-2" class="section level2" number="5.3">
<h2><span class="header-section-number">5.3</span> Discussion</h2>
<p><em>Ps aeruginosa</em> PAO1 was chosen as a suitable candidate for expression studies based
on the criteria outlined in Section 5.1, one of which was its genetic amenability. It was one
of the organisms tested in the original construction of plasmid pMMB67EH <span class="citation">[<a href="#ref-furste_molecular_1986" role="doc-biblioref">271</a>]</span>
and the early experiments in this chapter showed that transformation and plasmid
purification from the strain are quite straightforward (Figure 5.1). Having successfully
introduced the plasmid into <em>Ps. aeruginosa</em> PAO1, the next objective was to investigate the
growth of the transformed cells.</p>
<p>Aerobically, the growth of all cultures was similar, regardless of the presence of
nitrate or induction by IPTG (Figure 5.2). This was similar to the result obtained using <em>E. coli</em> JM83 in Chapter 4. However, anaerobically there was a pronounced growth defect in
cells carrying pMMBSE regardless of whether IPTG was present. These cultures reached a
maximum absorbance at 650 nm of only around 0.6-0.8 compared to around 1.6 in the wild
type cultures and entered stationary phase earlier. The cell pellets from anaerobic cultures
containing pMMBSE were also much paler when harvested. In <em>E. coli</em> there was some
suggestion of an anaerobic growth defect but it was not clear how marked this was, because
the combined stress of countering the antibiotic and growing in the minimal medium affected
growth regardless of the presence of the <em>nirS</em> gene on the plasmid. The growth of cultures of
<em>Ps. aeruginosa</em> containing only pMMB67EH was not tested but even so, the anaerobic
growth defect in cultures containing pMMBSE is clearly much more pronounced.</p>
<p>The concentration of nitrite in the culture medium under anaerobic condition was
measured, to assess whether a build up of toxic nitrite was responsible for the poor growth of
cells containing plasmid pMMBSE. Figure 5.3 shows that in both induced and non-induced
cultures of <em>Ps. aeruginosa</em> [pMMBSE], nitrite accumulates to a similar maximum level as
seen in the wild-type, but then remains at this concentration, rather than rapidly disappearing
from the medium. This indicates that for some reason, nitrite reduction is compromised in
cells containing pMMBSE. The haem-stained gel of extracts from each culture under
different growth conditions (Figure 5.6) shows that levels of the endogenous cytochrome <em>cd</em><span class="math inline">\(_1\)</span>
are substantially attenuated in cells containing pMMBSE. It appears that for some reason the
presence of the pMMBSE plasmid under anaerobic conditions leads to attenuated expression
of the endogenous cytochrome <em>cd</em><span class="math inline">\(1\)</span>. This in turn prevents the removal of nitrite produced
from the nitrate reductase activity of the cells and eventually leads to the premature cessation
of growth. The observation also suggests that any activity of the expressed <em>T. pantotropha</em>
<em>nirS</em> gene product is not sufficient to lower the nitrite concentration.</p>
<p>As this effect occurs only in cells containing pMMBSE under anaerobic conditions
regardless of induction, it is inferred that the effect is linked to a factor on the plasmid which
is only active anaerobically. One possible candidate might be the NNR binding site which is
present on pMMBSE. <em>Ps. aeruginosa</em> contains at least two proteins of the FNR family,
named ANR <span class="citation">[<a href="#ref-sawers_identification_1991" role="doc-biblioref">297</a>]</span> and DNR <span class="citation">[<a href="#ref-arai_expression_1995" role="doc-biblioref">197</a>]</span>. The second of these is in the same
phylogenetic group as the NNR protein <span class="citation">[<a href="#ref-de_boer_regulation_1996" role="doc-biblioref">168</a>]</span> and could conceivably interact with
the NNR binding site, to which it would not normally be exposed. This might reduce levels
of the protein and lead to the down-regulation of the endogenous nitrite reductase. However,
there are arguments against this interpretation. First, the plasmid pMMBSE is quite low in
copy number; the exact number of copies per cell is not known, but is likely to be in the
order of 12 copies <span class="citation">[<a href="#ref-frey_replication_1992" role="doc-biblioref">298</a>]</span>, as the plasmid is based on the RSF1010 replicon. If
the DNR (or ANR) molecule is expressed in a similar way to the FNR protein in <em>E. coli</em>, it
could be present at around 2 400 copies per cell <span class="citation">[<a href="#ref-unden_role_1987" role="doc-biblioref">299</a>]</span> and so would
be unlikely to be titrated out by the plasmid. A fall in free concentration of DNR/ANR
under anaerobic conditions might also be expected to be compensated for by autoregulation,
which is seen in the case of the FNR gene <span class="citation">[<a href="#ref-spiro_regulation_1987" role="doc-biblioref">181</a>]</span>. However these
arguments are based on analogy to FNR and as we know little of how DNR/ANR works
there may still be a case for interaction between this molecule and the NNR binding site on
pMMBSE, which is not understood at present.</p>
<p>The only other relevant portion of the pMMBSE plasmid is the truncated <em>nirE</em> gene,
the first 648 bp of which are present in this construct. This encodes 216 out of a total 287
amino acids and it may be that the truncated protein is expressed under anaerobic growth, so
interfering with the normal regulation of denitrification in <em>Ps. aeruginosa</em> and leading to the
effects detailed above. Possible effects of the plasmid-borne NNR binding site and the <em>nirE</em>
gene could be clarified by control experiments, examining the growth of cultures containing
plasmid pMMB67EH, plasmids that carried only the NNR binding site or only the <em>nirS</em>
structural gene. The nature of the anaerobic growth defect is, therefore, quite complex and
not understood at present, though its main practical consequence is a reduced cell yield, to be
taken into account when considering the amount of recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> that might
be obtained.</p>
<p>Expression of the <em>nirS</em> gene from pMMBSE in <em>Ps. aeruginosa</em> is similar to that seen
in <em>E. coli</em> JM83, in that expression only occurs following induction with IPTG, when cells
are grown either aerobically or anaerobically. This indicates that the <em>T. pantotropha</em> <em>nirS</em>
promoter is non-functional in <em>Ps. aeruginosa</em>. Both <em>T. pantotropha</em> <span class="citation">[<a href="#ref-steinrucke_genetics_1993" role="doc-biblioref">257</a>]</span> and <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-west_codon_1988" role="doc-biblioref">300</a>]</span> have a higher genomic G+C content
than <em>E. coli</em>, but apparently there is no precedent for a GC-based promoter structure based
solely upon a higher G+C genomic content. These results also demonstrate that the <em>tac</em>
promoter and the <em>lacI</em><span class="math inline">\(^q\)</span> gene are functional in <em>Ps. aeruginosa</em>, a result consistent with
previous studies <span class="citation">[<a href="#ref-fujita_vitro_1992" role="doc-biblioref">301</a>, <a href="#ref-brunschwig_two-component_1992" role="doc-biblioref">302</a>]</span>. The expressed
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is the same size as the native <em>T. pantotropha</em> protein and has <em>c</em>-type haem
attached, indicating that as in <em>E. coli</em>, translocation to the periplasm and subsequent
processing have occurred normally. When expressed anaerobically a smeared band of high
molecular weight was observed by Western blotting, in addition to the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> band
(Figure 5.5). This appears to be aggregated cytochrome <em>cd</em><span class="math inline">\(_1\)</span> rather than a different cross
reacting protein; the anti-cytochrome <em>cd</em><span class="math inline">\(_1\)</span> antibody is very specific, as shown by its lack of
reactivity towards the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> of <em>Ps. aeruginosa</em> (and also <em>Ps. stutzeri</em>, not shown).
This observation is in agreement with previous immunological studies, in which an antibody
to cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. stutzeri</em> ATCC 14405 was found to be almost strain specific,
reacting only weakly with the protein from only a very few other strains of <em>Ps. stutzeri</em>
<span class="citation">[<a href="#ref-ward_antibody_1993" role="doc-biblioref">303</a>]</span>. The aggregated material may originate <em>in vivo</em> or during sample boiling
prior to SDS-PAGE, but the former explanation appears more likely as this phenomenon was
specific to extracts from anaerobically-grown induced cultures. The protein seems to be
expressed at a higher level in <em>Ps. aeruginosa</em> compared to <em>E. coli</em> under these conditions and
this may contribute to the aggregation problem; aggregation of over-expressed proteins,
including those found in the periplasm has been described previously <em>e.g.</em> for T-cell receptor
protein <span class="citation">[<a href="#ref-wulfing_correctly_1994" role="doc-biblioref">304</a>]</span>, insulin-like growth factor-1 <span class="citation">[<a href="#ref-wong_expression_1988" role="doc-biblioref">305</a>]</span> and
anti-influenza neuraminidase antibody NC41 <span class="citation">[<a href="#ref-kortt_solution_1995" role="doc-biblioref">306</a>]</span>.</p>
<p>Attempts to partially purify recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Ps. aeruginosa</em> PAO1
proved to be quite challenging. To begin with the cells cannot be spheroplasted in the
normal way to release the periplasm, as they are prone to lysis in the presence of EDTA.
Other attempts at spheroplasting that omitted EDTA or used a washing step with 0.2 M
MgCl<span class="math inline">\(_2\)</span> <span class="citation">[<a href="#ref-cheng_alkaline_1970" role="doc-biblioref">307</a>]</span> yielded an extract with an extremely pale colour and so it was
decided to begin the purification with a total soluble extract from sonicated cells. However,
extract from anaerobically grown cells could not be eluted cleanly from a small DEAE
Sepharose ion exchange column. The flow rate under these circumstances was very uneven
and a resolved elution profile was not obtainable. This occurred several times, even after
extensive ultracentrifugation of the extract. It was assumed initially that the aggregated
material seen by Western blotting might be causing a blockage in the column, leading to the
uneven elution observed. Other possible causes include the presence of polysaccharide or
nucleic acids in the sample. The use of a small column (1 x 10 cm) may have contributed to
the poor resolution, due to disturbances in the flow caused by surface-tension effects across
the narrow column diameter. However, this did not seem to be a major problem during
purification of soluble extract from <em>E. coli</em> JM83 (see Figure 4.12, Chapter 4), or during
other purifications.</p>
<p>To alleviate the possible effects of all the factors outlined above, a number of
changes were made to the procedure. First, <em>Ps. aeruginosa</em> PAO1 [pMMBSE] was grown
and induced aerobically in the presence of 50 mM potassium nitrate. The rationale for this
approach was that the protein is expressed aerobically following induction as shown by
Western blotting but the aggregated material is not observed and, additionally, it is been
shown that the presence of nitrate under aerobic conditions in <em>Ps. aeruginosa</em> is sufficient to
promote the synthesis of active holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> (and therefore <em>d</em><span class="math inline">\(_1\)</span> haem <span class="citation">[<a href="#ref-zennaro_nitrite_1993" role="doc-biblioref">215</a>]</span>. Cultures of 1 l volume were grown in 5 l flasks at a low shaking rate in the hope that this would lower the oxygen tension to some degree, increasing the expression of nitrite
reductase but preventing aggregation. Additionally, the cells were lysed by a lysozyme
freeze-thaw treatment to reduce polysaccharides and after centrifugation, the viscous
supernatant was treated with DNase and filtered. Finally the column diameter was increased
to 2 cm and the gradient was run against gravity to improve resolution <span class="citation">[<a href="#ref-scopes_protein_1987" role="doc-biblioref">234</a>]</span>. The
orange-brown soluble extract obtained was bound to the column, washed and eluted and this
time an even flow rate was observed with a clean, well-resolved elution profile exhibiting
three peaks (Figure 5.7). Haem-staining of the fractions from the top of each peak showed
that peaks 2 and 3 contained a <em>c</em>-type cytochrome with the same mobility as the native
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> (Figure 5.11). This is what would be expected were the
cytochromes to be semi-apo (peak 2) and holo- (peak 3) cytochrome <em>cd</em><span class="math inline">\(_1\)</span>; the latter protein
elutes at around 250 mM NaCl, with the semi-apo protein eluting slightly earlier on anion
exchange columns. A haem-stained sample of total extract from this culture (lane 1) shows
clearly that the cytochromes from peaks 2 and 3 correspond to the band that appears in this
culture on induction with IPTG, which is slightly higher than the endogenous cytochrome
<em>cd</em><span class="math inline">\(_1\)</span> band. It was therefore clear that both peaks, 2 and 3, contained recombinant <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.</p>
<p>Spectroscopy was used to characterise further the proteins from each of the three
peaks. Peak 1 exhibited a broad peak at 405 nm as isolated, and two peaks at 425 and
557 nm in the dithionite-reduced form. These features indicate a haem-containing protein,
possibly a <em>b</em>- or <em>c</em>-type cytochrome. The sample was not investigated further. Peak 2
exhibited absorbance maxima that identified it as being semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The
spectrum was virtually identical to that of the recombinant semi-apo cytochrome cd isolated
from <em>E. coli</em> JM83 (Chapter 4) and to that of the chemically-prepared semi-apo protein in the
reduced state. Peak 3 had a stronger A<span class="math inline">\(_{410}\)</span> absorbance than peak 2. The spectrum of the
pooled fractions from the peak (Figure 5.9a) exhibited peak absorbances at 409, 523, 547
and 628 nm. Overall, the spectrum resembled cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the oxidised state, but the
slight (red-shift) of the <em>c</em>-haem <span class="math inline">\(\beta\)</span> and <span class="math inline">\(\gamma\)</span> bands and the small absorbance at 547 nm indicated
that a small proportion of the enzyme was reduced. Absorbance in the 600-700 nm region
was small but significant, indicating the presence of <em>d</em><span class="math inline">\(_1\)</span> haem.</p>
<p>Several small additions of sodium dithionite resulted in a gradual reduction of the
protein and the final recorded spectrum is shown in Figure 5.9b. This spectrum had peak
absorbances at 418, 522, 548, 554 and 607 nm. There was a pronounced shoulder at 460 nm
and the absorbance peak of 607 nm was quite broad, spreading across the 600-650 nm
region. These features are again quite similar to the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> holoprotein in the
reduced form, but there are some significant differences; in Figure 5.9b the absorbance at
460 nm is attenuated compared to the native protein. Also, the absorbance in the 600-650
nm region is different, with the peak absorbance significantly blue-shifted and comprising a
single broad absorbance. However, difference spectra of the reduced minus as-isolated
material from peak 3 (Figure 5.10a) and of the reduced peak 3 minus peak 2 (Figure 5.10b)
indicate that the changes in the reduced spectrum of material from peak 3 have occurred in
regions where <em>d</em><span class="math inline">\(_1\)</span> haem absorbs, at 460 nm and in the 600-650 nm region.</p>
<p>Table 5.2 shows the ratios of some of the peak absorbances in the spectra of the
semiapo- and putative holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> expressed in <em>Ps. aeruginosa</em>, compared with the
native protein from <em>T. pantotropha</em>.</p>
<table>
<caption>
<span id="tab:table52">Table 5.2: </span>Comparison of the ratios of the absorbance maxima in the reduced state of native holo- and chemically-prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> with those of the partially purified recombinant protein from <em>Ps. aeruginosa</em> or <em>E. coli</em>.
Absorbance maxima of the recombinant proteins and of the chemically-prepared semiapo protein were recorded directly from spectroscopic measurements. The ratios for the purified holoprotein from <em>T. pantotropha</em> were calculated from extinction coefficients supplied by Alrik KoppenhÃ¶fer. Chemically-prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was reduced using sodium dithionite, whereas the recombinant semi-apoproteins were isolated in the reduced state and were not further reduced by the addition of sodium dithionite. The symbol (-)
indicates that the ratio is not relevant for the particular case.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Absorbance ratio
</th>
<th style="text-align:left;">
<em>T. pantotropha</em> holoprotein, reduced
</th>
<th style="text-align:left;">
<em>Ps. aeruginosa</em> recombinant holoprotein, reduced
</th>
<th style="text-align:left;">
<em>T. pantotropha</em> chemically-prepared semi-apoprotein, reduced
</th>
<th style="text-align:left;">
<em>Ps. aeruginosa</em> recombinant semi-apoprotein, reduced
</th>
<th style="text-align:left;">
<em>E. coli</em> recombinant semi-apoprotein, reduced
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
418:522
</td>
<td style="text-align:left;">
7.48
</td>
<td style="text-align:left;">
2.92
</td>
<td style="text-align:left;">
2.72
</td>
<td style="text-align:left;">
6.16
</td>
<td style="text-align:left;">
5.73
</td>
</tr>
<tr>
<td style="text-align:left;">
418:548
</td>
<td style="text-align:left;">
6.60
</td>
<td style="text-align:left;">
2.88
</td>
<td style="text-align:left;">
2.69
</td>
<td style="text-align:left;">
6.59
</td>
<td style="text-align:left;">
5.72
</td>
</tr>
<tr>
<td style="text-align:left;">
418:554
</td>
<td style="text-align:left;">
6.68
</td>
<td style="text-align:left;">
2.90
</td>
<td style="text-align:left;">
2.67
</td>
<td style="text-align:left;">
6.50
</td>
<td style="text-align:left;">
5.62
</td>
</tr>
<tr>
<td style="text-align:left;">
418:650
</td>
<td style="text-align:left;">
8.06
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
522:548
</td>
<td style="text-align:left;">
0.88
</td>
<td style="text-align:left;">
0.99
</td>
<td style="text-align:left;">
0.99
</td>
<td style="text-align:left;">
1.07
</td>
<td style="text-align:left;">
1.00
</td>
</tr>
<tr>
<td style="text-align:left;">
522:554
</td>
<td style="text-align:left;">
0.89
</td>
<td style="text-align:left;">
0.99
</td>
<td style="text-align:left;">
0.98
</td>
<td style="text-align:left;">
1.06
</td>
<td style="text-align:left;">
0.98
</td>
</tr>
<tr>
<td style="text-align:left;">
522:650
</td>
<td style="text-align:left;">
1.08
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
548:554
</td>
<td style="text-align:left;">
1.01
</td>
<td style="text-align:left;">
1.01
</td>
<td style="text-align:left;">
0.99
</td>
<td style="text-align:left;">
0.99
</td>
<td style="text-align:left;">
0.98
</td>
</tr>
<tr>
<td style="text-align:left;">
548:650
</td>
<td style="text-align:left;">
1.22
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
554:650
</td>
<td style="text-align:left;">
1.21
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
418:607
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
3.87
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
522:607
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
1.32
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
548:607
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
1.34
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:left;">
554:607
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
1.34
</td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
<td style="text-align:left;">
<ul>
<li></td>
</tr>
</tbody>
</table></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<p>These data indicate that the <em>c</em> haem:<em>d</em><span class="math inline">\(_1\)</span> haem ratio is somewhat lower in the expressed material. One explanation for this is that the material is not
spectroscopically pure and may be contaminated with some semi-apo protein because of
incomplete resolution of peaks 2 and 3. However, the peaks appear quite well separated in
the elution profile (Figure 5.7) and spectroscopic analysis of the later fractions from peak 3
before concentration gave similar results to those obtained after concentration (not shown).
A second explanation is that <em>d</em><span class="math inline">\(_1\)</span> haem incorporation differs somewhat in the expressed
protein and this is supported to some extent by analysis of the activity of the expressed
protein. As expected, peak 2, the recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> shows neither
oxidase or nitrite reductase activity, in the absence of the <em>d</em><span class="math inline">\(_1\)</span> haem. However, the
recombinant holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> has an oxidase activity of around 4.3 nmol cytochrome <em>c</em>
oxidised per minute per mg of protein. Using the extinction coefficient determined for
native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> at 548 nm, this converts to a turnover number of approximately 0.64
min<span class="math inline">\(^{-1}\)</span>. Purified native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> assayed under the same
conditions gave a turnover number of approximately 3 min<span class="math inline">\(^{-1}\)</span>. In contrast, the nitrite
reductase activity seemed less affected in the expressed protein; although these assays were
more variable, a turnover number of around 60 min<span class="math inline">\(^{-1}\)</span> was calculated for the expressed
protein, compared to about 124 min<span class="math inline">\(^{-1}\)</span> for the native protein. However, it may not be
appropriate to use the extinction coefficients determined for the native protein in these
circumstances, due to the spectroscopic changes observed in the recombinant material and
indeed, using different peak absorbances for these calculations gives different estimates of
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> concentration. An approximate idea of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> concentration might
be obtained from measuring the protein concentration of a pure sample, though in this case
not enough material was obtained for further purification. The most reasonable summary of
these results is that the recombinant protein has both oxidase and nitrite reductase activities,
which appear to be somewhat lower than those of the native protein, while spectroscopic
features that indicate the presence of <em>d</em><span class="math inline">\(_1\)</span> haem but suggest that it is not bound in an identical
fashion to that seen in the native protein. Inefficient <em>d</em><span class="math inline">\(_1\)</span> haem incorporation may also explain
why some of the recombinant material was expressed as the semi-apo protein, though it may
also be that under the semi-aerobic growth conditions, insufficient <em>d</em><span class="math inline">\(_1\)</span> haem was produced to
complex with all of the expressed polypeptide.</p>
<p>The observed differences between the expressed and native holocytochromes <em>cd</em><span class="math inline">\(_1\)</span> may
result from improper interactions between the expressed polypeptide and the biosynthetic
apparatus for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in <em>Ps. aeruginosa</em>. Although several homologous genes
involved in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> biosynthesis have been identified in <em>Ps. aeruginosa</em>, <em>Ps. stutzeri</em>
and <em>P. denitrificans</em>, the specific genes found in each organism and their physical
arrangement differs <span class="citation">[<a href="#ref-zumft_enzyme_1997" role="doc-biblioref">167</a>]</span> and it is likely that each system is adapted to
its own specific cytochrome <em>cd</em><span class="math inline">\(_1\)</span> polypeptide to some degree. Another possibility is that
under semi-aerobic conditions a proportion of the haem inserted in to the expressed protein is
not of the <em>d</em><span class="math inline">\(_1\)</span> type, but some other haem. <em>Ps. aeruginosa</em> synthesises a number of haem
molecules including haems <em>o</em> <span class="citation">[<a href="#ref-matsushita_membrane-bound_1980" role="doc-biblioref">308</a>]</span>, <em>a</em><span class="math inline">\(_3\)</span> and <em>a</em> <span class="citation">[<a href="#ref-fujiwara_novel_1992" role="doc-biblioref">309</a>]</span>, the
latter of which was shown to insert into semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <em>in vitro</em> and result in a low
oxidase activity <span class="citation">[<a href="#ref-hill_reconstitution_1978" role="doc-biblioref">118</a>]</span>. Alternative haems might be identifiable by
organic extraction and spectroscopy of the haem group; this was attempted on a small
amount of the expressed protein but was inconclusive, again due to the small amounts of
material obtained. However, it seems more likely that the protein contains <em>d</em><span class="math inline">\(_1\)</span> haem that is
bound differently from the native protein, as the observed absorbances were in the correct
positions for haem <em>d</em><span class="math inline">\(_1\)</span>, if not identical to the native protein, and the colours of the expressed
and native proteins were similar. Notably, incomplete reconstitution of the recombinant
semi-apo protein from <em>E. coli</em> produces cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in which the <em>d</em><span class="math inline">\(_1\)</span> haem absorbance
features observed in the present chapter (a broad absorbance at 600-650 nm with a blue
shifted peak and an attenuated 460 nm peak) are also observed.</p>
<p>The experiments described in this chapter suggest that <em>Ps. aeruginosa</em> is of some use
as a heterologous host for expression of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em>, but there are
disadvantages. The recombinant protein had some activity, which indicates that the system
could be used to determine whether mutants of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> possessed activity when
compared with the native protein. However, the apparent differences in the spectrum of the
expressed protein and in its activity, as compared with the native protein, warrant further
investigation. Additionally, the comments from Chapter 4 pertaining to methods of
improving the levels of expression apply here, as it would be advantageous to obtain more
material for further purification and subsequent analysis. These experiments show that the
method of using a denitrifying heterologous host for expression is a valid one, and it may
also be possible to try expression in other hosts, such as <em>Ps. stutzeri</em> or <em>A. eutrophus</em>. The
latter approach may also be advantageous as <em>Ps. aeruginosa</em> is an opportunistically
pathogenic organism that requires some care in handling and, as a further practical
consideration, requires the use of high concentrations of carbenicillin, an expensive
antibiotic, to maintain the plasmid pMMBSE.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-hill_reconstitution_1978" class="csl-entry">
118. Hill KE, Wharton DC. Reconstitution of the apoenzyme of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa with heme d1 and other heme groups. The Journal of Biological Chemistry. 1978;253:489â95.
</div>
<div id="ref-zumft_enzyme_1997" class="csl-entry">
167. Zumft WG, KÃ¶rner H. Enzyme diversity and mosaic gene organization in denitrification. Antonie Van Leeuwenhoek. 1997;71:43â58.
</div>
<div id="ref-de_boer_regulation_1996" class="csl-entry">
168. Boer T de. Regulation of denitrification in <span>Paracoccus</span> denitrificans. PhD thesis. VU Amsterdam; 1996. <a href="https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans">https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans</a>.
</div>
<div id="ref-spiro_regulation_1987" class="csl-entry">
181. Spiro S, Guest JR. Regulation and over-expression of the fnr gene of <span>Escherichia</span> coli. Journal of General Microbiology. 1987;133:3279â88.
</div>
<div id="ref-arai_expression_1995" class="csl-entry">
197. Arai H, Igarashi Y, Kodama T. Expression of the nir and nor genes for denitrification of <span>Pseudomonas</span> aeruginosa requires a novel <span>CRP</span>/<span>FNR</span>-related transcriptional regulator, <span>DNR</span>, in addition to <span>ANR</span>. FEBS letters. 1995;371:73â6.
</div>
<div id="ref-zennaro_nitrite_1993" class="csl-entry">
215. Zennaro E, Ciabatti I, Cutruzzola F, DâAlessandro R, Silvestrini MC. The nitrite reductase gene of <span>Pseudomonas</span> aeruginosa: Effect of growth conditions on the expression and construction of a mutant by gene disruption. FEMS microbiology letters. 1993;109:243â50.
</div>
<div id="ref-scopes_protein_1987" class="csl-entry">
234. Scopes RK. Protein <span>Purification</span>: <span>Principles</span> and <span>Practice</span>. 1987.
</div>
<div id="ref-steinrucke_genetics_1993" class="csl-entry">
257. SteinrÃ¼cke P, Ludwig B. Genetics of <span>Paracoccus</span> denitrificans. FEMS microbiology reviews. 1993;10:83â117.
</div>
<div id="ref-furste_molecular_1986" class="csl-entry">
271. FÃ¼rste JP, Pansegrau W, Frank R, BlÃ¶cker H, Scholz P, Bagdasarian M, et al. Molecular cloning of the plasmid <span>RP4</span> primase region in a multi-host-range <span class="nocase">tacP</span> expression vector. Gene. 1986;48:119â31.
</div>
<div id="ref-horio_purification_1961" class="csl-entry">
294. Horio T, Higashi T, Yamanaka T, Matsubara H, Okunuki K. Purification and properties of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. The Journal of Biological Chemistry. 1961;236:944â51.
</div>
<div id="ref-rothmel_genetic_1991" class="csl-entry">
295. Rothmel RK, Chakrabarty AM, Berry A, Darzins A. Genetic systems in <span>Pseudomonas</span>. Methods in Enzymology. 1991;204:485â514.
</div>
<div id="ref-hull_isoelectrophoretic_1993" class="csl-entry">
296. Hull HH, Wharton DC. Isoelectrophoretic characterization of <span>Pseudomonas</span> cytochrome oxidase/nitrite reductase and its heme d1-containing domain. Archives of Biochemistry and Biophysics. 1993;301:85â90.
</div>
<div id="ref-sawers_identification_1991" class="csl-entry">
297. Sawers RG. Identification and molecular characterization of a transcriptional regulator from <span>Pseudomonas</span> aeruginosa <span>PAO1</span> exhibiting structural and functional similarity to the <span>FNR</span> protein of <span>Escherichia</span> coli. Molecular Microbiology. 1991;5:1469â81.
</div>
<div id="ref-frey_replication_1992" class="csl-entry">
298. Frey J, Bagdasarian MM, Bagdasarian M. Replication and copy number control of the broad-host-range plasmid <span>RSF1010</span>. Gene. 1992;113:101â6.
</div>
<div id="ref-unden_role_1987" class="csl-entry">
299. Unden G, Duchene A. On the role of cyclic <span>AMP</span> and the <span>Fnr</span> protein in <span>Escherichia</span> coli growing anaerobically. Archives of Microbiology. 1987;147:195â200.
</div>
<div id="ref-west_codon_1988" class="csl-entry">
300. West SE, Iglewski BH. Codon usage in <span>Pseudomonas</span> aeruginosa. Nucleic Acids Research. 1988;16:9323â35.
</div>
<div id="ref-fujita_vitro_1992" class="csl-entry">
301. Fujita M, Amemura A. In vitro interactions of <span>Pseudomonas</span> <span>RNA</span> polymerases with tac and <span>RNA</span> <span>I</span> promoters. Bioscience, Biotechnology, and Biochemistry. 1992;56:1644â8.
</div>
<div id="ref-brunschwig_two-component_1992" class="csl-entry">
302. Brunschwig E, Darzins A. A two-component <span>T7</span> system for the overexpression of genes in <span>Pseudomonas</span> aeruginosa. Gene. 1992;111:35â41.
</div>
<div id="ref-ward_antibody_1993" class="csl-entry">
303. Ward BB, Cockcroft AR, Kilpatrick KA. Antibody and <span>DNA</span> probes for detection of nitrite reductase in seawater. Journal of General Microbiology. 1993;139:2285â93.
</div>
<div id="ref-wulfing_correctly_1994" class="csl-entry">
304. WÃ¼lfing C, PlÃ¼ckthun A. Correctly folded <span>T</span>-cell receptor fragments in the periplasm of <span>Escherichia</span> coli. <span>Influence</span> of folding catalysts. Journal of Molecular Biology. 1994;242:655â69.
</div>
<div id="ref-wong_expression_1988" class="csl-entry">
305. Wong EY, Seetharam R, Kotts CE, Heeren RA, Klein BK, Braford SR, et al. Expression of secreted insulin-like growth factor-1 in <span>Escherichia</span> coli. Gene. 1988;68:193â203.
</div>
<div id="ref-kortt_solution_1995" class="csl-entry">
306. Kortt AA, Guthrie RE, Hinds MG, Power BE, Ivancic N, Caldwell JB, et al. Solution properties of <span>Escherichia</span> coli-expressed <span>VH</span> domain of anti-neuraminidase antibody <span>NC41</span>. Journal of Protein Chemistry. 1995;14:167â78.
</div>
<div id="ref-cheng_alkaline_1970" class="csl-entry">
307. Cheng KJ, Ingram JM, Costerton JW. Alkaline phosphatase localization and spheroplast formation of <span>Pseudomonas</span> aeruginosa. Canadian Journal of Microbiology. 1970;16:1319â24.
</div>
<div id="ref-matsushita_membrane-bound_1980" class="csl-entry">
308. Matsushita K, Yamada M, Shinagawa E, Adachi O, Ameyama M. Membrane-bound respiratory chain of <span>Pseudomonas</span> aeruginosa grown aerobically. Journal of Bacteriology. 1980;141:389â92.
</div>
<div id="ref-fujiwara_novel_1992" class="csl-entry">
309. Fujiwara T, Fukumori Y, Yamanaka T. A novel terminal oxidase, cytochrome baa3 purified from aerobically grown <span>Pseudomonas</span> aeruginosa: It shows a clear difference between resting state and pulsed state. Journal of Biochemistry. 1992;112:290â8.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="4-chapter04.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="6-chapter06.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
