<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Construction of an unmarked deletion in the nirS gene of T. pantotropha: complementation of the mutant and expression of a site-directed mutant of cytochrome cd\(_1\) nitrite reductase | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Construction of an unmarked deletion in the nirS gene of T. pantotropha: complementation of the mutant and expression of a site-directed mutant of cytochrome cd\(_1\) nitrite reductase | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Construction of an unmarked deletion in the nirS gene of T. pantotropha: complementation of the mutant and expression of a site-directed mutant of cytochrome cd\(_1\) nitrite reductase | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="5-chapter05.html"/>
<link rel="next" href="7-chapter07.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter06" class="section level1" number="6">
<h1><span class="header-section-number">Chapter 6</span> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</h1>
<div id="introduction-3" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> Introduction</h2>
<p>In the previous two chapters, expression of the cloned <em>nirS</em> gene from <em>T. pantotropha</em> was investigated in the heterologous hosts
<em>E. coli</em> and <em>Ps. aeruginosa</em>. In each of these cases holo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> could be obtained, either by reconstitution <em>in vitro</em> with haem <em>d</em><span class="math inline">\(_1\)</span> in the
former case or by expression under denitrifying conditions in the latter. It was seen that
neither of these approaches were ideal in terms of both the uniformity of the material
obtained and the difficulties in comparing the expressed material to the wild-type protein.
The experiments in this chapter therefore focus on a third possible expression system,
utilising a mutant of <em>T. pantotropha</em> in which part of the <em>nirS</em> gene has been deleted. Such
mutants would not synthesise cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase and would not be expected to
grow well under anaerobic conditions, using nitrate as a terminal electron acceptor, because
the nitrite that would accumulate in the absence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> is toxic. In theory it
should be possible to complement the mutants using the <em>nirS</em> gene on a plasmid and restore
growth on nitrate. However, there are three possible problems to consider using this
approach: first, over-expression of the plasmid-borne <em>nirS</em> gene may interfere with the
normal regulation of denitrification in the cell (for instance by out-stripping the supply of
haem <em>d</em><span class="math inline">\(_1\)</span> to the polypeptide), second, disruption of the <em>nirS</em> gene may adversely affect <em>d</em><span class="math inline">\(_1\)</span>
haem biosynthesis and third, expression of inactive mutants of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> will not
restore growth on nitrate to the cells which will die because of the build-up of nitrite.</p>
<p>The experiments described in this chapter fall into three sections. In the first part the
construction of two mutagenic plasmids is described; these are used to insert a kanamycin
marker cassette upstream of the <em>nirS</em> gene and then to construct the unmarked deletion in the
gene itself. The second part of the chapter concentrates on the analysis of this mutant and its
complementation using the cloned <em>nirS</em> gene. Finally, in part three, attempts to express the
first site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in which Tyr-25 is replaced with phenylalanine
are described.</p>
</div>
<div id="results-3" class="section level2" number="6.2">
<h2><span class="header-section-number">6.2</span> Results</h2>
<div id="construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene" class="section level3" number="6.2.1">
<h3><span class="header-section-number">6.2.1</span> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</h3>
<p>The methods used to construct an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em> were
developed by van Spanning (1991) <span class="citation">[<a href="#ref-van_spanning_genes_1991" role="doc-biblioref">310</a>]</span>. To create the deletion two mutagenesis steps were used.
First, a kanamycin resistance marker was inserted upstream of the <em>nirS</em> gene cloned in the
suicide plasmid pGRPd1. The plasmid was then mobilised into <em>T. pantotropha</em> and cells in
which the marker had inserted into the chromosome by homologous recombination were
selected. This procedure is outlined in Figure 6.1.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-1"></span>
<img src="figure/fig6-1.png" alt="Construction of a kanamycin insertional mutant in the _nir_ region of the _T. pantotropha_ genome. (a) Plasmid pGRPN3K (Table 6.1) is introduced into the cell and undergoes a single homologous recombination with the _nir_ region of the chromosome. (b) This results in a plasmid integrant strain which is both kanamycin- and spectinomycin resistant due to the spectinomycin resistance gene on the vector. (c) A second homologous recombination results in integration of the _kan_ gene upstream of the _nirS_ gene and loss of the spectinomycin resistance determinant on the suicide plasmid. Enzymes: P, _PstI_, Sp, _SphI_." width="60%" />
<p class="caption">
Figure 6.1: Construction of a kanamycin insertional mutant in the <em>nir</em> region of the <em>T. pantotropha</em> genome. (a) Plasmid pGRPN3K (Table 6.1) is introduced into the cell and undergoes a single homologous recombination with the <em>nir</em> region of the chromosome. (b) This results in a plasmid integrant strain which is both kanamycin- and spectinomycin resistant due to the spectinomycin resistance gene on the vector. (c) A second homologous recombination results in integration of the <em>kan</em> gene upstream of the <em>nirS</em> gene and loss of the spectinomycin resistance determinant on the suicide plasmid. Enzymes: P, <em>PstI</em>, Sp, <em>SphI</em>.
</p>
</div>
<p>In the second step, a copy of the <em>nirS</em> gene from which a fragment had been deleted was cloned into a second suicide plasmid,
pRVS1. This construct was then mobilised into the strain of <em>T. pantotropha</em> carrying the
kanamycin marker and cells in which the marker and the wild-type <em>nirS</em> gene were replaced
by the copy carrying the deletion were selected. This step is outlined in Figure 6.2.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-2"></span>
<img src="figure/fig6-2.png" alt="Construction of a _nirS_ deletion mutant strain in _T. pantotropha_. (a) Plasmid pRVS$\Delta$nir (Table 6.1) is introduced into the cell and undergoes a single homologous recombination with the _nir_ region of the chromosome. (b) This results in a plasmid integrant strain which is kanamycin- and spectinomycin resistant and colours blue on XGal plates due to the _lacZ_ gene on the vector. (c) A second homologous recombination removes the _kan_ gene from the chromosome and inserts the cloned _nirS_ gene which contains the deletion. The _lacZ_ and spectinomycin resistance genes are lost on the suicide plasmid. Enzymes: (B), (E) 382 bp _BamHI_ - _EcoRI_ fragment removed, ends blunted and religated. N, _NotI_." width="60%" />
<p class="caption">
Figure 6.2: Construction of a <em>nirS</em> deletion mutant strain in <em>T. pantotropha</em>. (a) Plasmid pRVS<span class="math inline">\(\Delta\)</span>nir (Table 6.1) is introduced into the cell and undergoes a single homologous recombination with the <em>nir</em> region of the chromosome. (b) This results in a plasmid integrant strain which is kanamycin- and spectinomycin resistant and colours blue on XGal plates due to the <em>lacZ</em> gene on the vector. (c) A second homologous recombination removes the <em>kan</em> gene from the chromosome and inserts the cloned <em>nirS</em> gene which contains the deletion. The <em>lacZ</em> and spectinomycin resistance genes are lost on the suicide plasmid. Enzymes: (B), (E) 382 bp <em>BamHI</em> - <em>EcoRI</em> fragment removed, ends blunted and religated. N, <em>NotI</em>.
</p>
</div>
<p>The plasmids and strains of <em>T. pantotropha</em> constructed in this study are described in Table 6.1.</p>
<table>
<caption>
<span id="tab:table61">Table 6.1: </span>Bacterial strains and plasmids referred to in this chapter
</caption>
<thead>
<tr>
<th style="text-align:left;">
Plasmid name
</th>
<th style="text-align:left;">
Description
</th>
<th style="text-align:left;">
Source or reference
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
pUC4K
</td>
<td style="text-align:left;">
Kanamycin resistance gene (<em>kan</em>) from Tn903 cloned in pUC4
</td>
<td style="text-align:left;">
Pharmacia
</td>
</tr>
<tr>
<td style="text-align:left;">
pGEM5zf(+)
</td>
<td style="text-align:left;">
General purpose cloning vector
</td>
<td style="text-align:left;">
Promega
</td>
</tr>
<tr>
<td style="text-align:left;">
pGRPd1
</td>
<td style="text-align:left;">
6.7 kbp suicide vector
</td>
<td style="text-align:left;">
Van Spanning <em>et al</em>. (1990) <span class="citation">[<a href="#ref-van_spanning_mutagenesis_1990" role="doc-biblioref">311</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
pRVS1
</td>
<td style="text-align:left;">
11.6 kbp suicide vector containing <em>lacZ</em> gene for blue-white selection of plasmid integrants; used for construction of umarked deletion in <em>nirS</em>
</td>
<td style="text-align:left;">
Van Spanning <em>et al</em>. (1991a) <span class="citation">[<a href="#ref-van_spanning_method_1991" role="doc-biblioref">312</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMB67EH
</td>
<td style="text-align:left;">
8.8 kbp broad host range expression vector
</td>
<td style="text-align:left;">
Fürste <em>et al</em>. (1986) <span class="citation">[<a href="#ref-furste_molecular_1986" role="doc-biblioref">271</a>]</span>
</td>
</tr>
<tr>
<td style="text-align:left;">
pALTER-1
</td>
<td style="text-align:left;">
Cloning vector for site-directed mutagenesis using Altered Sites II system
</td>
<td style="text-align:left;">
Promega
</td>
</tr>
<tr>
<td style="text-align:left;">
pTNIR3
</td>
<td style="text-align:left;">
7.5 kbp <em>SphI</em> fragment, containing the nirS gene and surrounding region cloned in pGEM7zf(+)
</td>
<td style="text-align:left;">
Obtained from Thon de Boer, Vrije Universiteit, Amsterdam
</td>
</tr>
<tr>
<td style="text-align:left;">
pBNIR
</td>
<td style="text-align:left;">
2.4 kbp <em>PstI</em>-<em>SphI</em> fragment of pTNIR3 containing the <em>nirS</em> gene cloned in pBS(-)
</td>
<td style="text-align:left;">
This work (Chapter 4)
</td>
</tr>
<tr>
<td style="text-align:left;">
pTNIR3K
</td>
<td style="text-align:left;">
1.2 kbp fragment of pUC4K containing <em>kan</em> gene cloned upstream of <em>nirS</em> gene in pTNIR3
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pGRPN3K
</td>
<td style="text-align:left;">
8.3 kbp <em>SphI</em> fragment of pTNIR3 containing <em>nirS</em> and <em>kan</em> genes cloned in pGRPd1
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pLN3N
</td>
<td style="text-align:left;">
4.2 kbp <em>NotI</em> fragment of pTNIR3 containing <em>nirS</em> gene cloned in pGEM5zf(+)
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pLN3N<span class="math inline">\(\Delta\)</span>nir
</td>
<td style="text-align:left;">
pLN3N with 382 bp <em>BamHI-EcoRI</em> fragment removed from the <em>nirS</em> gene
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pRVS<span class="math inline">\(\Delta\)</span>nir
</td>
<td style="text-align:left;">
3.8 kbp <em>NotI</em> fragment from pLN3N<span class="math inline">\(\Delta\)</span>nir containing <em>nirS</em> gene deletion cloned in <em>SmaI</em> site of pRVS1
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMBSEK
</td>
<td style="text-align:left;">
3.6 kbp <em>XbaI-SphI</em> fragment of pBNIR containing the <em>nirS</em> gene and <em>kan</em> genes cloned in pMMB67EH
</td>
<td style="text-align:left;">
This work (Chapter 4)
</td>
</tr>
<tr>
<td style="text-align:left;">
pVA191-Y25F
</td>
<td style="text-align:left;">
600 bp <em>EcoRI</em> fragment of pBNIR containing the Y25F mutation in <em>nirS</em> cloned in pALTER-1
</td>
<td style="text-align:left;">
This work, obtained from Dr. Simon Baker
</td>
</tr>
<tr>
<td style="text-align:left;">
pLN3NY25F
</td>
<td style="text-align:left;">
As pLN3N but with Y25F mutation in the <em>nirS</em> gene
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pBY25F
</td>
<td style="text-align:left;">
2.4 kbp <em>PstI-NotI</em> fragment of pLN3NY25F containing <em>nirS</em> gene with Y25F mutation cloned in pBluescript II KS(+)
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pBY25FK
</td>
<td style="text-align:left;">
1.2 kbp <em>PstI</em> fragment of pUC4K containing <em>kan</em> gene cloned in pBY25F
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
pMMBY25F
</td>
<td style="text-align:left;">
As pMMBSEK but with Y25F mutation in the <em>nirS</em> gene
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
<strong>Strain name</strong>
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>T. pantotropha</em> K7-K10
</td>
<td style="text-align:left;">
<em>T. pantotropha</em> containing kanamycin resistance gene in promoter region of <em>nirI</em>
</td>
<td style="text-align:left;">
This work
</td>
</tr>
<tr>
<td style="text-align:left;">
<em>T. pantotropha</em> <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span>, <span class="math inline">\(\Delta8\)</span>
</td>
<td style="text-align:left;">
<em>T. pantotropha</em> containing a 382 bp deletion in the <em>nirS</em> gene
</td>
<td style="text-align:left;">
This work
</td>
</tr>
</tbody>
</table>
<p>The strategy for construction of the suicide plasmid containing the <em>nirS</em> gene and the kanamycin marker is shown in Figure 6.3.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-3"></span>
<img src="figure/fig6-3.png" alt="Construction of the suicide plasmid pGRPN3K. Full details of plasmids are given in Table 6.1. Only the relevant restriction fragments are shown. Plasmid pGRPN3K was used to construct the kanamycin insertion mutant _T. pantotropha_ K7. Enzymes: B, _BamHI_, E, _EcoRI_, H, _HindIII_, N, _NotI_, P, _PstI_, S, _SalI_, Sp, _SphI_." width="60%" />
<p class="caption">
Figure 6.3: Construction of the suicide plasmid pGRPN3K. Full details of plasmids are given in Table 6.1. Only the relevant restriction fragments are shown. Plasmid pGRPN3K was used to construct the kanamycin insertion mutant <em>T. pantotropha</em> K7. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, H, <em>HindIII</em>, N, <em>NotI</em>, P, <em>PstI</em>, S, <em>SalI</em>, Sp, <em>SphI</em>.
</p>
</div>
<p>To create this plasmid, a 1.2 kbp <em>PstI</em> fragment
containing the kanamycin resistance gene (<em>kan</em>) from the plasmid pUC4K <span class="citation">[<a href="#ref-vieira_puc_1982" role="doc-biblioref">313</a>]</span> was isolated and ligated into the <em>PstI</em> site of the plasmid pTNIR3. The latter
site was located 27 bp upstream of the <em>nirS</em> NNR box. The new plasmid, designated
pTNIR3K, was digested with <em>SphI</em> to generate an 8.3 kbp fragment containing the <em>kan</em> gene
and having 5’- and 3’-ends in the <em>norB</em> and <em>nirE</em> genes respectively, as was described in
Chapter 4 (Figure 4.2). The <em>SphI</em> fragment was ligated into the <em>SphI</em> site of the suicide
plasmid pGRPd1, to give the final construct for use in insertional mutagenesis which was
designated pGRPN3K. The plasmid was analysed by restriction digestion to confirm that it
was the desired construct (Figure 6.4).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-4"></span>
<img src="figure/fig6-4.png" alt="Restriction digest analysis of plasmid pGRPN3K. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using _EcoRI_ (lanes 1-6), the DNA fragments were separated by electrophoresis on a 0.8% agarose gel. The possible orientations of the _SphI_ insert from pTNIR3K are shown below the gel. Two _EcoRI_ sites in the insert and a third site in the vector produce three fragments of approximate sizes 6.3, 5.6 and 3.6 kbp (first orientation) or 7.0, 5.6 and 2.9 Kbp (second orientation). These are seen in lanes 1 and 5 respectively. Enzymes: E, _EcoRI_, Sp, _SphI_. Lanes labelled III and II contain DNA size standards, the sizes of some of which are indicated to the left of the gel." width="60%" />
<p class="caption">
Figure 6.4: Restriction digest analysis of plasmid pGRPN3K. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using <em>EcoRI</em> (lanes 1-6), the DNA fragments were separated by electrophoresis on a 0.8% agarose gel. The possible orientations of the <em>SphI</em> insert from pTNIR3K are shown below the gel. Two <em>EcoRI</em> sites in the insert and a third site in the vector produce three fragments of approximate sizes 6.3, 5.6 and 3.6 kbp (first orientation) or 7.0, 5.6 and 2.9 Kbp (second orientation). These are seen in lanes 1 and 5 respectively. Enzymes: E, <em>EcoRI</em>, Sp, <em>SphI</em>. Lanes labelled III and II contain DNA size standards, the sizes of some of which are indicated to the left of the gel.
</p>
</div>
<p><em>T. pantotropha</em> was transformed with <em>pGRPN3K</em> by conjugal transfer of the plasmid
from <em>E. coli</em> S17-1. The mated cells were harvested, serially diluted and plated on minimal
acetate medium containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin and 1 mM PbNO<span class="math inline">\(_3\)</span> to select for <em>T. pantotropha</em> that had taken up the plasmid. 50 transconjugant colonies were picked from
these plates and streaked onto fresh minimal acetate medium containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>
kanamycin and onto identical plates that contained additionally 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> spectinomycin.
The selection step distinguished vector integrant strains arising from a single crossover
(phenotype Km<span class="math inline">\(^r\)</span>Sp<span class="math inline">\(^r\)</span>) from the desired insertion mutant strains which arose from a double
crossover (phenotype Km<span class="math inline">\(^r\)</span>Sp<span class="math inline">\(^s\)</span>). Fifteen of the latter type were identified, although each
streak was still contaminated to some degree with <em>E. coli</em> S17-1. To remove the
contaminants one of the streaks was picked with a sterile loop, diluted in LB medium and
lated on LB medium containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin and 1 mM PbNO<span class="math inline">\(_3\)</span>. Four single
colonies were obtained from this plate and designated <em>T. pantotropha</em> K7-K10. The colonies
were confirmed as having the required Km<span class="math inline">\(^r\)</span>Sp<span class="math inline">\(^s\)</span> phenotype. To confirm correct insertion of
the kanamycin resistance marker into the chromosome, genomic DNA was isolated from <em>T. pantotropha K7</em>, digested with <em>HindIII</em> and probed by Southern blotting using the 1.2 kbp
fragment of pUC4K labelled with digoxigenin. Fragments of the expected sizes (4.8 and 1.1
kbp) were observed on the blot (Figure 6.5), confirming that the kan gene had inserted only
once in the expected site.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-5"></span>
<img src="figure/fig6-5.png" alt="Southern blot of genomic DNA from _T. pantotropha_ K7 using the kanamycin resistance gene as a probe. Genomic DNA from wild-type _T. pantotropha_, _T. pantotropha_ K7 and _T. pantotropha_ $\Delta7$ was digested using _HindIII_ and probed with the 1.2 kbp _PstI_ fragment of pUC4K, labelled with digoxigenin-alkaline phosphatase conjugate. The two fragments of molecular weights 4.8 and 1.1 kbp in lane 2 arise from the insertion of the _kan_ gene upstream of the _nirS_ gene (Figure 6.3). Lanes: 1, _T. pantotropha_ wild-type, 2, _T. pantotropha_ K7,3, _T. pantotropha_ $\Delta7$, 4, _T. pantotropha_ $\Delta8$." width="50%" />
<p class="caption">
Figure 6.5: Southern blot of genomic DNA from <em>T. pantotropha</em> K7 using the kanamycin resistance gene as a probe. Genomic DNA from wild-type <em>T. pantotropha</em>, <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> was digested using <em>HindIII</em> and probed with the 1.2 kbp <em>PstI</em> fragment of pUC4K, labelled with digoxigenin-alkaline phosphatase conjugate. The two fragments of molecular weights 4.8 and 1.1 kbp in lane 2 arise from the insertion of the <em>kan</em> gene upstream of the <em>nirS</em> gene (Figure 6.3). Lanes: 1, <em>T. pantotropha</em> wild-type, 2, <em>T. pantotropha</em> K7,3, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, 4, <em>T. pantotropha</em> <span class="math inline">\(\Delta8\)</span>.
</p>
</div>
<p>Construction of the suicide plasmid containing a deletion in the <em>nirS</em> gene is outlined in
Figure 6.6.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-6"></span>
<img src="figure/fig6-6.png" alt="Construction of the suicide plasmid pRVS$\Delta$nir. Only the relevant restriction fragments are shown. Full details of the plasmids are given in Table 6.1. Plasmid pRVS$\Delta$nir was used to construct the _nirS_ deletion mutants $\Delta5$, $\Delta7$, $\Delta8$. Enzymes: B, _BamHI_, E, _EcoRI_, H, _HindIII_, N, _NotI_, S, _SalI_." width="60%" />
<p class="caption">
Figure 6.6: Construction of the suicide plasmid pRVS<span class="math inline">\(\Delta\)</span>nir. Only the relevant restriction fragments are shown. Full details of the plasmids are given in Table 6.1. Plasmid pRVS<span class="math inline">\(\Delta\)</span>nir was used to construct the <em>nirS</em> deletion mutants <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span>, <span class="math inline">\(\Delta8\)</span>. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, H, <em>HindIII</em>, N, <em>NotI</em>, S, <em>SalI</em>.
</p>
</div>
<p>A 4.2 kbp <em>NotI</em> fragment containing the <em>nirS</em> gene and approximately 1.9 kbp
and 0.5 kbp of upstream and downstream DNA respectively was isolated from the plasmid
pTNIR3 and ligated into the <em>NotI</em> site of the plasmid pGEM5zf(+), giving a plasmid which
was designated pLN3N. pLN3N was digested using <em>BamHI</em> and <em>EcoRI</em> to remove an internal
fragment of 382 bp from the <em>nirS</em> gene. The fragment corresponded to amino acids 9 to 136
in the mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span> polypeptide and included the haem ligands His-17 and Tyr-25.
The DNA fragment from which the 382 bp fragment had been removed was isolated, the
ends were blunted using T4 DNA polymerase and the plasmid was religated. The absence of
the 382 bp fragment was demonstrated with digests using the enzymes <em>PstI</em>, <em>NotI</em> and <em>SalI</em> (Figure 6.7).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-7"></span>
<img src="figure/fig6-7.png" alt="Restriction digest analysis of plasmids pLN3N and pLN3N$\Delta$nir. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using _PstI_ plus _NotI_ (lanes 1 and 2) or _SalI_ (lanes 3 and 4). The digested DNA was separated by electrophoresis on a 0.8% agarose gel. Lanes 1 and 3 contain DNA from pLN3N, lanes 2 and 4 contain DNA from pLN3N$\Delta$nir. In lane 1, three fragments of approximate sizes 3, 2.4 and 1.8 kbp are seen. The two smaller fragments correspond to the _NotI_ insert from pLN3N. In lane 2 the 2.4 kbp fragment has moved to 2 kbp, due to the removal of the 382 bp _BamHI_ - _EcoRI_ fragment. Lane 3 shows two _SalI_ fragments from pLN3N. In lane 4 the internal _SalI_ site in the 382 bp _BamHI_ - _EcoRI_ fragment has been lost, giving a linearised plasmid of approximate size 6.9 kbp. Enzymes: B, _BamHI_, E, _EcoRI_, H, _HindIII_, N, _NotI_, P, I, S, _SalI_. Lanes labelled III and II contain DNA size standards, some of which are indicated at the left of the gel." width="60%" />
<p class="caption">
Figure 6.7: Restriction digest analysis of plasmids pLN3N and pLN3N<span class="math inline">\(\Delta\)</span>nir. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using <em>PstI</em> plus <em>NotI</em> (lanes 1 and 2) or <em>SalI</em> (lanes 3 and 4). The digested DNA was separated by electrophoresis on a 0.8% agarose gel. Lanes 1 and 3 contain DNA from pLN3N, lanes 2 and 4 contain DNA from pLN3N<span class="math inline">\(\Delta\)</span>nir. In lane 1, three fragments of approximate sizes 3, 2.4 and 1.8 kbp are seen. The two smaller fragments correspond to the <em>NotI</em> insert from pLN3N. In lane 2 the 2.4 kbp fragment has moved to 2 kbp, due to the removal of the 382 bp <em>BamHI</em> - <em>EcoRI</em> fragment. Lane 3 shows two <em>SalI</em> fragments from pLN3N. In lane 4 the internal <em>SalI</em> site in the 382 bp <em>BamHI</em> - <em>EcoRI</em> fragment has been lost, giving a linearised plasmid of approximate size 6.9 kbp. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, H, <em>HindIII</em>, N, <em>NotI</em>, P, I, S, <em>SalI</em>. Lanes labelled III and II contain DNA size standards, some of which are indicated at the left of the gel.
</p>
</div>
<p>These digests showed clearly that the plasmid had lost 382 bp and that a <em>SalI</em> site which lay between the <em>BamHI</em> and <em>EcoRI</em> sites was not present. The new plasmid with
the 382 bp deletion was named pLN3N<span class="math inline">\(\Delta\)</span>nir.
The <em>NotI</em> fragment of pLN3N<span class="math inline">\(\Delta\)</span>nir, now 3.8 kbp long, was isolated, the ends were
blunted using Sequenase (T7 DNA polymerase) and the fragment was ligated into the <em>SmaI</em>
site of the suicide vector pRVS1 to give the final construct for mutagenesis, designated
pRVS<span class="math inline">\(\Delta\)</span>nir. The presence of the insert was confirmed by digestion with <em>EcoRI</em>, <em>StyI</em> and <em>SalI</em>
(Figure 6.8) and by a PCR reaction using pRVS<span class="math inline">\(\Delta\)</span>nir as template DNA and the primers nirSF
and MIDR (Table 3.1, Chapter 3).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-8"></span>
<img src="figure/fig6-8.png" alt="Restriction digest analysis of plasmids pRVS1 and pRVS$\Delta$nir. Following ligation and transformation, plasmid DNA was isolated from transformed _E. coli_ and digested using _EcoRI_ (lanes 1 and 2), _NruI_ (lanes 3 and 4), _SalI_ (lanes 5 and 6) and _StyI_ (lanes 7 and 8). The digested DNA was then separated by electrophoresis on a 0.8% agarose gel. Lanes 1, 3, 5 and 7 contain DNA from plasmid pRVS1, lanes 2, 4, 6 and 8 contain DNA from plasmid pRVS$\Delta$nir. As described in the text, restriction analysis of these plasmids was difficult to interpret due to the lack of a restriction map for pRVS1. However, the digests in lanes 1-2 and 5-6 clearly show that pRVS$\Delta$nir contains an insert of around 3.8 kbp, the expected size of the insert. The digests in lanes 3-4 and 7-8 show extra _NruI_ and _StyI_ sites in pRVS$\Delta$nir which can be related to the known sequence of the _nirS_ gene and to restriction analysis of plasmids containing the _nir_ region (Chapter 4). The lane labelled II contains DNA size standards, the sizes of some of which are indicated at the left of the gel." width="50%" />
<p class="caption">
Figure 6.8: Restriction digest analysis of plasmids pRVS1 and pRVS<span class="math inline">\(\Delta\)</span>nir. Following ligation and transformation, plasmid DNA was isolated from transformed <em>E. coli</em> and digested using <em>EcoRI</em> (lanes 1 and 2), <em>NruI</em> (lanes 3 and 4), <em>SalI</em> (lanes 5 and 6) and <em>StyI</em> (lanes 7 and 8). The digested DNA was then separated by electrophoresis on a 0.8% agarose gel. Lanes 1, 3, 5 and 7 contain DNA from plasmid pRVS1, lanes 2, 4, 6 and 8 contain DNA from plasmid pRVS<span class="math inline">\(\Delta\)</span>nir. As described in the text, restriction analysis of these plasmids was difficult to interpret due to the lack of a restriction map for pRVS1. However, the digests in lanes 1-2 and 5-6 clearly show that pRVS<span class="math inline">\(\Delta\)</span>nir contains an insert of around 3.8 kbp, the expected size of the insert. The digests in lanes 3-4 and 7-8 show extra <em>NruI</em> and <em>StyI</em> sites in pRVS<span class="math inline">\(\Delta\)</span>nir which can be related to the known sequence of the <em>nirS</em> gene and to restriction analysis of plasmids containing the <em>nir</em> region (Chapter 4). The lane labelled II contains DNA size standards, the sizes of some of which are indicated at the left of the gel.
</p>
</div>
<p>This primer pair gave a 903 bp PCR product from the complete <em>nirS</em> gene, but as the 382 bp deletion lies between the primers the expected size of
the PCR fragment should have been shortened to 521 bp. The smaller product was clearly
seen using pRVS<span class="math inline">\(\Delta\)</span>nir as the template DNA (Figure 6.9).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-9"></span>
<img src="figure/fig6-9.png" alt="PCR analysis of plasmid pRVS$\Delta$nir showing a 382 bp deletion in the cloned _nirS_ gene. Plasmid pRVS$\Delta$nir was used as the template in a PCR reaction with the primers nirSF (forward) and MIDR (reverse) described in Chapter 3, Table 3.1. The primer pair flanks the site of the deletion in the _nirS_ gene, generating a PCR product of 903 bp in the wild-type gene (Chapter 3, Figure 3.1). The 382 bp deletion in this region generates a PCR product of 521 bp, which can be seen in lane 1. The lane labelled M contains a 100 bp DNA size standard ladder with some of the fragment sizes indicated at the left of the gel." width="60%" />
<p class="caption">
Figure 6.9: PCR analysis of plasmid pRVS<span class="math inline">\(\Delta\)</span>nir showing a 382 bp deletion in the cloned <em>nirS</em> gene. Plasmid pRVS<span class="math inline">\(\Delta\)</span>nir was used as the template in a PCR reaction with the primers nirSF (forward) and MIDR (reverse) described in Chapter 3, Table 3.1. The primer pair flanks the site of the deletion in the <em>nirS</em> gene, generating a PCR product of 903 bp in the wild-type gene (Chapter 3, Figure 3.1). The 382 bp deletion in this region generates a PCR product of 521 bp, which can be seen in lane 1. The lane labelled M contains a 100 bp DNA size standard ladder with some of the fragment sizes indicated at the left of the gel.
</p>
</div>
<p><em>T. pantotropha</em> K7 was transformed with pRVS<span class="math inline">\(\Delta\)</span>nir by conjugal transfer from <em>E. coli</em>
S17-1 and transconjugants were selected initially on LB medium containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>
kanamycin, 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> spectinomycin and 40 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> X-Gal. These conditions selected for
vector integrant strains formed by a single crossover, the <em>lacZ</em> gene on the integrated plasmid
causing the colonies to be coloured pale blue. To promote the required double crossover
event, in which the vector together with the kanamycin resistance gene were excised and the
deletion was inserted into the chromosomal <em>nirS</em> gene, a single blue colony was inoculated
into 1.5 ml of LB medium that contained no antibiotics and grown aerobically overnight.
This culture was inoculated into fresh LB medium and grown again. The culture was then
serially diluted and plated on LB medium containing X-Gal at a concentration of 80 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>.
Most of the colonies obtained were vector integrants which formed small dark-blue colonies
on these plates. However, 25 white colonies were obtained and these were streaked onto
either LB medium, LB medium plus 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin or LB medium plus 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>
spectinomycin. Of these, three exhibited the required white, Km<span class="math inline">\(^s\)</span>Sp<span class="math inline">\(^s\)</span> phenotype; they were
stocked and designated <em>T. pantotropha</em> <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span> and <span class="math inline">\(\Delta8\)</span>.
To confirm that the deletion in the chromosomal <em>nirS</em> gene was correct, genomic DNA
was prepared from wild-type T. pantotropha, <em>T. pantotropha</em> K7 and the three deletion
mutants, digested using <em>StyI</em> and probed by Southern blotting using a 141 bp fragment of the
<em>nirS</em> gene that had been labelled with digoxigenin by PCR. The Southern blot (Figure 6.10)
showed all of the expected hybridising fragments, confirming that the deletion mutants were
the desired ones.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-10"></span>
<img src="figure/fig6-10.png" alt="Southern blot of genomic DNA from wild-type _T. pantotropha_, _T. pantotropha_ K7 and the _nirS_ deletion mutants $\Delta5$, $\Delta7$ and $\Delta8$ using a _nirS_-derived probe. Genomic DNA was digested using _StyI_ and probed using a 141 bp PCR product derived from the _nirS_ gene using the primers nirSF and 228R (Table 3.1, Chapter 3), labelled with digoxigenin-alkaline phosphatase conjugate. The map below the blot indicates the _StyI_ sites in the 4.2 kbp _NotI_ fragment in which the deletion was made to give pRVS$\Delta$nir. All of the expected fragments are visible on the blot except for the 2.1 kbp fragment in the insertion mutant strain; the larger size of this fragment and the small degree of overlap of the probe resulted in a very weak hybridisation that was barely visible on the original blot and cannot be seen on the photograph. The position of the 2.1 kbp fragment is indicated at the right of the gel, along with the other fragments. Lanes: 1, _T. pantotropha_ wild-type, 2, _T. pantotropha_ K7, 3-5, _T. pantotropha_ $\Delta5$, $\Delta7$ and $\Delta8$. Enzymes: S, _StyI_, P, _PstI_, B, _BamHI_, E, _EcoRI_" width="60%" />
<p class="caption">
Figure 6.10: Southern blot of genomic DNA from wild-type <em>T. pantotropha</em>, <em>T. pantotropha</em> K7 and the <em>nirS</em> deletion mutants <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span> and <span class="math inline">\(\Delta8\)</span> using a <em>nirS</em>-derived probe. Genomic DNA was digested using <em>StyI</em> and probed using a 141 bp PCR product derived from the <em>nirS</em> gene using the primers nirSF and 228R (Table 3.1, Chapter 3), labelled with digoxigenin-alkaline phosphatase conjugate. The map below the blot indicates the <em>StyI</em> sites in the 4.2 kbp <em>NotI</em> fragment in which the deletion was made to give pRVS<span class="math inline">\(\Delta\)</span>nir. All of the expected fragments are visible on the blot except for the 2.1 kbp fragment in the insertion mutant strain; the larger size of this fragment and the small degree of overlap of the probe resulted in a very weak hybridisation that was barely visible on the original blot and cannot be seen on the photograph. The position of the 2.1 kbp fragment is indicated at the right of the gel, along with the other fragments. Lanes: 1, <em>T. pantotropha</em> wild-type, 2, <em>T. pantotropha</em> K7, 3-5, <em>T. pantotropha</em> <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span> and <span class="math inline">\(\Delta8\)</span>. Enzymes: S, <em>StyI</em>, P, <em>PstI</em>, B, <em>BamHI</em>, E, <em>EcoRI</em>
</p>
</div>
</div>
<div id="analysis-of-the-nirs-insertion-and-deletion-mutants" class="section level3" number="6.2.2">
<h3><span class="header-section-number">6.2.2</span> Analysis of the <em>nirS</em> insertion and deletion mutants</h3>
<p>The kanamycin insertion mutant, unmarked deletion mutant and wild-type strains of <em>T. pantotropha</em> were compared first by growing each strain anaerobically in minimal succinate
medium plus 20 mM KNO<span class="math inline">\(_3\)</span> (Figure 6.11).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-11"></span>
<img src="figure/fig6-11.png" alt="Comparison of the anaerobic growth of _T. pantotropha_ with that of _T. pantotropha_ K7 and _T. pantotropha_ $\Delta7$. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increased absorbance at 650 nm (logarithmic scale). Symbols: (&amp;#9670;) _T. pantotropha_, (&amp;#9632;) _T. pantotropha_ K7, (&amp;#9650;) _T. pantotropha_ $\Delta7$. The other deletion mutant strains ($\Delta5$ and $\Delta8$) grew identically to $\Delta7$." width="60%" />
<p class="caption">
Figure 6.11: Comparison of the anaerobic growth of <em>T. pantotropha</em> with that of <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increased absorbance at 650 nm (logarithmic scale). Symbols: (◆) <em>T. pantotropha</em>, (■) <em>T. pantotropha</em> K7, (▲) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>. The other deletion mutant strains (<span class="math inline">\(\Delta5\)</span> and <span class="math inline">\(\Delta8\)</span>) grew identically to <span class="math inline">\(\Delta7\)</span>.
</p>
</div>
<p><em>T. pantotropha</em> wild-type and strain K7 grew at similar rates and reached a similar final optical density of around 1.3 at 650 nm. However,
the three <em>nirS</em> deletion mutants grew poorly; each one entered stationary phase earlier than
the wild-type or kanamycin insertion strains and reached a final A<span class="math inline">\(_{650}\)</span> of only around 0.52.
In the unmarked deletion mutants cytochrome <em>cd</em><span class="math inline">\(_1\)</span> should be absent, leading to a build
up of toxic nitrite due to nitrate reductase activity. To confirm this, 1 ml samples were taken
from each of the three cultures, harvested and assayed colorimetrically for nitrite (Figure
6.12).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-12"></span>
<img src="figure/fig6-12.png" alt="Comparison of (a) the anaerobic growth and (b) nitrite accumulation of _T. pantotropha_ with that of _T. pantotropha_ K7 and _T. pantotropha_ $\Delta7$. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples of culture were removed at each time point, centrifuged and the supernatant was assayed colorimetrically for nitrite. Symbols: (&amp;#9670;) _T. pantotropha_, (&amp;#9632;) _T. pantotropha_ K7, (&amp;#9650;) _T. pantotropha_ $\Delta7$." width="60%" />
<p class="caption">
Figure 6.12: Comparison of (a) the anaerobic growth and (b) nitrite accumulation of <em>T. pantotropha</em> with that of <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples of culture were removed at each time point, centrifuged and the supernatant was assayed colorimetrically for nitrite. Symbols: (◆) <em>T. pantotropha</em>, (■) <em>T. pantotropha</em> K7, (▲) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>.
</p>
</div>
<p>Culture medium that contained wild-type <em>T. pantotropha</em> showed a steady rise in
nitrite concentration to a maximum of about 3 mM near the end of the exponential growth
phase, followed by a rapid decline to undetectable levels in under two hours. A similar
process was observed for <em>T. pantotropha</em> K7, although the maximum concentration of nitrite
reached was higher at 13 mM. In the unmarked deletion mutant nitrite concentration
continued to rise as growth slowed and reached a maximum of almost 20 mM, indicating
that almost all the nitrate originally present was converted to nitrite. The cells stopped
growing at about the time that maximum nitrite concentration was reached.</p>
<p>After growth, each of the cultures was harvested and a total soluble extract was
prepared by sonication and ultracentrifugation. The colour of the cell pellets from each
culture differed; the wild-type cells were brown when harvested, whereas cells of <em>T. pantotropha</em> K7 were pink and cells from the deletion mutants were off-white. 40 <span class="math inline">\(\mu\)</span>g of
protein from each extract was separated by SDS-PAGE, transferred to nitrocellulose and
probed by Western blotting using mouse antibody to <em>P. denitrificans</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (Figure
6.13).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-13"></span>
<img src="figure/fig6-13.png" alt="Western blot of total soluble extracts from wild-type _T. pantotropha_, _T. pantotropha_ K7, _T. pantotropha_ $\Delta7$, T. pantotropha $\Delta7$ [pMMBSEK] and _T. pantotropha_ $\Delta7$ [pMMBY25F] using antibody raised against _P. denitrificans_ cytochrome _cd_$_1$. 40 $\mu$g of protein was loaded in each lane and the samples were separated in a 6% acrylamide SDS-PAGE gel, transferred to nitrocellulose by semi-dry electroblotting and probed using the anti-cytochrome _cd_$_1$ antibody. The 63 kDa band (mature cytochrome _cd_$_1$) and 47 kDa band (truncated cytochrome _cd_$_1$) are indicated by the upper and lower arrows respectively. Lanes: 1, _T. pantotropha_ wild-type, 2, _T. pantotropha_ K7, 3, _T. pantotropha_ $\Delta7$, 4, _T. pantotropha_ $\Delta7$ [pMMBSEK], 5, _T. pantotropha_ $\Delta7$ [pMMBY25F]." width="60%" />
<p class="caption">
Figure 6.13: Western blot of total soluble extracts from wild-type <em>T. pantotropha</em>, <em>T. pantotropha</em> K7, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, T. pantotropha <span class="math inline">\(\Delta7\)</span> [pMMBSEK] and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] using antibody raised against <em>P. denitrificans</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. 40 <span class="math inline">\(\mu\)</span>g of protein was loaded in each lane and the samples were separated in a 6% acrylamide SDS-PAGE gel, transferred to nitrocellulose by semi-dry electroblotting and probed using the anti-cytochrome <em>cd</em><span class="math inline">\(_1\)</span> antibody. The 63 kDa band (mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span>) and 47 kDa band (truncated cytochrome <em>cd</em><span class="math inline">\(_1\)</span>) are indicated by the upper and lower arrows respectively. Lanes: 1, <em>T. pantotropha</em> wild-type, 2, <em>T. pantotropha</em> K7, 3, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, 4, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK], 5, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F].
</p>
</div>
<p>Lanes that contained extract from <em>T. pantotropha</em> and <em>T. pantotropha</em> K7 each
contained a band of the same mobility (63 kDa) that corresponded to mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span>.
This band was absent in the extract from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> and a new band of lower
mobility (approximately 47 kDa) was detected in this extract.
The plasmid pMMBSEK (described in Chapter 4) was used to complement the <em>nirS</em>
deletion mutant with the wild-type <em>nirS</em> gene. The plasmid contained the <em>nirS</em> gene with its
upstream promoter region, as well as a kanamycin resistance gene in the <em>PstI</em> site upstream
of the NNR binding site. This arrangement provided a selectable marker in <em>T. pantotropha</em>
and also meant that induction using IPTG was unnecessary. Plasmid pMMBSEK was
transferred to <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> by conjugation from <em>E. coli</em> S17-1 and the transconjugants
were selected on minimal succinate medium containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin. The
presence of the plasmid in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> was confirmed by plasmid isolation and
restriction digests using <em>HindIII</em> (not shown). The transformed cells were grown
anaerobically in minimal succinate medium plus nitrate as before and the growth was
compared with <em>T. pantotropha</em>, <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> (Figure 6.14).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-14"></span>
<img src="figure/fig6-14.png" alt="Complementation of _T. pantotropha_ $\Delta7$ using plasmid pMMBSEK. Overnight cultures of _T. pantotropha_, _T. pantotropha_ $\Delta7$ and _T. pantotropha_ $\Delta7$ [pMMBSEK] were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). Symbols: (&amp;#9670;) _T. pantotropha_, (&amp;#9632;) _T. pantotropha_ $\Delta7$, (&amp;#9650;) _T. pantotropha_ $\Delta7$ [pMMBSEK]." width="60%" />
<p class="caption">
Figure 6.14: Complementation of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> using plasmid pMMBSEK. Overnight cultures of <em>T. pantotropha</em>, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] were inoculated at a dilution of 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). Symbols: (◆) <em>T. pantotropha</em>, (■) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, (▲) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK].
</p>
</div>
<p>Cells of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> that contained pMMBSEK grew normally and reached a similar
final optical density to the wild-type culture. The nitrite concentration in the culture medium
from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] also followed a similar time course to that in wild-
type culture (Figure 6.15).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-15"></span>
<img src="figure/fig6-15.png" alt="(a) Anaerobic growth of _T. pantotropha_ $\Delta7$ [pMMBSEK] and (b) nitrite concentration in the medium during growth. An overnight culture of _T. pantotropha_ $\Delta7$ [pMMBSEK] was inoculated 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples were removed at each time point, centrifuged and the supernatant was assayed colorimetrically for nitrite." width="60%" />
<p class="caption">
Figure 6.15: (a) Anaerobic growth of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] and (b) nitrite concentration in the medium during growth. An overnight culture of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] was inoculated 1/100 into minimal succinate medium plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples were removed at each time point, centrifuged and the supernatant was assayed colorimetrically for nitrite.
</p>
</div>
<p>A soluble extract was prepared from <em>T. pantotropha</em> [pMMBSEK] and probed for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> using Western blotting. The expression of
mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was restored to the cells by pMMBSEK and the 63 kDa band stained
slightly more strongly in this extract than in the wild-type lane (Figure 6.13). The 47 kDa
hand observed in the extract from the deletion mutant <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> was also observed
in the sample from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK].</p>
</div>
<div id="construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7" class="section level3" number="6.2.3">
<h3><span class="header-section-number">6.2.3</span> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></h3>
<p>Once it had been demonstrated that the <em>nirS</em> deletion mutant could be complemented
successfully using pMMBSEK, the next step was to try to express a site-directed mutant of
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> using the same system. The first such mutant to be tried was one in which
Tyr-25 was replaced with phenylalanine (Y25F). The role of this residue was discussed in
Chapter 3 and reasons for the choice of this mutant are discussed further in the next section.
The DNA fragment containing the Y25F mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was constructed by
Dr. Simon Baker using the Promega Altered Sites II system. The mutagenic plasmid is
outlined in Figure 6.16.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-16"></span>
<img src="figure/fig6-16.png" alt="Construction of the plasmid pVA191-Y25F containing the Y25F mutation in the _nirS_ gene of _T. pantotropha_. A 600 bp _EcoRI_ fragment from the plasmid pBNIR (Chapter 4, Table 4.1) containing approximately the first 500 bp of the _nirS_ gene was cloned into the vector pALTER-1. The Promega Altered Sites II site-directed mutagenesis system was then used to anneal a mutagenic oligonucleotide to the gene, substituting phenylalanine for Tyr-25 and introducing a silent _MluI_ restriction site for screening during later manipulations. The insert was then sequenced to check for secondary mutations. The relevant sequence of the gene is shown, with the Y25F and _MluI_ mutations underlined. Enzymes: B, _BamHI_, E, _EcoRI_, H, _HindIII_, P, _PstI_, S, _SalI_. Plasmid pVA191-Y25F was constructed by Dr. Simon Baker." width="60%" />
<p class="caption">
Figure 6.16: Construction of the plasmid pVA191-Y25F containing the Y25F mutation in the <em>nirS</em> gene of <em>T. pantotropha</em>. A 600 bp <em>EcoRI</em> fragment from the plasmid pBNIR (Chapter 4, Table 4.1) containing approximately the first 500 bp of the <em>nirS</em> gene was cloned into the vector pALTER-1. The Promega Altered Sites II site-directed mutagenesis system was then used to anneal a mutagenic oligonucleotide to the gene, substituting phenylalanine for Tyr-25 and introducing a silent <em>MluI</em> restriction site for screening during later manipulations. The insert was then sequenced to check for secondary mutations. The relevant sequence of the gene is shown, with the Y25F and <em>MluI</em> mutations underlined. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, H, <em>HindIII</em>, P, <em>PstI</em>, S, <em>SalI</em>. Plasmid pVA191-Y25F was constructed by Dr. Simon Baker.
</p>
</div>
<p>A 600 bp <em>EcoRI</em> fragment from the plasmid pBNIR (Chapter 4, Table 4.1), containing the first 500 bp of the <em>nirS</em> gene was ligated into the plasmid
pALTER-1. Tyr-25 was mutated to phenylalanine using a mutant oligonucleotide that additionally contained a silent mutation to introduce a restriction site for <em>MluI</em> into the
fragment. The latter mutation allowed for rapid screening of mutant clones during later
manipulations and the mutant plasmid was named pVA191-Y25F. The strategy for insertion
of the Y25F mutation into the full-length <em>nirS</em> gene is summarised in Figure 6.17.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-17"></span>
<img src="figure/fig6-17.png" alt="Construction of the expression plasmid pMMBY25F containing the Y25F mutation in the _nirS_ gene of _T. pantotropha_. Only the relevant restriction fragments of each plasmid are shown. Full details of the plasmids are given in Table 6.1. Enzymes: B, _BamHI_, E, _EcoRI_, K, _KpnI_, M, _MluI_, N, _NotI_, P, (PstI), S, _SacI_." width="60%" />
<p class="caption">
Figure 6.17: Construction of the expression plasmid pMMBY25F containing the Y25F mutation in the <em>nirS</em> gene of <em>T. pantotropha</em>. Only the relevant restriction fragments of each plasmid are shown. Full details of the plasmids are given in Table 6.1. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, K, <em>KpnI</em>, M, <em>MluI</em>, N, <em>NotI</em>, P, (PstI), S, <em>SacI</em>.
</p>
</div>
<p>First, a <em>BamHI</em> - <em>EcoRI</em> fragment was removed from pVA191-Y25F. This fragment carried the Y25F
mutation and was the same 382 bp of DNA that was removed from the <em>nirS</em> gene to construct
unmarked deletion mutants described in Section 6.2.1. The 382 bp fragment was ligated
into the plasmid pLN3N after first removing the equivalent fragment from the cloned <em>nirS</em>
gene in pLN3N and the new plasmid, designated pLN3NY25F was screened for the presence
of the new <em>MluI</em> site. The <em>nirS</em> gene containing the Y25F mutation was then cut from
pLN3NY25F as a 2.4 kbp <em>PstI</em> - <em>NotI</em> fragment and ligated into the vector pBluescript II
KS(+) which had been cut using the same enzymes. This construct was named pBY25F and
was analysed by restriction digestion and by partial sequencing using the 610F primer
(Chapter 3, Table 3.1) to confirm the cloning (Figure 6.18).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-18"></span>
<img src="figure/fig6-18.png" alt="Restriction digest analysis of plasmid pBY25FK. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using _BamHI_ (lane 1), _EcoRI_ (lane 2), _HindIII_ (lane 3), _SalI_ (lane 4), _StyI_ (lane 5), _KpnI_ - _SacI_ (lane 6) and _NruI_ (lane 7). Plasmid pBY25F from which pBY25FK was derived had previously been screened for the presence of the mutagenic _MluI_ site. The pBluescript (II) vector contained one additional site for _EcoRI_, _HindIII_ and _SalI_. Analysis of the digests showed that the construct was correct as shown in the diagram below the gel. Enzymes: B, _BamHI_, E, _EcoRI_, H, _HindIII_, K, _KpnI_, N, _NruI_, S, _SalI_, Sa, _SacI_, St, _StyI_. The lane labelled II contained DNA size standards, some of which are indicated at the left of the gel." width="60%" />
<p class="caption">
Figure 6.18: Restriction digest analysis of plasmid pBY25FK. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using <em>BamHI</em> (lane 1), <em>EcoRI</em> (lane 2), <em>HindIII</em> (lane 3), <em>SalI</em> (lane 4), <em>StyI</em> (lane 5), <em>KpnI</em> - <em>SacI</em> (lane 6) and <em>NruI</em> (lane 7). Plasmid pBY25F from which pBY25FK was derived had previously been screened for the presence of the mutagenic <em>MluI</em> site. The pBluescript (II) vector contained one additional site for <em>EcoRI</em>, <em>HindIII</em> and <em>SalI</em>. Analysis of the digests showed that the construct was correct as shown in the diagram below the gel. Enzymes: B, <em>BamHI</em>, E, <em>EcoRI</em>, H, <em>HindIII</em>, K, <em>KpnI</em>, N, <em>NruI</em>, S, <em>SalI</em>, Sa, <em>SacI</em>, St, <em>StyI</em>. The lane labelled II contained DNA size standards, some of which are indicated at the left of the gel.
</p>
</div>
<p>To construct an expression plasmid containing the <em>nirS</em> Y25F mutant gene, the 1.2 kbp <em>PstI</em> fragment from pUC4K
containing the kanamycin resistance gene was ligated into the <em>PstI</em> site of pBY25F and a 3.7
bp <em>KpnI</em> - <em>SacI</em> fragment containing this marker together with the mutated <em>nirS</em> gene was
ligated into the broad host range vector pMMB67EH that had been cut with the same
enzymes. The final plasmid, pMMBY25F, was thus identical to pMMBSEK except for the
Y25F mutation in the <em>nirS</em> gene and was checked by digestion using <em>MluI</em> (Figure 6.19).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-19"></span>
<img src="figure/fig6-19.png" alt="Restriction digest analysis of plasmid pMMBY25F using _MluI_. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using _MluI_ to check for the presence of the Y25F mutation in the _nirS_ gene. Plasmid pMMB67EH into which the _KpnI_ - _SacI_ insert from pBY25FK was cloned contains a single _MluI_ site, so this digest generated two fragments of approximate sizes 10.3 and 2.8 kbp, seen in lanes 1-7. Enzymes: K, _KpnI_, M, _MluI_, S, _SacI_. The lane labelled II contains DNA size standards, some of which are indicated at the left of the gel." width="60%" />
<p class="caption">
Figure 6.19: Restriction digest analysis of plasmid pMMBY25F using <em>MluI</em>. Following ligation and transformation, plasmid DNA was isolated from the transformed cells and digested using <em>MluI</em> to check for the presence of the Y25F mutation in the <em>nirS</em> gene. Plasmid pMMB67EH into which the <em>KpnI</em> - <em>SacI</em> insert from pBY25FK was cloned contains a single <em>MluI</em> site, so this digest generated two fragments of approximate sizes 10.3 and 2.8 kbp, seen in lanes 1-7. Enzymes: K, <em>KpnI</em>, M, <em>MluI</em>, S, <em>SacI</em>. The lane labelled II contains DNA size standards, some of which are indicated at the left of the gel.
</p>
</div>
<p>pMMBY25F was transferred to <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> by conjugal transfer from <em>E. coli</em>
S17-1 and the transconjugants were selected on minimal succinate medium containing 100
<span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin. <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] was then grown anaerobically in
minimal succinate medium plus nitrate containing 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> kanamycin (Figure 6.20a).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-20"></span>
<img src="figure/fig6-20.png" alt="Comparison of (a) anaerobic growth and (b) nitrite accumulation in _T. pantotropha_ $\Delta7$ [pMMBY25F] compared with that of _T. pantotropha_, _T. pantotropha_ $\Delta7$ and _T. pantotropha_ $\Delta7$ [pMMBSEK]. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate media plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples of each culture were removed at each time point and assayed colorimetrically for nitrite. Symbols: (&amp;#9670;) _T. pantotropha_, (&amp;#9632;) _T. pantotropha_ $\Delta7$, (&amp;#9650;) _T. pantotropha_ $\Delta7$ [pMMBSEK], (x) _T. pantotropha_ $\Delta7$ [pMMBY25F]." width="60%" />
<p class="caption">
Figure 6.20: Comparison of (a) anaerobic growth and (b) nitrite accumulation in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] compared with that of <em>T. pantotropha</em>, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK]. Overnight cultures of each strain were inoculated at a dilution of 1/100 into minimal succinate media plus 20 mM nitrate and growth was monitored by the increase in absorbance at 650 nm (logarithmic scale). 1 ml samples of each culture were removed at each time point and assayed colorimetrically for nitrite. Symbols: (◆) <em>T. pantotropha</em>, (■) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, (▲) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK], (x) <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F].
</p>
</div>
<p>Growth of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] was virtually indistinguishable from that of the
deletion mutants. Additionally, nitrite accumulated in the culture medium of <em>T. pantotropha</em>
[pMMBY25F] to almost 20 mM just as was seen in the deletion mutant cultures (Figure
6.20b). A total soluble extract was prepared from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] and
probed for cytochrome <em>cd</em><span class="math inline">\(_1\)</span> by Western blotting as described earlier (Figure 6.13). The 63
kDa band that corresponded to mature cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was much fainter in this extract than
in the extracts from wild-type, <em>T. pantotropha</em> K7 or <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] and
the 47 kDa band seen in the extracts from the deletion mutant was also present.</p>
<p>Although the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> Y25F mutant appeared to be inactive (as cultures of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] accumulated nitrite as though nitrite reductase were absent)
and present in low amounts, partial purification of the protein was attempted. 20 l of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] was grown anaerobically overnight, harvested and
spheroplasted to give 94 ml of periplasmic extract. The extract was bound to a DEAE
Sepharose column and eluted using a linear gradient of NaCl. Elution of haem proteins was
followed by measuring the absorbance of the fractions at 410 nm (Figure 6.21).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-21"></span>
<img src="figure/fig6-21.png" alt="Elution profile during ion exchange chromatography of soluble extract from anaerobically grown _T. pantotropha_ $\Delta7$ [pMMBY25F]. 20 l of _T. pantotropha_ $\Delta7$ [pMMBY25F] were grown anaerobically in minimal succinate medium plus 20 mM nitrate. A total soluble extract prepared from the cells was bound to a DEAE-Sepharose column (2 x 20 cm) and eluted with a 350 ml linear gradient of 0-400 mM NaCl in 50 mM Tris-HCl pH 8.0 (&amp;#9670;). Elution of haem proteins was followed by measuring the absorbance of the fractions at 410 nm (&amp;#9632;). Elution of the blue copper protein pseudoazurin in fractions 28-35 was monitored by absorbance at 595 nm (&amp;#9650;). The three major elution peaks are labelled numbers 2-4. As described in the text, a fourth _c_-type cytochrome eluted from the column either in the early fractions or during washing. In the above experiment this protein eluted during washing and was collected; however it is still referred to as peak 1." width="60%" />
<p class="caption">
Figure 6.21: Elution profile during ion exchange chromatography of soluble extract from anaerobically grown <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F]. 20 l of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] were grown anaerobically in minimal succinate medium plus 20 mM nitrate. A total soluble extract prepared from the cells was bound to a DEAE-Sepharose column (2 x 20 cm) and eluted with a 350 ml linear gradient of 0-400 mM NaCl in 50 mM Tris-HCl pH 8.0 (◆). Elution of haem proteins was followed by measuring the absorbance of the fractions at 410 nm (■). Elution of the blue copper protein pseudoazurin in fractions 28-35 was monitored by absorbance at 595 nm (▲). The three major elution peaks are labelled numbers 2-4. As described in the text, a fourth <em>c</em>-type cytochrome eluted from the column either in the early fractions or during washing. In the above experiment this protein eluted during washing and was collected; however it is still referred to as peak 1.
</p>
</div>
<p>A pink band that bound weakly to the column was eluted during washing in some trials, or in the early
part of the gradient in others; in Figure 6.21 this peak is not visible as it eluted during the
wash, but it is referred to as Peak 1. Three other major peaks (2-4) eluted from the column at
approximately 100, 150 and 220 mM NaCl respectively; these peak fractions were coloured
pink, yellow-brown and dark brown respectively. Additionally, a blue coloured peak eluted
from the column at approximately 50 mM NaCl; this peak was recorded by measuring its
absorbance at 595 nm as shown in Figure 6.21.
15 <span class="math inline">\(\mu\)</span>l of the fraction from the top of Peaks 1-4 was separated by SDS-PAGE and
stained in order to identify <em>c</em>-type cytochromes (Figure 6.22).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-22"></span>
<img src="figure/fig6-22.png" alt="SDS-PAGE gel of fractions eluting from the DEAE-Sepharose column during chromatography of soluble extract from _T. pantotropha_ $\Delta7$ [pMMBY25F], stained for _c_-type cytochromes. 15 $\mu$l of the top fraction from each peak was loaded on a 15% acrylamide SDS-PAGE gel, electrophoresed and stained for covalently-attached haem. The lane numbers refer to the peaks in Figure 6.21. Estimated molecular weights of the bands determined from an identical Coomassie-stained gel are indicated at the left of the gel." width="60%" />
<p class="caption">
Figure 6.22: SDS-PAGE gel of fractions eluting from the DEAE-Sepharose column during chromatography of soluble extract from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F], stained for <em>c</em>-type cytochromes. 15 <span class="math inline">\(\mu\)</span>l of the top fraction from each peak was loaded on a 15% acrylamide SDS-PAGE gel, electrophoresed and stained for covalently-attached haem. The lane numbers refer to the peaks in Figure 6.21. Estimated molecular weights of the bands determined from an identical Coomassie-stained gel are indicated at the left of the gel.
</p>
</div>
<p>The weakly-binding pink fractions under Peak 1 contained a <em>c</em>-type cytochrome with a molecular weight of
approximately 16 kDa. Peaks 2, 3 and 4 also contained <em>c</em>-type cytochromes with molecular
weights on SDS-PAGE of 17, 12 and 45 kDa respectively. Peaks 1 and 3 did not stain
strongly when Coomassie Blue was used to stain the gel (not shown). UV-visible spectroscopy was used to characterise further the coloured fractions. As isolated, peak 1
exhibited absorbance maxima at 406 and 528 nm (Figure 6.23a).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-23"></span>
<img src="figure/fig6-23.png" alt="Visible spectra of an unidentified _c_-type cytochrome from _T. pantotropha_ [pMMBY25F]. As isolated, peak 1 showed absorbance maxima at 406 and 528 nm (a). When reduced with dithionite in the presence of air, the cytochrome auto-oxidised rapidly. Reduction under argon produced new absorbance maxima at 416, 521 and 550 nm (b). The spectra are characteristic of a cytochrome _c_." width="60%" />
<p class="caption">
Figure 6.23: Visible spectra of an unidentified <em>c</em>-type cytochrome from <em>T. pantotropha</em> [pMMBY25F]. As isolated, peak 1 showed absorbance maxima at 406 and 528 nm (a). When reduced with dithionite in the presence of air, the cytochrome auto-oxidised rapidly. Reduction under argon produced new absorbance maxima at 416, 521 and 550 nm (b). The spectra are characteristic of a cytochrome <em>c</em>.
</p>
</div>
<p>The cytochrome auto-oxidised rapidly after reduction using sodium dithionite, but when the reduction was
performed under argon, absorbance maxima appeared at 416, 521 and 550 nm (Figure
623b). Fractions from under peak 1 had no nitrate reductase, nitrite reductase or cytochrome
<em>c</em> oxidase activities (not shown). Spectroscopy of peaks 2 (Figure 6.24a) and 3 (Figure
6.24b) after reduction with sodium dithionite identified them as cytochrome c<span class="math inline">\(_{550}\)</span> and
cytochrome <em>c</em>’ respectively.</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-24"></span>
<img src="figure/fig6-24.png" alt="Visible spectra of (a) cytochrome _c_$_{550}$ and (b) cytochrome _c_' from _T. pantotropha_ [pMMBY25F]. Peak 2 had absorbance maxima at 415, 522 and 550 nm when reduced. It was identified by spectroscopy, elution position and size on SDS-PAGE as cytochrome _c_$_{550}$. Peak 3 had absorbance maxima at 410, 500 and 604 nm when reduced and showed a shoulder to the red side of the peak at 410 nm. It was identified as being cytochrome _c_' using the methods employed to identify cytochrome _c_$_{550}$." width="60%" />
<p class="caption">
Figure 6.24: Visible spectra of (a) cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> and (b) cytochrome <em>c</em>’ from <em>T. pantotropha</em> [pMMBY25F]. Peak 2 had absorbance maxima at 415, 522 and 550 nm when reduced. It was identified by spectroscopy, elution position and size on SDS-PAGE as cytochrome <em>c</em><span class="math inline">\(_{550}\)</span>. Peak 3 had absorbance maxima at 410, 500 and 604 nm when reduced and showed a shoulder to the red side of the peak at 410 nm. It was identified as being cytochrome <em>c</em>’ using the methods employed to identify cytochrome <em>c</em><span class="math inline">\(_{550}\)</span>.
</p>
</div>
<p>The spectrum of the dark-brown coloured peak 3 exhibited absorbance maxima at 408,
526 and 639 nm as isolated (Figure 6.25a) and at 419, 524, 556 and 620 nm when reduced
with sodium dithionite (Figure 6.25b).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-25"></span>
<img src="figure/fig6-25.png" alt="Visible spectra of cytochrome _c_ peroxidase from _T. pantotropha_ [pMMBY25F]. Peak 4 as isolated showed absorbance maxima at 408, 526 and 639 nm. When reduced with sodium dithionite, absorbance maxima were seen at 419, 524, 556 and 620 with a shoulder at the blue side of the 556 nm peak. Peak 4 was identified by spectroscopy, size on SDS-PAGE and enzyme assay as cytochrome _c_ peroxidase." width="60%" />
<p class="caption">
Figure 6.25: Visible spectra of cytochrome <em>c</em> peroxidase from <em>T. pantotropha</em> [pMMBY25F]. Peak 4 as isolated showed absorbance maxima at 408, 526 and 639 nm. When reduced with sodium dithionite, absorbance maxima were seen at 419, 524, 556 and 620 with a shoulder at the blue side of the 556 nm peak. Peak 4 was identified by spectroscopy, size on SDS-PAGE and enzyme assay as cytochrome <em>c</em> peroxidase.
</p>
</div>
<p>These spectra strongly resembled the published spectra of cytochrome <em>c</em> peroxidase from <em>P. denitrificans</em> <span class="citation">[<a href="#ref-gilmour_spectroscopic_1993" role="doc-biblioref">314</a>]</span> and when
assayed the peak 3 fractions did have a significant peroxidase activity, oxidising 3.2 <span class="math inline">\(\mu\)</span>mol of
horse heart cytochrome <em>c</em> per minute per <span class="math inline">\(\mu\)</span>l of fraction in the presence of 18 <span class="math inline">\(\mu\)</span>M H<span class="math inline">\(_2\)</span>O<span class="math inline">\(_2\)</span>
(Figure 6.26).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-26"></span>
<img src="figure/fig6-26.png" alt="Assay of cytochrome _c_ peroxidase activity in peak 4 following ion exchange chromatography. The assay was performed in a volume of 1 ml, containing 50 mM Tris-HCl pH 8.0, 25 $\mu$M horse heart cytochrome _c_ and varying volumes of the fractions from the top of peak 4. After incubation of these components for 1 minute to reduce the peroxidase (Gilmour _et al_., 1994), the reaction was initiated by the addition of 18 $\mu$M H$_2$O$_2$. Oxidation of cytochrome _c_ was followed by the decrease in absorbance at 550 nm." width="60%" />
<p class="caption">
Figure 6.26: Assay of cytochrome <em>c</em> peroxidase activity in peak 4 following ion exchange chromatography. The assay was performed in a volume of 1 ml, containing 50 mM Tris-HCl pH 8.0, 25 <span class="math inline">\(\mu\)</span>M horse heart cytochrome <em>c</em> and varying volumes of the fractions from the top of peak 4. After incubation of these components for 1 minute to reduce the peroxidase (Gilmour <em>et al</em>., 1994), the reaction was initiated by the addition of 18 <span class="math inline">\(\mu\)</span>M H<span class="math inline">\(_2\)</span>O<span class="math inline">\(_2\)</span>. Oxidation of cytochrome <em>c</em> was followed by the decrease in absorbance at 550 nm.
</p>
</div>
<p>In addition to the cytochrome-containing fractions, the blue fractions 28-35
were identified by visible spectroscopy as the small copper protein pseudoazurin (Figure
6.27).</p>
<div class="figure" style="text-align: center"><span id="fig:fig6-27"></span>
<img src="figure/fig6-27.png" alt="Visible spectrum of pseudoazurin from _T. pantotropha_ [pMMBY25F]. Fractions 28-35 (Figure 6.21) were coloured pale-blue. The blue material had absorbance maxima at 408, 587 and 739 nm as isolated. Although the spectrum does not exactly match published spectra for pseudoazurin from _T. pantotropha_ [@moir_purification_1993], it could be identified by size on SDS-PAGE and elution position. The peak at 408 nm overlays a peak at 450 nm and probably indicates contamination of the pseudoazurin peak by the small cytochrome _c_ in peak 1." width="60%" />
<p class="caption">
Figure 6.27: Visible spectrum of pseudoazurin from <em>T. pantotropha</em> [pMMBY25F]. Fractions 28-35 (Figure 6.21) were coloured pale-blue. The blue material had absorbance maxima at 408, 587 and 739 nm as isolated. Although the spectrum does not exactly match published spectra for pseudoazurin from <em>T. pantotropha</em> <span class="citation">[<a href="#ref-moir_purification_1993" role="doc-biblioref">101</a>]</span>, it could be identified by size on SDS-PAGE and elution position. The peak at 408 nm overlays a peak at 450 nm and probably indicates contamination of the pseudoazurin peak by the small cytochrome <em>c</em> in peak 1.
</p>
</div>
<p>However, no elution peak that corresponded to cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was observed in
several trials. A cytochrome <em>cd</em><span class="math inline">\(_1\)</span> elution peak was also absent when a hydrophobic phenyl
Sepharose column was employed as the first purification step (not shown).</p>
</div>
</div>
<div id="discussion-3" class="section level2" number="6.3">
<h2><span class="header-section-number">6.3</span> Discussion</h2>
<p>The method used to construct an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em> has
been used previously in the closely-related organism <em>P. denitrificans</em> PD1222 to mutate a
number of genes including those encoding amicyanin <span class="citation">[<a href="#ref-van_spanning_mutagenesis_1990" role="doc-biblioref">311</a>]</span>,
cytochrome <em>c</em><span class="math inline">\(_{551i}\)</span>, cytochrome c<span class="math inline">\(_{550}\)</span> and cytochrome c<span class="math inline">\(_{553i}\)</span> <span class="citation">[<a href="#ref-van_spanning_method_1991" role="doc-biblioref">312</a>]</span>, the
cytochrome <em>c</em> oxidases <em>aa</em><span class="math inline">\(_3\)</span> and <em>cbb</em><span class="math inline">\(_3\)</span> <span class="citation">[<a href="#ref-van_der_oost_reversed_1995" role="doc-biblioref">315</a>]</span> and the genes <em>moxJ</em> and
<em>moxG</em> which are involved in methanol oxidation <span class="citation">[<a href="#ref-van_spanning_isolation_1991" role="doc-biblioref">316</a>]</span>. The method
has a number of advantages as compared to simple insertional mutagenesis using an
antibiotic resistance cassette; first, in-frame deletions can be constructed in order to prevent
polar effects on genes downstream of the mutated target gene and, second, the method allows
for the construction of multiple mutations by the repeated insertion of a kanamycin marker
into several targets followed by its replacement with the unmarked deletion. This is
especially important for <em>P. denitrificans</em> in which few selectable markers have been reported
<span class="citation">[<a href="#ref-steinrucke_genetics_1993" role="doc-biblioref">257</a>]</span>.</p>
<p>The first step of the mutagenic procedure, insertion of the kanamycin cassette into the
<em>PstI</em> site upstream of <em>nirS</em>, was accomplished quite easily. A large number of kanamycin
resistant transconjugants were selected after mating and 15 out of 100 of these proved to be
exconjugants when tested for spectinomycin susceptibility. This compares well with the
frequency of 40% reported for kanamycin insertion mutagenesis of the <em>cycA</em> gene in <em>P. denitrificans</em> <span class="citation">[<a href="#ref-van_spanning_genes_1991" role="doc-biblioref">310</a>]</span>. The ease of this procedure was probably due in part to the extensive homologous regions each side of the kanamycin cassette in the mutagenic
plasmid pGRPN3K, in which there was approximately 5.1 kbp and 2.1 kbp of DNA
upstream and downstream respectively of the kanamycin cassette. It has been suggested that
at least 1 kb of homologous DNA each side of the insertion region is desirable for efficient
homologous recombination of both arms in <em>P. denitrificans</em> (Dr. Simon Baker, personal
communication). Southern blotting using the labelled kanamycin cassette showed clearly
that the fragment has inserted only once into the chromosome and the hybridisation of the
fragment to genomic DNA digested with <em>HindIII</em> indicated that the insertion was at the
correct site. This was confirmed by hybridisation of the labelled <em>nirS</em> gene fragment to DNA
digested using <em>StyI</em>, although in this experiment hybridisation to the larger of the two
expected fragments was very weak, probably due to the low region of overlap between the
large fragment and the small <em>nirS</em>-derived probe.</p>
<p>The second step of the method, replacement of the <em>nirS</em> gene and the kanamycin
marker by an unmarked mutation was somewhat more difficult to achieve. This was largely
due to difficulties experienced in ligating the blunted <em>NotI</em> fragment from pLN3N<span class="math inline">\(\Delta\)</span>nir into
the blunt-ended <em>SmaI</em> site of pRVS1. Eventually this cloning was achieved by blunting the
<em>NotI</em> ends using T7 DNA polymerase. Restriction analysis of the plasmid pRVS<span class="math inline">\(\Delta\)</span>nir was
complicated by the lack of a published restriction map for this plasmid and by the
observation that with certain enzymes (<em>e.g.</em> <em>PstI</em>) an anomalous restriction pattern was
observed in which the sizes of the fragments produced did not appear to add up to the total
size of the plasmid (11.6 kbp). However, the correctness of the construct was confirmed by a
combination of restriction and PCR analyses, and ultimately by Southern blotting of the
mutants obtained and their subsequent analysis. In future it would be advantageous to
engineer fragments for cloning into pRVS1 to have the appropriate restriction sites at their
termini and to have a detailed restriction map of the plasmid.</p>
<p>Replacement of the chromosomal <em>nirS</em> gene with the unmarked deletion also proved to
be quite challenging. To begin with, the procedure for selection of cells in which pRVS<span class="math inline">\(\Delta\)</span>nir
had become integrated into the chromosome proved to be slightly different to the published
protocol <span class="citation">[<a href="#ref-van_spanning_genes_1991" role="doc-biblioref">310</a>]</span>, in that a concentration of 20 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> of X-Gal in the plates
was not sufficient to turn the colonies blue. At 40 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span> the colonies were pale-blue.
However, the presence of spectinomycin and kanamycin, both at 100 <span class="math inline">\(\mu\)</span>g ml<span class="math inline">\(^{-1}\)</span>, should have
been sufficient to detect plasmid integrant strains. In the next stage of the published protocol
cells are scraped from these plates, replated in the absence of antibiotics to promote excision
of the plasmid from the chromosome, then collected once more and plated in the presence of
a high X-Gal concentration to detect exconjugants, which colour white under these
conditions. In the case of <em>T. pantotropha</em> very few white colonies were obtained in this way,
and so a single colony was grown for two cycles in liquid culture to promote
recombinational excision of the plasmid. The number of white colonies obtained was
increased but was still very low; 25 were obtained after the second culture was diluted 10<span class="math inline">\(^7\)</span>
fold and plated on 10 plates, of which three colonies were kanamycin-sensitive and
contained the deletion. The frequency is somewhat lower than that reported for deletion
mutagenesis of the <em>cycA</em> gene in <em>P. denitrificans</em> (resolution frequency of 5 x 10<span class="math inline">\(^{-4}\)</span>, of which
25% were kanamycin-sensitive <span class="citation">[<a href="#ref-van_spanning_genes_1991" role="doc-biblioref">310</a>]</span>.</p>
<p>Growth of the deletion mutant strains revealed the expected phenotype as compared
with the wild-type cells; in the absence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> the nitrite produced by nitrate
reductase activity accumulates rapidly to levels beyond which the cells no longer grow. It is
notable that the maximum nitrite concentration reached was practically equal to that of the
nitrate supplied originally in the medium, indicating that even if the cells are not growing
nitrate reductase activity is able to continue quite efficiently. The peak nitrite concentration
reached in cultures containing the kanamycin insertion was somewhat higher than in the
wild-type strain, indicating that nitrite reduction in this strain was impaired. On a Western
blot, slightly less cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was observed in this mutant strain, and when harvested, the
cells were noticeably pink compared to the wild-type. The kanamycin insertion lies in the
promoter region of the <em>nirI</em> gene, which is known to be involved in the regulation of <em>nirS</em>
expression [de_boer_regulation_1996]. It is probable that in this mutant expression of <em>nirI</em> is impaired
but not abolished, allowing the formation of some active cytochrome <em>cd</em><span class="math inline">\(_1\)</span>. The pink colour of
the cell pellet is possibly due to the expression of some semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, which is
pink as it lacks the <em>d</em><span class="math inline">\(_1\)</span> haem, and/or to the attenuated level of holo-cytochrome <em>cd</em><span class="math inline">\(_1\)</span> which
normally colours the cell pellet brown.</p>
<p>Western blotting of the cell extracts from the deletion mutants showed that cytochrome
<em>cd</em><span class="math inline">\(_1\)</span> was absent, but revealed the presence of a new band with an approximate molecular
weight of 47 kDa. This band was specific to the deletion mutant and cells derived from it (it
was also observed in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> carrying the plasmids pMMBSEK and pMMBY25F,
but not in the extracts from wild-type cells or from <em>T. pantotropha</em> K7). The most likely
explanation for this band is that it represents a truncated form of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> generated
by the removal of the 382 bp <em>BamHI</em> - <em>EcoRI</em> fragment. During the latter procedure the
cohesive termini generated by restriction using these enzymes were filled in using T4 DNA
polymerase. Without sequencing across this junction it is not possible to know precisely
what junction was formed, as the polymerase may have removed some nucleotides from
either terminus as well as filling in the ends. However, analysis of the truncated gene using
the program SeqAid <span class="citation">[<a href="#ref-roufa_seqaid_1989" role="doc-biblioref">317</a>]</span> indicates that the procedure may have
generated two new open reading frames. The first of these begins at the original ATG start
codon and is 264 bp long, encoding for 88 amino acids and is unlikely to be relevant. The
second begins at an ATG codon after the deletion and encodes a polypeptide of 429 amino
acids long (47.8 kDa) and this protein may be the species that is cross-reacting on the
Western blot. The presence of this polypeptide is not significant in terms of the later
complementation and mutagenesis experiments as it is unlikely to be periplasmic in the
absence of a targeting sequence, is clearly not active and does not contain the region in
which the Y25F mutation was made and so could not exchange this residue with the mutated
residue on plasmid pMMBY25F.</p>
<p>Complementation of the deletion mutant <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> using the plasmid
pMMBSEK was successful, as shown by the restoration of growth on nitrate and the
presence of the cytochrome <em>cd</em><span class="math inline">\(_1\)</span> polypeptide by Western blotting. The latter technique
showed that there was a significant, but not excessive, increase in the amount of expressed
protein from the plasmid as compared with the chromosomal copy of the gene, indicating
that the copy number is quite low in <em>T. pantotropha</em> as was discussed in Chapter 5. The
increased levels of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> do not seem to have had a deleterious effect on the
regulation of denitrification in the cells as judged by analysis of the growth curves and of
nitrite levels in the culture medium, both of which were very similar to wild-type cultures.
pMMBSEK proved to be a good choice of plasmid for these experiments; it contains the
kanamycin resistance gene which is selectable in <em>T. pantotropha</em> and the upstream promoter
region which seems to be fully functional, removing the need for induction using IPTG. The
last point is important as the <em>tac</em> promoter is believed to function quite poorly in <em>P. denitrificans</em>
(Dr. M.L.D. Page and Dr. E. Gordon, personal communication).</p>
<p>Having demonstrated that the <em>nirS</em> deletion mutant could be complemented
successfully with a plasmid the next step was to try complementation using a site-directed
mutant of the <em>nirS</em> gene. The first mutant chosen was one in which Tyr-25 was mutated to
phenylalanine. The role of Tyr-25 was discussed in Chapter 3; it is a ligand to the <em>d</em><span class="math inline">\(_1\)</span> haem
in the oxidised enzyme structure and is postulated to displace nitric oxide from the haem iron
during the catalytic cycle of nitrite reduction <span class="citation">[<a href="#ref-fulop_anatomy_1995" role="doc-biblioref">127</a>]</span>. This crucial role made
Tyr-25 an attractive candidate for mutagenesis. Phenylalanine was chosen as a replacement
residue as it has the same core structure as tyrosine, lacking only the hydroxyl group; the
latter feature would be expected to prevent bonding to the <em>d</em><span class="math inline">\(_1\)</span> haem iron. Additionally, Tyr-25
is located on the 382 bp <em>BamHI</em> - <em>EcoRI</em> fragment that was removed from the chromosomal <em>nirS</em> gene
to make the deletion mutant. This facilitated the construction of the mutant as the
same fragment could be inserted into the expression plasmid as a cassette and screened for
using the silent <em>MluI</em> restriction site. The absence of the residue in the chromosomal copy of
the gene also meant that it could not be transferred to the plasmid by homologous
recombination, although the probability of this kind of transfer is quite low and can always
be monitored by isolation of the plasmid from the cells and restriction analysis or
sequencing. One other possibility is that of reversion, in which the Y25F mutant of the <em>nirS</em>
gene might be transferred to the chromosome by homologous recombination and then
repaired by random mutation to give the wild-type gene. However, this is most unlikely as
the amount of homologous overlapping DNA between the plasmid and the chromosome is
quite low, especially at the 5’-end and, additionally, reversion is extremely rare in <em>P. denitrificans</em>
except under conditions of extreme selective pressure <span class="citation">[<a href="#ref-steinrucke_genetics_1993" role="doc-biblioref">257</a>]</span>.</p>
<p>Growth of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F) anaerobically on nitrate showed a
phenotype that was virtually identical to that of the deletion mutants, with poor growth and
an accumulation of nitrite to toxic levels. This was taken initially as evidence that the
expressed protein was not active. However, Western blotting showed that only very low
amounts of the protein were present. This might have been a consequence of the low yield
of cells (an ideal control experiment at this stage, although not possible in the present work,
would have been to compare the level of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> to that in cells in which
denitrification was impaired due to another cause, such as mutations in other electron
transfer components), but the amount of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> seemed abnormally low even taking
the poor growth into account. To investigate this further 20 l of cells were grown and partial
purification of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was attempted from a periplasmic extract. Despite several
trials no cytochrome <em>cd</em><span class="math inline">\(_1\)</span> was isolated from a DEAE-Sepharose ion-exchange column in
these experiments. By contrast the amounts of cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> and cytochrome <em>c</em>’ (proteins
that are normally observed during cytochrome <em>cd</em><span class="math inline">\(_1\)</span> purification) seemed to be close to or even
slightly higher than what would have been expected when taking into account the reduced
growth of the cells as compared to wild-type cultures. These results indicate that the Y25F
mutation resulted in a greatly attenuated level of expressed cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, perhaps as a
result of degradation of the protein. Unfortunately, this result does not allow a definite
conclusion as to whether Tyr-25 is essential for the enzymatic activity of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>,
since it was not possible to isolate sufficient quantities of the mutant enzyme for kinetic
analyses. However, as the mutant was expressed <em>in vivo</em> in <em>T. pantotropha</em>, it can be stated
that Tyr-25 is certainly a vital residue in some sense. One possible reason for the postulated
degradation could be that Tyr-25 plays an important structural role during assembly of the
enzyme. As well as ligating to the <em>d</em><span class="math inline">\(_1\)</span> haem iron, Tyr-25 is hydrogen-bonded via a water
molecule to the residues His-345 and His-388 in the oxidised structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>
<span class="citation">[<a href="#ref-baker_cytochrome_1997" role="doc-biblioref">262</a>]</span> and so is structurally important. It may be that some or all of the above
interactions are important during the assembly and folding of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <em>in vivo</em> and
that their absence in the Y25F mutant accounts for its degradation. However, an equally
valid argument is that the absence of enzymatic activity in the Y25F mutant leads to the
degradation of the enzyme as the cells experience increasingly non-physiological conditions
because of the accumulation of nitrite. Notably, Tyr-25 is disordered in the reduced structure
of cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, which is likely to be the physiologically-relevant form (Chapter 3) and so
at least some degree of complementation using pMMBY25F might have been expected, had
the Y25F mutant been active.</p>
<p>The growth of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] is essentially identical to that of the
deletion mutant alone and some interesting observations can be made concerning the
cytochrome content of the cells. First, a small <em>c</em>-type cytochrome was observed
(approximately 16 kDa on SDS-PAGE) which bound only weakly to the DEAE-Sepharose
column, stained poorly with Coomassie Blue and auto-oxidised following reduction using
sodium dithionite. This cytochrome has not been observed previously during purification of
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from wild-type cultures of <em>T. pantotropha</em> and its function is unknown. The
spectrum of the cytochrome did not correspond to any known published spectrum of <em>c</em>-type
cytochromes from <em>P. denitrificans</em> or other organisms, and so N-terminal sequencing would
be a useful first step in an attempt to identify this protein. It may be expressed as a
consequence of stress placed on the cells by the high nitrite concentration. Further evidence
that the mutant cells grown under these conditions are under some form of oxidative stress
was deduced from the elevated levels of cytochrome <em>c</em> peroxidase found in the cells.
Normally this protein is seen only as a small absorbance at 410 nm eluting just before
cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from the DEAE-Sepharose column when wild-type cells are grown
anaerobically with nitrate. However, in the mutant culture cytochrome <em>c</em> peroxidase was the
major <em>c</em>-type cytochrome. It was also notable that elevated levels of the small blue copper
protein pseudoazurin were present, giving a blue colour to the fractions in which the protein
eluted that was clearly visible to the eye. Although the yield of pseudoazurin from wild-type
T. pantotropha has been reported as 0.6 mg l<span class="math inline">\(^{-1}\)</span> <span class="citation">[<a href="#ref-moir_aspects_1993" role="doc-biblioref">238</a>]</span>, the protein has been
consistently difficult to detect during routine purification of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> over the course
of the present work. The amounts observed in this case seem especially significant when the
poor cell yield is considered. This observation may indicate that pseudoazurin is produced in
response to oxidative stress and/or it may be an electron donor to cytochrome <em>c</em> peroxidase in
an analogous role to the similar protein azurin <span class="citation">[<a href="#ref-ronnberg_reaction_1981" role="doc-biblioref">318</a>]</span>.</p>
<p>The complementation of the <em>nirS</em> deletion mutant using pMMBSEK has shown that
this is a viable expression system for cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, provided that the recombinant protein
is stable. Although the growth of the cells is poor the final optical density is still equivalent
to that of wild-type cells in early exponential phase and the yield of other cytochromes from
the ion-exchange column indicates that enough cytochrome <em>cd</em><span class="math inline">\(_1\)</span> could be obtained for further
studies, provided that it was not degraded. It may also be possible to increase the cell yield
by dialysis culture to remove the toxic nitrite produced or by simply growing larger culture
volumes. However, the potential instability of mutants such as Y25F presents a different
problem. As the mutant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> could not be isolated from scaled-up cultures, it was
not possible to find out whether Tyr-25 was an essential residue for catalysis. This question
might be answered using the reconstitution of the expressed semi-apo protein from <em>E. coli</em>
that was described in Chapter 4. In this case the assembly of the protein differs to that <em>in vivo</em>, in that
it does not depend on the presence of Tyr-25 to ligand with the <em>d</em><span class="math inline">\(_1\)</span> haem during synthesis and folding
because the <em>d</em><span class="math inline">\(_1\)</span> haem is not present. It may therefore be possible to
insert the haem by reconstitution into semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> Y25F and determine whether
the protein is catalytically active. The comments from Chapter 4 concerning improvements
to the expression system and standardisation of the reconstitution protocol are equally valid
here. Additionally, there are a large number of other potential targets for site-directed
mutagenesis that may still be tried using expression in <em>T pantotropha</em> <span class="math inline">\(\Delta7\)</span> or in <em>E. coli</em>.
These include His-17 which is the proximal ligand to the <em>c</em>-haem in the oxidised structure,
Met-106, the proximal <em>c</em>-haem ligand in the reduced structure and His-345 or His-388, which
are believed to act as proton donors during the dehydration of nitrite at the active site. The
first two of these lie in the 382 bp deleted region of the <em>nirS</em> gene and could be constructed in
much the same way as the Y25F mutant described in this chapter.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-moir_purification_1993" class="csl-entry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Moir JW, Baratta D, Richardson DJ, Ferguson SJ. The purification of a cd1-type nitrite reductase from, and the absence of a copper-type nitrite reductase from, the aerobic denitrifier <span>Thiosphaera</span> pantotropha; the role of pseudoazurin as an electron donor. European Journal of Biochemistry. 1993;212: 377–385. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb17672.x">10.1111/j.1432-1033.1993.tb17672.x</a></div>
</div>
<div id="ref-fulop_anatomy_1995" class="csl-entry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Fülöp V, Moir JW, Ferguson SJ, Hajdu J. The anatomy of a bifunctional enzyme: Structural basis for reduction of oxygen to water and synthesis of nitric oxide by cytochrome cd1. Cell. 1995;81: 369–377. doi:<a href="https://doi.org/10.1016/0092-8674(95)90390-9">10.1016/0092-8674(95)90390-9</a></div>
</div>
<div id="ref-moir_aspects_1993" class="csl-entry">
<div class="csl-left-margin">238. </div><div class="csl-right-inline">Moir J. Aspects of electron transport in <span>Thiosphaera</span> pantotropha and <span>Paracoccus</span> denitrificans. PhD thesis, University of Oxford. 1993. Available: <a href="https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635">https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635</a></div>
</div>
<div id="ref-steinrucke_genetics_1993" class="csl-entry">
<div class="csl-left-margin">257. </div><div class="csl-right-inline">Steinrücke P, Ludwig B. Genetics of <span>Paracoccus</span> denitrificans. FEMS microbiology reviews. 1993;10: 83–117. doi:<a href="https://doi.org/10.1016/0378-1097(93)90505-v">10.1016/0378-1097(93)90505-v</a></div>
</div>
<div id="ref-baker_cytochrome_1997" class="csl-entry">
<div class="csl-left-margin">262. </div><div class="csl-right-inline">Baker SC, Saunders NF, Willis AC, Ferguson SJ, Hajdu J, Fülöp V. Cytochrome cd1 structure: Unusual haem environments in a nitrite reductase and analysis of factors contributing to beta-propeller folds. Journal of Molecular Biology. 1997;269: 440–455. doi:<a href="https://doi.org/10.1006/jmbi.1997.1070">10.1006/jmbi.1997.1070</a></div>
</div>
<div id="ref-furste_molecular_1986" class="csl-entry">
<div class="csl-left-margin">271. </div><div class="csl-right-inline">Fürste JP, Pansegrau W, Frank R, Blöcker H, Scholz P, Bagdasarian M, et al. Molecular cloning of the plasmid <span>RP4</span> primase region in a multi-host-range <span class="nocase">tacP</span> expression vector. Gene. 1986;48: 119–131. doi:<a href="https://doi.org/10.1016/0378-1119(86)90358-6">10.1016/0378-1119(86)90358-6</a></div>
</div>
<div id="ref-van_spanning_genes_1991" class="csl-entry">
<div class="csl-left-margin">310. </div><div class="csl-right-inline">Spanning R van. Genes involved in respiration of <span>Paracoccus</span> denitrificans. PhD thesis, VU Amsterdam. 1991. Available: <a href="https://research.vu.nl/en/publications/genes-involved-in-respiration-of-paracoccus-denitrificans">https://research.vu.nl/en/publications/genes-involved-in-respiration-of-paracoccus-denitrificans</a></div>
</div>
<div id="ref-van_spanning_mutagenesis_1990" class="csl-entry">
<div class="csl-left-margin">311. </div><div class="csl-right-inline">Spanning RJ van, Wansell CW, Reijnders WN, Oltmann LF, Stouthamer AH. Mutagenesis of the gene encoding amicyanin of <span>Paracoccus</span> denitrificans and the resultant effect on methylamine oxidation. FEBS letters. 1990;275: 217–220. doi:<a href="https://doi.org/10.1016/0014-5793(90)81475-4">10.1016/0014-5793(90)81475-4</a></div>
</div>
<div id="ref-van_spanning_method_1991" class="csl-entry">
<div class="csl-left-margin">312. </div><div class="csl-right-inline">Van Spanning RJ, Wansell CW, Reijnders WN, Harms N, Ras J, Oltmann LF, et al. A method for introduction of unmarked mutations in the genome of <span>Paracoccus</span> denitrificans: Construction of strains with multiple mutations in the genes encoding periplasmic cytochromes c550, c551i, and c553i. Journal of Bacteriology. 1991;173: 6962–6970. doi:<a href="https://doi.org/10.1128/jb.173.21.6962-6970.1991">10.1128/jb.173.21.6962-6970.1991</a></div>
</div>
<div id="ref-vieira_puc_1982" class="csl-entry">
<div class="csl-left-margin">313. </div><div class="csl-right-inline">Vieira J, Messing J. The <span class="nocase">pUC</span> plasmids, an <span>M13mp7</span>-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene. 1982;19: 259–268. doi:<a href="https://doi.org/10.1016/0378-1119(82)90015-4">10.1016/0378-1119(82)90015-4</a></div>
</div>
<div id="ref-gilmour_spectroscopic_1993" class="csl-entry">
<div class="csl-left-margin">314. </div><div class="csl-right-inline">Gilmour R, Goodhew CF, Pettigrew GW, Prazeres S, Moura I, Moura JJ. Spectroscopic characterization of cytochrome c peroxidase from <span>Paracoccus</span> denitrificans. The Biochemical Journal. 1993;294 ( Pt 3): 745–752. doi:<a href="https://doi.org/10.1042/bj2940745">10.1042/bj2940745</a></div>
</div>
<div id="ref-van_der_oost_reversed_1995" class="csl-entry">
<div class="csl-left-margin">315. </div><div class="csl-right-inline">Oost J van der, Schepper M, Stouthamer AH, Westerhoff HV, Spanning RJ van, Gier JW de. Reversed electron transfer through the bc1 complex enables a cytochrome c oxidase mutant (delta aa3/cbb3) of <span>Paracoccus</span> denitrificans to grow on methylamine. FEBS letters. 1995;371: 267–270. doi:<a href="https://doi.org/10.1016/0014-5793(95)00900-t">10.1016/0014-5793(95)00900-t</a></div>
</div>
<div id="ref-van_spanning_isolation_1991" class="csl-entry">
<div class="csl-left-margin">316. </div><div class="csl-right-inline">Van Spanning RJ, Wansell CW, De Boer T, Hazelaar MJ, Anazawa H, Harms N, et al. Isolation and characterization of the <span class="nocase">moxJ</span>, <span class="nocase">moxG</span>, <span class="nocase">moxI</span>, and <span class="nocase">moxR</span> genes of <span>Paracoccus</span> denitrificans: Inactivation of <span class="nocase">moxJ</span>, <span class="nocase">moxG</span>, and <span class="nocase">moxR</span> and the resultant effect on methylotrophic growth. Journal of Bacteriology. 1991;173: 6948–6961. doi:<a href="https://doi.org/10.1128/jb.173.21.6948-6961.1991">10.1128/jb.173.21.6948-6961.1991</a></div>
</div>
<div id="ref-roufa_seqaid_1989" class="csl-entry">
<div class="csl-left-margin">317. </div><div class="csl-right-inline">Roufa D, Rhoads D. <span>SeqAid</span> <span>II</span>. Genetics Laboratory, Kansas State University, Kansas; 1989. Available: <a href="https://edutice.archives-ouvertes.fr/edutice-00001128/document">https://edutice.archives-ouvertes.fr/edutice-00001128/document</a></div>
</div>
<div id="ref-ronnberg_reaction_1981" class="csl-entry">
<div class="csl-left-margin">318. </div><div class="csl-right-inline">Ronnberg M, Araiso T, Ellfolk N, Dunford HB. The reaction between reduced azurin and oxidized cytochrome c peroxidase from <span>Pseudomonas</span> aeruginosa. The Journal of Biological Chemistry. 1981;256: 2471–2474. doi:<a href="https://doi.org/10.1016/S0021-9258(19)69805-4">10.1016/S0021-9258(19)69805-4</a></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-chapter05.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="7-chapter07.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
