<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Transcriptional analysis of the nirS gene from T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Transcriptional analysis of the nirS gene from T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Transcriptional analysis of the nirS gene from T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="6-chapter06.html"/>
<link rel="next" href="8-chapter08.html"/>
<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter07" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></h1>
<div id="introduction-4" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Introduction</h2>
<p>Transposon mutagenesis and the use of heterologous gene probes have led to the
identification of gene clusters containing over thirty genes involved in denitrification in <em>Ps. stutzeri</em> Zobell, <em>Ps. aeruginosa</em> and <em>P. denitrificans</em> <span class="citation">[<a href="#ref-zumft_enzyme_1997" role="doc-biblioref">167</a>]</span>. Some of these genes have been well characterised, in particular the structural genes that encode the
enzymes of denitrification and some of the accessory genes involved in their biosynthesis
such as those involved in the assembly of the <em>d</em><span class="math inline">\(_1\)</span> haem group in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> <span class="citation">[<a href="#ref-palmedo_resolution_1995" role="doc-biblioref">319</a>]</span>.
However, very little transcriptional analysis of denitrification gene operons has
been reported. Most of the work in this field has been performed in <em>Ps. stutzeri</em> Zobell. In
this organism, transcription start site analysis has been reported for the genes <em>fnrA</em> <span class="citation">[<a href="#ref-cuypers_anaerobic_1993" role="doc-biblioref">204</a>]</span>, <em>norCB</em> <span class="citation">[<a href="#ref-zumft_nitric_1994" role="doc-biblioref">320</a>]</span> and <em>nosZ</em> <span class="citation">[<a href="#ref-cuypers_multiple_1995" role="doc-biblioref">219</a>]</span> and Northern blotting has been used to determine the major transcripts from these genes. Northern
blotting of transcripts from the <em>nirS</em> gene region has also been reported <span class="citation">[<a href="#ref-zumft_diversity_1995" role="doc-biblioref">220</a>]</span>.
Similar analysis of the <em>nor</em> genes <span class="citation">[<a href="#ref-bartnikas_characterization_1997" role="doc-biblioref">321</a>]</span> and the <em>nirK</em> gene which encodes a
copper nitrite reductase <span class="citation">[<a href="#ref-tosques_characterization_1997" role="doc-biblioref">322</a>]</span> have been performed using <em>Rhodobacter sphaeroides</em>.
In other cases sequence analysis has been employed, as in the <em>nir</em> operon of <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-kawasaki_gene_1997" role="doc-biblioref">221</a>]</span>
to look for transcription terminator features that might indicate the size of gene transcripts. In <em>P. denitrificans</em> transcriptional analysis is very much
in its infancy; very few genes have had their transcription start sites determined and little is
known about the promoter regions of any gene family. More experimental data from <em>P. denitrificans</em> would be extremely valuable as much of the regulation of denitrification is
likely to occur at the level of transcription. Identification of promoters and the proteins that
interact with them could improve our understanding of how denitrifying organisms sense
environmental factors such as oxygen tension and nitrate concentration and co-ordinate the
expression of the many genes involved in denitrification.</p>
<p>The experiments described in this chapter had two objectives: first to measure the
size of the transcript from the <em>nirS</em> gene and second, to determine the transcription start site
using two different methods. Following discussion of the results an analysis of NNR
dependent genes is presented with the aim of discussing some of the possible ways in which
the transcription of the <em>nirS</em> gene may be regulated. Finally, some ideas for future
experiments in this area are discussed.</p>
</div>
<div id="results-4" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Results</h2>
<p>The starting material for the experiments described in this chapter was total RNA extracted
from <em>T. pantotropha</em> that had been grown anaerobically to mid-exponential phase. The
extraction method was based on a number of methods rather than on one specific protocol;
cell lysis under denaturing conditions and phenol-chloroform extraction was based on the
method of Chattopadhyay <em>et al</em>. (1993) <span class="citation">[<a href="#ref-chattopadhyay_inexpensive_1993" role="doc-biblioref">247</a>]</span>. A 50 ml culture typically yielded 150-200 <span class="math inline">\(\mu\)</span>g of
total RNA with a ratio of absorbances at A<span class="math inline">\(_{260}\)</span>:A<span class="math inline">\(_{280}\)</span> of 1.5-1.9, which is considered to be of
sufficient quality for further work <span class="citation">[<a href="#ref-chirgwin_isolation_1979" role="doc-biblioref">323</a>]</span>. The extracted total RNA was
separated using denaturing formaldehyde agarose gel electrophoresis to determine whether
degradation had occurred during the extraction procedure. Figure 7.1 shows a gel containing
a sample of RNA from <em>T. pantotropha</em> that has been run alongside a total RNA extract from
<em>Escherichia coli</em> (prepared in the same way as that from T. pantotropha) and a sample of
total RNA from human cells (obtained from Dr. Stuart Wilson, University of Oxford).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-1"></span>
<img src="figure/fig7-1.png" alt="Analysis of total RNA samples from human cells, _E. coli_ and _T. pantotropha_ using denaturing agarose gel electrophoresis. Total RNA samples from human HeLa cells (lane 1, 10 $\mu$g, obtained from Dr. Stuart Wilson, University of Oxford), _E. coli_ (lane 2, 10 $\mu$g) and _T. pantotropha_ (lane 3, 15 $\mu$g from anaerobically grown cells and lane 4, 15 $\mu$g from aerobically grown cells) were denatured by heating in 50% formamide as described by Grierson (1990) [@griersen_notitle_1991] and electrophoresed in a 1% agarose-formaldehyde gel. The samples were stained by adding ethidium bromide at 0.033 $\mu$g/$\mu$l to the sample loading buffer. The sizes of RNA size standards (not visible on the photograph) are indicated at the left of the gel. Ribosomal RNA sizes (28S = 4718 bases, 23S = 2904 bases, 18S = 1874 bases, 16S = 1541 bases) are shown at the right of the gel." width="50%" />
<p class="caption">
Figure 7.1: Analysis of total RNA samples from human cells, <em>E. coli</em> and <em>T. pantotropha</em> using denaturing agarose gel electrophoresis. Total RNA samples from human HeLa cells (lane 1, 10 <span class="math inline">\(\mu\)</span>g, obtained from Dr. Stuart Wilson, University of Oxford), <em>E. coli</em> (lane 2, 10 <span class="math inline">\(\mu\)</span>g) and <em>T. pantotropha</em> (lane 3, 15 <span class="math inline">\(\mu\)</span>g from anaerobically grown cells and lane 4, 15 <span class="math inline">\(\mu\)</span>g from aerobically grown cells) were denatured by heating in 50% formamide as described by Grierson (1990) <span class="citation">[<a href="#ref-griersen_notitle_1991" role="doc-biblioref">324</a>]</span> and electrophoresed in a 1% agarose-formaldehyde gel. The samples were stained by adding ethidium bromide at 0.033 <span class="math inline">\(\mu\)</span>g/<span class="math inline">\(\mu\)</span>l to the sample loading buffer. The sizes of RNA size standards (not visible on the photograph) are indicated at the left of the gel. Ribosomal RNA sizes (28S = 4718 bases, 23S = 2904 bases, 18S = 1874 bases, 16S = 1541 bases) are shown at the right of the gel.
</p>
</div>
<p>Under these conditions two bright bands of ribosomal RNA and a faint smear of messenger
RNA should be observed. In the case of <em>E. coli</em> the 23S and 16S ribosomal RNA molecules
are 2904 <span class="citation">[<a href="#ref-brosius_complete_1980" role="doc-biblioref">325</a>]</span> and 1541 nucleotides long <span class="citation">[<a href="#ref-brosius_complete_1978" role="doc-biblioref">326</a>]</span>, respectively.
Undegraded RNA should exhibit a 2:1 staining ratio for the 23S and 16S rRNA species
<span class="citation">[<a href="#ref-chirgwin_isolation_1979" role="doc-biblioref">323</a>]</span>. The sample from <em>T. pantotropha</em> contained two strongly-staining
bands of RNA, but the upper band ran with the same mobility as the 16S RNA species
whereas the lower band was smaller, with an estimated size of approximately 900 bases.
This result is discussed further in the following section, but it results from the fragmentation
of the 23S rRNA molecule and in this case does not indicate that the RNA is degraded.</p>
<p>The length of the <em>nirS</em> transcript was determined using Northern blotting. For this
experiment 5, 10 and 20 <span class="math inline">\(\mu\)</span>g samples of total RNA from <em>T. pantotropha</em> were separated by
electrophoresis on a 1% formaldehyde-agarose gel and transferred to a positively charged
nylon membrane by capillary transfer using 20x SSC. The lane containing RNA size
standards was removed from the blot and stained using methylene blue. The transferred
RNA was then hybridised with a DNA probe of 141 bp in length, which had been generated
from <em>T. pantotropha</em> genomic DNA using a PCR reaction with the primers nirSF and 228R
(Table 3.1, Chapter 3). The probe was labelled with dUTP-digoxigenin-alkaline phosphatase
conjugate. Following hybridisation and washing, the hybridised probe was visualised
colorimetrically using a stain for the conjugated alkaline phosphatase activity (Figure 7.2).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-2"></span>
<img src="figure/fig7-2.png" alt="Northern blot of total RNA from anaerobically grown _T. pantotropha_ using a _nirS_-derived probe. 10 $\mu$g of RNA from _E. coli_ (lane 1) and 5, 10 and 20 $\mu$g of RNA from anaerobically grown _T. pantotropha_ (lanes 2-4) were separated by electrophoresis on a 1% agarose-formaldehyde gel transferred to a positive nylon membrane and hybridised with a probe to the _nirS_ gene, labelled with digoxigenin alkaline-phosphatase conjugate. After washing, the hybridised probe was detected colorimetrically. A single band with a mobility of approximately 1.85 kb was detected in the lanes containing RNA from _T. pantotropha_. The relative positions of the 23S (2904 bases) and 16S (1542 bases) ribosomal RNA species are shown at the left of the blot." width="50%" />
<p class="caption">
Figure 7.2: Northern blot of total RNA from anaerobically grown <em>T. pantotropha</em> using a <em>nirS</em>-derived probe. 10 <span class="math inline">\(\mu\)</span>g of RNA from <em>E. coli</em> (lane 1) and 5, 10 and 20 <span class="math inline">\(\mu\)</span>g of RNA from anaerobically grown <em>T. pantotropha</em> (lanes 2-4) were separated by electrophoresis on a 1% agarose-formaldehyde gel transferred to a positive nylon membrane and hybridised with a probe to the <em>nirS</em> gene, labelled with digoxigenin alkaline-phosphatase conjugate. After washing, the hybridised probe was detected colorimetrically. A single band with a mobility of approximately 1.85 kb was detected in the lanes containing RNA from <em>T. pantotropha</em>. The relative positions of the 23S (2904 bases) and 16S (1542 bases) ribosomal RNA species are shown at the left of the blot.
</p>
</div>
<p>In all lanes a single band with an approximate size of 1.85 kb was detected. Northern
blotting was also performed using chemiluminescent detection of the probe with the alkaline
phosphatase substrate CSPD. This technique is several times more sensitive than the
colorimetric detection method and was used to look for minor transcripts of different sizes.
However, only the same 1.85 kb band was detected in these experiments (Figure 7.3).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-3"></span>
<img src="figure/fig7-3.png" alt="Northern blot of total RNA from anaerobically grown _T. pantotropha_, using a _nirS_-derived probe together with chemiluminescent detection. 10 $\mu$g of RNA from _T. pantotropha_ grown anaerobically (lane 3) and aerobically (lane 2) and 10 $\mu$g of RNA from _E. coli_ (lane 1) were separated by electrophoresis on a 1% agarose-formaldehyde gel, transferred to positive nylon and hybridised to a probe for the _nirS_ gene as in Figure 7.2. The hybridised probe was detected using the chemiluminescent alkaline phosphatase substrate CSPD and the blot was exposed to X-ray film for 1 hour. A single band of approximate mobility 1.85 kb was detected in the lane containing RNA from anaerobically grown _T. pantotropha_. The very high stringency of the chemiluminescent detection protocol meant that no background was visible on the developed film; the outline of the original blot is the frame of the photograph. The relative positions of the 23S (2904 bases) and 16S (1541 bases) ribosomal RNAs species are shown at the left of the blot." width="50%" />
<p class="caption">
Figure 7.3: Northern blot of total RNA from anaerobically grown <em>T. pantotropha</em>, using a <em>nirS</em>-derived probe together with chemiluminescent detection. 10 <span class="math inline">\(\mu\)</span>g of RNA from <em>T. pantotropha</em> grown anaerobically (lane 3) and aerobically (lane 2) and 10 <span class="math inline">\(\mu\)</span>g of RNA from <em>E. coli</em> (lane 1) were separated by electrophoresis on a 1% agarose-formaldehyde gel, transferred to positive nylon and hybridised to a probe for the <em>nirS</em> gene as in Figure 7.2. The hybridised probe was detected using the chemiluminescent alkaline phosphatase substrate CSPD and the blot was exposed to X-ray film for 1 hour. A single band of approximate mobility 1.85 kb was detected in the lane containing RNA from anaerobically grown <em>T. pantotropha</em>. The very high stringency of the chemiluminescent detection protocol meant that no background was visible on the developed film; the outline of the original blot is the frame of the photograph. The relative positions of the 23S (2904 bases) and 16S (1541 bases) ribosomal RNAs species are shown at the left of the blot.
</p>
</div>
<p>Northern blotting was also attempted using a 428 bp <em>SphI</em> DNA fragment containing the ends
of the <em>nirE</em> (3’-end) and <em>nirC</em> (5’-end) genes as a probe, but no transcripts were detected in
these experiments (results not shown).</p>
<p>A primer extension experiment was performed using a sample of <em>T. pantotropha</em>
RNA in order to determine the transcription start point of the mRNA transcribed from the
<em>nirS</em> gene. Three oligonucleotides were synthesised, designated 284R, 242R and 228R, that
were homologous to the reverse strand of the early region of the <em>nirS</em> gene (Figure 7.4).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-4"></span>
<img src="figure/fig7-4.png" alt="Location of primers used in primer extension and 5'-RACE experiments to determine the transcription start site of the _T. pantotropha nirS_ gene. The upstream region of the _nirS_ gene is shown, covering the putative promoter region and the first 88 codons of the _nirS_ gene. The NNR box is in bold, the ATG start codon is underlined and a putative ribosome binding site is dash-underlined. The locations of the three reverse primers 284R, 242R and 228R are indicated by arrows." width="60%" />
<p class="caption">
Figure 7.4: Location of primers used in primer extension and 5’-RACE experiments to determine the transcription start site of the <em>T. pantotropha nirS</em> gene. The upstream region of the <em>nirS</em> gene is shown, covering the putative promoter region and the first 88 codons of the <em>nirS</em> gene. The NNR box is in bold, the ATG start codon is underlined and a putative ribosome binding site is dash-underlined. The locations of the three reverse primers 284R, 242R and 228R are indicated by arrows.
</p>
</div>
<p>The specificity of the oligonucleotides was analysed using PCR reactions with genomic DNA
from <em>T. pantotropha</em> and the nirSF primer (Chapter 3, Table 3.1), which gave specific
products of 197, 155 and 141 bp respectively (not shown). Early trials of the primer
extension experiment indicated that the 228R primer was most specific for annealing to
mRNA and so this primer was chosen for further work.</p>
<p>The 228R primer was labelled at the 5’-end with <span class="math inline">\(\gamma\)</span>-<span class="math inline">\(^{32}\)</span>P-ATP and approximately 10<span class="math inline">\(^5\)</span>
cpm of labelled primer was annealed to 40 <span class="math inline">\(\mu\)</span>g of total RNA from <em>T. pantotropha</em>. Avian
myeloblastosis virus (AMV) reverse transcriptase was then used to synthesise a cDNA
extending from the labelled primer to the 5’-end of the <em>nirS</em> mRNA molecule (the 3’-end of
the cDNA). The labelled cDNA was recovered by organic extraction and precipitation. It
was then electrophoresed on a standard 6% acrylamide sequencing gel alongside a
sequencing ladder generated using the 228R primer and the plasmid pTNIR3 (Chapter
4 Figure 4.2) as template DNA. The relevant portion of this gel is shown in Figure 7.5.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-5"></span>
<img src="figure/fig7-5.png" alt="Primer extension product synthesised from _nirS_ mRNA, compared with a sequencing ladder generated from the _nirS_ gene using the same primer. 10 pmol of the primer 228R was labelled at the 5'-end using 50 $\mu$Ci of $\gamma$-$^{32}$P-ATP and approximately 10$^5$ cpm of the labelled primer was annealed to 40 $\mu$g of total RNA from anaerobically-grown _T. pantotropha_. cDNA was extended from the labelled primer-template using 200 U of MMLV reverse transcriptase. The labelled cDNA product was precipitated redissolved and run on a 6% acrylamide sequencing gel alongside a sequencing ladder generated from the plasmid pTNIR3 (Chapter 4) and the 228R primer. Part of the sequence is shown at the left of the sequencing ladder and the G residue indicating the transcription start site with an asterisk. The sequencing ladder is not of high quality but as described in the text, previous knowledge of the _nirS_ gene sequence and the results of the 5'-RACE experiment (Figure 7.10) allowed the transcription start site to be assigned unambiguously." width="60%" />
<p class="caption">
Figure 7.5: Primer extension product synthesised from <em>nirS</em> mRNA, compared with a sequencing ladder generated from the <em>nirS</em> gene using the same primer. 10 pmol of the primer 228R was labelled at the 5’-end using 50 <span class="math inline">\(\mu\)</span>Ci of <span class="math inline">\(\gamma\)</span>-<span class="math inline">\(^{32}\)</span>P-ATP and approximately 10<span class="math inline">\(^5\)</span> cpm of the labelled primer was annealed to 40 <span class="math inline">\(\mu\)</span>g of total RNA from anaerobically-grown <em>T. pantotropha</em>. cDNA was extended from the labelled primer-template using 200 U of MMLV reverse transcriptase. The labelled cDNA product was precipitated redissolved and run on a 6% acrylamide sequencing gel alongside a sequencing ladder generated from the plasmid pTNIR3 (Chapter 4) and the 228R primer. Part of the sequence is shown at the left of the sequencing ladder and the G residue indicating the transcription start site with an asterisk. The sequencing ladder is not of high quality but as described in the text, previous knowledge of the <em>nirS</em> gene sequence and the results of the 5’-RACE experiment (Figure 7.10) allowed the transcription start site to be assigned unambiguously.
</p>
</div>
<p>The sequencing reactions were not of high quality in this case and it was not possible to replicate
the reactions before the activity of the labelled cDNA had reached unusable levels.
However, it was clear that the labelled cDNA product ran with the same mobility as a
guanine residue, 29 residues upstream of the <em>nirS</em> ATG start codon on the sequencing ladder.
The NNR-binding site upstream of the <em>nirS</em> gene was also clearly visible on the sequencing
gel and as the segment of DNA has been sequenced many times previously during
sequencing of the <em>nirS</em> gene, it was possible to assign confidently a transcription start site for
the <em>nirS</em> mRNA, as shown in Figure 7.6.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-6"></span>
<img src="figure/fig7-6.png" alt="The upstream region of the _T. pantotropha nirS_ gene, showing the transcription start site as determined using primer extension and 5'-RACE. The transcription start site is a G residue (double-underlined) 29 bp upstream of the ATG start codon (single-underlined). Other features are the putative ribosome binding site (dotted underlined) and the NNR box (bold). The reverse primers used in each experiment are shown by arrows. The bases 10, 35 and 41 bp upstream of the transcription start site are also indicated." width="60%" />
<p class="caption">
Figure 7.6: The upstream region of the <em>T. pantotropha nirS</em> gene, showing the transcription start site as determined using primer extension and 5’-RACE. The transcription start site is a G residue (double-underlined) 29 bp upstream of the ATG start codon (single-underlined). Other features are the putative ribosome binding site (dotted underlined) and the NNR box (bold). The reverse primers used in each experiment are shown by arrows. The bases 10, 35 and 41 bp upstream of the transcription start site are also indicated.
</p>
</div>
<p>To confirm this result, the method of 5’-RACE <span class="citation">[<a href="#ref-frohman_rapid_1993" role="doc-biblioref">250</a>]</span> was employed to
provide an independent estimate of the transcription start site. The basis of the method is
outlined in Figure 7.7.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-7"></span>
<img src="figure/fig7-7.png" alt="Schematic diagram illustrating the principle of the 5'-RACE method. (a) A reverse primer, R1, is used to synthesise a cDNA molecule from mRNA. The 3'-end of the cDNA is the complement of the transcription start site. (b) A poly-A tail is added to the 3'-end of the cDNA. (c) This allows several PCR cycles using the reverse primer R1 and the forward primer QTF, which contains a poly-T sequence and binding sites for two further forward primers Q0F and Q1F. (d) PCR is performed using the forward primer QOF and a second reverse primer R2. (e) final PCR reaction is performed using Q1F and a third reverse primer, R3, to increase specific (f) The final PCR product is cloned and sequenced; the transcription start site corresponds to the first base after the poly-A region." width="60%" />
<p class="caption">
Figure 7.7: Schematic diagram illustrating the principle of the 5’-RACE method. (a) A reverse primer, R1, is used to synthesise a cDNA molecule from mRNA. The 3’-end of the cDNA is the complement of the transcription start site. (b) A poly-A tail is added to the 3’-end of the cDNA. (c) This allows several PCR cycles using the reverse primer R1 and the forward primer QTF, which contains a poly-T sequence and binding sites for two further forward primers Q0F and Q1F. (d) PCR is performed using the forward primer QOF and a second reverse primer R2. (e) final PCR reaction is performed using Q1F and a third reverse primer, R3, to increase specific (f) The final PCR product is cloned and sequenced; the transcription start site corresponds to the first base after the poly-A region.
</p>
</div>
<p>As in the primer extension, experiment an oligonucleotide (R1) is
first annealed to the specific mRNA and a cDNA is synthesised, the 3’-end of which
corresponds to the 5’-end of the mRNA transcript. A tail of dATP residues is then added to
the 3’-end of the cDNA using terminal deoxynucleotidyl transferase. This allows a set of
nested PCR reactions to be performed, using three forward primers (QTF, QOF and Q1F <span class="citation">[<a href="#ref-frohman_rapid_1993" role="doc-biblioref">250</a>]</span>)
and two further reverse primers, R2 and R3. The PCR product so generated
can be cloned and sequenced conventionally and the 5’-end of the mRNA corresponds to the
first residue seen after the poly-dATP tail on the sequencing ladder.</p>
<p>In the 5’-RACE experiment the primer 284R was annealed to <em>nirS</em> mRNA and
extended using Moloney murine leukaemia virus (MMLV) reverse transcriptase to generate
cDNA. Following initial PCR amplification using the QTF primer and 284R, nested PCR
reactions were carried out using first QOF plus 242R and then Q1F plus 228R An aliquot of
the last PCR reaction was electrophoresed on a 1.4% agarose gel (Figure 7.8).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-8"></span>
<img src="figure/fig7-8.png" alt="Analysis using agarose gel electrophoresis of the PCR product generated from _nirS_ mRNA by 5'-RACE. Following cDNA synthesis using the primer 284R (Figure 7.4) and the addition of a poly-A tail to the cDNA product, three PCR reactions were performed using the primer pairs QTF + 284R, QOF + 242R and Q1F + 228R (see Figure 7.7). 10 $\mu$l of the final PCR reaction were separated by electrophoresis on a 1.2% agarose gel. Lanes 1 and 2 show a single PCR product of approximately 200 bp in length. To optimise the PCR reaction different conditions were used in the last PCR reaction; lane 1 used a high-salt buffer and lane 2 used low-salt buffer. The lane labelled M contains DNA size standards, the sizes of some of which are indicated at the left of the gel." width="50%" />
<p class="caption">
Figure 7.8: Analysis using agarose gel electrophoresis of the PCR product generated from <em>nirS</em> mRNA by 5’-RACE. Following cDNA synthesis using the primer 284R (Figure 7.4) and the addition of a poly-A tail to the cDNA product, three PCR reactions were performed using the primer pairs QTF + 284R, QOF + 242R and Q1F + 228R (see Figure 7.7). 10 <span class="math inline">\(\mu\)</span>l of the final PCR reaction were separated by electrophoresis on a 1.2% agarose gel. Lanes 1 and 2 show a single PCR product of approximately 200 bp in length. To optimise the PCR reaction different conditions were used in the last PCR reaction; lane 1 used a high-salt buffer and lane 2 used low-salt buffer. The lane labelled M contains DNA size standards, the sizes of some of which are indicated at the left of the gel.
</p>
</div>
<p>After optimisation of the PCR reactions, a single PCR product of approximately 200 bp was
present in the sample. The PCR product was purified from the gel and ligated into
plasmid pGEM-T. The presence of the insert was confirmed, following blue-white selection,
by digestion using the restriction enzymes <em>SphI</em> and <em>NdeI</em> (Figure 7.9).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-9"></span>
<img src="figure/fig7-9.png" alt="Restriction digest analysis of the cloned PCR products generated from _nirS_ mRNA by 5'-RACE. The 200 bp PCR product generated by PCR using the primers Q1F and 284R (Figure 7.8) was cloned into the T-vector pGEM-T. Following blue-white selection, positive clones were digested using the enzymes _SphI_ and _NdeI_. These enzymes cut once each side of the multiple cloning site but not in the _nirS_ gene sequence and therefore were a good screen for the presence of a cloned insert. Three of the clones (lanes 1, 2 and 5) contained the expected 200 bp insert. the other clones (lanes 3 and 4) contained a smaller insert. Sequencing of the latter clones showed that the smaller insert originated in the coding region of the _nirS_ gene and therefore resulted for incomplete cDNA synthesis during 5'-RACE. The lanes labelled III and II contained DNA size standards, some of which are indicated to of the gel." width="50%" />
<p class="caption">
Figure 7.9: Restriction digest analysis of the cloned PCR products generated from <em>nirS</em> mRNA by 5’-RACE. The 200 bp PCR product generated by PCR using the primers Q1F and 284R (Figure 7.8) was cloned into the T-vector pGEM-T. Following blue-white selection, positive clones were digested using the enzymes <em>SphI</em> and <em>NdeI</em>. These enzymes cut once each side of the multiple cloning site but not in the <em>nirS</em> gene sequence and therefore were a good screen for the presence of a cloned insert. Three of the clones (lanes 1, 2 and 5) contained the expected 200 bp insert. the other clones (lanes 3 and 4) contained a smaller insert. Sequencing of the latter clones showed that the smaller insert originated in the coding region of the <em>nirS</em> gene and therefore resulted for incomplete cDNA synthesis during 5’-RACE. The lanes labelled III and II contained DNA size standards, some of which are indicated to of the gel.
</p>
</div>
<p>The digests revealed that some of the clones contained a smaller insert than expected. Digestion of the plasmids
using <em>BamHI</em> produced a single linear fragment of 3.2 kbp, consistent with the presence of a
single <em>BamHI</em> site in the early region of the <em>nirS</em> gene (not shown). One of each of the
plasmids containing the larger and smaller inserts was sequenced using the M13 universal
forward and reverse primers. The sequencing ladder generated from the plasmid containing
the larger insert is shown in Figure 7.10.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-10"></span>
<img src="figure/fig7-10.png" alt="Sequence of the cloned PCR product synthesised from _nirS_ mRNA by 5'-RACE. The PCR product generated using the primers Q1F and 228R (see text for details) was cloned into the vector pGEM-T (Figure 7.9) and sequenced using the M13F universal primer. The Q1F primer can be read from the sequence and is followed by 11 T residues, representing the poly-A tail that was added to the cDNA following its synthesis from the _nirS_ mRNA. The residue immediately following the poly-T sequence is a G residue (asterisked). This residue indicates the 5'-end of the mRNA and is the same residue as that identified by primer extension." width="60%" />
<p class="caption">
Figure 7.10: Sequence of the cloned PCR product synthesised from <em>nirS</em> mRNA by 5’-RACE. The PCR product generated using the primers Q1F and 228R (see text for details) was cloned into the vector pGEM-T (Figure 7.9) and sequenced using the M13F universal primer. The Q1F primer can be read from the sequence and is followed by 11 T residues, representing the poly-A tail that was added to the cDNA following its synthesis from the <em>nirS</em> mRNA. The residue immediately following the poly-T sequence is a G residue (asterisked). This residue indicates the 5’-end of the mRNA and is the same residue as that identified by primer extension.
</p>
</div>
<p>The Q1F primer sequence and poly-dATP tail were clearly visible. Following the poly-dATP tail was a guanine residue and comparison to the
sequence of the <em>nirS</em> gene showed that this was the same guanine residue detected by the
primer extension experiment. Sequence from the plasmid containing the smaller insert
started within the coding region of the <em>nirS</em> gene, indicating that the cDNA generated in this
case was truncated and had not reached the 5’-end of the transcript.</p>
</div>
<div id="discussion-4" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Discussion</h2>
<p>Isolation of prokaryotic RNA can be technically challenging due to the short half-life of
bacterial mRNA (typically 1-2 minutes <span class="citation">[<a href="#ref-runar_ra_characterization_1995" role="doc-biblioref">327</a>]</span>) and the problem of
contamination with both endogenous and exogenous RNases. The quality of isolated RNA
is determined by two main measurements: a ratio of absorbances at 260 and 280 nm of 1.7
2.0 and a 23S:16S rRNA intensity ratio of 2:1 when stained with ethidium bromide after
denaturing gel electrophoresis. Total RNA isolated from <em>T. pantotropha</em> contained two
major bands that stained with ethidium bromide in a 2:1 ratio, but comparison with RNA
from <em>E. coli</em> showed that the 23S rRNA band was largely absent in these samples, having
apparently degraded to a 16S rRNA sized species and a smaller species. However this is not
indicative of general degradation in the RNA samples. Specific degradation of the 23S
RNA molecule has been observed during RNA isolation from the bacterium <em>Rhodobacter capsulatus</em> <span class="citation">[<a href="#ref-yu_molecular_1982" role="doc-biblioref">328</a>]</span>,
an organism that is very closely related to <em>T. pantotropha</em>. In the latter case it was demonstrated that intact 23S rRNA could be isolated using sucrose
density gradient centrifugation and the conditions of gel electrophoresis (in particular a low
concentration of divalent cations) were responsible for the observed degradation.
Fragmentation of the 23S rRNA molecule has also been observed in <em>Rhodospirillum rubrum</em>
<span class="citation">[<a href="#ref-chow_properties_1976" role="doc-biblioref">329</a>]</span>, <em>Rhodobacter sphaeroides</em> <span class="citation">[<a href="#ref-marrs_23_1970" role="doc-biblioref">330</a>]</span> and <em>Agrobacterium tumefaciens</em> <span class="citation">[<a href="#ref-grienenberger_structure_1975" role="doc-biblioref">331</a>]</span>.
In the latter case degradation of the 23S rRNA is proposed to occur <em>in vivo</em> during maturation of the ribosome. Thus it is clear that
23S rRNA degradation is common to several members of the <span class="math inline">\(\alpha\)</span>-Proteobacteria, including
species that are closely related to <em>T. pantotropha</em>. The positive results of the experiments
described in this chapter also show that the <em>nirS</em> mRNA that was isolated cannot have been
severely degraded.</p>
<p>Northern blotting using a DNA probe for the <em>nirS</em> gene identified a single transcript
that was approximately 1.85 kb long. The coding region of the <em>nirS</em> gene is 1788 bp long
and in <em>P. denitrificans</em> PD1222 is followed by the genes <em>nirE</em> (864 bp), <em>nirC</em> (309 bp), <em>nirF</em>
(1155 bp) and <em>nirD</em> (645 bp) <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>. Preliminary sequencing and restriction
analysis of the <em>nir</em> locus in <em>T. pantotropha</em> indicates the presence of at least <em>nirE</em> and <em>nirC</em>.
The 1.85 kb mRNA species is therefore a <em>nirS</em> transcript. Knowledge of the <em>nirS</em> gene
transcription start site described later in this chapter allows an approximate placement of the
end of the <em>nirS</em> transcript between the <em>nirS</em> and <em>nirE</em> genes (Figure 7.11).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-11"></span>
<img src="figure/fig7-11.png" alt="Model for transcriptional termination of the _T. pantotropha nirS_ gene. The length of the _nirS_ mRNA is estimated at 1.85 kb by Northern blotting (Figure 7.2). Alignment of the mRNA with the _nirS_ and _nirE_ genes places the end of the mRNA between the two genes, just before a strong inverted repeat. The TGA stop codon of the _nirS_ gene is underlined. The repeat could form a stem-loop structure as shown in the diagram and is followed by a relatively AT-rich region. This feature strongly resembles _rho_-dependent terminators in other genes. A second inverted repeat of unknown function before the putative terminator is indicated by inward-facing arrows. The intergenic sequence is numbered according to the sequence presented in Chapter 3." width="60%" />
<p class="caption">
Figure 7.11: Model for transcriptional termination of the <em>T. pantotropha nirS</em> gene. The length of the <em>nirS</em> mRNA is estimated at 1.85 kb by Northern blotting (Figure 7.2). Alignment of the mRNA with the <em>nirS</em> and <em>nirE</em> genes places the end of the mRNA between the two genes, just before a strong inverted repeat. The TGA stop codon of the <em>nirS</em> gene is underlined. The repeat could form a stem-loop structure as shown in the diagram and is followed by a relatively AT-rich region. This feature strongly resembles <em>rho</em>-dependent terminators in other genes. A second inverted repeat of unknown function before the putative terminator is indicated by inward-facing arrows. The intergenic sequence is numbered according to the sequence presented in Chapter 3.
</p>
</div>
<p>An imperfect inverted repeat 18 bp long (described in Chapter 3) lies between these two genes This
feature was originally annotated as an oxygen responsive element in the <em>P. denitrificans</em>
PD1222 <em>nir</em> locus, but it bears little resemblance to other ORE features and the annotation
has subsequently been withdrawn <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>. It seems more likely that the
inverted repeat is involved in terminating the transcription of the <em>nirS</em> gene. The GC-rich
region can form a stem-loop structure as shown in Figure 7.11 and together with an AT-rich
region downstream of the repeat, the structure is very similar to a <em>rho</em>-independent
termination site <span class="citation">[<a href="#ref-farnham_model_1980" role="doc-biblioref">332</a>]</span>.</p>
<p>Transcription of the <em>nir</em> region in <em>T. pantotropha</em> thus appears to differ from that of
the <em>nir</em> genes in <em>Ps. stutzeri</em> Zobell and <em>Ps. aeruginosa</em>. In <em>Ps. stutzeri</em> Zobell two transcripts
containing either <em>nirS</em> or <em>nirS</em> plus <em>nirT</em> have been detected <span class="citation">[<a href="#ref-zumft_diversity_1995" role="doc-biblioref">220</a>]</span>, whilst in <em>Ps. aeruginosa</em>
no transcript analysis has been performed to date, although analysis of the <em>nir</em> region suggests a large single transcript containing the genes <em>nirSMCFDLGHJEN</em> terminated
by a stem-loop structure <span class="citation">[<a href="#ref-bartnikas_characterization_1997" role="doc-biblioref">321</a>]</span>. Chemiluminescent detection of the <em>nirS</em>
gene probe using a high-stringency protocol that is at least as sensitive as <span class="math inline">\(^{32}\)</span>P-based methods
<span class="citation">[<a href="#ref-engler-blum_reduction_1993" role="doc-biblioref">249</a>]</span> failed to detect any longer transcripts from <em>T. pantotropha</em> in the
present work. However, it is still possible that longer transcripts including the <em>nirS</em> gene are
present <em>in vivo</em>; they may be present at only very low levels or, alternatively, very large
transcripts may not be efficiently transferred to the nylon membrane prior to hybridisation.
Transfer of large mRNA molecules can be improved by partial hydrolysis prior to blotting
<span class="citation">[<a href="#ref-sambrook_molecular_1989" role="doc-biblioref">235</a>]</span> and it would be of interest to repeat the experiment using this
technique. Northern blotting was attempted using a labelled DNA probe derived from parts
of the <em>nirE</em> and <em>nirC</em> gene but no transcript was detected. It may be that the transcript
containing these genes was not detectable for the reasons described above. Alternatively the
probe may not have been of sufficient quality; although the <em>nirS</em> gene probe efficiently
detected the <em>nirS</em> transcript, single-stranded RNA probes are preferred for Northern blotting
<span class="citation">[<a href="#ref-melton_efficient_1984" role="doc-biblioref">333</a>]</span>. In future experiments it would be valuable to synthesise riboprobes
from the <em>nir</em> genes for Northern blotting.</p>
<p>The transcription start site of the <em>nirS</em> gene was identified as a guanine residue 29 bp
upstream of the <em>nirS</em> ATG start codon. The sequencing ladder used in the primer extension
experiment was not of high quality, but as described in the results section assignment of the
start site was possible with prior knowledge of the <em>nirS</em> gene sequence in the relevant region.
Additionally, the independent method of 5’-RACE gave the same result for the start site.
The guanine residue is a good candidate for the authentic transcription start site as mRNA
transcripts most often begin with a G or an A residue <span class="citation">[<a href="#ref-hawley_compilation_1983" role="doc-biblioref">334</a>]</span> and the
start site is -41.5 bp downstream of the centre of the NNR box. The spacing is the same as
that observed between the FNR-binding site and the transcription start site in <em>E. coli</em> genes
that are activated by FNR <span class="citation">[<a href="#ref-eiglmeier_molecular_1989" role="doc-biblioref">335</a>]</span>. Once the transcription start site of the
<em>nirS</em> gene had been identified it was possible to look for upstream sequence features that
might be involved in transcriptional regulation. The first observation is that there are no
sequences which resemble the -10 and -35 hexamers seen upstream of sigma-70 dependent
genes from other species of bacteria, particularly <em>E. coli</em>. In <em>E. coli</em> these regions are well
characterised and consist of the consensus sequence TTGACA-N<span class="math inline">\(_{16-18}\)</span>-TATAAT, where the
TTGACA sequence is centred at -35 and the TATAAT sequence (or Pribnow box) is centred
at -10 from the transcription start site <span class="citation">[<a href="#ref-hawley_compilation_1983" role="doc-biblioref">334</a>]</span>. The -35 and -10
sequences are involved in binding the <span class="math inline">\(\sigma^{70}\)</span> subunit of the RNA polymerase holoenzyme
<span class="citation">[<a href="#ref-siebenlist_e_1980" role="doc-biblioref">336</a>]</span>. There is evidence that <em>T. pantotropha</em> probably does contain a <span class="math inline">\(\sigma^{70}\)</span>
dependent RNA polymerase; a recent survey of genes from <em>P. denitrificans</em> and other <span class="math inline">\(\alpha\)</span>-Proteobacteria
in which the transcription start site is known has shown that a similar <span class="math inline">\(\sigma^{70}\)</span> consensus sequence can be discerned in some instances
<span class="citation">[<a href="#ref-baker_molecular_1998" role="doc-biblioref">337</a>]</span>.
Additionally, the gene <em>xylE</em> cloned in a broad host range vector under the control of the <span class="math inline">\(\sigma^{70}\)</span>-dependent
bacteriophage T4 gene 32 promoter was transcribed from an identical start site in a range of
Gram-negative bacteria, including <em>Paracoccus denitrificans</em> <span class="citation">[<a href="#ref-frey_bacteriophage_1988" role="doc-biblioref">338</a>]</span>, suggesting a
common mechanism of <span class="math inline">\(\sigma^{70}\)</span> recognition. In the case of the <em>nirS</em> gene it might be expected
that the RNA polymerase binding sites differ from those normally found in <span class="math inline">\(\sigma^{70}\)</span> dependent
genes as the NNR-binding site overlays the -35 region and the NNR protein contains a
putative site for interaction with RNA polymerase <span class="citation">[<a href="#ref-de_boer_regulation_1996" role="doc-biblioref">168</a>]</span>. However, due to the lack
of an obvious -10 site in the <em>nirS</em> gene it is concluded that transcription of the <em>nirS</em> gene is
probably independent of <span class="math inline">\(\sigma^{70}\)</span>. A less likely possibility is that the specificity of the <em>T. pantotropha</em> <span class="math inline">\(\sigma^{70}\)</span>
subunit differs from that of the E. coli <span class="math inline">\(\sigma^{70}\)</span> and can bind to a much weaker -10 consensus,
particularly when binding is enhanced by the NNR protein. Experiments in
<em>Ps. aeruginosa</em> using the ANR and FNR proteins indicated that the <em>Ps. aeruginosa</em> <span class="math inline">\(\sigma^{70}\)</span>-
dependent RNA polymerase may have different specificity from that of the <em>E. coli</em> enzyme
<span class="citation">[<a href="#ref-winteler_homologous_1996" role="doc-biblioref">203</a>]</span> and <em>in vitro</em> data also suggested that the purified polymerase may
be less stringent <span class="citation">[<a href="#ref-gao_rna_1991" role="doc-biblioref">339</a>]</span>.</p>
<p>An alternative -10/-35 promoter consensus sequence was proposed by Steinrücke and
Ludwig (1993) <span class="citation">[<a href="#ref-steinrucke_genetics_1993" role="doc-biblioref">257</a>]</span>, based on the analysis of several genes from <em>P. denitrificans</em> and closely-
related species. This consensus was TTCGGN-N<span class="math inline">\(_{16-20}\)</span>-GATNGG/C. A match to this, with
four of the six positions filled in each hexamer can be discerned in the <em>nirS</em> upstream
sequence if it is assumed that as in <em>E. coli</em> the -10 site is allowed to fall anywhere in the first
hexamer. However, this consensus is now thought unlikely to be accurate as very little of the
data was based on experimentally determined transcription start sites for <em>P. denitrificans</em>
genes. Furthermore, genes that encoded proteins from a range of metabolic activities were
included in the analysis, with little consideration of whether the genes were expressed
constitutively or under specific conditions.</p>
<p>The last point is very important as in order to analyse the transcription of the <em>nirS</em>
gene in a meaningful manner, it should really be compared with the transcription of other
NNR-dependent (or possibly FNR-dependent) genes from <em>P. denitrificans</em> or its close
relatives. Unfortunately no information concerning transcripts from the <em>nir</em> or <em>nor</em> genes in
<em>P. denitrificans</em> is available apart from that reported in this chapter, as most work in the area
has been carried out using <em>Ps. stutzeri</em> Zobell. However, it was possible to examine the
sequences of the NNR-dependent genes <em>nirS</em> and <em>norCB</em> from <em>P. denitrificans</em> and <em>T. pantotropha</em>
by aligning the NNR-binding sites of these sequences and looking for common
features lying between the NNR-binding site and the ATG start codon (Figure 7.12).</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-12"></span>
<img src="figure/fig7-12.png" alt="Alignments of the upstream regions of NNR-regulated genes from _P. denitrificans_ and _T. pantotropha_ showing regions of conserved sequence. Group 1 shows genes with the consensus hexamer TTGCGC in the upstream region. The _nirI_ gene is assumed to be regulated by NNR (see text for details). Also included are the non-NNR-regulated genes _nirE_ and _nnr_. Group 2 shows a consensus resembling the RpoN binding site in the _nirI_ and _nnr_ upstream regions. The NNR binding site is shown in bold, the ATG start codon is in bold-italic. In Group 1 the TTGCGC hexamer is underlined, in Group 2 residues matching the RpoN consensus are underlined. +1 indicates the transcription start site of the _nirS_ gene. A perfect 10 bp inverted repeat in the _nirS_ gene is indicated by inward-facing arrows. Sequence data for _P. denitrificans_ PD1222 and _P. denitrificans_ 12442 was taken from GenBank accession number U05002 and Ohshima et. al. (1993) [@ohshima_cloning_1993], respectively." width="75%" />
<p class="caption">
Figure 7.12: Alignments of the upstream regions of NNR-regulated genes from <em>P. denitrificans</em> and <em>T. pantotropha</em> showing regions of conserved sequence. Group 1 shows genes with the consensus hexamer TTGCGC in the upstream region. The <em>nirI</em> gene is assumed to be regulated by NNR (see text for details). Also included are the non-NNR-regulated genes <em>nirE</em> and <em>nnr</em>. Group 2 shows a consensus resembling the RpoN binding site in the <em>nirI</em> and <em>nnr</em> upstream regions. The NNR binding site is shown in bold, the ATG start codon is in bold-italic. In Group 1 the TTGCGC hexamer is underlined, in Group 2 residues matching the RpoN consensus are underlined. +1 indicates the transcription start site of the <em>nirS</em> gene. A perfect 10 bp inverted repeat in the <em>nirS</em> gene is indicated by inward-facing arrows. Sequence data for <em>P. denitrificans</em> PD1222 and <em>P. denitrificans</em> 12442 was taken from GenBank accession number U05002 and Ohshima et. al. (1993) <span class="citation">[<a href="#ref-ohshima_cloning_1993" role="doc-biblioref">140</a>]</span>, respectively.
</p>
</div>
<p>The <em>nirI</em> genes from these organisms were also included in this analysis as they may be regulated
by NNR (see below). Several regions of homology between some or all of the sequences are
apparent from this alignment. First, the <em>nirS</em> genes contain a perfect inverted repeat of 10 bp
which begins five base pairs after the last nucleotide of the NNR-binding site. These repeats
were noted during the sequencing of the <em>nirS</em> gene described in Chapter 3. Less well-defined
repeats can be discerned in some of the other sequences, but the repeats in the <em>nirS</em> sequences
stand out because they are perfect and because of a striking structural conservation; the first
and last dinucleotides in the repeat from the <em>T. pantotropha</em> sequence are TT and AA
respectively whereas those in the two sequences from the <em>P. denitrificans</em> <em>nirS</em> genes are GG
and CC. Thus although the sequences differ, a perfect repeat is conserved in each case. This
feature suggests a structural role for the repeat. A possible role is in the binding of
a transcriptional repressor protein which would serve to prevent transcription of the <em>nirS</em> gene
under non-denitrifying conditions. Repeats of similar size and structure that serve to bind
repressors are common to many prokaryotic genes <span class="citation">[<a href="#ref-schmitt_cloning_1994" role="doc-biblioref">340</a>]</span>.</p>
<p>The second striking feature in the upstream sequences is the presence of a
hexanucleotide with the sequence TTGCGC in the <em>nirI</em> and two of the <em>nirS</em> sequences.
sequence (TTGCCG) and in two sequences that are apparently not regulated by NNR: <em>nirE</em>
Similar sequences are seen in the <em>T. pantotropha</em> <em>nirS</em> gene sequence (TTCCGC), the <em>norCB</em>
(TTCGCG) and the <em>nnr</em> gene itself (TTGCGC). In the case of the <em>T. pantotropha</em> <em>nirS</em> gene
the hexanucleotide is centred at -16.5 relative to the transcription start site. This sequence
lies in a similar position relative to the NNR box in all of the NNR-regulated genes and the
conservation in terms of sequence and position strongly suggests a role for this region of
sequence. Closer examination of the region that includes the TTGCGC hexanucleotide
reveals some similarity to a family of promoters that are dependent on an alternative sigma
factor, designated variously as <span class="math inline">\(\sigma^{54}\)</span>, NtrA or RpoN. RpoN-dependent promoters were first
identified in genes involved in nitrogen fixation <span class="citation">[<a href="#ref-beynon_nif_1983" role="doc-biblioref">341</a>]</span> and have since been
identified in several organisms where they regulate a diverse range of functions
<span class="citation">[<a href="#ref-romermann_rpon-like_1989" role="doc-biblioref">342</a>]</span>. RpoN differs from the better-known <span class="math inline">\(\sigma^{70}\)</span>-dependent RNA
polymerase in that it requires an ATP-binding activator protein in order to initiate
transcription from an open promoter-polymerase complex <span class="citation">[<a href="#ref-kustu_expression_1989" role="doc-biblioref">343</a>]</span>. Additionally,
the consensus sequence that is recognised by RpoN differs from the -35/-10 <span class="math inline">\(\sigma^{70}\)</span> sequences
found in <em>E. coli</em> (Figure 7.12). The minimum requirement for an RpoN-dependent promoter
is a GC dinucleotide found 11-14 base pairs upstream of the transcription start site (most
often found at the -12 position) and an invariant GG dinucleotide 10 bp further upstream.
most often at the -24 position <span class="citation">[<a href="#ref-thony_2412_1989" role="doc-biblioref">344</a>]</span>. In Figure 7.12 it can be
seen that the regions upstream of <em>nirI</em> and <em>nnr</em> contain these dinucleotides with
the correct spacing relative to each other and some additional sequence conservation at other residues.
However, the <em>nirS</em>, <em>nirE</em> and <em>norCB</em> genes lack one of the guanine residues in the invariant
GG dinucleotide. Additionally, the GC dinucleotide begins at the -17 position in the <em>nirS</em>
genes which is rather far from the more normal -12 position.</p>
<p>It therefore seems unlikely that the sigma factor involved in transcription of the
denitrification genes is the RpoN protein. However, it is still plausible that a similar type of
protein might be involved. Putative RpoN binding sites have been identified in the upstream
regions of some genes involved in denitrification, including the <em>Ps. stutzeri</em> Zobell <em>nosZ</em> gene
<span class="citation">[<a href="#ref-cuypers_notitle_1992" role="doc-biblioref">345</a>]</span> and the <em>nnrR</em> gene of <em>R. sphaeroides</em> 2.43 <span class="citation">[<a href="#ref-tosques_cloning_1996" role="doc-biblioref">202</a>]</span>.
The effects on denitrification of knocking out the rpon gene in these organisms have yet to
be investigated, although it is known that disruption of the <em>rpoN</em> gene in <em>Ps. aeruginosa</em> has
no effect on denitrification <span class="citation">[<a href="#ref-totten_rpon_1990" role="doc-biblioref">346</a>]</span>. In contrast, disruption of the <em>rpoN</em>
(formerly <em>hno</em>) gene in <em>Alcaligenes eutrophus</em> exerts a wide-ranging pleiotropic effect
including the loss of denitrification <span class="citation">[<a href="#ref-roemermann_pleotropic_1988" role="doc-biblioref">347</a>]</span>. Although <em>rpoN</em> disruption in
<em>Ps. aeruginosa</em> had no effect on denitrification, it is known that there are promoters in this
organism which resemble RpoN-dependent promoters, in that they possess the GG-N<span class="math inline">\(_{10}\)</span>-GC
motif but the GC dinucleotide is upstream of the normal -12 position and the genes are fully
functional in the absence of RpoN. Intriguingly, one such gene is that which encodes the
FNR homologue ANR in <em>Ps. aeruginosa</em> <span class="citation">[<a href="#ref-savioz_pseudomonas_1993" role="doc-biblioref">348</a>]</span> and it has been suggested that
these “RpoN-like” promoter regions depend on an alternative sigma factor. In
<em>Bradyrhizobium japonicum</em>, an organism quite closely related to <em>T. pantotropha</em> there are
two <em>rpoN</em> genes encoding the proteins RpoN, and RpoN2, with distinct specificities <span class="citation">[<a href="#ref-kullik_bradyrhizobium_1991" role="doc-biblioref">349</a>]</span>.
It is tempting to speculate that this is also the case in <em>T. pantotropha</em> and <em>P. denitrificans</em>
and that one of the RpoN homologues would be involved in denitrification.</p>
<p>Whatever the identity of the sigma factor involved in transcription of denitrifying
genes, a model for its interaction with the promoter and with other regulatory components
would be extremely valuable. At present genes encoding four types of regulatory protein
have been identified in the denitrification gene clusters of the principal denitrifying
organisms. The properties of these proteins are summarised in Table 7.1.</p>
<table>
<caption>
<span id="tab:table71">Table 7.1: </span>Proteins involved in regulating the expression of denitrification enzymes in <em>P. denitrificans</em>
</caption>
<thead>
<tr>
<th style="text-align:left;">
Regulatory protein
</th>
<th style="text-align:left;">
Cellular location and putative features
</th>
<th style="text-align:left;">
Comments and references
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
FnrP
</td>
<td style="text-align:left;">
Cytoplasmic. N-terminal iron-sulphur cluster, RNA polymerase contact site, dimerisation domain, helix-turn-helix DNA-binding motif.
</td>
<td style="text-align:left;">
<em>P. denitrificans</em> homologue of the <em>E. coli</em> FNR protein. Regulates transcription of membrane-bound nitrate reductase, cytochrome <em>c</em> peroxidase and <em>cbb</em><span class="math inline">\(_3\)</span>-type cytochrome <em>c</em> oxidase (van Spanning <em>et al</em>. 1997) <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">193</a>]</span>.
</td>
</tr>
<tr>
<td style="text-align:left;">
Nnr
</td>
<td style="text-align:left;">
Cytoplasmic. Similar to FnrP, but lacks N-terminal cysteine residues involved in binding iron-sulphur cluster.
</td>
<td style="text-align:left;">
Homologue of FNR, specifically regulates transcription of nitrite and nitric oxide reductases (van Spanning <em>et al</em>. 1995) <span class="citation">[<a href="#ref-van_spanning_nitrite_1995" role="doc-biblioref">195</a>]</span>.
</td>
</tr>
<tr>
<td style="text-align:left;">
NosR
</td>
<td style="text-align:left;">
Membrane-bound. Iron-sulphur cluster. Helix-turn-helix DNA-binding motif.
</td>
<td style="text-align:left;">
Identified in <em>P. denitrificans</em> but not characterised (Hoeren <em>et al</em>. 1993) <span class="citation">[<a href="#ref-hoeren_sequence_1993" role="doc-biblioref">96</a>]</span>. Homologue in <em>Ps. stutzeri</em> regulates transcription of nitrous oxide reductase (Cuypers <em>et al</em>. 1992) <span class="citation">[<a href="#ref-cuypers_nosr_1992" role="doc-biblioref">172</a>]</span>.
</td>
</tr>
<tr>
<td style="text-align:left;">
NirI
</td>
<td style="text-align:left;">
Membrane-bound. Iron-sulphur cluster. Less clear helix-turn-helix DNA-binding motif than NosR.
</td>
<td style="text-align:left;">
Homologue of NosR, but not well characterised. Regulates transcription of nitrite reductase (de Boer <em>et al</em>. 1994) <span class="citation">[<a href="#ref-de_boer_isolation_1994" role="doc-biblioref">141</a>]</span>.
</td>
</tr>
<tr>
<td style="text-align:left;">
NorQ
</td>
<td style="text-align:left;">
Cytoplasmic. Nucleotide-binding motif. Less clear DNA-binding motif than NirQ.
</td>
<td style="text-align:left;">
Identified in <em>P. denitrificans</em> but not well characterised. Mutants have reduced levels of nitrite and nitric oxide reductases (de Boer <em>et al</em>. 1996) <span class="citation">[<a href="#ref-de_boer_mutational_1996" role="doc-biblioref">171</a>]</span>. <em>Ps. stutzeri</em> homologue, NirQ, resembles NtrC family of transcriptional activators, mutants have reduced nitrite and nitric oxide reductase activity but similar level of the proteins (Jüngst and Zumft 1992) <span class="citation">[<a href="#ref-jungst_interdependence_1992" role="doc-biblioref">170</a>]</span>.
</td>
</tr>
</tbody>
</table>
<p>Four of these regulatory proteins (FnrP, NNR, NosR and Nirl) are known to be involved with the
transcription of one or more denitrification genes. NosR mutants in <em>Ps. stutzeri</em> fail to
synthesise nitrous oxide reductase <span class="citation">[<a href="#ref-cuypers_nosr_1992" role="doc-biblioref">172</a>]</span>, whilst NNR mutants in <em>P. denitrificans</em> fail
to synthesise the nitrite and nitric oxide reductases <span class="citation">[<a href="#ref-van_spanning_nitrite_1995" role="doc-biblioref">195</a>]</span>. FNR and its close homologues
appear to regulate the synthesis of nitrate and nitrous oxide reductases, as well as a number of other genes that are induced under microaerobic
conditions <span class="citation">[<a href="#ref-van_spanning_fnrp_1997" role="doc-biblioref">193</a>]</span>. The NirQ protein of <em>Ps. stutzeri</em> seems to be related
to the NtrC family of transcriptional regulators <span class="citation">[<a href="#ref-kustu_expression_1989" role="doc-biblioref">343</a>]</span> in that it contains a
nucleotide binding site and a DNA-binding helix-turn-helix motif, but disruption of the <em>nirQ</em>
gene gives rise to a mutant in which nitrite and nitric oxide reductases are present at normal
or slightly elevated levels but are not active <span class="citation">[<a href="#ref-jungst_interdependence_1992" role="doc-biblioref">170</a>]</span>. This suggests some
role for NirQ in the modulation of the enzyme activity of these two reductases. The situation
is different again for the NirQ homologue found in <em>P. denitrificans</em>, NorQ; disruption of this
gene substantially reduces the levels of nitrite and nitric oxide reductases <span class="citation">[<a href="#ref-de_boer_mutational_1996" role="doc-biblioref">171</a>]</span>.
This may be due to its differing location in the gene cluster; in <em>P. denitrificans</em> <em>norQ</em>
lies amongst the structural genes for nitric oxide reductase <span class="citation">[<a href="#ref-de_boer_mutational_1996" role="doc-biblioref">171</a>]</span> whereas in
<em>Ps. stutzeri</em> it is transcribed divergently upstream of the <em>nirS</em> gene <span class="citation">[<a href="#ref-jungst_interdependence_1992" role="doc-biblioref">170</a>]</span>.</p>
<p>In <em>P. denitrificans</em> NirI and NNR are the only proteins so far identified as having a
role in transcription of the nirS gene and an understanding of their action is far from
complete. NirI is predicted to be an integral inner membrane protein with seven
transmembrane helices and a putative iron-sulphur cluster <span class="citation">[<a href="#ref-de_boer_regulation_1996" role="doc-biblioref">168</a>]</span>. The latter feature
may bind a signal molecule, perhaps as part of a redox-sensing pathway. Although its
homologue NosR in <em>Ps. stutzeri</em> contains a putative DNA-binding helix-turn-helix motif
<span class="citation">[<a href="#ref-cuypers_nosr_1992" role="doc-biblioref">172</a>]</span> it is difficult to conceive of a membrane-bound protein acting directly
as a transcriptional activator, although such a role has also been suggested for the <em>E. coli</em>
FecI protein which is involved in iron dicitrate transport <span class="citation">[<a href="#ref-van_hove_novel_1990" role="doc-biblioref">350</a>]</span> and for the
ToxR protein of <em>Vibrio cholerae</em>, which activates the transcription of cholera toxin <span class="citation">[<a href="#ref-miller_cholera_1987" role="doc-biblioref">351</a>]</span>.
Therefore, NirI may be a transcriptional activator or it may be linked to a second
as yet unidentified protein which activates transcription from the <em>nirS</em> promoter, in a similar
fashion to the two component regulators such as the NarX-NarL system <span class="citation">[<a href="#ref-nohno_narx_1989" role="doc-biblioref">39</a>]</span>.
As the NNR protein probably binds to the <em>nirS</em> NNR-box (although this has not been
demonstrated experimentally) an interaction between NirI and NNR cannot be ruled out.</p>
<p>If NirI is involved in transcriptional activation of the <em>nirS</em> gene the role of the NNR
box upstream of the <em>nirS</em> gene needs to be clarified. The <em>nirI</em> gene itself could be under the
control of NNR as shown in Figure 7.13; due to the palindromic nature of the NNR-binding
site the <em>nirS</em> NNR-box could function on the opposite DNA strand to regulate the expression
of the divergently-transcribed <em>nirI</em> gene.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-13"></span>
<img src="figure/fig7-13.png" alt="Putative factors influencing the transcription of nitrite and nitric oxide reductases in _T. pantotropha_. In this diagram the Nnr protein is assumed to be a transcriptional activator for the _nirS_, _nirI_ and _norCB_ genes. The factor(s) that activate transcription of the _nnr_ gene are unknown. NirI also activates transcription of the _nirS_ gene, possibly through an as-yet unidentified cytoplasmic protein. RpoD represents an alternative sigma factor involved in transcription of the denitrification genes; it may not be the same protein in all cases. The circle labelled 'R' indicates an as-yet unidentified repressor protein that binds upstream of the _nirS_ gene. The NorQ protein is thought to modulate the activities of the nitrite and nitric oxide reductases; its mode of action is not known. More details are given in the text." width="75%" />
<p class="caption">
Figure 7.13: Putative factors influencing the transcription of nitrite and nitric oxide reductases in <em>T. pantotropha</em>. In this diagram the Nnr protein is assumed to be a transcriptional activator for the <em>nirS</em>, <em>nirI</em> and <em>norCB</em> genes. The factor(s) that activate transcription of the <em>nnr</em> gene are unknown. NirI also activates transcription of the <em>nirS</em> gene, possibly through an as-yet unidentified cytoplasmic protein. RpoD represents an alternative sigma factor involved in transcription of the denitrification genes; it may not be the same protein in all cases. The circle labelled ‘R’ indicates an as-yet unidentified repressor protein that binds upstream of the <em>nirS</em> gene. The NorQ protein is thought to modulate the activities of the nitrite and nitric oxide reductases; its mode of action is not known. More details are given in the text.
</p>
</div>
<p>The alignment in Figure 7.12 shows a number of conserved residues in the upstream regions of the <em>nirI</em> and <em>nirS</em> genes suggesting that this is
probably the case and strengthening the argument for an interaction between NNR and NirI.
Experiments in <em>R. sphaeroides</em> 2.4.3 have shown that the <em>nnrR</em> gene is expressed
constitutively at a low level throughout growth and indicate that the Nnr protein is
activated by an effector, which has been suggested to be nitric oxide. The initial NO signal
would be generated from a low level constitutive activity of the nitrite reductase <span class="citation">[<a href="#ref-tosques_cloning_1996" role="doc-biblioref">202</a>]</span>.
In <em>T. pantotropha</em> this is an attractive proposition for two reasons: first, NNR
seems to regulate specifically the nitrite and nitric oxide reductases which are involved with
the production and consumption of nitric oxide and second, these reductases probably only
function at very low oxygen concentrations. As nitric oxide is highly reactive with oxygen,
it would be diverted away from bioenergetically-useful reduction by nitric oxide reductase at
high oxygen tensions <span class="citation">[<a href="#ref-tosques_cloning_1996" role="doc-biblioref">202</a>]</span>. NirI and NNR might, therefore, function
together as a sensor of redox state or of nitric oxide concentration at lower oxygen
concentrations than FNR, to co-ordinate the expression of the nitrite and nitric oxide
reductases. Two important experiments would clarify the respective roles of NirI and NNR:
determination of whether NirI is expressed in an <em>nnr</em> mutant strain and conversely, whether
<em>nnr</em> and <em>norCB</em> which are NNR-dependent are expressed in a <em>nirI</em> mutant.</p>
<p>It is clear that much remains to be learned about the mechanisms of regulation during
denitrification in <em>P. denitrificans</em>, <em>T. pantotropha</em> and other micro-organisms. It is very
likely that further components of the regulatory pathway remain to be discovered and the
role of other environmental factors, such as N-oxides remains to be elucidated. For example,
it is known that nitrate and nitrite can influence the expression of denitrification genes, most
probably through a system similar to the NarX-NarL system of <em>E. coli</em>, although these genes
have not yet been discovered in any denitrifying bacterium. In terms of the transcription of
the <em>nirS</em> gene in <em>T. pantotropha</em>, a number of future experiments can be suggested. First, it
needs to be demonstrated that the <em>nirS</em> and <em>nirE</em> genes each have their own promoter as
suggested by the Northern blotting experiments. This should be achieved quite soon using
transcriptional or translational fusions with reporter genes downstream of the putative
promoter regions. The roles of individual residues in the promoter region can then be
investigated further, initially by deletions of whole parts of these regions and ultimately
using site-directed mutagenesis. The nature of the RNA polymerase also needs to be
elucidated; the <em>T. pantotropha</em> genome should be probed for the presence of one or more
copies of an RpoN-like gene and disruption of the gene(s) carried out to determine if this
sigma factor is involved in denitrification. Whatever the sigma factor proves to be, the
ultimate proof of its function will be to show that in conjunction with NNR it drives
transcription <em>in vitro</em> from the <em>nirS</em> promoter. DNA protection studies of the promoter region
using NNR and RNA polymerase will eventually show directly the location of the RNA
polymerase contact sites and may allow the definition of a consensus binding site for the
RNA polymerase.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-nohno_narx_1989" class="csl-entry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Nohno T, Noji S, Taniguchi S, Saito T. The <span class="nocase">narX</span> and <span class="nocase">narL</span> genes encoding the nitrate-sensing regulators of <span>Escherichia</span> coli are homologous to a family of prokaryotic two-component regulatory genes. Nucleic Acids Research. 1989;17: 2947–2957. doi:<a href="https://doi.org/10.1093/nar/17.8.2947">10.1093/nar/17.8.2947</a></div>
</div>
<div id="ref-hoeren_sequence_1993" class="csl-entry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Hoeren FU, Berks BC, Ferguson SJ, McCarthy JE. Sequence and expression of the gene encoding the respiratory nitrous-oxide reductase from <span>Paracoccus</span> denitrificans. <span>New</span> and conserved structural and regulatory motifs. European Journal of Biochemistry. 1993;218: 49–57. doi:<a href="https://doi.org/10.1111/j.1432-1033.1993.tb18350.x">10.1111/j.1432-1033.1993.tb18350.x</a></div>
</div>
<div id="ref-ohshima_cloning_1993" class="csl-entry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Ohshima T, Sugiyama M, Uozumi N, Iijima S, Kobayashi T. Cloning and sequencing of a gene encoding nitrite reductase from <span>Paracoccus</span> denitrificans and expression of the gene in <span>Escherichia</span> coli. Journal of Fermentation and Bioengineering. 1993;76: 82–88. doi:<a href="https://doi.org/10.1016/0922-338X(93)90061-C">10.1016/0922-338X(93)90061-C</a></div>
</div>
<div id="ref-de_boer_isolation_1994" class="csl-entry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Boer AP de, Reijnders WN, Kuenen JG, Stouthamer AH, Spanning RJ van. Isolation, sequencing and mutational analysis of a gene cluster involved in nitrite reduction in <span>Paracoccus</span> denitrificans. Antonie Van Leeuwenhoek. 1994;66: 111–127. doi:<a href="https://doi.org/10.1007/BF00871635">10.1007/BF00871635</a></div>
</div>
<div id="ref-zumft_enzyme_1997" class="csl-entry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline">Zumft WG, Körner H. Enzyme diversity and mosaic gene organization in denitrification. Antonie Van Leeuwenhoek. 1997;71: 43–58. doi:<a href="https://doi.org/10.1023/a:1000112008026">10.1023/a:1000112008026</a></div>
</div>
<div id="ref-de_boer_regulation_1996" class="csl-entry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline">Boer T de. Regulation of denitrification in <span>Paracoccus</span> denitrificans. PhD thesis, VU Amsterdam. 1996. Available: <a href="https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans">https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans</a></div>
</div>
<div id="ref-jungst_interdependence_1992" class="csl-entry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline">Jüngst A, Zumft WG. Interdependence of respiratory <span>NO</span> reduction and nitrite reduction revealed by mutagenesis of <span class="nocase">nirQ</span>, a novel gene in the denitrification gene cluster of <span>Pseudomonas</span> stutzeri. FEBS letters. 1992;314: 308–314. doi:<a href="https://doi.org/10.1016/0014-5793(92)81495-8">10.1016/0014-5793(92)81495-8</a></div>
</div>
<div id="ref-de_boer_mutational_1996" class="csl-entry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline">Boer AP de, Oost J van der, Reijnders WN, Westerhoff HV, Stouthamer AH, Spanning RJ van. Mutational analysis of the nor gene cluster which encodes nitric-oxide reductase from <span>Paracoccus</span> denitrificans. European Journal of Biochemistry. 1996;242: 592–600. doi:<a href="https://doi.org/10.1111/j.1432-1033.1996.0592r.x">10.1111/j.1432-1033.1996.0592r.x</a></div>
</div>
<div id="ref-cuypers_nosr_1992" class="csl-entry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline">Cuypers H, Viebrock-Sambale A, Zumft WG. <span>NosR</span>, a membrane-bound regulatory component necessary for expression of nitrous oxide reductase in denitrifying <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1992;174: 5332–5339. doi:<a href="https://doi.org/10.1128/jb.174.16.5332-5339.1992">10.1128/jb.174.16.5332-5339.1992</a></div>
</div>
<div id="ref-van_spanning_fnrp_1997" class="csl-entry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline">Van Spanning RJM, De Boer APN, Reijnders WNM, Westerhoff HV, Stouthamer AH, Van Der Oost J. <span>FnrP</span> and <span>NNR</span> of <em><span>Paracoccus</span> denitrificans</em> are both members of the <span>FNR</span> family of transcriptional activators but have distinct roles in respiratory adaptation in response to oxygen limitation. Molecular Microbiology. 1997;23: 893–907. doi:<a href="https://doi.org/10.1046/j.1365-2958.1997.2801638.x">10.1046/j.1365-2958.1997.2801638.x</a></div>
</div>
<div id="ref-van_spanning_nitrite_1995" class="csl-entry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline">Van Spanning RJ, De Boer AP, Reijnders WN, Spiro S, Westerhoff HV, Stouthamer AH, et al. Nitrite and nitric oxide reduction in <span>Paracoccus</span> denitrificans is under the control of <span>NNR</span>, a regulatory protein that belongs to the <span>FNR</span> family of transcriptional activators. FEBS letters. 1995;360: 151–154. doi:<a href="https://doi.org/10.1016/0014-5793(95)00091-m">10.1016/0014-5793(95)00091-m</a></div>
</div>
<div id="ref-tosques_cloning_1996" class="csl-entry">
<div class="csl-left-margin">202. </div><div class="csl-right-inline">Tosques IE, Shi J, Shapleigh JP. Cloning and characterization of <span class="nocase">nnrR</span>, whose product is required for the expression of proteins involved in nitric oxide metabolism in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1996;178: 4958–4964. doi:<a href="https://doi.org/10.1128/jb.178.16.4958-4964.1996">10.1128/jb.178.16.4958-4964.1996</a></div>
</div>
<div id="ref-winteler_homologous_1996" class="csl-entry">
<div class="csl-left-margin">203. </div><div class="csl-right-inline">Winteler HV, Haas D. The homologous regulators <span>ANR</span> of <span>Pseudomonas</span> aeruginosa and <span>FNR</span> of <span>Escherichia</span> coli have overlapping but distinct specificities for anaerobically inducible promoters. Microbiology (Reading, England). 1996;142 ( Pt 3): 685–693. doi:<a href="https://doi.org/10.1099/13500872-142-3-685">10.1099/13500872-142-3-685</a></div>
</div>
<div id="ref-cuypers_anaerobic_1993" class="csl-entry">
<div class="csl-left-margin">204. </div><div class="csl-right-inline">Cuypers H, Zumft WG. Anaerobic control of denitrification in <span>Pseudomonas</span> stutzeri escapes mutagenesis of an fnr-like gene. Journal of Bacteriology. 1993;175: 7236–7246. doi:<a href="https://doi.org/10.1128/jb.175.22.7236-7246.1993">10.1128/jb.175.22.7236-7246.1993</a></div>
</div>
<div id="ref-cuypers_multiple_1995" class="csl-entry">
<div class="csl-left-margin">219. </div><div class="csl-right-inline">Cuypers H, Jürgen B, Zumft WG. Multiple <span class="nocase">nosZ</span> promoters and anaerobic expression of nos genes necessary for <span>Pseudomonas</span> stutzeri nitrous oxide reductase and assembly of its copper centers. Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression. 1995;1264: 183–190. doi:<a href="https://doi.org/10.1016/0167-4781(95)00128-4">10.1016/0167-4781(95)00128-4</a></div>
</div>
<div id="ref-zumft_diversity_1995" class="csl-entry">
<div class="csl-left-margin">220. </div><div class="csl-right-inline">Zumft W. Diversity of gene organization and enzymes of denitrification. Beijerinck <span>Centennial</span> <span>Microbial</span> <span>Physiology</span> and <span>Gene</span> <span>Regulation</span>: <span>Emerging</span> <span>Principles</span> and <span>Applications</span>. Den Haag: Delft University Press; 1995. pp. 122–123. Available: <a href="https://repository.tudelft.nl/islandora/object/uuid:e30b10c4-0fad-4517-8874-5b3208e18f6e">https://repository.tudelft.nl/islandora/object/uuid:e30b10c4-0fad-4517-8874-5b3208e18f6e</a></div>
</div>
<div id="ref-kawasaki_gene_1997" class="csl-entry">
<div class="csl-left-margin">221. </div><div class="csl-right-inline">Kawasaki S, Arai H, Kodama T, Igarashi Y. Gene cluster for dissimilatory nitrite reductase (nir) from <span>Pseudomonas</span> aeruginosa: Sequencing and identification of a locus for heme d1 biosynthesis. Journal of Bacteriology. 1997;179: 235–242. doi:<a href="https://doi.org/10.1128/jb.179.1.235-242.1997">10.1128/jb.179.1.235-242.1997</a></div>
</div>
<div id="ref-sambrook_molecular_1989" class="csl-entry">
<div class="csl-left-margin">235. </div><div class="csl-right-inline">Sambrook J, Fritsch E, Maniatis T. Molecular cloning: <span>A</span> laboratory manual: <span>Vol</span>. 2. 2. ed. S.l.: Cold Spring Harbor; 1989. Available: <a href="https://www.worldcat.org/title/molecular-cloning-vol-2-a-laboratory-manual/oclc/730815487">https://www.worldcat.org/title/molecular-cloning-vol-2-a-laboratory-manual/oclc/730815487</a></div>
</div>
<div id="ref-chattopadhyay_inexpensive_1993" class="csl-entry">
<div class="csl-left-margin">247. </div><div class="csl-right-inline">Chattopadhyay N, Kher R, Godbole M. Inexpensive <span>SDS</span>/phenol method for <span>RNA</span> extraction from tissues. BioTechniques. 1993;15: 24–26. </div>
</div>
<div id="ref-engler-blum_reduction_1993" class="csl-entry">
<div class="csl-left-margin">249. </div><div class="csl-right-inline">Engler-Blum G, Meier M, Frank J, Müller GA. Reduction of background problems in nonradioactive northern and <span>Southern</span> blot analyses enables higher sensitivity than <span>32P</span>-based hybridizations. Analytical Biochemistry. 1993;210: 235–244. doi:<a href="https://doi.org/10.1006/abio.1993.1189">10.1006/abio.1993.1189</a></div>
</div>
<div id="ref-frohman_rapid_1993" class="csl-entry">
<div class="csl-left-margin">250. </div><div class="csl-right-inline">Frohman MA. Rapid amplification of complementary <span>DNA</span> ends for generation of full-length complementary <span>DNAs</span>: Thermal <span>RACE</span>. Methods in Enzymology. 1993;218: 340–356. doi:<a href="https://doi.org/10.1016/0076-6879(93)18026-9">10.1016/0076-6879(93)18026-9</a></div>
</div>
<div id="ref-steinrucke_genetics_1993" class="csl-entry">
<div class="csl-left-margin">257. </div><div class="csl-right-inline">Steinrücke P, Ludwig B. Genetics of <span>Paracoccus</span> denitrificans. FEMS microbiology reviews. 1993;10: 83–117. doi:<a href="https://doi.org/10.1016/0378-1097(93)90505-v">10.1016/0378-1097(93)90505-v</a></div>
</div>
<div id="ref-palmedo_resolution_1995" class="csl-entry">
<div class="csl-left-margin">319. </div><div class="csl-right-inline">Palmedo G, Seither P, Körner H, Matthews JC, Burkhalter RS, Timkovich R, et al. Resolution of the <span class="nocase">nirD</span> locus for heme d1 synthesis of cytochrome cd1 (respiratory nitrite reductase) from <span>Pseudomonas</span> stutzeri. European Journal of Biochemistry. 1995;232: 737–746. doi:<a href="https://doi.org/10.1111/j.1432-1033.1995.0737a.x">10.1111/j.1432-1033.1995.0737a.x</a></div>
</div>
<div id="ref-zumft_nitric_1994" class="csl-entry">
<div class="csl-left-margin">320. </div><div class="csl-right-inline">Zumft WG, Braun C, Cuypers H. Nitric oxide reductase from <span>Pseudomonas</span> stutzeri. <span>Primary</span> structure and gene organization of a novel bacterial cytochrome bc complex. European Journal of Biochemistry. 1994;219: 481–490. doi:<a href="https://doi.org/10.1111/j.1432-1033.1994.tb19962.x">10.1111/j.1432-1033.1994.tb19962.x</a></div>
</div>
<div id="ref-bartnikas_characterization_1997" class="csl-entry">
<div class="csl-left-margin">321. </div><div class="csl-right-inline">Bartnikas TB, Tosques IE, Laratta WP, Shi J, Shapleigh JP. Characterization of the nitric oxide reductase-encoding region in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1997;179: 3534–3540. doi:<a href="https://doi.org/10.1128/jb.179.11.3534-3540.1997">10.1128/jb.179.11.3534-3540.1997</a></div>
</div>
<div id="ref-tosques_characterization_1997" class="csl-entry">
<div class="csl-left-margin">322. </div><div class="csl-right-inline">Tosques IE, Kwiatkowski AV, Shi J, Shapleigh JP. Characterization and regulation of the gene encoding nitrite reductase in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1997;179: 1090–1095. doi:<a href="https://doi.org/10.1128/jb.179.4.1090-1095.1997">10.1128/jb.179.4.1090-1095.1997</a></div>
</div>
<div id="ref-chirgwin_isolation_1979" class="csl-entry">
<div class="csl-left-margin">323. </div><div class="csl-right-inline">Chirgwin JM, Przybyla AE, MacDonald RJ, Rutter WJ. Isolation of biologically active ribonucleic acid from sources enriched in ribonuclease. Biochemistry. 1979;18: 5294–5299. doi:<a href="https://doi.org/10.1021/bi00591a005">10.1021/bi00591a005</a></div>
</div>
<div id="ref-griersen_notitle_1991" class="csl-entry">
<div class="csl-left-margin">324. </div><div class="csl-right-inline">Griersen D. Gel <span>Electrophoresis</span> of <span>Nucleic</span> <span>Acids</span> <span>A</span> <span>Practical</span> <span>Approach</span> (<span>Second</span> <span>Edition</span>). 1991. p. 102. Available: <a href="https://linkinghub.elsevier.com/retrieve/pii/0307441291900416">https://linkinghub.elsevier.com/retrieve/pii/0307441291900416</a></div>
</div>
<div id="ref-brosius_complete_1980" class="csl-entry">
<div class="csl-left-margin">325. </div><div class="csl-right-inline">Brosius J, Dull TJ, Noller HF. Complete nucleotide sequence of a <span>23S</span> ribosomal <span>RNA</span> gene from <span>Escherichia</span> coli. Proceedings of the National Academy of Sciences of the United States of America. 1980;77: 201–204. doi:<a href="https://doi.org/10.1073/pnas.77.1.201">10.1073/pnas.77.1.201</a></div>
</div>
<div id="ref-brosius_complete_1978" class="csl-entry">
<div class="csl-left-margin">326. </div><div class="csl-right-inline">Brosius J, Palmer ML, Kennedy PJ, Noller HF. Complete nucleotide sequence of a <span>16S</span> ribosomal <span>RNA</span> gene from <span>Escherichia</span> coli. Proceedings of the National Academy of Sciences of the United States of America. 1978;75: 4801–4805. doi:<a href="https://doi.org/10.1073/pnas.75.10.4801">10.1073/pnas.75.10.4801</a></div>
</div>
<div id="ref-runar_ra_characterization_1995" class="csl-entry">
<div class="csl-left-margin">327. </div><div class="csl-right-inline">Runar Ra S, Saris PE. Characterization of prokaryotic <span class="nocase">mRNAs</span> by <span>RT</span>-<span>PCR</span>. BioTechniques. 1995;18: 792–795. </div>
</div>
<div id="ref-yu_molecular_1982" class="csl-entry">
<div class="csl-left-margin">328. </div><div class="csl-right-inline">Yu P-L, Hohn B, Falk H, Drews G. Molecular cloning of the ribosomal <span>RNA</span> genes of the photosynthetic bacterium <span>Rhodopseudomonas</span> capsulata. Molecular and General Genetics MGG. 1982;188: 392–398. doi:<a href="https://doi.org/10.1007/BF00330039">10.1007/BF00330039</a></div>
</div>
<div id="ref-chow_properties_1976" class="csl-entry">
<div class="csl-left-margin">329. </div><div class="csl-right-inline">Chow CT. Properties of ribonucleic acids from photosynthetic and heterotrophic <span>Rhodospirillum</span> rubrum. Canadian Journal of Microbiology. 1976;22: 228–236. doi:<a href="https://doi.org/10.1139/m76-031">10.1139/m76-031</a></div>
</div>
<div id="ref-marrs_23_1970" class="csl-entry">
<div class="csl-left-margin">330. </div><div class="csl-right-inline">Marrs B, Kaplan S. 23 s precursor ribosomal <span>RNA</span> of <span>Rhodopseudomonas</span> spheroides. Journal of Molecular Biology. 1970;49: 297–317. doi:<a href="https://doi.org/10.1016/0022-2836(70)90247-0">10.1016/0022-2836(70)90247-0</a></div>
</div>
<div id="ref-grienenberger_structure_1975" class="csl-entry">
<div class="csl-left-margin">331. </div><div class="csl-right-inline">Grienenberger JM, Simon D. Structure and biosynthesis of the ribosomal ribonucleic acids from the oncogenic bacterium <span>Agrobacterium</span> tumefaciens. The Biochemical Journal. 1975;149: 23–30. doi:<a href="https://doi.org/10.1042/bj1490023">10.1042/bj1490023</a></div>
</div>
<div id="ref-farnham_model_1980" class="csl-entry">
<div class="csl-left-margin">332. </div><div class="csl-right-inline">Farnham PJ, Platt T. A model for transcription termination suggested by studies on the trp attenuator in vitro using base analogs. Cell. 1980;20: 739–748. doi:<a href="https://doi.org/10.1016/0092-8674(80)90320-7">10.1016/0092-8674(80)90320-7</a></div>
</div>
<div id="ref-melton_efficient_1984" class="csl-entry">
<div class="csl-left-margin">333. </div><div class="csl-right-inline">Melton DA, Krieg PA, Rebagliati MR, Maniatis T, Zinn K, Green MR. Efficient in vitro synthesis of biologically active <span>RNA</span> and <span>RNA</span> hybridization probes from plasmids containing a bacteriophage <span>SP6</span> promoter. Nucleic Acids Research. 1984;12: 7035–7056. doi:<a href="https://doi.org/10.1093/nar/12.18.7035">10.1093/nar/12.18.7035</a></div>
</div>
<div id="ref-hawley_compilation_1983" class="csl-entry">
<div class="csl-left-margin">334. </div><div class="csl-right-inline">Hawley DK, McClure WR. Compilation and analysis of <span>Escherichia</span> coli promoter <span>DNA</span> sequences. Nucleic Acids Research. 1983;11: 2237–2255. doi:<a href="https://doi.org/10.1093/nar/11.8.2237">10.1093/nar/11.8.2237</a></div>
</div>
<div id="ref-eiglmeier_molecular_1989" class="csl-entry">
<div class="csl-left-margin">335. </div><div class="csl-right-inline">Eiglmeier K, Honoré N, Iuchi S, Lin EC, Cole ST. Molecular genetic analysis of <span>FNR</span>-dependent promoters. Molecular Microbiology. 1989;3: 869–878. doi:<a href="https://doi.org/10.1111/j.1365-2958.1989.tb00236.x">10.1111/j.1365-2958.1989.tb00236.x</a></div>
</div>
<div id="ref-siebenlist_e_1980" class="csl-entry">
<div class="csl-left-margin">336. </div><div class="csl-right-inline">Siebenlist U, Simpson RB, Gilbert W. E. Coli <span>RNA</span> polymerase interacts homologously with two different promoters. Cell. 1980;20: 269–281. doi:<a href="https://doi.org/10.1016/0092-8674(80)90613-3">10.1016/0092-8674(80)90613-3</a></div>
</div>
<div id="ref-baker_molecular_1998" class="csl-entry">
<div class="csl-left-margin">337. </div><div class="csl-right-inline">Baker SC, Ferguson SJ, Ludwig B, Page MD, Richter OM, Spanning RJ van. Molecular genetics of the genus <span>Paracoccus</span>: Metabolically versatile bacteria with bioenergetic flexibility. Microbiology and molecular biology reviews: MMBR. 1998;62: 1046–1078. doi:<a href="https://doi.org/10.1128/MMBR.62.4.1046-1078.1998">10.1128/MMBR.62.4.1046-1078.1998</a></div>
</div>
<div id="ref-frey_bacteriophage_1988" class="csl-entry">
<div class="csl-left-margin">338. </div><div class="csl-right-inline">Frey J, Mudd EA, Krisch HM. A bacteriophage <span>T4</span> expression cassette that functions efficiently in a wide range of gram-negative bacteria. Gene. 1988;62: 237–247. doi:<a href="https://doi.org/10.1016/0378-1119(88)90562-8">10.1016/0378-1119(88)90562-8</a></div>
</div>
<div id="ref-gao_rna_1991" class="csl-entry">
<div class="csl-left-margin">339. </div><div class="csl-right-inline">Gao JG, Gussin GN. <span>RNA</span> polymerases from <span>Pseudomonas</span> aeruginosa and <span>Pseudomonas</span> syringae respond to <span>Escherichia</span> coli activator proteins. Journal of Bacteriology. 1991;173: 394–397. doi:<a href="https://doi.org/10.1128/jb.173.1.394-397.1991">10.1128/jb.173.1.394-397.1991</a></div>
</div>
<div id="ref-schmitt_cloning_1994" class="csl-entry">
<div class="csl-left-margin">340. </div><div class="csl-right-inline">Schmitt MP, Holmes RK. Cloning, sequence, and footprint analysis of two promoter/operators from <span>Corynebacterium</span> diphtheriae that are regulated by the diphtheria toxin repressor (<span>DtxR</span>) and iron. Journal of Bacteriology. 1994;176: 1141–1149. doi:<a href="https://doi.org/10.1128/JB.176.4.1141-1149.1994">10.1128/JB.176.4.1141-1149.1994</a></div>
</div>
<div id="ref-beynon_nif_1983" class="csl-entry">
<div class="csl-left-margin">341. </div><div class="csl-right-inline">Beynon J, Cannon M, Buchanan-Wollaston V, Cannon F. The nif promoters of <span>Klebsiella</span> pneumoniae have a characteristic primary structure. Cell. 1983;34: 665–671. doi:<a href="https://doi.org/10.1016/0092-8674(83)90399-9">10.1016/0092-8674(83)90399-9</a></div>
</div>
<div id="ref-romermann_rpon-like_1989" class="csl-entry">
<div class="csl-left-margin">342. </div><div class="csl-right-inline">Römermann D, Warrelmann J, Bender RA, Friedrich B. An <span class="nocase">rpoN</span>-like gene of <span>Alcaligenes</span> eutrophus and <span>Pseudomonas</span> facilis controls expression of diverse metabolic pathways, including hydrogen oxidation. Journal of Bacteriology. 1989;171: 1093–1099. doi:<a href="https://doi.org/10.1128/jb.171.2.1093-1099.1989">10.1128/jb.171.2.1093-1099.1989</a></div>
</div>
<div id="ref-kustu_expression_1989" class="csl-entry">
<div class="csl-left-margin">343. </div><div class="csl-right-inline">Kustu S, Santero E, Keener J, Popham D, Weiss D. Expression of sigma 54 (<span class="nocase">ntrA</span>)-dependent genes is probably united by a common mechanism. Microbiological Reviews. 1989;53: 367–376. doi:<a href="https://doi.org/10.1128/mr.53.3.367-376.1989">10.1128/mr.53.3.367-376.1989</a></div>
</div>
<div id="ref-thony_2412_1989" class="csl-entry">
<div class="csl-left-margin">344. </div><div class="csl-right-inline">Thony B, Hennecke H. The −24/−12 promoter comes of age. FEMS Microbiology Letters. 1989;63: 341–357. doi:<a href="https://doi.org/10.1111/j.1574-6968.1989.tb03404.x">10.1111/j.1574-6968.1989.tb03404.x</a></div>
</div>
<div id="ref-cuypers_notitle_1992" class="csl-entry">
<div class="csl-left-margin">345. </div><div class="csl-right-inline">Cuypers H, Zumft W. Pseudomonas: <span>Molecular</span> <span>Biology</span> and <span>Biotechnology</span>. American Society for Microbiology; 1992. pp. 188–197. Available: <a href="https://www.worldcat.org/title/pseudomonas-molecular-biology-and-biotechnology/oclc/25411667">https://www.worldcat.org/title/pseudomonas-molecular-biology-and-biotechnology/oclc/25411667</a></div>
</div>
<div id="ref-totten_rpon_1990" class="csl-entry">
<div class="csl-left-margin">346. </div><div class="csl-right-inline">Totten PA, Lara JC, Lory S. The <span class="nocase">rpoN</span> gene product of <span>Pseudomonas</span> aeruginosa is required for expression of diverse genes, including the flagellin gene. Journal of Bacteriology. 1990;172: 389–396. doi:<a href="https://doi.org/10.1128/jb.172.1.389-396.1990">10.1128/jb.172.1.389-396.1990</a></div>
</div>
<div id="ref-roemermann_pleotropic_1988" class="csl-entry">
<div class="csl-left-margin">347. </div><div class="csl-right-inline">Roemermann D, Lohmeyer M, Friedrich CG, Friedrich B. Pleotropic mutants from <span>Alcaligenes</span> eutrophus defective in the metabolism of hydrogen, nitrate, urea, and fumarate. Archives of Microbiology. 1988;149: 471–475. doi:<a href="https://doi.org/10.1007/BF00425590">10.1007/BF00425590</a></div>
</div>
<div id="ref-savioz_pseudomonas_1993" class="csl-entry">
<div class="csl-left-margin">348. </div><div class="csl-right-inline">Savioz A, Zimmermann A, Haas D. Pseudomonas aeruginosa promoters which contain a conserved <span>GG</span>-<span>N10</span>-<span>GC</span> motif but appear to be <span>RpoN</span>-independent. Molecular &amp; general genetics: MGG. 1993;238: 74–80. doi:<a href="https://doi.org/10.1007/BF00279533">10.1007/BF00279533</a></div>
</div>
<div id="ref-kullik_bradyrhizobium_1991" class="csl-entry">
<div class="csl-left-margin">349. </div><div class="csl-right-inline">Kullik I, Fritsche S, Knobel H, Sanjuan J, Hennecke H, Fischer HM. Bradyrhizobium japonicum has two differentially regulated, functional homologs of the sigma 54 gene (<span class="nocase">rpoN</span>). Journal of Bacteriology. 1991;173: 1125–1138. doi:<a href="https://doi.org/10.1128/jb.173.3.1125-1138.1991">10.1128/jb.173.3.1125-1138.1991</a></div>
</div>
<div id="ref-van_hove_novel_1990" class="csl-entry">
<div class="csl-left-margin">350. </div><div class="csl-right-inline">Van Hove B, Staudenmaier H, Braun V. Novel two-component transmembrane transcription control: Regulation of iron dicitrate transport in <span>Escherichia</span> coli <span>K</span>-12. Journal of Bacteriology. 1990;172: 6749–6758. doi:<a href="https://doi.org/10.1128/jb.172.12.6749-6758.1990">10.1128/jb.172.12.6749-6758.1990</a></div>
</div>
<div id="ref-miller_cholera_1987" class="csl-entry">
<div class="csl-left-margin">351. </div><div class="csl-right-inline">Miller VL, Taylor RK, Mekalanos JJ. Cholera toxin transcriptional activator <span class="nocase">toxR</span> is a transmembrane <span>DNA</span> binding protein. Cell. 1987;48: 271–279. doi:<a href="https://doi.org/10.1016/0092-8674(87)90430-2">10.1016/0092-8674(87)90430-2</a></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="6-chapter06.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="8-chapter08.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
