<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>B Classification of T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="B Classification of T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="B Classification of T. pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"/>
<link rel="next" href="references.html"/>
<script src="libs/header-attrs-2.10/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="classification-of-t.-pantotropha" class="section level1" number="10">
<h1><span class="header-section-number">B</span> Classification of <em>T. pantotropha</em></h1>
<p>Throughout the present work the principal organism under study has been referred to by the name <em>Thiosphaera pantotropha</em>. However, during the course of the project the taxonomic position of this organism has come under scrutiny by a number of workers. In particular, it has been suggested that the organism should be re-classified as <em>Paracoccus denitrificans</em> <span class="citation">[<a href="#ref-ludwig_transfer_1993" role="doc-biblioref">367</a>]</span>. This argument was based on two observations: first, the 16S rRNA sequence of <em>T. pantotropha</em> is identical to that of <em>P. denitrificans</em> strain LMG 4218 and second, 100% DNA homology is observed between these two organisms in hybridisation experiments. Additionally, some of the metabolic differences that originally distinguished <em>T. pantotropha</em> from <em>P. denitrificans</em> <span class="citation">[<a href="#ref-robertson_thiosphaera_1983" role="doc-biblioref">225</a>]</span>, particularly aerobic denitrification, appear to have become attenuated since the original isolation of <em>T. pantotropha</em>.</p>
<p>The taxonomic status of <em>T. pantotropha</em> therefore remains undecided, although some workers have recently adopted its transfer to <em>P. denitrificans</em>, based on the work of Ludwig <em>et al</em>. (1993) <span class="citation">[<a href="#ref-ludwig_transfer_1993" role="doc-biblioref">367</a>]</span>. However, in the present work the name <em>T. pantotropha</em> has been retained, to avoid confusion between previous work in which this name was used and the much larger body of literature that refers specifically to <em>P. denitrificans</em>.
However, recent work has shown that there is some confusion as to the history of various strains of <em>P. denitrificans</em> which are deposited in several culture collections around the world. The relationship between strains of <em>P. denitrificans</em> has been re-investigated by examining the <em>c</em>-type cytochrome profiles using SDS-PAGE and by analysis of the protein sequences of cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> from each strain <span class="citation">[<a href="#ref-goodhew_cytochromes_1996" role="doc-biblioref">238</a>]</span>. The analysis indicates that <em>P. denitrificans</em> LMG 4218 is closely related to <em>T. pantotropha</em>, but also that <em>P. denitrificans</em> LMG 4218 is not derived from the original type strain of <em>P. denitrificans</em> <span class="citation">[<a href="#ref-beijerinck_bildung_1910" role="doc-biblioref">368</a>]</span>, as was thought previously. Instead, <em>P. denitrificans</em> LMG 4218 is placed in a group that includes a later strain of <em>P. denitrificans</em>, <em>P. denitrificans</em> LMD 52.44. The revised grouping of the strains would affect the way in which they were sub-divided, were consideration to be given to the creation of separate species within the genus <em>Paracoccus</em>.</p>
<p>To address the latter question, fatty acid methyl ester analysis (FAMES) has been used <span class="citation">[<a href="#ref-thompson_variation_1997" role="doc-biblioref">224</a>]</span>. This technique uses gas chromatography to analyse the relative amounts of different fatty acids in the cell membranes of bacteria. Taxonomic classification based on these measurements has been used successfully <span class="citation">[<a href="#ref-rainey_genome_1994" role="doc-biblioref">369</a>]</span> and the method is particularly well-suited to the grouping of strains at the sub-species level <span class="citation">[<a href="#ref-thompson_subgrouping_1993" role="doc-biblioref">370</a>]</span>. The results are so far unpublished, but they broadly support the assertion that <em>P. denitrificans</em> LMD 52.44, <em>P. denitrificans</em> LMG 4218 and <em>T. pantotropha</em> form a separate group within the genus <em>Paracoccus</em>, and also correlate well with a recent analysis of 16S rRNA sequences which suggested that the organism <em>Thiobacillus versutus</em> should be transferred to <em>Paracoccus versutus</em> <span class="citation">[<a href="#ref-katayama_paracoccus_1995" role="doc-biblioref">371</a>]</span>). Although all of the <em>P. denitrificans</em> strains studied were very similar in many respects, Thompson <em>et al</em>. (1997) <span class="citation">[<a href="#ref-thompson_variation_1997" role="doc-biblioref">224</a>]</span> assert that there is sufficient variation within the genus to propose the creation of separate species names, and it is suggested that <em>T. pantotropha</em> be renamed <em>P. pantotrophus</em>, to reflect the findings and to promote continuity of the <em>pantotrophus</em> species name.</p>

<!--
The bib chunk below must go last in this document according to how R Markdown renders.  More info is at http://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html
-->
<!-- 
If you'd like to change the name of the bibliography to something else,
delete "References" and replace it.
-->
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-thompson_variation_1997" class="csl-entry">
<div class="csl-left-margin">224. </div><div class="csl-right-inline">Thompson I, Ferguson S, Baker S. Variation within <span>Paracoccus</span> denitrificans and revival of the species <span>Thiosphaera</span> pantotropha as <span>Paracoccus</span> pantotrophus comb. nov. 1997. </div>
</div>
<div id="ref-robertson_thiosphaera_1983" class="csl-entry">
<div class="csl-left-margin">225. </div><div class="csl-right-inline">Robertson LA, Kuenen J. Thiosphaera pantotropha gen. Nov. Sp. Nov., A <span>Facultatively</span> <span>Anaerobic</span>, <span>Facultatively</span> <span>Autotrophic</span> <span>Sulphur</span> <span>Bacterium</span>. Microbiology,. 1983;129: 2847–2855. doi:<a href="https://doi.org/10.1099/00221287-129-9-2847">10.1099/00221287-129-9-2847</a></div>
</div>
<div id="ref-goodhew_cytochromes_1996" class="csl-entry">
<div class="csl-left-margin">238. </div><div class="csl-right-inline">Goodhew CF, Pettigrew GW, Devreese B, Beeumen jozef, Spanning RJM, Baker SC, et al. The cytochromes <em>c</em> -550 of <em><span>Paracoccus</span> denitrificans</em> and <em><span>Thiosphaera</span> pantotropha</em> : A need for re-evaluation of the history of <em>paracoccus</em> cultures. FEMS Microbiology Letters. 1996;137: 95–101. doi:<a href="https://doi.org/10.1111/j.1574-6968.1996.tb08089.x">10.1111/j.1574-6968.1996.tb08089.x</a></div>
</div>
<div id="ref-ludwig_transfer_1993" class="csl-entry">
<div class="csl-left-margin">367. </div><div class="csl-right-inline">Ludwig W, Mittenhuber G, Friedrich CG. Transfer of <span>Thiosphaera</span> pantotropha to <span>Paracoccus</span> denitrificans. International Journal of Systematic Bacteriology. 1993;43: 363–367. doi:<a href="https://doi.org/10.1099/00207713-43-2-363">10.1099/00207713-43-2-363</a></div>
</div>
<div id="ref-beijerinck_bildung_1910" class="csl-entry">
<div class="csl-left-margin">368. </div><div class="csl-right-inline">Beijerinck M, Minkman D. Bildung und verbrauch von stickoxydul durch bakterien. Zentralbl Bakteriol II. 1910;25: 30–63. </div>
</div>
<div id="ref-rainey_genome_1994" class="csl-entry">
<div class="csl-left-margin">369. </div><div class="csl-right-inline">Rainey PB, Thompson IP, Palleroni NJ. Genome and fatty acid analysis of <span>Pseudomonas</span> stutzeri. International Journal of Systematic Bacteriology. 1994;44: 54–61. doi:<a href="https://doi.org/10.1099/00207713-44-1-54">10.1099/00207713-44-1-54</a></div>
</div>
<div id="ref-thompson_subgrouping_1993" class="csl-entry">
<div class="csl-left-margin">370. </div><div class="csl-right-inline">Thompson IP, Bailey MJ, Ellis RJ, Purdy KJ. Subgrouping of bacterial populations by cellular fatty acid composition. FEMS Microbiology Letters. 1993;102: 75–84. doi:<a href="https://doi.org/10.1111/j.1574-6968.1993.tb05798.x">10.1111/j.1574-6968.1993.tb05798.x</a></div>
</div>
<div id="ref-katayama_paracoccus_1995" class="csl-entry">
<div class="csl-left-margin">371. </div><div class="csl-right-inline">Katayama Y, Hiraishi A, Kuraishi H. Paracoccus thiocyanatus sp. Nov., A new species of thiocyanate-utilizing facultative chemolithotroph, and transfer of <span>Thiobacillus</span> versutus to the genus <span>Paracoccus</span> as <span>Paracoccus</span> versutus comb. Nov. With emendation of the genus. Microbiology (Reading, England). 1995;141 ( Pt 6): 1469–1477. doi:<a href="https://doi.org/10.1099/13500872-141-6-1469">10.1099/13500872-141-6-1469</a></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
