<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.34.2 and GitBook 2.6.7" />

  <meta property="og:title" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="1-chapter01.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t.-pantotropha.html"><a href="B-classification-of-t.-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></h1>
<p class="author"><em>Neil F.W. Saunders <br />St. Cross College</em></p>
<p class="date"><em>September 1997</em></p>
</div>
<div id="preliminary-content" class="section level1 unnumbered hasAnchor">
<h1>Preliminary Content<a href="index.html#preliminary-content" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>A thesis submitted to the board of the Faculty of Biological Sciences at the
University of Oxford in partial fulfillment of the requirements for the degree of
Doctor of Philosophy.</p>
<div id="abstract" class="section level2 unnumbered hasAnchor">
<h2>Abstract<a href="index.html#abstract" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The <em>nirS</em> gene, which encodes the enzyme cytochrome <em>cd</em><sub>1</sub> nitrite reductase in the denitrifying
bacterium <em>Thiosphaera pantotropha</em>, was cloned and sequenced. The sequence of this
cytochrome <em>cd</em><sub>1</sub> was found to be very similar to those of the proteins not only from the
closely-related organism <em>P. denitrificans</em> but also several other species of bacteria. However,
the N-terminal region of these proteins shows less similarity than the overall protein
sequence. Furthermore, sequence analysis in combination with spectroscopic and structural
data indicates that the ligands to the <em>c</em>- and <em>d</em><sub>1</sub>-haem groups are not identical in cytochromes
<em>cd</em><sub>1</sub> from different sources. Implications for the mechanism of the enzyme are discussed.</p>
<p>Three different approaches were taken to the problem of expressing recombinant
cytochrome <em>cd</em><sub>1</sub> from the cloned <em>nirS</em> gene. First, the gene was expressed in <em>Escherichia coli</em>,
yielding a semi-apo enzyme that contained only the <em>c</em>-type haem. The protein could be
reconstituted to some degree using purified haem <em>d</em><sub>1</sub>, producing an enzyme with some of the
spectroscopic features of the native holoenzyme and around 50% of the native nitrite
reductase activity. This approach should be of use in the future expression of site-directed
mutants of cytochrome <em>cd</em><sub>1</sub>, which would be structurally unstable or enzymically inactive if
expressed as the holoenzyme in a heterologous denitrifying host.</p>
<p>Second, the <em>T. pantotropha</em> <em>nirS</em> gene was expressed in the denitrifying bacterium <em>Ps. aeruginosa</em> which synthesises an endogenous cytochrome <em>cd</em><sub>1</sub>. Under the growth conditions employed, a mixture of holo- and semi-apo recombinant cytochrome <em>cd</em><sub>1</sub> was produced. Differences in the visible spectrum of the holoenzyme and attenuated nitrite reductase and cytochrome <em>c</em> oxidase activities suggested that it was not fully assembled in this system. However, the system may be of use in distinguishing site-directed mutants of the enzyme that are enzymically inactive from those that retain some activity.</p>
<p>Third, an unmarked deletion was constructed in the genomic copy of the <em>T. pantotropha</em> <em>nirS</em> gene which abolished the synthesis of cytochrome <em>cd</em><sub>1</sub>. It then proved possible to restore denitrifying growth to the mutant by complementation with the cloned wild-type <em>nirS</em> gene. However, complementation using a site-directed mutant of cytochrome <em>cd</em><sub>1</sub>, in which the <em>d</em><sub>1</sub> haem ligand Tyr-25 was replaced with phenylalanine failed to restore growth and the cells contained greatly attenuated levels of the mutant cytochrome <em>cd</em><sub>1</sub>. Possible reasons for this result are discussed. These data illustrate the problem of finding an expression system that can synthesise the holoenzyme but does not rely upon its activity to maintain the growth of the host organism.</p>
<p>The transcription start site of the <em>nirS</em> gene was determined using 5’-RACE and primer
extension. The <em>nirS</em> gene was found to be transcribed as a monocistronic mRNA.
Comparison of the upstream region of the <em>nirS</em> gene with other denitrification genes indicates
a common mechanism of transcriptional regulation and suggests that such genes may be
transcribed using a specific sigma factor.</p>
</div>
<div id="acknowledgements" class="section level2 unnumbered hasAnchor">
<h2>Acknowledgements<a href="index.html#acknowledgements" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>My thanks first to Stuart, for taking me on in the first place, organising the funding and for his advice and encouragement during the project, particularly in the writing of this thesis. Thanks are also due to the Wellcome Trust, for their absurdly generous studentship, to Graham Pettigrew, who got me into bacterial <em>c</em>-type cytochromes in the first place, and to Thon de Boer, for his cytochrome <em>cd</em><sub>1</sub> antibody and for the gift of pTNIR3, the plasmid that saved the project! Thanks too, to Val for a huge number of perfect primers and to Colin for excellent photography.</p>
<p>Lab 5-9 has been graced by a number of wonderful postdocs; my thanks especially to Simon, for his support throughout the project and for not laughing (too much) during my transformation (ho ho) to a molecular biologist. And a big thank-you to Dudley, for advice on almost everything and conversation during the late nights, long weekends and even the more normal times of day. Variety has certainly been the spice of life in 5-9 thanks to Dave, Ben, Yoshi, Fiona, Alrik, Neeti, Isobel, our latest arrivals and all our visiting students. Outside of 5-9, thanks especially to Pam, Vilmos and Janos, Andy, the 6th floor crowd and of course Gerry and Rachel for much-needed caffeine and sugar, with a smile.</p>
<p>On the brief occasions that I escaped to the world outside, the Oxford “1st wave”, John, Hannah, Jacqui, Caroline and Sara had a lot to answer for. Thanks too to the Oxford “2nd wave”, especially Mat and Lucy, Jem and Lisa, Ralph, Charlie, Craig, Mark and Amanda and of course Marian, for rescuing me and making me realise it wasn’t all bad! Most of all, thanks to the folks in Edinburgh, for great holidays, time spent amongst mountains, accommodation, beers, entertainment and empathy: Mark, Leo, Jo, Anita and Lorna, and to my fellow “refugees in England” or further abroad; Mary Jane, James and Nim, Marc and Karen, and Colin and Michelle. I’d never have made it without all of you. I must also thank Mark E. Smith and The Fall, for twenty years of brilliant records.</p>
<p>Last, but definitely not least, a big thank-you to my parents, for all their support over the last eight years. I guess this is for you.</p>
</div>
<div id="publications" class="section level2 unnumbered hasAnchor">
<h2>Publications<a href="index.html#publications" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Some of the work in this thesis has been published.</p>
<ol style="list-style-type: decimal">
<li><p>Baker, S.C., Saunders, N.F.W., Willis, A.C., Ferguson, S.J., Hajdu, J. and Fülöp, V. (1997). Cytochrome <em>cd</em><sub>1</sub> structure: unusual haem environments in a nitrite reductase and analysis of factors contributing to β-propeller folds. J. Mol. Biol. 269: 440-455.</p></li>
<li><p>Williams, P.A., Fülöp, V., Garman, E.F., Saunders, N.F.W., Ferguson, S.J. and Hajdu, J. (1997). Haem-ligand switching during catalysis in crystals of a nitrogen cycle enzyme. Nature 389: 406-412.</p></li>
</ol>
<p><strong>Conference abstract</strong></p>
<ol start="3" style="list-style-type: decimal">
<li>Baker, S.C., Saunders, N., Fülöp, V., Hajdu, J. and Ferguson, S.J. (1995). The structure of <em>cd</em><sub>1</sub> nitrite reductase and analysis of the nirS gene of the denitrifying bacterium <em>Thiosphaera pantotropha</em>. Poster presented at the Beijerinck Centennial Conference on Microbial Physiology and Gene Regulation: Emerging Principles and Applications. 10-14 December 1995, The Hague, The Netherlands.</li>
</ol>
<p><strong>Publications not directly related to the present work</strong></p>
<ol start="4" style="list-style-type: decimal">
<li><p>Goodhew, C.F., Pettigrew, G.W., Devreese, B., Beeumen, J.V., van Spanning, R.J.M., Baker, S.C., Saunders, N.F.W., Ferguson, S.J. and Thompson, I.P. (1996). The cytochromes <em>c</em>-550 of <em>Paracoccus denitrificans</em> and <em>Thiosphaera pantotropha</em>: a need for re-evaluation of the history of <em>Paracoccus cultures</em>. FEMS Microbiol. Letts. 137: 95-101.</p></li>
<li><p>Page, M.D., Saunders, N.F.W. and Ferguson, S.J. (1997). Disruption of the <em>Pseudomonas aeruginosa dipZ</em> gene, encoding a putative protein-disulfide reductase, leads to partial pleiotropic deficiency in <em>c</em>-type cytochrome biogenesis. Microbiol. 143: 3111-3122.</p></li>
<li><p>Hu, W., Van Driessche, G., Devreese, B., Goodhew, C.F., McGinnity, D.F., Saunders, N., Fülöp, V., Pettigrew, G.W. and Van Beeumen, J.J. (1997). Structural characterization of <em>Paracoccus denitrificans</em> cytochrome <em>c</em> peroxidase and assignment of the low and high potential heme sites. Biochemistry 36: 7958-7966.</p></li>
</ol>
</div>
<div id="list-of-figures" class="section level2 unnumbered hasAnchor">
<h2>List of Figures<a href="index.html#list-of-figures" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Chapter</th>
<th align="left">Figure</th>
<th align="right">Title</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Chapter 1</td>
<td align="left"><a href="1-chapter01.html#fig:fig1-1">1.1</a></td>
<td align="right">The Nitrogen Cycle</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-2">1.2</a></td>
<td align="right"><nobr>The denitrifying electron transport chain of <em>Paracoccus denitrificans</em></nobr></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-3">1.3</a></td>
<td align="right">Structure of (a) <em>d</em><span class="math inline">\(_1\)</span> haem and (b) of protoporphyrin IX</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-4">1.4</a></td>
<td align="right">Crystal structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the oxidised form from <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-5">1.5</a></td>
<td align="right">Proposed mechanism for nitrite reduction by cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>Thiosphaera pantotropha</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-6">1.6</a></td>
<td align="right">Denitrification gene clusters in <em>Ps. stutzeri</em> Zobell, <em>Ps. aeruginosa</em> and <em>P. denitrificans</em> PD1222</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#fig:fig1-7">1.7</a></td>
<td align="right">Unrooted phylogenetic tree of the FNR family of transcriptional activator proteins</td>
</tr>
<tr class="even">
<td align="left">Chapter 3</td>
<td align="left"><a href="3-chapter03.html#fig:fig3-1">3.1</a></td>
<td align="right">PCR products used to sequence the <em>nirS</em> gene of <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-2">3.2</a></td>
<td align="right">Sequence of the <em>nirS</em> gene and its product, cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, from <em>T. pantotropha</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-3">3.3</a></td>
<td align="right">Predicted cleavage site of the periplasmic targeting sequence in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-4">3.4</a></td>
<td align="right">Multiple alignment of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-5">3.5</a></td>
<td align="right">Phylogenetic tree of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> sequences compared with a phylogenetic tree of 16S rRNA sequences from the same organisms</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-6">3.6</a></td>
<td align="right">Putative NNR binding sites in the upstream regions of the <em>nirS</em> gene aligned with the FNR consensus binding site</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-7">3.7</a></td>
<td align="right">Early region of the <em>nirS</em> gene from <em>Ps. stutzeri</em> ATCC 14405, showing primers used for re-sequencing and the sequence obtained</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-8">3.8</a></td>
<td align="right">Visible spectra of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> (a) in the oxidised and (b) in the reduced forms from <em>Ps. stutzeri</em> Zobell</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-9">3.9</a></td>
<td align="right">Visible spectra of (a) holocytochrome <em>cd</em><span class="math inline">\(_1\)</span> and (b) semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> in the reduced forms from <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#fig:fig3-10">3.10</a></td>
<td align="right">Alternative reaction schemes for the reduction of nitrite by cytochrome <em>cd</em><span class="math inline">\(_1\)</span> starting from the fully oxidised enzyme</td>
</tr>
<tr class="even">
<td align="left">Chapter 4</td>
<td align="left"><a href="4-chapter04.html#fig:fig4-1">4.1</a></td>
<td align="right">Physical map of the plasmid pTNIR3 produced using six restriction enzymes</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-2">4.2</a></td>
<td align="right">Location of denitrification genes of <em>T. pantotropha</em> on the plasmid pTNIR3 as determined by physical mapping and limited sequencing.</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-3">4.3</a></td>
<td align="right">Diagram of the broad host range expression vector pMMB67EH, showing relevant features</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-4">4.4</a></td>
<td align="right">Restriction digests illustrating the problem of vector carryover during cloning of the <em>T. pantotropha</em> <em>nirS</em> gene into pMMB67EH</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-5">4.5</a></td>
<td align="right">Restriction digest analysis of the plasmids pMMBSE and pMMBSEK</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-6">4.6</a></td>
<td align="right">Comparison of the aerobic growth of <em>E. coli</em>, <em>E. coli</em> [pMMB67EH], <em>E. coli</em> [pMMBSEK] and <em>E. coli</em> [pMMBSE]</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-7">4.7</a></td>
<td align="right">Comparison of the anaerobic growth of <em>E. coli</em> JM83 with that of <em>E. coli</em> [pMMB67EH]</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-8">4.8</a></td>
<td align="right">Comparison of the protein content of <em>E. coli</em> JM83 with that of <em>E. coli</em> [pMMB67EH], <em>E. coli</em> [pMMBSEK] and <em>E. coli</em> [pMMBSE].</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-9">4.9</a></td>
<td align="right">Western blot of protein samples from <em>E. coli</em> JM83, <em>E. coli</em> [pMMB67EH], <em>E. coli</em> [pMMBSEK] and <em>E. coli</em> [pMMBSE], probed using antibody to cytochrome <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-10">4.10</a></td>
<td align="right">Comparison of the <em>c</em>-type cytochrome content of <em>E. coli</em> [pMMBSE] grown under aerobic or anaerobic conditions</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-11">4.11</a></td>
<td align="right">Visible spectrum of soluble extract from <em>E. coli</em> [pMMBSE] grown anaerobically and induced with 1 mM IPTG</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-12">4.12</a></td>
<td align="right">Elution profile during ion exchange chromatography of soluble extract from anaerobically-grown <em>E. coli</em> [pMMBSE] induced using IPTG</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-13">4.13</a></td>
<td align="right">UV-visible spectrum of recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following ion-exchange chromatography</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-14">4.14</a></td>
<td align="right">Visible spectra of recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following reconstitution with purified haem <em>d</em><span class="math inline">\(_1\)</span> (a) as prepared, (b) following reduction using sodium dithionite, compared with (c) native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#fig:fig4-15">4.15</a></td>
<td align="right">Assays for nitrite reductase activity using reduced methyl viologen as an electron donor: (a) recombinant semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span>, (b) recombinant cytochrome <em>cd</em><span class="math inline">\(_1\)</span> following reconstitution with <em>d</em><span class="math inline">\(_1\)</span> haem, (c) native cytochrome <em>cd</em><span class="math inline">\(_1\)</span> purified from <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left">Chapter 5</td>
<td align="left"><a href="5-chapter05.html#fig:fig5-1">5.1</a></td>
<td align="right">Re-isolation of intact plasmid pMMBSE from <em>Ps. aeruginosa</em> PAO1</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-2">5.2</a></td>
<td align="right">Comparison of the growth curves of <em>Ps. aeruginosa</em> and <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-3">5.3</a></td>
<td align="right">Comparison of (a) the growth curve and (b) nitrite accumulation of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE]</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-4">5.4</a></td>
<td align="right">Comparison of the protein content of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-5">5.5</a></td>
<td align="right">Western blot of protein samples from <em>Ps. aeruginosa</em> or <em>Ps. aeruginosa</em> [pMMBSE] grown under different growth conditions, probed using antibody to cytochrome <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-6">5.6</a></td>
<td align="right">Comparison of the <em>c</em>-type cytochrome content of <em>Ps. aeruginosa</em> with that of <em>Ps. aeruginosa</em> [pMMBSE] under different growth conditions</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-7">5.7</a></td>
<td align="right">Elution profile during ion-exchange chromatography of soluble extract from <em>Ps. aeruginosa</em> [pMMBSE] grown semi-aerobically with nitrate and induced using IPTG</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-8">5.8</a></td>
<td align="right">Analysis of the <em>c</em>-type cytochrome content in the elution profile of soluble extract from <em>Ps. aeruginosa</em> [pMMBSE] following ion exchange chromatography</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-9">5.9</a></td>
<td align="right">UV-visible spectrum of concentrated fractions eluting under peak 2 following DEAE-Sepharose chromatography</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-10">5.10</a></td>
<td align="right">a) UV-visible spectrum of concentrated fractions eluting under peak 3 after DEAE-Sepharose chromatography, as isolated and (b) after the addition of excess sodium dithionite</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-11">5.11</a></td>
<td align="right">Difference spectra of (a) dithionite-reduced minus as-isolated spectra from peak 3 and (b) dithionite-reduced peak 3 minus peak 2</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#fig:fig5-12">5.12</a></td>
<td align="right">Cytochrome <em>c</em> oxidase and (b) nitrite reductase activities of the concentrated fractions from peak 3 following ion exchange chromatography</td>
</tr>
<tr class="odd">
<td align="left">Chapter 6</td>
<td align="left"><a href="6-chapter06.html#fig:fig6-1">6.1</a></td>
<td align="right">Construction of a kanamycin insertional mutant in the <em>nir</em> region of the <em>T. pantotropha</em> genome</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-2">6.2</a></td>
<td align="right">Construction of a <em>nirS</em> deletion mutant strain in <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-3">6.3</a></td>
<td align="right">Construction of the suicide plasmid pGRPN3K</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-4">6.4</a></td>
<td align="right">Restriction digest analysis of plasmid pGRPN3K</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-5">6.5</a></td>
<td align="right">Southern blot of genomic DNA from <em>T. pantotropha</em> K7 using the kanamycin resistance gene as a probe</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-6">6.6</a></td>
<td align="right">Construction of the suicide plasmid pRVS<span class="math inline">\(\Delta\)</span>nir</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-7">6.7</a></td>
<td align="right">Restriction digest analysis of plasmids pLN3N and pLN3N<span class="math inline">\(\Delta\)</span>nir</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-8">6.8</a></td>
<td align="right">Restriction digest analysis of plasmids pRVS1 and pRVS<span class="math inline">\(\Delta\)</span>nir</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-9">6.9</a></td>
<td align="right">PCR analysis of plasmid pRVS<span class="math inline">\(\Delta\)</span>nir showing a 382 bp deletion in the cloned <em>nirS</em> gene</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-10">6.10</a></td>
<td align="right">Southern blot of genomic DNA from wild-type <em>T. pantotropha</em>, <em>T. pantotropha</em> K7 and the <em>nirS</em> deletion mutants <span class="math inline">\(\Delta5\)</span>, <span class="math inline">\(\Delta7\)</span> and <span class="math inline">\(\Delta8\)</span> using a <em>nirS</em>-derived probe</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-11">6.11</a></td>
<td align="right">Comparison of the anaerobic growth of <em>T. pantotropha</em> with that of <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-12">6.12</a></td>
<td align="right">Comparison of (a) the anaerobic growth and (b) nitrite accumulation of <em>T. pantotropha</em> with that of <em>T. pantotropha</em> K7 and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-13">6.13</a></td>
<td align="right">Western blot of total soluble extracts from wild-type <em>T. pantotropha</em>, <em>T. pantotropha</em> K7, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span>, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] using antibody raised against <em>P. denitrificans</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-14">6.14</a></td>
<td align="right">Complementation of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> using plasmid pMMBSEK</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-15">6.15</a></td>
<td align="right">(a) Anaerobic growth of <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK] and (b) nitrite concentration in the medium during growth</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-16">6.16</a></td>
<td align="right">Construction of the plasmid pVA191-Y25F containing the Y25F mutation in the <em>nirS</em> gene of <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-17">6.17</a></td>
<td align="right">Construction of the expression plasmid pMMBY25F containing the Y25F mutation in the <em>nirS</em> gene of <em>T. pantotropha</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-18">6.18</a></td>
<td align="right">Restriction digest analysis of plasmid pBY25FK</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-19">6.19</a></td>
<td align="right">Restriction digest analysis of plasmid pMMBY25F using <em>MluI</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-20">6.20</a></td>
<td align="right">Comparison of (a) anaerobic growth and (b) nitrite accumulation in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F] compared with that of <em>T. pantotropha</em>, <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> and <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBSEK]</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-21">6.21</a></td>
<td align="right">Elution profile during ion exchange chromatography of soluble extract from anaerobically grown <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F]</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-22">6.22</a></td>
<td align="right">SDS-PAGE gel of fractions eluting from the DEAE-Sepharose column during chromatography of soluble extract from <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span> [pMMBY25F], stained for <em>c</em>-type cytochromes</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-23">6.23</a></td>
<td align="right">Visible spectra of an unidentified <em>c</em>-type cytochrome from <em>T. pantotropha</em> [pMMBY25F]</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-24">6.24</a></td>
<td align="right">Visible spectra of (a) cytochrome <em>c</em><span class="math inline">\(_{550}\)</span> and (b) cytochrome <em>c</em>’ from <em>T. pantotropha</em> [pMMBY25F]</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-25">6.25</a></td>
<td align="right">Visible spectra of cytochrome <em>c</em> peroxidase from <em>T. pantotropha</em> [pMMBY25F]</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-26">6.26</a></td>
<td align="right">Assay of cytochrome <em>c</em> peroxidase activity in peak 4 following ion exchange chromatography</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="6-chapter06.html#fig:fig6-27">6.27</a></td>
<td align="right">Visible spectrum of pseudoazurin from <em>T. pantotropha</em> [pMMBY25F]</td>
</tr>
<tr class="even">
<td align="left">Chapter 7</td>
<td align="left"><a href="7-chapter07.html#fig:fig7-1">7.1</a></td>
<td align="right">Analysis of total RNA samples from human cells, <em>E. coli</em> and <em>T. pantotropha</em> using denaturing agarose gel electrophoresis</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-2">7.2</a></td>
<td align="right">Northern blot of total RNA from anaerobically grown <em>T. pantotropha</em> using a <em>nirS</em>-derived probe</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-3">7.3</a></td>
<td align="right">Northern blot of total RNA from anaerobically grown <em>T. pantotropha</em>, using a <em>nirS</em>-derived probe together with chemiluminescent detection</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-4">7.4</a></td>
<td align="right">Location of primers used in primer extension and 5’-RACE experiments to determine the transcription start site of the <em>T. pantotropha nirS</em> gene</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-5">7.5</a></td>
<td align="right">Primer extension product synthesised from <em>nirS</em> mRNA, compared with a sequencing ladder generated from the <em>nirS</em> gene using the same primer</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-6">7.6</a></td>
<td align="right">The upstream region of the <em>T. pantotropha nirS</em> gene, showing the transcription start site as determined using primer extension and 5’-RACE</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-7">7.7</a></td>
<td align="right">Schematic diagram illustrating the principle of the 5’-RACE method</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-8">7.8</a></td>
<td align="right">Analysis using agarose gel electrophoresis of the PCR product generated from <em>nirS</em> mRNA by 5’-RACE</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-9">7.9</a></td>
<td align="right">Restriction digest analysis of the cloned PCR products generated from <em>nirS</em> mRNA by 5’-RACE</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-10">7.10</a></td>
<td align="right">Sequence of the cloned PCR product synthesised from <em>nirS</em> mRNA by 5’-RACE</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-11">7.11</a></td>
<td align="right">Model for transcriptional termination of the <em>T. pantotropha nirS</em> gene</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-12">7.12</a></td>
<td align="right">Alignments of the upstream regions of NNR-regulated genes from <em>P. denitrificans</em> and <em>T. pantotropha</em> showing regions of conserved sequence</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="7-chapter07.html#fig:fig7-13">7.13</a></td>
<td align="right">Putative factors influencing the transcription of nitrite and nitric oxide reductases in <em>T. pantotropha</em></td>
</tr>
<tr class="odd">
<td align="left">Appendix A</td>
<td align="left"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd_1-with-other-beta-propeller-structures.html#fig:figA-1">A.1</a></td>
<td align="right">Schematic diagram of the <span class="math inline">\(\beta\)</span>-propeller <em>d</em><span class="math inline">\(_1\)</span>-haem binding domain of <em>T. pantotropha</em> cytochrome <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
</tbody>
</table>
</div>
<div id="list-of-tables" class="section level2 unnumbered hasAnchor">
<h2>List of Tables<a href="index.html#list-of-tables" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<colgroup>
<col width="33%" />
<col width="33%" />
<col width="33%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Chapter</th>
<th align="left">Table</th>
<th align="right">Title</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Chapter 1</td>
<td align="left"><a href="1-chapter01.html#tab:table11">1.1</a></td>
<td align="right">Reduction potentials and thermodynamics of the reactions of denitrification</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="1-chapter01.html#tab:table12">1.2</a></td>
<td align="right">Standard reduction potentials reported by other workers of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
<tr class="odd">
<td align="left">Chapter 3</td>
<td align="left"><a href="3-chapter03.html#tab:table31">3.1</a></td>
<td align="right">Primers used to sequence the <em>nirS</em> gene from <em>T. pantotropha</em></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#tab:table32">3.2</a></td>
<td align="right">Comparison of inverted repeats in the DNA sequence of the <em>nir</em> region from <em>T. pantotropha</em> with those in the same region of <em>P. denitrificans</em> PD1222</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#tab:table33">3.3</a></td>
<td align="right">Matrix of similarities and identities between the cytochromes <em>cd</em><span class="math inline">\(_1\)</span></td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="3-chapter03.html#tab:table34">3.4</a></td>
<td align="right">Comparison of the physicochemical properties of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from different sources</td>
</tr>
<tr class="odd">
<td align="left">Chapter 4</td>
<td align="left"><a href="4-chapter04.html#tab:table41">4.1</a></td>
<td align="right">Plasmids referred to in this chapter</td>
</tr>
<tr class="even">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#tab:table42">4.2</a></td>
<td align="right">Different growth conditions used to compare the growth of <em>E. coli</em> JM83 with that of <em>E. coli</em> containing either pMMB67EH, pMMBSEK or pMMBSE</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="4-chapter04.html#tab:table43">4.3</a></td>
<td align="right">Comparison of the ratios of the absorbance maxima in (left) the reduced and (right) the oxidised states of native holo- and chemically prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> with those of the recombinant protein from <em>E. coli</em></td>
</tr>
<tr class="even">
<td align="left">Chapter 5</td>
<td align="left"><a href="5-chapter05.html#tab:table51">5.1</a></td>
<td align="right">Different growth conditions used to compare the growth of <em>Ps. aeruginosa</em> PAO1 with that of <em>Ps. aeruginosa</em> [pMMBSE]</td>
</tr>
<tr class="odd">
<td align="left"></td>
<td align="left"><a href="5-chapter05.html#tab:table52">5.2</a></td>
<td align="right">Comparison of the ratios of the absorbance maxima in the reduced state of native holo- and chemically-prepared semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em> with those of the partially purified recombinant protein from <em>Ps. aeruginosa</em> or <em>E. coli</em></td>
</tr>
<tr class="even">
<td align="left">Chapter 6</td>
<td align="left"><a href="6-chapter06.html#tab:table61">6.1</a></td>
<td align="right">Bacterial strains and plasmids referred to in this chapter</td>
</tr>
<tr class="odd">
<td align="left">Chapter 7</td>
<td align="left"><a href="7-chapter07.html#tab:table71">7.1</a></td>
<td align="right">Proteins involved in regulating the expression of denitrification enzymes in <em>P. denitrificans</em></td>
</tr>
</tbody>
</table>
</div>
<div id="list-of-abbreviations" class="section level2 unnumbered hasAnchor">
<h2>List of Abbreviations<a href="index.html#list-of-abbreviations" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<table>
<thead>
<tr class="header">
<th align="left">Abbreviation</th>
<th align="right">Meaning</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">AMV</td>
<td align="right">Avian myeloblastosis virus</td>
</tr>
<tr class="even">
<td align="left">ATP</td>
<td align="right">Adenosine triphosphate</td>
</tr>
<tr class="odd">
<td align="left">A<span class="math inline">\(_{xxx}\)</span></td>
<td align="right">Absorbance at wavelength XXX nm</td>
</tr>
<tr class="even">
<td align="left">bp</td>
<td align="right">Base pairs</td>
</tr>
<tr class="odd">
<td align="left">BSA</td>
<td align="right">Bovine serum albumin</td>
</tr>
<tr class="even">
<td align="left">DEAE</td>
<td align="right">Diethylaminoethyl</td>
</tr>
<tr class="odd">
<td align="left"><span class="math inline">\(\Delta\)</span><em>G</em>°’</td>
<td align="right">Standard Gibbs free energy change at pH 7.0</td>
</tr>
<tr class="even">
<td align="left">DIG</td>
<td align="right">Digoxigenin</td>
</tr>
<tr class="odd">
<td align="left">DIG-11-dUTP</td>
<td align="right">Digoxigenin-11-deoxyuridine triphosphate</td>
</tr>
<tr class="even">
<td align="left">DMSO</td>
<td align="right">Dimethylsulphoxide</td>
</tr>
<tr class="odd">
<td align="left">DNase</td>
<td align="right">Deoxyribonuclease</td>
</tr>
<tr class="even">
<td align="left">dNTP</td>
<td align="right">Deoxynucleotide triphosphate</td>
</tr>
<tr class="odd">
<td align="left">DTT</td>
<td align="right">Dithiothreitol</td>
</tr>
<tr class="even">
<td align="left">e<span class="math inline">\(^-\)</span></td>
<td align="right">Electron</td>
</tr>
<tr class="odd">
<td align="left"><em>E</em>°’</td>
<td align="right">Standard redox potential at pH 7.0</td>
</tr>
<tr class="even">
<td align="left">EPR</td>
<td align="right">Electron paramagnetic resonance</td>
</tr>
<tr class="odd">
<td align="left">IPTG</td>
<td align="right">Isopropylthio-<span class="math inline">\(\beta\)</span>-D-galactoside</td>
</tr>
<tr class="even">
<td align="left">LB</td>
<td align="right">Luria Bertani</td>
</tr>
<tr class="odd">
<td align="left">MCD</td>
<td align="right">Magnetic circular dichroism</td>
</tr>
<tr class="even">
<td align="left">MGD</td>
<td align="right">Molybdopterin guanine dinucleotide</td>
</tr>
<tr class="odd">
<td align="left">MMLV</td>
<td align="right">Moloney murine leukaemia virus</td>
</tr>
<tr class="even">
<td align="left">Na<span class="math inline">\(_2\)</span>EDTA/EDTA</td>
<td align="right">Disodium ethylenediamine tetraacetic acid</td>
</tr>
<tr class="odd">
<td align="left">NAD</td>
<td align="right">Nicotinamide adenine dinucleotide</td>
</tr>
<tr class="even">
<td align="left">NMR</td>
<td align="right">Nuclear magnetic resonance</td>
</tr>
<tr class="odd">
<td align="left">O.D. (xxx)</td>
<td align="right">Optical density at XXX nm</td>
</tr>
<tr class="even">
<td align="left">PBS</td>
<td align="right">Phosphate buffered saline</td>
</tr>
<tr class="odd">
<td align="left">PCR</td>
<td align="right">Polymerase chain reaction</td>
</tr>
<tr class="even">
<td align="left">PEG</td>
<td align="right">Polyethylene glycol</td>
</tr>
<tr class="odd">
<td align="left">RNase</td>
<td align="right">Ribonuclease</td>
</tr>
<tr class="even">
<td align="left">SDS-PAGE</td>
<td align="right">Sodium dodecyl sulphate polyacrylamide gel electrophoresis</td>
</tr>
<tr class="odd">
<td align="left">SSC</td>
<td align="right">0.15 M NaCl, 0.15 mM sodium citrate</td>
</tr>
<tr class="even">
<td align="left">TAE</td>
<td align="right">40 mM Tris-acetate pH 8.0, 1 mM EDTA</td>
</tr>
<tr class="odd">
<td align="left">TBE</td>
<td align="right">45 mM Tris-borate pH 8.3, 1 mM EDTA</td>
</tr>
<tr class="even">
<td align="left">TE</td>
<td align="right">10 mM Tris-HCl, 1 mM EDTA</td>
</tr>
<tr class="odd">
<td align="left">TEMED</td>
<td align="right">N, N, N’, N’-tetramethylethylenediamine</td>
</tr>
<tr class="even">
<td align="left">UQ/UQH<span class="math inline">\(_2\)</span></td>
<td align="right">Ubiquinone/ubiquinol</td>
</tr>
<tr class="odd">
<td align="left">X-Gal</td>
<td align="right">5-bromo-4-chloro-3-indolyl-<span class="math inline">\(\beta\)</span>-D-galactoside</td>
</tr>
</tbody>
</table>
</div>
<div id="quote" class="section level2 unnumbered hasAnchor">
<h2>Quote<a href="index.html#quote" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<blockquote>
<p>If we do not succeed, then we run the risk of failure”</p>
</blockquote>
<p><em>Former U.S. Vice-President Dan Quayle, 1990 (attrib.)</em></p>
<!-- The {.unnumbered} option here means that the introduction will be 
"Chapter 0." You can also use {-} for no numbers on chapters.
-->

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="1-chapter01.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
