<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha</title>
  <meta name="description" content="Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd\(_1\) nitrite reductase, from Thiosphaera pantotropha" />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Cloning, sequence analysis and studies on the expression of the nirS gene, encoding cytochrome cd_1 nitrite reductase, from Thiosphaera pantotropha" />
  
  
  

<meta name="author" content="Neil F.W. Saunders  St. Cross College" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="B-classification-of-t-pantotropha.html"/>

<script src="libs/header-attrs-2.9/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preliminary Content</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#publications"><i class="fa fa-check"></i>Publications</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-figures"><i class="fa fa-check"></i>List of Figures</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-tables"><i class="fa fa-check"></i>List of Tables</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#list-of-abbreviations"><i class="fa fa-check"></i>List of Abbreviations</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#quote"><i class="fa fa-check"></i>Quote</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-chapter01.html"><a href="1-chapter01.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#denitrification-and-the-nitrogen-cycle"><i class="fa fa-check"></i><b>1.1</b> Denitrification and the nitrogen cycle</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-importance-of-denitrification"><i class="fa fa-check"></i><b>1.1.1</b> The importance of denitrification</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-denitrifying-micro-organisms"><i class="fa fa-check"></i><b>1.2</b> The denitrifying micro-organisms</a></li>
<li class="chapter" data-level="1.3" data-path="1-chapter01.html"><a href="1-chapter01.html#organisation-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.3</b> Organisation of the denitrifying electron transport chain</a></li>
<li class="chapter" data-level="1.4" data-path="1-chapter01.html"><a href="1-chapter01.html#enzymes-of-the-denitrifying-electron-transport-chain"><i class="fa fa-check"></i><b>1.4</b> Enzymes of the denitrifying electron transport chain</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="1-chapter01.html"><a href="1-chapter01.html#the-membrane-bound-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.1</b> The membrane-bound nitrate reductase</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-periplasmic-nitrate-reductase"><i class="fa fa-check"></i><b>1.4.2</b> The periplasmic nitrate reductase</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-chapter01.html"><a href="1-chapter01.html#the-copper-nitrite-reductase"><i class="fa fa-check"></i><b>1.4.3</b> The copper nitrite reductase</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitric-oxide-reductase"><i class="fa fa-check"></i><b>1.4.4</b> The nitric oxide reductase</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-nitrous-oxide-reductase"><i class="fa fa-check"></i><b>1.4.5</b> The nitrous oxide reductase</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-chapter01.html"><a href="1-chapter01.html#small-electron-transfer-proteins"><i class="fa fa-check"></i><b>1.4.6</b> Small electron transfer proteins</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-chapter01.html"><a href="1-chapter01.html#the-cytochrome-cd_1-nitrite-reductase"><i class="fa fa-check"></i><b>1.5</b> The cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="1.5.1" data-path="1-chapter01.html"><a href="1-chapter01.html#subunit-structure"><i class="fa fa-check"></i><b>1.5.1</b> Subunit structure</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-chapter01.html"><a href="1-chapter01.html#properties-of-the-haem-groups-in-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.2</b> Properties of the haem groups in cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.3" data-path="1-chapter01.html"><a href="1-chapter01.html#folding-and-conformation-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>1.5.3</b> Folding and conformation of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="1.5.4" data-path="1-chapter01.html"><a href="1-chapter01.html#kinetic-and-mechanistic-studies-of-the-enzyme"><i class="fa fa-check"></i><b>1.5.4</b> Kinetic and mechanistic studies of the enzyme</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="1-chapter01.html"><a href="1-chapter01.html#genetic-analyses-of-denitrification"><i class="fa fa-check"></i><b>1.6</b> Genetic analyses of denitrification</a></li>
<li class="chapter" data-level="1.7" data-path="1-chapter01.html"><a href="1-chapter01.html#regulation-of-denitrification"><i class="fa fa-check"></i><b>1.7</b> Regulation of denitrification</a>
<ul>
<li class="chapter" data-level="1.7.1" data-path="1-chapter01.html"><a href="1-chapter01.html#anaerobic-response-regulators-the-fnr-family"><i class="fa fa-check"></i><b>1.7.1</b> Anaerobic response regulators: the FNR family</a></li>
<li class="chapter" data-level="1.7.2" data-path="1-chapter01.html"><a href="1-chapter01.html#the-arcab-system"><i class="fa fa-check"></i><b>1.7.2</b> The ArcA/B system</a></li>
<li class="chapter" data-level="1.7.3" data-path="1-chapter01.html"><a href="1-chapter01.html#sensors-of-nitrate-and-nitrite"><i class="fa fa-check"></i><b>1.7.3</b> Sensors of nitrate and nitrite</a></li>
</ul></li>
<li class="chapter" data-level="1.8" data-path="1-chapter01.html"><a href="1-chapter01.html#aims-of-the-present-work"><i class="fa fa-check"></i><b>1.8</b> Aims of the present work</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-chapter02.html"><a href="2-chapter02.html"><i class="fa fa-check"></i><b>2</b> Materials and Methods</a>
<ul>
<li class="chapter" data-level="2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#bacterial-strains"><i class="fa fa-check"></i><b>2.1</b> Bacterial strains</a></li>
<li class="chapter" data-level="2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-media"><i class="fa fa-check"></i><b>2.2</b> Growth media</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="2-chapter02.html"><a href="2-chapter02.html#luria-bertani-lb-medium"><i class="fa fa-check"></i><b>2.2.1</b> Luria-Bertani (LB) medium</a></li>
<li class="chapter" data-level="2.2.2" data-path="2-chapter02.html"><a href="2-chapter02.html#t.-pantotropha-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.2</b> <em>T. pantotropha</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#minimal-succinate-medium"><i class="fa fa-check"></i><b>2.2.3</b> Minimal succinate medium</a></li>
<li class="chapter" data-level="2.2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-aeruginosa-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.4</b> <em>Ps. aeruginosa</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.5" data-path="2-chapter02.html"><a href="2-chapter02.html#ps.-stutzeri-anaerobic-medium"><i class="fa fa-check"></i><b>2.2.5</b> <em>Ps. stutzeri</em> anaerobic medium</a></li>
<li class="chapter" data-level="2.2.6" data-path="2-chapter02.html"><a href="2-chapter02.html#e.-coli-anaerobic-minimal-medium"><i class="fa fa-check"></i><b>2.2.6</b> <em>E. coli</em> anaerobic minimal medium</a></li>
<li class="chapter" data-level="2.2.7" data-path="2-chapter02.html"><a href="2-chapter02.html#growth-on-solid-medium"><i class="fa fa-check"></i><b>2.2.7</b> Growth on solid medium</a></li>
<li class="chapter" data-level="2.2.8" data-path="2-chapter02.html"><a href="2-chapter02.html#antibiotics-and-other-selective-additions-to-media"><i class="fa fa-check"></i><b>2.2.8</b> Antibiotics and other selective additions to media</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-biochemistry-techniques"><i class="fa fa-check"></i><b>2.3</b> Protein biochemistry techniques</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-total-soluble-extracts"><i class="fa fa-check"></i><b>2.3.1</b> Preparation of total soluble extracts</a></li>
<li class="chapter" data-level="2.3.2" data-path="2-chapter02.html"><a href="2-chapter02.html#preparation-of-periplasmic-extracts"><i class="fa fa-check"></i><b>2.3.2</b> Preparation of periplasmic extracts</a></li>
<li class="chapter" data-level="2.3.3" data-path="2-chapter02.html"><a href="2-chapter02.html#protein-purification"><i class="fa fa-check"></i><b>2.3.3</b> Protein purification</a></li>
<li class="chapter" data-level="2.3.4" data-path="2-chapter02.html"><a href="2-chapter02.html#sodium-dodecyl-sulphate-polyacrylamide-gel-electrophoresis-sds-page"><i class="fa fa-check"></i><b>2.3.4</b> Sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS-PAGE)</a></li>
<li class="chapter" data-level="2.3.5" data-path="2-chapter02.html"><a href="2-chapter02.html#determination-of-protein-concentration"><i class="fa fa-check"></i><b>2.3.5</b> Determination of protein concentration</a></li>
<li class="chapter" data-level="2.3.6" data-path="2-chapter02.html"><a href="2-chapter02.html#uv-visible-spectroscopy"><i class="fa fa-check"></i><b>2.3.6</b> UV-visible spectroscopy</a></li>
<li class="chapter" data-level="2.3.7" data-path="2-chapter02.html"><a href="2-chapter02.html#assays-for-enzyme-activity"><i class="fa fa-check"></i><b>2.3.7</b> Assays for enzyme activity</a></li>
<li class="chapter" data-level="2.3.8" data-path="2-chapter02.html"><a href="2-chapter02.html#chemical-preparation-of-semi-apo-cytochrome-cd_1"><i class="fa fa-check"></i><b>2.3.8</b> Chemical preparation of semi-apo cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="2.3.9" data-path="2-chapter02.html"><a href="2-chapter02.html#purification-of-haem-d_1"><i class="fa fa-check"></i><b>2.3.9</b> Purification of haem <em>d</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="2-chapter02.html"><a href="2-chapter02.html#genetic-techniques"><i class="fa fa-check"></i><b>2.4</b> Genetic techniques</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="2-chapter02.html"><a href="2-chapter02.html#dna-manipulation"><i class="fa fa-check"></i><b>2.4.1</b> DNA manipulation</a></li>
<li class="chapter" data-level="2.4.2" data-path="2-chapter02.html"><a href="2-chapter02.html#rna-manipulation"><i class="fa fa-check"></i><b>2.4.2</b> RNA manipulation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-chapter03.html"><a href="3-chapter03.html"><i class="fa fa-check"></i><b>3</b> Sequencing and analysis of the <em>nirS</em> gene from <em>Thiosphaera pantotropha</em></a>
<ul>
<li class="chapter" data-level="3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#introduction"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#sequencing-strategy"><i class="fa fa-check"></i><b>3.2</b> Sequencing strategy</a></li>
<li class="chapter" data-level="3.3" data-path="3-chapter03.html"><a href="3-chapter03.html#results"><i class="fa fa-check"></i><b>3.3</b> Results</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="3-chapter03.html"><a href="3-chapter03.html#dna-sequence-features"><i class="fa fa-check"></i><b>3.3.1</b> DNA sequence features</a></li>
<li class="chapter" data-level="3.3.2" data-path="3-chapter03.html"><a href="3-chapter03.html#primary-sequence-of-cytochrome-cd_1-from-t.-pantotropha"><i class="fa fa-check"></i><b>3.3.2</b> Primary sequence of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> from <em>T. pantotropha</em></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="3-chapter03.html"><a href="3-chapter03.html#discussion"><i class="fa fa-check"></i><b>3.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="3-chapter03.html"><a href="3-chapter03.html#features-of-the-dna-sequence"><i class="fa fa-check"></i><b>3.4.1</b> Features of the DNA sequence</a></li>
<li class="chapter" data-level="3.4.2" data-path="3-chapter03.html"><a href="3-chapter03.html#analysis-of-the-seven-primary-sequences-of-cytochromes-cd_1"><i class="fa fa-check"></i><b>3.4.2</b> Analysis of the seven primary sequences of cytochromes <em>cd</em><span class="math inline">\(_1\)</span></a></li>
<li class="chapter" data-level="3.4.3" data-path="3-chapter03.html"><a href="3-chapter03.html#relationship-of-the-primary-sequence-to-the-structure-of-cytochrome-cd_1"><i class="fa fa-check"></i><b>3.4.3</b> Relationship of the primary sequence to the structure of cytochrome <em>cd</em><span class="math inline">\(_1\)</span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-chapter04.html"><a href="4-chapter04.html"><i class="fa fa-check"></i><b>4</b> Cloning and expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in <em>Escherichia coli</em></a>
<ul>
<li class="chapter" data-level="4.1" data-path="4-chapter04.html"><a href="4-chapter04.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="4-chapter04.html"><a href="4-chapter04.html#results-1"><i class="fa fa-check"></i><b>4.2</b> Results</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="4-chapter04.html"><a href="4-chapter04.html#sub-cloning-of-the-full-length-nirs-gene-from-t.-pantotropha"><i class="fa fa-check"></i><b>4.2.1</b> Sub-cloning of the full-length <em>nirS</em> gene from <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="4.2.2" data-path="4-chapter04.html"><a href="4-chapter04.html#cloning-the-nirs-gene-into-the-broad-host-range-expression-vector-pmmb67eh"><i class="fa fa-check"></i><b>4.2.2</b> Cloning the <em>nirS</em> gene into the broad host range expression vector pMMB67EH</a></li>
<li class="chapter" data-level="4.2.3" data-path="4-chapter04.html"><a href="4-chapter04.html#expression-from-pmmbse-in-e.-coli-jm83-under-different-growth-conditions"><i class="fa fa-check"></i><b>4.2.3</b> Expression from pMMBSE in <em>E. coli</em> JM83 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-chapter04.html"><a href="4-chapter04.html#discussion-1"><i class="fa fa-check"></i><b>4.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-chapter05.html"><a href="5-chapter05.html"><i class="fa fa-check"></i><b>5</b> Heterologous expression of the <em>nirS</em> gene from <em>T. pantotropha</em> in the denitrifying host <em>Ps. aeruginosa</em></a>
<ul>
<li class="chapter" data-level="5.1" data-path="5-chapter05.html"><a href="5-chapter05.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a></li>
<li class="chapter" data-level="5.2" data-path="5-chapter05.html"><a href="5-chapter05.html#results-2"><i class="fa fa-check"></i><b>5.2</b> Results</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="5-chapter05.html"><a href="5-chapter05.html#transformation-of-ps.-aeruginosa-pao1-with-the-plasmid-pmmbse"><i class="fa fa-check"></i><b>5.2.1</b> Transformation of <em>Ps. aeruginosa</em> PAO1 with the plasmid pMMBSE</a></li>
<li class="chapter" data-level="5.2.2" data-path="5-chapter05.html"><a href="5-chapter05.html#analysis-of-expression-from-pmmbse-in-ps.-aeruginosa-pao1-under-different-growth-conditions"><i class="fa fa-check"></i><b>5.2.2</b> Analysis of expression from pMMBSE in <em>Ps. aeruginosa</em> PAO1 under different growth conditions</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="5-chapter05.html"><a href="5-chapter05.html#discussion-2"><i class="fa fa-check"></i><b>5.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="6-chapter06.html"><a href="6-chapter06.html"><i class="fa fa-check"></i><b>6</b> Construction of an unmarked deletion in the <em>nirS</em> gene of <em>T. pantotropha</em>: complementation of the mutant and expression of a site-directed mutant of cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase</a>
<ul>
<li class="chapter" data-level="6.1" data-path="6-chapter06.html"><a href="6-chapter06.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="6-chapter06.html"><a href="6-chapter06.html#results-3"><i class="fa fa-check"></i><b>6.2</b> Results</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-mutagenic-plasmids-and-of-the-unmarked-deletion-mutant-in-the-nirs-gene"><i class="fa fa-check"></i><b>6.2.1</b> Construction of mutagenic plasmids and of the unmarked deletion mutant in the <em>nirS</em> gene</a></li>
<li class="chapter" data-level="6.2.2" data-path="6-chapter06.html"><a href="6-chapter06.html#analysis-of-the-nirs-insertion-and-deletion-mutants"><i class="fa fa-check"></i><b>6.2.2</b> Analysis of the <em>nirS</em> insertion and deletion mutants</a></li>
<li class="chapter" data-level="6.2.3" data-path="6-chapter06.html"><a href="6-chapter06.html#construction-of-a-cytochrome-cd_1-mutant-in-which-tyr-25-is-replaced-by-phenylalanine-and-expression-of-the-mutant-in-t.-pantotropha-delta7"><i class="fa fa-check"></i><b>6.2.3</b> Construction of a cytochrome <em>cd</em><span class="math inline">\(_1\)</span> mutant in which Tyr-25 is replaced by phenylalanine and expression of the mutant in <em>T. pantotropha</em> <span class="math inline">\(\Delta7\)</span></a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="6-chapter06.html"><a href="6-chapter06.html#discussion-3"><i class="fa fa-check"></i><b>6.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="7-chapter07.html"><a href="7-chapter07.html"><i class="fa fa-check"></i><b>7</b> Transcriptional analysis of the <em>nirS</em> gene from <em>T. pantotropha</em></a>
<ul>
<li class="chapter" data-level="7.1" data-path="7-chapter07.html"><a href="7-chapter07.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="7-chapter07.html"><a href="7-chapter07.html#results-4"><i class="fa fa-check"></i><b>7.2</b> Results</a></li>
<li class="chapter" data-level="7.3" data-path="7-chapter07.html"><a href="7-chapter07.html#discussion-4"><i class="fa fa-check"></i><b>7.3</b> Discussion</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="8-chapter08.html"><a href="8-chapter08.html"><i class="fa fa-check"></i><b>8</b> Conclusions and future research perspectives</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><a href="A-comparison-of-the-beta-propeller-structure-in-cytochrome-cd-1-with-other-beta-propeller-structures.html"><i class="fa fa-check"></i><b>A</b> Comparison of the <span class="math inline">\(\beta\)</span>-propeller structure in cytochrome <em>cd</em><span class="math inline">\(_1\)</span> with other <span class="math inline">\(\beta\)</span>-propeller structures</a></li>
<li class="chapter" data-level="B" data-path="B-classification-of-t-pantotropha.html"><a href="B-classification-of-t-pantotropha.html"><i class="fa fa-check"></i><b>B</b> Classification of <em>T. pantotropha</em></a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cloning, sequence analysis and studies on the expression of the <em>nirS</em> gene, encoding cytochrome <em>cd</em><span class="math inline">\(_1\)</span> nitrite reductase, from <em>Thiosphaera pantotropha</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<!--
This manually sets the header for this unnumbered chapter.
-->
<!--
To remove the indentation of the first entry.
-->
<p></p>
<!--
To create a hanging indent and spacing between entries.  These three lines may need to be removed for styles that don't require the hanging indent.
-->
<!--
This is just for testing with more citations for the bibliography at the end.  Add other entries into the list here if you'd like them to appear in the bibliography even if they weren't explicitly cited in the document.
-->

<div id="refs" class="references csl-bib-body">
<div class="csl-entry">
1. Society for General Microbiology, Fry JC, editors. Nitrogen removal from water and waste. In: Microbial control of pollution. Cambridge [England] ; New York, NY, USA: Cambridge University Press; 1992. p. 227–67.
</div>
<div class="csl-entry">
2. Gayon U, Dupetit G. Recherches sur la reduction des nitrates par les miiniment petits. Mem Soc Sci Phys Nat Bordeaux. 1886;2:201–307.
</div>
<div class="csl-entry">
3. Rasmussen RA, Khalil MA. Atmospheric trace gases: Trends and distributions over the last decade. Science (New York, NY). 1986;232:1623–4.
</div>
<div class="csl-entry">
4. Hauck R. In: Terrestrial <span>Nitrogen</span> <span>Cycles</span>. Stockholm: Swedish Natural Resource Council; 1981. p. 558.
</div>
<div class="csl-entry">
5. Hutchins SR. Biodegradation of monoaromatic hydrocarbons by aquifer microorganisms using oxygen, nitrate, or nitrous oxide as the terminal electron acceptor. Applied and Environmental Microbiology. 1991;57:2403–7.
</div>
<div class="csl-entry">
6. Olson RA. Fertilizers for food production vs energy needs and environmental quality. Ecotoxicology and Environmental Safety. 1977;1:311–26.
</div>
<div class="csl-entry">
7. Levallois P, Phaneuf D. [<span>Contamination</span> of drinking water by nitrates: Analysis of health risks]. Canadian Journal of Public Health = Revue Canadienne De Sante Publique. 1994;85:192–6.
</div>
<div class="csl-entry">
8. Calmels S, Ohshima H, Henry Y, Bartsch H. Characterization of bacterial cytochrome cd(1)-nitrite reductase as one enzyme responsible for catalysis of nitrosation of secondary amines. Carcinogenesis. 1996;17:533–6.
</div>
<div class="csl-entry">
9. Bredt DS, Snyder SH. Nitric oxide: A physiologic messenger molecule. Annual Review of Biochemistry. 1994;63:175–95.
</div>
<div class="csl-entry">
10. Shoun H, Tanimoto T. Denitrification by the fungus <span>Fusarium</span> oxysporum and involvement of cytochrome <span>P</span>-450 in the respiratory nitrite reduction. The Journal of Biological Chemistry. 1991;266:11078–82.
</div>
<div class="csl-entry">
11. Usuda K, Toritsuka N, Matsuo Y, Kim DH, Shoun H. Denitrification by the fungus <span>Cylindrocarpon</span> tonkinense: Anaerobic cell growth and two isozyme forms of cytochrome <span>P</span>-450nor. Applied and Environmental Microbiology. 1995;61:883–9.
</div>
<div class="csl-entry">
12. Kobayashi M, Matsuo Y, Takimoto A, Suzuki S, Maruo F, Shoun H. Denitrification, a novel type of respiratory metabolism in fungal mitochondrion. The Journal of Biological Chemistry. 1996;271:16263–7.
</div>
<div class="csl-entry">
13. Kobayashi M, Shoun H. The copper-containing dissimilatory nitrite reductase involved in the denitrifying system of the fungus <span>Fusarium</span> oxysporum. The Journal of Biological Chemistry. 1995;270:4146–51.
</div>
<div class="csl-entry">
14. Nakahara K, Tanimoto T, Hatano K, Usuda K, Shoun H. Cytochrome <span>P</span>-450 <span>55A1</span> (<span>P</span>-<span>450dNIR</span>) acts as nitric oxide reductase employing <span>NADH</span> as the direct electron donor. The Journal of Biological Chemistry. 1993;268:8350–5.
</div>
<div class="csl-entry">
15. Enoch HG, Lester RL. The role of a novel cytochrome b-containing nitrate reductase and quinone in the in vitro reconstruction of formate-nitrate reductase activity of <span>E</span>. coli. Biochemical and Biophysical Research Communications. 1974;61:1234–41.
</div>
<div class="csl-entry">
16. Berks BC, Richardson DJ, Reilly A, Willis AC, Ferguson SJ. The <span class="nocase">napEDABC</span> gene cluster encoding the periplasmic nitrate reductase system of <span>Thiosphaera</span> pantotropha. The Biochemical Journal. 1995;309 ( Pt 3):983–92.
</div>
<div class="csl-entry">
17. Richterich P, Lakey N, Gryan G, Jaehn L, Mintz L, Robison K, et al. U00008. Genbank. 1993. <a href="https://www.ncbi.nlm.nih.gov/nuccore/u00008">https://www.ncbi.nlm.nih.gov/nuccore/u00008</a>.
</div>
<div class="csl-entry">
18. Godden JW, Turley S, Teller DC, Adman ET, Liu MY, Payne WJ, et al. The 2.3 angstrom <span>X</span>-ray structure of nitrite reductase from <span>Achromobacter</span> cycloclastes. Science (New York, NY). 1991;253:438–42.
</div>
<div class="csl-entry">
19. Kukimoto M, Nishiyama M, Murphy ME, Turley S, Adman ET, Horinouchi S, et al. X-ray structure and site-directed mutagenesis of a nitrite reductase from <span>Alcaligenes</span> faecalis <span>S</span>-6: Roles of two copper atoms in nitrite reduction. Biochemistry. 1994;33:5246–52.
</div>
<div class="csl-entry">
20. Zumft W. The denitrifying prokaryotes. In: The <span>Prokaryotes</span>. 2nd edition. Springer-Verlag; 1992. p. 554–82. <a href="https://www.springer.com/gp/book/9781475721911">https://www.springer.com/gp/book/9781475721911</a>.
</div>
<div class="csl-entry">
21. John P, Whatley FR. Paracoccus denitrificans and the evolutionary origin of the mitochondrion. Nature. 1975;254:495–8.
</div>
<div class="csl-entry">
22. Trumpower BL. Cytochrome bc1 complexes of microorganisms. Microbiological Reviews. 1990;54:101–29.
</div>
<div class="csl-entry">
23. Ballard AL, Ferguson SJ. Respiratory nitrate reductase from <span>Paracoccus</span> denitrificans. <span>Evidence</span> for two b-type haems in the gamma subunit and properties of a water-soluble active enzyme containing alpha and beta subunits. European Journal of Biochemistry. 1988;174:207–12.
</div>
<div class="csl-entry">
24. Clegg RA. Purification and some properties of nitrate reductase (<span>EC</span> 1.7.99.4) from <span>Escherichia</span> coli <span>K12</span>. The Biochemical Journal. 1976;153:533–41.
</div>
<div class="csl-entry">
25. Ballard AL, Ferguson SJ. Molecular properties of the respiratory nitrate reductase of <span>Paracoccus</span> denitrificans. Biochemical Society Transactions. 1987;15:937–8. doi:<a href="https://doi.org/10.1042/bst0150937">10.1042/bst0150937</a>.
</div>
<div class="csl-entry">
26. Demoss JA, Fan TY, Scott RH. Characterization of subunit structural alterations which occur during purification of nitrate reductase from <span>Escherichia</span> coli. Archives of Biochemistry and Biophysics. 1981;206:54–64.
</div>
<div class="csl-entry">
27. Oppenheim DS, Yanofsky C. Translational coupling during expression of the tryptophan operon of <span>Escherichia</span> coli. Genetics. 1980;95:785–95.
</div>
<div class="csl-entry">
28. Berks BC, Page MD, Richardson DJ, Reilly A, Cavill A, Outen F, et al. Sequence analysis of subunits of the membrane-bound nitrate reductase from a denitrifying bacterium: The integral membrane subunit provides a prototype for the dihaem electron-carrying arm of a redox loop. Molecular Microbiology. 1995;15:319–31.
</div>
<div class="csl-entry">
29. Johnson MK, Bennett DE, Morningstar JE, Adams MW, Mortenson LE. The iron-sulfur cluster composition of <span>Escherichia</span> coli nitrate reductase. The Journal of Biological Chemistry. 1985;260:5456–63.
</div>
<div class="csl-entry">
30. Wootton JC, Nicolson RE, Mark Cock J, Walters DE, Burke JF, Doyle WA, et al. Enzymes depending on the pterin molybdenum cofactor: Sequence families, spectroscopic properties of molybdenum and possible cofactor-binding domains. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1991;1057:157–85. doi:<a href="https://doi.org/10.1016/S0005-2728(05)80100-8">10.1016/S0005-2728(05)80100-8</a>.
</div>
<div class="csl-entry">
31. Berks BC, Ferguson SJ, Moir JW, Richardson DJ. Enzymes and associated electron transport systems that catalyse the respiratory reduction of nitrogen oxides and oxyanions. Biochimica Et Biophysica Acta. 1995;1232:97–173.
</div>
<div class="csl-entry">
32. Craig JA, Holm RH. Reduction of nitrate to nitrite by molybdenum-mediated atom transfer: A nitrate reductase analog reaction system. Journal of the American Chemical Society. 1989;111:2111–5. doi:<a href="https://doi.org/10.1021/ja00188a026">10.1021/ja00188a026</a>.
</div>
<div class="csl-entry">
33. Cramer S. Molybdenum enzymes: A survey of structural information from <span>EXAFS</span> and <span>EPR</span> spectroscopy. Adv Inorg Bioinorg Mech. 1983;259–316.
</div>
<div class="csl-entry">
34. Bellion E. The <span>Biological</span> <span>Chemistry</span> of the <span>Elements</span>: <span>The</span> <span>Inorganic</span> <span>Chemistry</span> of <span>Life</span> (da <span>Silva</span>, <span>J</span>. <span>J</span>. <span>R</span>. <span>Frausto</span>; <span>Williams</span>, <span>R</span>. <span>J</span>. <span>P</span>.). Journal of Chemical Education. 1992;69:A326. doi:<a href="https://doi.org/10.1021/ed069pA326.1">10.1021/ed069pA326.1</a>.
</div>
<div class="csl-entry">
35. Schindelin H, Kisker C, Hilton J, Rajagopalan KV, Rees DC. Crystal structure of <span>DMSO</span> reductase: Redox-linked changes in molybdopterin coordination. Science (New York, NY). 1996;272:1615–21.
</div>
<div class="csl-entry">
36. Boyington JC, Gladyshev VN, Khangulov SV, Stadtman TC, Sun PD. Crystal structure of formate dehydrogenase <span>H</span>: Catalysis involving <span>Mo</span>, molybdopterin, selenocysteine, and an <span>Fe4S4</span> cluster. Science (New York, NY). 1997;275:1305–8.
</div>
<div class="csl-entry">
37. Sodergren EJ, DeMoss JA. <span class="nocase">narI</span> region of the <span>Escherichia</span> coli nitrate reductase (nar) operon contains two genes. Journal of Bacteriology. 1988;170:1721–9.
</div>
<div class="csl-entry">
38. Blasco F, Iobbi C, Ratouchniak J, Bonnefoy V, Chippaux M. Nitrate reductases of <span>Escherichia</span> coli: Sequence of the second nitrate reductase and comparison with that encoded by the <span class="nocase">narGHJI</span> operon. Molecular &amp; general genetics: MGG. 1990;222:104–11.
</div>
<div class="csl-entry">
39. Nohno T, Noji S, Taniguchi S, Saito T. The <span class="nocase">narX</span> and <span class="nocase">narL</span> genes encoding the nitrate-sensing regulators of <span>Escherichia</span> coli are homologous to a family of prokaryotic two-component regulatory genes. Nucleic Acids Research. 1989;17:2947–57.
</div>
<div class="csl-entry">
40. Rowe JJ, Ubbink-Kok T, Molenaar D, Konings WN, Driessen AJ. <span>NarK</span> is a nitrite-extrusion system involved in anaerobic nitrate respiration by <span>Escherichia</span> coli. Molecular Microbiology. 1994;12:579–86.
</div>
<div class="csl-entry">
41. DeMoss JA, Hsu PY. <span>NarK</span> enhances nitrate uptake and nitrite excretion in <span>Escherichia</span> coli. Journal of Bacteriology. 1991;173:3303–10.
</div>
<div class="csl-entry">
42. Peakman T, Crouzet J, Mayaux JF, Busby S, Mohan S, Harborne N, et al. Nucleotide sequence, organisation and structural analysis of the products of genes in the <span class="nocase">nirB</span>-<span class="nocase">cysG</span> region of the <span>Escherichia</span> coli <span>K</span>-12 chromosome. European Journal of Biochemistry. 1990;191:315–23.
</div>
<div class="csl-entry">
43. Kajie S, Anraku Y. Purification of a hexaheme cytochrome c552 from <span>Escherichia</span> coli <span>K</span> 12 and its properties as a nitrite reductase. European Journal of Biochemistry. 1986;154:457–63.
</div>
<div class="csl-entry">
44. Omata T. Structure, function and regulation of the nitrate transport system of the cyanobacterium <span>Synechococcus</span> sp. <span>PCC7942</span>. Plant &amp; Cell Physiology. 1995;36:207–13.
</div>
<div class="csl-entry">
45. Rodríguez R, Lara C, Guerrero MG. Nitrate transport in the cyanobacterium <span>Anacystis</span> nidulans <span>R2</span>. <span>Kinetic</span> and energetic aspects. The Biochemical Journal. 1992;282 ( Pt 3):639–43.
</div>
<div class="csl-entry">
46. McEwan AG, Wetzstein HG, Meyer O, Jackson JB, Ferguson SJ. The periplasmic nitrate reductase of <span>Rhodobacter</span> capsulatus; purification, characterisation and distinction from a single reductase for trimethylamine-<span>N</span>-oxide, dimethylsulphoxide and chlorate. Archives of Microbiology. 1987;147:340–5. doi:<a href="https://doi.org/10.1007/BF00406130">10.1007/BF00406130</a>.
</div>
<div class="csl-entry">
47. Berks BC, Richardson DJ, Robinson C, Reilly A, Aplin RT, Ferguson SJ. Purification and characterization of the periplasmic nitrate reductase from <span>Thiosphaera</span> pantotropha. European Journal of Biochemistry. 1994;220:117–24.
</div>
<div class="csl-entry">
48. Siddiqui RA, Warnecke-Eberz U, Hengsberger A, Schneider B, Kostka S, Friedrich B. Structure and function of a periplasmic nitrate reductase in <span>Alcaligenes</span> eutrophus <span>H16</span>. Journal of Bacteriology. 1993;175:5867–76.
</div>
<div class="csl-entry">
49. Reyes F, Roldán MD, Klipp W, Castillo F, Moreno-Vivián C. Isolation of periplasmic nitrate reductase genes from <span>Rhodobacter</span> sphaeroides <span>DSM</span> 158: Structural and functional differences among prokaryotic nitrate reductases. Molecular Microbiology. 1996;19:1307–18.
</div>
<div class="csl-entry">
50. Carter JP, Hsaio YH, Spiro S, Richardson DJ. Soil and sediment bacteria capable of aerobic nitrate respiration. Applied and Environmental Microbiology. 1995;61:2852–8.
</div>
<div class="csl-entry">
51. Breton J, Berks BC, Reilly A, Thomson AJ, Ferguson SJ, Richardson DJ. Characterization of the paramagnetic iron-containing redox centres of <span>Thiosphaera</span> pantotropha periplasmic nitrate reductase. FEBS letters. 1994;345:76–80.
</div>
<div class="csl-entry">
52. Thöny-Meyer L, Loferer H, Ritz D, Hennecke H. Bacterial genes and proteins involved in the biogenesis of c-type cytochromes and terminal oxidases. Biochimica Et Biophysica Acta. 1994;1187:260–3.
</div>
<div class="csl-entry">
53. Stouthamer AH. Metabolic pathways in <span>Paracoccus</span> denitrificans and closely related bacteria in relation to the phylogeny of prokaryotes. Antonie Van Leeuwenhoek. 1992;61:1–33.
</div>
<div class="csl-entry">
54. Noji S, Taniguchi S. Molecular oxygen controls nitrate transport of <span>Escherichia</span> coli nitrate-respiring cells. The Journal of Biological Chemistry. 1987;262:9441–3.
</div>
<div class="csl-entry">
55. Robertson LA, Dalsgaard T, Revsbech N-P, Kuenen JG. Confirmation of <span>‘aerobic denitrification’</span> in batch cultures, using gas chromatography and <span>15N</span> mass spectrometry. FEMS Microbiology Ecology. 1995;18:113–20. doi:<a href="https://doi.org/10.1111/j.1574-6941.1995.tb00168.x">10.1111/j.1574-6941.1995.tb00168.x</a>.
</div>
<div class="csl-entry">
56. Baumann B, Snozzi M, Zehnder AJ, Van Der Meer JR. Dynamics of denitrification activity of <span>Paracoccus</span> denitrificans in continuous culture during aerobic-anaerobic changes. Journal of Bacteriology. 1996;178:4367–74.
</div>
<div class="csl-entry">
57. Richardson DJ, Ferguson SJ. The influence of carbon substrate on the activity of the periplasmic nitrate reductase in aerobically grown <span>Thiosphaera</span> pantotropha. Archives of Microbiology. 1992;157:535–7. doi:<a href="https://doi.org/10.1007/BF00276774">10.1007/BF00276774</a>.
</div>
<div class="csl-entry">
58. Richardson DJ, King GF, Kelly DJ, McEwan AG, Ferguson SJ, Jackson JB. The role of auxiliary oxidants in maintaining redox balance during phototrophic growth of <span>Rhodobacter</span> capsulatus on propionate or butyrate. Archives of Microbiology. 1988;150:131–7. doi:<a href="https://doi.org/10.1007/BF00425152">10.1007/BF00425152</a>.
</div>
<div class="csl-entry">
59. Averill BA. Dissimilatory <span>Nitrite</span> and <span>Nitric</span> <span>Oxide</span> <span>Reductases</span>. Chemical Reviews. 1996;96:2951–64.
</div>
<div class="csl-entry">
60. Sann R, Kostka S, Friedrich B. A cytochrome cd1-type nitrite reductase mediates the first step of denitrification in <span>Alcaligenes</span> eutrophus. Archives of Microbiology. 1994;161:453–9.
</div>
<div class="csl-entry">
61. Iwasaki H, Matsubara T. Cytochrome c-557 (551) and cytochrome cd of <span>Alcaligenes</span> faecalis. Journal of Biochemistry. 1971;69:847–57.
</div>
<div class="csl-entry">
62. Kakutani T, Watanabe H, Arima K, Beppu T. Purification and properties of a copper-containing nitrite reductase from a denitrifying bacterium, <span>Alcaligenes</span> faecalis strain <span>S</span>-6. Journal of Biochemistry. 1981;89:453–61.
</div>
<div class="csl-entry">
63. Glockner AB, Jüngst A, Zumft WG. Copper-containing nitrite reductase from <span>Pseudomonas</span> aureofaciens is functional in a mutationally cytochrome cd1-free background (<span>NirS</span>-) of <span>Pseudomonas</span> stutzeri. Archives of Microbiology. 1993;160:18–26.
</div>
<div class="csl-entry">
64. Iwasaki H, Matsubara T. A nitrite reductase from <span>Achromobacter</span> cycloclastes. Journal of Biochemistry. 1972;71:645–52.
</div>
<div class="csl-entry">
65. Zumft WG, Gotzmann DJ, Kroneck PM. Type 1, blue copper proteins constitute a respiratory nitrite-reducing system in <span>Pseudomonas</span> aureofaciens. European Journal of Biochemistry. 1987;168:301–7.
</div>
<div class="csl-entry">
66. Denariaz G, Payne WJ, LeGall J. The denitrifying nitrite reductase of <span>Bacillus</span> halodenitrificans. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1991;1056:225–32. doi:<a href="https://doi.org/10.1016/S0005-2728(05)80053-2">10.1016/S0005-2728(05)80053-2</a>.
</div>
<div class="csl-entry">
67. Han J, Loehr TM, Lu Y, Valentine JS, Averill BA, Sanders-Loehr J. Resonance <span>Raman</span> excitation profiles indicate multiple <span>Cys</span>-<span>&gt;</span><span>Cu</span> charge transfer transitions in type 1 copper proteins. Journal of the American Chemical Society. 1993;115:4256–63. doi:<a href="https://doi.org/10.1021/ja00063a048">10.1021/ja00063a048</a>.
</div>
<div class="csl-entry">
68. Kukimoto M, Nishiyama M, Tanokura M, Adman ET, Horinouchi S. Studies on protein-protein interaction between copper-containing nitrite reductase and pseudoazurin from <span>Alcaligenes</span> faecalis <span>S</span>-6. The Journal of Biological Chemistry. 1996;271:13680–3.
</div>
<div class="csl-entry">
69. Strange RW, Dodd FE, Abraham ZH, Grossmann JG, Brüser T, Eady RR, et al. The substrate-binding site in <span>Cu</span> nitrite reductase and its similarity to <span>Zn</span> carbonic anhydrase. Nature Structural Biology. 1995;2:287–92.
</div>
<div class="csl-entry">
70. Hulse CL, Tiedje JM, Averill BA. Evidence for a copper-nitrosyl intermediate in denitrification by the copper-containing nitrite reductase of <span>Achromobacter</span> cycloclastes. Journal of the American Chemical Society. 1989;111:2322–3. doi:<a href="https://doi.org/10.1021/ja00188a067">10.1021/ja00188a067</a>.
</div>
<div class="csl-entry">
71. Jackson MA, Tiedje JM, Averill BA. Evidence for a <span>NO</span>-rebound mechanism for production of <span>N2O</span> from nitrite by the copper-containing nitrite reductase from <span>Achromobacter</span> cycloclastes. FEBS letters. 1991;291:41–4.
</div>
<div class="csl-entry">
72. Goretski J, Hollocher TC. Trapping of nitric oxide produced during denitrification by extracellular hemoglobin. The Journal of Biological Chemistry. 1988;263:2316–23.
</div>
<div class="csl-entry">
73. Zumft WG, Döhler K, Körner H, Löchelt S, Viebrock A, Frunzke K. Defects in cytochrome cd1-dependent nitrite respiration of transposon <span>Tn5</span>-induced mutants from <span>Pseudomonas</span> stutzeri. Archives of Microbiology. 1988;149:492–8.
</div>
<div class="csl-entry">
74. Carr GJ, Ferguson SJ. The nitric oxide reductase of <span>Paracoccus</span> denitrificans. The Biochemical Journal. 1990;269:423–9.
</div>
<div class="csl-entry">
75. Jones AM, Hollocher TC. Nitric oxide reductase of <span>Achromobacter</span> cycloclastes. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1993;1144:359–66. doi:<a href="https://doi.org/10.1016/0005-2728(93)90121-U">10.1016/0005-2728(93)90121-U</a>.
</div>
<div class="csl-entry">
76. Heiss B, Frunzke K, Zumft WG. Formation of the <span>N</span>-<span>N</span> bond from nitric oxide by a membrane-bound cytochrome bc complex of nitrate-respiring (denitrifying) <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1989;171:3288–97.
</div>
<div class="csl-entry">
77. Goretski J, Zafiriou OC, Hollocher TC. Steady-state nitric oxide concentrations during denitrification. The Journal of Biological Chemistry. 1990;265:11535–8.
</div>
<div class="csl-entry">
78. Kastrau DH, Heiss B, Kroneck PM, Zumft WG. Nitric oxide reductase from <span>Pseudomonas</span> stutzeri, a novel cytochrome bc complex. <span>Phospholipid</span> requirement, electron paramagnetic resonance and redox properties. European Journal of Biochemistry. 1994;222:293–303.
</div>
<div class="csl-entry">
79. Fujiwara T, Fukumori Y. Cytochrome cb-type nitric oxide reductase with cytochrome c oxidase activity from <span>Paracoccus</span> denitrificans <span>ATCC</span> 35512. Journal of Bacteriology. 1996;178:1866–71.
</div>
<div class="csl-entry">
80. Girsch P, Vries S de. Purification and initial kinetic and spectroscopic characterization of <span>NO</span> reductase from <span>Paracoccus</span> denitrificans. Biochimica Et Biophysica Acta. 1997;1318:202–16.
</div>
<div class="csl-entry">
81. Oost J van der, Boer AP de, Gier JW de, Zumft WG, Stouthamer AH, Spanning RJ van. The heme-copper oxidase family consists of three distinct types of terminal oxidases and is related to nitric oxide reductase. FEMS microbiology letters. 1994;121:1–9.
</div>
<div class="csl-entry">
82. Iwata S, Ostermeier C, Ludwig B, Michel H. Structure at 2.8 <span>A</span> resolution of cytochrome c oxidase from <span>Paracoccus</span> denitrificans. Nature. 1995;376:660–9.
</div>
<div class="csl-entry">
83. Saraste M, Castresana J. Cytochrome oxidase evolved by tinkering with denitrification enzymes. FEBS letters. 1994;341:1–4.
</div>
<div class="csl-entry">
84. Zhao XJ, Sampath V, Caughey WS. Cytochrome c oxidase catalysis of the reduction of nitric oxide to nitrous oxide. Biochemical and Biophysical Research Communications. 1995;212:1054–60.
</div>
<div class="csl-entry">
85. Coyle CL, Zumft WG, Kroneck PM, Körner H, Jakob W. Nitrous oxide reductase from denitrifying <span>Pseudomonas</span> perfectomarina. <span>Purification</span> and properties of a novel multicopper enzyme. European Journal of Biochemistry. 1985;153:459–67.
</div>
<div class="csl-entry">
86. Snyder SW, Hollocher TC. Purification and some characteristics of nitrous oxide reductase from <span>Paracoccus</span> denitrificans. The Journal of Biological Chemistry. 1987;262:6515–25.
</div>
<div class="csl-entry">
87. SooHoo CK, Hollocher TC. Purification and characterization of nitrous oxide reductase from <span>Pseudomonas</span> aeruginosa strain <span>P2</span>. The Journal of Biological Chemistry. 1991;266:2203–9.
</div>
<div class="csl-entry">
88. Matsubara T, Iwasaki H. A new-type of copper-protein from <span>Alcaligenes</span> faecalis. Journal of Biochemistry. 1972;71:747–50.
</div>
<div class="csl-entry">
89. Kelly M, Lappalainen P, Talbo G, Haltia T, Oost J van der, Saraste M. Two cysteines, two histidines, and one methionine are ligands of a binuclear purple copper center. The Journal of Biological Chemistry. 1993;268:16781–7.
</div>
<div class="csl-entry">
90. Zumft WG, Dreusch A, Löchelt S, Cuypers H, Friedrich B, Schneider B. Derived amino acid sequences of the <span class="nocase">nosZ</span> gene (respiratory <span>N2O</span> reductase) from <span>Alcaligenes</span> eutrophus, <span>Pseudomonas</span> aeruginosa and <span>Pseudomonas</span> stutzeri reveal potential copper-binding residues. <span>Implications</span> for the <span>CuA</span> site of <span>N2O</span> reductase and cytochrome-c oxidase. European Journal of Biochemistry. 1992;208:31–40.
</div>
<div class="csl-entry">
91. Farrar JA, Thomson AJ, Cheesman MR, Dooley DM, Zumft WG. A model of the copper centres of nitrous oxide reductase (<span>Pseudomonas</span> stutzeri). <span>Evidence</span> from optical, <span>EPR</span> and <span>MCD</span> spectroscopy. FEBS letters. 1991;294:11–5.
</div>
<div class="csl-entry">
92. Riester J, Zumft WG, Kroneck PM. Nitrous oxide reductase from <span>Pseudomonas</span> stutzeri. <span>Redox</span> properties and spectroscopic characterization of different forms of the multicopper enzyme. European Journal of Biochemistry. 1989;178:751–62.
</div>
<div class="csl-entry">
93. Kristjansson JK, Hollocher TC. First practical assay for soluble nitrous oxide reductase of denitrifying bacteria and a partial kinetic characterization. The Journal of Biological Chemistry. 1980;255:704–7.
</div>
<div class="csl-entry">
94. Berks BC, Baratta D, Richardson J, Ferguson SJ. Purification and characterization of a nitrous oxide reductase from <span>Thiosphaera</span> pantotropha. <span>Implications</span> for the mechanism of aerobic nitrous oxide reduction. European Journal of Biochemistry. 1993;212:467–76.
</div>
<div class="csl-entry">
95. Braun C, Zumft WG. The structural genes of the nitric oxide reductase complex from <span>Pseudomonas</span> stutzeri are part of a 30-kilobase gene cluster for denitrification. Journal of Bacteriology. 1992;174:2394–7.
</div>
<div class="csl-entry">
96. Hoeren FU, Berks BC, Ferguson SJ, McCarthy JE. Sequence and expression of the gene encoding the respiratory nitrous-oxide reductase from <span>Paracoccus</span> denitrificans. <span>New</span> and conserved structural and regulatory motifs. European Journal of Biochemistry. 1993;218:49–57.
</div>
<div class="csl-entry">
97. Heijne G von. Signal sequences. <span>The</span> limits of variation. Journal of Molecular Biology. 1985;184:99–105.
</div>
<div class="csl-entry">
98. Berks BC. A common export pathway for proteins binding complex redox cofactors? Molecular Microbiology. 1996;22:393–404.
</div>
<div class="csl-entry">
99. Ambler R. The structure and classification of cytochromes c. In: From <span>Cyclotrons</span> <span>To</span> <span>Cytochromes</span>: <span>Essays</span> in <span>Molecular</span> <span>Biology</span> and <span>Chemistry</span>. Saint Louis: Elsevier Science; 1980. <a href="http://qut.eblib.com.au/patron/FullRecord.aspx?p=1183623">http://qut.eblib.com.au/patron/FullRecord.aspx?p=1183623</a>. Accessed 29 Apr 2021.
</div>
<div class="csl-entry">
100. Hall J, Zha XH, Durham B, O’Brien P, Vieira B, Davis D, et al. Reaction of cytochromes c and c2 with the <span>Rhodobacter</span> sphaeroides reaction center involves the heme crevice domain. Biochemistry. 1987;26:4494–500.
</div>
<div class="csl-entry">
101. Moir JW, Baratta D, Richardson DJ, Ferguson SJ. The purification of a cd1-type nitrite reductase from, and the absence of a copper-type nitrite reductase from, the aerobic denitrifier <span>Thiosphaera</span> pantotropha; the role of pseudoazurin as an electron donor. European Journal of Biochemistry. 1993;212:377–85.
</div>
<div class="csl-entry">
102. Parr SR, Barber D, Greenwood C. A purification procedure for the soluble cytochrome oxidase and some other respiratory proteins from <span>Pseudomonas</span> aeruginosa. The Biochemical Journal. 1976;157:423–30.
</div>
<div class="csl-entry">
103. Husain M, Davidson VL, Smith AJ. Properties of <span>Paracoccus</span> denitrificans amicyanin. Biochemistry. 1986;25:2431–6.
</div>
<div class="csl-entry">
104. Kamp M van de, Silvestrini MC, Brunori M, Van Beeumen J, Hali FC, Canters GW. Involvement of the hydrophobic patch of azurin in the electron-transfer reactions with cytochrome <span>C551</span> and nitrite reductase. European Journal of Biochemistry. 1990;194:109–18.
</div>
<div class="csl-entry">
105. Williams PA, Fülöp V, Leung YC, Chan C, Moir JW, Howlett G, et al. Pseudospecific docking surfaces on electron transfer proteins as illustrated by pseudoazurin, cytochrome c550 and cytochrome cd1 nitrite reductase. Nature Structural Biology. 1995;2:975–82.
</div>
<div class="csl-entry">
106. Moir JWB, Ferguson SJ. Properties of a <span>Paracoccus</span> denitrificans mutant deleted in cytochrome c550 indicate that a copper protein can substitute for this cytochrome in electron transport to nitrite, nitric oxide and nitrous oxide. Microbiology. 1994;140:389–97. doi:<a href="https://doi.org/10.1099/13500872-140-2-389">10.1099/13500872-140-2-389</a>.
</div>
<div class="csl-entry">
107. Jenney FE, Daldal F. A novel membrane-associated c-type cytochrome, cyt cy, can mediate the photosynthetic growth of <span>Rhodobacter</span> capsulatus and <span>Rhodobacter</span> sphaeroides. The EMBO journal. 1993;12:1283–92.
</div>
<div class="csl-entry">
108. Husain M, Davidson VL. Characterization of two inducible periplasmic c-type cytochromes from <span>Paracoccus</span> denitrificans. The Journal of Biological Chemistry. 1986;261:8577–80.
</div>
<div class="csl-entry">
109. Horio T. <span>TERMINAL</span> <span>OXIDATION</span> <span>SYSTEM</span> <span>IN</span> <span>BACTERIA</span>. The Journal of Biochemistry. 1958;45:195–205. doi:<a href="https://doi.org/10.1093/oxfordjournals.jbchem.a126857">10.1093/oxfordjournals.jbchem.a126857</a>.
</div>
<div class="csl-entry">
110. Yamanaka T, Kijimoto S, Okunuki K. Biological significance of <span>Pseudomonas</span> cytochrome oxidase in <span>Pseudomonas</span> aeruginosa. Journal of Biochemistry. 1963;53:416–21.
</div>
<div class="csl-entry">
111. Timkovich R, Robinson MK. Evidence for water as the product for oxygen reduction by cytochrome cd. Biochemical and Biophysical Research Communications. 1979;88:649–55.
</div>
<div class="csl-entry">
112. Lam Y, Nicholas DJ. A nitrite reductase with cytochrome oxidase activity from <span>Micrococcus</span> denitrificans. Biochimica Et Biophysica Acta. 1969;180:459–72.
</div>
<div class="csl-entry">
113. Sawhney V, Nicholas DJD. Sulphide-linked <span>Nitrite</span> <span>Reductase</span> from <span>Thiobacillus</span> denitrificans with <span>Cytochrome</span> <span>Oxidase</span> <span>Activity</span>: <span>Purification</span> and <span>Properties</span>. Journal of General Microbiology. 1978;106:119–28. doi:<a href="https://doi.org/10.1099/00221287-106-1-119">10.1099/00221287-106-1-119</a>.
</div>
<div class="csl-entry">
114. Yamazaki T, Oyanagi H, Fujiwara T, Fukumori Y. Nitrite <span>Reductase</span> from the <span>Magnetotactic</span> <span>Bacterium</span> <span>Magnetospirillum</span> magnetotacticum. <span>A</span> <span>Novel</span> <span>Cytochrome</span> cd1 with <span>Fe</span>(<span>II</span>): <span>Nitrite</span> <span>Oxidoreductase</span> <span>Activity</span>. European Journal of Biochemistry. 1995;233:665–71. doi:<a href="https://doi.org/10.1111/j.1432-1033.1995.665_2.x">10.1111/j.1432-1033.1995.665_2.x</a>.
</div>
<div class="csl-entry">
115. Doi M, Shioi Y, Morita M, Takamiya K. Two types of cytochrome cd1 in the aerobic photosynthetic bacterium, <span>Erythrobacter</span> sp. <span>OCh</span> 114. European Journal of Biochemistry. 1989;184:521–7.
</div>
<div class="csl-entry">
116. Kuronen T, Ellfolk N. A new purification procedure and molecular properties of <span>Pseudomonas</span> cytochrome oxidase. Biochimica Et Biophysica Acta. 1972;275:308–18.
</div>
<div class="csl-entry">
117. Chang CK, Timkovich R, Wu W. Evidence that heme d1 is a 1,3-porphyrindione. Biochemistry. 1986;25:8447–53.
</div>
<div class="csl-entry">
118. Hill KE, Wharton DC. Reconstitution of the apoenzyme of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa with heme d1 and other heme groups. The Journal of Biological Chemistry. 1978;253:489–95.
</div>
<div class="csl-entry">
119. Walsh TA, Johnson MK, Greenwood C, Barber D, Springall JP, Thomson AJ. Some magnetic properties of <span>Pseudomonas</span> cytochrome oxidase. The Biochemical Journal. 1979;177:29–39.
</div>
<div class="csl-entry">
120. Walsh TA, Johnson MK, Thomson AJ, Barber D, Greenwood C. The characterization and magnetic properties of the azide and imidazole derivatives of <span>Pseudomonas</span> nitrite reductase. Journal of Inorganic Biochemistry. 1981;14:1–4.
</div>
<div class="csl-entry">
121. Barber D, Parr SR, Greenwood C. The reactions of <span>Pseudomonas</span> cytochrome c-551 oxidase with potassium cyanide. The Biochemical Journal. 1978;175:239–49.
</div>
<div class="csl-entry">
122. Parr SR, Wilson MT, Greenwood C. The reaction of <span>Pseudomonas</span> aeruginosa cytochrome c oxidase with carbon monoxide. The Biochemical Journal. 1975;151:51–9.
</div>
<div class="csl-entry">
123. Gilmour R, Goodhew CF, Pettigrew GW, Prazeres S, Moura JJ, Moura I. The kinetics of the oxidation of cytochrome c by <span>Paracoccus</span> cytochrome c peroxidase. The Biochemical Journal. 1994;300 ( Pt 3):907–14.
</div>
<div class="csl-entry">
124. Sutherland J, Greenwood C, Peterson J, Thomson AJ. An investigation of the ligand-binding properties of <span>Pseudomonas</span> aeruginosa nitrite reductase. The Biochemical Journal. 1986;233:893–8.
</div>
<div class="csl-entry">
125. Cheesman MR, Ferguson SJ, Moir JW, Richardson DJ, Zumft WG, Thomson AJ. Two enzymes with a common function but different heme ligands in the forms as isolated. <span>Optical</span> and magnetic properties of the heme groups in the oxidized forms of nitrite reductase, cytochrome cd1, from <span>Pseudomonas</span> stutzeri and <span>Thiosphaera</span> pantotropha. Biochemistry. 1997;36:16267–76.
</div>
<div class="csl-entry">
126. Huynh BH, Lui MC, Moura JJ, Moura I, Ljungdahl PO, Münck E, et al. Mössbauer and <span>EPR</span> studies on nitrite reductase from <span>Thiobacillus</span> denitrificans. The Journal of Biological Chemistry. 1982;257:9576–81.
</div>
<div class="csl-entry">
127. Fülöp V, Moir JW, Ferguson SJ, Hajdu J. The anatomy of a bifunctional enzyme: Structural basis for reduction of oxygen to water and synthesis of nitric oxide by cytochrome cd1. Cell. 1995;81:369–77.
</div>
<div class="csl-entry">
128. Horio T, Kamen MD, De Klerk H. Relative oxidation-reduction potentials of heme groups in two soluble doubleheme proteins. The Journal of Biological Chemistry. 1961;236:2783–7.
</div>
<div class="csl-entry">
129. Kamen MD, Horio T. Bacterial cytochromes. <span>I</span>. <span>Structural</span> aspects. Annual Review of Biochemistry. 1970;39:673–700.
</div>
<div class="csl-entry">
130. Wharton DC, Gudat JC, Gibson QH. Cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. <span>II</span>. <span>Reaction</span> with copper protein. Biochimica et Biophysica Acta (BBA) - Bioenergetics. 1973;292:611–20. doi:<a href="https://doi.org/10.1016/0005-2728(73)90009-1">10.1016/0005-2728(73)90009-1</a>.
</div>
<div class="csl-entry">
131. Shimada H, Orii Y. Oxidation-reduction behavior of the heme c and heme d moieties of <span>Pseudomonas</span> aeruginosa nitrite reductase and the formation of an oxygenated intermediate at heme d1. Journal of Biochemistry. 1976;80:135–40.
</div>
<div class="csl-entry">
132. Blatt Y, Pecht I. Allosteric cooperative interactions among redox sites of <span>Pseudomonas</span> cytochrome oxidase. Biochemistry. 1979;18:2917–22.
</div>
<div class="csl-entry">
133. Schichman SA, Gray HB. Kinetics of the anaerobic reduction of ferricytochrome cd1 by <span>Fe</span>(<span>EDTA</span>)2-. <span>Evidence</span> for bimolecular and intramolecular electron transfers to the d1 hemes. Journal of the American Chemical Society. 1981;103:7794–5. doi:<a href="https://doi.org/10.1021/ja00416a020">10.1021/ja00416a020</a>.
</div>
<div class="csl-entry">
134. Silvestrini MC, Tordi MG, Colosimo A, Antonini E, Brunori M. The kinetics of electron transfer between pseudomonas aeruginosa cytochrome c-551 and its oxidase. The Biochemical Journal. 1982;203:445–51.
</div>
<div class="csl-entry">
135. Carson SD, Ching YC, Wells CA, Wharton DC, Ondrias MR. Variations in the oxidation-reduction behavior of liganded species of <span>Pseudomonas</span> cytochrome oxidase. Biochemistry. 1986;25:787–90.
</div>
<div class="csl-entry">
136. Besson S, Carneiro C, Moura JJ, Moura I, Fauque G. A cytochrome cd1-type nitrite reductase isolated from the marine denitrifier <span>Pseudomonas</span> nautica 617: Purification and characterization. Anaerobe. 1995;1:219–26.
</div>
<div class="csl-entry">
137. Silvestrini MC, Galeotti CL, Gervais M, Schininà E, Barra D, Bossa F, et al. Nitrite reductase from <span>Pseudomonas</span> aeruginosa: Sequence of the gene and the protein. FEBS letters. 1989;254:33–8.
</div>
<div class="csl-entry">
138. Jüngst A, Wakabayashi S, Matsubara H, Zumft WG. The <span class="nocase">nirSTBM</span> region coding for cytochrome cd1-dependent nitrite respiration of <span>Pseudomonas</span> stutzeri consists of a cluster of mono-, di-, and tetraheme proteins. FEBS letters. 1991;279:205–9.
</div>
<div class="csl-entry">
139. Smith GB, Tiedje JM. Isolation and characterization of a nitrite reductase gene and its use as a probe for denitrifying bacteria. Applied and Environmental Microbiology. 1992;58:376–84.
</div>
<div class="csl-entry">
140. Ohshima T, Sugiyama M, Uozumi N, Iijima S, Kobayashi T. Cloning and sequencing of a gene encoding nitrite reductase from <span>Paracoccus</span> denitrificans and expression of the gene in <span>Escherichia</span> coli. Journal of Fermentation and Bioengineering. 1993;76:82–8. doi:<a href="https://doi.org/10.1016/0922-338X(93)90061-C">10.1016/0922-338X(93)90061-C</a>.
</div>
<div class="csl-entry">
141. Boer AP de, Reijnders WN, Kuenen JG, Stouthamer AH, Spanning RJ van. Isolation, sequencing and mutational analysis of a gene cluster involved in nitrite reduction in <span>Paracoccus</span> denitrificans. Antonie Van Leeuwenhoek. 1994;66:111–27.
</div>
<div class="csl-entry">
142. Rees E, Siddiqui RA, Köster F, Schneider B, Friedrich B. Structural gene (<span class="nocase">nirS</span>) for the cytochrome cd1 nitrite reductase of <span>Alcaligenes</span> eutrophus <span>H16</span>. Applied and Environmental Microbiology. 1997;63:800–2.
</div>
<div class="csl-entry">
143. Horowitz PM, Muhoberac BB, Falksen K, Wharton DC. Controlled proteolysis by subtilisin as a probe for cyanide-induced conformational changes in <span>Pseudomonas</span> cytochrome oxidase. The Journal of Biological Chemistry. 1982;257:2140–3.
</div>
<div class="csl-entry">
144. Tordi MG, Silvestrini MC, Colosimo A, Provencher S, Brunori M. Circular-dichroic properties and secondary structure of <span>Pseudomonas</span> aeruginosa soluble cytochrome c oxidase. The Biochemical Journal. 1984;218:907–12.
</div>
<div class="csl-entry">
145. Berger H, Wharton DC. Small-angle <span>X</span>-ray scattering studies of oxidized and reduced cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. Biochimica Et Biophysica Acta. 1980;622:355–9.
</div>
<div class="csl-entry">
146. Mitra S, Donovan JW, Bersohn R. Heat stabilization dependence on redox state of cytochrome cd1 oxidase from <span>Pseudomonas</span> aeruginosa. Biochemical and Biophysical Research Communications. 1981;98:140–6.
</div>
<div class="csl-entry">
147. Akey CW, Moffat K, Wharton DC, Edelstein SJ. Characterization of crystals of a cytochrome oxidase (nitrite reductase) from <span>Pseudomonas</span> aeruginosa by x-ray diffraction and electron microscopy. Journal of Molecular Biology. 1980;136:19–43.
</div>
<div class="csl-entry">
148. Mitra S, Bersohn R. Location of the heme groups in cytochrome cd1 oxidase from <span>Pseudomonas</span> aeruginosa. Biochemistry. 1980;19:3200–3.
</div>
<div class="csl-entry">
149. Yamanaka T, Okunuki K. Crystalline <span>Pseudomonas</span> cytochrome oxidase. <span>I</span>. <span>Enzymic</span> properties with special reference to the biological specificity. Biochimica Et Biophysica Acta. 1963;67:379–93.
</div>
<div class="csl-entry">
150. Moore GR, Pettigrew GW. Cytochromes <span>C</span> <span>Evolutionary</span>, <span>Structural</span> and <span>Physicochemical</span> <span>Aspects</span>. Berlin, Heidelberg: Springer Berlin / Heidelberg; 1990. <a href="http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030">http://public.eblib.com/choice/PublicFullRecord.aspx?p=6499030</a>. Accessed 28 Apr 2021.
</div>
<div class="csl-entry">
151. Xia ZX, Dai WW, Xiong JP, Hao ZP, Davidson VL, White S, et al. The three-dimensional structures of methanol dehydrogenase from two methylotrophic bacteria at 2.6-<span>A</span> resolution. The Journal of Biological Chemistry. 1992;267:22289–97.
</div>
<div class="csl-entry">
152. Vellieux FM, Huitema F, Groendijk H, Kalk KH, Jzn JF, Jongejan JA, et al. Structure of quinoprotein methylamine dehydrogenase at 2.25 <span>A</span> resolution. The EMBO journal. 1989;8:2171–8.
</div>
<div class="csl-entry">
153. Varghese JN, Laver WG, Colman PM. Structure of the influenza virus glycoprotein antigen neuraminidase at 2.9 <span>A</span> resolution. Nature. 1983;303:35–40.
</div>
<div class="csl-entry">
154. Dhesi R, Timkovich R. Patterns of product inhibition for bacterial nitrite reductase. Biochemical and Biophysical Research Communications. 1984;123:966–72.
</div>
<div class="csl-entry">
155. Robinson MK, Martinkus K, Kennelly PJ, Timkovich R. Implications of the integrated rate law for the reactions of <span>Paracoccus</span> denitrificans nitrite reductase. Biochemistry. 1979;18:3921–6.
</div>
<div class="csl-entry">
156. Parr SR, Barber D, Greenwood C, Brunori M. The electron-transfer reaction between azurin and the cytochrome c oxidase from <span>Pseudomonas</span> aeruginosa. The Biochemical Journal. 1977;167:447–55.
</div>
<div class="csl-entry">
157. Kim CH, Hollocher TC. Catalysis of nitrosyl transfer reactions by a dissimilatory nitrite reductase (cytochrome c,d1). The Journal of Biological Chemistry. 1984;259:2092–9.
</div>
<div class="csl-entry">
158. Kim CH, Hollocher TC. <span>15N</span> tracer studies on the reduction of nitrite by the purified dissimilatory nitrite reductase of <span>Pseudomonas</span> aeruginosa. <span>Evidence</span> for direct production of <span>N2O</span> without free <span>NO</span> as an intermediate. The Journal of Biological Chemistry. 1983;258:4861–3.
</div>
<div class="csl-entry">
159. Carr GJ, Page MD, Ferguson SJ. The energy-conserving nitric-oxide-reductase system in <span>Paracoccus</span> denitrificans. <span>Distinction</span> from the nitrite reductase that catalyses synthesis of nitric oxide and evidence from trapping experiments for nitric oxide as a free intermediate during denitrification. European Journal of Biochemistry. 1989;179:683–92.
</div>
<div class="csl-entry">
160. Wang Y, Averill BA. Direct <span>Observation</span> by <span>FTIR</span> <span>Spectroscopy</span> of the <span>Ferrous</span> <span>Heme</span>−<span>NO</span> <span class="math inline">\(^{\textrm{+}}\)</span> <span>Intermediate</span> in <span>Reduction</span> of <span>Nitrite</span> by a <span>Dissimilatory</span> <span>Heme</span> <em>cd</em> <span class="math inline">\(_{\textrm{1}}\)</span> <span>Nitrite</span> <span>Reductase</span>. Journal of the American Chemical Society. 1996;118:3972–3. doi:<a href="https://doi.org/10.1021/ja9538647">10.1021/ja9538647</a>.
</div>
<div class="csl-entry">
161. Silvestrini MC, Tordi MG, Musci G, Brunori M. The reaction of <span>Pseudomonas</span> nitrite reductase and nitrite. <span>A</span> stopped-flow and <span>EPR</span> study. The Journal of Biological Chemistry. 1990;265:11783–7.
</div>
<div class="csl-entry">
162. LeGall J, Payne WJ, Morgan TV, DerVartanian D. On the purification of nitrite reductase from <span>Thiobacillus</span> denitrificans and its reaction with nitrite under reducing conditions. Biochemical and Biophysical Research Communications. 1979;87:355–62.
</div>
<div class="csl-entry">
163. Traylor TG, Sharma VS. Why <span>NO</span>? Biochemistry. 1992;31:2847–9.
</div>
<div class="csl-entry">
164. Kobayashi K, Koppenhöfer A, Ferguson SJ, Tagawa S. Pulse radiolysis studies on cytochrome cd1 nitrite reductase from <span>Thiosphaera</span> pantotropha: Evidence for a fast intramolecular electron transfer from c-heme to d1-heme. Biochemistry. 1997;36:13611–6.
</div>
<div class="csl-entry">
165. Williams P. Time-resolved structural studies on macromolecules. PhD thesis. University of Oxford; 1996.
</div>
<div class="csl-entry">
166. Zumft WG, Döhler K, Körner H. Isolation and characterization of transposon <span>Tn5</span>-induced mutants of <span>Pseudomonas</span> perfectomarina defective in nitrous oxide respiration. Journal of Bacteriology. 1985;163:918–24.
</div>
<div class="csl-entry">
167. Zumft WG, Körner H. Enzyme diversity and mosaic gene organization in denitrification. Antonie Van Leeuwenhoek. 1997;71:43–58.
</div>
<div class="csl-entry">
168. Boer T de. Regulation of denitrification in <span>Paracoccus</span> denitrificans. PhD thesis. VU Amsterdam; 1996. <a href="https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans">https://research.vu.nl/en/publications/regulation-of-denitrification-in-paracoccus-denitrificans</a>.
</div>
<div class="csl-entry">
169. Arai H, Sanbongi Y, Igarashi Y, Kodama T. Cloning and sequencing of the gene encoding cytochrome c-551 from <span>Pseudomonas</span> aeruginosa. FEBS letters. 1990;261:196–8.
</div>
<div class="csl-entry">
170. Jüngst A, Zumft WG. Interdependence of respiratory <span>NO</span> reduction and nitrite reduction revealed by mutagenesis of <span class="nocase">nirQ</span>, a novel gene in the denitrification gene cluster of <span>Pseudomonas</span> stutzeri. FEBS letters. 1992;314:308–14.
</div>
<div class="csl-entry">
171. Boer AP de, Oost J van der, Reijnders WN, Westerhoff HV, Stouthamer AH, Spanning RJ van. Mutational analysis of the nor gene cluster which encodes nitric-oxide reductase from <span>Paracoccus</span> denitrificans. European Journal of Biochemistry. 1996;242:592–600.
</div>
<div class="csl-entry">
172. Cuypers H, Viebrock-Sambale A, Zumft WG. <span>NosR</span>, a membrane-bound regulatory component necessary for expression of nitrous oxide reductase in denitrifying <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1992;174:5332–9.
</div>
<div class="csl-entry">
173. Glockner AB, Zumft WG. Sequence analysis of an internal 9.72-kb segment from the 30-kb denitrification gene cluster of <span>Pseudomonas</span> stutzeri. Biochimica Et Biophysica Acta. 1996;1277:6–12.
</div>
<div class="csl-entry">
174. Mushegian AR, Koonin EV. Gene order is not conserved in bacterial evolution. Trends in genetics: TIG. 1996;12:289–90.
</div>
<div class="csl-entry">
175. Lambden PR, Guest JR. Mutants of <span>Escherichia</span> coli <span>K12</span> unable to use fumarate as an anaerobic electron acceptor. Journal of General Microbiology. 1976;97:145–60.
</div>
<div class="csl-entry">
176. Shaw DJ, Rice DW, Guest JR. Homology between <span>CAP</span> and <span>Fnr</span>, a regulator of anaerobic respiration in <span>Escherichia</span> coli. Journal of Molecular Biology. 1983;166:241–7.
</div>
<div class="csl-entry">
177. Green J, Bennett B, Jordan P, Ralph ET, Thomson AJ, Guest JR. Reconstitution of the [<span>4Fe</span>-<span>4S</span>] cluster in <span>FNR</span> and demonstration of the aerobic-anaerobic transcription switch in vitro. The Biochemical Journal. 1996;316 ( Pt 3):887–92.
</div>
<div class="csl-entry">
178. Khoroshilova N, Beinert H, Kiley PJ. Association of a polynuclear iron-sulfur center with a mutant <span>FNR</span> protein enhances <span>DNA</span> binding. Proceedings of the National Academy of Sciences of the United States of America. 1995;92:2499–503.
</div>
<div class="csl-entry">
179. Unden G, Trageser M, Duchêne A. Effect of positive redox potentials (greater than +400 <span class="nocase">mV</span>) on the expression of anaerobic respiratory enzymes in <span>Escherichia</span> coli. Molecular Microbiology. 1990;4:315–9.
</div>
<div class="csl-entry">
180. Walker MS, DeMoss JA. Promoter sequence requirements for <span>Fnr</span>-dependent activation of transcription of the <span class="nocase">narGHJI</span> operon. Molecular Microbiology. 1991;5:353–60.
</div>
<div class="csl-entry">
181. Spiro S, Guest JR. Regulation and over-expression of the fnr gene of <span>Escherichia</span> coli. Journal of General Microbiology. 1987;133:3279–88.
</div>
<div class="csl-entry">
182. Sharrocks AD, Green J, Guest JR. <span>FNR</span> activates and represses transcription in vitro. Proceedings Biological Sciences. 1991;245:219–26.
</div>
<div class="csl-entry">
183. Ziegelhoffer EC, Kiley PJ. In vitro analysis of a constitutively active mutant form of the <span>Escherichia</span> coli global transcription factor <span>FNR</span>. Journal of Molecular Biology. 1995;245:351–61.
</div>
<div class="csl-entry">
184. Park SJ, Tseng CP, Gunsalus RP. Regulation of succinate dehydrogenase (<span class="nocase">sdhCDAB</span>) operon expression in <span>Escherichia</span> coli in response to carbon supply and anaerobiosis: Role of <span>ArcA</span> and <span>Fnr</span>. Molecular Microbiology. 1995;15:473–82.
</div>
<div class="csl-entry">
185. Williams R, Bell A, Sims G, Busby S. The role of two surface exposed loops in transcription activation by the <span>Escherichia</span> coli <span>CRP</span> and <span>FNR</span> proteins. Nucleic Acids Research. 1991;19:6705–12.
</div>
<div class="csl-entry">
186. Spiro S. The <span>FNR</span> family of transcriptional regulators. Antonie Van Leeuwenhoek. 1994;66:23–36.
</div>
<div class="csl-entry">
187. Spiro S, Gaston KL, Bell AI, Roberts RE, Busby SJ, Guest JR. Interconversion of the <span>DNA</span>-binding specificities of two related transcription regulators, <span>CRP</span> and <span>FNR</span>. Molecular Microbiology. 1990;4:1831–8.
</div>
<div class="csl-entry">
188. Sawers G, Kaiser M, Sirko A, Freundlich M. Transcriptional activation by <span>FNR</span> and <span>CRP</span>: Reciprocity of binding-site recognition. Molecular Microbiology. 1997;23:835–45.
</div>
<div class="csl-entry">
189. Thompson JD, Higgins DG, Gibson TJ. <span>CLUSTAL</span> <span>W</span>: Improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Research. 1994;22:4673–80.
</div>
<div class="csl-entry">
190. Saitou N, Nei M. The neighbor-joining method: A new method for reconstructing phylogenetic trees. Molecular Biology and Evolution. 1987;4:406–25.
</div>
<div class="csl-entry">
191. Felsenstein J. <span>CONFIDENCE</span> <span>LIMITS</span> <span>ON</span> <span>PHYLOGENIES</span>: <span>AN</span> <span>APPROACH</span> <span>USING</span> <span>THE</span> <span>BOOTSTRAP</span>. Evolution; International Journal of Organic Evolution. 1985;39:783–91.
</div>
<div class="csl-entry">
192. Page RD. <span>TreeView</span>: An application to display phylogenetic trees on personal computers. Computer applications in the biosciences: CABIOS. 1996;12:357–8.
</div>
<div class="csl-entry">
193. Van Spanning RJM, De Boer APN, Reijnders WNM, Westerhoff HV, Stouthamer AH, Van Der Oost J. <span>FnrP</span> and <span>NNR</span> of <em><span>Paracoccus</span> denitrificans</em> are both members of the <span>FNR</span> family of transcriptional activators but have distinct roles in respiratory adaptation in response to oxygen limitation. Molecular Microbiology. 1997;23:893–907. doi:<a href="https://doi.org/10.1046/j.1365-2958.1997.2801638.x">10.1046/j.1365-2958.1997.2801638.x</a>.
</div>
<div class="csl-entry">
194. Frías JE, Mérida A, Herrero A, Martín-Nieto J, Flores E. General distribution of the nitrogen control gene <span class="nocase">ntcA</span> in cyanobacteria. Journal of Bacteriology. 1993;175:5710–3.
</div>
<div class="csl-entry">
195. Van Spanning RJ, De Boer AP, Reijnders WN, Spiro S, Westerhoff HV, Stouthamer AH, et al. Nitrite and nitric oxide reduction in <span>Paracoccus</span> denitrificans is under the control of <span>NNR</span>, a regulatory protein that belongs to the <span>FNR</span> family of transcriptional activators. FEBS letters. 1995;360:151–4.
</div>
<div class="csl-entry">
196. Ye RW, Haas D, Ka JO, Krishnapillai V, Zimmermann A, Baird C, et al. Anaerobic activation of the entire denitrification pathway in <span>Pseudomonas</span> aeruginosa requires <span>Anr</span>, an analog of <span>Fnr</span>. Journal of Bacteriology. 1995;177:3606–9.
</div>
<div class="csl-entry">
197. Arai H, Igarashi Y, Kodama T. Expression of the nir and nor genes for denitrification of <span>Pseudomonas</span> aeruginosa requires a novel <span>CRP</span>/<span>FNR</span>-related transcriptional regulator, <span>DNR</span>, in addition to <span>ANR</span>. FEBS letters. 1995;371:73–6.
</div>
<div class="csl-entry">
198. Batut J, Daveran-Mingot ML, David M, Jacobs J, Garnerone AM, Kahn D. <span class="nocase">fixK</span>, a gene homologous with fnr and crp from <span>Escherichia</span> coli, regulates nitrogen fixation genes both positively and negatively in <span>Rhizobium</span> meliloti. The EMBO journal. 1989;8:1279–86.
</div>
<div class="csl-entry">
199. Irvine AS, Guest JR. Lactobacillus casei contains a member of the <span>CRP</span>-<span>FNR</span> family. Nucleic Acids Research. 1993;21:753.
</div>
<div class="csl-entry">
200. Tomura D, Obika K, Fukamizu A, Shoun H. Nitric oxide reductase cytochrome <span>P</span>-450 gene, <span>CYP</span> 55, of the fungus <span>Fusarium</span> oxysporum containing a potential binding-site for <span>FNR</span>, the transcription factor involved in the regulation of anaerobic growth of <span>Escherichia</span> coli. Journal of Biochemistry. 1994;116:88–94.
</div>
<div class="csl-entry">
201. Anthamatten D, Scherb B, Hennecke H. Characterization of a <span class="nocase">fixLJ</span>-regulated <span>Bradyrhizobium</span> japonicum gene sharing similarity with the <span>Escherichia</span> coli fnr and <span>Rhizobium</span> meliloti <span class="nocase">fixK</span> genes. Journal of Bacteriology. 1992;174:2111–20.
</div>
<div class="csl-entry">
202. Tosques IE, Shi J, Shapleigh JP. Cloning and characterization of <span class="nocase">nnrR</span>, whose product is required for the expression of proteins involved in nitric oxide metabolism in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1996;178:4958–64.
</div>
<div class="csl-entry">
203. Winteler HV, Haas D. The homologous regulators <span>ANR</span> of <span>Pseudomonas</span> aeruginosa and <span>FNR</span> of <span>Escherichia</span> coli have overlapping but distinct specificities for anaerobically inducible promoters. Microbiology (Reading, England). 1996;142 ( Pt 3):685–93.
</div>
<div class="csl-entry">
204. Cuypers H, Zumft WG. Anaerobic control of denitrification in <span>Pseudomonas</span> stutzeri escapes mutagenesis of an fnr-like gene. Journal of Bacteriology. 1993;175:7236–46.
</div>
<div class="csl-entry">
205. Stock JB, Ninfa AJ, Stock AM. Protein phosphorylation and regulation of adaptive responses in bacteria. Microbiological Reviews. 1989;53:450–90.
</div>
<div class="csl-entry">
206. Iuchi S, Matsuda Z, Fujiwara T, Lin EC. The <span class="nocase">arcB</span> gene of <span>Escherichia</span> coli encodes a sensor-regulator protein for anaerobic repression of the arc modulon. Molecular Microbiology. 1990;4:715–27.
</div>
<div class="csl-entry">
207. Iuchi S, Lin EC. Purification and phosphorylation of the <span>Arc</span> regulatory components of <span>Escherichia</span> coli. Journal of Bacteriology. 1992;174:5617–23.
</div>
<div class="csl-entry">
208. Unden G, Becker S, Bongaerts J, Schirawski J, Six S. Oxygen regulated gene expression in facultatively anaerobic bacteria. Antonie Van Leeuwenhoek. 1994;66:3–22.
</div>
<div class="csl-entry">
209. Lynch AS, Lin EC. Transcriptional control mediated by the <span>ArcA</span> two-component response regulator protein of <span>Escherichia</span> coli: Characterization of <span>DNA</span> binding at target promoters. Journal of Bacteriology. 1996;178:6238–49.
</div>
<div class="csl-entry">
210. Tseng CP, Albrecht J, Gunsalus RP. Effect of microaerophilic cell growth conditions on expression of the aerobic (<span class="nocase">cyoABCDE</span> and <span class="nocase">cydAB</span>) and anaerobic (<span class="nocase">narGHJI</span>, <span class="nocase">frdABCD</span>, and <span class="nocase">dmsABC</span>) respiratory pathway genes in <span>Escherichia</span> coli. Journal of Bacteriology. 1996;178:1094–8.
</div>
<div class="csl-entry">
211. Stewart V. Nitrate regulation of anaerobic respiratory gene expression in <span>Escherichia</span> coli. Molecular Microbiology. 1993;9:425–34.
</div>
<div class="csl-entry">
212. Rabin RS, Stewart V. Dual response regulators (<span>NarL</span> and <span>NarP</span>) interact with dual sensors (<span>NarX</span> and <span>NarQ</span>) to control nitrate- and nitrite-regulated gene expression in <span>Escherichia</span> coli <span>K</span>-12. Journal of Bacteriology. 1993;175:3259–68.
</div>
<div class="csl-entry">
213. Cavicchioli R, Chiang RC, Kalman LV, Gunsalus RP. Role of the periplasmic domain of the <span>Escherichia</span> coli <span>NarX</span> sensor-transmitter protein in nitrate-dependent signal transduction and gene regulation. Molecular Microbiology. 1996;21:901–11.
</div>
<div class="csl-entry">
214. Tyson KL, Cole JA, Busby SJ. Nitrite and nitrate regulation at the promoters of two <span>Escherichia</span> coli operons encoding nitrite reductase: Identification of common target heptamers for both <span>NarP</span>- and <span>NarL</span>-dependent regulation. Molecular Microbiology. 1994;13:1045–55.
</div>
<div class="csl-entry">
215. Zennaro E, Ciabatti I, Cutruzzola F, D’Alessandro R, Silvestrini MC. The nitrite reductase gene of <span>Pseudomonas</span> aeruginosa: Effect of growth conditions on the expression and construction of a mutant by gene disruption. FEMS microbiology letters. 1993;109:243–50.
</div>
<div class="csl-entry">
216. Körner H, Zumft WG. Expression of denitrification enzymes in response to the dissolved oxygen level and respiratory substrate in continuous culture of <span>Pseudomonas</span> stutzeri. Applied and Environmental Microbiology. 1989;55:1670–6.
</div>
<div class="csl-entry">
217. Kaiser M, Sawers G. Overlapping promoters modulate <span>Fnr</span>- and <span>ArcA</span>-dependent anaerobic transcriptional activation of the <span class="nocase">focApfl</span> operon in <span>Escherichia</span> coli. Microbiology (Reading, England). 1997;143 ( Pt 3):775–83.
</div>
<div class="csl-entry">
218. Bongaerts J, Zoske S, Weidner U, Unden G. Transcriptional regulation of the proton translocating <span>NADH</span> dehydrogenase genes (<span class="nocase">nuoA</span>-<span>N</span>) of <span>Escherichia</span> coli by electron acceptors, electron donors and gene regulators. Molecular Microbiology. 1995;16:521–34.
</div>
<div class="csl-entry">
219. Cuypers H, Jürgen B, Zumft WG. Multiple <span class="nocase">nosZ</span> promoters and anaerobic expression of nos genes necessary for <span>Pseudomonas</span> stutzeri nitrous oxide reductase and assembly of its copper centers. Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression. 1995;1264:183–90. doi:<a href="https://doi.org/10.1016/0167-4781(95)00128-4">10.1016/0167-4781(95)00128-4</a>.
</div>
<div class="csl-entry">
220. Zumft W. 1995.
</div>
<div class="csl-entry">
221. Kawasaki S, Arai H, Kodama T, Igarashi Y. Gene cluster for dissimilatory nitrite reductase (nir) from <span>Pseudomonas</span> aeruginosa: Sequencing and identification of a locus for heme d1 biosynthesis. Journal of Bacteriology. 1997;179:235–42.
</div>
<div class="csl-entry">
222. Thompson I, Ferguson S, Baker S. Variation within <span>Paracoccus</span> denitrificans and revival of the species <span>Thiosphaera</span> pantotropha as <span>Paracoccus</span> pantotrophus comb. nov. 1997.
</div>
<div class="csl-entry">
223. Robertson LA, Kuenen J. Thiosphaera pantotropha gen. Nov. Sp. Nov., A <span>Facultatively</span> <span>Anaerobic</span>, <span>Facultatively</span> <span>Autotrophic</span> <span>Sulphur</span> <span>Bacterium</span>. Microbiology,. 1983;129:2847–55. doi:<a href="https://doi.org/10.1099/00221287-129-9-2847">10.1099/00221287-129-9-2847</a>.
</div>
<div class="csl-entry">
224. Dohler K, Huss VAR, Zumft WG. Transfer of <span>Pseudomonas</span> perfectomarina <span>Baumann</span>, <span>Bowditch</span>, <span>Baumann</span>, and <span>Beaman</span> 1983 to <span>Pseudomonas</span> stutzeri (<span>Lehmann</span> and <span>Neumann</span> 1896) <span>Sijderius</span> 1946. International Journal of Systematic Bacteriology. 1987;37:1–3. doi:<a href="https://doi.org/10.1099/00207713-37-1-1">10.1099/00207713-37-1-1</a>.
</div>
<div class="csl-entry">
225. Holloway BW, Krishnapillai V, Morgan AF. Chromosomal genetics of <span>Pseudomonas</span>. Microbiological Reviews. 1979;43:73–102.
</div>
<div class="csl-entry">
226. Yanisch-Perron C, Vieira J, Messing J. Improved <span>M13</span> phage cloning vectors and host strains: Nucleotide sequences of the <span>M13mp18</span> and <span class="nocase">pUC19</span> vectors. Gene. 1985;33:103–19.
</div>
<div class="csl-entry">
227. Hanahan D. Studies on transformation of <span>Escherichia</span> coli with plasmids. Journal of Molecular Biology. 1983;166:557–80.
</div>
<div class="csl-entry">
228. Simon R, Priefer U, Pühler A. A <span>Broad</span> <span>Host</span> <span>Range</span> <span>Mobilization</span> <span>System</span> for <span>In</span> <span>Vivo</span> <span>Genetic</span> <span>Engineering</span>: <span>Transposon</span> <span>Mutagenesis</span> in <span>Gram</span> <span>Negative</span> <span>Bacteria</span>. Bio/Technology. 1983;1:784–91. doi:<a href="https://doi.org/10.1038/nbt1183-784">10.1038/nbt1183-784</a>.
</div>
<div class="csl-entry">
229. Vishniac W, Santer M. The thiobacilli. Bacteriological Reviews. 1957;21:195–213.
</div>
<div class="csl-entry">
230. Burnell JN, John P, Whatley FR. The reversibility of active sulphate transport in membrane vesicles of <span>Paracoccus</span> denitrificans. The Biochemical Journal. 1975;150:527–36.
</div>
<div class="csl-entry">
231. Pope NR, Cole JA. Generation of a membrane potential by one of two independent pathways for nitrite reduction by <span>Escherichia</span> coli. Journal of General Microbiology. 1982;128:219–22.
</div>
<div class="csl-entry">
232. Cole JA, Coleman KJ, Compton BE, Kavanagh BM, Keevil CW. Nitrite and ammonia assimilation by anaerobic continuous cultures of <span>Escherichia</span> coli. Journal of General Microbiology. 1974;85:11–22.
</div>
<div class="csl-entry">
233. Boer HA de, Comstock LJ, Vasser M. The tac promoter: A functional hybrid derived from the trp and lac promoters. Proceedings of the National Academy of Sciences of the United States of America. 1983;80:21–5.
</div>
<div class="csl-entry">
234. Scopes RK. Protein <span>Purification</span>: <span>Principles</span> and <span>Practice</span>. 1987.
</div>
<div class="csl-entry">
235. Sambrook J, Fritsch E, Maniatis T. Molecular cloning: <span>A</span> laboratory manual: <span>Vol</span>. 2. 2. ed. S.l.: Cold Spring Harbor; 1989.
</div>
<div class="csl-entry">
236. Goodhew CF, Pettigrew GW, Devreese B, Beeumen jozef, Spanning RJM, Baker SC, et al. The cytochromes <em>c</em> -550 of <em><span>Paracoccus</span> denitrificans</em> and <em><span>Thiosphaera</span> pantotropha</em> : A need for re-evaluation of the history of <em>paracoccus</em> cultures. FEMS Microbiology Letters. 1996;137:95–101. doi:<a href="https://doi.org/10.1111/j.1574-6968.1996.tb08089.x">10.1111/j.1574-6968.1996.tb08089.x</a>.
</div>
<div class="csl-entry">
237. Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Analytical Biochemistry. 1976;72:248–54.
</div>
<div class="csl-entry">
238. Moir J. Aspects of electron transport in <span>Thiosphaera</span> pantotropha and <span>Paracoccus</span> denitrificans. PhD thesis. University of Oxford; 1993. <a href="https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635">https://ethos.bl.uk/OrderDetails.do?uin=uk.bl.ethos.386635</a>.
</div>
<div class="csl-entry">
239. Donald Nicholas DJ, Nason A. Determination of nitrate and nitrite. In: Methods in <span>Enzymology</span>. Elsevier; 1957. p. 981–4. doi:<a href="https://doi.org/10.1016/S0076-6879(57)03489-8">10.1016/S0076-6879(57)03489-8</a>.
</div>
<div class="csl-entry">
240. Birnboim HC, Doly J. A rapid alkaline extraction procedure for screening recombinant plasmid <span>DNA</span>. Nucleic Acids Research. 1979;7:1513–23.
</div>
<div class="csl-entry">
241. Mierendorf RC, Pfeffer D. Direct sequencing of denatured plasmid <span>DNA</span>. Methods in Enzymology. 1987;152:556–62.
</div>
<div class="csl-entry">
242. Cohen SN, Chang AC, Hsu L. Nonchromosomal antibiotic resistance in bacteria: Genetic transformation of <span>Escherichia</span> coli by <span>R</span>-factor <span>DNA</span>. Proceedings of the National Academy of Sciences of the United States of America. 1972;69:2110–4.
</div>
<div class="csl-entry">
243. Clark JM. Novel non-templated nucleotide addition reactions catalyzed by procaryotic and eucaryotic <span>DNA</span> polymerases. Nucleic Acids Research. 1988;16:9677–86.
</div>
<div class="csl-entry">
244. Sanger F, Nicklen S, Coulson AR. <span>DNA</span> sequencing with chain-terminating inhibitors. Proceedings of the National Academy of Sciences of the United States of America. 1977;74:5463–7.
</div>
<div class="csl-entry">
245. Kretz KA, O’Brien JS. Direct sequencing of polymerase chain reaction products from low melting temperature agarose. Methods in Enzymology. 1993;218:72–9.
</div>
<div class="csl-entry">
246. Devereux J, Haeberli P, Smithies O. A comprehensive set of sequence analysis programs for the <span>VAX</span>. Nucleic Acids Research. 1984;12 1 Pt 1:387–95.
</div>
<div class="csl-entry">
247. Chattopadhyay N, Kher R, Godbole M. Inexpensive <span>SDS</span>/phenol method for <span>RNA</span> extraction from tissues. BioTechniques. 1993;15:24–6.
</div>
<div class="csl-entry">
248. Majumdar D, Avissar YJ, Wyche JH. Simultaneous and rapid isolation of bacterial and eukaryotic <span>DNA</span> and <span>RNA</span>: A new approach for isolating <span>DNA</span>. BioTechniques. 1991;11:94–101.
</div>
<div class="csl-entry">
249. Engler-Blum G, Meier M, Frank J, Müller GA. Reduction of background problems in nonradioactive northern and <span>Southern</span> blot analyses enables higher sensitivity than <span>32P</span>-based hybridizations. Analytical Biochemistry. 1993;210:235–44.
</div>
<div class="csl-entry">
250. Frohman MA. Rapid amplification of complementary <span>DNA</span> ends for generation of full-length complementary <span>DNAs</span>: Thermal <span>RACE</span>. Methods in Enzymology. 1993;218:340–56.
</div>
<div class="csl-entry">
251. Weeg-Aerssens E, Wu WS, Ye RW, Tiedje JM, Chang CK. Purification of cytochrome cd1 nitrite reductase from <span>Pseudomonas</span> stutzeri <span>JM300</span> and reconstitution with native and synthetic heme d1. The Journal of Biological Chemistry. 1991;266:7496–502.
</div>
<div class="csl-entry">
252. Hole UH, Vollack K-U, Zumft WG, Eisenmann E, Siddiqui RA, Friedrich B, et al. Characterization of the membranous denitrification enzymes nitrite reductase (cytochrome cd 1 ) and copper-containing nitrous oxide reductase from <span>Thiobacillus</span> denitrificans. Archives of Microbiology. 1996;165:55–61. doi:<a href="https://doi.org/10.1007/s002030050296">10.1007/s002030050296</a>.
</div>
<div class="csl-entry">
253. Heijne G von. A new method for predicting signal sequence cleavage sites. Nucleic Acids Research. 1986;14:4683–90.
</div>
<div class="csl-entry">
254. Nielsen H, Engelbrecht J, Brunak S, Heijne G von. Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. Protein Engineering. 1997;10:1–6.
</div>
<div class="csl-entry">
255. Barton GJ. <span>ALSCRIPT</span>: A tool to format multiple sequence alignments. Protein Engineering. 1993;6:37–40.
</div>
<div class="csl-entry">
256. Appel RD, Bairoch A, Hochstrasser DF. A new generation of information retrieval tools for biologists: The example of the <span>ExPASy</span> <span>WWW</span> server. Trends in Biochemical Sciences. 1994;19:258–60.
</div>
<div class="csl-entry">
257. Steinrücke P, Ludwig B. Genetics of <span>Paracoccus</span> denitrificans. FEMS microbiology reviews. 1993;10:83–117.
</div>
<div class="csl-entry">
258. Roth JR, Lawrence JG, Rubenfield M, Kieffer-Higgins S, Church GM. Characterization of the cobalamin (vitamin <span>B12</span>) biosynthetic genes of <span>Salmonella</span> typhimurium. Journal of Bacteriology. 1993;175:3303–16.
</div>
<div class="csl-entry">
259. Klug G. A <span>DNA</span> sequence upstream of the puf operon of <span>Rhodobacter</span> capsulatus is involved in its oxygen-dependent regulation and functions as a protein binding site. Molecular &amp; general genetics: MGG. 1991;226:167–76.
</div>
<div class="csl-entry">
260. Olsen GJ, Woese CR, Overbeek R. The winds of (evolutionary) change: Breathing new life into microbiology. Journal of Bacteriology. 1994;176:1–6.
</div>
<div class="csl-entry">
261. Fox GE, Wisotzkey JD, Jurtshuk P. How close is close: <span>16S</span> <span class="nocase">rRNA</span> sequence identity may not be sufficient to guarantee species identity. International Journal of Systematic Bacteriology. 1992;42:166–70.
</div>
<div class="csl-entry">
262. Baker SC, Saunders NF, Willis AC, Ferguson SJ, Hajdu J, Fülöp V. Cytochrome cd1 structure: Unusual haem environments in a nitrite reductase and analysis of factors contributing to beta-propeller folds. Journal of Molecular Biology. 1997;269:440–55.
</div>
<div class="csl-entry">
263. Wallace CJ, Clark-Lewis I. Functional role of heme ligation in cytochrome c. <span>Effects</span> of replacement of methionine 80 with natural and non-natural residues by semisynthesis. The Journal of Biological Chemistry. 1992;267:3852–61.
</div>
<div class="csl-entry">
264. Dawson JH, Bracete AM, Huff AM, Kadkhodayan S, Zeitler CM, Sono M, et al. The active site structure of <span>E</span>. Coli <span>HPII</span> catalase. <span>Evidence</span> favoring coordination of a tyrosinate proximal ligand to the chlorin iron. FEBS letters. 1991;295:123–6.
</div>
<div class="csl-entry">
265. Nagai M, Yoneyama Y. Reduction of methemoglobins <span>M</span> <span>Hyde</span> <span>Park</span>, <span>M</span> <span>Saskatoon</span>, and <span>M</span> <span>Milwaukee</span> by ferredoxin and ferredoxin-nicotinamide adenine dinucleotide phosphate reductase system. The Journal of Biological Chemistry. 1983;258:14379–84.
</div>
<div class="csl-entry">
266. Hildebrand DP, Burk DL, Maurus R, Ferrer JC, Brayer GD, Mauk AG. The proximal ligand variant <span>His93Tyr</span> of horse heart myoglobin. Biochemistry. 1995;34:1997–2005.
</div>
<div class="csl-entry">
267. Andersson LA, Johnson AK, Simms MD, Willingham TR. Comparative analysis of catalases: Spectral evidence against heme-bound water for the solution enzymes. FEBS letters. 1995;370:97–100.
</div>
<div class="csl-entry">
268. Silvestrini MC, Cutruzzolà F, D’Alessandro R, Brunori M, Fochesato N, Zennaro E. Expression of <span>Pseudomonas</span> aeruginosa nitrite reductase in <span>Pseudomonas</span> putida and characterization of the recombinant protein. The Biochemical Journal. 1992;285 ( Pt 2):661–6.
</div>
<div class="csl-entry">
269. Iobbi-Nivol C, Crooke H, Griffiths L, Grove J, Hussain H, Pommier J, et al. A reassessment of the range of c-type cytochromes synthesized by <span>Escherichia</span> coli <span>K</span>-12. FEMS microbiology letters. 1994;119:89–94.
</div>
<div class="csl-entry">
270. Hitti YS, Bertino AM. Proteinase <span>K</span> and <span>T4</span> <span>DNA</span> polymerase facilitate the blunt-end subcloning of <span>PCR</span> products. BioTechniques. 1994;16:802–5.
</div>
<div class="csl-entry">
271. Fürste JP, Pansegrau W, Frank R, Blöcker H, Scholz P, Bagdasarian M, et al. Molecular cloning of the plasmid <span>RP4</span> primase region in a multi-host-range <span class="nocase">tacP</span> expression vector. Gene. 1986;48:119–31.
</div>
<div class="csl-entry">
272. Gronenborn B. Overproduction of phage lambda repressor under control of the lac promotor of <span>Escherichia</span> coli. Molecular &amp; general genetics: MGG. 1976;148:243–50.
</div>
<div class="csl-entry">
273. Derbyshire KM, Willetts NS. Mobilization of the non-conjugative plasmid <span>RSF1010</span>: A genetic analysis of its origin of transfer. Molecular &amp; general genetics: MGG. 1987;206:154–60.
</div>
<div class="csl-entry">
274. Darwin AJ, Stewart V. Nitrate and nitrite regulation of the <span>Fnr</span>-dependent aeg-46.5 promoter of <span>Escherichia</span> coli <span>K</span>-12 is mediated by competition between homologous response regulators (<span>NarL</span> and <span>NarP</span>) for a common <span>DNA</span>-binding site. Journal of Molecular Biology. 1995;251:15–29.
</div>
<div class="csl-entry">
275. Thöny-Meyer L, Fischer F, Künzler P, Ritz D, Hennecke H. Escherichia coli genes required for cytochrome c maturation. Journal of Bacteriology. 1995;177:4321–6.
</div>
<div class="csl-entry">
276. Darie S, Gunsalus RP. Effect of heme and oxygen availability on <span class="nocase">hemA</span> gene expression in <span>Escherichia</span> coli: Role of the fnr, <span class="nocase">arcA</span>, and <span class="nocase">himA</span> gene products. Journal of Bacteriology. 1994;176:5270–6.
</div>
<div class="csl-entry">
277. Woodard SI, Dailey HA. Regulation of heme biosynthesis in <span>Escherichia</span> coli. Archives of Biochemistry and Biophysics. 1995;316:110–5.
</div>
<div class="csl-entry">
278. Page MD, Ferguson SJ. Apo forms of cytochrome c550 and cytochrome cd1 are translocated to the periplasm of <span>Paracoccus</span> denitrificans in the absence of haem incorporation caused either mutation or inhibition of haem synthesis. Molecular Microbiology. 1990;4:1181–92.
</div>
<div class="csl-entry">
279. Grisshammer R, Oeckl C, Michel H. Expression in <span>Escherichia</span> coli of c-type cytochrome genes from <span>Rhodopseudomonas</span> viridis. Biochimica Et Biophysica Acta. 1991;1088:183–90.
</div>
<div class="csl-entry">
280. Sambongi Y, Stoll R, Ferguson SJ. Alteration of haem-attachment and signal-cleavage sites for <span>Paracoccus</span> denitrificans cytochrome <span>C550</span> probes pathway of c-type cytochrome biogenesis in <span>Escherichia</span> coli. Molecular Microbiology. 1996;19:1193–204.
</div>
<div class="csl-entry">
281. Pollock WB, Voordouw G. Aerobic expression of the cyf gene encoding cytochrome c-553 from <span>Desulfovibrio</span> vulgaris <span>Hildenborough</span> in <span>Escherichia</span> coli. Microbiology (Reading, England). 1994;140 ( Pt 4):879–87.
</div>
<div class="csl-entry">
282. Fu R, Wall JD, Voordouw G. <span>DcrA</span>, a c-type heme-containing methyl-accepting protein from <span>Desulfovibrio</span> vulgaris <span>Hildenborough</span>, senses the oxygen concentration or redox potential of the environment. Journal of Bacteriology. 1994;176:344–50.
</div>
<div class="csl-entry">
283. Ubbink M, Van Beeumen J, Canters GW. Cytochrome c550 from <span>Thiobacillus</span> versutus: Cloning, expression in <span>Escherichia</span> coli, and purification of the heterologous holoprotein. Journal of Bacteriology. 1992;174:3707–14.
</div>
<div class="csl-entry">
284. Sambongi Y, Ferguson SJ. Synthesis of holo <span>Paracoccus</span> denitrificans cytochrome c550 requires targeting to the periplasm whereas that of holo <span>Hydrogenobacter</span> thermophilus cytochrome c552 does not. <span>Implications</span> for c-type cytochrome biogenesis. FEBS letters. 1994;340:65–70.
</div>
<div class="csl-entry">
285. Garrett RM, Rajagopalan KV. Molecular cloning of rat liver sulfite oxidase. <span>Expression</span> of a eukaryotic <span>Mo</span>-pterin-containing enzyme in <span>Escherichia</span> coli. The Journal of Biological Chemistry. 1994;269:272–6.
</div>
<div class="csl-entry">
286. Vidakovic MS, Fraczkiewicz G, Germanas JP. Expression and spectroscopic characterization of the hydrogenosomal [<span>2Fe</span>-<span>2S</span>] ferredoxin from the protozoan <span>Trichomonas</span> vaginalis. The Journal of Biological Chemistry. 1996;271:14734–9.
</div>
<div class="csl-entry">
287. Leung YC, Chan C, Reader JS, Willis AC, Spanning RJ van, Ferguson SJ, et al. The pseudoazurin gene from <span>Thiosphaera</span> pantotropha: Analysis of upstream putative regulatory sequences and overexpression in <span>Escherichia</span> coli. The Biochemical Journal. 1997;321 ( Pt 3):699–705.
</div>
<div class="csl-entry">
288. Viebrock A, Zumft WG. Molecular cloning, heterologous expression, and primary structure of the structural gene for the copper enzyme nitrous oxide reductase from denitrifying <span>Pseudomonas</span> stutzeri. Journal of Bacteriology. 1988;170:4658–68.
</div>
<div class="csl-entry">
289. Darwin A, Hussain H, Griffiths L, Grove J, Sambongi Y, Busby S, et al. Regulation and sequence of the structural gene for cytochrome c552 from <span>Escherichia</span> coli: Not a hexahaem but a 50 <span class="nocase">kDa</span> tetrahaem nitrite reductase. Molecular Microbiology. 1993;9:1255–65.
</div>
<div class="csl-entry">
290. Barber D, Parr SR, Greenwood C. Some spectral and steady-state kinetic properties of <span>Pseudomonas</span> cytochrome oxidase. The Biochemical Journal. 1976;157:431–8.
</div>
<div class="csl-entry">
291. Parr SR, Wilson MT, Greenwood C. The reaction of <span>Pseudomonas</span> aeurginosa cytochrome c oxidase with sodium metabisulphite. The Biochemical Journal. 1974;139:273–6.
</div>
<div class="csl-entry">
292. Yamamoto K, Uozumi T, Beppu T. The blue copper protein gene of <span>Alcaligenes</span> faecalis <span>S</span>-6 directs secretion of blue copper protein from <span>Escherichia</span> coli cells. Journal of Bacteriology. 1987;169:5648–52.
</div>
<div class="csl-entry">
293. Studier FW, Rosenberg AH, Dunn JJ, Dubendorff JW. Use of <span>T7</span> <span>RNA</span> polymerase to direct expression of cloned genes. Methods in Enzymology. 1990;185:60–89.
</div>
<div class="csl-entry">
294. Horio T, Higashi T, Yamanaka T, Matsubara H, Okunuki K. Purification and properties of cytochrome oxidase from <span>Pseudomonas</span> aeruginosa. The Journal of Biological Chemistry. 1961;236:944–51.
</div>
<div class="csl-entry">
295. Rothmel RK, Chakrabarty AM, Berry A, Darzins A. Genetic systems in <span>Pseudomonas</span>. Methods in Enzymology. 1991;204:485–514.
</div>
<div class="csl-entry">
296. Hull HH, Wharton DC. Isoelectrophoretic characterization of <span>Pseudomonas</span> cytochrome oxidase/nitrite reductase and its heme d1-containing domain. Archives of Biochemistry and Biophysics. 1993;301:85–90.
</div>
<div class="csl-entry">
297. Sawers RG. Identification and molecular characterization of a transcriptional regulator from <span>Pseudomonas</span> aeruginosa <span>PAO1</span> exhibiting structural and functional similarity to the <span>FNR</span> protein of <span>Escherichia</span> coli. Molecular Microbiology. 1991;5:1469–81.
</div>
<div class="csl-entry">
298. Frey J, Bagdasarian MM, Bagdasarian M. Replication and copy number control of the broad-host-range plasmid <span>RSF1010</span>. Gene. 1992;113:101–6.
</div>
<div class="csl-entry">
299. Unden G, Duchene A. On the role of cyclic <span>AMP</span> and the <span>Fnr</span> protein in <span>Escherichia</span> coli growing anaerobically. Archives of Microbiology. 1987;147:195–200.
</div>
<div class="csl-entry">
300. West SE, Iglewski BH. Codon usage in <span>Pseudomonas</span> aeruginosa. Nucleic Acids Research. 1988;16:9323–35.
</div>
<div class="csl-entry">
301. Fujita M, Amemura A. In vitro interactions of <span>Pseudomonas</span> <span>RNA</span> polymerases with tac and <span>RNA</span> <span>I</span> promoters. Bioscience, Biotechnology, and Biochemistry. 1992;56:1644–8.
</div>
<div class="csl-entry">
302. Brunschwig E, Darzins A. A two-component <span>T7</span> system for the overexpression of genes in <span>Pseudomonas</span> aeruginosa. Gene. 1992;111:35–41.
</div>
<div class="csl-entry">
303. Ward BB, Cockcroft AR, Kilpatrick KA. Antibody and <span>DNA</span> probes for detection of nitrite reductase in seawater. Journal of General Microbiology. 1993;139:2285–93.
</div>
<div class="csl-entry">
304. Wülfing C, Plückthun A. Correctly folded <span>T</span>-cell receptor fragments in the periplasm of <span>Escherichia</span> coli. <span>Influence</span> of folding catalysts. Journal of Molecular Biology. 1994;242:655–69.
</div>
<div class="csl-entry">
305. Wong EY, Seetharam R, Kotts CE, Heeren RA, Klein BK, Braford SR, et al. Expression of secreted insulin-like growth factor-1 in <span>Escherichia</span> coli. Gene. 1988;68:193–203.
</div>
<div class="csl-entry">
306. Kortt AA, Guthrie RE, Hinds MG, Power BE, Ivancic N, Caldwell JB, et al. Solution properties of <span>Escherichia</span> coli-expressed <span>VH</span> domain of anti-neuraminidase antibody <span>NC41</span>. Journal of Protein Chemistry. 1995;14:167–78.
</div>
<div class="csl-entry">
307. Cheng KJ, Ingram JM, Costerton JW. Alkaline phosphatase localization and spheroplast formation of <span>Pseudomonas</span> aeruginosa. Canadian Journal of Microbiology. 1970;16:1319–24.
</div>
<div class="csl-entry">
308. Matsushita K, Yamada M, Shinagawa E, Adachi O, Ameyama M. Membrane-bound respiratory chain of <span>Pseudomonas</span> aeruginosa grown aerobically. Journal of Bacteriology. 1980;141:389–92.
</div>
<div class="csl-entry">
309. Fujiwara T, Fukumori Y, Yamanaka T. A novel terminal oxidase, cytochrome baa3 purified from aerobically grown <span>Pseudomonas</span> aeruginosa: It shows a clear difference between resting state and pulsed state. Journal of Biochemistry. 1992;112:290–8.
</div>
<div class="csl-entry">
310. Spanning R van. Genes involved in respiration of <span>Paracoccus</span> denitrificans. PhD thesis. VU Amsterdam; 1991. <a href="https://research.vu.nl/en/publications/genes-involved-in-respiration-of-paracoccus-denitrificans">https://research.vu.nl/en/publications/genes-involved-in-respiration-of-paracoccus-denitrificans</a>.
</div>
<div class="csl-entry">
311. Spanning RJ van, Wansell CW, Reijnders WN, Oltmann LF, Stouthamer AH. Mutagenesis of the gene encoding amicyanin of <span>Paracoccus</span> denitrificans and the resultant effect on methylamine oxidation. FEBS letters. 1990;275:217–20.
</div>
<div class="csl-entry">
312. Van Spanning RJ, Wansell CW, Reijnders WN, Harms N, Ras J, Oltmann LF, et al. A method for introduction of unmarked mutations in the genome of <span>Paracoccus</span> denitrificans: Construction of strains with multiple mutations in the genes encoding periplasmic cytochromes c550, c551i, and c553i. Journal of Bacteriology. 1991;173:6962–70.
</div>
<div class="csl-entry">
313. Vieira J, Messing J. The <span class="nocase">pUC</span> plasmids, an <span>M13mp7</span>-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene. 1982;19:259–68.
</div>
<div class="csl-entry">
314. Gilmour R, Goodhew CF, Pettigrew GW, Prazeres S, Moura I, Moura JJ. Spectroscopic characterization of cytochrome c peroxidase from <span>Paracoccus</span> denitrificans. The Biochemical Journal. 1993;294 ( Pt 3):745–52.
</div>
<div class="csl-entry">
315. Oost J van der, Schepper M, Stouthamer AH, Westerhoff HV, Spanning RJ van, Gier JW de. Reversed electron transfer through the bc1 complex enables a cytochrome c oxidase mutant (delta aa3/cbb3) of <span>Paracoccus</span> denitrificans to grow on methylamine. FEBS letters. 1995;371:267–70.
</div>
<div class="csl-entry">
316. Van Spanning RJ, Wansell CW, De Boer T, Hazelaar MJ, Anazawa H, Harms N, et al. Isolation and characterization of the <span class="nocase">moxJ</span>, <span class="nocase">moxG</span>, <span class="nocase">moxI</span>, and <span class="nocase">moxR</span> genes of <span>Paracoccus</span> denitrificans: Inactivation of <span class="nocase">moxJ</span>, <span class="nocase">moxG</span>, and <span class="nocase">moxR</span> and the resultant effect on methylotrophic growth. Journal of Bacteriology. 1991;173:6948–61.
</div>
<div class="csl-entry">
317. Roufa D, Rhoads D. <span>SeqAid</span> <span>II</span>. 1989.
</div>
<div class="csl-entry">
318. Ronnberg M, Araiso T, Ellfolk N, Dunford HB. The reaction between reduced azurin and oxidized cytochrome c peroxidase from <span>Pseudomonas</span> aeruginosa. The Journal of Biological Chemistry. 1981;256:2471–4.
</div>
<div class="csl-entry">
319. Palmedo G, Seither P, Körner H, Matthews JC, Burkhalter RS, Timkovich R, et al. Resolution of the <span class="nocase">nirD</span> locus for heme d1 synthesis of cytochrome cd1 (respiratory nitrite reductase) from <span>Pseudomonas</span> stutzeri. European Journal of Biochemistry. 1995;232:737–46.
</div>
<div class="csl-entry">
320. Zumft WG, Braun C, Cuypers H. Nitric oxide reductase from <span>Pseudomonas</span> stutzeri. <span>Primary</span> structure and gene organization of a novel bacterial cytochrome bc complex. European Journal of Biochemistry. 1994;219:481–90.
</div>
<div class="csl-entry">
321. Bartnikas TB, Tosques IE, Laratta WP, Shi J, Shapleigh JP. Characterization of the nitric oxide reductase-encoding region in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1997;179:3534–40.
</div>
<div class="csl-entry">
322. Tosques IE, Kwiatkowski AV, Shi J, Shapleigh JP. Characterization and regulation of the gene encoding nitrite reductase in <span>Rhodobacter</span> sphaeroides 2.4.3. Journal of Bacteriology. 1997;179:1090–5.
</div>
<div class="csl-entry">
323. Chirgwin JM, Przybyla AE, MacDonald RJ, Rutter WJ. Isolation of biologically active ribonucleic acid from sources enriched in ribonuclease. Biochemistry. 1979;18:5294–9.
</div>
<div class="csl-entry">
324. Griersen D. In: Gel <span>Electrophoresis</span> of <span>Nucleic</span> <span>Acids</span> <span>A</span> <span>Practical</span> <span>Approach</span> (<span>Second</span> <span>Edition</span>). 1991. p. 102. <a href="https://linkinghub.elsevier.com/retrieve/pii/0307441291900416">https://linkinghub.elsevier.com/retrieve/pii/0307441291900416</a>. Accessed 29 Apr 2021.
</div>
<div class="csl-entry">
325. Brosius J, Dull TJ, Noller HF. Complete nucleotide sequence of a <span>23S</span> ribosomal <span>RNA</span> gene from <span>Escherichia</span> coli. Proceedings of the National Academy of Sciences of the United States of America. 1980;77:201–4.
</div>
<div class="csl-entry">
326. Brosius J, Palmer ML, Kennedy PJ, Noller HF. Complete nucleotide sequence of a <span>16S</span> ribosomal <span>RNA</span> gene from <span>Escherichia</span> coli. Proceedings of the National Academy of Sciences of the United States of America. 1978;75:4801–5.
</div>
<div class="csl-entry">
327. Runar Ra S, Saris PE. Characterization of prokaryotic <span class="nocase">mRNAs</span> by <span>RT</span>-<span>PCR</span>. BioTechniques. 1995;18:792–5.
</div>
<div class="csl-entry">
328. Yu P-L, Hohn B, Falk H, Drews G. Molecular cloning of the ribosomal <span>RNA</span> genes of the photosynthetic bacterium <span>Rhodopseudomonas</span> capsulata. Molecular and General Genetics MGG. 1982;188:392–8. doi:<a href="https://doi.org/10.1007/BF00330039">10.1007/BF00330039</a>.
</div>
<div class="csl-entry">
329. Chow CT. Properties of ribonucleic acids from photosynthetic and heterotrophic <span>Rhodospirillum</span> rubrum. Canadian Journal of Microbiology. 1976;22:228–36.
</div>
<div class="csl-entry">
330. Marrs B, Kaplan S. 23 s precursor ribosomal <span>RNA</span> of <span>Rhodopseudomonas</span> spheroides. Journal of Molecular Biology. 1970;49:297–317.
</div>
<div class="csl-entry">
331. Grienenberger JM, Simon D. Structure and biosynthesis of the ribosomal ribonucleic acids from the oncogenic bacterium <span>Agrobacterium</span> tumefaciens. The Biochemical Journal. 1975;149:23–30.
</div>
<div class="csl-entry">
332. Farnham PJ, Platt T. A model for transcription termination suggested by studies on the trp attenuator in vitro using base analogs. Cell. 1980;20:739–48.
</div>
<div class="csl-entry">
333. Melton DA, Krieg PA, Rebagliati MR, Maniatis T, Zinn K, Green MR. Efficient in vitro synthesis of biologically active <span>RNA</span> and <span>RNA</span> hybridization probes from plasmids containing a bacteriophage <span>SP6</span> promoter. Nucleic Acids Research. 1984;12:7035–56.
</div>
<div class="csl-entry">
334. Hawley DK, McClure WR. Compilation and analysis of <span>Escherichia</span> coli promoter <span>DNA</span> sequences. Nucleic Acids Research. 1983;11:2237–55.
</div>
<div class="csl-entry">
335. Eiglmeier K, Honoré N, Iuchi S, Lin EC, Cole ST. Molecular genetic analysis of <span>FNR</span>-dependent promoters. Molecular Microbiology. 1989;3:869–78.
</div>
<div class="csl-entry">
336. Siebenlist U, Simpson RB, Gilbert W. E. Coli <span>RNA</span> polymerase interacts homologously with two different promoters. Cell. 1980;20:269–81.
</div>
<div class="csl-entry">
337. Baker SC, Ferguson SJ, Ludwig B, Page MD, Richter OM, Spanning RJ van. Molecular genetics of the genus <span>Paracoccus</span>: Metabolically versatile bacteria with bioenergetic flexibility. Microbiology and molecular biology reviews: MMBR. 1998;62:1046–78.
</div>
<div class="csl-entry">
338. Frey J, Mudd EA, Krisch HM. A bacteriophage <span>T4</span> expression cassette that functions efficiently in a wide range of gram-negative bacteria. Gene. 1988;62:237–47.
</div>
<div class="csl-entry">
339. Gao JG, Gussin GN. <span>RNA</span> polymerases from <span>Pseudomonas</span> aeruginosa and <span>Pseudomonas</span> syringae respond to <span>Escherichia</span> coli activator proteins. Journal of Bacteriology. 1991;173:394–7.
</div>
<div class="csl-entry">
340. Schmitt MP, Holmes RK. Cloning, sequence, and footprint analysis of two promoter/operators from <span>Corynebacterium</span> diphtheriae that are regulated by the diphtheria toxin repressor (<span>DtxR</span>) and iron. Journal of Bacteriology. 1994;176:1141–9. doi:<a href="https://doi.org/10.1128/JB.176.4.1141-1149.1994">10.1128/JB.176.4.1141-1149.1994</a>.
</div>
<div class="csl-entry">
341. Beynon J, Cannon M, Buchanan-Wollaston V, Cannon F. The nif promoters of <span>Klebsiella</span> pneumoniae have a characteristic primary structure. Cell. 1983;34:665–71.
</div>
<div class="csl-entry">
342. Römermann D, Warrelmann J, Bender RA, Friedrich B. An <span class="nocase">rpoN</span>-like gene of <span>Alcaligenes</span> eutrophus and <span>Pseudomonas</span> facilis controls expression of diverse metabolic pathways, including hydrogen oxidation. Journal of Bacteriology. 1989;171:1093–9.
</div>
<div class="csl-entry">
343. Kustu S, Santero E, Keener J, Popham D, Weiss D. Expression of sigma 54 (<span class="nocase">ntrA</span>)-dependent genes is probably united by a common mechanism. Microbiological Reviews. 1989;53:367–76.
</div>
<div class="csl-entry">
344. Thöny B, Hennecke H. The -24/-12 promoter comes of age. FEMS Microbiological Reviews. 1989;63:341–58.
</div>
<div class="csl-entry">
345. Cuypers H, Zumft W. In: Pseudomonas: <span>Molecular</span> <span>Biology</span> and <span>Biotechnology</span>. American Society for Microbiology; 1992. p. 188–97.
</div>
<div class="csl-entry">
346. Totten PA, Lara JC, Lory S. The <span class="nocase">rpoN</span> gene product of <span>Pseudomonas</span> aeruginosa is required for expression of diverse genes, including the flagellin gene. Journal of Bacteriology. 1990;172:389–96.
</div>
<div class="csl-entry">
347. Roemermann D, Lohmeyer M, Friedrich CG, Friedrich B. Pleotropic mutants from <span>Alcaligenes</span> eutrophus defective in the metabolism of hydrogen, nitrate, urea, and fumarate. Archives of Microbiology. 1988;149:471–5. doi:<a href="https://doi.org/10.1007/BF00425590">10.1007/BF00425590</a>.
</div>
<div class="csl-entry">
348. Savioz A, Zimmermann A, Haas D. Pseudomonas aeruginosa promoters which contain a conserved <span>GG</span>-<span>N10</span>-<span>GC</span> motif but appear to be <span>RpoN</span>-independent. Molecular &amp; general genetics: MGG. 1993;238:74–80.
</div>
<div class="csl-entry">
349. Kullik I, Fritsche S, Knobel H, Sanjuan J, Hennecke H, Fischer HM. Bradyrhizobium japonicum has two differentially regulated, functional homologs of the sigma 54 gene (<span class="nocase">rpoN</span>). Journal of Bacteriology. 1991;173:1125–38.
</div>
<div class="csl-entry">
350. Van Hove B, Staudenmaier H, Braun V. Novel two-component transmembrane transcription control: Regulation of iron dicitrate transport in <span>Escherichia</span> coli <span>K</span>-12. Journal of Bacteriology. 1990;172:6749–58.
</div>
<div class="csl-entry">
351. Miller VL, Taylor RK, Mekalanos JJ. Cholera toxin transcriptional activator <span class="nocase">toxR</span> is a transmembrane <span>DNA</span> binding protein. Cell. 1987;48:271–9.
</div>
<div class="csl-entry">
352. Cutruzzolà F, Arese M, Grasso S, Bellelli A, Brunori M. Mutagenesis of nitrite reductase from <span>Pseudomonas</span> aeruginosa: Tyrosine-10 in the c heme domain is not involved in catalysis. FEBS letters. 1997;412:365–9.
</div>
<div class="csl-entry">
353. Ghosh M, Anthony C, Harlos K, Goodwin MG, Blake C. The refined structure of the quinoprotein methanol dehydrogenase from <span>Methylobacterium</span> extorquens at 1.94 <span>A</span>. Structure (London, England: 1993). 1995;3:177–87.
</div>
<div class="csl-entry">
354. Xia Z, Dai W, Zhang Y, White SA, Boyd GD, Mathews FS. Determination of the gene sequence and the three-dimensional structure at 2.4 angstroms resolution of methanol dehydrogenase from <span>Methylophilus</span> <span>W3A1</span>. Journal of Molecular Biology. 1996;259:480–501.
</div>
<div class="csl-entry">
355. Ito N, Phillips SE, Stevens C, Ogel ZB, McPherson MJ, Keen JN, et al. Novel thioether bond revealed by a 1.7 <span>A</span> crystal structure of galactose oxidase. Nature. 1991;350:87–90.
</div>
<div class="csl-entry">
356. Sondek J, Bohm A, Lambright DG, Hamm HE, Sigler PB. Crystal structure of a <span>G</span>-protein beta gamma dimer at 2.<span>1A</span> resolution. Nature. 1996;379:369–74.
</div>
<div class="csl-entry">
357. Gaskell A, Crennell S, Taylor G. The three domains of a bacterial sialidase: A beta-propeller, an immunoglobulin module and a galactose-binding jelly-roll. Structure (London, England: 1993). 1995;3:1197–205.
</div>
<div class="csl-entry">
358. Faber HR, Groom CR, Baker HM, Morgan WT, Smith A, Baker EN. 1.8 <span>A</span> crystal structure of the <span>C</span>-terminal domain of rabbit serum haemopexin. Structure (London, England: 1993). 1995;3:551–9.
</div>
<div class="csl-entry">
359. Li J, Brick P, O’Hare MC, Skarzynski T, Lloyd LF, Curry VA, et al. Structure of full-length porcine synovial collagenase reveals a <span>C</span>-terminal domain containing a calcium-linked, four-bladed beta-propeller. Structure (London, England: 1993). 1995;3:541–9.
</div>
<div class="csl-entry">
360. Russell RB, Barton GJ. Multiple protein sequence alignment from tertiary structure comparison: Assignment of global and residue confidence levels. Proteins. 1992;14:309–23.
</div>
<div class="csl-entry">
361. Cozier GE, Giles IG, Anthony C. The structure of the quinoprotein alcohol dehydrogenase of <span>Acetobacter</span> aceti modelled on that of methanol dehydrogenase from <span>Methylobacterium</span> extorquens. The Biochemical Journal. 1995;308 ( Pt 2):375–9.
</div>
<div class="csl-entry">
362. Garcia-Higuera I, Fenoglio J, Li Y, Lewis C, Panchenko MP, Reiner O, et al. Folding of proteins with <span>WD</span>-repeats: Comparison of six members of the <span>WD</span>-repeat superfamily to the <span>G</span> protein beta subunit. Biochemistry. 1996;35:13985–94.
</div>
<div class="csl-entry">
363. Bork P, Doolittle RF. Drosophila kelch motif is derived from a common enzyme fold. Journal of Molecular Biology. 1994;236:1277–82.
</div>
<div class="csl-entry">
364. Murzin AG. Structural principles for the propeller assembly of beta-sheets: The preference for seven-fold symmetry. Proteins. 1992;14:191–201.
</div>
<div class="csl-entry">
365. Ludwig W, Mittenhuber G, Friedrich CG. Transfer of <span>Thiosphaera</span> pantotropha to <span>Paracoccus</span> denitrificans. International Journal of Systematic Bacteriology. 1993;43:363–7.
</div>
<div class="csl-entry">
366. Beijerinck M, Minkman D. Bildung und verbrauch von stickoxydul durch bakterien. Zentralbl Bakteriol II. 1910;25:30–63.
</div>
<div class="csl-entry">
367. Rainey PB, Thompson IP, Palleroni NJ. Genome and fatty acid analysis of <span>Pseudomonas</span> stutzeri. International Journal of Systematic Bacteriology. 1994;44:54–61.
</div>
<div class="csl-entry">
368. Thompson IP, Bailey MJ, Ellis RJ, Purdy KJ. Subgrouping of bacterial populations by cellular fatty acid composition. FEMS Microbiology Letters. 1993;102:75–84. doi:<a href="https://doi.org/10.1111/j.1574-6968.1993.tb05798.x">10.1111/j.1574-6968.1993.tb05798.x</a>.
</div>
<div class="csl-entry">
369. Katayama Y, Hiraishi A, Kuraishi H. Paracoccus thiocyanatus sp. Nov., A new species of thiocyanate-utilizing facultative chemolithotroph, and transfer of <span>Thiobacillus</span> versutus to the genus <span>Paracoccus</span> as <span>Paracoccus</span> versutus comb. Nov. With emendation of the genus. Microbiology (Reading, England). 1995;141 ( Pt 6):1469–77.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="B-classification-of-t-pantotropha.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
